{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Arxiv Research Paper Dataset Generation Projection\n",
    "\n",
    "TEAM MEMBERS\n",
    "\n",
    "- Naren Garapati (ng658@drexel.edu)\n",
    "- Milan Varghese(mv644@drexel.edu)\n",
    "- Daniel Lai (dl3285@drexel.edu)\n",
    "- Rutvij Hiteshkumar Upadhyay (ru45@drexel.edu)​"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "import bs4\n",
    "from bs4 import BeautifulSoup\n",
    "import csv\n",
    "import os \n",
    "import time\n",
    "import io\n",
    "import PyPDF2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO: Not scanning for jupyter notebooks.\n",
      "<unknown>:17: SyntaxWarning: invalid escape sequence '\\d'\n",
      "<unknown>:44: SyntaxWarning: invalid escape sequence '\\d'\n",
      "WARNING: Import named \"beautifulsoup4\" not found locally. Trying to resolve it at the PyPI server.\n",
      "WARNING: Import named \"beautifulsoup4\" was resolved to \"beautifulsoup4:4.12.3\" package (https://pypi.org/project/beautifulsoup4/).\n",
      "Please, verify manually the final list of requirements.txt to avoid possible dependency confusions.\n",
      "WARNING: Import named \"Requests\" not found locally. Trying to resolve it at the PyPI server.\n",
      "WARNING: Import named \"Requests\" was resolved to \"requests:2.32.3\" package (https://pypi.org/project/requests/).\n",
      "Please, verify manually the final list of requirements.txt to avoid possible dependency confusions.\n",
      "WARNING: Import named \"twython\" not found locally. Trying to resolve it at the PyPI server.\n",
      "WARNING: Import named \"twython\" was resolved to \"twython:3.9.1\" package (https://pypi.org/project/twython/).\n",
      "Please, verify manually the final list of requirements.txt to avoid possible dependency confusions.\n",
      "INFO: Successfully saved requirements file in /Users/narengarapati/Desktop/requirements.txt\n"
     ]
    }
   ],
   "source": [
    "!pipreqs ~/Desktop/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: arxivscraper in /opt/anaconda3/lib/python3.12/site-packages (0.0.5)\n"
     ]
    }
   ],
   "source": [
    "!pip3 install arxivscraper"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import arxivscraper\n",
    "scraper = arxivscraper.Scraper(category='q-fin', date_from='2024-01-01',date_until='2024-12-03')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "fetching up to  1000 records...\n",
      "Got 503. Retrying after 30 seconds.\n",
      "fetching up to  1000 records...\n",
      "Got 503. Retrying after 30 seconds.\n",
      "fetching up to  1000 records...\n",
      "fetching up to  2000 records...\n",
      "Got 503. Retrying after 30 seconds.\n",
      "fetching up to  2000 records...\n",
      "fetching up to  3000 records...\n",
      "Got 503. Retrying after 30 seconds.\n",
      "fetching up to  3000 records...\n",
      "fetching up to  4000 records...\n",
      "Got 503. Retrying after 30 seconds.\n",
      "fetching up to  4000 records...\n",
      "fetching up to  5000 records...\n",
      "Got 503. Retrying after 30 seconds.\n",
      "fetching up to  5000 records...\n",
      "fetching up to  6000 records...\n",
      "Got 503. Retrying after 30 seconds.\n",
      "fetching up to  6000 records...\n",
      "fetching is completed in 318.9 seconds.\n",
      "Total number of records 5254\n"
     ]
    }
   ],
   "source": [
    "output = scraper.scrape()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'title': 'thermodynamic constraints on neural dimensions, firing rates, brain   temperature and size',\n",
       "  'id': '0905.3690',\n",
       "  'abstract': 'there have been suggestions that heat caused by cerebral metabolic activity may constrain mammalian brain evolution, architecture, and function. this article investigates physical limits on brain wiring and corresponding changes in brain temperature that are imposed by thermodynamics of heat balance determined mainly by na$^{+}$/k$^{+}$-atpase, cerebral blood flow, and heat conduction. it is found that even moderate firing rates cause significant intracellular na$^{+}$ build-up, and the atp consumption rate associated with pumping out these ions grows nonlinearly with frequency. surprisingly, the power dissipated by the na$^{+}$/k$^{+}$ pump depends biphasically on frequency, which can lead to the biphasic dependence of brain temperature on frequency as well. both the total power of sodium pumps and brain temperature diverge for very small fiber diameters, indicating that too thin fibers are not beneficial for thermal balance. for very small brains blood flow is not a sufficient cooling mechanism deep in the brain. the theoretical lower bound on fiber diameter above which brain temperature is in the operational regime is strongly frequency dependent but finite due to synaptic depression. for normal neurophysiological conditions this bound is at least an order of magnitude smaller than average values of empirical fiber diameters, suggesting that neuroanatomy of the mammalian brains operates in the thermodynamically safe regime. analytical formulas presented can be used to estimate average firing rates in mammals, and relate their changes to changes in brain temperature, which can have important practical applications. in general, activity in larger brains is found to be slower than in smaller brains.',\n",
       "  'categories': 'q-bio.nc q-bio.cb q-bio.sc',\n",
       "  'doi': '10.1007/s10827-009-0153-7',\n",
       "  'created': '2009-05-22',\n",
       "  'updated': '',\n",
       "  'authors': ['jan karbowski'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/0905.3690'},\n",
       " {'title': 'category theoretic analysis of hierarchical protein materials and social   networks',\n",
       "  'id': '1103.2273',\n",
       "  'abstract': 'materials in biology span all the scales from angstroms to meters and typically consist of complex hierarchical assemblies of simple building blocks. here we describe an application of category theory to describe structural and resulting functional properties of biological protein materials by developing so-called ologs. an olog is like a \"concept web\" or \"semantic network\" except that it follows a rigorous mathematical formulation based on category theory. this key difference ensures that an olog is unambiguous, highly adaptable to evolution and change, and suitable for sharing concepts with other olog. we consider simple cases of alpha-helical and amyloid-like protein filaments subjected to axial extension and develop an olog representation of their structural and resulting mechanical properties. we also construct a representation of a social network in which people send text-messages to their nearest neighbors and act as a team to perform a task. we show that the olog for the protein and the olog for the social network feature identical category-theoretic representations, and we proceed to precisely explicate the analogy or isomorphism between them. the examples presented here demonstrate that the intrinsic nature of a complex system, which in particular includes a precise relationship between structure and function at different hierarchical levels, can be effectively represented by an olog. this, in turn, allows for comparative studies between disparate materials or fields of application, and results in novel approaches to derive functionality in the design of de novo hierarchical systems. we discuss opportunities and challenges associated with the description of complex biological materials by using ologs as a powerful tool for analysis and design in the context of materiomics, and we present the potential impact of this approach for engineering, life sciences, and medicine.',\n",
       "  'categories': 'math.ct cond-mat.mes-hall cond-mat.mtrl-sci cond-mat.soft q-bio.mn',\n",
       "  'doi': '10.1371/journal.pone.0023911',\n",
       "  'created': '2011-03-11',\n",
       "  'updated': '2011-07-10',\n",
       "  'authors': ['david i. spivak',\n",
       "   'tristan giesa',\n",
       "   'elizabeth wood',\n",
       "   'markus j. buehler'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1103.2273'},\n",
       " {'title': 'the fundamental theorem of phyllotaxis revisited',\n",
       "  'id': '1201.1641',\n",
       "  'abstract': \"jean's `fundamental theorem of phyllotaxis' (\\\\emph{phyllotaxis: a systematic study in plant morphogenesis}, cup 1994) describes the relationship between the count numbers of observed spirals in cylindrical lattices and the horizontal angle between vertically successive spots in the lattice. it is indeed fundamental to observational studies of phyllotactic counts, and especially to the evaluation of hypotheses about the origin of fibonacci structure within lattices. unfortunately the textbook version of the theorem is incomplete in that it is incorrect for an important special case. this paper provides a complete statement and proof of the theorem.\",\n",
       "  'categories': 'q-bio.to',\n",
       "  'doi': '',\n",
       "  'created': '2012-01-08',\n",
       "  'updated': '2024-07-09',\n",
       "  'authors': ['jonathan swinton'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1201.1641'},\n",
       " {'title': 'boolean nested canalizing functions: a comprehensive analysis',\n",
       "  'id': '1204.5203',\n",
       "  'abstract': 'boolean network models of molecular regulatory networks have been used successfully in computational systems biology. the boolean functions that appear in published models tend to have special properties, in particular the property of being nested canalizing, a concept inspired by the concept of canalization in evolutionary biology. it has been shown that networks comprised of nested canalizing functions have dynamic properties that make them suitable for modeling molecular regulatory networks, namely a small number of (large) attractors, as well as relatively short limit cycles.   this paper contains a detailed analysis of this class of functions, based on a novel normal form as polynomial functions over the boolean field. the concept of layer is introduced that stratifies variables into different classes depending on their level of dominance. using this layer concept a closed form formula is derived for the number of nested canalizing functions with a given number of variables. additional metrics considered include hamming weight, the activity number of any variable, and the average sensitivity of the function. it is also shown that the average sensitivity of any nested canalizing function is between 0 and 2. this provides a rationale for why nested canalizing functions are stable, since a random boolean function in n variables has average sensitivity n/2. the paper also contains experimental evidence that the layer number is an important factor in network stability.',\n",
       "  'categories': 'math.ds q-bio.mn',\n",
       "  'doi': '10.1016/j.tcs.2013.02.020',\n",
       "  'created': '2012-04-23',\n",
       "  'updated': '2013-01-17',\n",
       "  'authors': ['yuan li',\n",
       "   'john o. adeyeye',\n",
       "   'david murrugarra',\n",
       "   'boris aguilar',\n",
       "   'reinhard laubenbacher'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1204.5203'},\n",
       " {'title': 'perfect taxon sampling and fixing taxon traceability: introducing a   class of phylogenetically decisive taxon sets',\n",
       "  'id': '1206.3472',\n",
       "  'abstract': 'phylogenetically decisive sets of taxon sets have the property that if trees are chosen for each of their elements, as long as these trees are compatible, the resulting supertree is unique. this means that as long as the trees describing the phylogenetic relationships of the (input) species sets are compatible, they can only be combined into a common supertree in precisely one way. this setting is sometimes also referred to as \\\\enquote{perfect taxon sampling}. while for rooted trees, the decision if a given set of input taxon sets is phylogenetically decisive can be made in polynomial time, the decision problem to determine whether a set of taxon sets is phylogenetically decisive concerning \\\\emph{unrooted} trees is unfortunately conp-complete and therefore in practice hard to solve for large instances. this shows that recognizing such sets is often difficult. in this paper, we explain phylogenetic decisiveness and introduce a class of input taxon sets, namely so-called \\\\emph{fixing taxon traceable} sets, which are guaranteed to be phylogenetically decisive and which can be recognized in polynomial time. using both combinatorial approaches as well as simulations, we compare properties of fixing taxon traceability and phylogenetic decisiveness, e.g., by deriving lower and upper bounds for the number of quadruple sets (i.e., sets of 4-tuples) needed in the input set for each of these properties. in particular, we correct an erroneous lower bound concerning phylogenetic decisiveness from the literature.   we have implemented the algorithm to determine if a given set of taxon sets is fixing taxon traceable in \\\\textsf{r} and made our software package \\\\verb+fixingtaxontracer+ publicly available.',\n",
       "  'categories': 'q-bio.pe math.co',\n",
       "  'doi': '',\n",
       "  'created': '2012-06-15',\n",
       "  'updated': '2024-08-19',\n",
       "  'authors': ['mareike fischer', 'janne pott'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1206.3472'},\n",
       " {'title': 'development of hydrogen bonding magnetic reaction-based gene regulation   through cyclic electromagnetic dna simulation in double-stranded dna',\n",
       "  'id': '1210.7091',\n",
       "  'abstract': 'the proton-magnetic reaction is commonly used in mri machines with a strong magnetic field of over 1 t, while this study hypothesized that the electron magnetic reaction of hydrogen could affect the hydrogen bonds of double-stranded dna (dsdna) at a low magnetic field below 0.01 t. the goal is to develop a hydrogen bonding magnetic reaction-based gene regulation (hbmr-gr) system. the polarities of dna base pairs are derived from the relative electrostatic charge between purines and pyrimidines, which become positively and negatively charged, respectively. the pyu dsdnas with pyrimidine(s)-purine(s) sequences, ds3t3a, ds3c3g, and ds3c3a, showed stronger dna hybridization potential, increased infrared absorption at 3400-3200 cm-1, and a unique dna conformation in hplc analysis compared to the corresponding puy dsdnas. to target the three-dimensional structure of dsdna based on the dna base pair polarities, one can use cyclic electromagnetic dna simulation (ceds) with approximately 25% efficiency for randomly oriented dsdnas. ceds was found to induce sequence-specific hybridization of target oligo-dsdnas in 0.005m nacl solution and sequence-specific conformation of oligo-dsdnas in 0.1m nacl solution. it was found that the pyu oligo-dsdnas were more responsible for the hybridization and conformational changes by ceds than the puy oligo-dsdnas. ceds decreased ethidium bromide (etbr) dna intercalation and spermidine dna condensation depending on ceds time in the binding assay. the results also included that the pyu oligo-dsdnas were more responsible for ceds by forming stable and unique conformation of oligo-dsdna than the therefore, it is postulated that the low-level hbmr-based ceds can enhance the hybridization potential of oligo-dsdnas and subsequently lead to the unique dna conformation required for the initiation of various dna functions.',\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '',\n",
       "  'created': '2012-10-26',\n",
       "  'updated': '2024-06-29',\n",
       "  'authors': ['yeon sook kim', 'dae gwan lee', 'suk keun lee'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1210.7091'},\n",
       " {'title': 'sensitivity analysis of circadian entrainment in the space of phase   response curves',\n",
       "  'id': '1211.7317',\n",
       "  'abstract': 'sensitivity analysis is a classical and fundamental tool to evaluate the role of a given parameter in a given system characteristic. because the phase response curve is a fundamental input--output characteristic of oscillators, we developed a sensitivity analysis for oscillator models in the space of phase response curves. the proposed tool can be applied to high-dimensional oscillator models without facing the curse of dimensionality obstacle associated with numerical exploration of the parameter space. application of this tool to a state-of-the-art model of circadian rhythms suggests that it can be useful and instrumental to biological investigations.',\n",
       "  'categories': 'math.ds q-bio.qm',\n",
       "  'doi': '10.1007/978-94-017-9047-5_3',\n",
       "  'created': '2012-11-30',\n",
       "  'updated': '2013-01-24',\n",
       "  'authors': ['pierre sacré', 'rodolphe sepulchre'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1211.7317'},\n",
       " {'title': 'prior biological knowledge and epigenetic information enhances   prediction accuracy of bayesian wnt pathway',\n",
       "  'id': '1307.4296',\n",
       "  'abstract': 'computational modeling of wnt signaling pathway has gained prominence for its use as computer aided diagnostic tool to develop therapeutic cancer target drugs and predict of test samples as cancerous and non cancerous. this manuscript focuses on development of simple static bayesian network models of varying complexity that encompasses prior partially available biological knowledge about intra and extra cellular factors affecting the wnt pathway and incorporates epigenetic information like methylation and histone modification of a few genes known to have inhibitory affect on wnt pathway. it might be expected that such models not only increase cancer prediction accuracies and also form basis for understanding wnt signaling activity in different states of tumorigenesis. initial results in human colorectal cancer cases indicate that incorporation of epigenetic information increases prediction accuracy of test samples as being tumorous or normal. receiver operator curves (roc) and their respective area under the curve (auc) measurements, obtained from predictions of state of test sample and corresponding predictions of the state of activation of transcription complex of the wnt pathway for the test sample, indicate that there is significant difference between the wnt pathway being on (off) and its association with the sample being tumorous (normal). two sample kolmogorov-smirnov test confirm the statistical deviation between the distributions of these predictions. at a preliminary stage, use of these models may help in understanding the yet unknown effect of certain factors like dkk2, dkk3-1 and sfrp-2/3/5 on {\\\\beta}-catenin transcription complex.',\n",
       "  'categories': 'q-bio.mn cs.ce',\n",
       "  'doi': '',\n",
       "  'created': '2013-07-15',\n",
       "  'updated': '2024-11-25',\n",
       "  'authors': ['shriprakash sinha', 'marcel j. t. reinders', 'wim verhaegh'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1307.4296'},\n",
       " {'title': 'estimation of erythrocyte surface area in mammals',\n",
       "  'id': '1403.7660',\n",
       "  'abstract': 'measures of erythrocytes volume and surface are helpful in several physiological studies, both for zoologists and veterinarians. whilst diameter and volume are assessed with ease from observations of blood smears and complete blood count, respectively, thickness and surface area, instead, are much more difficult to be obtained. the accurate description of the erythrocyte geometry is given by the equation of the oval of cassini, but the formulas deriving from it are very complex, comprising elliptic integrals. in this article three solids are proposed as models approximating the erythrocyte: sphere, cylinder and a spheroid with concave caps. volumes and surface areas obtained with these models are compared to those effectively measured.',\n",
       "  'categories': 'q-bio.qm q-bio.to',\n",
       "  'doi': '10.3390/mps7010011',\n",
       "  'created': '2014-03-29',\n",
       "  'updated': '',\n",
       "  'authors': ['ion udroiu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1403.7660'},\n",
       " {'title': 'one file to share them all: using the combine archive and the omex   format to share all information about a modeling project',\n",
       "  'id': '1407.4992',\n",
       "  'abstract': 'background: with the ever increasing use of computational models in the biosciences, the need to share models and reproduce the results of published studies efficiently and easily is becoming more important. to this end, various standards have been proposed that can be used to describe models, simulations, data or other essential information in a consistent fashion. these constitute various separate components required to reproduce a given published scientific result.   results: we describe the open modeling exchange format (omex). together with the use of other standard formats from the computational modeling in biology network (combine), omex is the basis of the combine archive, a single file that supports the exchange of all the information necessary for a modeling and simulation experiment in biology. an omex file is a zip container that includes a manifest file, listing the content of the archive, an optional metadata file adding information about the archive and its content, and the files describing the model. the content of a combine archive consists of files encoded in combine standards whenever possible, but may include additional files defined by an internet media type. several tools that support the combine archive are available, either as independent libraries or embedded in modeling software.   conclusions: the combine archive facilitates the reproduction of modeling and simulation experiments in biology by embedding all the relevant information in one file. having all the information stored and exchanged at once also helps in building activity logs and audit trails. we anticipate that the combine archive will become a significant help for modellers, as the domain moves to larger, more complex experiments such as multi-scale models of organs, digital organisms, and bioengineering.',\n",
       "  'categories': 'cs.dl q-bio.mn',\n",
       "  'doi': '10.1186/s12859-014-0369-z',\n",
       "  'created': '2014-07-18',\n",
       "  'updated': '2014-09-30',\n",
       "  'authors': ['frank t. bergmann',\n",
       "   'richard adams',\n",
       "   'stuart moodie',\n",
       "   'jonathan cooper',\n",
       "   'mihai glont',\n",
       "   'martin golebiewski',\n",
       "   'michael hucka',\n",
       "   'camille laibe',\n",
       "   'andrew k. miller',\n",
       "   'david p. nickerson',\n",
       "   'brett g. olivier',\n",
       "   'nicolas rodriguez',\n",
       "   'herbert m. sauro',\n",
       "   'martin scharm',\n",
       "   'stian soiland-reyes',\n",
       "   'dagmar waltemath',\n",
       "   'florent yvon',\n",
       "   'nicolas le novère'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1407.4992'},\n",
       " {'title': 'optimizing hybrid spreading in metapopulations',\n",
       "  'id': '1409.7291',\n",
       "  'abstract': 'epidemic spreading phenomena are ubiquitous in nature and society. examples include the spreading of diseases, information, and computer viruses. epidemics can spread by local spreading, where infected nodes can only infect a limited set of direct target nodes and global spreading, where an infected node can infect every other node. in reality, many epidemics spread using a hybrid mixture of both types of spreading. in this study we develop a theoretical framework for studying hybrid epidemics, and examine the optimum balance between spreading mechanisms in terms of achieving the maximum outbreak size. we show the existence of critically hybrid epidemics where neither spreading mechanism alone can cause a noticeable spread but a combination of the two spreading mechanisms would produce an enormous outbreak. our results provide new strategies for maximising beneficial epidemics and estimating the worst outcome of damaging hybrid epidemics.',\n",
       "  'categories': 'physics.soc-ph cs.ai cs.si q-bio.pe',\n",
       "  'doi': '10.1038/srep09924',\n",
       "  'created': '2014-09-25',\n",
       "  'updated': '2015-03-31',\n",
       "  'authors': ['changwang zhang',\n",
       "   'shi zhou',\n",
       "   'joel c. miller',\n",
       "   'ingemar j. cox',\n",
       "   'benjamin m. chain'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1409.7291'},\n",
       " {'title': 'calculating complexity of large randomized libraries',\n",
       "  'id': '1410.5851',\n",
       "  'abstract': 'randomized libraries are increasingly popular in protein engineering and other biomedical research fields. statistics of the libraries are useful to guide and evaluate randomized library construction. previous works only give the mean of the number of unique sequences in the library, and they can only handle equal molar ratio of the four nucleotides at a small number of mutation sites. we derive formulas to calculate the mean and variance of the number of unique sequences in libraries generated by cassette mutagenesis with mixtures of arbitrary nucleotide ratios. computer program was developed which utilizes arbitrary numerical precision software package to calculate the statistics of large libraries. the statistics of library with mutations in more than $20$ amino acids can be calculated easily. results show that the nucleotide ratios have significant effects on these statistics. the more skewed the ratio, the larger the library size is needed to obtain the same expected number of unique sequences. the program is freely available at \\\\url{http://graphics.med.yale.edu/cgi-bin/lib_comp.pl}.',\n",
       "  'categories': 'q-bio.qm stat.co',\n",
       "  'doi': '10.1016/j.jtbi.2009.04.008',\n",
       "  'created': '2014-10-21',\n",
       "  'updated': '',\n",
       "  'authors': ['yong kong'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1410.5851'},\n",
       " {'title': 'motor skill learning by increasing the movement planning horizon',\n",
       "  'id': '1410.6049',\n",
       "  'abstract': 'we investigated motor skill learning using a path tracking task, where human subjects had to track various curved paths as fast as possible, in the absence of any external perturbations. subjects became better with practice, producing faster and smoother movements even when tracking novel untrained paths. using a \"searchlight\" paradigm, where only a short segment of the path ahead of the cursor was shown, we found that subjects with a higher tracking skill took a longer chunk of the future path into account when computing the control policy for the upcoming movement segment. we observed the same effects in a second experiment where tracking speed was fixed and subjects were practicing to increase their accuracy. these findings demonstrate that human subjects increase their planning horizon when acquiring a motor skill.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1152/jn.00631.2020',\n",
       "  'created': '2014-10-22',\n",
       "  'updated': '2015-10-17',\n",
       "  'authors': ['luke bashford', 'dmitry kobak', 'carsten mehring'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1410.6049'},\n",
       " {'title': 'btrim: a fast, lightweight adapter and quality trimming program for   next-generation sequencing technologies',\n",
       "  'id': '1410.6455',\n",
       "  'abstract': \"btrim is a fast and lightweight software to trim adapters and low quality regions in reads from ultra high-throughput next-generation sequencing machines. it also can reliably identify barcodes and assign the reads to the original samples. based on a modified myers's bit-vector dynamic programming algorithm, btrim can handle indels in adapters and barcodes. it removes low quality regions and trims off adapters at both or either end of the reads. a typical trimming of 30m reads with two sets of adapter pairs can be done in about a minute with a small memory footprint. btrim is a versatile stand-alone tool that can be used as the first step in virtually all next-generation sequence analysis pipelines. the program is available at \\\\url{http://graphics.med.yale.edu/trim/}.\",\n",
       "  'categories': 'q-bio.gn cs.ce cs.ds',\n",
       "  'doi': '10.1016/j.ygeno.2011.05.009',\n",
       "  'created': '2014-10-23',\n",
       "  'updated': '',\n",
       "  'authors': ['yong kong'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1410.6455'},\n",
       " {'title': 'statistical distributions of sequencing by synthesis with probabilistic   nucleotide incorporation',\n",
       "  'id': '1411.2547',\n",
       "  'abstract': 'sequencing by synthesis is used in many next-generation dna sequencing technologies. some of the technologies, especially those exploring the principle of single-molecule sequencing, allow incomplete nucleotide incorporation in each cycle. we derive statistical distributions for sequencing by synthesis by taking into account the possibility that nucleotide incorporation may not be complete in each flow cycle. the statistical distributions are expressed in terms of nucleotide probabilities of the target sequences and the nucleotide incorporation probabilities for each nucleotide. we give exact distributions both for fixed number of flow cycles and for fixed sequence length. explicit formulas are derived for the mean and variance of these distributions. the results are generalizations of our previous work for pyrosequencing. incomplete nucleotide incorporation leads to significant change in the mean and variance of the distributions, but still they can be approximated by normal distributions with the same mean and variance. the results are also generalized to handle sequence context dependent incorporation. the statistical distributions will be useful for instrument and software development for sequencing by synthesis platforms.',\n",
       "  'categories': 'q-bio.gn cs.dm math.co stat.co',\n",
       "  'doi': '10.1089/cmb.2008.0215',\n",
       "  'created': '2014-11-05',\n",
       "  'updated': '',\n",
       "  'authors': ['yong kong'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1411.2547'},\n",
       " {'title': 'length distribution of sequencing by synthesis: fixed flow cycle model',\n",
       "  'id': '1411.2548',\n",
       "  'abstract': 'sequencing by synthesis is the underlying technology for many next-generation dna sequencing platforms. we developed a new model, the fixed flow cycle model, to derive the distributions of sequence length for a given number of flow cycles under the general conditions where the nucleotide incorporation is probabilistic and may be incomplete, as in some single-molecule sequencing technologies. unlike the previous model, the new model yields the probability distribution for the sequence length. explicit closed form formulas are derived for the mean and variance of the distribution.',\n",
       "  'categories': 'q-bio.gn cs.dm math.co stat.co',\n",
       "  'doi': '10.1007/s00285-012-0556-3',\n",
       "  'created': '2014-11-05',\n",
       "  'updated': '',\n",
       "  'authors': ['yong kong'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1411.2548'},\n",
       " {'title': 'distributions of positive signals in pyrosequencing',\n",
       "  'id': '1411.2549',\n",
       "  'abstract': 'pyrosequencing is one of the important next-generation sequencing technologies. we derive the distribution of the number of positive signals in pyrograms of this sequencing technology as a function of flow cycle numbers and nucleotide probabilities of the target sequences. as for the distribution of sequence length, we also derive the distribution of positive signals for the fixed flow cycle model. explicit formulas are derived for the mean and variance of the distributions. a simple result for the mean of the distribution is that the mean number of positive signals in a pyrogram is approximately twice the number of flow cycles, regardless of nucleotide probabilities. the statistical distributions will be useful for instrument and software development for pyrosequencing and other related platforms.',\n",
       "  'categories': 'q-bio.gn cs.dm math.co stat.co',\n",
       "  'doi': '10.1007/s00285-013-0691-5',\n",
       "  'created': '2014-11-05',\n",
       "  'updated': '',\n",
       "  'authors': ['yong kong'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1411.2549'},\n",
       " {'title': 'statistical distributions of pyrosequencing',\n",
       "  'id': '1411.2668',\n",
       "  'abstract': 'pyrosequencing is emerging as one of the important next-generation sequencing technologies. we derive the statistical distributions of this technique in terms of nucleotide probabilities of the target sequences. we give exact distributions both for fixed number of flow cycles and for fixed sequence length. explicit formulas are derived for the mean and variance of these distributions. in both cases, the distributions can be approximated accurately by normal distributions with the same mean and variance. the statistical distributions will be useful for instrument and software development for pyrosequencing platforms.',\n",
       "  'categories': 'q-bio.gn cs.dm math.co stat.co',\n",
       "  'doi': '10.1089/cmb.2008.0106',\n",
       "  'created': '2014-11-05',\n",
       "  'updated': '',\n",
       "  'authors': ['yong kong'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1411.2668'},\n",
       " {'title': 'energy cost and optimisation in breath-hold diving',\n",
       "  'id': '1503.02904',\n",
       "  'abstract': 'we present a new model for calculating locomotion costs in breath-hold divers. starting from basic mechanics principles, we calculate the work that the diver must provide through propulsion to counterbalance the action of drag, the buoyant force and weight during immersion. compared to those in previous studies, the model presented here accurately analyses breath-hold divers which alternate active swimming with prolonged glides during the dive (as is the case in mammals). the energy cost of the dive is strongly dependent on these prolonged gliding phases. here we investigate the length and impacts on energy cost of these glides with respect to the diver characteristics, and compare them with those observed in different breath-hold diving species. taking into account the basal metabolic rate and chemical energy to propulsion transformation efficiency, we calculate optimal swim velocity and the corresponding total energy cost (including metabolic rate) and compare them with observations. energy cost is minimised when the diver passes through neutral buoyancy conditions during the dive. this generally implies the presence of prolonged gliding phases in both ascent and descent, where the buoyancy (varying with depth) is best used against the drag, reducing energy cost. this is in agreement with past results (miller et al., 2012; sato et al., 2013) where, when the buoyant force is considered constant during the dive, the energy cost was minimised for neutral buoyancy. in particular, our model confirms the good physical adaption of dolphins for diving, compared to other breath-hold diving species which are mostly positively buoyant (penguins for example). the presence of prolonged glides implies a non-trivial dependency of optimal speed on maximal depth of the dive. this extends previous findings (sato et al., 2010; watanabe et al., 2011) which found no dependency of optimal speed on dive depth for particular conditions. the energy cost of the dive can be further diminished by reducing the volume of gas-filled body parts in divers close to neutral buoyancy. this provides a possible additional explanation for the observed exhalation of air before diving in phocid seals to minimise dive energy cost. until now the only explanation for this phenomenon has been a reduction in the risk of decompression sickness.',\n",
       "  'categories': 'q-bio.qm physics.bio-ph physics.flu-dyn',\n",
       "  'doi': '10.1016/j.jtbi.2016.02.009',\n",
       "  'created': '2015-03-10',\n",
       "  'updated': '2016-01-19',\n",
       "  'authors': ['martino trassinelli'],\n",
       "  'affiliation': ['insp'],\n",
       "  'url': 'https://arxiv.org/abs/1503.02904'},\n",
       " {'title': 'multiland: a neutral landscape generator designed for theoretical   studies',\n",
       "  'id': '1503.07215',\n",
       "  'abstract': 'the main goal of multiland is to generate neutral landscapes made of several types of regions, with an exact control of the proportions occupied by each type of region. an important feature of the software is that it allows a control of the landscape fragmentation. it is intended to theoretical studies on the effect of landscape structure in applied sciences. it has been developed in the framework of the peerless anr project \"predictive ecological engineering for landscape ecosystem services and sustainability\". it includes both matlab and python versions.',\n",
       "  'categories': 'q-bio.pe q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2015-03-24',\n",
       "  'updated': '2024-11-11',\n",
       "  'authors': ['lionel roques'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1503.07215'},\n",
       " {'title': 'hybrid spreading mechanisms and t cell activation shape the dynamics of   hiv-1 infection',\n",
       "  'id': '1503.08992',\n",
       "  'abstract': \"hiv-1 can disseminate between susceptible cells by two mechanisms: cell-free infection following fluid-phase diffusion of virions and by highly-efficient direct cell-to-cell transmission at immune cell contacts. the contribution of this hybrid spreading mechanism, which is also a characteristic of some important computer worm outbreaks, to hiv-1 progression in vivo remains unknown. here we present a new mathematical model that explicitly incorporates the ability of hiv-1 to use hybrid spreading mechanisms and evaluate the consequences for hiv-1 pathogenenesis. the model captures the major phases of the hiv-1 infection course of a cohort of treatment naive patients and also accurately predicts the results of the short pulse anti-retroviral therapy at seroconversion (spartac) trial. using this model we find that hybrid spreading is critical to seed and establish infection, and that cell-to-cell spread and increased cd4+ t cell activation are important for hiv-1 progression. notably, the model predicts that cell-to-cell spread becomes increasingly effective as infection progresses and thus may present a considerable treatment barrier. deriving predictions of various treatments' influence on hiv-1 progression highlights the importance of earlier intervention and suggests that treatments effectively targeting cell-to-cell hiv-1 spread can delay progression to aids. this study suggests that hybrid spreading is a fundamental feature of hiv infection, and provides the mathematical framework incorporating this feature with which to evaluate future therapeutic strategies.\",\n",
       "  'categories': 'q-bio.pe cs.ai cs.ce physics.bio-ph q-bio.cb',\n",
       "  'doi': '10.1371/journal.pcbi.1004179',\n",
       "  'created': '2015-03-31',\n",
       "  'updated': '',\n",
       "  'authors': ['changwang zhang',\n",
       "   'shi zhou',\n",
       "   'elisabetta groppelli',\n",
       "   'pierre pellegrino',\n",
       "   'ian williams',\n",
       "   'persephone borrow',\n",
       "   'benjamin m. chain',\n",
       "   'clare jolly'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1503.08992'},\n",
       " {'title': 'electrical neurostimulation for chronic pain: on selective relay of   sensory neural activities in myelinated nerve fibers',\n",
       "  'id': '1507.02716',\n",
       "  'abstract': 'chronic pain affects about 100 million adults in the us. despite their great need, neuropharmacology and neurostimulation therapies for chronic pain have been associated with suboptimal efficacy and limited long-term success, as their mechanisms of action are unclear. yet current computational models of pain transmission suffer from several limitations. in particular, dorsal column models do not include the fundamental underlying sensory activity traveling in these nerve fibers. we developed a (simple) simulation test bed of electrical neurostimulation of myelinated nerve fibers with underlying sensory activity. this paper reports our findings so far. interactions between stimulation-evoked and underlying activities are mainly due to collisions of action potentials and losses of excitability due to the refractory period following an action potential. in addition, intuitively, the reliability of sensory activity decreases as the stimulation frequency increases. this first step opens the door to a better understanding of pain transmission and its modulation by neurostimulation therapies.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1109/embc.2015.7319444',\n",
       "  'created': '2015-07-09',\n",
       "  'updated': '',\n",
       "  'authors': ['pierre sacré',\n",
       "   'sridevi v. sarma',\n",
       "   'yun guan',\n",
       "   'william s. anderson'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1507.02716'},\n",
       " {'title': 'molecular network control through boolean canalization',\n",
       "  'id': '1508.03026',\n",
       "  'abstract': 'boolean networks are an important class of computational models for molecular interaction networks. boolean canalization, a type of hierarchical clustering of the inputs of a boolean function, has been extensively studied in the context of network modeling where each layer of canalization adds a degree of stability in the dynamics of the network. recently, dynamic network control approaches have been used for the design of new therapeutic interventions and for other applications such as stem cell reprogramming. this work studies the role of canalization in the control of boolean molecular networks. it provides a method for identifying the potential edges to control in the wiring diagram of a network for avoiding undesirable state transitions. the method is based on identifying appropriate input-output combinations on undesirable transitions that can be modified using the edges in the wiring diagram of the network. moreover, a method for estimating the number of changed transitions in the state space of the system as a result of an edge deletion in the wiring diagram is presented. the control methods of this paper were applied to a mutated cell-cycle model and to a p53-mdm2 model to identify potential control targets.',\n",
       "  'categories': 'q-bio.mn',\n",
       "  'doi': '10.1186/s13637-015-0029-2',\n",
       "  'created': '2015-08-12',\n",
       "  'updated': '2015-10-24',\n",
       "  'authors': ['david murrugarra', 'elena s. dimitrova'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1508.03026'},\n",
       " {'title': 'a sequential approach to calibrate ecosystem models with multiple time   series data',\n",
       "  'id': '1509.06123',\n",
       "  'abstract': 'ecosystem approach to fisheries requires a thorough understanding of fishing impacts on ecosystem status and processes as well as predictive tools such as ecosystem models to provide useful information for management. the credibility of such models is essential when used as decision making tools, and model fitting to observed data is one major criterion to assess such credibility. however, more attention has been given to the exploration of model behavior than to a rigorous confrontation to observations, as the calibration of ecosystem models is challenging in many ways. first, ecosystem models can only be simulated numerically and are generally too complex for mathematical analysis and explicit parameter estimation; secondly, the complex dynamics represented in ecosystem models allow species-specific parameters to impact other species parameters through ecological interactions; thirdly, critical data about non-commercial species are often poor; lastly, technical aspects can be impediments to the calibration with regard to the high computational cost potentially involved and the scarce documentation published on fitting complex ecosystem models to data. this work highlights some issues related to the confrontation of complex ecosystem models to data and proposes a methodology for a sequential multi-phases calibration of ecosystem models. we propose criteria to classify the parameters of a model: model dependency and time variability of the parameters. these criteria and the availability of approximate initial estimates are used as decision rules to determine which parameters need to be estimated, and their precedence order in the sequential calibration process. the end-to-end ecosystem model roms-pisces-osmose applied to the northern humboldt current ecosystem is used as an illustrative case study.',\n",
       "  'categories': 'q-bio.qm q-bio.pe stat.me',\n",
       "  'doi': '10.1016/j.pocean.2017.01.002',\n",
       "  'created': '2015-09-21',\n",
       "  'updated': '',\n",
       "  'authors': ['ricardo oliveros-ramos', 'philippe verley', 'yunne-jai shin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1509.06123'},\n",
       " {'title': 'pathway tools version 28.0: integrated software for pathway/genome   informatics and systems biology',\n",
       "  'id': '1510.03964',\n",
       "  'abstract': 'pathway tools is a bioinformatics software environment with a broad set of capabilities. the software provides genome-informatics tools such as a genome browser, sequence alignments, a genome-variant analyzer, and comparative-genomics operations. it offers metabolic-informatics tools, such as metabolic reconstruction, quantitative metabolic modeling, prediction of reaction atom mappings, and metabolic route search. pathway tools also provides regulatory-informatics tools, such as the ability to represent and visualize a wide range of regulatory interactions. the software creates and manages a type of organism-specific database called a pathway/genome database (pgdb), which the software enables database curators to interactively edit. it supports web publishing of pgdbs and provides a large number of query, visualization, and omics-data analysis tools. scientists around the world have created more than 45,000 pgdbs by using pathway tools, many of which are curated databases for important model organisms. those pgdbs can be exchanged using a peer-to-peer database-sharing system called the pgdb registry.',\n",
       "  'categories': 'q-bio.gn q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2015-10-14',\n",
       "  'updated': '2024-09-18',\n",
       "  'authors': ['peter d. karp',\n",
       "   'suzanne m. paley',\n",
       "   'markus krummenacker',\n",
       "   'anamika kothari',\n",
       "   'peter e. midford',\n",
       "   'pallavi subhraveti',\n",
       "   'austin swart',\n",
       "   'lisa moore',\n",
       "   'ron caspi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1510.03964'},\n",
       " {'title': 'local complexity predicts global synchronization of hierarchically   networked oscillators',\n",
       "  'id': '1512.01922',\n",
       "  'abstract': 'we study the global synchronization of hierarchically-organized stuart-landau oscillators, where each subsystem consists of three oscillators with activity-dependent couplings. we consider all possible coupling signs between the three oscillators, and find that they can generate different numbers of phase attractors depending on the network motif. here, the subsystems are coupled through mean activities of total oscillators. under weak inter-subsystem couplings, we demonstrate that the synchronization between subsystems is highly correlated with the number of attractors in uncoupled subsystems. among the network motifs, perfect anti-symmetric ones are unique to generate both single and multiple attractors depending on the activities of oscillators. the flexible local complexity can make global synchronization controllable.',\n",
       "  'categories': 'nlin.ao q-bio.nc',\n",
       "  'doi': '10.1063/1.4995961',\n",
       "  'created': '2015-12-07',\n",
       "  'updated': '2024-02-12',\n",
       "  'authors': ['jin xu', 'dong-ho park', 'junghyo jo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1512.01922'},\n",
       " {'title': 'perspectives on theory at the interface of physics and biology',\n",
       "  'id': '1512.08954',\n",
       "  'abstract': \"theoretical physics is the search for simple and universal mathematical descriptions of the natural world. in contrast, much of modern biology is an exploration of the complexity and diversity of life. for many, this contrast is prima facie evidence that theory, in the sense that physicists use the word, is impossible in a biological context. for others, this contrast serves to highlight a grand challenge. i'm an optimist, and believe (along with many colleagues) that the time is ripe for the emergence of a more unified theoretical physics of biological systems, building on successes in thinking about particular phenomena. in this essay i try to explain the reasons for my optimism, through a combination of historical and modern examples.\",\n",
       "  'categories': 'physics.bio-ph cond-mat.dis-nn physics.hist-ph q-bio.qm',\n",
       "  'doi': '10.1088/1361-6633/aa995b',\n",
       "  'created': '2015-12-30',\n",
       "  'updated': '',\n",
       "  'authors': ['william bialek'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1512.08954'},\n",
       " {'title': 'the unfitted discontinuous galerkin method for solving the eeg forward   problem',\n",
       "  'id': '1601.07810',\n",
       "  'abstract': 'objective: the purpose of this study is to introduce and evaluate the unfitted discontinuous galerkin finite element method (udg-fem) for solving the electroencephalography (eeg) forward problem. methods: this new approach for source analysis does not use a geometry conforming volume triangulation, but instead uses a structured mesh that does not resolve the geometry. the geometry is described using level set functions and is incorporated implicitly in its mathematical formulation. as no triangulation is necessary, the complexity of a simulation pipeline and the need for manual interaction for patient specific simulations can be reduced and is comparable with that of the fem for hexahedral meshes. in addition, it maintains conservation laws on a discrete level. here, we present the theory for udg-fem forward modeling, its verification using quasi-analytical solutions in multi-layer sphere models and an evaluation in a comparison with a discontinuous galerkin (dg-fem) method on hexahedral and on conforming tetrahedral meshes. we furthermore apply the udg-fem forward approach in a realistic head model simulation study. results: the given results show convergence and indicate a good overall accuracy of the udg-fem approach. udg-fem performs comparable or even better than dg-fem on a conforming tetrahedral mesh while providing a less complex simulation pipeline. when compared to dg-fem on hexahedral meshes, an overall better accuracy is achieved. conclusion: the udg-fem approach is an accurate, flexible and promising method to solve the eeg forward problem. significance: this study shows the first application of the udg-fem approach to the eeg forward problem.',\n",
       "  'categories': 'cs.ce cs.na math.na q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2016-01-28',\n",
       "  'updated': '2016-05-02',\n",
       "  'authors': ['andreas nüßing',\n",
       "   'carsten h. wolters',\n",
       "   'heinrich brinck',\n",
       "   'christian engwer'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1601.07810'},\n",
       " {'title': 'winning versus losing during gambling and its neural correlates',\n",
       "  'id': '1602.03493',\n",
       "  'abstract': 'humans often make decisions which maximize an internal utility function. for example, humans often maximize their expected reward when gambling and this is considered as a \"rational\" decision. however, humans tend to change their betting strategies depending on how they \"feel\". if someone has experienced a losing streak, they may \"feel\" that they are more likely to win on the next hand even though the odds of the game have not changed. that is, their decisions are driven by their emotional state. in this paper, we investigate how the human brain responds to wins and losses during gambling. using a combination of local field potential recordings in human subjects performing a financial decision-making task, spectral analyses, and non-parametric cluster statistics, we investigated whether neural responses in different cognitive and limbic brain areas differ between wins and losses after decisions are made. in eleven subjects, the neural activity modulated significantly between win and loss trials in one brain region: the anterior insula ($p=0.01$). in particular, gamma activity (30-70 hz) increased in the anterior insula when subjects just realized that they won. modulation of metabolic activity in the anterior insula has been observed previously in functional magnetic resonance imaging studies during decision making and when emotions are elicited. however, our study is able to characterize temporal dynamics of electrical activity in this brain region at the millisecond resolution while decisions are made and after outcomes are revealed.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1109/ciss.2016.7460563',\n",
       "  'created': '2016-02-10',\n",
       "  'updated': '',\n",
       "  'authors': ['pierre sacré',\n",
       "   'matthew s. d. kerr',\n",
       "   'sandya subramanian',\n",
       "   'kevin kahn',\n",
       "   'jorge gonzalez-martinez',\n",
       "   'matthew a. johnson',\n",
       "   'john t. gale',\n",
       "   'sridevi v. sarma'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1602.03493'},\n",
       " {'title': 'calibrar: an r package for fitting complex ecological models',\n",
       "  'id': '1603.03141',\n",
       "  'abstract': \"the fitting or parameter estimation of complex ecological models is a challenging optimisation task, with a notable lack of tools for fitting complex, long runtime or stochastic models. calibrar is an r package that is dedicated to the fitting of complex models to data. it is a generic tool that can be used for any type of model, especially those with non-differentiable objective functions and long runtime, including individual based models. calibrar supports multiple phases and constrained optimisation, includes 18 optimisation algorithms, including derivative-based and heuristic ones. it supports any type of parallelization, the restart of interrupted optimisations for long runtime models and the combination of different optimisation methods during the multiple phases of the calibration. user-level expertise in r is necessary to handle calibration experiments with calibrar, but there is no need to modify the model's code, which can be programmed in any language. it implements maximum likelihood estimation methods and automated construction of the objective function from simulated model outputs. for more experienced users, calibrar allows the implementation of user-defined objective functions. the package source code is fully accessible and can be installed directly from cran.\",\n",
       "  'categories': 'q-bio.qm math.oc stat.co',\n",
       "  'doi': '',\n",
       "  'created': '2016-03-09',\n",
       "  'updated': '2024-04-27',\n",
       "  'authors': ['ricardo oliveros-ramos', 'yunne-jai shin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1603.03141'},\n",
       " {'title': 'the influence of canalization on the robustness of boolean networks',\n",
       "  'id': '1607.04474',\n",
       "  'abstract': 'time- and state-discrete dynamical systems are frequently used to model molecular networks. this paper provides a collection of mathematical and computational tools for the study of robustness in boolean network models. the focus is on networks governed by $k$-canalizing functions, a recently introduced class of boolean functions that contains the well-studied class of nested canalizing functions. the activities and sensitivity of a function quantify the impact of input changes on the function output. this paper generalizes the latter concept to $c$-sensitivity and provides formulas for the activities and $c$-sensitivity of general $k$-canalizing functions as well as canalizing functions with more precisely defined structure. a popular measure for the robustness of a network, the derrida value, can be expressed as a weighted sum of the $c$-sensitivities of the governing canalizing functions, and can also be calculated for a stochastic extension of boolean networks. these findings provide a computationally efficient way to obtain derrida values of boolean networks, deterministic or stochastic, that does not involve simulation.',\n",
       "  'categories': 'math.ds nlin.cd physics.bio-ph q-bio.mn',\n",
       "  'doi': '10.1016/j.physd.2017.05.002',\n",
       "  'created': '2016-07-15',\n",
       "  'updated': '2016-11-18',\n",
       "  'authors': ['claus kadelka', 'jack kuipers', 'reinhard laubenbacher'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1607.04474'},\n",
       " {'title': 'a local counter-regulatory motif modulates the global phase of hormonal   oscillations',\n",
       "  'id': '1607.05386',\n",
       "  'abstract': 'counter-regulatory elements maintain dynamic equilibrium ubiquitously in living systems. the most prominent example, which is critical to mammalian survival, is that of pancreatic {\\\\alpha} and {\\\\beta} cells producing glucagon and insulin for glucose homeostasis. these cells are not found in a single gland but are dispersed in multiple micro-organs known as the islets of langerhans. within an islet, these two reciprocal cell types interact with each other and with an additional cell type: the {\\\\delta} cell. by testing all possible motifs governing the interactions of these three cell types, we found that a unique set of positive/negative intra-islet interactions between different islet cell types functions not only to reduce the superficially wasteful zero-sum action of glucagon and insulin but also to enhance/suppress the synchronization of hormone secretions between islets under high/normal glucose conditions. this anti-symmetric interaction motif confers effective controllability for network (de)synchronization.',\n",
       "  'categories': 'q-bio.to',\n",
       "  'doi': '10.1038/s41598-017-01806-0',\n",
       "  'created': '2016-07-18',\n",
       "  'updated': '2024-02-12',\n",
       "  'authors': ['dong-ho park',\n",
       "   'taegeun song',\n",
       "   'danh-tai hoang',\n",
       "   'jin xu',\n",
       "   'junghyo jo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1607.05386'},\n",
       " {'title': 'estimating propensity parameters using google pagerank and genetic   algorithms',\n",
       "  'id': '1608.00535',\n",
       "  'abstract': 'stochastic boolean networks, or more generally, stochastic discrete networks, are an important class of computational models for molecular interaction networks. the stochasticity stems from the updating schedule. standard updating schedules include the synchronous update, where all the nodes are updated at the same time, and the asynchronous update where a random node is updated at each time step. the former produces a deterministic dynamics while the latter a stochastic dynamics. a more general stochastic setting considers propensity parameters for updating each node. stochastic discrete dynamical systems (sdds) is a modeling framework that considers two propensity parameters for updating each node and uses one when the update has a positive impact on the variable, that is, when the update causes the variable to increase its value, and uses the other when the update has a negative impact, that is, when the update causes it to decrease its value. this framework offers additional features for simulations but also adds a complexity in parameter estimation of the propensities. this paper presents a method for estimating the propensity parameters for sdds. the method is based on adding noise to the system using the google pagerank approach to make the system ergodic and thus guaranteeing the existence of a stationary distribution. then with the use of a genetic algorithm, the propensity parameters are estimated. approximation techniques that make the search algorithms efficient are also presented and matlab/octave code to test the algorithms are available at http://www.ms.uky.edu/~dmu228/geneticalg/code.html.',\n",
       "  'categories': 'q-bio.mn',\n",
       "  'doi': '10.3389/fnins.2016.00513',\n",
       "  'created': '2016-08-01',\n",
       "  'updated': '2016-10-17',\n",
       "  'authors': ['david murrugarra', 'jacob miller', 'alex mueller'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1608.00535'},\n",
       " {'title': 'spirals and heteroclinic cycles in a spatially extended   rock-paper-scissors model of cyclic dominance',\n",
       "  'id': '1609.04141',\n",
       "  'abstract': 'spatially extended versions of the cyclic-dominance rock-paper-scissors model have traveling wave (in one dimension) and spiral (in two dimensions) behavior. the far field of the spirals behave like traveling waves, which themselves have profiles reminiscent of heteroclinic cycles. we compute numerically a nonlinear dispersion relation between the wavelength and wavespeed of the traveling waves, and, together with insight from heteroclinic bifurcation theory and further numerical results from 2d simulations, we are able to make predictions about the overall structure and stability of spiral waves in 2d cyclic dominance models.',\n",
       "  'categories': 'nlin.ps q-bio.pe',\n",
       "  'doi': '10.1209/0295-5075/117/48006',\n",
       "  'created': '2016-09-14',\n",
       "  'updated': '2024-04-15',\n",
       "  'authors': ['c. m. postlethwaite', 'a. m. rucklidge'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1609.04141'},\n",
       " {'title': 'partner selection supported by opaque reputation promotes cooperative   behavior',\n",
       "  'id': '1610.01993',\n",
       "  'abstract': \"reputation plays a major role in human societies, and it has been proposed as an explanation for the evolution of cooperation. while the majority of previous studies equates reputation with a transparent and complete history of players' past decisions, in real life, reputations are often ambiguous and opaque. using web-based experiments, we explore the extent to which opaque reputation works in isolating defectors, with and without partner selection opportunities. our results show that low reputation works as a signal of untrustworthiness, whereas medium or high reputation are not taken into account by participants for orienting their choices. we also find that reputation without partner selection does not promote cooperative behavior; that is, defectors do not turn into cooperators only for the sake of getting a positive reputation. finally, in a third study, we find that, when reputation is pivotal to selection, then a substantial proportion of would-be-defectors turn into cooperators. taken together, these results provide insights on the characteristics of reputation and on the way in which humans make use of it when selecting partners but also when knowing that they will be selected.\",\n",
       "  'categories': 'physics.soc-ph q-bio.pe',\n",
       "  'doi': '10.1017/s1930297500004800',\n",
       "  'created': '2016-10-06',\n",
       "  'updated': '2016-10-07',\n",
       "  'authors': ['valerio capraro',\n",
       "   'francesca giardini',\n",
       "   'daniele vilone',\n",
       "   'mario paolucci'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1610.01993'},\n",
       " {'title': 'nonequilibrium and nonlinear kinetics as key determinants for   bistability in fission yeast g2-m transition',\n",
       "  'id': '1610.09637',\n",
       "  'abstract': 'a living cell is an open, nonequilibrium biochemical system where atp hydrolysis serves as the energy source for a wide range of intracellular processes, possibly including the assurance for decision-making. in the fission yeast cell cycle, the transition from g2 to m phase is driven by the activation of cdc13/cdc2 and cdc25 and the deactivation of wee1 through phosphorylation-dephosphorylation cycles with feedback loops. here, we present a kinetic description of the g2-m circuit which reveals that both cellular atp level and atp hydrolysis free energy critically control cdc2 activation. using fission yeast nucleoplasmic extract (ynpe), we experimentally verify that increased atp level drives the activation of cdc2 which exhibits bistability and hysteresis in response to changes in cellular atp level and atp hydrolysis energy. these findings suggest that cellular atp level and atp hydrolysis energy are determinants of the bistability and robustness of cdc2 activation during g2-m transition.',\n",
       "  'categories': 'q-bio.mn q-bio.sc',\n",
       "  'doi': '',\n",
       "  'created': '2016-10-30',\n",
       "  'updated': '2024-05-06',\n",
       "  'authors': ['de zhao',\n",
       "   'teng wang',\n",
       "   'jian zhao',\n",
       "   'dianjie li',\n",
       "   'zhili lin',\n",
       "   'zeyan chen',\n",
       "   'qi ouyang',\n",
       "   'hong qian',\n",
       "   'yu v. fu',\n",
       "   'fangting li'],\n",
       "  'affiliation': ['peking university, beijing',\n",
       "   'peking university, beijing',\n",
       "   'chinese academy of sciences, beijing',\n",
       "   'peking university, beijing',\n",
       "   'peking university, beijing',\n",
       "   'peking university, beijing',\n",
       "   'peking university, beijing',\n",
       "   'university of washington, seattle',\n",
       "   'chinese academy of sciences, beijing',\n",
       "   'peking university, beijing'],\n",
       "  'url': 'https://arxiv.org/abs/1610.09637'},\n",
       " {'title': 'metabolic allometric scaling of multicellular organisms as a product of   evolutionary development and optimization of food chains',\n",
       "  'id': '1612.00098',\n",
       "  'abstract': \"production of energy is a foundation of life. metabolic rate of organisms (amount of energy produced per unit time) generally increases slower than organisms' mass, which has important implications for life organization. this phenomenon, when considered across different taxa, is called interspecific allometric scaling. its origin has puzzled scientists for many decades, and still is considered unknown. in this paper we posit that natural selection, as determined by evolutionary pressures, leads to distribution of resources, and accordingly energy, within a food chain, which is optimal from the perspective of stability of the food chain, when each species has sufficient amount of resources for continuous reproduction, but not too much to jeopardize existence of other species. metabolic allometric scaling is then a quantitative representation of this optimal distribution. taking locomotion and the primary mechanism for distribution of energy, we developed a biomechanical model to find energy expenditures, considering limb length, skeleton mass and speed. using the interspecific allometric exponents for these three measures and substituting into the locomotion-derived model for energy expenditure, we calculated allometric exponents for mammals, reptiles, fish and birds, and compared these values with allometric exponents derived from experimental observations. the calculated allometric exponents were nearly identical to experimentally observed exponents for mammals, and very close for fish, reptiles and the basal metabolic rate of birds. the main result of the study is that the metabolic allometric scaling is a function of a mechanism of optimal energy distribution between the species of a food chain. this optimized sharing of common resources provides stability of a food chain for a given habitat and is guided by evolutionary pressures and natural selection.\",\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '10.1142/s1793048024500061',\n",
       "  'created': '2016-11-30',\n",
       "  'updated': '2024-10-07',\n",
       "  'authors': ['yuri k. shestopaloff'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1612.00098'},\n",
       " {'title': 'metabolic allometric scaling of unicellular organisms as a product of   selection guided by optimization of nutrients distribution in food chains',\n",
       "  'id': '1612.04695',\n",
       "  'abstract': 'one of the major characteristics of living organisms is metabolic rate, which is the amount of energy produced per unit of time. when the mass of organisms increases, the metabolic rate also increases (usually as a power function of mass), but usually slower than mass. this effect is called metabolic allometric scaling. its causes are considered unknown. the effect has important implications for individual and population organismal development. it was shown in the first part of this study, presented in a separate paper, that in the case of multicellular organisms, this effect is a consequence of natural selection and optimization of nutrient distribution between the species of a food chain, sharing resources of a common habitat. here, in the second part that studies unicellular organisms, we discover that the same principle of natural selection guided by optimization of nutrient distribution between the species of a food chain defines also metabolic allometric scaling of unicellular organisms. to find that, we consider the metabolic properties of amoeba proteus, fission yeast schizosaccharomyces pombe, escherichia coli, bacillus subtilis, staphylococcus. the sharing of nutrients is optimized in such a way that bigger microorganisms have progressively bigger nutrient influx per unit of surface. this evolutionary arrangement secures the stability of a food chain by providing certain metabolic advantages for bigger organisms. accounting for this regular increase of nutrient influx with mass increase, we obtained allometric exponents and their ranges close to experimental values, thus proving that metabolic allometric scaling of both multicellular and unicellular organisms is defined by the same fundamental evolutionary principle of optimized sharing of nutrients between the species of a food chain.',\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '10.1142/s1793048024500097',\n",
       "  'created': '2016-12-14',\n",
       "  'updated': '2024-11-28',\n",
       "  'authors': ['yuri k. shestopaloff'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1612.04695'},\n",
       " {'title': 'optimal decoding of information from a genetic network',\n",
       "  'id': '1612.08084',\n",
       "  'abstract': \"gene expression levels carry information about signals that have functional significance for the organism. using the gap gene network in the fruit fly embryo as an example, we show how this information can be decoded, building a dictionary that translates expression levels into a map of implied positions. the optimal decoder makes use of graded variations in absolute expression level, resulting in positional estimates that are precise to ~1% of the embryo's length. we test this optimal decoder by analyzing gap gene expression in embryos lacking some of the primary maternal inputs to the network. the resulting maps are distorted, and these distortions predict, with no free parameters, the positions of expression stripes for the pair-rule genes in the mutant embryos.\",\n",
       "  'categories': 'q-bio.mn physics.bio-ph',\n",
       "  'doi': '10.1016/j.cell.2019.01.007',\n",
       "  'created': '2016-12-23',\n",
       "  'updated': '',\n",
       "  'authors': ['mariela d. petkova',\n",
       "   'gašper tkačik',\n",
       "   'william bialek',\n",
       "   'eric f. wieschaus',\n",
       "   'thomas gregor'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1612.08084'},\n",
       " {'title': 'selection originating from protein stability/foldability: relationships   between protein folding free energy, sequence ensemble, and fitness',\n",
       "  'id': '1612.09379',\n",
       "  'abstract': 'assuming that mutation and fixation processes are reversible markov processes, we prove that the equilibrium ensemble of sequences obeys a boltzmann distribution with $\\\\exp(4n_e m(1 - 1/(2n)))$, where $m$ is malthusian fitness and $n_e$ and $n$ are effective and actual population sizes. on the other hand, the probability distribution of sequences with maximum entropy that satisfies a given amino acid composition at each site and a given pairwise amino acid frequency at each site pair is a boltzmann distribution with $\\\\exp(-\\\\psi_n)$, where $\\\\psi_n$ is represented as the sum of one body and pairwise potentials. a protein folding theory indicates that homologous sequences obey a canonical ensemble characterized by $\\\\exp(-\\\\delta g_{nd}/k_b t_s)$ or by $\\\\exp(- g_{n}/k_b t_s)$ if an amino acid composition is kept constant, where $\\\\delta g_{nd} \\\\equiv g_n - g_d$, $g_n$ and $g_d$ are the native and denatured free energies, and $t_s$ is selective temperature. thus, $4n_e m (1 - 1/(2n))$, $-\\\\delta \\\\psi_{nd}$, and $-\\\\delta g_{nd}/k_b t_s$ must be equivalent to each other. based on the analysis of the changes ($\\\\delta \\\\psi_n$) of $\\\\psi_n$ due to single nucleotide nonsynonymous substitutions, $t_s$, and then glass transition temperature $t_g$, and $\\\\delta g_{nd}$ are estimated with reasonable values for 14 protein domains. in addition, approximating the probability density function (pdf) of $\\\\delta \\\\psi_n$ by a log-normal distribution, pdfs of $\\\\delta \\\\psi_n$ and $k_a/k_s$, which is the ratio of nonsynonymous to synonymous substitution rate per site, in all and in fixed mutants are estimated. it is confirmed that $t_s$ negatively correlates with the mean of $k_a/k_s$. stabilizing mutations are significantly fixed by positive selection, and balance with destabilizing mutations fixed by random drift. supporting the nearly neutral theory, neutral selection is not significant.',\n",
       "  'categories': 'q-bio.pe q-bio.bm',\n",
       "  'doi': '10.1016/j.jtbi.2017.08.018',\n",
       "  'created': '2016-12-29',\n",
       "  'updated': '2024-08-19',\n",
       "  'authors': ['sanzo miyazawa'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1612.09379'},\n",
       " {'title': 'an evolutionary theory for the variability hypothesis',\n",
       "  'id': '1703.04184',\n",
       "  'abstract': 'an elementary biostatistical theory based on a selectivity-variability principle is proposed to address a question raised by charles darwin, namely, how one sex of a sexually dimorphic species might tend to evolve with greater variability than the other sex. briefly, the theory says that if one sex is relatively selective then from one generation to the next, more variable subpopulations of the opposite sex will generally tend to prevail over those with lesser variability. moreover, the perhaps less intuitive converse also holds: if a sex is relatively non-selective, then less variable subpopulations of the opposite sex will prevail over those with greater variability. this theory requires certain regularity conditions on the distributions, but makes no assumptions about differences in means between the sexes, nor does it presume that one sex is selective and the other non-selective. two mathematical models of the selectivity-variability principle are presented: a discrete-time one-step probabilistic model of short-term behavior with an example using normally distributed perceived fitness values; and a continuous-time deterministic model for the long-term asymptotic behavior of the expected sizes of the subpopulations with an example using exponentially distributed fitness levels.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2017-03-12',\n",
       "  'updated': '2024-10-02',\n",
       "  'authors': ['theodore p. hill'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1703.04184'},\n",
       " {'title': 'memory effects, transient growth, and wave breakup in a model of paced   atrium',\n",
       "  'id': '1704.03399',\n",
       "  'abstract': 'the mechanisms underlying cardiac fibrillation have been investigated for over a century, but we are still finding surprising results that change our view of this phenomenon. the present study focuses on the transition from normal rhythm to atrial fibrillation associated with a gradual increase in the pacing rate. while some of our findings are consistent with existing experimental, numerical, and theoretical studies of this problem, one result appears to contradict the accepted picture. specifically we show that, in a two-dimensional model of paced homogeneous atrial tissue, transition from discordant alternans to conduction block, wave breakup, reentry, and spiral wave chaos is associated with transient growth of finite amplitude disturbances rather than a conventional instability. it is mathematically very similar to subcritical, or bypass, transition from laminar fluid flow to turbulence, which allows many of the tools developed in the context of fluid turbulence to be used for improving our understanding of cardiac arrhythmias.',\n",
       "  'categories': 'q-bio.to nlin.ps',\n",
       "  'doi': '10.1063/1.4999601',\n",
       "  'created': '2017-04-11',\n",
       "  'updated': '',\n",
       "  'authors': ['alejandro garzón', 'roman o. grigoriev'],\n",
       "  'affiliation': ['department of mathematics, universidad sergio arboleda',\n",
       "   'school of physics, georgia institute of technology'],\n",
       "  'url': 'https://arxiv.org/abs/1704.03399'},\n",
       " {'title': 'spectral analysis of the blood flow in the foot microvascular bed during   thermal testing in patients with diabetes mellitus',\n",
       "  'id': '1707.02110',\n",
       "  'abstract': 'timely diagnostics of microcirculatory system abnormalities which are the most severe diabetic complications, is a significant problem of modern health care. functional abnormalities manifest themselves earlier than the structural one and their assessment is the focus of present-day studies. in this study, the laser doppler flowmetry, a noninvasive technique for the cutaneous blood flow monitoring, was used together with local temperature tests and wavelet analysis. the study of the blood flow in the microvascular bed of toes was carried out in control group of 40 subjects, and two diabetic groups differing in the type of diabetes mellitus (17 of dm1 and 23 of dm2).   the temperature tests demonstrated that diabetic patients have impaired vasodilation in response to local heating. the study of oscillating components shows a significant difference of the spectral properties even in the basal conditions. low frequency pulsations of the blood flow associated with endothelial and activities are lower in both diabetes groups as well as the ones connected with cardiac activity. local thermal tests induce variations both in the perfusion and its spectral characteristics, which are different in the groups under consideration. we assume that the results obtained provide a deeper understanding of pathological processes involved in the progress of microvascular abnormalities due to diabetes mellitus.',\n",
       "  'categories': 'q-bio.to',\n",
       "  'doi': '10.1016/j.mvr.2018.05.005',\n",
       "  'created': '2017-07-07',\n",
       "  'updated': '',\n",
       "  'authors': ['irina mizeva',\n",
       "   'elena zharkikh',\n",
       "   'victor dremin',\n",
       "   'evgeny zherebtsov',\n",
       "   'irina makovik',\n",
       "   'elena potapova',\n",
       "   'andrey dunaev'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1707.02110'},\n",
       " {'title': 'fast flowing populations are not well mixed',\n",
       "  'id': '1709.10442',\n",
       "  'abstract': 'in evolutionary dynamics, well-mixed populations are almost always associated with all-to-all interactions; mathematical models are based on complete graphs. in most cases, these models do not predict fixation probabilities in groups of individuals mixed by flows. we propose an analytical description in the fast-flow limit. this approach is valid for processes with global and local selection, and accurately predicts the suppression of selection as competition becomes more local. it provides a modelling tool for biological or social systems with individuals in motion.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1038/s41598-018-22062-w',\n",
       "  'created': '2017-09-29',\n",
       "  'updated': '',\n",
       "  'authors': ['francisco herrerías-azcué',\n",
       "   'vicente pérez-muñuzuri',\n",
       "   'tobias galla'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1709.10442'},\n",
       " {'title': 'outward sodium current, fine-structure constant and ferroelectric   hysteresis regimes in the giant squid axon propagating action potential: a   phase space approach',\n",
       "  'id': '1711.03575',\n",
       "  'abstract': \"a phase space cable equation for the propagation of the functioning action potential is derived. plots of the ionic current in the recovery region reveal the presence of two outward currents with quasilinear segments. the first one has never been detected with voltage clamp experiments. the plot of currents in the action potential frame of reference reveals that the modulated ohm's current determines the reversal of the ionic current from incoming to outward in the recovery region. we show that this current is sodium current caused by ohm current buildup of the electrostatic gradient across the membrane opposing and overwhelming sodium's concentration gradient. fits of fractions of open sodium channels with the avrami equation reveal the continuous phase changes of the sodium channels lattice at the inception and at the peak of the action potential. the correlation between the avrami parameter a and the ionic channel's temperature dependent time rate reveals the role of the fine-structure constant in the passage of ions through channels. plots of ionic time rates as boltzmann's kinetic rates yield temperature independent ionic activation and deactivation energies suggesting a potential quantum mechanical influence, such as tunneling. the optimum sodium channel density is calculated that is close to the observed one. it is shown that action potential traverses a heat releasing ferroelectric hysteresis loop with sodium channels traversing two states with different symmetries. presented results provide the framework for further analysis of excitability, channel's lattice structure, thermodynamic phase behavior, and the role of quantum mechanics mediating the flow of ions through channels. the role of ferroelectric sodium channels behavior has implications for encoding and storing of memories.\",\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '',\n",
       "  'created': '2017-11-09',\n",
       "  'updated': '2024-11-16',\n",
       "  'authors': ['nikola k. jurisic', 'fred cooper'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1711.03575'},\n",
       " {'title': 'broad cross-reactivity of the t-cell repertoire achieves specific and   sufficiently rapid target searching',\n",
       "  'id': '1712.04633',\n",
       "  'abstract': 'the molecular recognition of t-cell receptors is the hallmark of the adaptive immunity. given the finiteness of the t-cell repertoire, individual t-cell receptors are necessary to be cross-reactive to multiple antigenic peptides. in this study, we quantify the variability of the cross-reactivity by using a string model that estimates the binding affinity between two sequences of amino acids. we examine sequences of 10,000 human t-cell receptors and 10,000 antigenic peptides, and obtain a full spectrum of cross-reactivity of the receptor-peptide binding. then, we find that the cross-reactivity spectrum is broad. some t cells are reactive to 1,000 peptides, but some t cells are reactive to only one or two peptides. since the degree of cross-reactivity has a correlation with the (un)binding affinity of receptors, we further investigate how the broad crossreactivity affects the target searching of t cells. high cross-reactive t cells may not require many trials for searching correct targets, but they may spend long time to unbind from incorrect targets. in contrast, low cross-reactive t cells may not spend long time to ignore incorrect targets, but they require many trials for screening correct targets. we evaluate this hypothesis, and show that the broad cross-reactivity of the natural t-cell repertoire can balance the trade-off between the rapid screening and unbinding penalty.',\n",
       "  'categories': 'q-bio.cb',\n",
       "  'doi': '10.1016/j.jtbi.2019.01.025',\n",
       "  'created': '2017-12-13',\n",
       "  'updated': '2024-02-13',\n",
       "  'authors': ['jin xu', 'junghyo jo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1712.04633'},\n",
       " {'title': 'emergent memory in cell signaling: persistent adaptive dynamics in   cascades can arise from the diversity of relaxation time-scales',\n",
       "  'id': '1801.04057',\n",
       "  'abstract': 'the mitogen-activated protein kinase (mapk) signaling cascade, an evolutionarily conserved motif present in all eukaryotic cells, is involved in coordinating critical cell-fate decisions, regulating protein synthesis, and mediating learning and memory. while the steady-state behavior of the pathway stimulated by a time-invariant signal is relatively well-understood, we show using a computational model that it exhibits a rich repertoire of transient adaptive responses to changes in stimuli. when the signal is switched on, the response is characterized by long-lived modulations in frequency as well as amplitude. on withdrawing the stimulus, the activity decays over timescales much longer than that of phosphorylation-dephosphorylation processes, exhibiting reverberations characterized by repeated spiking in the activated mapk concentration. the long-term persistence of such post-stimulus activity suggests that the cascade retains memory of the signal for a significant duration following its removal, even in the absence of any explicit feedback or cross-talk with other pathways. we find that the molecular mechanism underlying this behavior is related to the existence of distinct relaxation rates for the different cascade components. this results in the imbalance of fluxes between different layers of the cascade, with the repeated reuse of activated kinases as enzymes when they are released from sequestration in complexes leading to one or more spike events following the removal of the stimulus. the persistent adaptive response reported here, indicative of a cellular \"short-term\" memory, suggests that this ubiquitous signaling pathway plays an even more central role in information processing by eukaryotic cells.',\n",
       "  'categories': 'q-bio.sc nlin.ao physics.bio-ph',\n",
       "  'doi': '10.1038/s41598-018-31626-9',\n",
       "  'created': '2018-01-12',\n",
       "  'updated': '',\n",
       "  'authors': ['tanmay mitra', 'shakti n. menon', 'sitabhra sinha'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1801.04057'},\n",
       " {'title': 'collective dynamics of self-propelled semiflexible filaments',\n",
       "  'id': '1802.07480',\n",
       "  'abstract': 'the collective behavior of active semiflexible filaments is studied with a model of tangentially driven self-propelled worm-like chains. the combination of excluded-volume interactions and self-propulsion leads to several distinct dynamic phases as a function of bending rigidity, activity, and aspect ratio of individual filaments. we consider first the case of intermediate filament density. for high-aspect-ratio filaments, we identify a transition with increasing propulsion from a state of free-swimming filaments to a state of spiraled filaments with nearly frozen translational motion. for lower aspect ratios, this gas-of-spirals phase is suppressed with growing density due to filament collisions; instead, filaments form clusters similar to self-propelled rods, as activity increases. finite bending rigidity strongly effects the dynamics and phase behavior. flexible filaments form small and transient clusters, while stiffer filaments organize into giant clusters, similarly as self-propelled rods, but with a reentrant phase behavior from giant to smaller clusters as activity becomes large enough to bend the filaments. for high filament densities, we identify a nearly frozen jamming state at low activities, a nematic laning state at intermediate activities, and an active-turbulence state at high activities. the latter state is characterized by a power-law decay of the energy spectrum as a function of wave number. the resulting phase diagrams encapsulate tunable non-equilibrium steady states that can be used in the organization of living matter.',\n",
       "  'categories': 'physics.bio-ph cond-mat.soft q-bio.sc',\n",
       "  'doi': '10.1039/c8sm00282g',\n",
       "  'created': '2018-02-21',\n",
       "  'updated': '',\n",
       "  'authors': ['ozer duman',\n",
       "   'rolf e. isele-holder',\n",
       "   'jens elgeti',\n",
       "   'gerhard gompper'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1802.07480'},\n",
       " {'title': 'a primer on the use of probability generating functions in infectious   disease modeling',\n",
       "  'id': '1803.05136',\n",
       "  'abstract': 'we explore the application of probability generating functions (pgfs) to invasive processes, focusing on infectious disease introduced into large populations. our goal is to acquaint the reader with applications of pgfs, moreso than to derive new results. pgfs help predict a number of properties about early outbreak behavior while the population is still effectively infinite, including the probability of an epidemic, the size distribution after some number of generations, and the cumulative size distribution of non-epidemic outbreaks. we show how pgfs can be used in both discrete-time and continuous-time settings, and discuss how to use these results to infer disease parameters from observed outbreaks. in the large population limit for susceptible-infected-recovered (sir) epidemics pgfs lead to survival-function based models that are equivalent the the usual mass-action sir models but with fewer odes. we use these to explore properties such as the final size of epidemics or even the dynamics once stochastic effects are negligible. we target this tutorial to biologists and public health researchers who want to learn how to apply pgfs to invasive diseases, but it could also be used in an introductory mathematics course on pgfs. we include many exercises to help demonstrate concepts and to give practice applying the results. we summarize our main results in a few tables. additionally we provide a small python package which performs many of the relevant calculations.',\n",
       "  'categories': 'q-bio.pe physics.soc-ph q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2018-03-14',\n",
       "  'updated': '2024-10-02',\n",
       "  'authors': ['joel c. miller'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1803.05136'},\n",
       " {'title': 'biogeographical network analysis of plant species distribution in the   mediterranean region',\n",
       "  'id': '1803.05275',\n",
       "  'abstract': \"the delimitation of bioregions helps to understand historical and ecological drivers of species distribution. in this work, we performed a network analysis of the spatial distribution patterns of plants in south of france (languedoc-roussillon and provence-alpes-c\\\\^ote d'azur) to analyze the biogeographical structure of the french mediterranean flora at different scales. we used a network approach to identify and characterize biogeographical regions, based on a large database containing 2.5 million of geolocalized plant records corresponding to more than 3500 plant species. this methodology is performed following five steps, from the biogeographical bipartite network construction, to the identification of biogeographical regions under the form of spatial network communities, the analysis of their interactions and the identification of clusters of plant species based on the species contribution to the biogeographical regions. first, we identified two sub-networks that distinguish mediterranean and temperate biota. then, we separated eight statistically significant bioregions that present a complex spatial structure. some of them are spatially well delimited, and match with particular geological entities. on the other hand fuzzy transitions arise between adjacent bioregions that share a common geological setting, but are spread along a climatic gradient. the proposed network approach illustrates the biogeographical structure of the flora in southern france, and provides precise insights into the relationships between bioregions. this approach sheds light on ecological drivers shaping the distribution of mediterranean biota: the interplay between a climatic gradient and geological substrate shapes biodiversity patterns. finally this work exemplifies why fragmented distributions are common in the mediterranean region, isolating groups of species that share a similar eco-evolutionary history.\",\n",
       "  'categories': 'q-bio.pe q-bio.qm',\n",
       "  'doi': '10.1002/ece3.4718',\n",
       "  'created': '2018-03-06',\n",
       "  'updated': '2024-02-08',\n",
       "  'authors': ['maxime lenormand',\n",
       "   'guillaume papuga',\n",
       "   'olivier argagnon',\n",
       "   'maxence soubeyrand',\n",
       "   'guilhem de barros',\n",
       "   'samuel alleaume',\n",
       "   'sandra luque'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1803.05275'},\n",
       " {'title': 'reconciliation of weak pairwise spike-train correlations and highly   coherent local field potentials across space',\n",
       "  'id': '1805.10235',\n",
       "  'abstract': 'multi-electrode arrays covering several square millimeters of neural tissue provide simultaneous access to population signals such as extracellular potentials and spiking activity of one hundred or more individual neurons. the interpretation of the recorded data calls for multiscale computational models with corresponding spatial dimensions and signal predictions. multi-layer spiking neuron network models of local cortical circuits covering about 1 mm$^2$ have been developed, integrating experimentally obtained neuron-type-specific connectivity data and reproducing features of observed in-vivo spiking statistics. local field potentials (lfps) can be computed from the simulated spiking activity. we here extend a local network and lfp model to an area of 4x4 mm$^2$, preserving the neuron density and introducing distance-dependent connection probabilities and conduction delays. we find that the upscaling procedure preserves the overall spiking statistics of the original model and reproduces asynchronous irregular spiking across populations and weak pairwise spike-train correlations in agreement with experimental recordings from sensory cortex. also compatible with experimental observations, the correlation of lfp signals is strong and decays over a distance of several hundred micrometers. enhanced spatial coherence in the low-gamma band around 50 hz may explain the recent report of an apparent band-pass filter effect in the spatial reach of the lfp.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1093/cercor/bhae405',\n",
       "  'created': '2018-05-25',\n",
       "  'updated': '2024-09-23',\n",
       "  'authors': ['johanna senk',\n",
       "   'espen hagen',\n",
       "   'sacha j. van albada',\n",
       "   'markus diesmann'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1805.10235'},\n",
       " {'title': 'ebic: an open source software for high-dimensional and big data   biclustering analyses',\n",
       "  'id': '1807.09932',\n",
       "  'abstract': 'motivation: in this paper we present the latest release of ebic, a next-generation biclustering algorithm for mining genetic data. the major contribution of this paper is adding support for big data, making it possible to efficiently run large genomic data mining analyses. additional enhancements include integration with r and bioconductor and an option to remove influence of missing value on the final result.   results: ebic was applied to datasets of different sizes, including a large dna methylation dataset with 436,444 rows. for the largest dataset we observed over 6.6 fold speedup in computation time on a cluster of 8 gpus compared to running the method on a single gpu. this proves high scalability of the algorithm.   availability: the latest version of ebic could be downloaded from http://github.com/epistasislab/ebic . installation and usage instructions are also available online.',\n",
       "  'categories': 'q-bio.gn cs.lg stat.ml',\n",
       "  'doi': '10.1093/bioinformatics/btz027',\n",
       "  'created': '2018-07-25',\n",
       "  'updated': '2024-09-04',\n",
       "  'authors': ['patryk orzechowski', 'jason h. moore'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1807.09932'},\n",
       " {'title': 'immunological recognition by artificial neural networks',\n",
       "  'id': '1808.03386',\n",
       "  'abstract': 'the binding affinity between the t-cell receptors (tcrs) and antigenic peptides mainly determines immunological recognition. it is not a trivial task that t cells identify the digital sequences of peptide amino acids by simply relying on the integrated binding affinity between tcrs and antigenic peptides. to address this problem, we examine whether the affinity-based discrimination of peptide sequences is learnable and generalizable by artificial neural networks (anns) that process the digital experimental amino acid sequence information of receptors and peptides. a pair of tcr and peptide sequences correspond to the input for anns, while the success or failure of the immunological recognition correspond to the output. the output is obtained by both theoretical model and experimental data. in either case, we confirmed that anns could learn the immunological recognition. we also found that a homogenized encoding of amino acid sequence was more effective for the supervised learning task.',\n",
       "  'categories': 'q-bio.cb q-bio.qm',\n",
       "  'doi': '10.3938/jkps.73.1908',\n",
       "  'created': '2018-08-09',\n",
       "  'updated': '2024-02-12',\n",
       "  'authors': ['jin xu', 'junghyo jo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1808.03386'},\n",
       " {'title': 'complete inference of causal relations between dynamical systems',\n",
       "  'id': '1808.10806',\n",
       "  'abstract': \"from ancient philosophers to modern economists, biologists, and other researchers, there has been a continuous effort to unveil causal relations. the most formidable challenge lies in deducing the nature of the causal relationship: whether it is unidirectional, bidirectional, or merely apparent - implied by an unobserved common cause.   while modern technology equips us with tools to collect data from intricate systems such as the planet's ecosystem or the human brain, comprehending their functioning requires the identification and differentiation of causal relationships among the components, all without external interventions.   in this context, we introduce a novel method capable of distinguishing and assigning probabilities to the presence of all potential causal relations between two or more time series within dynamical systems. the efficacy of this method is verified using synthetic datasets and applied to eeg (electroencephalographic) data recorded from epileptic patients.   given the universal applicability of our method, it holds promise for diverse scientific fields.\",\n",
       "  'categories': 'q-bio.qm q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2018-08-31',\n",
       "  'updated': '2024-01-17',\n",
       "  'authors': ['zsigmond benkő',\n",
       "   'ádám zlatniczki',\n",
       "   'marcell stippinger',\n",
       "   'dániel fabó',\n",
       "   'andrás sólyom',\n",
       "   'loránd erőss',\n",
       "   'andrás telcs',\n",
       "   'zoltán somogyvári'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1808.10806'},\n",
       " {'title': 'localization of brain activity from eeg/meg using mv-pure framework',\n",
       "  'id': '1809.03930',\n",
       "  'abstract': 'we consider the problem of localization of sources of brain electrical activity from electroencephalographic (eeg) and magnetoencephalographic (meg) measurements using spatial filtering techniques. we propose novel reduced-rank activity indices based on the minimum-variance pseudo-unbiased reduced-rank estimation (mv-pure) framework. the main results of this paper establish the key unbiasedness property of the proposed indices and their higher spatial resolution compared with full-rank indices in challenging task of localizing closely positioned and possibly highly correlated sources, especially in low signal-to-noise regime. numerical examples are provided to illustrate the practical applicability of the proposed activity indices using both simulated and real data.',\n",
       "  'categories': 'eess.sp q-bio.nc',\n",
       "  'doi': '10.1016/j.bspc.2020.102243',\n",
       "  'created': '2018-09-11',\n",
       "  'updated': '2024-08-01',\n",
       "  'authors': ['tomasz piotrowski', 'jan nikadon', 'alexander moiseev'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1809.03930'},\n",
       " {'title': 'mechanics of tissue competition: interfaces stabilize coexistence',\n",
       "  'id': '1809.10990',\n",
       "  'abstract': 'mechanical forces influence the dynamics of growing tissues. computer simulations are employed to study the importance of interfacial effects in tissue competition. it was speculated that mechanical pressure determines the competition, where the determining quantity is the homeostatic pressure - the pressure where division and apoptosis balance; the tissue with the higher homeostatic pressure overwhelms the other. surprisingly, a weaker tissue can persist in stable coexistence with a stronger tissue, if adhesion between them is small enough. an analytic continuum description can quantitatively describe the underlying mechanism and reproduce the resulting pressures and cell-number fractions. computer simulations furthermore display a variety of coexisting structures, ranging from spherical inclusions to a bicontinuous state.',\n",
       "  'categories': 'physics.bio-ph q-bio.cb',\n",
       "  'doi': '10.1088/1367-2630/ab2475',\n",
       "  'created': '2018-09-28',\n",
       "  'updated': '',\n",
       "  'authors': ['nirmalendu ganai',\n",
       "   'tobias buscher',\n",
       "   'gerhard gompper',\n",
       "   'jens elgeti'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1809.10990'},\n",
       " {'title': 'traction force microscopy with optimized regularization and automated   bayesian parameter selection for comparing cells',\n",
       "  'id': '1810.05848',\n",
       "  'abstract': 'adherent cells exert traction forces on to their environment, which allows them to migrate, to maintain tissue integrity, and to form complex multicellular structures. this traction can be measured in a perturbation-free manner with traction force microscopy (tfm). in tfm, traction is usually calculated via the solution of a linear system, which is complicated by undersampled input data, acquisition noise, and large condition numbers for some methods. therefore, standard tfm algorithms either employ data filtering or regularization. however, these approaches require a manual selection of filter- or regularization parameters and consequently exhibit a substantial degree of subjectiveness. this shortcoming is particularly serious when cells in different conditions are to be compared because optimal noise suppression needs to be adapted for every situation, which invariably results in systematic errors. here, we systematically test the performance of new methods from computer vision and bayesian inference for solving the inverse problem in tfm. we compare two classical schemes, l1- and l2-regularization, with three previously untested schemes, namely elastic net regularization, proximal gradient lasso, and proximal gradient elastic net. overall, we find that elastic net regularization, which combines l1 and l2 regularization, outperforms all other methods with regard to accuracy of traction reconstruction. next, we develop two methods, bayesian l2 regularization and advanced bayesian l2 regularization, for automatic, optimal l2 regularization. using artificial data and experimental data, we show that these methods enable robust reconstruction of traction without requiring a difficult selection of regularization parameters specifically for each data set. thus, bayesian methods can mitigate the considerable uncertainty inherent in comparing cellular traction forces.',\n",
       "  'categories': 'physics.bio-ph q-bio.cb q-bio.qm',\n",
       "  'doi': '10.1038/s41598-018-36896-x',\n",
       "  'created': '2018-10-13',\n",
       "  'updated': '',\n",
       "  'authors': ['yunfei huang',\n",
       "   'christoph schell',\n",
       "   'tobias b. huber',\n",
       "   'ahmet n. simsek',\n",
       "   'nils hersch',\n",
       "   'rudolf merkel',\n",
       "   'gerhard gompper',\n",
       "   'benedikt sabass'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1810.05848'},\n",
       " {'title': 'aggression heuristics underlie animal dominance hierarchies and provide   evidence of group-level social information',\n",
       "  'id': '1810.07215',\n",
       "  'abstract': 'members of a social species need to make appropriate decisions about who, how, and when to interact with others in their group. however, it has been difficult for researchers to detect the inputs to these decisions and, in particular, how much information individuals actually have about their social context. we present a new method that can serve as a social assay to quantify how patterns of aggression depend upon information about the ranks of individuals within social dominance hierarchies. applied to existing data on aggression in 172 social groups across 85 species in 23 orders, it reveals three main patterns of rank-dependent social dominance: the downward heuristic (aggress uniformly against lower-ranked opponents), close competitors (aggress against opponents ranked slightly below self), and bullying (aggress against opponents ranked much lower than self). the majority of the groups (133 groups, 77%) follow a downward heuristic, but a significant minority (38 groups, 22%) show more complex social dominance patterns (close competitors or bullying) consistent with higher levels of social information use. these patterns are not phylogenetically constrained and different groups within the same species can use different patterns, suggesting that heuristics use may depend on context and the structuring of aggression by social information should not be considered a fixed characteristic of a species. our approach provides new opportunities to study the use of social information within and across species and the evolution of social complexity and cognition.',\n",
       "  'categories': 'q-bio.pe nlin.ao physics.soc-ph q-bio.nc',\n",
       "  'doi': '10.1073/pnas.2022912118',\n",
       "  'created': '2018-10-16',\n",
       "  'updated': '2020-11-12',\n",
       "  'authors': ['elizabeth a. hobson', 'dan mønster', 'simon dedeo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1810.07215'},\n",
       " {'title': 'optical probe pressure effects on cutaneous blood flow',\n",
       "  'id': '1811.11443',\n",
       "  'abstract': 'the variation of blood flow characteristics caused by the probe pressure during noninvasive studies is of particular interest within the context of fundamental and applied research. it has been shown previously that the weak local pressure induces vasodilation, whereas the increased pressure is able to stop the blood flow in the compressed area, as well as to significantly change optical signals.   the blood flow oscillations measured by laser doppler flowmetry (ldf) characterize the functional state of the microvascular system and can be used for noninvasive diagnostics of its abnormality. this study was intended to identify the patterns of the relationship between the oscillating components of blood flow registered by the ldf method under different levels of pressure applied to an optical fiber probe.   for this purpose we have developed an original optical probe capable of regulating the applied pressure. the developed protocol included six sequential records of the blood perfusion at pressure within the 0 to 200 mmhg range with unloading at the last stage.   using wavelet analyses, we traced the variation of energy of oscillations for these records in five frequency bands associated with different vascular tone regulation mechanisms. six young volunteers of the same age (three males and three females) were included in this preliminary study and the protocol was repeated five times in each volunteer. totally 30 ldf records were available for the analyses. as expected, the ldf signal increases at weak pressure (30 mmhg) and decreases at increased pressure. the statistically stable amplification of endothelial associated blood flow oscillations under the 90 mmhg pressure allowed us to put forward a hypothesis that the endothelial activity increases. the possible causes of this phenomenon are discussed.',\n",
       "  'categories': 'q-bio.to',\n",
       "  'doi': '10.3233/ch-180459',\n",
       "  'created': '2018-11-28',\n",
       "  'updated': '',\n",
       "  'authors': ['irina a. mizeva',\n",
       "   'elena v. potapova',\n",
       "   'viktor v. dremin',\n",
       "   'evgeny a. zherebtsov',\n",
       "   'mikhail a. mezentsev',\n",
       "   'valerii v. shupletsov',\n",
       "   'andrey v. dunaev'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1811.11443'},\n",
       " {'title': 'a shortest-path based clustering algorithm for joint human-machine   analysis of complex datasets',\n",
       "  'id': '1812.11850',\n",
       "  'abstract': 'clustering is a technique for the analysis of datasets obtained by empirical studies in several disciplines with a major application for biomedical research. essentially, clustering algorithms are executed by machines aiming at finding groups of related points in a dataset. however, the result of grouping depends on both metrics for point-to-point similarity and rules for point-to-group association. indeed, non-appropriate metrics and rules can lead to undesirable clustering artifacts. this is especially relevant for datasets, where groups with heterogeneous structures co-exist. in this work, we propose an algorithm that achieves clustering by exploring the paths between points. this allows both, to evaluate the properties of the path (such as gaps, density variations, etc.), and expressing the preference for certain paths. moreover, our algorithm supports the integration of existing knowledge about admissible and non-admissible clusters by training a path classifier. we demonstrate the accuracy of the proposed method on challenging datasets including points from synthetic shapes in publicly available benchmarks and microscopy data.',\n",
       "  'categories': 'q-bio.qm cs.ai cs.hc cs.lg',\n",
       "  'doi': '10.1126/sciadv.aax377',\n",
       "  'created': '2018-12-31',\n",
       "  'updated': '',\n",
       "  'authors': ['diego ulisse pizzagalli',\n",
       "   'santiago fernandez gonzalez',\n",
       "   'rolf krause'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1812.11850'},\n",
       " {'title': 'the power of moral words: loaded language generates framing effects in   the extreme dictator game',\n",
       "  'id': '1901.02314',\n",
       "  'abstract': \"understanding whether preferences are sensitive to the frame has been a major topic of debate in the last decades. for example, several works have explored whether the dictator game in the give frame gives rise to a different rate of pro-sociality than the same game in the take frame, leading to mixed results. here we contribute to this debate with two experiments. in study 1 ($n=567$) we implement an extreme dictator game in which the dictator either gets \\\\$0.50 and the recipient gets nothing, or the opposite (i.e., the recipient gets \\\\$0.50 and the dictator gets nothing). we experimentally manipulate the words describing the available actions using six terms, from very negative (e.g., stealing) to very positive (e.g., donating) connotations. we find that the rate of pro-sociality is affected by the words used to describe the available actions. in study 2 ($n=221$) we ask brand new participants to rate each of the words used in study 1 from ``extremely wrong'' to ``extremely right'' . we find that these moral judgments explain the framing effect in study 1. in sum, our studies provide evidence that framing effects in an extreme dictator game can be generated using morally loaded language.\",\n",
       "  'categories': 'physics.soc-ph cs.gt q-bio.pe',\n",
       "  'doi': '10.1017/s1930297500004356',\n",
       "  'created': '2019-01-08',\n",
       "  'updated': '2019-04-06',\n",
       "  'authors': ['valerio capraro', 'andrea vanzo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1901.02314'},\n",
       " {'title': 'influence of nicotine and alcohol on sleep latency: the reward-attention   circuit model',\n",
       "  'id': '1902.05515',\n",
       "  'abstract': 'we previously developed a neurocomputational model called the reward-attention circuit, which has been used to investigate the separate effects of nicotine and alcohol on attention focus. the model is based on known circuits linking midbrain dopaminergic neurons to the thalamocortical loop and is sufficiently versatile to capture diverse phenomena. in this work, we used the model to study the influence of both nicotine and alcohol on sleep latency (time to fall asleep). in line with findings from other studies, our simulations suggest that nicotine can have a stimulating effect that increases sleep latency, promoting wakefulness, and, in contrast, alcohol can have a sedating effect that induces sleep. together, our results suggest insights into alterations in mesothalamic dopamine activity and sleep. also, they raise speculations about aspects of insomnia and low mesothalamic dopamine.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2019-02-14',\n",
       "  'updated': '2024-02-02',\n",
       "  'authors': ['karine guimarães'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1902.05515'},\n",
       " {'title': 'a physical mechanism of heterogeneity in stem cell, cancer and cancer   stem cell',\n",
       "  'id': '1902.10353',\n",
       "  'abstract': 'heterogeneity is ubiquitous in stem cells (sc), cancer cells (cs), and cancer stem cells (csc). sc and csc heterogeneity is manifested as diverse sub-populations with self-renewing and unique regeneration capacity. moreover, the csc progeny possesses multiple plasticity and cancerous characteristics. many studies have demonstrated that cancer heterogeneity is one of the greatest obstacle for therapy. this leads to the incomplete anti-cancer therapies and transitory efficacy. furthermore, numerous micro-metastasis leads to the wide spread of the tumor cells across the body which is the beginning of metastasis. the epigenetic processes (dna methylation or histone remodification etc.) can provide a source for certain heterogeneity. in this study, we develop a mathematical model to quantify the heterogeneity of sc, csc and cancer taking both genetic and epigenetic effects into consideration. we uncovered the roles and physical mechanisms of heterogeneity from the three aspects (sc, csc and cancer). in the adiabatic regime (relatively fast regulatory binding and effective coupling among genes), seven native states (sc, csc, cancer, premalignant, normal, lesion and hyperplasia) emerge. in non-adiabatic regime (relatively slow regulatory binding and effective weak coupling among genes), multiple meta-stable sc, cs, csc and differentiated states emerged which can explain the origin of heterogeneity. in other words, the slow regulatory binding mimicking the epigenetics can give rise to heterogeneity. elucidating the origin of heterogeneity and dynamical interrelationship between intra-tumoral cells has clear clinical significance in helping to understand the cellular basis of treatment response, therapeutic resistance, and tumor relapse.',\n",
       "  'categories': 'q-bio.mn physics.bio-ph q-bio.cb',\n",
       "  'doi': '10.1063/5.0078196',\n",
       "  'created': '2019-02-27',\n",
       "  'updated': '',\n",
       "  'authors': ['chong yu', 'qiong liu', 'cong chen', 'jin wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1902.10353'},\n",
       " {'title': 'span observables - \"when is a foraging rabbit no longer hungry?\"',\n",
       "  'id': '1903.06036',\n",
       "  'abstract': 'be $x_t$ a random walk. we study its span $s$, i.e. the size of the domain visited up to time $t$. we want to know the probability that $s$ reaches $1$ for the first time, as well as the density of the span given $t$. analytical results are presented, and checked against numerical simulations. we then generalize this to include drift, and one or two reflecting boundaries. we also derive the joint probability of the maximum and minimum of a process. our results are based on the diffusion propagator with reflecting or absorbing boundaries, for which a set of useful formulas is derived.',\n",
       "  'categories': 'cond-mat.stat-mech q-bio.qm',\n",
       "  'doi': '10.1007/s10955-019-02446-6',\n",
       "  'created': '2019-03-13',\n",
       "  'updated': '',\n",
       "  'authors': ['kay joerg wiese'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1903.06036'},\n",
       " {'title': 'exploiting product molecule number to consider reaction rate fluctuation   in elementary reactions',\n",
       "  'id': '1903.06375',\n",
       "  'abstract': 'in many chemical reactions, reaction rate fluctuation is inevitable. reaction rates are different whenever chemical reaction occurs due to their dependence on the number of reaction events or the product number. as such, understanding the impact of rate fluctuation on product number counting statistics is of the utmost importance when developing a quantitative explanation of chemical reactions. in this work, we present a master equation that describes reaction rates as a function of product number and time. our equal reveals the relationship between the reaction rate and product number fluctuation. product number counting statistics uncovers a stochastic property of the product number; product number directly manipulates the reaction rate. specifically, we find that product number shows super-poisson characteristics when the product number increases, inducing an increase in the reaction rate. while, on the other hand, when the product number shows sub-poisson characteristics with an increase in the product number, this is induced by a decrease in the reaction rate. furthermore, our analysis exploits reaction rate fluctuation, enabling the quantification of the deviation of an elementary reaction process from a renewal process.',\n",
       "  'categories': 'q-bio.mn',\n",
       "  'doi': '10.1063/5.0091597',\n",
       "  'created': '2019-03-15',\n",
       "  'updated': '2019-03-18',\n",
       "  'authors': ['seong jun park'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1903.06375'},\n",
       " {'title': 'a general mathematical framework for understanding the behavior of   heterogeneous stem cell regeneration',\n",
       "  'id': '1903.11448',\n",
       "  'abstract': 'stem cell heterogeneity is essential for the homeostasis in tissue development. this paper established a general formulation for understanding the dynamics of stem cell regeneration with cell heterogeneity and random transitions of epigenetic states. the model generalizes the classical g0 cell cycle model, and incorporates the epigenetic states of stem cells that are represented by a continuous multidimensional variable and the kinetic rates of cell behaviors, including proliferation, differentiation, and apoptosis, that are dependent on their epigenetic states. moreover, the random transition of epigenetic states is represented by an inheritance probability that can be described as a conditional beta distribution. this model can be extended to investigate gene mutation-induced tumor development. the proposed formula is a generalized formula that helps us to understand various dynamic processes of stem cell regeneration, including tissue development, degeneration, and abnormal growth.',\n",
       "  'categories': 'q-bio.pe q-bio.cb',\n",
       "  'doi': '10.1016/j.jtbi.2020.110196',\n",
       "  'created': '2019-03-27',\n",
       "  'updated': '2024-07-11',\n",
       "  'authors': ['jinzhi lei'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1903.11448'},\n",
       " {'title': 'the superstatistics of confined ants: from serenity to panic',\n",
       "  'id': '1904.03236',\n",
       "  'abstract': 'nonequilibrium systems with strong parameter fluctuations are difficult to describe with ordinary statistical mechanics. superstatistics -- that can be seen as statistics of an statistics -- has emerged as a tool able to describe these complex systems. it has been successfully applied, for example, to hydrodynamic turbulence, internal convection, and even dna architecture, but not to macroscopic biological systems. here we present fully controlled laboratory experiments in confined ants both in normal and in \"panic\" conditions. using a data-driven superstatistical model, we reproduce key signatures of their motion: from non-gaussian velocity distributions driven by a log-normal diffusion coefficient, to clustering away from the source of danger information.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2019-04-05',\n",
       "  'updated': '2024-10-31',\n",
       "  'authors': ['a. reyes',\n",
       "   'm. curbelo',\n",
       "   'f. tejera',\n",
       "   'a. rivera',\n",
       "   'o. ramos',\n",
       "   'm. s. turner',\n",
       "   'e. altshuler'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1904.03236'},\n",
       " {'title': 'a concept-value network as a brain model',\n",
       "  'id': '1904.04579',\n",
       "  'abstract': \"this paper suggests a statistical framework for describing the relations between the physical and conceptual entities of a brain-like model. features and concept instances are put into context, where the paper suggests that features may be the electrical wiring, although chemical connections are also possible. with this idea, the actual length of the connection is important, because it is related to firing rates and neuron synchronization, but the signal type is less important. the paper then suggests that concepts are neuron groups that link feature sets and concept instances are determined by chemical signals from those groups. therefore, features become the static horizontal framework of the neural system and concepts are vertically interconnected combinations of these. with regards to functionality, the neuron is then considered to be functional and the more horizontal memory structures can even be glial. this would also suggest that features can be distributed entities and not concentrated to a single area. another aspect could be signal 'breaks' that compartmentalise a pattern and may help with neural binding.\",\n",
       "  'categories': 'cs.ne cs.ai q-bio.nc',\n",
       "  'doi': '10.3390/neurosci5040039',\n",
       "  'created': '2019-04-09',\n",
       "  'updated': '2024-09-26',\n",
       "  'authors': ['kieran greer'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1904.04579'},\n",
       " {'title': 'beyond the nucleus: cytoplasmic dominance in follicular thyroid   carcinoma detection using single-cell raman imaging across multiple devices',\n",
       "  'id': '1904.05675',\n",
       "  'abstract': 'cytological diagnosis of follicular thyroid carcinoma (ftc) is one of major challenges in the field of endocrine oncology due to absence of evident morphological indicators. morphological abnormalities in the nucleus are typically key indicators in cancer cytopathology and are attributed to a range of biochemical alterations in nuclear components. consequently, raman spectroscopy has been widely used to detect cancer in various cytological samples, often identifying biochemical changes prior to observable morphological alterations. however, in the case of ftc, cytoplasmic features such as carotenoids, cytochromes, and lipid droplets have shown greater diagnostic relevance compared to nuclear features. this study leverages single-cell raman imaging to explore the spatial origin of diagnostic signals in ftc and normal thyroid (nt) cells, assessing the contributions of the nucleus and cytoplasm independently. our results demonstrate that raman spectra from the cytoplasmic region can distinguish between ftc and nt cells with an accuracy of 84% under co-culture conditions, maintaining robustness across multiple devices. in contrast, classification based on nuclear spectra achieved only 53% accuracy, suggesting that biochemical alterations in the cytoplasm play a more significant role in ftc detection than those in the nucleus. our work elevates the promise of raman-based cytopathology by providing complementary organelle-dependent information to traditional diagnostic methods and demonstrating transferability across different devices.',\n",
       "  'categories': 'q-bio.qm q-bio.cb',\n",
       "  'doi': '',\n",
       "  'created': '2019-04-11',\n",
       "  'updated': '2024-11-22',\n",
       "  'authors': ['aurelien pelissier',\n",
       "   'kosuke hashimoto',\n",
       "   'kentaro mochizuki',\n",
       "   'j. nicholas taylor',\n",
       "   'jean-emmanuel clement',\n",
       "   'yasuaki kumamoto',\n",
       "   'katsumasa fujita',\n",
       "   'yoshinori harada',\n",
       "   'tamiki komatsuzaki'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1904.05675'},\n",
       " {'title': 'magnetic resonance imaging of mean cell size in human breast tumors',\n",
       "  'id': '1905.07818',\n",
       "  'abstract': 'purpose: cell size is a fundamental characteristic of all tissues, and changes in cell size in cancer reflect tumor status and response to treatments, such as apoptosis and cell cycle arrest. unfortunately, cell size can only be obtained by pathologic evaluation of the tumor in the current standard of care. previous imaging approaches can be implemented on only animal mri scanners or require relatively long acquisition times that are undesirable for clinical imaging. there is a need to develop cell size imaging for clinics. experimental design: we propose a new method, impulsed (imaging microstructural parameters using limited spectrally edited diffusion) that can characterize mean cell sizes in solid tumors. we report the use of combined sequences with different gradient waveforms on human mri and analytical equations that link dwi signals of real gradient waveforms and specific microstructural parameters such as cell size. we also describe comprehensive validations using computer simulations, cell experiments in vitro, and animal experiments in vivo and finally demonstrate applications in pre-operative breast cancer patients. results: with fast acquisitions (~ 7 mins), impulsed can provide high-resolution (1.3 mm in-plane) mapping of mean cell size of human tumors in vivo on currently-available 3t mri scanners. all validations suggest impulsed provide accurate and reliable measurements of mean cell size. conclusion: the proposed impulsed method can assess cell size variations in the tumor of breast cancer patients, which may have the potential to assess early response to neoadjuvant therapy.',\n",
       "  'categories': 'physics.bio-ph physics.med-ph q-bio.qm',\n",
       "  'doi': '10.1002/mrm.28056',\n",
       "  'created': '2019-05-19',\n",
       "  'updated': '',\n",
       "  'authors': ['junzhong xu',\n",
       "   'xiaoyu jiang',\n",
       "   'hua li',\n",
       "   'lori r. arlinghaus',\n",
       "   'eliot t. mckinley',\n",
       "   'sean p. devan',\n",
       "   'benjamin m. hardy',\n",
       "   'jingping xie',\n",
       "   'hakmook kang',\n",
       "   'anuradha b. chakravarthy',\n",
       "   'john c. gore'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1905.07818'},\n",
       " {'title': 'cortical divisive normalization from wilson-cowan neural dynamics',\n",
       "  'id': '1906.08246',\n",
       "  'abstract': 'divisive normalization and the wilson-cowan equations are influential models of neural interaction and saturation [carandini and heeger nat.rev.neurosci. 2012; wilson and cowan kybernetik 1973]. however, they have not been analytically related yet. in this work we show that divisive normalization can be obtained from the wilson-cowan model. specifically, assuming that divisive normalization is the steady state solution of the wilson-cowan differential equation, we find that the kernel that controls neural interactions in divisive normalization depends on the wilson-cowan kernel but also has a signal-dependent contribution. a standard stability analysis of a wilson-cowan model with the parameters obtained from our relation shows that the divisive normalization solution is a stable node. this stability demonstrates the consistency of our steady state assumption.   the proposed theory provides a physiological foundation (a relation to a dynamical network with fixed wiring among neurons) for the functional suggestions that have been done on the need of signal-dependent divisive normalization [e.g. in coen-cagli et al., plos comp.biol. 2012]. moreover, this theory explains the modifications that had to be introduced ad-hoc in gaussian kernels of divisive normalization in [martinez et al. front. neurosci. 2019] to reproduce contrast responses. the proposed relation implies that the wilson-cowan dynamics also reproduces visual masking and subjective image distortion metrics, which up to now had been mainly explained via divisive normalization. finally, this relation allows to apply to divisive normalization the methods which up to now had been developed for dynamical systems such as wilson-cowan networks.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2019-06-19',\n",
       "  'updated': '2023-12-28',\n",
       "  'authors': ['j. malo', 'j. j. esteve-taboada', 'm. bertalmío'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1906.08246'},\n",
       " {'title': 'towards a consensus on rheological models for shear waves in soft   tissues',\n",
       "  'id': '1906.08612',\n",
       "  'abstract': 'a rising wave of technologies and instruments are enabling more labs and clinics to make a variety of measurements related to tissue viscoelastic properties. these instruments include elastography imaging scanners, rheological shear viscometers, and a variety of calibrated stress-strain analyzers. from these many sources of disparate data, a common step in analyzing results is to fit the measurements of tissue response to some viscoelastic model. in the best scenario, this places the measurements within a theoretical framework and enables meaningful comparisons of the parameters against other types of tissues. however, there is a large set of established rheological models, even within the class of linear, causal, viscoelastic solid models, so which of these should be chosen? is it simply a matter of best fit to a minimum mean squared error of the model to several data points? we argue that the long history of biomechanics, including the concept of the extended relaxation spectrum, along with data collected from viscoelastic soft tissues over an extended range of times and frequencies, and the theoretical framework of multiple relaxation models which model the multi-scale nature of physical tissues, all lead to the conclusion that fractional derivative models represent the most succinct and meaningful models of soft tissue viscoelastic behavior. these arguments are presented with the goal of clarifying some distinctions between, and consequences of, some of the most commonly used models, and with the longer term goal of reaching a consensus among different sub-fields in acoustics, biomechanics, and elastography that have common interests in comparing tissue measurements.',\n",
       "  'categories': 'q-bio.to q-bio.qm',\n",
       "  'doi': '10.1088/1361-6560/ab453d',\n",
       "  'created': '2019-06-20',\n",
       "  'updated': '2019-09-19',\n",
       "  'authors': ['kevin j. parker', 'thomas szabo', 'sverre holm'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1906.08612'},\n",
       " {'title': 'derivation of the variational bayes equations',\n",
       "  'id': '1906.08804',\n",
       "  'abstract': \"the derivation of key equations for the variational bayes approach is well-known in certain circles. however, translating the fundamental derivations (e.g., as found in beal's work) to friston's notation is somewhat delicate. further, the notion of using variational bayes in the context of a system with a markov blanket requires special attention. this technical report presents the derivation in detail. it further illustrates how the variational bayes method provides a framework for a new computational engine, incorporating the 2-d cluster variation method (cvm), which provides a necessary free energy equation that can be minimized across both the external and representational systems' states, respectively.\",\n",
       "  'categories': 'cs.ne q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2019-06-20',\n",
       "  'updated': '2024-08-18',\n",
       "  'authors': ['alianna j. maren'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1906.08804'},\n",
       " {'title': 'quantifying the total effect of edge interventions in discrete   multistate networks',\n",
       "  'id': '1906.09465',\n",
       "  'abstract': 'developing efficient computational methods to assess the impact of external interventions on the dynamics of a network model is an important problem in systems biology. this paper focuses on quantifying the global changes that result from the application of an intervention to produce a desired effect, which we define as the total effect of the intervention. the type of mathematical models that we will consider are discrete dynamical systems which include the widely used boolean networks and their generalizations. the potential interventions can be represented by a set of nodes and edges that can be manipulated to produce a desired effect on the system. we use a class of regulatory rules called nested canalizing functions that frequently appear in published models and were inspired by the concept of canalization in evolutionary biology. in this paper, we provide a polynomial normal form based on the canalizing properties of regulatory functions. using this polynomial normal form, we give a set of formulas for counting the maximum number of transitions that will change in the state space upon an edge deletion in the wiring diagram. these formulas rely on the canalizing structure of the target function since the number of changed transitions depends on the canalizing layer that includes the input to be deleted. we also present computations on random networks to compare the exact number of changes with the upper bounds provided by our formulas. finally, we provide statistics on the sharpness of these upper bounds in random networks.',\n",
       "  'categories': 'q-bio.mn',\n",
       "  'doi': '10.1016/j.automatica.2020.109453',\n",
       "  'created': '2019-06-22',\n",
       "  'updated': '2020-10-11',\n",
       "  'authors': ['david murrugarra', 'elena dimitrova'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1906.09465'},\n",
       " {'title': 'improving rna secondary structure prediction via state inference with   deep recurrent neural networks',\n",
       "  'id': '1906.10819',\n",
       "  'abstract': 'the problem of determining which nucleotides of an rna sequence are paired or unpaired in the secondary structure of an rna, which we call rna state inference, can be studied by different machine learning techniques. successful state inference of rna sequences can be used to generate auxiliary information for data-directed rna secondary structure prediction. bidirectional long short-term memory (lstm) neural networks have emerged as a powerful tool that can model global nonlinear sequence dependencies and have achieved state-of-the-art performances on many different classification problems. this paper presents a practical approach to rna secondary structure inference centered around a deep learning method for state inference. state predictions from a deep bidirectional lstm are used to generate synthetic shape data that can be incorporated into rna secondary structure prediction via the nearest neighbor thermodynamic model (nntm). this method produces predicted secondary structures for a diverse test set of 16s ribosomal rna that are, on average, 25 percentage points more accurate than undirected mfe structures. these improvements range from several percentage points for some sequences to nearly 50 percentage points for others. accuracy is highly dependent on the success of our state inference method, and investigating the global features of our state predictions reveals that accuracy of both our state inference and structure inference methods are highly dependent on the similarity of the sequence to the dataset. this paper presents a deep learning state inference tool, trained and tested on 16s ribosomal rna. converting these state predictions into synthetic shape data with which to direct nntm can result in large improvements in secondary structure prediction accuracy, as shown on a test set of 16s rrna.',\n",
       "  'categories': 'q-bio.bm',\n",
       "  'doi': '10.1515/cmb-2020-0002',\n",
       "  'created': '2019-06-25',\n",
       "  'updated': '2020-02-23',\n",
       "  'authors': ['devin willmott', 'david murrugarra', 'qiang ye'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1906.10819'},\n",
       " {'title': 'modeling response time distributions with generalized beta prime',\n",
       "  'id': '1907.00070',\n",
       "  'abstract': \"we use generalized beta prime distribution, also known as gb2, for fitting response time distributions. this distribution, characterized by one scale and three shape parameters, is incredibly flexible in that it can mimic behavior of many other distributions. gb2 exhibits power-law behavior at both front and tail ends and is a steady-state distribution of a simple stochastic differential equation. we apply gb2 in contrast studies between two distinct groups -- in this case children with dyslexia and a control group -- and show that it provides superior fitting. we compare aggregate response time distributions of the two groups for scale and shape differences (including several scale-independent measures of variability, such as hoover index), which may in turn reflect on cognitive dynamics differences. in this approach, response time distribution of an individual can be considered as a random variate of that individual's group distribution.\",\n",
       "  'categories': 'q-bio.nc stat.ap',\n",
       "  'doi': '10.5890/dnc.2020.09.009',\n",
       "  'created': '2019-06-28',\n",
       "  'updated': '',\n",
       "  'authors': ['m. dashti moghaddam',\n",
       "   'jiong liu',\n",
       "   'john g. holden',\n",
       "   'r. a. serota'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1907.00070'},\n",
       " {'title': 'mechanisms underlying the response of mouse cortical networks to   optogenetic manipulation',\n",
       "  'id': '1907.00816',\n",
       "  'abstract': 'gabaergic interneurons can be subdivided into three subclasses: parvalbumin positive (pv), somatostatin positive (som) and serotonin positive neurons. with principal cells (pcs) they form complex networks. we examine pcs and pv responses in mouse anterior lateral motor cortex (alm) and barrel cortex (s1) upon pv photostimulation in vivo. in layer 5, the pv response is paradoxical: photoexcitation reduces their activity. this is not the case in alm layer 2/3. we combine analytical calculations and numerical simulations to investigate how these results constrain the architecture. two-population models cannot account for the results. networks with three inhibitory populations and v1-like architecture account for the data in alm layer 2/3. our data in layer 5 can be accounted for if som neurons receive inputs only from pcs and pv neurons. in both four-population models, the paradoxical effect implies not too strong recurrent excitation. it is not evidence for stabilization by inhibition.',\n",
       "  'categories': 'q-bio.nc cond-mat.dis-nn',\n",
       "  'doi': '',\n",
       "  'created': '2019-07-01',\n",
       "  'updated': '2024-09-02',\n",
       "  'authors': ['alexandre mahrach',\n",
       "   'guang chen',\n",
       "   'nuo li',\n",
       "   'carl van vreeswijk',\n",
       "   'david hansel'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1907.00816'},\n",
       " {'title': 'sers discrimination of single amino acid residue in single peptide by   plasmonic nanocavities',\n",
       "  'id': '1908.04206',\n",
       "  'abstract': 'surface-enhanced raman spectroscopy (sers) is a sensitive label-free optical method that can provide fingerprint raman spectra of biomolecules such as dna, amino acids and proteins. while sers of single dna molecule has been recently demonstrated, raman analysis of single protein sequence was not possible because the sers spectra of proteins are usually dominated by signals of aromatic amino acid residues. here, we used electroplasmonic approach to trap single gold nanoparticle in a nanohole for generating a plasmonic nanocavity between the trapped nanoparticle and the nanopore wall. the giant field generated in the nanocavity was so sensitive and localized that it enables sers discrimination of 10 distinct amino acids at single-molecule level. the obtained spectra are used to analyze the spectra of 2 biomarkers (vasopressin and oxytocin) made of a short sequence of 9 amino-acids. significantly, we demonstrated identification of single non-aromatic amino acid residues in a single short peptide chain as well as discrimination between two peptides with sequences distinguishable in 2 specific amino-acids. our result demonstrate the high sensitivity of our method to identify single amino acid residue in a protein chain and a potential for further applications in proteomics and single-protein sequencing.',\n",
       "  'categories': 'q-bio.qm physics.optics',\n",
       "  'doi': '10.1002/anie.202000489',\n",
       "  'created': '2019-08-09',\n",
       "  'updated': '2019-12-13',\n",
       "  'authors': ['jian-an huang',\n",
       "   'mansoureh z. mousavi',\n",
       "   'giorgia giovannini',\n",
       "   'yingqi zhao',\n",
       "   'aliaksandr hubarevich',\n",
       "   'denis garoli',\n",
       "   'francesco de angelis'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1908.04206'},\n",
       " {'title': 'evolution of cooperation in networks with well-connected cooperators',\n",
       "  'id': '1908.05923',\n",
       "  'abstract': 'cooperative behavior constitutes a key aspect of human society and non-human animal systems, but explaining how cooperation evolves represents a major scientific challenge. it is now well established that social network structure plays a central role for the viability of cooperation. however, not much is known about the importance of the positions of cooperators in the networks for the evolution of cooperation. here, we investigate how the spread of cooperation is affected by correlations between cooperativeness and individual social connectedness (such that cooperators occupy well-connected network positions). using simulation models, we find that these correlations enhance cooperation in standard scale-free networks but not in standard poisson networks. in contrast, when degree assortativity is increased such that individuals cluster with others of similar social connectedness, we find that poisson networks can maintain high levels of cooperation, which can even exceed those of scale-free networks. we show that this is due to dynamics where bridge areas between social clusters act as barriers to the spread of defection. we also find that this positive effect on cooperation is sensitive to the presence of trojan horses (defectors placed within cooperator clusters), which allow defection to invade. the results provide new knowledge about the conditions under which cooperation may evolve, and are also relevant to consider in regard to the design of cooperation studies.',\n",
       "  'categories': 'q-bio.pe physics.soc-ph',\n",
       "  'doi': '10.1017/nws.2024.11',\n",
       "  'created': '2019-08-16',\n",
       "  'updated': '2024-10-18',\n",
       "  'authors': ['josefine bohr brask', 'jonatan bohr brask'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1908.05923'},\n",
       " {'title': 'linear stability analysis for large dynamical systems on directed random   graphs',\n",
       "  'id': '1908.07092',\n",
       "  'abstract': 'we present a linear stability analysis of stationary states (or fixed points) in large dynamical systems defined on random directed graphs with a prescribed distribution of indegrees and outdegrees. we obtain two remarkable results for such dynamical systems: first, infinitely large systems on directed graphs can be stable even when the degree distribution has unbounded support; this result is surprising since their counterparts on nondirected graphs are unstable when system size is large enough. second, we show that the phase transition between the stable and unstable phase is universal in the sense that it depends only on a few parameters, such as, the mean degree and a degree correlation coefficient. in addition, in the unstable regime we characterize the nature of the destabilizing mode, which also exhibits universal features. these results follow from an exact theory for the leading eigenvalue of infinitely large graphs that are locally tree-like and oriented, as well as, for the right and left eigenvectors associated with the leading eigenvalue. we corroborate analytical results for infinitely large graphs with numerical experiments on random graphs of finite size. we discuss how the presented theory can be extended to graphs with diagonal disorder and to graphs that contain nondirected links. finally, we discuss the influence of small cycles and how they can destabilize large dynamical systems when they induce strong enough feedback loops.',\n",
       "  'categories': 'cond-mat.stat-mech cond-mat.dis-nn cs.si physics.soc-ph q-bio.pe',\n",
       "  'doi': '10.1103/physrevresearch.2.033313',\n",
       "  'created': '2019-08-19',\n",
       "  'updated': '2024-05-21',\n",
       "  'authors': ['izaak neri', 'fernando lucas metz'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1908.07092'},\n",
       " {'title': 'boltzmann machine learning and regularization methods for inferring   evolutionary fields and couplings from a multiple sequence alignment',\n",
       "  'id': '1909.05006',\n",
       "  'abstract': 'the inverse potts problem to infer a boltzmann distribution for homologous protein sequences from their single-site and pairwise amino acid frequencies recently attracts a great deal of attention in the studies of protein structure and evolution. we study regularization and learning methods and how to tune regularization parameters to correctly infer interactions in boltzmann machine learning. using $l_2$ regularization for fields, group $l_1$ for couplings is shown to be very effective for sparse couplings in comparison with $l_2$ and $l_1$. two regularization parameters are tuned to yield equal values for both the sample and ensemble averages of evolutionary energy. both averages smoothly change and converge, but their learning profiles are very different between learning methods. the adam method is modified to make stepsize proportional to the gradient for sparse couplings. it is shown by first inferring interactions from protein sequences and then from monte carlo samples that the fields and couplings can be well recovered, but that recovering the pairwise correlations in the resolution of a total energy is harder for the natural proteins than for the protein-like sequences. selective temperature for folding/structural constrains in protein evolution is also estimated.',\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech cs.lg q-bio.bm stat.ml',\n",
       "  'doi': '10.1109/tcbb.2020.2993232',\n",
       "  'created': '2019-09-10',\n",
       "  'updated': '2024-07-21',\n",
       "  'authors': ['sanzo miyazawa'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1909.05006'},\n",
       " {'title': 'exact power spectrum in a minimal hybrid model of stochastic gene   expression oscillations',\n",
       "  'id': '1909.09769',\n",
       "  'abstract': \"stochastic oscillations in individual cells are usually characterized by a non-monotonic power spectrum with an oscillatory autocorrelation function. here we develop an analytical approach of stochastic oscillations in a minimal hybrid model of stochastic gene expression including promoter state switching, protein synthesis and degradation, as well as a genetic feedback loop. the oscillations observed in our model are noise-induced since the deterministic theory predicts stable fixed points. the autocorrelated function, power spectrum, and steady-state distribution of protein concentration fluctuations are computed in closed form without making any approximations. using the exactly solvable model, we illustrate sustained oscillations as a circular motion along a stochastic hysteresis loop induced by gene state switching. a triphasic stochastic bifurcation upon the increasing strength of negative feedback is observed, which reveals how stochastic bursts evolve into stochastic oscillations. in our model, oscillations tend to occur when the protein is relatively stable and when gene switching is relatively slow. translational bursting is found to enhance the robustness and broaden the region of stochastic oscillations. these results provide deeper insights into r. thomas' two conjectures for single-cell gene expression kinetics.\",\n",
       "  'categories': 'q-bio.mn nlin.cd physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2019-09-20',\n",
       "  'updated': '2024-02-07',\n",
       "  'authors': ['chen jia', 'hong qian', 'michael q. zhang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1909.09769'},\n",
       " {'title': 'tissue evolution: mechanical interplay of adhesion, pressure, and   heterogeneity',\n",
       "  'id': '1910.03263',\n",
       "  'abstract': 'the evolution of various competing cell types in tissues, and the resulting persistent tissue population, is studied numerically and analytically in a particle-based model of active tissues. mutations change the properties of cells in various ways, including their mechanical properties. each mutation results in an advantage or disadvantage to grow in the competition between different cell types. while changes in signaling processes and biochemistry play an important role, we focus on changes in the mechanical properties by studying the result of variation of growth force and adhesive cross-interactions between cell types. for independent mutations of growth force and adhesion strength, the tissue evolves towards cell types with high growth force and low internal adhesion strength, as both increase the homeostatic pressure. motivated by biological evidence, we postulate a coupling between both parameters, such that an increased growth force comes at the cost of a higher internal adhesion strength or vice versa. this tradeoff controls the evolution of the tissue, ranging from unidirectional evolution to very heterogeneous and dynamic populations. the special case of two competing cell types reveals three distinct parameter regimes: two in which one cell type outcompetes the other, and one in which both cell types coexist in a highly mixed state. interestingly, a single mutated cell alone suffices to reach the mixed state, while a finite mutation rate affects the results only weakly. finally, the coupling between changes in growth force and adhesion strength reveals a mechanical explanation for the evolution towards intra-tumor heterogeneity, in which multiple species coexist even under a constant evolutianary pressure.',\n",
       "  'categories': 'q-bio.pe q-bio.to',\n",
       "  'doi': '10.1088/1367-2630/ab74a5',\n",
       "  'created': '2019-10-08',\n",
       "  'updated': '',\n",
       "  'authors': ['tobias büscher',\n",
       "   'nirmalendu ganai',\n",
       "   'gerhard gompper',\n",
       "   'jens elgeti'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1910.03263'},\n",
       " {'title': 'statistical linear models in virus genomic alignment-free   classification: application to hepatitis c viruses',\n",
       "  'id': '1910.05421',\n",
       "  'abstract': 'viral sequence classification is an important task in pathogen detection, epidemiological surveys and evolutionary studies. statistical learning methods are widely used to classify and identify viral sequences in samples from environments. these methods face several challenges associated with the nature and properties of viral genomes such as recombination, mutation rate and diversity. also, new generations of sequencing technologies rise other difficulties by generating massive amounts of fragmented sequences. while linear classifiers are often used to classify viruses, there is a lack of exploration of the accuracy space of existing models in the context of alignment free approaches. in this study, we present an exhaustive assessment procedure exploring the power of linear classifiers in genotyping and subtyping partial and complete genomes. it is applied to the hepatitis c viruses (hcv). several variables are considered in this investigation such as classifier types (generative and discriminative) and their hyper-parameters (smoothing value and regularization penalty function), the classification task (genotyping and subtyping), the length of the tested sequences (partial and complete) and the length of k-mer words. overall, several classifiers perform well given a set of precise combination of the experimental variables mentioned above. finally, we provide the procedure and benchmark data to allow for more robust assessment of classification from virus genomes.',\n",
       "  'categories': 'cs.lg q-bio.gn stat.ml',\n",
       "  'doi': '10.1109/bibm47256.2019.8983375',\n",
       "  'created': '2019-10-11',\n",
       "  'updated': '2024-05-28',\n",
       "  'authors': ['amine m. remita', 'abdoulaye baniré diallo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1910.05421'},\n",
       " {'title': \"beyond classical hamilton's rule. state distribution asymmetry and the   dynamics of altruism\",\n",
       "  'id': '1912.00518',\n",
       "  'abstract': 'this paper analyzes relationships between demographic and state-based evolutionary game framework and inclusive fitness and hamilton\\'s rule. it is shown that the classical hamilton\\'s rule (counterfactual method) combined with demographic payoff functions, leads to easily testable models. it works well in the case when the roles of donor and receiver are randomly drawn during each interaction event. this is illustrated by the alarm call example. however, we can imagine situations in which role-switching results from some external mechanism. for example, fluxes of individuals between the border and the interior of the habitat, when only border individuals may spot the threat and warn their neighbors. to cover these cases, a new model is extended to the case with explicit dynamics of the role distributions among carriers of different strategies, driven by some general mechanisms. it is thereby shown that even in the case when fluxes between roles are driven by neutral mechanisms acting in the same way on all strategies, differences in mortality in the focal interaction lead to different distributions of roles for different strategies. this leads to a more complex rule for cooperation than the classical hamilton\\'s rule. in addition to the classical cost and benefit, the new rule contains a third component weighted by the difference in proportions of donors among carriers of both strategies. depending on the sign, this component can be termed the survival surplus, or the sacrifice cost, when the receiver\\'s survival exceeds that of the donor. for the \"surplus\" case, cooperators can win even in the case when the assortment mechanism is inefficient (i.e probability of receiving help for noncooperators is slightly greater than for cooperators), which is impossible in the classical hamilton\\'s rule.',\n",
       "  'categories': 'q-bio.pe nlin.ao',\n",
       "  'doi': '',\n",
       "  'created': '2019-12-01',\n",
       "  'updated': '2024-06-26',\n",
       "  'authors': ['krzysztof argasinski', 'ryszard rudnicki'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1912.00518'},\n",
       " {'title': 'airlift: a fast and comprehensive technique for remapping alignments   between reference genomes',\n",
       "  'id': '1912.08735',\n",
       "  'abstract': 'airlift is the first read remapping tool that enables users to quickly and comprehensively map a read set, that had been previously mapped to one reference genome, to another similar reference. users can then quickly run a downstream analysis of read sets for each latest reference release. compared to the state-of-the-art method for remapping reads (i.e., full mapping), airlift reduces the overall execution time to remap read sets between two reference genome versions by up to 27.4x. we validate our remapping results with gatk and find that airlift provides high accuracy in identifying ground truth snp/indel variants   airlift source code and readme describing how to reproduce our results are available at https://github.com/cmu-safari/airlift.',\n",
       "  'categories': 'q-bio.gn cs.ce',\n",
       "  'doi': '',\n",
       "  'created': '2019-12-18',\n",
       "  'updated': '2024-09-11',\n",
       "  'authors': ['jeremie s. kim',\n",
       "   'can firtina',\n",
       "   'meryem banu cavlak',\n",
       "   'damla senol cali',\n",
       "   'mohammed alser',\n",
       "   'nastaran hajinazar',\n",
       "   'can alkan',\n",
       "   'onur mutlu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1912.08735'},\n",
       " {'title': 'evolution of cooperation in networked heterogeneous fluctuating   environments',\n",
       "  'id': '1912.09205',\n",
       "  'abstract': 'fluctuating environments are situations where the spatio-temporal stochasticity plays a significant role in the evolutionary dynamics. the study of the evolution of cooperation in these environments typically assumes a homogeneous, well mixed population, whose constituents are endowed with identical capabilities. in this paper, we generalize these results by developing a systematic study for the cooperation dynamics in fluctuating environments under the consideration of structured, heterogeneous populations with individual entities subjected to general behavioral rules. considering complex network topologies, and a behavioral rule based on generalized reciprocity, we perform a detailed analysis of the effect of the underlying interaction structure on the evolutionary stability of cooperation. we find that, in the presence of environmental fluctuations, the cooperation dynamics can lead to the creation of multiple network components, each with distinct evolutionary properties. this is paralleled to the freezing state in the random energy model. we utilize this result to examine the applicability of our generalized reciprocity behavioral rule in a variety of settings. we thereby show that the introduced rule leads to steady state cooperative behavior that is always greater than or equal to the one predicted by the evolutionary stability analysis of unconditional cooperation. as a consequence, the implementation of our results may go beyond explaining the evolution of cooperation. in particular, they can be directly applied in domains that deal with the development of artificial systems able to adequately mimic reality, such as reinforcement learning.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1016/j.physa.2021.125904',\n",
       "  'created': '2019-12-19',\n",
       "  'updated': '2021-03-09',\n",
       "  'authors': ['viktor stojkoski',\n",
       "   'marko karbevski',\n",
       "   'zoran utkovski',\n",
       "   'lasko basnarkov',\n",
       "   'ljupco kocarev'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1912.09205'},\n",
       " {'title': 'systematic clustering algorithm for chromatin accessibility data and its   application to hematopoietic cells',\n",
       "  'id': '1912.10641',\n",
       "  'abstract': 'the huge amount of data acquired by high-throughput sequencing requires data reduction for effective analysis. here we give a clustering algorithm for genome-wide open chromatin data using a new data reduction method. this method regards the genome as a string of $1$s and $0$s based on a set of peaks and calculates the hamming distances between the strings. this algorithm with the systematically optimized set of peaks enables us to quantitatively evaluate differences between samples of hematopoietic cells and classify cell types, potentially leading to a better understanding of leukemia pathogenesis.',\n",
       "  'categories': 'q-bio.gn cond-mat.stat-mech physics.data-an q-bio.qm',\n",
       "  'doi': '10.1371/journal.pcbi.1008422',\n",
       "  'created': '2019-12-23',\n",
       "  'updated': '2020-11-26',\n",
       "  'authors': ['azusa tanaka',\n",
       "   'yasuhiro ishitsuka',\n",
       "   'hiroki ohta',\n",
       "   'akihiro fujimoto',\n",
       "   'jun-ichirou yasunaga',\n",
       "   'masao matsuoka'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1912.10641'},\n",
       " {'title': 'linearpartition: linear-time approximation of rna folding partition   function and base pairing probabilities',\n",
       "  'id': '1912.13190',\n",
       "  'abstract': 'rna secondary structure prediction is widely used to understand rna function. recently, there has been a shift away from the classical minimum free energy (mfe) methods to partition function-based methods that account for folding ensembles and can therefore estimate structure and base pair probabilities. however, the classical partition function algorithm scales cubically with sequence length, and is therefore a slow calculation for long sequences. this slowness is even more severe than cubic-time mfe-based methods due to a larger constant factor in runtime. inspired by the success of our recently proposed linearfold algorithm that predicts the approximate mfe structure in linear time, we design a similar linear-time heuristic algorithm, linearpartition, to approximate the partition function and base pairing probabilities, which is shown to be orders of magnitude faster than vienna rnafold and contrafold (e.g., 2.5 days vs. 1.3 minutes on a sequence with length 32,753 nt). more interestingly, the resulting base pairing probabilities are even better correlated with the ground truth structures. linearpartition also leads to a small accuracy improvement when used for downstream structure prediction on families with the longest length sequences (16s and 23s rrna), as well as a substantial improvement on long-distance base pairs (500+ nt apart).',\n",
       "  'categories': 'q-bio.bm cs.ds physics.bio-ph q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2019-12-31',\n",
       "  'updated': '2020-02-01',\n",
       "  'authors': ['he zhang', 'liang zhang', 'david h. mathews', 'liang huang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/1912.13190'},\n",
       " {'title': 'a machine learning approach to investigate regulatory control circuits   in bacterial metabolic pathways',\n",
       "  'id': '2001.04794',\n",
       "  'abstract': \"in this work, a machine learning approach for identifying the multi-omics metabolic regulatory control circuits inside the pathways is described. therefore, the identification of bacterial metabolic pathways that are more regulated than others in term of their multi-omics follows from the analysis of these circuits . this is a consequence of the alternation of the omic values of codon usage and protein abundance along with the circuits. in this work, the e.coli's glycolysis and its multi-omic circuit features are shown as an example.\",\n",
       "  'categories': 'q-bio.mn cs.lg stat.ml',\n",
       "  'doi': '10.1093/bioinformatics/btaa966',\n",
       "  'created': '2020-01-13',\n",
       "  'updated': '',\n",
       "  'authors': ['francesco bardozzo', \"pietro lio'\", 'roberto tagliaferri'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2001.04794'},\n",
       " {'title': 'multiblock variable influence on orthogonal projections (mb-viop) for   enhanced interpretation of total, global, local and unique variations in   onpls models',\n",
       "  'id': '2001.06530',\n",
       "  'abstract': 'for multivariate data analysis involving only two input matrices, the previously published methods for variable influence on projection (e.g., vipopls or vipo2pls) are widely used for variable selection purposes, including (i) variable importance assessment, (ii) dimensionality reduction of big data and (iii) interpretation enhancement of pls, opls and o2pls models. for multiblock analysis, the onpls models find relationships among multiple data matrices by calculating latent variables; however, a method for improving the interpretation of these latent variables by assessing the importance of the input variables was not available up to now.   a method for variable selection in multiblock analysis, called multiblock variable influence on orthogonal projections (mb-viop) is explained in this paper. mb-viop is a model based variable selection method that uses the data matrices, the scores and the normalized loadings of an onpls model in order to sort the input variables of more than two data matrices according to their importance for both simplification and interpretation of the total multiblock model, and also of the unique, local and global model components separately. mb-viop has been tested using three multiblock datasets.   mb-viop assesses the variable importance in any type of data. mb-viop connects the input variables of different data matrices according to their relevance for the interpretation of each latent variable, yielding enhanced interpretability for each onpls model component. besides, mb-viop can deal with strong overlapping of types of variation, as well as with many data blocks with very different dimensionality. the ability of mb-viop for generating dimensionality reduced models with high interpretability makes this method ideal for big data mining, multi-omics data integration and any study that requires exploration and interpretation of large streams of data.',\n",
       "  'categories': 'stat.me q-bio.qm',\n",
       "  'doi': '10.1186/s12859-021-04015-9',\n",
       "  'created': '2020-01-17',\n",
       "  'updated': '2021-02-05',\n",
       "  'authors': ['b. galindo-prieto', 'p. geladi', 'j. trygg'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2001.06530'},\n",
       " {'title': 'oxygen depletion hypothesis remains controversial: a mathematical model   of oxygen depletion during flash radiation',\n",
       "  'id': '2001.10788',\n",
       "  'abstract': 'background: experiments have reported low normal tissue toxicities during flash radiation, but the mechanism has not been elaborated. several hypotheses have been proposed to explain the mechanism. the oxygen depletion hypothesis has been introduced and mostly studied qualitatively. methods: we present a computational model to describe the time-dependent change of oxygen concentration in the tissue. the kinetic equation of the model is solved numerically using the finite difference method. the model is used to analyze the flash effect with the oxygen depletion hypothesis, and the brain tissue is chosen as an example. results: the oxygen distribution is determined by the oxygen consumption rate of the tissue and the distance between capillaries. the change of oxygen concentration with time after radiation has been found to follow a negative exponential function, and the time constant is determined by the distance between capillaries. when the dose rate is high enough, the same dose results in the same change of oxygen concentration regardless of dose rate. the analysis of flash effect in the brain tissue based on this model does not support the explanation of the oxygen depletion hypothesis. conclusions: the oxygen depletion hypothesis remains controversial because oxygen in most normal tissues cannot be depleted by flash radiation according to the mathematical analysis with this model and experiments on the expression and distribution of the hypoxia-inducible factors.',\n",
       "  'categories': 'physics.med-ph q-bio.to',\n",
       "  'doi': '10.1667/rade-20-00260.1',\n",
       "  'created': '2020-01-29',\n",
       "  'updated': '',\n",
       "  'authors': ['ankang hu',\n",
       "   'rui qiu',\n",
       "   'zhen wu',\n",
       "   'chunyan li',\n",
       "   'hui zhang',\n",
       "   'junli li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2001.10788'},\n",
       " {'title': 'on the similarity of deep learning representations across didactic and   adversarial examples',\n",
       "  'id': '2002.06816',\n",
       "  'abstract': 'the increasing use of deep neural networks (dnns) has motivated a parallel endeavor: the design of adversaries that profit from successful misclassifications. however, not all adversarial examples are crafted for malicious purposes. for example, real world systems often contain physical, temporal, and sampling variability across instrumentation. adversarial examples in the wild may inadvertently prove deleterious for accurate predictive modeling. conversely, naturally occurring covariance of image features may serve didactic purposes. here, we studied the stability of deep learning representations for neuroimaging classification across didactic and adversarial conditions characteristic of mri acquisition variability. we show that representational similarity and performance vary according to the frequency of adversarial examples in the input space.',\n",
       "  'categories': 'cs.cv cs.lg eess.iv q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2020-02-17',\n",
       "  'updated': '2024-09-16',\n",
       "  'authors': ['pk douglas', 'farzad vasheghani farahani'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2002.06816'},\n",
       " {'title': 'feedback identification of conductance-based models',\n",
       "  'id': '2002.09626',\n",
       "  'abstract': 'this paper applies the classical prediction error method (pem) to the estimation of nonlinear discrete-time models of neuronal systems subject to input-additive noise. while the nonlinear system exhibits excitability, bifurcations, and limit-cycle oscillations, we prove consistency of the parameter estimation procedure under output feedback. hence, this paper provides a rigorous framework for the application of conventional nonlinear system identification methods to discrete-time stochastic neuronal systems. the main result exploits the elementary property that conductance-based models of neurons have an exponentially contracting inverse dynamics. this property is implied by the voltage-clamp experiment, which has been the fundamental modeling experiment of neurons ever since the pioneering work of hodgkin and huxley.',\n",
       "  'categories': 'eess.sy cs.sy q-bio.nc',\n",
       "  'doi': '10.1016/j.automatica.2020.109297',\n",
       "  'created': '2020-02-22',\n",
       "  'updated': '2020-12-16',\n",
       "  'authors': ['thiago b. burghi', 'maarten schoukens', 'rodolphe sepulchre'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2002.09626'},\n",
       " {'title': 'instability and fingering of interfaces in growing tissue',\n",
       "  'id': '2003.04601',\n",
       "  'abstract': 'interfaces in tissues are ubiquitous, both between tissue and environment as well as between populations of different cell types. the propagation of an interface can be driven mechanically. % e.g. by a difference in the respective homeostatic stress of the different cell types. computer simulations of growing tissues are employed to study the stability of the interface between two tissues on a substrate. from a mechanical perspective, the dynamics and stability of this system is controlled mainly by four parameters of the respective tissues: (i) the homeostatic stress (ii) cell motility (iii) tissue viscosity and (iv) substrate friction. for propagation driven by a difference in homeostatic stress, the interface is stable for tissue-specific substrate friction even for very large differences of homeostatic stress; however, it becomes unstable above a critical stress difference when the tissue with the larger homeostatic stress has a higher viscosity. a small difference in directed bulk motility between the two tissues suffices to result in propagation with a stable interface, even for otherwise identical tissues. larger differences in motility force, however, result in a finite-wavelength instability of the interface. interestingly, the instability is apparently bound by nonlinear effects and the amplitude of the interface undulations only grows to a finite value in time.',\n",
       "  'categories': 'q-bio.to',\n",
       "  'doi': '10.1088/1367-2630/ab9e88',\n",
       "  'created': '2020-03-10',\n",
       "  'updated': '',\n",
       "  'authors': ['tobias büscher',\n",
       "   'angel l. diez',\n",
       "   'gerhard gompper',\n",
       "   'jens elgeti'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2003.04601'},\n",
       " {'title': 'morphological reconstruction of detached dendritic spines via geodesic   path prediction',\n",
       "  'id': '2003.08809',\n",
       "  'abstract': 'morphological reconstruction of dendritic spines from fluorescent microscopy is a critical open problem in neuro-image analysis. existing segmentation tools are ill-equipped to handle thin spines with long, poorly illuminated neck membranes. we address this issue, and introduce an unsupervised path prediction technique based on a stochastic framework which seeks the optimal solution from a path-space of possible spine neck reconstructions. our method is specifically designed to reduce bias due to outliers, and is adept at reconstructing challenging shapes from images plagued by noise and poor contrast. experimental analyses on two photon microscopy data demonstrate the efficacy of our method, where an improvement of 12.5% is observed over the state-of-the-art in terms of mean absolute reconstruction error.',\n",
       "  'categories': 'eess.iv q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2020-03-19',\n",
       "  'updated': '2020-09-21',\n",
       "  'authors': ['sammit jain',\n",
       "   'suvadip mukherjee',\n",
       "   'lydia danglot',\n",
       "   'jean-christophe olivo-marin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2003.08809'},\n",
       " {'title': 'morphogen-regulated contact-mediated signaling between cells can drive   the transitions underlying body segmentation in vertebrates',\n",
       "  'id': '2003.12493',\n",
       "  'abstract': 'we propose a unified mechanism that reproduces the sequence of dynamical transitions observed during somitogenesis, the process of body segmentation during embryonic development, that is invariant across all vertebrate species. this is achieved by combining inter-cellular interactions mediated via receptor-ligand coupling with global spatial heterogeneity introduced through a morphogen gradient known to occur along the anteroposterior axis. our model reproduces synchronized oscillations in the gene expression in cells at the anterior of the presomitic mesoderm (psm) as it grows by adding new cells at its posterior, followed by traveling waves and subsequent arrest of activity, with the eventual appearance of somite-like patterns. this framework integrates a boundary-organized pattern formation mechanism, which uses positional information provided by a morphogen gradient, with the coupling-mediated self-organized emergence of collective dynamics, to explain the processes that lead to segmentation.',\n",
       "  'categories': 'q-bio.to nlin.ps physics.bio-ph',\n",
       "  'doi': '10.1088/1478-3975/ac31a3',\n",
       "  'created': '2020-03-27',\n",
       "  'updated': '2021-09-22',\n",
       "  'authors': ['chandrashekar kuyyamudi', 'shakti n. menon', 'sitabhra sinha'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2003.12493'},\n",
       " {'title': 'electrostatic precipitation of exhaled particles for tensiometric   examination of pulmonary surfactant',\n",
       "  'id': '2004.01399',\n",
       "  'abstract': 'objective: collecting exhaled particles that represent small droplets of the alveolar lining fluid shows great promise as a tool for pulmonary surfactant (ps) sampling. for this purpose, we present a setup consisting of two modules, namely, a module for droplet collecting, called an electrostatic aerosol trapping (esat) system, and a measurement module for studying ps properties. we suggest the way how to extract numerical values from the experimental data associated with ps properties.   methods: the operating principle of esat is based on the electrostatic precipitation of exhaled particles. the native material was collected directly on the water surface, where an accumulated adsorbed film of ps was examined with tensiometric method. the modified capillary waves method adapted to study small volume samples was utilized. the efficiency of the setup was verified in the experiments with healthy subjects.   conclusion: the accumulation of ps components on the water surface in an amount sufficient for tensiometric study was reported. it was shown how to extract the numerical values from the experimental data characterizing ps properties.   significance: the idea underlying the new concept used in this study may give impetus to further development of point-of-care facilities for collecting ps samples and for their express analysis.',\n",
       "  'categories': 'q-bio.to physics.bio-ph',\n",
       "  'doi': '10.1016/j.jaerosci.2020.105622',\n",
       "  'created': '2020-04-03',\n",
       "  'updated': '',\n",
       "  'authors': ['andrey shmyrov',\n",
       "   'alexey mizev',\n",
       "   'irina mizeva',\n",
       "   'anastasia shmyrova'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2004.01399'},\n",
       " {'title': \"search via parallel l\\\\'evy walks on $\\\\mathbb{z}^2$\",\n",
       "  'id': '2004.01562',\n",
       "  'abstract': \"motivated by the l\\\\'evy foraging hypothesis -- the premise that various animal species have adapted to follow l\\\\'evy walks to optimize their search efficiency -- we study the parallel hitting time of l\\\\'evy walks on the infinite two-dimensional grid. we consider $k$ independent discrete-time l\\\\'evy walks, with the same exponent $\\\\alpha \\\\in(1,\\\\infty)$, that start from the same node, and analyze the number of steps until the first walk visits a given target at distance $\\\\ell$. we show that for any choice of $k$ and $\\\\ell$ from a large range, there is a unique optimal exponent $\\\\alpha_{k,\\\\ell} \\\\in (2,3)$, for which the hitting time is $\\\\tilde o(\\\\ell^2/k)$ w.h.p., while modifying the exponent by an $\\\\epsilon$ term increases the hitting time by a polynomial factor, or the walks fail to hit the target almost surely. based on that, we propose a surprisingly simple and effective parallel search strategy, for the setting where $k$ and $\\\\ell$ are unknown: the exponent of each l\\\\'evy walk is just chosen independently and uniformly at random from the interval $(2,3)$. this strategy achieves optimal search time (modulo polylogarithmic factors) among all possible algorithms (even centralized ones that know $k$). our results should be contrasted with a line of previous work showing that the exponent $\\\\alpha = 2$ is optimal for various search problems. in our setting of $k$ parallel walks, we show that the optimal exponent depends on $k$ and $\\\\ell$, and that randomizing the choice of the exponents works simultaneously for all $k$ and $\\\\ell$.\",\n",
       "  'categories': 'cs.dc math.pr q-bio.qm',\n",
       "  'doi': '10.1145/3465084.3467921',\n",
       "  'created': '2020-04-03',\n",
       "  'updated': '2024-03-07',\n",
       "  'authors': ['andrea clementi',\n",
       "   \"francesco d'amore\",\n",
       "   'george giakkoupis',\n",
       "   'emanuele natale'],\n",
       "  'affiliation': ['coati', 'coati', 'wide', 'coati'],\n",
       "  'url': 'https://arxiv.org/abs/2004.01562'},\n",
       " {'title': 'integrative dynamic structural biology unveils conformers essential for   the oligomerization of a large gtpase',\n",
       "  'id': '2004.04229',\n",
       "  'abstract': \"guanylate binding proteins (gbps) are soluble dynamin-like proteins with structured domains that undergo a conformational transition for gtp-controlled oligomerization to exert their function as part of the innate immune system of mammalian cells - attacking intra-cellular parasites by disrupting their membranes. the structural basis and mechanism of this process is unknown. therefore, we apply neutron spin echo, x-ray scattering, fluorescence, and epr spectroscopy as techniques for integrative dynamic structural biology to human gbp1 (hgbp1). we mapped hgbp1's essential dynamics from nanoseconds to milliseconds by motional spectra of sub-domains. we find a gtp-independent flexibility of the c-terminal effector domain in the $\\\\mu$s-regime and structurally characterize conformers being essential that hgbp1 can open like a pocketknife for oligomerization. this unveils the intrinsic flexibility, a gtp-triggered association of the gtpase-domains and assembly-dependent gtp-hydrolysis as functional design principles of hgbp1 that control its reversible oligomerization in polar assemblies and the subsequent formation of condensates.\",\n",
       "  'categories': 'physics.bio-ph q-bio.bm',\n",
       "  'doi': '10.7554/elife.79565.sa0',\n",
       "  'created': '2020-04-08',\n",
       "  'updated': '',\n",
       "  'authors': ['thomas-otavio peulen',\n",
       "   'carola s. hengstenberg',\n",
       "   'ralf biehl',\n",
       "   'mykola dimura',\n",
       "   'charlotte lorenz',\n",
       "   'alessandro valeri',\n",
       "   'semra ince',\n",
       "   'tobias vöpel',\n",
       "   'bela faragó',\n",
       "   'holger gohlke',\n",
       "   'johann p. klare',\n",
       "   'andreas m. stadler',\n",
       "   'claus a. m. seidel',\n",
       "   'christian herrmann'],\n",
       "  'affiliation': ['chair for molecular physical chemistry, heinrich-heine-university düsseldorf, düsseldorf, germany',\n",
       "   'physical chemistry i, faculty of chemistry and biochemistry, ruhr-university bochum, bochum, germany',\n",
       "   'jülich centre for neutron science jcns and institute for complex system ics, jülich, germany',\n",
       "   'chair for molecular physical chemistry, heinrich-heine-university düsseldorf, düsseldorf, germany',\n",
       "   'jülich centre for neutron science jcns and institute for complex system ics, jülich, germany',\n",
       "   'chair for molecular physical chemistry, heinrich-heine-university düsseldorf, düsseldorf, germany',\n",
       "   'physical chemistry i, faculty of chemistry and biochemistry, ruhr-university bochum, bochum, germany',\n",
       "   'physical chemistry i, faculty of chemistry and biochemistry, ruhr-university bochum, bochum, germany',\n",
       "   'institut laue-langevin, grenoble, france',\n",
       "   'institut für pharmazeutische und medizinische chemie, heinrich-heine-universität düsseldorf, düsseldorf, germany',\n",
       "   'macromolecular structure group, department of physics, university of osnabrück, osnabrück, germany',\n",
       "   'jülich centre for neutron science jcns and institute for complex system ics, jülich, germany',\n",
       "   'chair for molecular physical chemistry, heinrich-heine-university düsseldorf, düsseldorf, germany',\n",
       "   'physical chemistry i, faculty of chemistry and biochemistry, ruhr-university bochum, bochum, germany'],\n",
       "  'url': 'https://arxiv.org/abs/2004.04229'},\n",
       " {'title': 'zig-zag sampling for discrete structures and non-reversible phylogenetic   mcmc',\n",
       "  'id': '2004.08807',\n",
       "  'abstract': 'we construct a zig-zag process targeting a posterior distribution defined on a hybrid state space consisting of both discrete and continuous variables. the construction does not require any assumptions on the structure among discrete variables. we demonstrate our method on two examples in genetics based on the kingman coalescent, showing that the zig-zag process can lead to efficiency gains of up to several orders of magnitude over classical metropolis-hastings algorithms, and that it is well suited to parallel computation. our construction resembles existing techniques for hamiltonian monte carlo on a hybrid state space, which suffers from implementationally and analytically complex boundary crossings when applied to the coalescent. we demonstrate that the continuous-time zig-zag process avoids these complications.',\n",
       "  'categories': 'stat.co math.st q-bio.pe stat.me stat.th',\n",
       "  'doi': '10.1080/10618600.2022.2032722',\n",
       "  'created': '2020-04-19',\n",
       "  'updated': '2022-01-18',\n",
       "  'authors': ['jere koskela'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2004.08807'},\n",
       " {'title': 'algorithm for optimized mrna design improves stability and   immunogenicity',\n",
       "  'id': '2004.10177',\n",
       "  'abstract': 'messenger rna (mrna) vaccines are being used for covid-19, but still suffer from the critical issue of mrna instability and degradation, which is a major obstacle in the storage, distribution, and efficacy of the vaccine. previous work showed that optimizing secondary structure stability lengthens mrna half-life, which, together with optimal codons, increases protein expression. therefore, a principled mrna design algorithm must optimize both structural stability and codon usage to improve mrna efficiency. however, due to synonymous codons, the mrna design space is prohibitively large, e.g., there are $\\\\sim\\\\!10^{632}$ mrnas for the sars-cov-2 spike protein, which poses insurmountable challenges to previous methods. here we provide a surprisingly simple solution to this hard problem by reducing it to a classical problem in computational linguistics, where finding the optimal mrna is akin to finding the most likely sentence among similar sounding alternatives. our algorithm, named lineardesign, takes only 11 minutes for the spike protein, and can jointly optimize stability and codon usage. experimentally, without chemical modification, our designs substantially improve mrna half-life and protein expression in vitro, and dramatically increase antibody response by up to 23$\\\\times$ in vivo, compared to the codon-optimized benchmark. our work enables the exploration of highly stable and efficient designs that are previously unreachable and is a timely tool not only for vaccines but also for mrna medicine encoding all therapeutic proteins (e.g., monoclonal antibodies and anti-cancer drugs).',\n",
       "  'categories': 'q-bio.bm',\n",
       "  'doi': '10.1038/s41586-023-06127-z',\n",
       "  'created': '2020-04-21',\n",
       "  'updated': '2022-03-17',\n",
       "  'authors': ['he zhang',\n",
       "   'liang zhang',\n",
       "   'ang lin',\n",
       "   'congcong xu',\n",
       "   'ziyu li',\n",
       "   'kaibo liu',\n",
       "   'boxiang liu',\n",
       "   'xiaopin ma',\n",
       "   'fanfan zhao',\n",
       "   'weiguo yao',\n",
       "   'hangwen li',\n",
       "   'david h. mathews',\n",
       "   'yujian zhang',\n",
       "   'liang huang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2004.10177'},\n",
       " {'title': \"self-reinforcing directionality generates l\\\\'evy walks without the   power-law assumption\",\n",
       "  'id': '2005.00498',\n",
       "  'abstract': \"we introduce a persistent random walk model with finite velocity and self-reinforcing directionality, which explains how exponentially distributed runs self-organize into truncated l\\\\'evy walks observed in active intracellular transport by chen et. al. [\\\\textit{nat. mat.}, 2015]. we derive the non-homogeneous in space and time, hyperbolic pde for the probability density function (pdf) of particle position. this pdf exhibits a bimodal density (aggregation phenomena) in the superdiffusive regime, which is not observed in classical linear hyperbolic and l\\\\'evy walk models. we find the exact solutions for the first and second moments and criteria for the transition to superdiffusion.\",\n",
       "  'categories': 'cond-mat.stat-mech q-bio.sc',\n",
       "  'doi': '10.1103/physreve.103.022132',\n",
       "  'created': '2020-05-01',\n",
       "  'updated': '2024-02-06',\n",
       "  'authors': ['daniel han',\n",
       "   'marco a. a. da silva',\n",
       "   'nickolay korabel',\n",
       "   'sergei fedotov'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2005.00498'},\n",
       " {'title': 'covid-19 mitigation strategies and overview on results from relevant   studies in europe',\n",
       "  'id': '2005.05249',\n",
       "  'abstract': 'in december 2019, the first patients in wuhan, china were diagnosed with a primary atypical pneumonia, which showed to be unknown and contagious. since then, known as covid-19 disease, the responsible viral pathogen, sars-cov-2, has spread around the world in a pandemic. decisions on how to deal with the crisis are often based on simulations of the pandemic spread of the virus. the results of some of these, as well as their methodology and possibilities for improvement, will be described in more detail in this paper in order to inform beyond the current public health dogma called \"flatten-the-curve\". there are several ways to model an epidemic in order to simulate the spread of diseases. depending on the timeliness, scope and quality of the associated real data, these multivariable models differ in the value of used parameters, but also in the selection of considered influencing factors. it was exemplarily shown that epidemics in their course are simulated more realistically by models that assume subexponential growth. furthermore, various simulations of the covid-19 pandemic were presented in an european perspective, compared against each other and discussed in more detail. it is difficult to estimate how credible the simulations of the pandemic models currently are, so it remains to be seen whether the spread of the pandemic can be effectively reduced by the measures taken. whether a model works well in reality is largely determined by the quality and scope of its underlying data. past studies have shown that countermeasures are able to reduce reproduction numbers or transmission rates in epidemics. in addition to that, the presented modelling study provides a good framework for the creation of subexponential-growth-models for assessing the spread of covid-19.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2020-05-11',\n",
       "  'updated': '2024-01-29',\n",
       "  'authors': ['philipp heider'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2005.05249'},\n",
       " {'title': 'monte carlo simulation of sars-cov-2 radiation-induced inactivation for   vaccine development',\n",
       "  'id': '2005.06201',\n",
       "  'abstract': 'immunization with an inactivated virus is one of the strategies currently being tested towards developing a sars-cov-2 vaccine. one of the methods used to inactivate viruses is exposure to high doses of ionizing radiation to damage their nucleic acids. although gamma-rays effectively induce lesions in the rna, envelope proteins are also highly damaged in the process. this in turn may alter their antigenic properties, affecting their capacity to induce an adaptive immune response able to confer effective protection. here, we modelled the impact of sparsely and densely ionizing radiation on sars-cov-2 using the monte carlo toolkit geant4-dna. with a realistic 3d target virus model, we calculated the expected number of lesions in the spike and membrane proteins, as well as in the viral rna. we show that gamma-rays produce significant spike protein damage, but densely ionizing charged particles induce less membrane damage for the same level of rna lesions, because a single ion traversal through the nuclear envelope is sufficient to inactivate the virus. we propose that accelerated charged particles produce inactivated viruses with little structural damage to envelope proteins, thereby representing a new and effective tool for developing vaccines against sars-cov-2 and other enveloped viruses.',\n",
       "  'categories': 'physics.bio-ph q-bio.bm',\n",
       "  'doi': '10.1667/rade-20-00241.1',\n",
       "  'created': '2020-05-13',\n",
       "  'updated': '2021-01-08',\n",
       "  'authors': ['ziad francis',\n",
       "   'sebastien incerti',\n",
       "   'sara a. zein',\n",
       "   'nathanael lampe',\n",
       "   'carlos a. guzman',\n",
       "   'marco durante'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2005.06201'},\n",
       " {'title': 'a minimal model for structure, dynamics, and tension of monolayered cell   colonies',\n",
       "  'id': '2006.04519',\n",
       "  'abstract': 'the motion of cells in tissues is an ubiquitous phenomenon. in particular, in monolayered cell colonies in vitro, pronounced collective behavior with swirl-like motion has been observed deep within a cell colony, while at the same time, the colony remains cohesive, with not a single cell escaping at the edge. thus, the colony displays liquid-like properties inside, in coexistence with a cell-free \"vacuum\" outside. how can adhesion be strong enough to keep cells together, while at the same time not jam the system in a glassy state? what kind of minimal model can describe such a behavior? which other signatures of activity arise from the internal fluidity? we propose a novel active brownian particle model with attraction, in which the interaction potential has a broad minimum to give particles enough wiggling space to be collectively in the fluid state. we demonstrate that for moderate propulsion, this model can generate the fluid-vacuum coexistence described above. in addition, the combination of the fluid nature of the colony with cohesion leads to preferred orientation of the cell polarity, pointing outward, at the edge, which in turn gives rise to a tensile stress in the colony -- as observed experimentally for epithelial sheets. for stronger propulsion, collective detachment of cell clusters is predicted. further addition of an alignment preference of cell polarity and velocity direction results in enhanced coordinated, swirl-like motion, increased tensile stress and cell-cluster detachment.',\n",
       "  'categories': 'physics.bio-ph cond-mat.soft q-bio.cb',\n",
       "  'doi': '10.1038/s42005-020-00515-x',\n",
       "  'created': '2020-06-08',\n",
       "  'updated': '',\n",
       "  'authors': ['debarati sarkar', 'gerhard gompper', 'jens elgeti'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2006.04519'},\n",
       " {'title': 'age-structured estimation of covid-19 icu demand from low quality data',\n",
       "  'id': '2006.06530',\n",
       "  'abstract': 'we sample aggravated cases following age-structured probabilities from confirmed cases and use icu occupation data to find a subnotification factor. a logistic fit is then employed to project the progression of the covid-19 epidemic with plateau scenarios taken from locations that have reached this stage. finally, the logistic curve found is corrected by the subnotification factor and sampled to project the future demand for icu beds.',\n",
       "  'categories': 'physics.soc-ph q-bio.pe stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2020-06-08',\n",
       "  'updated': '2020-06-15',\n",
       "  'authors': ['rodrigo veiga', 'rodrigo murta', 'renato vicente'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2006.06530'},\n",
       " {'title': 'pycro-manager: open-source software for integrated microscopy hardware   control and image processing',\n",
       "  'id': '2006.11330',\n",
       "  'abstract': '{\\\\mu}manager, an open-source microscopy acquisition software, has been an essential tool for many microscopy experiments over the past 15 years, but is not easy to use for experiments in which image acquisition and analysis are closely coupled. this is because {\\\\mu}manager libraries are written in c++ and java, whereas image processing is increasingly carried out with data science and machine learning tools most easily accessible through the python programming language. we present pycro-manager, a tool that enables rapid development of such experiments, while also providing access to the wealth of existing tools within {\\\\mu}manager through python.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.1038/s41592-021-01087-6',\n",
       "  'created': '2020-06-19',\n",
       "  'updated': '',\n",
       "  'authors': ['henry pinkard', 'nico stuurman', 'laura waller'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2006.11330'},\n",
       " {'title': 'toward the biological model of the hippocampus as the successor   representation agent',\n",
       "  'id': '2006.11975',\n",
       "  'abstract': 'the hippocampus is an essential brain region for spatial memory and learning. recently, a theoretical model of the hippocampus based on temporal difference (td) learning has been published. inspired by the successor representation (sr) learning algorithms, which decompose value function of td learning into reward and state transition, they argued that the rate of firing of ca1 place cells in the hippocampus represents the probability of state transition. this theory, called predictive map theory, claims that the hippocampus representing space learns the probability of transition from the current state to the future state. the neural correlates of expecting the future state are the firing rates of the ca1 place cells. this explanation is plausible for the results recorded in behavioral experiments, but it is lacking the neurobiological implications.   modifying the sr learning algorithm added biological implications to the predictive map theory. similar with the simultaneous needs of information of the current and future state in the sr learning algorithm, the ca1 place cells receive two inputs from ca3 and entorhinal cortex. mathematical transformation showed that the sr learning algorithm is equivalent to the heterosynaptic plasticity rule. the heterosynaptic plasticity phenomena in ca1 were discussed and compared with the modified sr update rule. this study attempted to interpret the td algorithm as the neurobiological mechanism occurring in place learning, and to integrate the neuroscience and artificial intelligence approaches in the field.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1016/j.biosystems.2022.104612',\n",
       "  'created': '2020-06-21',\n",
       "  'updated': '2021-10-26',\n",
       "  'authors': ['hyunsu lee'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2006.11975'},\n",
       " {'title': 'method to monitor the evolution of an epidemic in real time',\n",
       "  'id': '2007.01508',\n",
       "  'abstract': 'the emergence of an epidemic evokes the need to monitor its spread and assess and validate any mitigation measures enacted by governments and administrative bodies in real time. we present here a method to observe and quantify this spread and the response of affected populations and governing bodies and apply it to covid-19 as a case study. this method provides means to simultaneously track in real time quantities such as the mortality and the recovery rates as well as the number of new infections caused by an infected person. with sufficient data, this method enables thorough monitoring and assessment of an epidemic without assumptions regarding the evolution of the pandemic in the future.',\n",
       "  'categories': 'q-bio.pe physics.soc-ph',\n",
       "  'doi': '10.1016/j.physleta.2020.127074',\n",
       "  'created': '2020-07-03',\n",
       "  'updated': '',\n",
       "  'authors': ['justin trujillo', 'valerica raicu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2007.01508'},\n",
       " {'title': 'bayesian inference of minimally complex models with interactions of   arbitrary order',\n",
       "  'id': '2008.00520',\n",
       "  'abstract': 'finding the model that best describes a high-dimensional dataset is a daunting task, even more so if one aims to consider all possible high-order patterns of the data, going beyond pairwise models. for binary data, we show that this task becomes feasible when restricting the search to a family of simple models, that we call minimally complex models (mcms). mcms are maximum entropy models that have interactions of arbitrarily high order grouped into independent components of minimal complexity. they are simple in information-theoretic terms, which means they can only fit well certain types of data patterns and are therefore easy to falsify. we show that bayesian model selection restricted to these models is computationally feasible and has many advantages. first, the model evidence, which balances goodness-of-fit against complexity, can be computed efficiently without any parameter fitting, enabling very fast explorations of the space of mcms. second, the family of mcms is invariant under gauge transformations, which can be used to develop a representation-independent approach to statistical modeling. for small systems (up to 15 variables), combining these two results allows us to select the best mcm among all, even though the number of models is already extremely large. for larger systems, we propose simple heuristics to find optimal mcms in reasonable times. besides, inference and sampling can be performed without any computational effort. finally, because mcms have interactions of any order, they can reveal the presence of important high-order dependencies in the data, providing a new approach to explore high-order dependencies in complex systems. we apply our method to synthetic data and real-world examples, illustrating how mcms portray the structure of dependencies among variables in a simple manner, extracting falsifiable predictions on symmetries and invariance from the data.',\n",
       "  'categories': 'cs.ai math.st physics.data-an q-bio.qm stat.th',\n",
       "  'doi': '',\n",
       "  'created': '2020-08-02',\n",
       "  'updated': '2024-08-27',\n",
       "  'authors': ['clélia de mulatier', 'matteo marsili'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2008.00520'},\n",
       " {'title': 'what do we mean by the dimensionality of behavior?',\n",
       "  'id': '2008.09574',\n",
       "  'abstract': 'there is growing effort in the \"physics of behavior\" that aims at complete quantitative characterization of animal movements under more complex, naturalistic conditions. one reaction to the resulting explosion of data is the search for low dimensional structure. here i try to define more clearly what we mean by the dimensionality of behavior, where observable behavior may consist either of continuous trajectories or sequences of discrete states. this discussion also serves to isolate situations in which the dimensionality of behavior is effectively infinite. i conclude with some more general perspectives about the importance of quantitative phenomenology.',\n",
       "  'categories': 'q-bio.nc cond-mat.stat-mech q-bio.qm',\n",
       "  'doi': '10.1073/pnas.2021860119',\n",
       "  'created': '2020-08-21',\n",
       "  'updated': '',\n",
       "  'authors': ['william bialek'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2008.09574'},\n",
       " {'title': 'searching for long time scales without fine tuning',\n",
       "  'id': '2008.11674',\n",
       "  'abstract': 'most of animal and human behavior occurs on time scales much longer than the response times of individual neurons. in many cases, it is plausible that these long time scales emerge from the recurrent dynamics of electrical activity in networks of neurons. in linear models, time scales are set by the eigenvalues of a dynamical matrix whose elements measure the strengths of synaptic connections between neurons. it is not clear to what extent these matrix elements need to be tuned in order to generate long time scales; in some cases, one needs not just a single long time scale but a whole range. starting from the simplest case of random symmetric connections, we combine maximum entropy and random matrix theory methods to construct ensembles of networks, exploring the constraints required for long time scales to become generic. we argue that a single long time scale can emerge generically from realistic constraints, but a full spectrum of slow modes requires more tuning. langevin dynamics that generates patterns of synaptic connections drawn from these ensembles involves a combination of hebbian learning and activity-dependent synaptic scaling.',\n",
       "  'categories': 'physics.bio-ph nlin.ao q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2020-08-26',\n",
       "  'updated': '2024-08-22',\n",
       "  'authors': ['xiaowen chen', 'william bialek'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2008.11674'},\n",
       " {'title': 'a meta-analysis of boolean network models reveals design principles of   gene regulatory networks',\n",
       "  'id': '2009.01216',\n",
       "  'abstract': 'gene regulatory networks (grns) play a central role in cellular decision-making. understanding their structure and how it impacts their dynamics constitutes thus a fundamental biological question. grns are frequently modeled as boolean networks, which are intuitive, simple to describe, and can yield qualitative results even when data is sparse. we assembled the largest repository of expert-curated boolean grn models. a meta-analysis of this diverse set of models reveals several design principles. grns exhibit more canalization, redundancy and stable dynamics than expected. moreover, they are enriched for certain recurring network motifs. this raises the important question why evolution favors these design mechanisms.',\n",
       "  'categories': 'q-bio.mn math.ds nlin.ao',\n",
       "  'doi': '10.1126/sciadv.adj0822',\n",
       "  'created': '2020-09-02',\n",
       "  'updated': '2023-09-11',\n",
       "  'authors': ['claus kadelka',\n",
       "   'taras-michael butrie',\n",
       "   'evan hilton',\n",
       "   'jack kinseth',\n",
       "   'addison schmidt',\n",
       "   'haris serdarevic'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2009.01216'},\n",
       " {'title': 'a joint network optimization framework to predict clinical severity from   resting state functional mri data',\n",
       "  'id': '2009.03238',\n",
       "  'abstract': 'we propose a novel optimization framework to predict clinical severity from resting state fmri (rs-fmri) data. our model consists of two coupled terms. the first term decomposes the correlation matrices into a sparse set of representative subnetworks that define a network manifold. these subnetworks are modeled as rank-one outer-products which correspond to the elemental patterns of co-activation across the brain; the subnetworks are combined via patient-specific non-negative coefficients. the second term is a linear regression model that uses the patient-specific coefficients to predict a measure of clinical severity. we validate our framework on two separate datasets in a ten fold cross validation setting. the first is a cohort of fifty-eight patients diagnosed with autism spectrum disorder (asd). the second dataset consists of sixty three patients from a publicly available asd database. our method outperforms standard semi-supervised frameworks, which employ conventional graph theoretic and statistical representation learning techniques to relate the rs-fmri correlations to behavior. in contrast, our joint network optimization framework exploits the structure of the rs-fmri correlation matrices to simultaneously capture group level effects and patient heterogeneity. finally, we demonstrate that our proposed framework robustly identifies clinically relevant networks characteristic of asd.',\n",
       "  'categories': 'q-bio.nc cs.lg eess.sp stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2020-08-27',\n",
       "  'updated': '2024-11-21',\n",
       "  'authors': [\"niharika shimona d'souza\",\n",
       "   'mary beth nebel',\n",
       "   'nicholas wymbs',\n",
       "   'stewart h. mostofsky',\n",
       "   'archana venkataraman'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2009.03238'},\n",
       " {'title': 'dynamical systems on large networks with predator-prey interactions are   stable and exhibit oscillations',\n",
       "  'id': '2009.11211',\n",
       "  'abstract': \"we analyse the stability of linear dynamical systems defined on sparse, random graphs with predator-prey, competitive, and mutualistic interactions. these systems are aimed at modelling the stability of fixed points in large systems defined on complex networks, such as, ecosystems consisting of a large number of species that interact through a food-web. we develop an exact theory for the spectral distribution and the leading eigenvalue of the corresponding sparse jacobian matrices. this theory reveals that the nature of local interactions have a strong influence on system's stability. we show that, in general, linear dynamical systems defined on random graphs with a prescribed degree distribution of unbounded support are unstable if they are large enough, implying a tradeoff between stability and diversity. remarkably, in contrast to the generic case, antagonistic systems that only contain interactions of the predator-prey type can be stable in the infinite size limit. this qualitatively feature for antagonistic systems is accompanied by a peculiar oscillatory behaviour of the dynamical response of the system after a perturbation, when the mean degree of the graph is small enough. moreover, for antagonistic systems we also find that there exist a dynamical phase transition and critical mean degree above which the response becomes non-oscillatory.\",\n",
       "  'categories': 'cond-mat.stat-mech cond-mat.dis-nn q-bio.pe',\n",
       "  'doi': '10.1103/physreve.105.014305',\n",
       "  'created': '2020-09-23',\n",
       "  'updated': '2024-07-22',\n",
       "  'authors': ['andrea marcello mambuca', 'chiara cammarota', 'izaak neri'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2009.11211'},\n",
       " {'title': 'deep neural network: an efficient and optimized machine learning   paradigm for reducing genome sequencing error',\n",
       "  'id': '2010.03420',\n",
       "  'abstract': 'genomic data i used in many fields but, it has become known that most of the platforms used in the sequencing process produce significant errors. this means that the analysis and inferences generated from these data may have some errors that need to be corrected. on the two main types of genome errors - substitution and indels - our work is focused on correcting indels. a deep learning approach was used to correct the errors in sequencing the chosen dataset',\n",
       "  'categories': 'q-bio.gn cs.cv',\n",
       "  'doi': '',\n",
       "  'created': '2020-10-06',\n",
       "  'updated': '',\n",
       "  'authors': ['ferdinand kartriku', 'robert sowah', 'charles saah'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2010.03420'},\n",
       " {'title': 'quantification of ebola virus replication kinetics in vitro',\n",
       "  'id': '2010.09898',\n",
       "  'abstract': 'mathematical modelling has successfully been used to provide quantitative descriptions of many viral infections, but for the ebola virus, which requires biosafety level 4 facilities for experimentation, modelling can play a crucial role. ebola modelling efforts have primarily focused on in vivo virus kinetics, e.g., in animal models, to aid the development of antivirals and vaccines. but, thus far, these studies have not yielded a detailed specification of the infection cycle, which could provide a foundational description of the virus kinetics and thus a deeper understanding of their clinical manifestation. here, we obtain a diverse experimental data set of the ebola infection in vitro, and then make use of bayesian inference methods to fully identify parameters in a mathematical model of the infection. our results provide insights into the distribution of time an infected cell spends in the eclipse phase (the period between infection and the start of virus production), as well as the rate at which infectious virions lose infectivity. we suggest how these results can be used in future models to describe co-infection with defective interfering particles, which are an emerging alternative therapeutic.',\n",
       "  'categories': 'q-bio.cb',\n",
       "  'doi': '10.1371/journal.pcbi.1008375',\n",
       "  'created': '2020-10-19',\n",
       "  'updated': '',\n",
       "  'authors': ['laura e. liao',\n",
       "   'jonathan carruthers',\n",
       "   'sophie j. smither',\n",
       "   'cl4 virology team',\n",
       "   'simon a. weller',\n",
       "   'diane williamson',\n",
       "   'thomas r. laws',\n",
       "   'isabel garcia-dorival',\n",
       "   'julian hiscox',\n",
       "   'benjamin p. holder',\n",
       "   'catherine a. a. beauchemin',\n",
       "   'alan s. perelson',\n",
       "   'martin lopez-garcia',\n",
       "   'grant lythe',\n",
       "   'john barr',\n",
       "   'carmen molina-paris'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2010.09898'},\n",
       " {'title': 'beating temporal phase sensitivity limit in off-axis interferometry   based quantitative phase microscopy',\n",
       "  'id': '2010.11477',\n",
       "  'abstract': 'phase sensitivity determines the lowest optical path length (opl) value that can be detected from the noise floor in a quantitative phase microscopy (qpm) system. the temporal phase sensitivity is known to be limited by both photon shot-noise and a variety of noise sources from electronic devices and environment. to beat temporal phase sensitivity limit, we explore different ways to reduce different noise factors in off-axis interferometry-based qpm using laser-illumination. using a high electron-well-capacity camera, we measured the temporal phase sensitivity values using non-common-path and common-path interferometry based qpm systems under different environmental conditions. a frame summing method and a spatiotemporal filtering method are further used to reduce the noise contributions, thus enabling us to push the overall temporal phase sensitivity to less than 2 picometers.',\n",
       "  'categories': 'q-bio.qm physics.bio-ph physics.optics',\n",
       "  'doi': '10.1063/5.0034515',\n",
       "  'created': '2020-10-22',\n",
       "  'updated': '',\n",
       "  'authors': ['yujie nie', 'renjie zhou'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2010.11477'},\n",
       " {'title': 'scaling study of diffusion in dynamic crowded spaces',\n",
       "  'id': '2011.02444',\n",
       "  'abstract': 'we formulate a scaling theory for the long-time diffusive motion in a space occluded by a high density of moving obstacles in dimensions 1, 2 and 3. our tracers diffuse anomalously over many decades in time, before reaching a diffusive steady state with an effective diffusion constant $d_\\\\mathrm{eff}$, which depends on the obstacle diffusivity and density. the scaling of $d_\\\\mathrm{eff}$, above and below a critical regime, is characterized by two independent critical parameters: the conductivity exponent $\\\\mu$, also found in models with frozen obstacles, and an exponent $\\\\psi$, which quantifies the effect of obstacle diffusivity.',\n",
       "  'categories': 'cond-mat.stat-mech q-bio.sc',\n",
       "  'doi': '10.1088/1751-8121/ad8496',\n",
       "  'created': '2020-11-04',\n",
       "  'updated': '2024-10-09',\n",
       "  'authors': ['h. bendekgey', 'g. huber', 'd. yllanes'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2011.02444'},\n",
       " {'title': 'homeostasis in networks with multiple input nodes and robustness in   bacterial chemotaxis',\n",
       "  'id': '2011.04415',\n",
       "  'abstract': \"a biological system achieve homeostasis when there is a regulated quantity that is maintained within a narrow range of values. here we consider homeostasis as a phenomenon of network dynamics. in this context, we improve a general theory for the analysis of homeostasis in network dynamical systems with distinguished input and output nodes, called `input-output networks'. the theory allows one to define `homeostasis types' of a given network in a `model independent' fashion, in the sense that the classification depends on the network topology rather than on the specific model equations. each `homeostasis type' represents a possible mechanism for generating homeostasis and is associated with a suitable `subnetwork motif' of the original network. our contribution is an extension of the theory to the case of networks with multiple input nodes. to showcase our theory, we apply it to bacterial chemotaxis, a paradigm for homeostasis in biochemical systems. by considering a representative model of escherichia coli chemotaxis, we verify that the corresponding abstract network has multiple input nodes. thus showing that our extension of the theory allows for the inclusion of an important class of models that were previously out of reach. moreover, from our abstract point of view, the occurrence of homeostasis in the studied model is caused by a new mechanism, called input counterweight homeostasis. this new homeostasis mechanism was discovered in the course of our investigation and is generated by a balancing between the several input nodes of the network -- therefore, it requires the existence of at least two input nodes to occur. finally, the framework developed here allows one to formalize a notion of `robustness' of homeostasis based on the concept of `genericity' from the theory dynamical systems. we discuss how this kind of robustness of homeostasis appears in the chemotaxis model.\",\n",
       "  'categories': 'q-bio.mn math.ds physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2020-11-05',\n",
       "  'updated': '2022-01-25',\n",
       "  'authors': ['joão luiz de oliveira madeira', 'fernando antoneli'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2011.04415'},\n",
       " {'title': 'exploring the impact of under-reported cases on the covid-19   spatiotemporal distribution using healthcare worker infection data',\n",
       "  'id': '2011.04892',\n",
       "  'abstract': 'a timely understanding of the spatiotemporal pattern and development trend of covid-19 is critical for timely prevention and control. however, the under-reporting of cases is widespread in fields associated with public health. it is also possible to draw biased inferences and formulate inappropriate prevention and control policies if the phenomenon of under-reporting is not taken into account. therefore, in this paper, a novel framework was proposed to explore the impact of under-reporting on covid-19 spatiotemporal distributions, and empirical analysis was carried out using infection data of healthcare workers in wuhan and hubei (excluding wuhan). the results show that (1) the lognormal distribution was the most suitable to describe the evolution of epidemic with time; (2) the estimated peak infection time of the reported cases lagged the peak infection time of the healthcare worker cases, and the estimated infection time interval of the reported cases was smaller than that of the healthcare worker cases. (3) the impact of under-reporting cases on the early stages of the pandemic was greater than that on its later stages, and the impact on the early onset area was greater than that on the late onset area. (4) although the number of reported cases was lower than the actual number of cases, a high spatial correlation existed between the cumulatively reported cases and healthcare worker cases. the proposed framework of this study is highly extensible, and relevant researchers can use data sources from other counties to carry out similar research.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1016/j.cities.2022.103593',\n",
       "  'created': '2020-11-09',\n",
       "  'updated': '2022-01-20',\n",
       "  'authors': ['peixiao wang', 'tao hu', 'hongqiang liu', 'xinyan zhu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2011.04892'},\n",
       " {'title': 'dynamics of a stochastic covid-19 epidemic model with jump-diffusion',\n",
       "  'id': '2011.06280',\n",
       "  'abstract': \"for a stochastic covid-19 model with jump-diffusion, we prove the existence and uniqueness of the global positive solution. we also investigate some conditions for the extinction and persistence of the disease. we calculate the threshold of the stochastic epidemic system which determines the extinction or permanence of the disease at different intensities of the stochastic noises. this threshold is denoted by $\\\\xi$ which depends on the white and jump noises. the effects of these noises on the dynamics of the model are studied. the numerical experiments show that the random perturbation introduced in the stochastic model suppresses disease outbreaks as compared to its deterministic counterpart. in other words, the impact of the noises on the extinction and persistence is high. when the noise is large or small, our numerical findings show that the covid-19 vanishes from the population if $\\\\xi <1;$ whereas the epidemic can't go out of control if $\\\\xi >1.$ from this, we observe that white noise and jump noise have a significant effect on the spread of covid-19 infection, i.e., we can conclude that the stochastic model is more realistic than the deterministic one. finally, to illustrate this phenomenon, we put some numerical simulations.\",\n",
       "  'categories': 'q-bio.pe math.ds physics.soc-ph',\n",
       "  'doi': '10.1186/s13662-021-03396-8',\n",
       "  'created': '2020-11-12',\n",
       "  'updated': '2024-10-16',\n",
       "  'authors': ['almaz tesfay',\n",
       "   'tareq saeed',\n",
       "   'anwar zeb',\n",
       "   'daniel tesfay',\n",
       "   'anas khalaf',\n",
       "   'james brannan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2011.06280'},\n",
       " {'title': 'nonparametric goodness-of-fit testing for parametric covariate models in   pharmacometric analyses',\n",
       "  'id': '2011.07539',\n",
       "  'abstract': 'the characterization of covariate effects on model parameters is a crucial step during pharmacokinetic/pharmacodynamic analyses. while covariate selection criteria have been studied extensively, the choice of the functional relationship between covariates and parameters, however, has received much less attention. often, a simple particular class of covariate-to-parameter relationships (linear, exponential, etc.) is chosen ad hoc or based on domain knowledge, and a statistical evaluation is limited to the comparison of a small number of such classes. goodness-of-fit testing against a nonparametric alternative provides a more rigorous approach to covariate model evaluation, but no such test has been proposed so far. in this manuscript, we derive and evaluate nonparametric goodness-of-fit tests for parametric covariate models, the null hypothesis, against a kernelized tikhonov regularized alternative, transferring concepts from statistical learning to the pharmacological setting. the approach is evaluated in a simulation study on the estimation of the age-dependent maturation effect on the clearance of a monoclonal antibody. scenarios of varying data sparsity and residual error are considered. the goodness-of-fit test correctly identified misspecified parametric models with high power for relevant scenarios. the case study provides proof-of-concept of the feasibility of the proposed approach, which is envisioned to be beneficial for applications that lack well-founded covariate models.',\n",
       "  'categories': 'stat.me q-bio.qm stat.ap',\n",
       "  'doi': '10.1002/psp4.12614',\n",
       "  'created': '2020-11-15',\n",
       "  'updated': '',\n",
       "  'authors': ['niklas hartung',\n",
       "   'martin wahl',\n",
       "   'abhishake rastogi',\n",
       "   'wilhelm huisinga'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2011.07539'},\n",
       " {'title': 'reconciling security and utility in next-generation epidemic risk   mitigation systems',\n",
       "  'id': '2011.08069',\n",
       "  'abstract': \"epidemics like the recent covid-19 require proactive contact tracing and epidemiological analysis to predict and subsequently contain infection transmissions. the proactive measures require large scale data collection, which simultaneously raise concerns regarding users' privacy. digital contact tracing systems developed in response to covid-19 either collected extensive data for effective analytics at the cost of users' privacy or collected minimal data for the sake of user privacy but were ineffective in predicting and mitigating the epidemic risks. we present silmarillion--in preparation for future epidemics--a system that reconciles user's privacy with rich data collection for higher utility. in silmarillion, user devices record bluetooth encounters with beacons installed in strategic locations. the beacons further enrich the encounters with geo-location, location type, and environment conditions at the beacon installation site. this enriched information enables detailed scientific analysis of disease parameters as well as more accurate personalized exposure risk notification. at the same time, silmarillion provides privacy to all participants and non-participants at the same level as that guaranteed in digital and manual contact tracing. we describe the design of silmarillion and its communication protocols that ensure user privacy and data security. we also evaluate a prototype of silmarillion built using low-end iot boards, showing that the power consumption and user latencies are adequately low for a practical deployment. finally, we briefly report on a small-scale deployment within a university building as a proof-of-concept.\",\n",
       "  'categories': 'cs.cr cs.cy cs.si q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2020-11-16',\n",
       "  'updated': '2024-05-09',\n",
       "  'authors': ['pierfrancesco ingo',\n",
       "   'nichole boufford',\n",
       "   'ming cheng jiang',\n",
       "   'rowan lindsay',\n",
       "   'matthew lentz',\n",
       "   'gilles barthe',\n",
       "   'manuel gomez-rodriguez',\n",
       "   'bernhard schölkopf',\n",
       "   'deepak garg',\n",
       "   'peter druschel',\n",
       "   'aastha mehta'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2011.08069'},\n",
       " {'title': 'maximum likelihood estimates of diffusion coefficients from   single-particle tracking experiments',\n",
       "  'id': '2011.09955',\n",
       "  'abstract': 'single-molecule localization microscopy allows practitioners to locate and track labeled molecules in biological systems. when extracting diffusion coefficients from the resulting trajectories, it is common practice to perform a linear fit on mean-square-displacement curves. however, this strategy is suboptimal and prone to errors. recently, it was shown that the increments between observed positions provide a good estimate for the diffusion coefficient, and their statistics are well-suited for likelihood-based analysis methods. here, we revisit the problem of extracting diffusion coefficients from single-particle tracking experiments subject to static and dynamic noise using the principle of maximum likelihood. taking advantage of an efficient real-space formulation, we extend the model to mixtures of subpopulations differing in their diffusion coefficients, which we estimate with the help of the expectation-maximization algorithm. this formulation naturally leads to a probabilistic assignment of trajectories to subpopulations. we employ the theory to analyze experimental tracking data that cannot be explained with a single diffusion coefficient. we test how well a dataset conforms to the assumptions of a diffusion model and determine the optimal number of subpopulations with the help of a quality factor of known analytical distribution. to facilitate use by practitioners, we provide a fast open-source implementation of the theory for the efficient analysis of multiple trajectories in arbitrary dimensions simultaneously.',\n",
       "  'categories': 'physics.bio-ph cond-mat.stat-mech q-bio.qm',\n",
       "  'doi': '10.1063/5.0038174',\n",
       "  'created': '2020-11-19',\n",
       "  'updated': '2021-05-21',\n",
       "  'authors': ['jakob tómas bullerjahn', 'gerhard hummer'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2011.09955'},\n",
       " {'title': 'in silico study on the contribution of the follicular route to dermal   permeability of small molecules',\n",
       "  'id': '2012.02728',\n",
       "  'abstract': 'purpose. this study investigates in silico the contribution of the hair follicle to the overall dermal permeability of small molecules, as published experimental work provides inconclusive information on whether the follicular route favours the permeation of hydrophobic or hydrophilic permeants. method. a study is conducted varying physico-chemical parameters of permeants such as lipophilicity, molecular weight and protein binding. the simulated data is compared to published experimental data to discuss how those properties can modulate the contribution of the hair follicle to the overall dermal permeation. results. the results indicate that the contribution of the follicular route to dermal permeation can range from negligible to notable depending on the combination of lipophilic/hydrophilic properties of the substance filling the follicular route and the permeant. conclusion. characterisation of the substance filling the follicular route is required for analysing the experimental data of dermal permeation of small molecules, as changes between in vivo and in vitro due to handling of samples and cessation of vital functions can modify the contribution of the follicular route to overall dermal permeation, hence hindering data interpretation.',\n",
       "  'categories': 'q-bio.to',\n",
       "  'doi': '',\n",
       "  'created': '2020-12-04',\n",
       "  'updated': '2024-02-04',\n",
       "  'authors': ['daniel sebastia-saez', 'guoping lian', 'tao chen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2012.02728'},\n",
       " {'title': 'a number sense as an emergent property of the manipulating brain',\n",
       "  'id': '2012.04132',\n",
       "  'abstract': \"the ability to understand and manipulate numbers and quantities emerges during childhood, but the mechanism through which humans acquire and develop this ability is still poorly understood. we explore this question through a model, assuming that the learner is able to pick up and place small objects from, and to, locations of its choosing, and will spontaneously engage in such undirected manipulation. we further assume that the learner's visual system will monitor the changing arrangements of objects in the scene and will learn to predict the effects of each action by comparing perception with a supervisory signal from the motor system. we model perception using standard deep networks for feature extraction and classification, and gradient descent learning. our main finding is that, from learning the task of action prediction, an unexpected image representation emerges exhibiting regularities that foreshadow the perception and representation of numbers and quantity. these include distinct categories for zero and the first few natural numbers, a strict ordering of the numbers, and a one-dimensional signal that correlates with numerical quantity. as a result, our model acquires the ability to estimate numerosity, i.e. the number of objects in the scene, as well as subitization, i.e. the ability to recognize at a glance the exact number of objects in small scenes. remarkably, subitization and numerosity estimation extrapolate to scenes containing many objects, far beyond the three objects used during training. we conclude that important aspects of a facility with numbers and quantities may be learned with supervision from a simple pre-training task. our observations suggest that cross-modal learning is a powerful learning mechanism that may be harnessed in artificial intelligence.\",\n",
       "  'categories': 'q-bio.nc cs.ai cs.cv cs.ro',\n",
       "  'doi': '10.1038/s41598-024-56828-2',\n",
       "  'created': '2020-12-07',\n",
       "  'updated': '2024-03-23',\n",
       "  'authors': ['neehar kondapaneni', 'pietro perona'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2012.04132'},\n",
       " {'title': 'sparse encoding for more-interpretable feature-selecting representations   in probabilistic matrix factorization',\n",
       "  'id': '2012.04171',\n",
       "  'abstract': 'dimensionality reduction methods for count data are critical to a wide range of applications in medical informatics and other fields where model interpretability is paramount. for such data, hierarchical poisson matrix factorization (hpf) and other sparse probabilistic non-negative matrix factorization (nmf) methods are considered to be interpretable generative models. they consist of sparse transformations for decoding their learned representations into predictions. however, sparsity in representation decoding does not necessarily imply sparsity in the encoding of representations from the original data features. hpf is often incorrectly interpreted in the literature as if it possesses encoder sparsity. the distinction between decoder sparsity and encoder sparsity is subtle but important. due to the lack of encoder sparsity, hpf does not possess the column-clustering property of classical nmf -- the factor loading matrix does not sufficiently define how each factor is formed from the original features. we address this deficiency by self-consistently enforcing encoder sparsity, using a generalized additive model (gam), thereby allowing one to relate each representation coordinate to a subset of the original data features. in doing so, the method also gains the ability to perform feature selection. we demonstrate our method on simulated data and give an example of how encoder sparsity is of practical use in a concrete application of representing inpatient comorbidities in medicare patients.',\n",
       "  'categories': 'cs.lg q-bio.qm stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2020-12-07',\n",
       "  'updated': '2020-12-29',\n",
       "  'authors': ['joshua c. chang',\n",
       "   'patrick fletcher',\n",
       "   'jungmin han',\n",
       "   'ted l. chang',\n",
       "   'shashaank vattikuti',\n",
       "   'bart desmet',\n",
       "   'ayah zirikly',\n",
       "   'carson c. chow'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2012.04171'},\n",
       " {'title': 'estimating the contribution of early and late noise in vision from   psychophysical data',\n",
       "  'id': '2012.06608',\n",
       "  'abstract': 'in many psychophysical detection and discrimination tasks human performance is thought to be limited by internal or inner noise when neuronal activity is converted into an overt behavioural response. it is unclear, however, to what extent the behaviourally limiting inner noise arises from early noise in the photoreceptors and the retina, or from late noise in cortex at or immediately prior to the decision stage. presumably, the behaviourally limiting inner noise is a non-trivial combination of both early and late noises. here we propose a method to quantify the contributions of early and late noise purely from psychophysical data. our analysis generalizes classical results for linear systems (burgess and colborne, 1988) by combining the theory of noise propagation through a nonlinear network (ahumada, 1987) with the expressions to obtain the perceptual metric along the nonlinear network (malo and simoncelli, 2006; laparra et al., 2010). we show that from threshold-only data the relative contribution of early and late noise can only be determined if the experiments include substantial external noise in some of the stimuli used during experiments. if experimenters collected full psychometric functions, however, then early and late noise sources can be quantified even in the absence of external noise. our psychophysical estimate of the magnitude of the early noise assuming a standard cascade of linear and nonlinear model stages is substantially lower than the noise in cone photocurrents computed via an accurate model of retinal physiology (brainard and wandell, 2020, isetbio). this is consistent with the idea that one of the fundamental tasks of early vision is to reduce the comparatively large retinal noise.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2020-12-11',\n",
       "  'updated': '2024-12-01',\n",
       "  'authors': ['jesus malo',\n",
       "   'jose juan esteve-taboada',\n",
       "   'guillermo aguilar',\n",
       "   'marianne maertens',\n",
       "   'felix a. wichmann'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2012.06608'},\n",
       " {'title': 'a comparative analysis of the ensemble methods for drug design',\n",
       "  'id': '2012.07640',\n",
       "  'abstract': 'quantitative structure-activity relationship (qsar) is a computer modeling technique for identifying relationships between the structural properties of chemical compounds and biological activity. qsar modeling is necessary for drug discovery, but it has many limitations. ensemble-based machine learning approaches have been used to overcome limitations and generate reliable predictions. ensemble learning creates a set of diverse models and combines them. in our comparative analysis, each ensemble algorithm was paired with each of the basic algorithms, but the basic algorithms were also investigated separately. in this configuration, 57 algorithms were developed and compared on 4 different datasets. thus, a technique for complex ensemble method is proposed that builds diversified models and integrates them. the proposed individual models did not show impressive results as a unified model, but it was considered the most important predictor when combined. we assessed whether ensembles always give better results than individual algorithms. the python code written to get experimental results in this article has been uploaded to github (https://github.com/rifqat/comparative-analysis).',\n",
       "  'categories': 'cs.lg cs.it math.it q-bio.qm',\n",
       "  'doi': '10.1063/5.0057487',\n",
       "  'created': '2020-12-11',\n",
       "  'updated': '',\n",
       "  'authors': ['rifkat davronova', 'fatima adilovab'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2012.07640'},\n",
       " {'title': 'system identification of biophysical neuronal models',\n",
       "  'id': '2012.07691',\n",
       "  'abstract': 'after sixty years of quantitative biophysical modeling of neurons, the identification of neuronal dynamics from input-output data remains a challenging problem, primarily due to the inherently nonlinear nature of excitable behaviors. by reformulating the problem in terms of the identification of an operator with fading memory, we explore a simple approach based on a parametrization given by a series interconnection of generalized orthonormal basis functions (gobfs) and static artificial neural networks. we show that gobfs are particularly well-suited to tackle the identification problem, and provide a heuristic for selecting gobf poles which addresses the ultra-sensitivity of neuronal behaviors. the method is illustrated on the identification of a bursting model from the crab stomatogastric ganglion.',\n",
       "  'categories': 'q-bio.nc cs.lg cs.sy eess.sy',\n",
       "  'doi': '10.1109/cdc42340.2020.9304363',\n",
       "  'created': '2020-12-14',\n",
       "  'updated': '',\n",
       "  'authors': ['thiago b. burghi', 'maarten schoukens', 'rodolphe sepulchre'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2012.07691'},\n",
       " {'title': 'computational interference microscopy enabled by deep learning',\n",
       "  'id': '2012.10239',\n",
       "  'abstract': 'quantitative phase imaging (qpi) has been widely applied in characterizing cells and tissues. spatial light interference microscopy (slim) is a highly sensitive qpi method, due to its partially coherent illumination and common path interferometry geometry. however, its acquisition rate is limited because of the four-frame phase-shifting scheme. on the other hand, off-axis methods like diffraction phase microscopy (dpm), allows for single-shot qpi. however, the laser-based dpm system is plagued by spatial noise due to speckles and multiple reflections. in a parallel development, deep learning was proven valuable in the field of bioimaging, especially due to its ability to translate one form of contrast into another. here, we propose using deep learning to produce synthetic, slim-quality, high-sensitivity phase maps from dpm, single-shot images as input. we used an inverted microscope with its two ports connected to the dpm and slim modules, such that we have access to the two types of images on the same field of view. we constructed a deep learning model based on u-net and trained on over 1,000 pairs of dpm and slim images. the model learned to remove the speckles in laser dpm and overcame the background phase noise in both the test set and new data. furthermore, we implemented the neural network inference into the live acquisition software, which now allows a dpm user to observe in real-time an extremely low-noise phase image. we demonstrated this principle of computational interference microscopy (cim) imaging using blood smears, as they contain both erythrocytes and leukocytes, in static and dynamic conditions.',\n",
       "  'categories': 'eess.iv physics.optics q-bio.qm',\n",
       "  'doi': '10.1063/5.0041901',\n",
       "  'created': '2020-12-17',\n",
       "  'updated': '',\n",
       "  'authors': ['yuheng jiao',\n",
       "   'yuchen r. he',\n",
       "   'mikhail e. kandel',\n",
       "   'xiaojun liu',\n",
       "   'wenlong lu',\n",
       "   'gabriel popescu'],\n",
       "  'affiliation': ['quantitative light imaging laboratory, department of electrical and computer engineering, beckman institute for advanced science and technology, university of illinois at urbana-champaign',\n",
       "   'quantitative light imaging laboratory, department of electrical and computer engineering, beckman institute for advanced science and technology, university of illinois at urbana-champaign',\n",
       "   'quantitative light imaging laboratory, department of electrical and computer engineering, beckman institute for advanced science and technology, university of illinois at urbana-champaign',\n",
       "   'state key laboratory of digital manufacturing equipment and technology, school of mechanical and engineering, huazhong university of science and technology',\n",
       "   'state key laboratory of digital manufacturing equipment and technology, school of mechanical and engineering, huazhong university of science and technology',\n",
       "   'quantitative light imaging laboratory, department of electrical and computer engineering, beckman institute for advanced science and technology, university of illinois at urbana-champaign'],\n",
       "  'url': 'https://arxiv.org/abs/2012.10239'},\n",
       " {'title': 'unconsciousness reconfigures modular brain network dynamics',\n",
       "  'id': '2012.10785',\n",
       "  'abstract': 'the dynamic core hypothesis posits that consciousness is correlated with simultaneously integrated and differentiated assemblies of transiently synchronized brain regions. we represented time-dependent functional interactions using dynamic brain networks, and assessed the integrityof the dynamic core by means of the flexibility and largest multilayer module of these networks. as a first step, we constrained parameter selection using a newly developed benchmark for module detection in heterogeneous temporal networks. next, we applied a multilayer modularity maximization algorithm to dynamic brain networks computed from functional magnetic resonance imaging (fmri) data acquired during deep sleep and under propofol anesthesia. we found that unconsciousness reconfigured network flexibility and reduced the size of the largest spatiotemporal module, which we identified with the dynamic core. our results present a first characterization of modular brain network dynamics during states of unconsciousness measured with fmri, adding support to the dynamic core hypothesis of human consciousness.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1063/5.0046047',\n",
       "  'created': '2020-12-19',\n",
       "  'updated': '',\n",
       "  'authors': ['sofia morena del pozo',\n",
       "   'helmut laufs',\n",
       "   'vincent bonhomme',\n",
       "   'steven laureys',\n",
       "   'pablo balenzuela',\n",
       "   'enzo tagliazucchi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2012.10785'},\n",
       " {'title': 'improving j-divergence of brain connectivity states by graph laplacian   denoising',\n",
       "  'id': '2012.11240',\n",
       "  'abstract': 'functional connectivity (fc) can be represented as a network, and is frequently used to better understand the neural underpinnings of complex tasks such as motor imagery (mi) detection in brain-computer interfaces (bcis). however, errors in the estimation of connectivity can affect the detection performances. in this work, we address the problem of denoising common connectivity estimates to improve the detectability of different connectivity states. specifically, we propose a denoising algorithm that acts on the network graph laplacian, which leverages recent graph signal processing results. further, we derive a novel formulation of the jensen divergence for the denoised laplacian under different states. numerical simulations on synthetic data show that the denoising method improves the jensen divergence of connectivity patterns corresponding to different task conditions. furthermore, we apply the laplacian denoising technique to brain networks estimated from real eeg data recorded during mi-bci experiments. using our novel formulation of the j-divergence, we are able to quantify the distance between the fc networks in the motor imagery and resting states, as well as to understand the contribution of each laplacian variable to the total j-divergence between two states. experimental results on real mi-bci eeg data demonstrate that the laplacian denoising improves the separation of motor imagery and resting mental states, and shortens the time interval required for connectivity estimation. we conclude that the approach shows promise for the robust detection of connectivity states while being appealing for implementation in real-time bci applications.',\n",
       "  'categories': 'q-bio.nc eess.sp',\n",
       "  'doi': '10.1109/tsipn.2021.3100302',\n",
       "  'created': '2020-12-21',\n",
       "  'updated': '2020-12-22',\n",
       "  'authors': ['tiziana cattai',\n",
       "   'gaetano scarano',\n",
       "   'marie-constance corsi',\n",
       "   'danielle s. bassett',\n",
       "   'fabrizio de vico fallani',\n",
       "   'stefania colonnese'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2012.11240'},\n",
       " {'title': 'optimal adaptive testing for epidemic control: combining molecular and   serology tests',\n",
       "  'id': '2101.00773',\n",
       "  'abstract': 'the covid-19 crisis highlighted the importance of non-medical interventions, such as testing and isolation of infected individuals, in the control of epidemics. here, we show how to minimize testing needs while maintaining the number of infected individuals below a desired threshold. we find that the optimal policy is adaptive, with testing rates that depend on the epidemic state. additionally, we show that such epidemic state is difficult to infer with molecular tests alone, which are highly sensitive but have a short detectability window. instead, we propose the use of baseline serology testing, which is less sensitive but detects past infections, for the purpose of state estimation. validation of such combined testing approach with a stochastic model of epidemics shows significant cost savings compared to non-adaptive testing strategies that are the current standard for covid-19.',\n",
       "  'categories': 'eess.sy cs.sy math.ds math.oc physics.soc-ph q-bio.pe',\n",
       "  'doi': '10.1016/j.automatica.2023.111391',\n",
       "  'created': '2021-01-03',\n",
       "  'updated': '',\n",
       "  'authors': ['d. acemoglu',\n",
       "   'a. fallah',\n",
       "   'a. giometto',\n",
       "   'd. huttenlocher',\n",
       "   'a. ozdaglar',\n",
       "   'f. parise',\n",
       "   's. pattathil'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2101.00773'},\n",
       " {'title': 'some paradoxes in the tilman model, how to avoid or accept them',\n",
       "  'id': '2101.07082',\n",
       "  'abstract': 'in making models for biological systems one expects to grasp the biology and be able to use ones intuition to predict the outcome. this paper is about the discrepancy between what is expected and what is the outcome of the analysis of the model, a paradox. the tilman model of consumers competing for resources has some aspects that may appear counterintuitive. within the standard tilman model for a single consumer and a single resource, when a very efficient consumer rapidly eats all available food, the resource density becomes zero, but if there is no food, how can the consumer survive? the paradox can be resolved by realising that in the short term indeed rapid consumption may lead to starvation and a decline in the consumer population, but in the long term a small resource and consumer density remain. a single consumer living on two essential nutrients leaves the density of the non-limiting nutrient above its critical level, so what is done with the extra food? we explain that the extra food is not consumed, just because it is available, but that in fact the additional food is not consumed at all, because the consumer has no use for it. for a model with two consumer species competing for a single nutrient one of the consumers will eventually disappear, even if the food economics of the surviving one is worse and there is still much food wasted. both are inherent aspects of the model, and the paradox can be avoided if the difference between the consumer species is small, in which case it will take very long to reach equilibrium. we argue that in general an extended stability analysis, in which not only the asymptotically stable state is considered, but also the unstable steady states and all time scales involved in the transient dynamics, gives more insight in the dynamics involved and thus can help in avoiding apparent contradictions in ecological models, or accepting them.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2021-01-18',\n",
       "  'updated': '2024-04-04',\n",
       "  'authors': ['joost h. j. van opheusden', 'lia hemerik'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2101.07082'},\n",
       " {'title': 'algebraic model selection and experimental design in biological data   science',\n",
       "  'id': '2101.09384',\n",
       "  'abstract': 'design of experiments and model selection, though essential steps in data science, are usually viewed as unrelated processes in the study and analysis of biological networks. not accounting for their inter-relatedness has the potential to introduce bias and increase the risk of missing salient features in the modeling process. we propose a data-driven computational framework to unify experimental design and model selection for discrete data sets and minimal polynomial models. we use a special affine transformation, called a linear shift, to provide both the data sets and the polynomial terms that form a basis for a model. this framework enables us to address two important questions that arise in biological data science research: finding the data which identify a set of known interactions and finding identifiable interactions given a set of data. we present the theoretical foundation for a web-accessible database. as an example, we apply this methodology to a previously constructed pharmacodynamic model of epidermal derived growth factor receptor (egfr) signaling.',\n",
       "  'categories': 'math.ag q-bio.qm',\n",
       "  'doi': '10.1016/j.aam.2021.102282',\n",
       "  'created': '2021-01-22',\n",
       "  'updated': '',\n",
       "  'authors': ['anyu zhang',\n",
       "   'jingzhen hu',\n",
       "   'qingzhong liang',\n",
       "   'elena s. dimitrova',\n",
       "   'brandilyn stigler'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2101.09384'},\n",
       " {'title': \"time to revisit the endpoint dilution assay and to replace tcid$_{50}$   and pfu as measures of a virus sample's infection concentration\",\n",
       "  'id': '2101.11526',\n",
       "  'abstract': \"the infectivity of a virus sample is measured by the infections it causes, via a plaque or focus forming assay (pfu or ffu) or an endpoint dilution (ed) assay (tcid$_{50}$, ccid$_{50}$, eid$_{50}$, etc., hereafter collectively id$_{50}$). the counting of plaques or foci at a given dilution intuitively and directly provides the concentration of infectious doses in the undiluted sample. however, it has many technical and experimental limitations. for example, it relies on one's judgement in distinguishing between two merged plaques and a larger one, or between small plaques and staining artifacts. in this regard, ed assays are more robust because one need only determine whether infection occurred. the output of the ed assay, the 50% infectious dose (id$_{50}$), is calculated using either the spearman-karber (sk, 1908,1931) or reed-muench (rm, 1938) mathematical approximations. however, these are often miscalculated and their id$_{50}$ approximation is biased. we propose that the pfu and ffu assays be abandoned, and that the measured output of the ed assay, the id$_{50}$, be replaced by a more useful measure we coined specific infections (sin). we introduce a free, open-source web-application, midsin, that computes the sin concentration in a virus sample from a standard ed assay, requiring no changes to current experimental protocols. we demonstrate that the sin/ml of a sample reliably corresponds to the number of infections the sample will cause per unit volume, and directly relates to the multiplicity of infection. midsin estimates are shown to be more accurate and robust than those using the rm and sk approximations. the impact of ed plate design choices (dilution factor, replicates per dilution) on measurement accuracy is also explored. the simplicity of sin as a measure and the greater accuracy of midsin make them an easy, superior replacement for the pfu, ffu, and id$_{50}$ measures.\",\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.1371/journal.pcbi.1009480',\n",
       "  'created': '2021-01-27',\n",
       "  'updated': '',\n",
       "  'authors': ['daniel cresta',\n",
       "   'donald c. warren',\n",
       "   'christian quirouette',\n",
       "   'amanda p. smith',\n",
       "   'lindey c. lane',\n",
       "   'amber m. smith',\n",
       "   'catherine a. a. beauchemin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2101.11526'},\n",
       " {'title': 'a step toward a reinforcement learning de novo genome assembler',\n",
       "  'id': '2102.02649',\n",
       "  'abstract': \"de novo genome assembly is a relevant but computationally complex task in genomics. although de novo assemblers have been used successfully in several genomics projects, there is still no 'best assembler', and the choice and setup of assemblers still rely on bioinformatics experts. thus, as with other computationally complex problems, machine learning may emerge as an alternative (or complementary) way for developing more accurate and automated assemblers. reinforcement learning has proven promising for solving complex activities without supervision - such games - and there is a pressing need to understand the limits of this approach to 'real' problems, such as the dfa problem. this study aimed to shed light on the application of machine learning, using reinforcement learning (rl), in genome assembly. we expanded upon the sole previous approach found in the literature to solve this problem by carefully exploring the learning aspects of the proposed intelligent agent, which uses the q-learning algorithm, and we provided insights for the next steps of automated genome assembly development. we improved the reward system and optimized the exploration of the state space based on pruning and in collaboration with evolutionary computing. we tested the new approaches on 23 new larger environments, which are all available on the internet. our results suggest consistent performance progress; however, we also found limitations, especially concerning the high dimensionality of state and action spaces. finally, we discuss paths for achieving efficient and automated genome assembly in real scenarios considering successful rl applications - including deep reinforcement learning.\",\n",
       "  'categories': 'q-bio.gn cs.ai cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2021-02-02',\n",
       "  'updated': '2024-03-07',\n",
       "  'authors': ['kleber padovani',\n",
       "   'roberto xavier',\n",
       "   'rafael cabral borges',\n",
       "   'andre carvalho',\n",
       "   'anna reali',\n",
       "   'annie chateau',\n",
       "   'ronnie alves'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2102.02649'},\n",
       " {'title': 'collective dynamics in the presence of finite-width pulses',\n",
       "  'id': '2102.03438',\n",
       "  'abstract': 'the idealisation of neuronal pulses as $\\\\delta$-spikes is a convenient approach in neuroscience but can sometimes lead to erroneous conclusions. we investigate the effect of a finite pulse-width on the dynamics of balanced neuronal networks. in particular, we study two populations of identical excitatory and inhibitory neurons in a random network of phase oscillators coupled through exponential pulses with different widths. we consider three coupling functions, inspired by leaky integrate-and-fire neurons with delay and type-i phase-response curves. by exploring the role of the pulse-widths for different coupling strengths we find a robust collective irregular dynamics, which collapses onto a fully synchronous regime if the inhibitory pulses are sufficiently wider than the excitatory ones. the transition to synchrony is accompanied by hysteretic phenomena (i.e. the co-existence of collective irregular and synchronous dynamics). our numerical results are supported by a detailed scaling and stability analysis of the fully synchronous solution. a conjectured first-order phase transition emerging for $\\\\delta$-spikes is smoothed out for finite-width pulses.',\n",
       "  'categories': 'q-bio.nc math.ds nlin.ao',\n",
       "  'doi': '10.1063/5.0046691',\n",
       "  'created': '2021-02-05',\n",
       "  'updated': '2021-04-13',\n",
       "  'authors': ['n/a afifurrahman', 'ekkehard ullner', 'antonio politi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2102.03438'},\n",
       " {'title': 'graphery: interactive tutorials for biological network algorithms',\n",
       "  'id': '2102.03469',\n",
       "  'abstract': 'networks provide a meaningful way to represent and analyze complex biological information, but the methodological details of network-based tools are often described for a technical audience. graphery is a hands-on tutorial webserver designed to help biological researchers understand the fundamental concepts behind commonly-used graph algorithms. each tutorial describes a graph concept along with executable python code that visualizes the concept in a code view and a graph view. graphery tutorials help researchers understand graph statistics (such as degree distribution and network modularity) and classic graph algorithms (such as shortest paths and random walks). users navigate each tutorial using their choice of real-world biological networks, ranging in scale from molecular interaction graphs to ecological networks. graphery also allows users to modify the code within each tutorial or write new programs, which all can be executed without requiring an account. discipline-focused tutorials will be essential to help researchers interpret their biological data. graphery accepts ideas for new tutorials and datasets that will be shaped by both computational and biological researchers, growing into a community-contributed learning platform. availability: graphery is available at https://graphery.reedcompbio.org/.',\n",
       "  'categories': 'q-bio.mn',\n",
       "  'doi': '10.1093/nar/gkab420',\n",
       "  'created': '2021-02-05',\n",
       "  'updated': '2024-02-15',\n",
       "  'authors': ['heyuan zeng',\n",
       "   'jinbiao zhang',\n",
       "   'gabriel a. preising',\n",
       "   'tobias rubel',\n",
       "   'pramesh singh',\n",
       "   'anna ritz'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2102.03469'},\n",
       " {'title': 'an overview of continuous and discrete phasor analysis of binned or   time-gated periodic decays',\n",
       "  'id': '2102.03676',\n",
       "  'abstract': 'time-resolved analysis of periodically excited luminescence decays by the phasor method in the presence of time-gating or binning is revisited. analytical expressions for discrete configurations of square gates are derived and the locus of the phasors of such modified periodic single-exponential decays is compared to the canonical universal semicircle. the effects of irf offset, decay truncation and gate shape are also discussed. finally, modified expressions for the phase and modulus lifetimes are provided for some simple cases. a discussion of a modified phasor calibration approach is presented.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.1063/5.0027834',\n",
       "  'created': '2021-02-06',\n",
       "  'updated': '2021-02-11',\n",
       "  'authors': ['xavier michalet'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2102.03676'},\n",
       " {'title': 'a survey of the monotonicity and non-contradiction of consensus methods   and supertree methods',\n",
       "  'id': '2102.04502',\n",
       "  'abstract': 'in a recent study, bryant, francis and steel investigated the concept of \\\\enquote{future-proofing} consensus methods in phylogenetics. that is, they investigated if such methods can be robust against the introduction of additional data like added trees or new species. in the present manuscript, we analyze consensus methods under a different aspect of introducing new data, namely concerning the discovery of new clades. in evolutionary biology, often formerly unresolved clades get resolved by refined reconstruction methods or new genetic data analyses. in our manuscript we investigate which properties of consensus methods can guarantee that such new insights do not disagree with previously found consensus trees, but merely refine them, a property termed \\\\emph{monotonicity}. along the lines of analyzing monotonicity, we also study two {established} supertree methods, namely matrix representation with parsimony (mrp) and matrix representation with compatibility (mrc), which have also been suggested as consensus methods in the literature. while we (just like bryant, francis and steel in their recent study) unfortunately have to conclude some negative answers concerning general consensus methods, we also state some relevant and positive results concerning the majority rule ($\\\\mathtt{mr}$) and strict consensus methods, which are amongst the most frequently used consensus methods. moreover, we show that there exist infinitely many consensus methods which are monotonic and have some other desirable properties.   \\\\textbf{keywords:} consensus tree, phylogenetics, majority rule, tree refinement, matrix representation with parsimony   \\\\textbf{msc:} c92b05, 05c05',\n",
       "  'categories': 'q-bio.pe math.co',\n",
       "  'doi': '',\n",
       "  'created': '2021-02-08',\n",
       "  'updated': '2024-10-11',\n",
       "  'authors': ['mareike fischer', 'michael hendriksen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2102.04502'},\n",
       " {'title': 'a general model of structured cell kinetics',\n",
       "  'id': '2102.06490',\n",
       "  'abstract': 'we present a modelling framework for the dynamics of cells structured by the concentration of a micromolecule they contain. we derive general equations for the evolution of the cell population and of the extra-cellular concentration of the molecule and apply this approach to models of silicosis, quorum sensing in gram-negative bacteria and magnetic ion exchange.',\n",
       "  'categories': 'q-bio.cb nlin.ps',\n",
       "  'doi': '',\n",
       "  'created': '2021-02-12',\n",
       "  'updated': '2024-10-29',\n",
       "  'authors': ['michael grinfeld', 'nigel mottram', 'jozsef farkas'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2102.06490'},\n",
       " {'title': \"a dynamic network model of societal complexity and resilience inspired   by tainter's theory of collapse\",\n",
       "  'id': '2102.06698',\n",
       "  'abstract': 'in recent years, several global events have severely disrupted economies and social structures, undermining confidence in the resilience of modern societies. while empirical evidence on the dynamics and drivers of past societal collapse is mounting, a process-based understanding of these dynamics is still in its infancy. here we aim to identify and illustrate the underlying drivers of such societal instability or even collapse. the inspiration for this work is joseph tainter\\'s theory of the \"collapse of complex societies\", which postulates that the complexity of societies increases as they solve problems, leading to diminishing returns on complexity investments, and ultimately to collapse. in this work we have abstracted this theory into a low-dimensional and stylised model of two classes of networked agents, hereafter referred to as \"laborers\" and \"administrators\". we numerically modeled the dynamics of societal complexity, measured as the fraction of \"administrators\", which is assumed to affect the productivity of connected energy-producing \"laborers\". we show that collapse becomes increasingly likely as the complexity of the model society continuously increases in response to external stresses that emulate tainter\\'s abstract notion of problems that societies must solve. we also provide an analytical approximation of the system\\'s dominant dynamics, which matches well with the numerical experiments, and use it to study the influence on network link density, social mobility and productivity. our work advances the understanding of social-ecological collapse and illustrates its potentially direct link to an ever-increasing societal complexity in response to external shocks or stresses via a self-reinforcing feedback.',\n",
       "  'categories': 'physics.soc-ph q-bio.pe',\n",
       "  'doi': '10.3390/e26020098',\n",
       "  'created': '2021-02-12',\n",
       "  'updated': '2023-12-21',\n",
       "  'authors': ['florian schunck',\n",
       "   'marc wiedermann',\n",
       "   'jobst heitzig',\n",
       "   'jonathan f. donges'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2102.06698'},\n",
       " {'title': 'polygenic risk score in africa population: progress and challenges',\n",
       "  'id': '2102.08468',\n",
       "  'abstract': \"polygenic risk score (prs) analysis is a powerful method been used to estimate an individual's genetic risk towards targeted traits. prs analysis could be used to obtain evidence of a genetic effect beyond genome-wide association studies (gwas) results i.e. when there are no significant markers. prs analysis has been widely applied to investigate the genetic basis of several traits including rare diseases. however, the accuracy of prs analysis depends on the genomic data of the underlying population. for instance, several studies showed that obtaining higher prediction power of prs analysis is challenging for non-europeans. in this manuscript, we reviewed the conventional prs methods and their application to sub-saharan africa communities. we concluded that the limiting factor of applying prs analysis to sub-saharan populations is the lack of sufficient gwas data. also, we recommended developing african-specific prs tools\",\n",
       "  'categories': 'q-bio.gn',\n",
       "  'doi': '10.12688/f1000research.76218.2',\n",
       "  'created': '2021-02-16',\n",
       "  'updated': '',\n",
       "  'authors': ['yagoub adam',\n",
       "   'suraju sadeeq',\n",
       "   'judit kumuthini',\n",
       "   'olabode ajayi',\n",
       "   'gordon wells',\n",
       "   'rotimi solomon',\n",
       "   'olubanke ogunlana',\n",
       "   'emmmanuel adetiba',\n",
       "   'emeka iweala',\n",
       "   'benedikt brors',\n",
       "   'ezekiel adebiyi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2102.08468'},\n",
       " {'title': 'learning to extend molecular scaffolds with structural motifs',\n",
       "  'id': '2103.03864',\n",
       "  'abstract': 'recent advancements in deep learning-based modeling of molecules promise to accelerate in silico drug discovery. a plethora of generative models is available, building molecules either atom-by-atom and bond-by-bond or fragment-by-fragment. however, many drug discovery projects require a fixed scaffold to be present in the generated molecule, and incorporating that constraint has only recently been explored. here, we propose moler, a graph-based model that naturally supports scaffolds as initial seed of the generative procedure, which is possible because it is not conditioned on the generation history. our experiments show that moler performs comparably to state-of-the-art methods on unconstrained molecular optimization tasks, and outperforms them on scaffold-based tasks, while being an order of magnitude faster to train and sample from than existing approaches. furthermore, we show the influence of a number of seemingly minor design choices on the overall performance.',\n",
       "  'categories': 'cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2021-03-05',\n",
       "  'updated': '2024-05-12',\n",
       "  'authors': ['krzysztof maziarz',\n",
       "   'henry jackson-flux',\n",
       "   'pashmina cameron',\n",
       "   'finton sirockin',\n",
       "   'nadine schneider',\n",
       "   'nikolaus stiefl',\n",
       "   'marwin segler',\n",
       "   'marc brockschmidt'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2103.03864'},\n",
       " {'title': 'continual developmental neurosimulation using embodied computational   agents',\n",
       "  'id': '2103.05753',\n",
       "  'abstract': 'there is much to learn through synthesis of developmental biology, cognitive science and computational modeling. our path forward involves a design for developmentally-inspired learning agents based on braitenberg vehicles. continual developmental neurosimulation allows us to consider the role of developmental trajectories in bridging the related phenomena of nervous system morphogenesis, developmental learning, and plasticity. being closely tied to continual learning, our approach is tightly integrated with developmental embodiment, and can be implemented using a type of agent called developmental braitenberg vehicles (dbvs). dbvs begin their lives as a set of undefined structures that transform into agent-based systems including a body, sensors, effectors, and nervous system. this phenotype is characterized in terms of developmental timing: with distinct morphogenetic, critical, and acquisition (developmental learning) periods. we further propose that network morphogenesis can be accomplished using a genetic algorithmic approach, while developmental learning can be implemented using a number of computational methodologies. this approach provides a framework for adaptive agent behavior that might result from a developmental approach: namely by exploiting critical periods or growth and acquisition, an explicitly embodied network architecture, and a distinction between the assembly of neuronal networks and active learning on these networks. in conclusion, we will consider agent learning and development at different timescales, from very short (<100ms) intervals to long-term evolution. the development, evolution, and learning in an embodied agent-based approach is key to an integrative view of biologically-inspired intelligence.',\n",
       "  'categories': 'q-bio.nc cs.ai cs.ne',\n",
       "  'doi': '',\n",
       "  'created': '2021-03-07',\n",
       "  'updated': '2024-07-12',\n",
       "  'authors': ['bradly alicea',\n",
       "   'rishabh chakrabarty',\n",
       "   'stefan dvoretskii',\n",
       "   'akshara gopi',\n",
       "   'avery lim',\n",
       "   'jesse parent'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2103.05753'},\n",
       " {'title': 'active inference tree search in large pomdps',\n",
       "  'id': '2103.13860',\n",
       "  'abstract': \"the ability to plan ahead efficiently is key for both living organisms and artificial systems. model-based planning and prospection are widely studied in cognitive neuroscience and artificial intelligence (ai), but from different perspectives--and with different desiderata in mind (biological realism versus scalability) that are difficult to reconcile. here, we introduce a novel method to plan in pomdps--active inference tree search (act)--that combines the normative character and biological realism of a leading planning theory in neuroscience (active inference) and the scalability of tree search methods in ai. this unification enhances both approaches. on the one hand, tree searches enable the biologically grounded, first principle method of active inference to be applied to large-scale problems. on the other hand, active inference provides a principled solution to the exploration-exploitation dilemma, which is often addressed heuristically in tree search methods. our simulations show that act successfully navigates binary trees that are challenging for sampling-based methods, problems that require adaptive exploration, and the large pomdp problem 'rocksample'--in which act reproduces state-of-the-art pomdp solutions. furthermore, we illustrate how act can be used to simulate neurophysiological responses (e.g., in the hippocampus and prefrontal cortex) of humans and other animals that solve large planning problems. these numerical analyses show that active tree search is a principled realisation of neuroscientific and ai planning theories, which offer both biological realism and scalability.\",\n",
       "  'categories': 'cs.ai math.pr q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2021-03-25',\n",
       "  'updated': '2024-10-08',\n",
       "  'authors': ['domenico maisto',\n",
       "   'francesco gregoretti',\n",
       "   'karl friston',\n",
       "   'giovanni pezzulo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2103.13860'},\n",
       " {'title': 'gatekeeper-gpu: fast and accurate pre-alignment filtering in short read   mapping',\n",
       "  'id': '2103.14978',\n",
       "  'abstract': 'at the last step of short read mapping, the candidate locations of the reads on the reference genome are verified to compute their differences from the corresponding reference segments using sequence alignment algorithms. calculating the similarities and differences between two sequences is still computationally expensive since approximate string matching techniques traditionally inherit dynamic programming algorithms with quadratic time and space complexity. we introduce gatekeeper-gpu, a fast and accurate pre-alignment filter that efficiently reduces the need for expensive sequence alignment. gatekeeper-gpu provides two main contributions: first, improving the filtering accuracy of gatekeeper (a lightweight pre-alignment filter), and second, exploiting the massive parallelism provided by the large number of gpu threads of modern gpus to examine numerous sequence pairs rapidly and concurrently. by reducing the work, gatekeeper-gpu provides an acceleration of 2.9x to sequence alignment and up to 1.4x speedup to the end-to-end execution time of a comprehensive read mapper (mrfast). gatekeeper-gpu is available at https://github.com/bilkentcompgen/gatekeeper-gpu.',\n",
       "  'categories': 'q-bio.gn cs.ar',\n",
       "  'doi': '10.1109/tc.2024.3365931',\n",
       "  'created': '2021-03-27',\n",
       "  'updated': '2024-02-22',\n",
       "  'authors': ['zülal bingöl',\n",
       "   'mohammed alser',\n",
       "   'onur mutlu',\n",
       "   'ozcan ozturk',\n",
       "   'can alkan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2103.14978'},\n",
       " {'title': 'dragon: determining regulatory associations using graphical models on   multi-omic networks',\n",
       "  'id': '2104.01690',\n",
       "  'abstract': 'the increasing quantity of multi-omics data, such as methylomic and transcriptomic profiles, collected on the same specimen, or even on the same cell, provide a unique opportunity to explore the complex interactions that define cell phenotype and govern cellular responses to perturbations. we propose a network approach based on gaussian graphical models (ggms) that facilitates the joint analysis of paired omics data. this method, called dragon (determining regulatory associations using graphical models on multi-omic networks), calibrates its parameters to achieve an optimal trade-off between the network\\'s complexity and estimation accuracy, while explicitly accounting for the characteristics of each of the assessed omics \"layers.\" in simulation studies, we show that dragon adapts to edge density and feature size differences between omics layers, improving model inference and edge recovery compared to state-of-the-art methods. we further demonstrate in an analysis of joint transcriptome - methylome data from tcga breast cancer specimens that dragon can identify key molecular mechanisms such as gene regulation via promoter methylation. in particular, we identify transcription factor ap-2 beta (tfap2b) as a potential multi-omic biomarker for basal-type breast cancer. dragon is available as open-source code in python through the network zoo package (netzoopy v0.8; netzoo.github.io).',\n",
       "  'categories': 'q-bio.mn',\n",
       "  'doi': '10.1093/nar/gkac1157',\n",
       "  'created': '2021-04-04',\n",
       "  'updated': '2022-09-21',\n",
       "  'authors': ['katherine h. shutta',\n",
       "   'deborah weighill',\n",
       "   'rebekka burkholz',\n",
       "   'marouen ben guebila',\n",
       "   'dawn l. demeo',\n",
       "   'helena u. zacharias',\n",
       "   'john quackenbush',\n",
       "   'michael altenbuchinger'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2104.01690'},\n",
       " {'title': 'oscillations and bifurcation structure of reaction-diffusion model for   cell polarity formation',\n",
       "  'id': '2104.04283',\n",
       "  'abstract': 'we investigate the oscillatory dynamics and bifurcation structure of a reaction-diffusion system with bistable nonlinearity and mass conservation, which was proposed by [otsuji et al, plos comp. biol. 3 (2007), e108]. the system is a useful model for understanding cell polarity formation. we show that this model exhibits four different spatiotemporal patterns including two types of oscillatory patterns, which can be regarded as cell polarity oscillations with the reversal and non-reversal of polarity, respectively. the trigger causing these patterns is a diffusion-driven (turing-like) instability. moreover, we investigate the effects of extracellular signals on the cell polarity oscillations.',\n",
       "  'categories': 'q-bio.cb math.ds',\n",
       "  'doi': '10.1007/s00285-022-01723-5',\n",
       "  'created': '2021-04-09',\n",
       "  'updated': '2022-02-22',\n",
       "  'authors': ['masataka kuwamura', 'hirofumi izuhara', 'shin-ichiro ei'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2104.04283'},\n",
       " {'title': 'lggnet: learning from local-global-graph representations for   brain-computer interface',\n",
       "  'id': '2105.02786',\n",
       "  'abstract': 'neuropsychological studies suggest that co-operative activities among different brain functional areas drive high-level cognitive processes. to learn the brain activities within and among different functional areas of the brain, we propose lggnet, a novel neurologically inspired graph neural network, to learn local-global-graph representations of electroencephalography (eeg) for brain-computer interface (bci). the input layer of lggnet comprises a series of temporal convolutions with multi-scale 1d convolutional kernels and kernel-level attentive fusion. it captures temporal dynamics of eeg which then serves as input to the proposed local and global graph-filtering layers. using a defined neurophysiologically meaningful set of local and global graphs, lggnet models the complex relations within and among functional areas of the brain. under the robust nested cross-validation settings, the proposed method is evaluated on three publicly available datasets for four types of cognitive classification tasks, namely, the attention, fatigue, emotion, and preference classification tasks. lggnet is compared with state-of-the-art methods, such as deepconvnet, eegnet, r2g-stnn, tsception, rgnn, amcnn-dgcn, hrnn and graphnet. the results show that lggnet outperforms these methods, and the improvements are statistically significant (p<0.05) in most cases. the results show that bringing neuroscience prior knowledge into neural network design yields an improvement of classification performance. the source code can be found at https://github.com/yi-ding-cs/lgg',\n",
       "  'categories': 'cs.ne cs.lg eess.sp q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2021-05-05',\n",
       "  'updated': '2022-12-05',\n",
       "  'authors': ['yi ding',\n",
       "   'neethu robinson',\n",
       "   'chengxuan tong',\n",
       "   'qiuhao zeng',\n",
       "   'cuntai guan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2105.02786'},\n",
       " {'title': 'dynamics of specialization in neural modules under resource constraints',\n",
       "  'id': '2106.02626',\n",
       "  'abstract': \"it has long been believed that the brain is highly modular both in terms of structure and function, although recent evidence has led some to question the extent of both types of modularity. we used artificial neural networks to test the hypothesis that structural modularity is sufficient to guarantee functional specialization, and find that in general, this doesn't necessarily hold. we then systematically tested which features of the environment and network do lead to the emergence of specialization. we used a simple toy environment, task and network, allowing us precise control, and show that in this setup, several distinct measures of specialization give qualitatively similar results. we further find that in this setup (1) specialization can only emerge in environments where features of that environment are meaningfully separable, (2) specialization preferentially emerges when the network is strongly resource-constrained, and (3) these findings are qualitatively similar across the different variations of network architectures that we tested, but that the quantitative relationships depend on the precise architecture. finally, we show that functional specialization varies dynamically across time, and demonstrate that these dynamics depend on both the timing and bandwidth of information flow in the network. we conclude that a static notion of specialization, based on structural modularity, is likely too simple a framework for understanding intelligence in situations of real-world complexity, from biology to brain-inspired neuromorphic systems. we propose that thoroughly stress testing candidate definitions of functional modularity in simplified scenarios before extending to more complex data, network models and electrophysiological recordings is likely to be a fruitful approach.\",\n",
       "  'categories': 'q-bio.nc cs.ai cs.lg cs.ne',\n",
       "  'doi': '',\n",
       "  'created': '2021-06-04',\n",
       "  'updated': '2024-10-12',\n",
       "  'authors': ['gabriel béna', 'dan f. m. goodman'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2106.02626'},\n",
       " {'title': 'one-shot learning of paired association navigation with biologically   plausible schemas',\n",
       "  'id': '2106.03580',\n",
       "  'abstract': \"schemas are knowledge structures that can enable rapid learning. rodent one-shot learning in a multiple paired association navigation task has been postulated to be schema-dependent. we still only poorly understand how schemas, conceptualized at marr's computational level, are neurally implemented. moreover, a biologically plausible computational model of the rodent learning has not been demonstrated. accordingly, we here compose an agent from schemas with biologically plausible neural implementations. the agent gradually learns a metric representation of its environment using a path integration temporal difference error, allowing it to localize in any environment. additionally, the agent contains an associative memory that can stably form numerous one-shot associations between sensory cues and goal coordinates, implemented with a feedforward layer or a reservoir of recurrently connected neurons whose plastic output weights are governed by a 4-factor reward-modulated exploratory hebbian (eh) rule. a third network performs vector subtraction between the agent's current and goal location to decide the direction of movement. we further show that schemas supplemented by an actor-critic allows the agent to succeed even if an obstacle prevents direct heading, and that temporal-difference learning of a working memory gating mechanism enables one-shot learning despite distractors. our agent recapitulates learning behavior observed in experiments and provides testable predictions that can be probed in future experiments.\",\n",
       "  'categories': 'cs.ne q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2021-06-07',\n",
       "  'updated': '2024-09-09',\n",
       "  'authors': ['m ganesh kumar',\n",
       "   'cheston tan',\n",
       "   'camilo libedinsky',\n",
       "   'shih-cheng yen',\n",
       "   'andrew yong-yi tan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2106.03580'},\n",
       " {'title': 'calibrating covid-19 seir models with time-varying effective contact   rates',\n",
       "  'id': '2106.04705',\n",
       "  'abstract': 'we describe the population-based seir (susceptible, exposed, infected, removed) model developed by the irish epidemiological modelling advisory group (iemag), which advises the irish government on covid-19 responses. the model assumes a time-varying effective contact rate (equivalently, a time-varying reproduction number) to model the effect of non-pharmaceutical interventions. a crucial technical challenge in applying such models is their accurate calibration to observed data, e.g., to the daily number of confirmed new cases, as the past history of the disease strongly affects predictions of future scenarios. we demonstrate an approach based on inversion of the seir equations in conjunction with statistical modelling and spline-fitting of the data, to produce a robust methodology for calibration of a wide class of models of this type.',\n",
       "  'categories': 'physics.soc-ph q-bio.pe',\n",
       "  'doi': '10.1098/rsta.2021.0120',\n",
       "  'created': '2021-06-08',\n",
       "  'updated': '',\n",
       "  'authors': ['james p. gleeson',\n",
       "   'thomas brendan murphy',\n",
       "   \"joseph d. o'brien\",\n",
       "   'nial friel',\n",
       "   'norma bargary',\n",
       "   \"david j. p. o'sullivan\"],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2106.04705'},\n",
       " {'title': 'transition behavior of the seizure dynamics modulated by the astrocyte   inositol triphosphate noise',\n",
       "  'id': '2106.05180',\n",
       "  'abstract': 'epilepsy is a neurological disorder with recurrent seizures of complexity and randomness. until now, the mechanism of epileptic randomness has not been fully elucidated. inspired by the recent finding that astrocyte gtpase-activating protein (g-protein)-coupled receptors could be involved in stochastic epileptic seizures, we proposed a neuron-astrocyte network model, incorporating the noise of the astrocytic second messager, inositol triphosphate (ip3) which is modulated by the g-protein)-coupled receptor activation. based on this model, we have statistically analysed the transitions of epileptic seizures by performing tens of simulation trials. our simulation results show that the increase of the ip3 noise intensity induces the depolarization-block epileptic seizures together with an increase in neuronal firing frequency. meanwhile, a bistable state of neuronal firing emerges under certain noise intensity, during which the neuronal firing pattern switches between regular sparse spiking and epileptic seizure states. this random presence of epileptic seizures is absent when the noise intensity continues to increase, accompanying with an increase in the epileptic depolarization block duration. the simulation results also shed light on the fact that calcium signals in astrocytes play significant roles in the pattern formations of the epileptic seizure. our results provide a potential pathway for understanding the epileptic randomness.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1063/5.0124123',\n",
       "  'created': '2021-05-26',\n",
       "  'updated': '2022-10-31',\n",
       "  'authors': ['jiajia li',\n",
       "   'peihua feng',\n",
       "   'liang zhao',\n",
       "   'junying chen',\n",
       "   'mengmeng du',\n",
       "   'yangyang yu',\n",
       "   'jian song',\n",
       "   'ying wu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2106.05180'},\n",
       " {'title': 'revealing the canalizing structure of boolean functions: algorithms and   applications',\n",
       "  'id': '2106.06580',\n",
       "  'abstract': 'boolean functions can be represented in many ways including logical forms, truth tables, and polynomials. additionally, boolean functions have different canonical representations such as minimal disjunctive normal forms. other canonical representation is based on the polynomial representation of boolean functions where they can be written as a nested product of canalizing layers and a polynomial that contains the noncanalizing variables. in this paper we study the problem of identifying the canalizing layers format of boolean functions. first, we show that the problem of finding the canalizing layers is np-hard. second, we present several algorithms for finding the canalizing layers of a boolean function, discuss their complexities, and compare their performances. third, we show applications where the computation of canalizing layers can be used for finding a disjunctive normal form of a nested canalizing function. another application deals with the reverse engineering of boolean networks with a prescribed layering format. finally, implementations of our algorithms in python and in the computer algebra system macaulay2 are available at https://github.com/ckadelka/booleancanalization.',\n",
       "  'categories': 'cs.cc q-bio.qm',\n",
       "  'doi': '10.1016/j.automatica.2022.110630',\n",
       "  'created': '2021-06-11',\n",
       "  'updated': '2022-05-20',\n",
       "  'authors': ['elena dimitrova',\n",
       "   'brandilyn stigler',\n",
       "   'claus kadelka',\n",
       "   'david murrugarra'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2106.06580'},\n",
       " {'title': 'the backpropagation algorithm implemented on spiking neuromorphic   hardware',\n",
       "  'id': '2106.07030',\n",
       "  'abstract': \"the capabilities of natural neural systems have inspired new generations of machine learning algorithms as well as neuromorphic very large-scale integrated (vlsi) circuits capable of fast, low-power information processing. however, it has been argued that most modern machine learning algorithms are not neurophysiologically plausible. in particular, the workhorse of modern deep learning, the backpropagation algorithm, has proven difficult to translate to neuromorphic hardware. in this study, we present a neuromorphic, spiking backpropagation algorithm based on synfire-gated dynamical information coordination and processing, implemented on intel's loihi neuromorphic research processor. we demonstrate a proof-of-principle three-layer circuit that learns to classify digits from the mnist dataset. to our knowledge, this is the first work to show a spiking neural network (snn) implementation of the backpropagation algorithm that is fully on-chip, without a computer in the loop. it is competitive in accuracy with off-chip trained snns and achieves an energy-delay product suitable for edge computing. this implementation shows a path for using in-memory, massively parallel neuromorphic processors for low-power, low-latency implementation of modern deep learning applications.\",\n",
       "  'categories': 'cs.ne cs.ai cs.lg q-bio.nc',\n",
       "  'doi': '10.1038/s41467-024-53827-9',\n",
       "  'created': '2021-06-13',\n",
       "  'updated': '2021-08-26',\n",
       "  'authors': ['alpha renner',\n",
       "   'forrest sheldon',\n",
       "   'anatoly zlotnik',\n",
       "   'louis tao',\n",
       "   'andrew sornborger'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2106.07030'},\n",
       " {'title': 'tests for partial correlation between repeatedly observed nonstationary   nonlinear timeseries',\n",
       "  'id': '2106.07096',\n",
       "  'abstract': 'we describe two families of statistical tests to detect partial correlation in vectorial timeseries. the tests measure whether an observed timeseries y can be predicted from a second series x, even after accounting for a third series z which may correlate with x. they do not make any assumptions on the nature of these timeseries, such as stationarity or linearity, but they do require that multiple statistically independent recordings of the 3 series are available. intuitively, the tests work by asking if the series y recorded on one experiment can be better predicted from x recorded on the same experiment than on a different experiment, after accounting for the prediction from z recorded on both experiments.',\n",
       "  'categories': 'stat.me q-bio.nc stat.ap',\n",
       "  'doi': '',\n",
       "  'created': '2021-06-13',\n",
       "  'updated': '2024-04-24',\n",
       "  'authors': ['kenneth d. harris', 'alex e. yuan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2106.07096'},\n",
       " {'title': 'instrumental variable estimation for compositional treatments',\n",
       "  'id': '2106.11234',\n",
       "  'abstract': 'many scientific datasets are compositional in nature. important biological examples include species abundances in ecology, cell-type compositions derived from single-cell sequencing data, and amplicon abundance data in microbiome research. here, we provide a causal view on compositional data in an instrumental variable setting where the composition acts as the cause. first, we crisply articulate potential pitfalls for practitioners regarding the interpretation of compositional causes from the viewpoint of interventions and warn against attributing causal meaning to common summary statistics such as diversity indices in microbiome data analysis. we then advocate for and develop multivariate methods using statistical data transformations and regression techniques that take the special structure of the compositional sample space into account while still yielding scientifically interpretable results. in a comparative analysis on synthetic and real microbiome data we show the advantages and limitations of our proposal. we posit that our analysis provides a useful framework and guidance for valid and informative cause-effect estimation in the context of compositional data.',\n",
       "  'categories': 'cs.lg q-bio.qm stat.ap stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2021-06-21',\n",
       "  'updated': '2024-05-28',\n",
       "  'authors': ['elisabeth ailer', 'christian l. müller', 'niki kilbertus'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2106.11234'},\n",
       " {'title': 'on the relationship between predictive coding and backpropagation',\n",
       "  'id': '2106.13082',\n",
       "  'abstract': 'artificial neural networks are often interpreted as abstract models of biological neuronal networks, but they are typically trained using the biologically unrealistic backpropagation algorithm and its variants. predictive coding has been proposed as a potentially more biologically realistic alternative to backpropagation for training neural networks. this manuscript reviews and extends recent work on the mathematical relationship between predictive coding and backpropagation for training feedforward artificial neural networks on supervised learning tasks. implications of these results for the interpretation of predictive coding and deep neural networks as models of biological learning are discussed along with a repository of functions, torch2pc, for performing predictive coding with pytorch neural network models.',\n",
       "  'categories': 'q-bio.nc cs.lg cs.ne',\n",
       "  'doi': '10.1371/journal.pone.0266102',\n",
       "  'created': '2021-06-20',\n",
       "  'updated': '2024-04-23',\n",
       "  'authors': ['robert rosenbaum'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2106.13082'},\n",
       " {'title': 'disorder in cellular packing can alter proliferation dynamics to   regulate growth',\n",
       "  'id': '2106.15578',\n",
       "  'abstract': 'controlling growth via cell division is crucial in the development of higher organisms, and yet the mechanisms through which this is achieved, e.g., in epithelial tissue, is not yet fully understood. we show that by coupling the cell cycle oscillator governing cell division to signals that encode inter-cellular contacts, this phenomenon can be seen as a collective dynamical transition in a system of coupled oscillators in lattices with changing degree of disorder. as the distribution of cellular morphological characteristics become more homogeneous over the course of development, the contact-induced signals to the cells increase beyond a critical value to trigger coordinated cessation of oscillations, eventually leading to growth arrest. our results suggest that the global phenomenon of growth rate reduction as a tissue approaches its appropriate size is causally related to the increasingly regular geometry of local cell-cell contact interfaces.',\n",
       "  'categories': 'q-bio.to nlin.ao physics.bio-ph',\n",
       "  'doi': '10.1103/physreve.104.l052401',\n",
       "  'created': '2021-06-29',\n",
       "  'updated': '',\n",
       "  'authors': ['chandrashekar kuyyamudi',\n",
       "   'shakti n. menon',\n",
       "   'fernando casares',\n",
       "   'sitabhra sinha'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2106.15578'},\n",
       " {'title': 'a survey on graph-based deep learning for computational histopathology',\n",
       "  'id': '2107.00272',\n",
       "  'abstract': 'with the remarkable success of representation learning for prediction problems, we have witnessed a rapid expansion of the use of machine learning and deep learning for the analysis of digital pathology and biopsy image patches. however, learning over patch-wise features using convolutional neural networks limits the ability of the model to capture global contextual information and comprehensively model tissue composition. the phenotypical and topological distribution of constituent histological entities play a critical role in tissue diagnosis. as such, graph data representations and deep learning have attracted significant attention for encoding tissue representations, and capturing intra- and inter- entity level interactions. in this review, we provide a conceptual grounding for graph analytics in digital pathology, including entity-graph construction and graph architectures, and present their current success for tumor localization and classification, tumor invasion and staging, image retrieval, and survival prediction. we provide an overview of these methods in a systematic manner organized by the graph representation of the input image, scale, and organ on which they operate. we also outline the limitations of existing techniques, and suggest potential future research directions in this domain.',\n",
       "  'categories': 'cs.lg cs.cv q-bio.to',\n",
       "  'doi': '10.1016/j.compmedimag.2021.102027',\n",
       "  'created': '2021-07-01',\n",
       "  'updated': '2021-09-27',\n",
       "  'authors': ['david ahmedt-aristizabal',\n",
       "   'mohammad ali armin',\n",
       "   'simon denman',\n",
       "   'clinton fookes',\n",
       "   'lars petersson'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2107.00272'},\n",
       " {'title': 'a generic self-stabilization mechanism for biomolecular adhesions under   load',\n",
       "  'id': '2107.03714',\n",
       "  'abstract': 'mechanical loading generally weakens adhesive structures and eventually leads to their rupture. however, biological systems can adapt to loads by strengthening adhesions, which is essential for maintaining the integrity of tissue and whole organisms. inspired by cellular focal adhesions, we suggest here a generic, molecular mechanism that allows adhesion systems to harness applied loads for self-stabilization under non-equilibrium conditions -- without any active feedback involved. the mechanism is based on conformation changes of adhesion molecules that are dynamically exchanged with a reservoir. tangential loading drives the occupation of some stretched conformation states out of equilibrium, which, for thermodynamic reasons, leads to association of further molecules with the adhesion cluster. self-stabilization robustly increases adhesion lifetimes in broad parameter ranges. unlike for catch-bonds, bond dissociation rates do not decrease with force. the self-stabilization principle can be realized in many ways in complex adhesion-state networks; we show how it naturally occurs in cellular adhesions involving the adaptor proteins talin and vinculin.',\n",
       "  'categories': 'physics.bio-ph cond-mat.soft q-bio.sc',\n",
       "  'doi': '10.1038/s41467-022-29823-2',\n",
       "  'created': '2021-07-08',\n",
       "  'updated': '',\n",
       "  'authors': ['andrea braeutigam',\n",
       "   'ahmet nihat simsek',\n",
       "   'gerhard gompper',\n",
       "   'benedikt sabass'],\n",
       "  'affiliation': ['theoretical physics of living matter, institute for biological information processes, forschungszentrum jülich, 52425 jülich, germany',\n",
       "   'theoretical physics of living matter, institute for biological information processes, forschungszentrum jülich, 52425 jülich, germany',\n",
       "   'theoretical physics of living matter, institute for biological information processes, forschungszentrum jülich, 52425 jülich, germany',\n",
       "   'theoretical physics of living matter, institute for biological information processes, forschungszentrum jülich, 52425 jülich, germany'],\n",
       "  'url': 'https://arxiv.org/abs/2107.03714'},\n",
       " {'title': 'graph autoencoders for embedding learning in brain networks and major   depressive disorder identification',\n",
       "  'id': '2107.12838',\n",
       "  'abstract': 'brain functional connectivity (fc) reveals biomarkers for identification of various neuropsychiatric disorders. recent application of deep neural networks (dnns) to connectome-based classification mostly relies on traditional convolutional neural networks using input connectivity matrices on a regular euclidean grid. we propose a graph deep learning framework to incorporate the non-euclidean information about graph structure for classifying functional magnetic resonance imaging (fmri)-derived brain networks in major depressive disorder (mdd). we design a novel graph autoencoder (gae) architecture based on the graph convolutional networks (gcns) to embed the topological structure and node content of large-sized fmri networks into low-dimensional latent representations. in network construction, we employ the ledoit-wolf (ldw) shrinkage method to estimate the high-dimensional fc metrics efficiently from fmri data. we consider both supervised and unsupervised approaches for the graph embedding learning. the learned embeddings are then used as feature inputs for a deep fully-connected neural network (fcnn) to discriminate mdd from healthy controls. evaluated on two resting-state fmri (rs-fmri) mdd datasets, results show that the proposed gae-fcnn model significantly outperforms several state-of-the-art methods for brain connectome classification, achieving the best accuracy using the ldw-fc edges as node features. the graph embeddings of fmri fc networks learned by the gae also reveal apparent group differences between mdd and hc. our new framework demonstrates feasibility of learning graph embeddings on brain networks to provide discriminative information for diagnosis of brain disorders.',\n",
       "  'categories': 'q-bio.nc cs.ai cs.lg',\n",
       "  'doi': '10.1109/jbhi.2024.3351177',\n",
       "  'created': '2021-07-27',\n",
       "  'updated': '2022-06-02',\n",
       "  'authors': ['fuad noman',\n",
       "   'chee-ming ting',\n",
       "   'hakmook kang',\n",
       "   'raphael c. -w. phan',\n",
       "   'brian d. boyd',\n",
       "   'warren d. taylor',\n",
       "   'hernando ombao'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2107.12838'},\n",
       " {'title': 'the evolution of forecasting for decision making in dynamic environments',\n",
       "  'id': '2108.00047',\n",
       "  'abstract': 'global change is reshaping ecosystems and societies. strategic choices that were best yesterday may be sub-optimal tomorrow; and environmental conditions that were once taken for granted may soon cease to exist. in this setting, how people choose behavioral strategies has important consequences for environmental dynamics. economic and evolutionary theories make similar predictions for strategic behavior in a static environment, even though one approach assumes perfect rationality and the other assumes no cognition whatsoever; but predictions differ in a dynamic environment. here we explore a middle ground between economic rationality and evolutionary myopia. starting from a population of myopic agents, we study the emergence of a new type that forms environmental forecasts when making strategic decisions. we show that forecasting types can have an advantage in changing environments, even when the act of forecasting is costly. forecasting types can invade but not overtake the population, producing a stable coexistence with myopic types. moreover, forecasters provide a public good by reducing the amplitude of environmental oscillations and increasing mean payoff to forecasting and myopic types alike. we interpret our results for understanding the evolution of different modes of decision-making. and we discuss implications for the management of environmental systems of great societal importance.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1177/26339137231221726',\n",
       "  'created': '2021-07-30',\n",
       "  'updated': '2022-08-26',\n",
       "  'authors': ['andrew r. tilman',\n",
       "   'vítor v. vasconcelos',\n",
       "   'erol akçay',\n",
       "   'joshua b. plotkin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2108.00047'},\n",
       " {'title': 'biocode: a data-driven procedure to learn the growth of biological   networks',\n",
       "  'id': '2108.04776',\n",
       "  'abstract': \"probabilistic biological network growth models have been utilized for many tasks including but not limited to capturing mechanism and dynamics of biological growth activities, null model representation, capturing anomalies, etc. well-known examples of these probabilistic models are kronecker model, preferential attachment model, and duplication-based model. however, we should frequently keep developing new models to better fit and explain the observed network features while new networks are being observed. additionally, it is difficult to develop a growth model each time we study a new network. in this paper, we propose biocode, a framework to automatically discover novel biological growth models matching user-specified graph attributes in directed and undirected biological graphs. biocode designs a basic set of instructions which are common enough to model a number of well-known biological graph growth models. we combine such instruction-wise representation with a genetic algorithm based optimization procedure to encode models for various biological networks. we mainly evaluate the performance of biocode in discovering models for biological collaboration networks, gene regulatory networks, metabolic networks, and protein interaction networks which features such as assortativity, clustering coefficient, degree distribution closely match with the true ones in the corresponding real biological networks. as shown by the tests on the simulated graphs, the variance of the distributions of biological networks generated by biocode is similar to the known models' variance for these biological network types.\",\n",
       "  'categories': 'q-bio.mn q-bio.qm',\n",
       "  'doi': '10.1109/tcbb.2022.3165092',\n",
       "  'created': '2021-08-10',\n",
       "  'updated': '2021-09-05',\n",
       "  'authors': ['emre sefer'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2108.04776'},\n",
       " {'title': 'deepgene transformer: transformer for the gene expression-based   classification of cancer subtypes',\n",
       "  'id': '2108.11833',\n",
       "  'abstract': 'cancer and its subtypes constitute approximately 30% of all causes of death globally and display a wide range of heterogeneity in terms of clinical and molecular responses to therapy. molecular subtyping has enabled the use of precision medicine to overcome these challenges and provide significant biological insights to predict prognosis and improve clinical decision-making. over the past decade, conventional machine learning (ml) and deep learning (dl) algorithms have been widely espoused for the classification of cancer subtypes from gene expression datasets. however, these methods are potentially biased toward the identification of cancer biomarkers. hence, an end-to-end deep learning approach, deepgene transformer, is proposed which addresses the complexity of high-dimensional gene expression with a multi-head self-attention module by identifying relevant biomarkers across multiple cancer subtypes without requiring feature selection as a pre-requisite for the current classification algorithms. comparative analysis reveals that the proposed deepgene transformer outperformed the commonly used traditional and state-of-the-art classification algorithms and can be considered an efficient approach for classifying cancer and its subtypes, indicating that any improvement in deep learning models in computational biologists can be reflected well in this domain as well.',\n",
       "  'categories': 'q-bio.qm cs.ai q-bio.gn',\n",
       "  'doi': '',\n",
       "  'created': '2021-08-26',\n",
       "  'updated': '2024-07-10',\n",
       "  'authors': ['anwar khan', 'boreom lee'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2108.11833'},\n",
       " {'title': 'rapidly and accurately estimating brain strain and strain rate across   head impact types with transfer learning and data fusion',\n",
       "  'id': '2108.13577',\n",
       "  'abstract': 'brain strain and strain rate are effective in predicting traumatic brain injury (tbi) caused by head impacts. however, state-of-the-art finite element modeling (fem) demands considerable computational time in the computation, limiting its application in real-time tbi risk monitoring. to accelerate, machine learning head models (mlhms) were developed, and the model accuracy was found to decrease when the training/test datasets were from different head impacts types. however, the size of dataset for specific impact types may not be enough for model training. to address the computational cost of fem, the limited strain rate prediction, and the generalizability of mlhms to on-field datasets, we propose data fusion and transfer learning to develop a series of mlhms to predict the maximum principal strain (mps) and maximum principal strain rate (mpsr). we trained and tested the mlhms on 13,623 head impacts from simulations, american football, mixed martial arts, car crash, and compared against the models trained on only simulations or only on-field impacts. the mlhms developed with transfer learning are significantly more accurate in estimating mps and mpsr than other models, with a mean absolute error (mae) smaller than 0.03 in predicting mps and smaller than 7 (1/s) in predicting mpsr on all impact datasets. the mlhms can be applied to various head impact types for rapidly and accurately calculating brain strain and strain rate. besides the clinical applications in real-time brain strain and strain rate monitoring, this model helps researchers estimate the brain strain and strain rate caused by head impacts more efficiently than fem.',\n",
       "  'categories': 'cs.lg physics.med-ph q-bio.qm q-bio.to',\n",
       "  'doi': '10.1109/tbme.2024.3354192',\n",
       "  'created': '2021-08-30',\n",
       "  'updated': '',\n",
       "  'authors': ['xianghao zhan',\n",
       "   'yuzhe liu',\n",
       "   'nicholas j. cecchi',\n",
       "   'olivier gevaert',\n",
       "   'michael m. zeineh',\n",
       "   'gerald a. grant',\n",
       "   'david b. camarillo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2108.13577'},\n",
       " {'title': 'using convolutional neural networks for the classification of breast   cancer images',\n",
       "  'id': '2108.13661',\n",
       "  'abstract': 'an important part of breast cancer staging is the assessment of the sentinel axillary node for early signs of tumor spreading. however, this assessment by pathologists is not always easy and retrospective surveys often requalify the status of a high proportion of sentinel nodes. convolutional neural networks (cnns) are a class of deep learning algorithms that have shown excellent performances in the most challenging visual classification tasks, with numerous applications in medical imaging. in this study i compare twelve different cnns and different hardware acceleration devices for the detection of breast cancer from microscopic images of breast cancer tissue. convolutional models are trained and tested on two public datasets. the first one is composed of more than 300,000 images of sentinel lymph node tissue from breast cancer patients, while the second one has more than 220,000 images from inductive breast carcinoma tissue, one of the most common forms of breast cancer. four different hardware acceleration cards were used, with an off-the-shelf deep learning framework. the impact of transfer learning and hyperparameters fine-tuning are tested. hardware acceleration device performance can improve training time by a factor of five to twelve, depending on the model used. on the other hand, increasing convolutional depth will augment the training time by a factor of four to six times, depending on the acceleration device used. increasing the depth and the complexity of the model generally improves performance, but the relationship is not linear and also depends on the architecture of the model. the performance of transfer learning is always worse compared to a complete retraining of the model. fine-tuning the hyperparameters of the model improves the results, with the best model showing a performance comparable to state-of-the-art models.',\n",
       "  'categories': 'q-bio.qm eess.iv',\n",
       "  'doi': '',\n",
       "  'created': '2021-08-31',\n",
       "  'updated': '2024-04-29',\n",
       "  'authors': ['eric bonnet'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2108.13661'},\n",
       " {'title': 'resting state fmri-based temporal coherence mapping',\n",
       "  'id': '2109.00146',\n",
       "  'abstract': 'long-range temporal coherence (lrtc) is quite common to dynamic systems and is fundamental to the system function. lrtc in the brain has been shown to be important to cognition. assessing lrtc may provide critical information for understanding the potential underpinnings of brain organization, function, and cognition. to facilitate this overarching goal, we provide a method, which is named temporal coherence mapping (tcm), to explicitly quantify lrtc using resting state fmri. tcm is based on correlation analysis of the transit states of the phase space reconstructed by temporal embedding. a few tcm properties were collected to measure lrtc, including the averaged correlation, anti-correlation, the ratio of correlation and anticorrelation, the mean coherent and incoherent duration, and the ratio between the coherent and incoherent time. tcm was first evaluated with simulations and then with the large human connectome project data. evaluation results showed that tcm metrics can successfully differentiate signals with different temporal coherence regardless of the parameters used to reconstruct the phase space. in human brain, tcm metrics except the ratio of the coherent/incoherent time showed high test-retest reproducibility; tcm metrics are related to age, sex, and total cognitive scores. in summary, tcm provides a first-of-its-kind tool to assess lrtc and the imbalance between coherence and incoherence; tcm properties are physiologically and cognitively meaningful.',\n",
       "  'categories': 'q-bio.nc q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2021-08-31',\n",
       "  'updated': '2024-07-09',\n",
       "  'authors': ['ze wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2109.00146'},\n",
       " {'title': 'inferring gene regulation dynamics from static snapshots of gene   expression variability',\n",
       "  'id': '2109.00392',\n",
       "  'abstract': 'inferring functional relationships within complex networks from static snapshots of a subset of variables is a ubiquitous problem in science. for example, a key challenge of systems biology is to translate cellular heterogeneity data obtained from single-cell sequencing or flow-cytometry experiments into regulatory dynamics. we show how static population snapshots of co-variability can be exploited to rigorously infer properties of gene expression dynamics when gene expression reporters probe their upstream dynamics on separate time-scales. this can be experimentally exploited in dual-reporter experiments with fluorescent proteins of unequal maturation times, thus turning an experimental bug into an analysis feature. we derive correlation conditions that detect the presence of closed-loop feedback regulation in gene regulatory networks. furthermore, we show how genes with cell-cycle dependent transcription rates can be identified from the variability of co-regulated fluorescent proteins. similar correlation constraints might prove useful in other areas of science in which static correlation snapshots are used to infer causal connections between dynamically interacting components.',\n",
       "  'categories': 'q-bio.mn q-bio.qm',\n",
       "  'doi': '10.1103/physreve.104.044406',\n",
       "  'created': '2021-09-01',\n",
       "  'updated': '',\n",
       "  'authors': ['euan joly-smith', 'zitong jerry wang', 'andreas hilfinger'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2109.00392'},\n",
       " {'title': 'dominance reversals: the resolution of genetic conflict and maintenance   of genetic variation',\n",
       "  'id': '2109.01571',\n",
       "  'abstract': 'beneficial reversals of dominance reduce the costs of genetic trade-offs and can enable selection to maintain genetic variation for fitness. beneficial dominance reversals are characterized by the beneficial allele for a given context (e.g. habitat, developmental stage, trait, or sex) being dominant in that context but recessive where deleterious. this context-dependence at least partially mitigates the fitness consequence of heterozygotes carrying one non-beneficial allele for their context and can result in balancing selection that maintains alternative alleles. dominance reversals are theoretically plausible and are supported by mounting empirical evidence. here we highlight the importance of beneficial dominance reversals as a mechanism for the mitigation of genetic conflict and review the theory and empirical evidence for them. we identify some areas in need of further research and development and outline three methods (dominance ordination, allele-specific expression, and allele-specific atac-seq) that could facilitate the identification of antagonistic genetic variation. there is ample scope for the development of new empirical methods as well as reanalysis of existing data through the lens of dominance reversals. a greater focus on this topic will expand our understanding of the mechanisms that resolve genetic conflict and whether they maintain genetic variation.',\n",
       "  'categories': 'q-bio.pe q-bio.qm',\n",
       "  'doi': '10.1098/rspb.2023.2816',\n",
       "  'created': '2021-09-03',\n",
       "  'updated': '2024-01-26',\n",
       "  'authors': ['karl grieshop', 'eddie k. h. ho', 'katja r. kasimatis'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2109.01571'},\n",
       " {'title': 'emergence of robust memory manifolds',\n",
       "  'id': '2109.03879',\n",
       "  'abstract': 'the ability to store continuous variables in the state of a biological system (e.g. a neural network) is critical for many behaviours. most models for implementing such a memory manifold require hand-crafted symmetries in the interactions or precise fine-tuning of parameters. we present a general principle that we refer to as {\\\\it frozen stabilisation} (fs), which allows a family of neural networks to self-organise to a critical state exhibiting multiple memory manifolds without parameter fine-tuning or symmetries. memory manifolds arising from fs exhibit a wide range of emergent relaxational timescales and can be used as general purpose integrators for inputs aligned with the manifold. moreover, fs allows robust memory manifolds in small networks, and this is relevant to debates of implementing continuous attractors with a small number of neurons in light of recent experimental discoveries.',\n",
       "  'categories': 'q-bio.nc cond-mat.dis-nn physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2021-09-08',\n",
       "  'updated': '2024-09-05',\n",
       "  'authors': ['tankut can', 'kamesh krishnamurthy'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2109.03879'},\n",
       " {'title': 'two-type branching processes with immigration, and the structured   coalescents',\n",
       "  'id': '2109.07645',\n",
       "  'abstract': 'we consider a population constituted by two types of individuals; each of them can produce offspring in two different islands (as a particular case the islands can be interpreted as active or dormant individuals). we model the evolution of the population of each type using a two-type feller diffusion with immigration, and we study the frequency of one of the types, in each island, when the total population size in each island is forced to be constant at a dense set of times. this leads to the solution of a sde which we call the asymmetric two-island frequency process. we derive properties of this process and obtain a large population limit when the total size of each island tends to infinity. additionally, we compute the fluctuations of the process around its deterministic limit. we establish conditions under which the asymmetric two-island frequency process has a moment dual. the dual is a continuous-time two-dimensional markov chain that can be interpreted in terms of mutation, branching, pairwise branching, coalescence, and a novel mixed selection-migration term. also, we conduct a stability analysis of the limiting deterministic dynamical system and present some numerical results to study fixation and a new form of balancing selection. when restricting to the seedbank model, we observe that some combinations of the parameters lead to balancing selection. besides finding yet another way in which genetic reservoirs increase the genetic variability, we find that if a population that sustains a seedbank competes with one that does not, the seed producers will have a selective advantage if they reproduce faster, but will not have a selective disadvantage if they reproduce slower: their worst case scenario is balancing selection.',\n",
       "  'categories': 'q-bio.pe math.pr',\n",
       "  'doi': '',\n",
       "  'created': '2021-09-15',\n",
       "  'updated': '2024-05-01',\n",
       "  'authors': ['maría emilia caballero',\n",
       "   'adrián gonzález casanova',\n",
       "   'josé luis pérez'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2109.07645'},\n",
       " {'title': 'the effects of random and seasonal environmental fluctuations on optimal   harvesting and stocking',\n",
       "  'id': '2109.13169',\n",
       "  'abstract': 'we analyze the harvesting and stocking of a population that is affected by random and seasonal environmental fluctuations. the main novelty comes from having three layers of environmental fluctuations. the first layer is due to the environment switching at random times between different environmental states. this is similar to having sudden environmental changes or catastrophes. the second layer is due to seasonal variation, where there is a significant change in the dynamics between seasons. finally, the third layer is due to the constant presence of environmental stochasticity -- between the seasonal or random regime switches, the species is affected by fluctuations which can be modelled by white noise. this framework is more realistic because it can capture both significant random and deterministic environmental shifts as well as small and frequent fluctuations in abiotic factors. our framework also allows for the price or cost of harvesting to change deterministically and stochastically, something that is more realistic from an economic point of view.   the combined effects of seasonal and random fluctuations make it impossible to find the optimal harvesting-stocking strategy analytically. we get around this roadblock by developing rigorous numerical approximations and proving that they converge to the optimal harvesting-stocking strategy. we apply our methods to multiple population models and explore how prices, or costs, and environmental fluctuations influence the optimal harvesting-stocking strategy. we show that in many situations the optimal way of harvesting and stocking is not of threshold type.',\n",
       "  'categories': 'math.pr math.oc q-bio.pe',\n",
       "  'doi': '10.1007/s00285-022-01750-2',\n",
       "  'created': '2021-09-27',\n",
       "  'updated': '2021-09-27',\n",
       "  'authors': ['alexandru hening', 'ky q. tran', 'sergiu c. ungureanu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2109.13169'},\n",
       " {'title': 'mining for potent inhibitors through artificial intelligence and   physics: a unified methodology for ligand based and structure based drug   design',\n",
       "  'id': '2110.01806',\n",
       "  'abstract': 'the viability of a new drug molecule is a time and resource intensive task that makes computer-aided assessments a vital approach to rapid drug discovery. here we develop a machine learning algorithm, iminer, that generates novel inhibitor molecules for target proteins by combining deep reinforcement learning with real-time 3d molecular docking using autodock vina, thereby simultaneously creating chemical novelty while constraining molecules for shape and molecular compatibility with target active sites. moreover, through the use of various types of reward functions, we can generate new molecules that are chemically similar to a target ligand, which can be grown from known protein bound fragments, as well as to create molecules that enforce interactions with target residues in the protein active site. the iminer algorithm is embedded in a composite workflow that filters out pan-assay interference compounds, lipinski rule violations, and poor synthetic accessibility, with options for cross-validation against other docking scoring functions and automation of a molecular dynamics simulation to measure pose stability. because our approach only relies on the structure of the target protein, iminer can be easily adapted for future development of other inhibitors or small molecule therapeutics of any target protein.',\n",
       "  'categories': 'q-bio.bm physics.bio-ph physics.chem-ph physics.data-an',\n",
       "  'doi': '',\n",
       "  'created': '2021-10-04',\n",
       "  'updated': '2024-01-10',\n",
       "  'authors': ['jie li',\n",
       "   'oufan zhang',\n",
       "   'yingze wang',\n",
       "   'kunyang sun',\n",
       "   'xingyi guan',\n",
       "   'dorian bagni',\n",
       "   'mojtaba haghighatlari',\n",
       "   'fiona l. kearns',\n",
       "   'conor parks',\n",
       "   'rommie e. amaro',\n",
       "   'teresa head-gordon'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2110.01806'},\n",
       " {'title': 'magorino: magnitude-only fat fraction and r2* estimation with rician   noise modelling',\n",
       "  'id': '2110.05400',\n",
       "  'abstract': 'purpose: magnitude-based fitting of chemical shift-encoded data enables proton density fat fraction (pdff) and r2* estimation where complex-based methods fail or when phase data is inaccessible or unreliable. however, traditional magnitude-based fitting algorithms do not account for rician noise, creating a source of bias. to address these issues, we propose an algorithm for magnitude-only pdff and r2* estimation with rician noise modelling (magorino). methods: simulations of multi-echo gradient echo signal intensities are used to investigate the performance and behavior of magorino over the space of clinically-plausible pdff, r2* and signal-to-noise ratio (snr) values. fitting performance is assessed through detailed simulation, including likelihood function visualization, and in a multi-site, multi-vendor and multi-field-strength phantom dataset and in vivo. results: simulations show that rician noise-based magnitude fitting outperforms existing gaussian noise-based fitting and reveals two key mechanisms underpinning the observed improvement. firstly, the likelihood functions exhibit two local optima; rician noise modelling increases the chance the global optimum corresponds to the ground truth. secondly, when the global optimum corresponds to ground truth for both noise models, the optimum from rician noise modelling is closer to ground truth. multisite phantom experiments show good agreement of magorino pdff with reference values, and in vivo experiments replicate the performance benefits observed in simulation. conclusion: magorino reduces rician noise-related bias in pdff and r2* estimation, thus addressing a key limitation of existing magnitude-only fitting methods. our results offer an insight into the importance of the noise model for selecting the correct optimum when multiple plausible optima exist.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.1002/mrm.29493',\n",
       "  'created': '2021-10-11',\n",
       "  'updated': '2024-11-04',\n",
       "  'authors': ['timothy jp bray',\n",
       "   'alan bainbridge',\n",
       "   'margaret a hall-craggs',\n",
       "   'hui zhang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2110.05400'},\n",
       " {'title': \"integrating fr\\\\'echet distance and ai reveals the evolutionary   trajectory and origin of sars-cov-2\",\n",
       "  'id': '2110.07696',\n",
       "  'abstract': \"a genome, composed of a precisely ordered sequence of four nucleotides (atcg), encompasses a multitude of specific genome features like aaa motif. mutations occurring within a genome disrupt the sequential order and composition of these features, thereby influencing the evolutionary trajectories and yielding variants. the evolutionary relatedness between a variant and its ancestor can be estimated by assessing evolutionary distances across a spectrum of genome features. this study develops a novel, alignment-free algorithm that considers both the sequential order and composition of genome features, enabling computation of the fr\\\\'echet distance (fr) across multiple genome features to quantify the evolutionary status of a variant. integrating this algorithm with an artificial recurrent neural network (rnn) reveals the quantitative evolutionary trajectory and origin of sars-cov-2, a puzzle unsolved by alignment-based phylogenetics. the rnn generates the evolutionary trajectory from fr data at two levels: genome sequence mutations and organism variants. at the genome sequence level, sars-cov-2 evolutionarily shortens its genome to enhance its infectious capacity. mutating signature features, such as tta and gct, increases its infectious potential and drives its evolution. at the organism level, variants mutating a single biomarker possess low infectious potential. however, mutating multiple markers dramatically increases their infectious capacity, propelling the covid-19 pandemic. sars-cov-2 likely originates from mink coronavirus variants, with its origin trajectory traced as follows: mink, cat, tiger, mouse, hamster, dog, lion, gorilla, leopard, bat, and pangolin. together, mutating multiple signature features and biomarkers delineates the evolutionary trajectory of mink-origin sars-cov-2, leading to the covid-19 pandemic. full text and detailed on https://combai.org/ai/covidgenome/\",\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '10.1002/jmv.29557',\n",
       "  'created': '2021-10-14',\n",
       "  'updated': '2024-03-23',\n",
       "  'authors': ['anyou wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2110.07696'},\n",
       " {'title': 'mathematical model of subthalamic nucleus neuron: characteristic   activity patterns and bifurcation analysis',\n",
       "  'id': '2110.10229',\n",
       "  'abstract': \"the subthalamic nucleus (stn) has an important role in the pathophysiology of the basal ganglia in parkinson's disease. the ability of stn cells to generate bursting rhythms under either transient or sustained hyperpolarization may underlie the excessively synchronous beta rhythms observed in parkinson's disease. in this study, we developed a conductance-based single compartment model of an stn neuron, which is able to generate characteristic activity patterns observed in experiments including hyperpolarization-induced bursts and post-inhibitory rebound bursts. this study focused on the role of three currents in rhythm generation: t-type calcium (cat) current, l-type calcium (cal) current, and hyperpolarization-activated cyclic nucleotide-gated (hcn) current. to investigate the effects of these currents in rhythm generation, we performed a bifurcation analysis using slow variables in these currents. bifurcation analysis showed that the hcn current promotes single-spike activity patterns rather than bursting in agreement with experimental results. it also showed that the cat current is necessary for characteristic bursting activity patterns. in particular, the cat current enables stn neurons to generate these activity patterns under hyperpolarizing stimuli. the cal current enriches and reinforces these characteristic activity patterns. in hyperpolarization-induced bursts or post-inhibitory rebound bursts, the cal current allows stn neurons to generate long bursting patterns. thus, bifurcation analysis explained the synergistic interaction of the cat and cal currents, which enables stn neurons to respond to hyperpolarizing stimuli in a salient way. the results of this study implicate the importance of cat and cal currents in the pathophysiology of the basal ganglia in parkinson's disease.\",\n",
       "  'categories': 'q-bio.nc physics.bio-ph',\n",
       "  'doi': '10.1063/5.0059773',\n",
       "  'created': '2021-10-19',\n",
       "  'updated': '',\n",
       "  'authors': ['choongseok park', 'leonid l. rubchinsky', 'sungwoo ahn'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2110.10229'},\n",
       " {'title': 'precision of morphogen-driven tissue patterning during development is   enhanced through contact-mediated cellular interactions',\n",
       "  'id': '2110.12927',\n",
       "  'abstract': 'embryonic development involves pattern formation characterized by the emergence of spatially localized domains characterized by distinct cell fates resulting from differential gene expression. the boundaries demarcating these domains are precise and consistent within a species despite stochastic fluctuations in the morphogen molecular concentration that provides positional information to the cells, as well as, the intrinsic noise in molecular processes that interpret this information to guide fate determination. we show that local interactions between physically adjacent cells mediated by receptor-ligand binding utilizes the asymmetry between the fate-determining genes to yield a switch-like response to the global signal provided by the morphogen. this results in robust developmental outcomes with a consistent identity of the gene that is dominantly expressed at each cellular location, thereby substantially reducing the uncertainty in the location of the boundary between distinct fates.',\n",
       "  'categories': 'q-bio.to nlin.ps physics.bio-ph',\n",
       "  'doi': '10.1103/physreve.107.024407',\n",
       "  'created': '2021-10-25',\n",
       "  'updated': '',\n",
       "  'authors': ['chandrashekar kuyyamudi', 'shakti n. menon', 'sitabhra sinha'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2110.12927'},\n",
       " {'title': 'multiscale convergence of the inverse problem for chemotaxis in the   bayesian setting',\n",
       "  'id': '2110.13787',\n",
       "  'abstract': \"chemotaxis describes the movement of an organism, such as single or multi-cellular organisms and bacteria, in response to a chemical stimulus. two widely used models to describe the phenomenon are the celebrated keller-segel equation and a chemotaxis kinetic equation. these two equations describe the organism movement at the macro- and mesoscopic level respectively, and are asymptotically equivalent in the parabolic regime. how the organism responds to a chemical stimulus is embedded in the diffusion/advection coefficients of the keller-segel equation or the turning kernel of the chemotaxis kinetic equation. experiments are conducted to measure the time dynamics of the organisms' population level movement when reacting to certain stimulation. from this one infers the chemotaxis response, which constitutes an inverse problem. \\\\\\\\ in this paper we discuss the relation between both the macro- and mesoscopic inverse problems, each of which is associated to two different forward models. the discussion is presented in the bayesian framework, where the posterior distribution of the turning kernel of the organism population is sought after. we prove the asymptotic equivalence of the two posterior distributions.\",\n",
       "  'categories': 'math.ap math.pr q-bio.qm',\n",
       "  'doi': '10.3390/computation9110119',\n",
       "  'created': '2021-10-26',\n",
       "  'updated': '2021-12-06',\n",
       "  'authors': ['kathrin hellmuth',\n",
       "   'christian klingenberg',\n",
       "   'qin li',\n",
       "   'min tang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2110.13787'},\n",
       " {'title': 'feature selection revisited in the single-cell era',\n",
       "  'id': '2110.14329',\n",
       "  'abstract': 'feature selection techniques are essential for high-dimensional data analysis. in the last two decades, their popularity has been fuelled by the increasing availability of high-throughput biomolecular data where high-dimensionality is a common data property. recent advances in biotechnologies enable global profiling of various molecular and cellular features at single-cell resolution, resulting in large-scale datasets with increased complexity. these technological developments have led to a resurgence in feature selection research and application in the single-cell field. here, we revisit feature selection techniques and summarise recent developments. we review their versatile application to a range of single-cell data types including those generated from traditional cytometry and imaging technologies and the latest array of single-cell omics technologies. we highlight some of the challenges and future directions on which feature selection could have a significant impact. finally, we consider the scalability and make general recommendations on the utility of each type of feature selection method. we hope this review serves as a reference point to stimulate future research and application of feature selection in the single-cell era.',\n",
       "  'categories': 'q-bio.qm cs.lg',\n",
       "  'doi': '10.1186/s13059-021-02544-3',\n",
       "  'created': '2021-10-27',\n",
       "  'updated': '',\n",
       "  'authors': ['pengyi yang', 'hao huang', 'chunlei liu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2110.14329'},\n",
       " {'title': 'learning-based importance sampling via stochastic optimal control for   stochastic reaction networks',\n",
       "  'id': '2110.14335',\n",
       "  'abstract': 'we explore efficient estimation of statistical quantities, particularly rare event probabilities, for stochastic reaction networks. consequently, we propose an importance sampling (is) approach to improve the monte carlo (mc) estimator efficiency based on an approximate tau-leap scheme. the crucial step in the is framework is choosing an appropriate change of probability measure to achieve substantial variance reduction. this task is typically challenging and often requires insights into the underlying problem. therefore, we propose an automated approach to obtain a highly efficient path-dependent measure change based on an original connection in the stochastic reaction network context between finding optimal is parameters within a class of probability measures and a stochastic optimal control formulation. optimal is parameters are obtained by solving a variance minimization problem. first, we derive an associated dynamic programming equation. analytically solving this backward equation is challenging, hence we propose an approximate dynamic programming formulation to find near-optimal control parameters. to mitigate the curse of dimensionality, we propose a learning-based method to approximate the value function using a neural network, where the parameters are determined via a stochastic optimization algorithm. our analysis and numerical experiments verify that the proposed learning-based is approach substantially reduces mc estimator variance, resulting in a lower computational complexity in the rare event regime, compared with standard tau-leap mc estimators.',\n",
       "  'categories': 'math.na cs.na math.oc q-bio.qm stat.co',\n",
       "  'doi': '10.1007/s11222-023-10222-6',\n",
       "  'created': '2021-10-27',\n",
       "  'updated': '2023-02-13',\n",
       "  'authors': ['chiheb ben hammouda',\n",
       "   'nadhir ben rached',\n",
       "   'raúl tempone',\n",
       "   'sophia wiechert'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2110.14335'},\n",
       " {'title': 'bioremediation of contaminated soil with crude oil using consortium of   bacteria',\n",
       "  'id': '2111.00594',\n",
       "  'abstract': \"this study included isolate petroleum hydrocarbons degradable bacteria and develops a consortium or a mixture of bacteria with high biodegradation capabilities which can be used in biological treatment units of the contaminated soils before release. in this studyt ten bacterial strains were isolated from soils contaminated with crude oil, by primary and secondary screening, and by using the sterile saline solution with 1% of crude oil, five bacterial isolates that can degrade oil were identified. the bacterial isolates were isolated from polluted soils with crude oil, the samples were collected from different areas of the baiji refinery, and samples of heavy crude oil extracted from qayyarah fields were used in designing the biodegradation experiments, the five isolates were diagnosed based on phenotypic, culture and biochemical characterizes, and detection of the gene sequence of bacterial isolates (16srrna). the (16srrna) gene sequence of the bacterial isolates was recorded under the accession numbers lc596402, lc596403, lc596406, lc596404, lc596405, for the genus (am-i-1, am-i-2, am-i-5) and the species (am-i-3, am-i-4) that following bacillus sp respectively, in the ncbi's genbank, the efficiency of the five isolates were examined for utilizing petroleum hydrocarbons using a sterile mineral salt medium ms supplemented with crude oil as the sole source of carbon with different concentrations, the results showed that the decomposition of petroleum hydrocarbons at the concentrations (0.5, 1, 1.5, 2, 3) % reached (57.32, 69.36,63.71, 75.20, 68.60) %, respectively, for mixed isolates, depending on the results of the gas chromatography (gc) analysis.\",\n",
       "  'categories': 'q-bio.bm',\n",
       "  'doi': '10.1063/5.0094117',\n",
       "  'created': '2021-10-31',\n",
       "  'updated': '',\n",
       "  'authors': ['ahmed mohamed taher', 'ibrahim omar saeed'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.00594'},\n",
       " {'title': 'tuning the weights: the impact of initial matrix configurations on   successor features learning efficacy',\n",
       "  'id': '2111.02017',\n",
       "  'abstract': 'the focus of this study is to investigate the impact of different initialization strategies for the weight matrix of successor features (sf) on learning efficiency and convergence in reinforcement learning (rl) agents. using a grid-world paradigm, we compare the performance of rl agents, whose sf weight matrix is initialized with either an identity matrix, zero matrix, or a randomly generated matrix (using xavier, he, or uniform distribution method). our analysis revolves around evaluating metrics such as value error, step length, pca of successor representation (sr) place field, and the distance of sr matrices between different agents. the results demonstrate that rl agents initialized with random matrices reach the optimal sr place field faster and showcase a quicker reduction in value error, pointing to more efficient learning. furthermore, these random agents also exhibit a faster decrease in step length across larger grid-world environments. the study provides insights into the neurobiological interpretations of these results, their implications for understanding intelligence, and potential future research directions. these findings could have profound implications for the field of artificial intelligence, particularly in the design of learning algorithms.',\n",
       "  'categories': 'q-bio.nc cs.ne',\n",
       "  'doi': '10.3390/electronics12204212',\n",
       "  'created': '2021-11-03',\n",
       "  'updated': '2023-09-12',\n",
       "  'authors': ['hyunsu lee'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.02017'},\n",
       " {'title': 'adaptive observers for biophysical neuronal circuits',\n",
       "  'id': '2111.02176',\n",
       "  'abstract': 'this paper presents adaptive observers for online state and parameter estimation of a class of nonlinear systems motivated by biophysical models of neuronal circuits. we first present a linear-in-the-parameters design that solves a classical recursive least squares problem. then, building on this simple design, we present an augmented adaptive observer for models with a nonlinearly parameterized internal dynamics, the parameters of which we interpret as structured uncertainty. we present a convergence and robustness analysis based on contraction theory, and illustrate the potential of the approach in neurophysiological applications by means of numerical simulations.',\n",
       "  'categories': 'eess.sy cs.sy q-bio.nc',\n",
       "  'doi': '10.1109/tac.2023.3344723',\n",
       "  'created': '2021-11-03',\n",
       "  'updated': '2023-06-28',\n",
       "  'authors': ['thiago b. burghi', 'rodolphe sepulchre'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.02176'},\n",
       " {'title': 'gabor frames from contact geometry in models of the primary visual   cortex',\n",
       "  'id': '2111.02307',\n",
       "  'abstract': 'we analyze the interplay between contact geometry and gabor filters signal analysis in geometric models of the primary visual cortex. we show in particular that a specific framed lattice and an associated gabor system is determined by the legendrian circle bundle structure of the $3$-manifold of contact elements on a surface (which models the v1-cortex), together with the presence of an almost-complex structure on the tangent bundle of the surface (which models the retinal surface). we identify a scaling of the lattice, also dictated by the manifold geometry, that ensures the frame condition is satisfied. we then consider a $5$-dimensional model where receptor profiles also involve a dependence on frequency and scale variables, in addition to the dependence on position and orientation. in this case we show that a proposed profile window function does not give rise to frames (even in a distributional sense), while a natural modification of the same generates gabor frames with respect to the appropriate lattice determined by the contact geometry.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.46298/mna.9766',\n",
       "  'created': '2021-11-03',\n",
       "  'updated': '2023-06-02',\n",
       "  'authors': ['vasiliki liontou', 'matilde marcolli'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.02307'},\n",
       " {'title': 'simulation of the behavior of the leopardus guigna using random walkers',\n",
       "  'id': '2111.03445',\n",
       "  'abstract': 'considering that there is very little information on the behavior habits of guigna cats, as well as investigations in which small populations are captured to place radiocollars on them and then release them in the place where they were captured, which is done with the intention of collecting data on their positions in a territory and thus make estimates of the mean distances they usually travel. under the hypothesis that guignas maintain a sedentary behavior in a specific area of a given territory, this paper shows one way to simulate a distribution of points in a territory using random walkers to emulate the distribution of the data that would be obtained by placing radiocollars in a population of guignas, with which it is possible to make estimates of the mean distances that move away from a certain fixed position, and the interactions they can have with points in the territory that represent a high probability of lethality, such as farms, packs of dogs, roads, urban areas, etc. it is necessary to mention that by estimating the possible interactions that a guignas population may have with possible predators in a territory with the help of a satellite image, it is possible to evaluate the points of a territory that represent a potentially lethal risk for the guignas, and thus generate relocation strategies that help preserve them.',\n",
       "  'categories': 'physics.soc-ph physics.bio-ph q-bio.ot',\n",
       "  'doi': '10.5121/mathsj.2021.8401',\n",
       "  'created': '2021-11-05',\n",
       "  'updated': '',\n",
       "  'authors': ['a. torres-hernandez',\n",
       "   'byron c. guzmán',\n",
       "   'melanie kaiser',\n",
       "   'julio c. hernández'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.03445'},\n",
       " {'title': 'emergence of collective behaviors from local voronoi topological   perception',\n",
       "  'id': '2111.03448',\n",
       "  'abstract': \"this article addresses how diverse collective behaviors arise from simple and realistic decisions made entirely at the level of each agent's personal space in the sense of the voronoi diagram. we present a discrete time model in 2d in which individual agents are aware of their local voronoi environment and may seek static target locations. in particular, agents only communicate directly with their voronoi neighbors and make decisions based on the geometry of their own voronoi cells. with two effective control parameters, it is shown numerically to capture a wide range of collective behaviors in different scenarios. further, we show that the voronoi topology facilitates the computation of several novel observables for quantifying discrete collective behaviors. these observables are applicable to all agent-based models and to empirical data.\",\n",
       "  'categories': 'physics.soc-ph math.ds q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2021-10-29',\n",
       "  'updated': '2024-02-13',\n",
       "  'authors': ['ivan gonzalez',\n",
       "   'jack tisdell',\n",
       "   'rustum choksi',\n",
       "   'jean-christophe nave'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.03448'},\n",
       " {'title': 'treatise on hearing: the temporal auditory imaging theory inspired by   optics and communication',\n",
       "  'id': '2111.04338',\n",
       "  'abstract': 'a new theory of mammalian hearing is presented, which accounts for the auditory image in the midbrain (inferior colliculus) of objects in the acoustical environment of the listener. it is shown that the ear is a temporal imaging system that comprises three transformations of the envelope functions: cochlear group-delay dispersion, cochlear time lensing, and neural group-delay dispersion. these elements are analogous to the optical transformations in vision of diffraction between the object and the eye, spatial lensing by the lens, and second diffraction between the lens and the retina. unlike the eye, it is established that the human auditory system is naturally defocused, so that coherent stimuli do not react to the defocus, whereas completely incoherent stimuli are impacted by it and may be blurred by design. it is argued that the auditory system can use this differential focusing to enhance or degrade the images of real-world acoustical objects that are partially coherent. the theory is founded on coherence and temporal imaging theories that were adopted from optics. in addition to the imaging transformations, the corresponding inverse-domain modulation transfer functions are derived and interpreted with consideration to the nonuniform neural sampling operation of the auditory nerve. these ideas are used to rigorously initiate the concepts of sharpness and blur in auditory imaging, auditory aberrations, and auditory depth of field. in parallel, ideas from communication theory are used to show that the organ of corti functions as a multichannel phase-locked loop (pll) that constitutes the point of entry for auditory phase locking and hence conserves the signal coherence. it provides an anchor for a dual coherent and noncoherent auditory detection in the auditory brain that culminates in auditory accommodation. implications on hearing impairments are discussed as well.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2021-11-08',\n",
       "  'updated': '2024-06-01',\n",
       "  'authors': ['adam weisser'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.04338'},\n",
       " {'title': 'the central role of metabolism in vascular morphogenesis',\n",
       "  'id': '2111.04657',\n",
       "  'abstract': 'as nutrients travel through microcirculation and are absorbed, their availability continuously decreases. however, a uniform nutrient distribution is critical, as it prevents tissue death in poorly supplied areas. how, then, do vascular networks achieve equi-perfusion? given the extensive number of vessels in animal vascular systems, the structure of smaller vessels cannot be fully genetically predetermined and thus relies on a self-organizing developmental mechanism. we propose a simple, optimization-based adaptation rule to control vessel radii, aiming to equalize perfusion while minimizing flow resistance and material cost. this adaptation balances three competing factors: perfusion efficiency, energy dissipation, and material expenditure, which together drive complex network morphologies. these morphologies range from hierarchical architectures optimized for minimal resistance and material cost to dense networks that achieve efficient perfusion. notably, metabolic demand is the primary factor modulating the transition between these contrasting structures. by applying our adaptation rule to networks in the rat mesentery and comparing the resulting optimal perfusion architectures with the experimental data, we can estimate the biologically relevant parameters, such as the oxygen absorption rate, for which the optimization and experimental networks align the most. surprisingly, the optimal absorption rate we derive matches independent in vivo measurements. this suggests that reduced-order models like ours can provide valuable insights into the development and function of real vascular networks.',\n",
       "  'categories': 'q-bio.to nlin.ao physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2021-11-08',\n",
       "  'updated': '2024-11-15',\n",
       "  'authors': ['georgios gounaris',\n",
       "   'mija jovchevska',\n",
       "   'miguel ruiz garcia',\n",
       "   'eleni katifori'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.04657'},\n",
       " {'title': 'heterogeneous popularity of metabolic reactions from evolution',\n",
       "  'id': '2111.04943',\n",
       "  'abstract': \"the composition of cellular metabolism is different across species. empirical data reveal that bacterial species contain similar numbers of metabolic reactions but that the cross-species popularity of reactions is so heterogenous that some reactions are found in all the species while others are in just few species, characterized by a power-law distribution with the exponent one. introducing an evolutionary model concretizing the stochastic recruitment of chemical reactions into the metabolism of different species at different times and their inheritance to descendants, we demonstrate that the exponential growth of the number of species containing a reaction and the saturated recruitment rate of brand-new reactions lead to the empirically identified power-law popularity distribution. furthermore, the structural characteristics of metabolic networks and the species' phylogeny in our simulations agree well with empirical observations.\",\n",
       "  'categories': 'q-bio.pe physics.bio-ph',\n",
       "  'doi': '10.1103/physrevlett.132.018401',\n",
       "  'created': '2021-11-08',\n",
       "  'updated': '2024-01-05',\n",
       "  'authors': ['mi jin lee', 'deok-sun lee'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.04943'},\n",
       " {'title': 'a rule-based epidemiological modelling framework',\n",
       "  'id': '2111.07336',\n",
       "  'abstract': \"motivated by chemical reaction rules, we introduce a rule-based epidemiological framework for the systematic mathematical modelling of future pandemics. here we stress that we do not have a specific model in mind, but a whole collection of models which can be transformed into each other, or represent different aspects of a pandemic, and these aspects can change during the course of the emergency, as happened during the covid-19 pandemic. as conditions for outbreaks in the modern world change on different time-scales, some rapidly, epidemiology has few 'laws', besides perhaps the fundamental infection process described by kermack-mckendrick. each single of our variety of models, called framework, is based on a mathematical formulation that we call a rule-based system. they have several advantages, for example that they can be both interpreted stochastically and deterministically, without changing the model structure. rule-based systems should be easier to communicate to non-specialists, when compared to differential equations. due to their combinatorial nature, the rule-based model framework we propose is ideal for systematic mathematical modelling, systematic links to statistics, data analysis in general and also machine learning leading to artificial intelligence.\",\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2021-11-14',\n",
       "  'updated': '2024-05-22',\n",
       "  'authors': ['david alonso',\n",
       "   'steffen bauer',\n",
       "   'markus kirkilionis',\n",
       "   'lisa maria kreusser',\n",
       "   'luca sbano'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.07336'},\n",
       " {'title': 'on minimizers and convolutional filters: theoretical connections and   applications to genome analysis',\n",
       "  'id': '2111.08452',\n",
       "  'abstract': 'minimizers and convolutional neural networks (cnns) are two quite distinct popular techniques that have both been employed to analyze categorical biological sequences. at face value, the methods seem entirely dissimilar. minimizers use min-wise hashing on a rolling window to extract a single important k-mer feature per window. cnns start with a wide array of randomly initialized convolutional filters, paired with a pooling operation, and then multiple additional neural layers to learn both the filters themselves and how they can be used to classify the sequence.   here, our main result is a careful mathematical analysis of hash function properties showing that for sequences over a categorical alphabet, random gaussian initialization of convolutional filters with max-pooling is equivalent to choosing a minimizer ordering such that selected k-mers are (in hamming distance) far from the k-mers within the sequence but close to other minimizers. in empirical experiments, we find that this property manifests as decreased density in repetitive regions, both in simulation and on real human telomeres. we additionally train from scratch a cnn embedding of synthetic short-reads from the sars-cov-2 genome into 3d euclidean space that locally recapitulates the linear sequence distance of the read origins, a modest step towards building a deep learning assembler, though it is at present too slow to be practical. in total, this manuscript provides a partial explanation for the effectiveness of cnns in categorical sequence analysis.',\n",
       "  'categories': 'cs.lg cs.ai q-bio.gn',\n",
       "  'doi': '',\n",
       "  'created': '2021-11-09',\n",
       "  'updated': '2024-01-26',\n",
       "  'authors': ['yun william yu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.08452'},\n",
       " {'title': 'the \"bayesian\" brain, with a bit less bayes',\n",
       "  'id': '2111.09063',\n",
       "  'abstract': 'the idea that the brain is a probabilistic (bayesian) inference machine, continuously trying to figure out the hidden causes of its inputs, has become very influential in cognitive (neuro)science over recent decades. here i present a relatively straightforward generalization of this idea: the primary computational task that the brain is faced with is to track the probabilistic structure of observations themselves, without recourse to hidden states. taking this starting point seriously turns out to have considerable explanatory power, and several key ideas are developed from it: (1) past experience, encoded in prior expectations, has an influence over the future that is analogous to regularization as known from machine learning; (2) action generation (interpreted as constraint satisfaction) is a special case of such regularization; (3) the concept of attractors in dynamical systems provides a useful lens through which prior expectations, regularization, and action induction can be viewed; these thus appear as different perspectives on the same phenomenon; (4) the phylogenetically ancient imperative of acting to ensure and thereby observe conditions beneficial for survival is likely the same as that which underlies perceptual inference. the bayesian brain hypothesis has been touted as promising to deliver a \"unified science of mind and action\". in this paper, i sketch an informal step towards fulfilling that promise, while avoiding some pitfalls that other such attempts have fallen prey to.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2021-11-17',\n",
       "  'updated': '2024-02-14',\n",
       "  'authors': ['eelke spaak'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.09063'},\n",
       " {'title': 'a neural network model of the entorhinal cortex and hippocampus for   event-order memory processing',\n",
       "  'id': '2111.10535',\n",
       "  'abstract': 'to solve a navigation task based on experiences, we need a mechanism to associate places with objects and to recall them along the course of action. in a reward-oriented task, if the route to a reward location is simulated in mind after experiencing it once, it might be possible that the reward is gained efficiently. one way to solve it is to incorporate a biologically plausible mechanism. in this study, we propose a neural network that stores a sequence of events associated with a reward. the proposed network recalls the reward location by tracing them in its mind in order. we simulated a virtual mouse that explores a figure-eight maze and recalls the route to the reward location. during the learning period, a sequence of events relating to firing along a passage was temporarily stored in the heteroassociative network, and the sequence of events is consolidated in the synaptic weight matrix when a reward is fed. for retrieval, the sequential activation of conjunctive cue-place cells toward the reward location is internally generated by an impetus input. in the figure-eight maze task, the location of the reward was estimated by mind travel, irrespective of whether the reward is in the counterclockwise or distant clockwise route. the mechanism of efficiently reaching the goal by mind travel in the brain based on experiences is beneficial for mobile service robots that perform autonomous navigation.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2021-11-20',\n",
       "  'updated': '',\n",
       "  'authors': ['hiroki nakagawa',\n",
       "   'katsumi tateno',\n",
       "   'kensuke takada',\n",
       "   'takashi morie'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.10535'},\n",
       " {'title': 'mathematical representation of the structure of neuron-glia networks',\n",
       "  'id': '2111.10719',\n",
       "  'abstract': 'network representations of the nervous system have been useful for the understanding of brain phenomena such as perception, motor coordination, and memory. although brains are composed of both neurons and glial cells, neuron-glial networks have been little studied so far. given the emergent role of glial cells in information transmission in the brain, we developed a mathematical representation for neuron-glial networks ($\\\\upsilon$-graph). we also defined the concepts of isomorphisms, unnested form (multidigraph) and matrix equation for $\\\\upsilon$-graphs. although we found several network motives where the isomorphism between unnested forms does not guarantees the isomorphism between their respective $\\\\upsilon$-graphs, we found that if the matrix equations satisfy some conditions, the unnested forms isomorphism guarantees the isomorphism between $\\\\upsilon$-graphs. finally, we introduced a novel approach to modeling the network shape. our work presents a mathematical framework for working with neuron-glia networks.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2021-11-20',\n",
       "  'updated': '2024-11-12',\n",
       "  'authors': ['marco peña-garcia',\n",
       "   'francesco peña-garcia',\n",
       "   'nelson castro',\n",
       "   'walter cabrera-febola'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.10719'},\n",
       " {'title': 'operating principles of peristaltic pumping through a dense array of   valves',\n",
       "  'id': '2111.11413',\n",
       "  'abstract': 'immersed nonlinear elements are prevalent in biological systems that require a preferential flow direction, such as the venous and the lymphatic system. we investigate here a certain class of models where the fluid is driven by peristaltic pumping and the nonlinear elements are ideal valves that completely suppress backflow. this highly nonlinear system produces discontinuous solutions that are difficult to study. we show that as the density of valves increases, the pressure and flow are well-approximated by a continuum of valves which can be analytically treated, and we demonstrate through numeric simulation that the approximation works well even for intermediate valve densities. we find that the induced flow is linear in the peristaltic amplitude for small peristaltic forces and, in the case of sinusoidal peristalsis, is independent of pumping direction. despite the continuum approximation used, the physical valve density is accounted for by modifying the resistance of the fluid appropriately. the suppression of backflow causes a net benefit in adding valves when the valve density is low, but once the density is high enough, valves predominately suppress forward flow, suggesting there is an optimum number of valves per wavelength. the continuum model for peristaltic pumping through an array of valves presented in this work can eventually provide insights about the design and operating principles of complex flow networks with a broad class of nonlinear elements.',\n",
       "  'categories': 'physics.flu-dyn physics.bio-ph q-bio.to',\n",
       "  'doi': '10.1017/jfm.2024.480',\n",
       "  'created': '2021-11-22',\n",
       "  'updated': '2023-11-17',\n",
       "  'authors': ['aaron winn', 'eleni katifori'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.11413'},\n",
       " {'title': 'multi-modality microscopy image style transfer for nuclei segmentation',\n",
       "  'id': '2111.12138',\n",
       "  'abstract': 'annotating microscopy images for nuclei segmentation is laborious and time-consuming. to leverage the few existing annotations, also across multiple modalities, we propose a novel microscopy-style augmentation technique based on a generative adversarial network (gan). unlike other style transfer methods, it can not only deal with different cell assay types and lighting conditions, but also with different imaging modalities, such as bright-field and fluorescence microscopy. using disentangled representations for content and style, we can preserve the structure of the original image while altering its style during augmentation. we evaluate our data augmentation on the 2018 data science bowl dataset consisting of various cell assays, lighting conditions, and imaging modalities. with our style augmentation, the segmentation accuracy of the two top-ranked mask r-cnn-based nuclei segmentation algorithms in the competition increases significantly. thus, our augmentation technique renders the downstream task more robust to the test data heterogeneity and helps counteract class imbalance without resampling of minority classes.',\n",
       "  'categories': 'eess.iv cs.cv q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2021-11-23',\n",
       "  'updated': '',\n",
       "  'authors': ['ye liu', 'sophia j. wagner', 'tingying peng'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.12138'},\n",
       " {'title': 'untangling the role of temporal and spatial variations in persistance of   populations',\n",
       "  'id': '2111.12633',\n",
       "  'abstract': 'we consider a population distributed between two habitats, in each of which it experiences a growth rate that switches periodically between two values, $1- \\\\varepsilon > 0$ or $ - (1 + \\\\varepsilon) < 0$. we study the specific case where the growth rate is positive in one habitat and negative in the other one for the first half of the period, and conversely for the second half of the period, that we refer as the $(\\\\pm 1)$ model. in the absence of migration, the population goes to $0$ exponentially fast in each environment. in this paper, we show that, when the period is sufficiently large, a small dispersal between the two patches is able to produce a very high positive exponential growth rate for the whole population, a phenomena called inflation. we prove in particular that the threshold of the dispersal rate at which the inflation appears is exponentially small with the period. we show that inflation is robust to random perturbation, by considering a model where the values of the growth rate in each patch are switched at random times: we prove, using theory of piecewise deterministic markov processes (pdmp) that inflation occurs for low switching rate and small dispersal. finally, we provide some extensions to more complicated models, especially epidemiological and density dependent models.',\n",
       "  'categories': 'math.ds math.pr q-bio.pe',\n",
       "  'doi': '10.1016/j.tpb.2023.07.003',\n",
       "  'created': '2021-11-24',\n",
       "  'updated': '2023-07-03',\n",
       "  'authors': ['michel benaïm',\n",
       "   'claude lobry',\n",
       "   'tewfik sari',\n",
       "   'edouard strickler'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.12633'},\n",
       " {'title': 'multi-modality fusion using canonical correlation analysis methods:   application in breast cancer survival prediction from histology and genomics',\n",
       "  'id': '2111.13987',\n",
       "  'abstract': 'the availability of multi-modality datasets provides a unique opportunity to characterize the same object of interest using multiple viewpoints more comprehensively. in this work, we investigate the use of canonical correlation analysis (cca) and penalized variants of cca (pcca) for the fusion of two modalities. we study a simple graphical model for the generation of two-modality data. we analytically show that, with known model parameters, posterior mean estimators that jointly use both modalities outperform arbitrary linear mixing of single modality posterior estimators in latent variable prediction. penalized extensions of cca (pcca) that incorporate domain knowledge can discover correlations with high-dimensional, low-sample data, whereas traditional cca is inapplicable. to facilitate the generation of multi-dimensional embeddings with pcca, we propose two matrix deflation schemes that enforce desirable properties exhibited by cca. we propose a two-stage prediction pipeline using pcca embeddings generated with deflation for latent variable prediction by combining all the above. on simulated data, our proposed model drastically reduces the mean-squared error in latent variable prediction. when applied to publicly available histopathology data and rna-sequencing data from the cancer genome atlas (tcga) breast cancer patients, our model can outperform principal components analysis (pca) embeddings of the same dimension in survival prediction.',\n",
       "  'categories': 'cs.lg eess.sp q-bio.qm stat.ap',\n",
       "  'doi': '',\n",
       "  'created': '2021-11-27',\n",
       "  'updated': '',\n",
       "  'authors': ['vaishnavi subramanian', 'tanveer syeda-mahmood', 'minh n. do'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.13987'},\n",
       " {'title': 'optimal and $h_\\\\infty$ control of stochastic reaction networks',\n",
       "  'id': '2111.14754',\n",
       "  'abstract': 'stochastic reaction networks is a powerful class of models for the representation a wide variety of population models including biochemistry. the control of such networks has been recently considered due to their important implications for the control of biological systems. their optimal control, however, has been relatively few studied until now. the continuous-time finite-horizon optimal control problem is formulated first and explicitly solved in the case of unimolecular reaction networks. the problems of the optimal sampled-data control, the continuous $h_\\\\infty$ control, and the sampled-data $h_\\\\infty$ control of such networks are addressed next. the results in the unimolecular case take the form of nonstandard riccati differential equations or differential lyapunov equations coupled with difference riccati equations, which can all be solved numerically by backward-in-time integration.',\n",
       "  'categories': 'math.oc cs.sy eess.sy q-bio.mn',\n",
       "  'doi': '',\n",
       "  'created': '2021-11-29',\n",
       "  'updated': '2024-09-20',\n",
       "  'authors': ['corentin briat', 'mustafa khammash'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2111.14754'},\n",
       " {'title': 'thermodynamics of darwinian evolution in molecular replicators',\n",
       "  'id': '2112.02809',\n",
       "  'abstract': 'we consider the relationship between thermodynamics, fitness, and darwinian evolution in autocatalytic molecular replicators. we uncover a thermodynamic bound that relates fitness, replication rate, and the gibbs free energy dissipated per copy. this bound applies to a broad range of systems, including elementary and non-elementary autocatalytic reactions, polymer-based replicators, and certain kinds of autocatalytic sets. in addition, we show that the critical selection coefficient (the minimal fitness difference visible to selection) is bounded by the gibbs free energy dissipated per replication. our results imply fundamental thermodynamic bounds on the strength of selection in molecular evolution, complementary to other bounds that arise from finite population sizes and error thresholds. these bounds may be relevant for understanding thermodynamic constraints faced by early replicators at the origin of life. we illustrate our approach on several examples, including a classic model of replicators in a chemostat.',\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2021-12-06',\n",
       "  'updated': '2024-02-01',\n",
       "  'authors': ['artemy kolchinsky'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2112.02809'},\n",
       " {'title': 'a mems-based optical scanning system for precise, high-speed neural   interfacing',\n",
       "  'id': '2112.04909',\n",
       "  'abstract': 'optical scanning is a prevalent technique for optical neural interfaces where light delivery with high spatial and temporal precision is desired. however, due to the sequential nature of point-scanning techniques, the settling time of optical modulators is a major bottleneck for throughput and limits random-access targeting capabilities. while fast lateral scanners exist, commercially available varifocal elements are constrained to >3ms settling times, limiting the speed of the overall system to hundreds of hz. faster focusing methods exist but cannot combine fast operation and dwelling capability with electrical and optical efficiency. here, we present a varifocal mirror comprised of an array of piston-motion mems micromirrors and a custom driver asic, offering fast operation with dwelling capability while maintaining high diffraction efficiency. the asic features a reconfigurable nonlinear dac to simultaneously compensate for the built-in nonlinearity of electrostatic actuators and the global process variations in mems mirrors. experimental results demonstrate a wide continuous sweeping range that spans 22 distinctly resolvable depth planes with refresh rates greater than 12 khz.',\n",
       "  'categories': 'physics.app-ph physics.optics q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2021-12-07',\n",
       "  'updated': '',\n",
       "  'authors': ['cem yalcin',\n",
       "   'nathan tessema ersaro',\n",
       "   'm. meraj ghanbari',\n",
       "   'george bocchetti',\n",
       "   'sina faraji alamouti',\n",
       "   'nick antipa',\n",
       "   'daniel lopez',\n",
       "   'nicolas c. pégard',\n",
       "   'laura waller',\n",
       "   'rikky muller'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2112.04909'},\n",
       " {'title': 'intraspecific predator interference promotes biodiversity in ecosystems',\n",
       "  'id': '2112.05098',\n",
       "  'abstract': 'explaining biodiversity is a fundamental issue in ecology. a long-standing puzzle lies in the paradox of the plankton: many species of plankton feeding on a limited variety of resources coexist, apparently flouting the competitive exclusion principle (cep), which holds that the number of predator (consumer) species cannot exceed that of the resources at a steady state. here, we present a mechanistic model and demonstrate that intraspecific interference among the consumers enables a plethora of consumer species to coexist at constant population densities with only one or a handful of resource species. this facilitated biodiversity is resistant to stochasticity, either with the stochastic simulation algorithm or individual-based modeling. our model naturally explains the classical experiments that invalidate the cep, quantitatively illustrates the universal s-shaped pattern of the rank-abundance curves across a wide range of ecological communities, and can be broadly used to resolve the mystery of biodiversity in many natural ecosystems.',\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech physics.bio-ph',\n",
       "  'doi': '10.7554/elife.93115.1',\n",
       "  'created': '2021-12-09',\n",
       "  'updated': '2024-04-30',\n",
       "  'authors': ['ju kang',\n",
       "   'shijie zhang',\n",
       "   'yiyuan niu',\n",
       "   'fan zhong',\n",
       "   'xin wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2112.05098'},\n",
       " {'title': 'computational bioacoustics with deep learning: a review and roadmap',\n",
       "  'id': '2112.06725',\n",
       "  'abstract': 'animal vocalisations and natural soundscapes are fascinating objects of study, and contain valuable evidence about animal behaviours, populations and ecosystems. they are studied in bioacoustics and ecoacoustics, with signal processing and analysis an important component. computational bioacoustics has accelerated in recent decades due to the growth of affordable digital sound recording devices, and to huge progress in informatics such as big data, signal processing and machine learning. methods are inherited from the wider field of deep learning, including speech and image processing. however, the tasks, demands and data characteristics are often different from those addressed in speech or music analysis. there remain unsolved problems, and tasks for which evidence is surely present in many acoustic signals, but not yet realised. in this paper i perform a review of the state of the art in deep learning for computational bioacoustics, aiming to clarify key concepts and identify and analyse knowledge gaps. based on this, i offer a subjective but principled roadmap for computational bioacoustics with deep learning: topics that the community should aim to address, in order to make the most of future developments in ai and informatics, and to use audio data in answering zoological and ecological questions.',\n",
       "  'categories': 'cs.sd eess.as q-bio.qm',\n",
       "  'doi': '10.7717/peerj.13152',\n",
       "  'created': '2021-12-13',\n",
       "  'updated': '',\n",
       "  'authors': ['dan stowell'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2112.06725'},\n",
       " {'title': 'relaxation to statistical equilibrium in stochastic michaelis-menten   kinetics',\n",
       "  'id': '2112.10362',\n",
       "  'abstract': \"the equilibration of enzyme and complex concentrations in deterministic michaelis-menten reaction networks underlies the hyperbolic dependence between the input (substrates) and output (products). this relationship was first obtained by michaelis and menten and then briggs and haldane in two asymptotic limits: `fast equilibrium' and `steady state'. in stochastic michaelis-menten networks, relevant to catalysis at single-molecule and mesoscopic concentrations, the classical analysis cannot be directly applied due to molecular discreteness and fluctuations. instead, as we show here, such networks require a more subtle asymptotic analysis based on the decomposition of the network into reversible and irreversible sub-networks and the exact solution of the chemical master equation (cme). the reversible and irreversible sub-networks reach detailed balance and stationarity, respectively, through a relaxation phase that we characterise in detail through several new statistical measures. since stochastic enzyme kinetics encompasses the single-molecule, mesoscopic and thermodynamic limits, our work provides a broader molecular viewpoint of the classical results, in much the same manner that statistical mechanics provides a broader understanding of thermodynamics.\",\n",
       "  'categories': 'q-bio.mn physics.bio-ph physics.chem-ph q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2021-12-20',\n",
       "  'updated': '2024-09-27',\n",
       "  'authors': ['subham pal', 'manmath panigrahy', 'r. adhikari', 'arti dua'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2112.10362'},\n",
       " {'title': 'mathematical modelling, selection and hierarchical inference to   determine the minimal dose in ifn$\\\\alpha$ therapy against myeloproliferative   neoplasms',\n",
       "  'id': '2112.10688',\n",
       "  'abstract': 'myeloproliferative neoplasms (mpn) are blood cancers that appear after acquiring a driver mutation in a hematopoietic stem cell. these hematological malignancies result in the overproduction of mature blood cells and, if not treated, induce a risk of cardiovascular events and thrombosis. pegylated ifn$\\\\alpha$ is commonly used to treat mpn, but no clear guidelines exist concerning the dose prescribed to patients. we applied a model selection procedure and ran a hierarchical bayesian inference method to decipher how dose variations impact the response to the therapy. we inferred that ifn$\\\\alpha$ acts on mutated stem cells by inducing their differentiation into progenitor cells; the higher the dose, the higher the effect. we found that the treatment can induce long-term remission when a sufficient (patient-dependent) dose is reached. we determined this minimal dose for individuals in a cohort of patients and estimated the most suitable starting dose to give to a new patient to increase the chances of being cured.',\n",
       "  'categories': 'q-bio.qm stat.ap stat.co',\n",
       "  'doi': '10.1093/imammb/dqae006',\n",
       "  'created': '2021-12-20',\n",
       "  'updated': '2024-09-14',\n",
       "  'authors': ['gurvan hermange',\n",
       "   'william vainchenker',\n",
       "   'isabelle plo',\n",
       "   'paul-henry cournède'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2112.10688'},\n",
       " {'title': 'curvature-driven transport of thin bingham fluid layers in airway   bifurcations',\n",
       "  'id': '2112.11084',\n",
       "  'abstract': 'the mucus on the bronchial wall forms a thin layer of non-newtonian fluid. one of the roles of mucus is to protect the lungs by capturing inhaled pollutants. it is transported by mucocilliary clearance toward the tracheo-pharyngeal bifurcation, where it is eliminated. due to the corrugation of its interface with air, the mucus layer is subject to surface tension forces that interact with its rheology. it is still not clear whether these forces can affect mucus displacement and, if they can, under what conditions and how this displacement can occur. in this work, we model the mucus as a thin bingham fluid layer located on the wall of idealized, multi-scaled airway bifurcations. we analyze the resulting physical system using lubrication theory and 3d simulations. the theoretical analysis allows us to characterize the nonlinear behavior of the system and determine the geometric conditions under which the bingham fluid can be moved by surface tension. 3d simulations are then used to quantify the effects in idealized airway bifurcations on a range of scales corresponding to those of bronchial bifurcations. our results suggest that surface tension effects can displace overly thick mucus layers in airway bifurcations, a typical situation in obstructive lung pathologies (asthma, bpco, cystic fobrosis, etc.). moreover, our results indicate that this movement can disrupt mucociliary clearance and the homogeneity of the layer thickness, thus increasing the risk of lung infection.',\n",
       "  'categories': 'physics.flu-dyn physics.bio-ph physics.comp-ph q-bio.to',\n",
       "  'doi': '',\n",
       "  'created': '2021-12-21',\n",
       "  'updated': '2024-06-10',\n",
       "  'authors': ['cyril karamaoun',\n",
       "   'haribalan kumar',\n",
       "   'médéric argentina',\n",
       "   'didier clamond',\n",
       "   'benjamin mauroy'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2112.11084'},\n",
       " {'title': 'a general multi-scale description of metastable adaptive motion across   fitness valleys',\n",
       "  'id': '2112.12675',\n",
       "  'abstract': 'we consider a stochastic individual-based model of adaptive dynamics on a finite trait graph $g=(v,e)$. the evolution is driven by a linear birth rate, a density dependent logistic death rate an the possibility of mutations along the (possibly directed) edges in $e$. we study the limit of small mutation rates for a simultaneously diverging population size. closing the gap between the works of bovier, coquille and smadi (2019) and coquille, kraut and smadi (2021), we give a precise description of transitions between evolutionary stable conditions (esc), where multiple mutations are needed to cross a valley in the fitness landscape. the system shows a metastable behaviour on several divergent time scales associated to a degree of stability. we develop the framework of a meta graph that is constituted of escs and possible metastable transitions between those. this allows for a concise description of the multi-scale jump chain arising from concatenating several jumps. finally, for each of the various time scale, we prove the convergence of the population process to a markov jump process visiting only escs of sufficiently high stability.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2021-12-23',\n",
       "  'updated': '2024-04-05',\n",
       "  'authors': ['manuel esser', 'anna kraut'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2112.12675'},\n",
       " {'title': 'sequential memory improves sample and memory efficiency in episodic   control',\n",
       "  'id': '2112.14734',\n",
       "  'abstract': 'state of the art deep reinforcement learning algorithms are sample inefficient due to the large number of episodes they require to achieve asymptotic performance. episodic reinforcement learning (erl) algorithms, inspired by the mammalian hippocampus, typically use extended memory systems to bootstrap learning from past events to overcome this sample-inefficiency problem. however, such memory augmentations are often used as mere buffers, from which isolated past experiences are drawn to learn from in an offline fashion (e.g., replay). here, we demonstrate that including a bias in the acquired memory content derived from the order of episodic sampling improves both the sample and memory efficiency of an episodic control algorithm. we test our sequential episodic control (sec) model in a foraging task to show that storing and using integrated episodes as event sequences leads to faster learning with fewer memory requirements as opposed to a standard erl benchmark, model-free episodic control, that buffers isolated events only. we also study the effect of memory constraints and forgetting on the sequential and non-sequential version of the sec algorithm. furthermore, we discuss how a hippocampal-like fast memory system could bootstrap slow cortical and subcortical learning subserving habit formation in the mammalian brain.',\n",
       "  'categories': 'cs.lg cs.ai cs.ro q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2021-12-29',\n",
       "  'updated': '2024-06-06',\n",
       "  'authors': ['ismael t. freire', 'adrián f. amil', 'paul f. m. j. verschure'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2112.14734'},\n",
       " {'title': 'self-consistent stochastic dynamics for finite-size networks of spiking   neurons',\n",
       "  'id': '2112.14867',\n",
       "  'abstract': 'despite the huge number of neurons composing a brain network, ongoing activity of local cell assemblies composing cortical columns is intrinsically stochastic. fluctuations in their instantaneous rate of spike firing $\\\\nu(t)$ scale with the size of the assembly and persist in isolated network, i.e., in absence of external source of noise. although deterministic chaos due to the quenched disorder of the synaptic couplings likely underlies this seemingly stochastic dynamics, an effective theory for the network dynamics of a finite ensemble of spiking neurons is lacking. here, we fill this gap by extending the so-called population density approach including an activity- and size-dependent stochastic source in the fokker-planck equation for the membrane potential density. the finite-size noise embedded in this stochastic partial derivative equation is analytically characterized leading to a self-consistent and non-perturbative description of $\\\\nu(t)$ valid for a wide class of spiking neuron networks. its power spectra of $\\\\nu(t)$ are found in excellent agreement with those from detailed simulations both in the linear regime and across a synchronization phase transition, when a size-dependent smearing of the critical dynamics emerges.',\n",
       "  'categories': 'q-bio.nc cond-mat.dis-nn',\n",
       "  'doi': '10.1103/physrevlett.130.097402',\n",
       "  'created': '2021-12-29',\n",
       "  'updated': '',\n",
       "  'authors': ['gianni v. vinci', 'roberto benzi', 'maurizio mattia'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2112.14867'},\n",
       " {'title': 'stochastic covid-19 model influenced by non-gaussian noise',\n",
       "  'id': '2112.14876',\n",
       "  'abstract': 'natural fluctuations have played a crucial role in affecting the dynamics of pervasive diseases such as the coronavirus. examining the effects of irregular unsettling disturbances on epidemic models is important for understanding these dynamics. in this study, we introduce a mathematical model for the sir (susceptible-infectious-recovered) dynamics of the coronavirus, incorporating perturbations in the contact rate through levy noise. the utilization of the levy process is essential for the protection and control of diseases. we delve into the dynamics of both the deterministic model and the global positive solution of the stochastic model, establishing their existence and uniqueness. additionally, we explore conditions for the termination and persistence of the infection. furthermore, we derive the basic reproduction number, a critical determinant of disease extinction or persistence. numerical results indicate that covid-19 dissipates from the population when the reproduction number is less than one in the presence of significant or minor noise. conversely, controlling epidemic diseases becomes challenging when the reproduction number exceeds one. to illustrate this phenomenon, we provide numerical simulations, offering insights into the dynamics of the disease and the efficacy of control measures',\n",
       "  'categories': 'math.ds physics.soc-ph q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2021-12-29',\n",
       "  'updated': '2024-04-02',\n",
       "  'authors': ['daniel tesfayb', 'almaz tesfay', 'james brannan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2112.14876'},\n",
       " {'title': 'persistent homological state-space estimation of functional human brain   networks at rest',\n",
       "  'id': '2201.00087',\n",
       "  'abstract': 'we introduce an innovative, data-driven topological data analysis (tda) technique for estimating the state spaces of dynamically changing functional human brain networks at rest. our method utilizes the wasserstein distance to measure topological differences, enabling the clustering of brain networks into distinct topological states. this technique outperforms the commonly used k-means clustering in identifying brain network state spaces by effectively incorporating the temporal dynamics of the data without the need for explicit model specification. we further investigate the genetic underpinnings of these topological features using a twin study design, examining the heritability of such state changes. our findings suggest that the topology of brain networks, particularly in their dynamic state changes, may hold significant hidden genetic information. matlab code for the method is available at https://github.com/laplcebeltrami/ph-stat.',\n",
       "  'categories': 'math.at cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2021-12-31',\n",
       "  'updated': '2024-04-16',\n",
       "  'authors': ['moo k. chung',\n",
       "   'shih-gu huang',\n",
       "   'ian c. carroll',\n",
       "   'vince d. calhoun',\n",
       "   'h. hill goldsmith'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2201.00087'},\n",
       " {'title': 'interconnection between density-regulation and stability in competitive   ecological network',\n",
       "  'id': '2201.03193',\n",
       "  'abstract': 'in natural ecosystems, species can be characterized by the nonlinear density-dependent self-regulation of their growth profile. species of many taxa show a substantial density-dependent reduction for low population size. nevertheless, many show the opposite trend; density regulation is minimal for small populations and increases significantly when the population size is near the carrying capacity. the theta-logistic growth equation can portray the intraspecific density regulation in the growth profile, theta being the density regulation parameter. in this study, we examine the role of these different growth profiles on the stability of a competitive ecological community with the help of a mathematical model of competitive species interactions. this manuscript deals with the random matrix theory to understand the stability of the classical theta-logistic models of competitive interactions. our results suggest that having more species with strong density dependence, which self-regulate at low densities, leads to more stable communities. with this, stability also depends on the complexity of the ecological network. species network connectance (link density) shows a consistent trend of increasing stability, whereas community size (species richness) shows a context-dependent effect. we also interpret our results from the aspect of two different life history strategies: r and k-selection. our results show that the stability of a competitive network increases with the fraction of r-selected species in the community. our result is robust, irrespective of different network architectures.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1016/j.tpb.2024.03.003',\n",
       "  'created': '2022-01-10',\n",
       "  'updated': '2024-08-24',\n",
       "  'authors': ['amit samadder',\n",
       "   'arnab chattopadhyay',\n",
       "   'anurag sau',\n",
       "   'sabyasachi bhattacharya'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2201.03193'},\n",
       " {'title': 'stochastic epidemic sir models with hidden states',\n",
       "  'id': '2201.06708',\n",
       "  'abstract': 'this paper focuses on and analyzes realistic sir models that take stochasticity into account. the proposed systems are applicable to most incidence rates that are used in the literature including the bilinear incidence rate, the beddington-deangelis incidence rate, and a holling type ii functional response. given that many diseases can lead to asymptomatic infections, we look at a system of stochastic differential equations that also includes a class of hidden state individuals, for which the infection status is unknown. we assume that the direct observation of the percentage of hidden state individuals that are infected, $\\\\alpha(t)$, is not given and only a noise-corrupted observation process is available. using the nonlinear filtering techniques in conjunction with an invasion type analysis (or analysis using lyapunov exponents from the dynamical system point of view), this paper proves that the long-term behavior of the disease is governed by a threshold $\\\\lambda\\\\in \\\\mathbb{r}$ that depends on the model parameters. it turns out that if $\\\\lambda<0$ the number $i(t)$ of infected individuals converges to zero exponentially fast, or the extinction happens. in contrast, if $\\\\lambda>0$, the infection is endemic and the system is permanent. we showcase our results by applying them in specific illuminating examples. numerical simulations are also given to illustrate our results.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '10.1016/j.nahs.2023.101368',\n",
       "  'created': '2022-01-17',\n",
       "  'updated': '',\n",
       "  'authors': ['nguyen du', 'alexandru hening', 'nhu nguyen', 'george yin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2201.06708'},\n",
       " {'title': 'natural selection and random matrix theory',\n",
       "  'id': '2201.09812',\n",
       "  'abstract': 'we will study the relationship between two well-known theories, genetic evolution and random matrix theory in the context of many-body systems. it is suggested that genetic evolution can be described by a random matrix theory with statistical distribution in which natural selection acts as a gross-witten-wadia phase transition.',\n",
       "  'categories': 'nlin.cd q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2022-01-24',\n",
       "  'updated': '2024-07-20',\n",
       "  'authors': ['mehdi ameri'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2201.09812'},\n",
       " {'title': 'deep discriminative to kernel density graph for in- and   out-of-distribution calibrated inference',\n",
       "  'id': '2201.13001',\n",
       "  'abstract': \"deep discriminative approaches like random forests and deep neural networks have recently found applications in many important real-world scenarios. however, deploying these learning algorithms in safety-critical applications raises concerns, particularly when it comes to ensuring confidence calibration for both in-distribution and out-of-distribution data points. many popular methods for in-distribution (id) calibration, such as isotonic and platt's sigmoidal regression, exhibit excellent id calibration performance. however, these methods are not calibrated for the entire feature space, leading to overconfidence in the case of out-of-distribution (ood) samples. on the other end of the spectrum, existing out-of-distribution (ood) calibration methods generally exhibit poor in-distribution (id) calibration. in this paper, we address id and ood calibration problems jointly. we leveraged the fact that deep models, including both random forests and deep-nets, learn internal representations which are unions of polytopes with affine activation functions to conceptualize them both as partitioning rules of the feature space. we replace the affine function in each polytope populated by the training data with a gaussian kernel. our experiments on both tabular and vision benchmarks show that the proposed approaches obtain well-calibrated posteriors while mostly preserving or improving the classification accuracy of the original algorithm for id region, and extrapolate beyond the training data to handle ood inputs appropriately.\",\n",
       "  'categories': 'cs.lg cs.ai cs.ds q-bio.nc stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2022-01-31',\n",
       "  'updated': '2024-06-07',\n",
       "  'authors': ['jayanta dey',\n",
       "   'haoyin xu',\n",
       "   'will levine',\n",
       "   'ashwin de silva',\n",
       "   'tyler m. tomita',\n",
       "   'ali geisa',\n",
       "   'tiffany chu',\n",
       "   'jacob desman',\n",
       "   'joshua t. vogelstein'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2201.13001'},\n",
       " {'title': 'orientation-aware graph neural networks for protein structure   representation learning',\n",
       "  'id': '2201.13299',\n",
       "  'abstract': 'by folding to particular 3d structures, proteins play a key role in living beings. to learn meaningful representation from a protein structure for downstream tasks, not only the global backbone topology but the local fine-grained orientational relations between amino acids should also be considered. in this work, we propose the orientation-aware graph neural networks (oagnns) to better sense the geometric characteristics in protein structure (e.g. inner-residue torsion angles, inter-residue orientations). extending a single weight from a scalar to a 3d vector, we construct a rich set of geometric-meaningful operations to process both the classical and so(3) representations of a given structure. to plug our designed perceptron unit into existing graph neural networks, we further introduce an equivariant message passing paradigm, showing superior versatility in maintaining so(3)-equivariance at the global scale. experiments have shown that our oagnns have a remarkable ability to sense geometric orientational features compared to classical networks. oagnns have also achieved state-of-the-art performance on various computational biology applications related to protein 3d structures.',\n",
       "  'categories': 'q-bio.bm cs.ai cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2022-01-28',\n",
       "  'updated': '2024-11-26',\n",
       "  'authors': ['jiahan li',\n",
       "   'shitong luo',\n",
       "   'congyue deng',\n",
       "   'chaoran cheng',\n",
       "   'jiaqi guan',\n",
       "   'leonidas guibas',\n",
       "   'jian peng',\n",
       "   'jianzhu ma'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2201.13299'},\n",
       " {'title': 'periodic traveling waves in an integro-difference equation with   non-monotonic growth and strong allee effect',\n",
       "  'id': '2202.00234',\n",
       "  'abstract': 'we derive sufficient conditions for the existence of a periodic traveling wave solution to an integro-difference equation with a piecewise constant growth function exhibiting a stable period2 cycle and strong allee effect. the mean traveling wave speed is shown to be the asymptotic spreading speed of solutions with compactly supported initial data under appropriate conditions. we then conduct case studies for the laplace kernel and uniform kernel.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2022-02-01',\n",
       "  'updated': '2024-09-06',\n",
       "  'authors': ['michael nestor', 'bingtuan li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2202.00234'},\n",
       " {'title': 'stable cooperation emerges in stochastic multiplicative growth',\n",
       "  'id': '2202.02787',\n",
       "  'abstract': 'understanding the evolutionary stability of cooperation is a central problem in biology, sociology, and economics. there exist only a few known mechanisms that guarantee the existence of cooperation and its robustness to cheating. here, we introduce a new mechanism for the emergence of cooperation in the presence of fluctuations. we consider agents whose wealth change stochastically in a multiplicative fashion. each agent can share part of her wealth as public good, which is equally distributed among all the agents. we show that, when agents operate with long time-horizons, cooperation produce an advantage at the individual level, as it effectively screens agents from the deleterious effect of environmental fluctuations.',\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech econ.th physics.bio-ph q-fin.gn',\n",
       "  'doi': '10.1103/physreve.108.l012401',\n",
       "  'created': '2022-02-06',\n",
       "  'updated': '',\n",
       "  'authors': ['lorenzo fant',\n",
       "   'onofrio mazzarisi',\n",
       "   'emanuele panizon',\n",
       "   'jacopo grilli'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2202.02787'},\n",
       " {'title': 'faster gr\\\\\"obner bases for lie derivatives of ode systems via monomial   orderings',\n",
       "  'id': '2202.06297',\n",
       "  'abstract': 'symbolic computation for systems of differential equations is often computationally expensive. many practical differential models have a form of polynomial or rational ode system with specified outputs. a basic symbolic approach to analyze these models is to compute and then symbolically process the polynomial system obtained by sufficiently many lie derivatives of the output functions with respect to the vector field given by the ode system.   in this paper, we present a method for speeding up gr\\\\\"obner basis computation for such a class of polynomial systems by using specific monomial ordering, including weights for the variables, coming from the structure of the ode model. we provide empirical results that show improvement across different symbolic computing frameworks and apply the method to speed up structural identifiability analysis of ode models.',\n",
       "  'categories': 'cs.sc cs.ms q-bio.qm',\n",
       "  'doi': '10.1145/3666000.3669695',\n",
       "  'created': '2022-02-13',\n",
       "  'updated': '2024-06-06',\n",
       "  'authors': ['mariya bessonov',\n",
       "   'ilia ilmer',\n",
       "   'tatiana konstantinova',\n",
       "   'alexey ovchinnikov',\n",
       "   'gleb pogudin',\n",
       "   'pedro soto'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2202.06297'},\n",
       " {'title': 'effect of expanding learning scope on the evolution of cooperation in   scale-free networks',\n",
       "  'id': '2202.07211',\n",
       "  'abstract': 'we study how expanding the scope for selecting game and learning (adaptation) partners affects the evolution of cooperation in public goods games on scale-free networks. we show the following three results. (i) expanding the scope for selecting game partners suppresses cooperation. (ii) expanding the scope for selecting learning partners promotes cooperation when cooperation evolution is difficult. (iii) when cooperation is more likely to evolve, slightly expanding the scope for selecting learning partners causes a significant drop in the cooperation rate, but expanding the scope further causes the cooperation rate to recover. although (i) is explained by the hub-centered mechanism, the well-known dynamic that promotes cooperation on scale-free networks, (ii) and (iii) are caused by a completely different mechanism that has heretofore been rarely mentioned.',\n",
       "  'categories': 'physics.soc-ph math.ds q-bio.pe',\n",
       "  'doi': '10.1038/s41598-023-37074-4',\n",
       "  'created': '2022-02-15',\n",
       "  'updated': '',\n",
       "  'authors': ['masaaki inaba', 'eizo akiyama'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2202.07211'},\n",
       " {'title': 'limits on the evolutionary rates of biological traits',\n",
       "  'id': '2202.07533',\n",
       "  'abstract': \"this paper focuses on the maximum speed at which biological evolution can occur. i derive inequalities that limit the rate of evolutionary processes driven by natural selection, mutations, or genetic drift. these \\\\emph{rate limits} link the variability in a population to evolutionary rates. in particular, high variances in the fitness of a population and of a quantitative trait allow for fast changes in the trait's average. in contrast, low variability makes a trait less susceptible to random changes due to genetic drift. the results in this article generalize fisher's fundamental theorem of natural selection to dynamics that allow for mutations and genetic drift, via trade-off relations that constrain the evolutionary rates of arbitrary traits. the rate limits can be used to probe questions in various evolutionary biology and ecology settings. they apply, for instance, to trait dynamics within or across species or to the evolution of bacteria strains. they apply to any quantitative trait, e.g., from species' weights to the lengths of dna strands.\",\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech physics.bio-ph q-bio.qm',\n",
       "  'doi': '10.1038/s41598-024-61872-z',\n",
       "  'created': '2022-02-15',\n",
       "  'updated': '2024-06-14',\n",
       "  'authors': ['luis pedro garcía-pintos'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2202.07533'},\n",
       " {'title': 'the functional architecture of the early vision and neurogeometric   models',\n",
       "  'id': '2202.10157',\n",
       "  'abstract': \"the initial sections of the paper give a concise presentation, specially designed for a mathematically oriented audience, of some of the most basic facts on the functional architecture of early vision. such information is usually scattered in a variety of papers and books, which are not easily accessible by non-specialists. our goal is thus to offer a handy and short introduction to this topics, which might be helpful for researchers willing to enter the area of the applications of modern differential geometry in studies on the visual systems, baptized neurogeometry by j. petitot. we then offer a survey of three of the most important neurogeometric models: petitot's contact model of the primary visual cortex, its extension to a. sarti, g. citti and j. petitot's symplectic model, and p. c. bressloff and j. d. cowan's spherical model of hypercolumns. we finally discuss the main points of the so-called ``conformal model'' for hypercolumns (a model that was briefly presented in [d. v. alekseevsky, conformal model of hypercolumns in v1 cortex and the moebius group, in ``geometric science of information'', pp. 65--72, springer, 2021] and given in detail in [d. v. alekseevsky and a. spiro, conformal models for hypercolumns in the primary visual cortex v1, arxiv 2024]), which can be considered as a synthesis of the symplectic and the spherical models.\",\n",
       "  'categories': 'q-bio.nc math.dg',\n",
       "  'doi': '',\n",
       "  'created': '2022-02-21',\n",
       "  'updated': '2024-10-26',\n",
       "  'authors': ['dmitri v. alekseevsky', 'andrea spiro'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2202.10157'},\n",
       " {'title': 'fault-tolerant neural networks from biological error correction codes',\n",
       "  'id': '2202.12887',\n",
       "  'abstract': 'it has been an open question in deep learning if fault-tolerant computation is possible: can arbitrarily reliable computation be achieved using only unreliable neurons? in the grid cells of the mammalian cortex, analog error correction codes have been observed to protect states against neural spiking noise, but their role in information processing is unclear. here, we use these biological error correction codes to develop a universal fault-tolerant neural network that achieves reliable computation if the faultiness of each neuron lies below a sharp threshold; remarkably, we find that noisy biological neurons fall below this threshold. the discovery of a phase transition from faulty to fault-tolerant neural computation suggests a mechanism for reliable computation in the cortex and opens a path towards understanding noisy analog systems relevant to artificial intelligence and neuromorphic computing.',\n",
       "  'categories': 'cs.lg cs.ne q-bio.nc stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2022-02-25',\n",
       "  'updated': '2024-02-09',\n",
       "  'authors': ['alexander zlokapa',\n",
       "   'andrew k. tan',\n",
       "   'john m. martyn',\n",
       "   'ila r. fiete',\n",
       "   'max tegmark',\n",
       "   'isaac l. chuang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2202.12887'},\n",
       " {'title': 'bayesian active learning for discrete latent variable models',\n",
       "  'id': '2202.13426',\n",
       "  'abstract': 'active learning seeks to reduce the amount of data required to fit the parameters of a model, thus forming an important class of techniques in modern machine learning. however, past work on active learning has largely overlooked latent variable models, which play a vital role in neuroscience, psychology, and a variety of other engineering and scientific disciplines. here we address this gap by proposing a novel framework for maximum-mutual-information input selection for discrete latent variable regression models. we first apply our method to a class of models known as \"mixtures of linear regressions\" (mlr). while it is well known that active learning confers no advantage for linear-gaussian regression models, we use fisher information to show analytically that active learning can nevertheless achieve large gains for mixtures of such models, and we validate this improvement using both simulations and real-world data. we then consider a powerful class of temporally structured latent variable models given by a hidden markov model (hmm) with generalized linear model (glm) observations, which has recently been used to identify discrete states from animal decision-making data. we show that our method substantially reduces the amount of data needed to fit glm-hmm, and outperforms a variety of approximate methods based on variational and amortized inference. infomax learning for latent variable models thus offers a powerful for characterizing temporally structured latent states, with a wide variety of applications in neuroscience and beyond.',\n",
       "  'categories': 'cs.lg q-bio.nc stat.ml',\n",
       "  'doi': '10.1162/neco_a_01646',\n",
       "  'created': '2022-02-27',\n",
       "  'updated': '2023-06-02',\n",
       "  'authors': ['aditi jha', 'zoe c. ashwood', 'jonathan w. pillow'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2202.13426'},\n",
       " {'title': 'precision-medicine-toolbox: an open-source python package for   facilitation of quantitative medical imaging and radiomics analysis',\n",
       "  'id': '2202.13965',\n",
       "  'abstract': 'medical image analysis plays a key role in precision medicine as it allows the clinicians to identify anatomical abnormalities and it is routinely used in clinical assessment. data curation and pre-processing of medical images are critical steps in the quantitative medical image analysis that can have a significant impact on the resulting model performance. in this paper, we introduce a precision-medicine-toolbox that allows researchers to perform data curation, image pre-processing and handcrafted radiomics extraction (via pyradiomics) and feature exploration tasks with python. with this open-source solution, we aim to address the data preparation and exploration problem, bridge the gap between the currently existing packages, and improve the reproducibility of quantitative medical imaging research.',\n",
       "  'categories': 'eess.iv cs.cv cs.lg q-bio.qm',\n",
       "  'doi': '10.1016/j.simpa.2023.100508',\n",
       "  'created': '2022-02-28',\n",
       "  'updated': '',\n",
       "  'authors': ['sergey primakov',\n",
       "   'elizaveta lavrova',\n",
       "   'zohaib salahuddin',\n",
       "   'henry c woodruff',\n",
       "   'philippe lambin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2202.13965'},\n",
       " {'title': 'multi-objective latent space optimization of generative molecular design   models',\n",
       "  'id': '2203.00526',\n",
       "  'abstract': 'molecular design based on generative models, such as variational autoencoders (vaes), has become increasingly popular in recent years due to its efficiency for exploring high-dimensional molecular space to identify molecules with desired properties. while the efficacy of the initial model strongly depends on the training data, the sampling efficiency of the model for suggesting novel molecules with enhanced properties can be further enhanced via latent space optimization. in this paper, we propose a multi-objective latent space optimization (lso) method that can significantly enhance the performance of generative molecular design (gmd). the proposed method adopts an iterative weighted retraining approach, where the respective weights of the molecules in the training data are determined by their pareto efficiency. we demonstrate that our multi-objective gmd lso method can significantly improve the performance of gmd for jointly optimizing multiple molecular properties.',\n",
       "  'categories': 'cs.lg q-bio.bm',\n",
       "  'doi': '10.1016/j.patter.2024.101042',\n",
       "  'created': '2022-03-01',\n",
       "  'updated': '2024-07-21',\n",
       "  'authors': ['a n m nafiz abeer',\n",
       "   'nathan urban',\n",
       "   'm ryan weil',\n",
       "   'francis j. alexander',\n",
       "   'byung-jun yoon'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2203.00526'},\n",
       " {'title': 'deepautopin: an automorphism orbits based deep neural network for   characterizing the organizational diversity of protein interactomes across   the tree of life',\n",
       "  'id': '2203.00999',\n",
       "  'abstract': \"the enormous diversity of life forms thriving in drastically different environmental milieus involves a complex interplay among constituent proteins interacting with each other. however, the organizational principles characterizing the evolution of protein interaction networks (pins) across the tree of life are largely unknown. here we study 4,738 pins belonging to 16 phyla to discover phyla-specific architectural features and examine if there are some evolutionary constraints imposed on the networks' topologies. we utilized positional information of a network's nodes by normalizing the frequencies of automorphism orbits appearing in graphlets of sizes 2-5. we report that orbit usage profiles (oups) of networks belonging to the three domains of life are contrastingly different not only at the domain level but also at the scale of phyla. integrating the information related to protein families, domains, subcellular location, gene ontology, and pathways, our results indicate that wiring patterns of pins in different phyla are not randomly generated rather they are shaped by evolutionary constraints imposed on them. there exist subtle but substantial variations in the wiring patterns of pins that enable oups to differentiate among different superfamilies. a deep neural network was trained on differentially expressed orbits resulting in a prediction accuracy of 85%.\",\n",
       "  'categories': 'q-bio.mn cs.ai q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2022-03-02',\n",
       "  'updated': '2024-01-29',\n",
       "  'authors': ['vikram singh', 'vikram singh'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2203.00999'},\n",
       " {'title': 'a mathematical study of the role of tbregs in breast cancer',\n",
       "  'id': '2203.01434',\n",
       "  'abstract': 'a model for the mathematical study of immune response to breast cancer is proposed and studied, both analytically and numerically. it is a simplification of a complex one, recently introduced by two of the present authors. it serves for a compact study of the dynamical role in cancer promotion of a relatively recently described subgroup of regulatory b cells, which are evoked by the tumour.',\n",
       "  'categories': 'q-bio.cb math.ds',\n",
       "  'doi': '10.1007/s11538-022-01054-y',\n",
       "  'created': '2022-03-02',\n",
       "  'updated': '2022-07-16',\n",
       "  'authors': ['vasiliki bitsouni', 'nikolaos gialelis', 'vasilis tsilidis'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2203.01434'},\n",
       " {'title': 'superanimal pretrained pose estimation models for behavioral analysis',\n",
       "  'id': '2203.07436',\n",
       "  'abstract': \"quantification of behavior is critical in applications ranging from neuroscience, veterinary medicine and animal conservation efforts. a common key step for behavioral analysis is first extracting relevant keypoints on animals, known as pose estimation. however, reliable inference of poses currently requires domain knowledge and manual labeling effort to build supervised models. we present a series of technical innovations that enable a new method, collectively called superanimal, to develop unified foundation models that can be used on over 45 species, without additional human labels. concretely, we introduce a method to unify the keypoint space across differently labeled datasets (via our generalized data converter) and for training these diverse datasets in a manner such that they don't catastrophically forget keypoints given the unbalanced inputs (via our keypoint gradient masking and memory replay approaches). these models show excellent performance across six pose benchmarks. then, to ensure maximal usability for end-users, we demonstrate how to fine-tune the models on differently labeled data and provide tooling for unsupervised video adaptation to boost performance and decrease jitter across frames. if the models are fine-tuned, we show superanimal models are 10-100$\\\\times$ more data efficient than prior transfer-learning-based approaches. we illustrate the utility of our models in behavioral classification in mice and gait analysis in horses. collectively, this presents a data-efficient solution for animal pose estimation.\",\n",
       "  'categories': 'cs.cv cs.ai q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-03-14',\n",
       "  'updated': '2023-12-30',\n",
       "  'authors': ['shaokai ye',\n",
       "   'anastasiia filippova',\n",
       "   'jessy lauer',\n",
       "   'steffen schneider',\n",
       "   'maxime vidal',\n",
       "   'tian qiu',\n",
       "   'alexander mathis',\n",
       "   'mackenzie weygandt mathis'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2203.07436'},\n",
       " {'title': 'labelled partitions in action: recombination, selection, mutation, and   more',\n",
       "  'id': '2203.10874',\n",
       "  'abstract': 'in this paper, we consider the evolution of an (infinitely large) population under recombination and additional evolutionary forces, modelled by a measure-valued ordinary differential equation. we provide a stochastic representation for the solution of this model via duality to a new labelled partitioning process with markovian labels. in the special case of single-crossover, this leads to a recursive solution formula. this extends (and unifies) previous results on the selection-recombination equation. as a concrete example, we consider the selection-mutation-recombination equation.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2022-03-21',\n",
       "  'updated': '2024-10-02',\n",
       "  'authors': ['frederic alberti'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2203.10874'},\n",
       " {'title': 'sparse inference and active learning of stochastic differential   equations from data',\n",
       "  'id': '2203.11010',\n",
       "  'abstract': 'automatic machine learning of empirical models from experimental data has recently become possible as a result of increased availability of computational power and dedicated algorithms. despite the successes of non-parametric inference and neural-network-based inference for empirical modelling, a physical interpretation of the results often remains challenging. here, we focus on direct inference of governing differential equations from data, which can be formulated as a linear inverse problem. a bayesian framework with a laplacian prior distribution is employed for finding sparse solutions efficiently. the superior accuracy and robustness of the method is demonstrated for various cases, including ordinary, partial, and stochastic differential equations. furthermore, we develop an active learning procedure for the automated discovery of stochastic differential equations. in this procedure, learning of the unknown dynamical equations is coupled to the application of perturbations to the measured system in a feedback loop. we demonstrate with simulations that the active learning procedure improves the inference of empirical, langevin-type descriptions of stochastic processes.',\n",
       "  'categories': 'cond-mat.stat-mech q-bio.qm',\n",
       "  'doi': '10.1038/s41598-022-25638-9',\n",
       "  'created': '2022-03-21',\n",
       "  'updated': '2022-12-16',\n",
       "  'authors': ['yunfei huang',\n",
       "   'youssef mabrouk',\n",
       "   'gerhard gompper',\n",
       "   'benedikt sabass'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2203.11010'},\n",
       " {'title': 'vitamin k content of cheese, yoghurt and meat products in australia',\n",
       "  'id': '2203.11402',\n",
       "  'abstract': 'vitamin k is vital for normal blood coagulation, and may influence bone, neurological and vascular health. data on the vitamin k content of australian foods are limited, preventing estimation of vitamin k intakes in the australian population. we measured phylloquinone (pk) and menaquinone (mk) -4 to -10 in cheese, yoghurt and meat products (48 composite samples from 288 primary samples) by liquid chromatography with electrospray ionisation-tandem mass spectrometry. at least one k vitamer was found in every sample. the greatest mean concentrations of pk, mk-4 and mk-9 were found in lamb liver, chicken leg meat and cheddar cheese, respectively. cheddar cheese and cream cheese contained mk-5. mk-8 was found in cheddar cheese only. as the k vitamer profile and concentrations appear to vary considerably by geographical location, australia needs a vitamin k food composition dataset that is representative of foods consumed in australia.',\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '10.1016/j.foodchem.2022.133772',\n",
       "  'created': '2022-03-21',\n",
       "  'updated': '',\n",
       "  'authors': ['eleanor dunlop',\n",
       "   'jette jakobsen',\n",
       "   'marie bagge jensen',\n",
       "   'jayashree arcot',\n",
       "   'liang qiao',\n",
       "   'judy cunningham',\n",
       "   'lucinda j black'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2203.11402'},\n",
       " {'title': 'mathematical modeling of glioma invasion and therapy approaches via   kinetic theory of active particles',\n",
       "  'id': '2203.11578',\n",
       "  'abstract': 'a multiscale model for glioma spread in brain tissue under the influence of vascularization and vascular endothelial growth factors is proposed. it accounts for the interplay between the different components of the neoplasm and the healthy tissue and it investigates and compares various therapy approaches. precisely, these involve radio- and chemotherapy in a concurrent or adjuvant manner together with anti-angiogenic therapy affecting the vascular component of the system. we assess tumor growth and spread on the basis of dti data, which allows us to reconstruct a realistic brain geometry and tissue structure, and we apply our model to real glioma patient data. in this latter case, a space-dependent radiotherapy description is considered using data about the corresponding isodose curves.',\n",
       "  'categories': 'q-bio.cb',\n",
       "  'doi': '',\n",
       "  'created': '2022-03-22',\n",
       "  'updated': '2024-04-25',\n",
       "  'authors': ['martina conte',\n",
       "   'yvonne dzierma',\n",
       "   'sven knobe',\n",
       "   'christina surulescu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2203.11578'},\n",
       " {'title': 'using active matter to introduce spatial heterogeneity to the   susceptible-infected-recovered model of epidemic spreading',\n",
       "  'id': '2203.13341',\n",
       "  'abstract': 'the widely used susceptible-infected-recovered (s-i-r) epidemic model assumes a uniform, well-mixed population, and incorporation of spatial heterogeneities remains a major challenge. understanding failures of the mixing assumption is important for designing effective disease mitigation approaches. we combine a run-and-tumble self-propelled active matter system with an s-i-r model to capture the effects of spatial disorder. working in the motility-induced phase separation regime both with and without quenched disorder, we find two epidemic regimes. for low transmissibility, quenched disorder lowers the frequency of epidemics and increases their average duration. for high transmissibility, the epidemic spreads as a front and the epidemic curves are less sensitive to quenched disorder; however, within this regime it is possible for quenched disorder to enhance the contagion by creating regions of higher particle densities. we discuss how this system could be realized using artificial swimmers with mobile optical traps operated on a feedback loop.',\n",
       "  'categories': 'cond-mat.soft cond-mat.stat-mech q-bio.qm',\n",
       "  'doi': '10.1038/s41598-022-15223-5',\n",
       "  'created': '2022-03-24',\n",
       "  'updated': '',\n",
       "  'authors': ['p. forgacs',\n",
       "   'a. libal',\n",
       "   'c. reichhardt',\n",
       "   'n. hengartner',\n",
       "   'c. j. o. reichhardt'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2203.13341'},\n",
       " {'title': 'surface vision transformers: attention-based modelling applied to   cortical analysis',\n",
       "  'id': '2203.16414',\n",
       "  'abstract': 'the extension of convolutional neural networks (cnns) to non-euclidean geometries has led to multiple frameworks for studying manifolds. many of those methods have shown design limitations resulting in poor modelling of long-range associations, as the generalisation of convolutions to irregular surfaces is non-trivial. motivated by the success of attention-modelling in computer vision, we translate convolution-free vision transformer approaches to surface data, to introduce a domain-agnostic architecture to study any surface data projected onto a spherical manifold. here, surface patching is achieved by representing spherical data as a sequence of triangular patches, extracted from a subdivided icosphere. a transformer model encodes the sequence of patches via successive multi-head self-attention layers while preserving the sequence resolution. we validate the performance of the proposed surface vision transformer (sit) on the task of phenotype regression from cortical surface metrics derived from the developing human connectome project (dhcp). experiments show that the sit generally outperforms surface cnns, while performing comparably on registered and unregistered data. analysis of transformer attention maps offers strong potential to characterise subtle cognitive developmental patterns.',\n",
       "  'categories': 'cs.cv eess.iv q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-03-30',\n",
       "  'updated': '',\n",
       "  'authors': ['simon dahan',\n",
       "   'abdulah fawaz',\n",
       "   'logan z. j. williams',\n",
       "   'chunhui yang',\n",
       "   'timothy s. coalson',\n",
       "   'matthew f. glasser',\n",
       "   'a. david edwards',\n",
       "   'daniel rueckert',\n",
       "   'emma c. robinson'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2203.16414'},\n",
       " {'title': 'a cognitive architecture for machine consciousness and artificial   superintelligence: thought is structured by the iterative updating of working   memory',\n",
       "  'id': '2203.17255',\n",
       "  'abstract': 'this article provides an analytical framework for how to simulate human-like thought processes within a computer. it describes how attention and memory should be structured, updated, and utilized to search for associative additions to the stream of thought. the focus is on replicating the dynamics of the mammalian working memory system, which features two forms of persistent activity: sustained firing (preserving information on the order of seconds) and synaptic potentiation (preserving information from minutes to hours). the article uses a series of figures to systematically demonstrate how the iterative updating of these working memory stores provides functional organization to behavior, cognition, and awareness.   in a machine learning implementation, these two memory stores should be updated continuously and in an iterative fashion. this means each state should preserve a proportion of the coactive representations from the state before it (where each representation is an ensemble of neural network nodes). this makes each state a revised iteration of the preceding state and causes successive configurations to overlap and blend with respect to the information they contain. thus, the set of concepts in working memory will evolve gradually and incrementally over time. transitions between states happen as persistent activity spreads activation energy throughout the hierarchical network, searching long-term memory for the most appropriate representation to be added to the global workspace. the result is a chain of associatively linked intermediate states capable of advancing toward a solution or goal. iterative updating is conceptualized here as an information processing strategy, a model of working memory, a theory of consciousness, and an algorithm for designing and programming artificial intelligence (ai, agi, and asi).',\n",
       "  'categories': 'q-bio.nc cs.cl cs.cv',\n",
       "  'doi': '',\n",
       "  'created': '2022-03-29',\n",
       "  'updated': '2024-11-13',\n",
       "  'authors': ['jared edward reser'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2203.17255'},\n",
       " {'title': 'mesoscopic description of hippocampal replay and metastability in   spiking neural networks with short-term plasticity',\n",
       "  'id': '2204.01675',\n",
       "  'abstract': 'bottom-up models of functionally relevant patterns of neural activity provide an explicit link between neuronal dynamics and computation. a prime example of functional activity pattern is hippocampal replay, which is critical for memory consolidation. the switchings between replay events and a low-activity state in neural recordings suggests metastable neural circuit dynamics. as metastability has been attributed to noise and/or slow fatigue mechanisms, we propose a concise mesoscopic model which accounts for both. crucially, our model is bottom-up: it is analytically derived from the dynamics of finite-size networks of linear-nonlinear poisson neurons with short-term synaptic depression. as such, noise is explicitly linked to spike noise and network size, and fatigue is explicitly linked to synaptic dynamics. to derive the mesosocpic model, we first consider a homogeneous spiking neural network and follow the temporal coarse-graining approach of gillespie (\"chemical langevin equation\"), which can be naturally interpreted as a stochastic neural mass model. the langevin equation is computationally inexpensive to simulate and enables a thorough study of metastable dynamics in classical setups (population spikes and up-down states dynamics) by means of phase-plane analysis. this stochastic neural mass model is the basic component of our mesoscopic model for replay. we show that our model faithfully captures the stochastic nature of individual replayed trajectories. moreover, compared to the deterministic romani-tsodyks model of place cell dynamics, it exhibits a higher level of variability in terms of content, direction and timing of replay events, compatible with biological evidence and could be functionally desirable. this variability is the product of a new dynamical regime where metastability emerges from a complex interplay between finite-size fluctuations and local fatigue.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1371/journal.pcbi.1010809',\n",
       "  'created': '2022-04-04',\n",
       "  'updated': '2022-04-29',\n",
       "  'authors': ['bastian pietras', 'valentin schmutz', 'tilo schwalger'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2204.01675'},\n",
       " {'title': 'mitigating mismatch compression in differential local field potentials',\n",
       "  'id': '2204.03778',\n",
       "  'abstract': 'bidirectional deep brain stimulation (bddbs) devices capable of recording differential local field potentials (dlfp) enable neural recordings alongside clinical therapy. efforts to identify objective signals of various brain disorders, or disease readouts, are challenging in dlfp, especially during active dbs. in this report we identified, characterized, and mitigated a major source of distortion in dlfp that we introduce as mismatch compression (mc). mc occurs secondary to impedance mismatches across the dlfp channel resulting in incomplete rejection of artifacts and downstream amplifier gain compression. using in silico and in vitro models we demonstrate that mc accounts for impedance-related distortions sensitive to dbs amplitude. we then use these models to develop and validate a mitigation strategy for mc that is provided as an opensource library for more reliable oscillatory disease readouts.',\n",
       "  'categories': 'eess.sp q-bio.nc',\n",
       "  'doi': '10.1109/tnsre.2022.3217469',\n",
       "  'created': '2022-04-07',\n",
       "  'updated': '',\n",
       "  'authors': ['vineet tiruvadi',\n",
       "   'sam james',\n",
       "   'bryan howell',\n",
       "   'mosadoluwa obatusin',\n",
       "   'andrea crowell',\n",
       "   'patricio riva-posse',\n",
       "   'ki sueng choi',\n",
       "   'allison waters',\n",
       "   'robert e. gross',\n",
       "   'cameron c. mcintyre',\n",
       "   'helen s. mayberg',\n",
       "   'robert butera'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2204.03778'},\n",
       " {'title': 'a spiking neural network based on neural manifold for augmenting   intracortical brain-computer interface data',\n",
       "  'id': '2204.05132',\n",
       "  'abstract': 'brain-computer interfaces (bcis), transform neural signals in the brain into in-structions to control external devices. however, obtaining sufficient training data is difficult as well as limited. with the advent of advanced machine learning methods, the capability of brain-computer interfaces has been enhanced like never before, however, these methods require a large amount of data for training and thus require data augmentation of the limited data available. here, we use spiking neural networks (snn) as data generators. it is touted as the next-generation neu-ral network and is considered as one of the algorithms oriented to general artifi-cial intelligence because it borrows the neural information processing from bio-logical neurons. we use the snn to generate neural spike information that is bio-interpretable and conforms to the intrinsic patterns in the original neural data. ex-periments show that the model can directly synthesize new spike trains, which in turn improves the generalization ability of the bci decoder. both the input and output of the spiking neural model are spike information, which is a brain-inspired intelligence approach that can be better integrated with bci in the future.',\n",
       "  'categories': 'q-bio.nc cs.ai',\n",
       "  'doi': '',\n",
       "  'created': '2022-03-26',\n",
       "  'updated': '',\n",
       "  'authors': ['shengjie zheng',\n",
       "   'wenyi li',\n",
       "   'lang qian',\n",
       "   'chenggang he',\n",
       "   'xiaojian li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2204.05132'},\n",
       " {'title': 'spatio-temporally graded causality: a model',\n",
       "  'id': '2204.07161',\n",
       "  'abstract': \"in this paper we consider a claim that in the natural world there is no fact of the matter about the spatio-temporal separation of events. in order to make sense of such a notion and construct useful models of the world, it is proposed to use elements of a non-classical logic. specifically, a model is proposed here, according to which causality can be considered to be spatio-temporally graded. it is outlined how this can be described using the formalism of fuzzy sets theory, with the degree of causality varying between 1, that is no separation between causes and effects, and 0, that is perfect separation between causes and their effects as in classical 'billiard balls' models of physical systems, namely such based on the notion of ideal mathematical point. our model posits that subjective moments of time are like fuzzy sets, with their extension determined by local degrees of causality, resulting from information integration processes extended gradually in space and time. this, we argue, is how a notion of causality could be, to a certain degree, spared and reconciled with a variant of bergsonian duration theory as formulated in the theory of continuous change. relation of the proposed viewpoint to other theories, as well as possible solutions it suggests to various problems, in particular the measurement problem, are also discussed.\",\n",
       "  'categories': 'q-bio.nc math-ph math.mp physics.hist-ph',\n",
       "  'doi': '10.1007/s10701-024-00761-x',\n",
       "  'created': '2022-03-17',\n",
       "  'updated': '2023-10-23',\n",
       "  'authors': ['bartosz jura'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2204.07161'},\n",
       " {'title': 'rapid 3d multiparametric mapping of brain metastases with deep   learning-based phase-sensitive mr fingerprinting',\n",
       "  'id': '2204.07313',\n",
       "  'abstract': 'in mr fingerprinting (mrf) reconstruction, measured data is pattern-matched to simulated signals to extract quantitative tissue parameters. a critical drawback to this approach is the exponentially increasing compute time for mapping of multiple parameters. previously, a deep learning (dl) reconstruction method called drone was shown to overcome this constraint by mapping the magnitude time-series signal to the underlying tissue parameters. however, relaxometry from magnitude images is susceptible to errors arising from ambiguities in the zero crossing of the signal or the non-zero noise mean. the aim of this study is to develop rapid acquisition and quantification methods to enable accurate multiparametric tissue mapping from complex data. an optimized epi based mrf sequence is developed along with a novel phasesensitive dl quantification allowing the use of real-valued neural networks to reconstruct complex measured data and providing an additional quantitative map of the phase. phantom experiments demonstrate the accuracy of the proposed approach. a comparison to previous drone methods in a healthy subject shows improved fidelity to known t1 and t2 values for the phase-sensitive approach. by processing the estimated phase map with conventional quantitative susceptibility mapping algorithms, we demonstrate the feasibility of simultaneous quantification of proton density, t1, t2, transmitter b1+ field and the quantitative susceptibility maps. in vivo experiments in a healthy volunteer and a subject with metastatic brain cancer are used to illustrate potential applications of this technology for treatment response assessment and tumor characterization.',\n",
       "  'categories': 'physics.med-ph q-bio.qm',\n",
       "  'doi': '10.1016/j.mri.2024.03.023',\n",
       "  'created': '2022-04-14',\n",
       "  'updated': '',\n",
       "  'authors': ['victoria y. yu',\n",
       "   'kathryn r. tringale',\n",
       "   'ricardo otazo',\n",
       "   'ouri cohen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2204.07313'},\n",
       " {'title': 'biocosmology: towards the birth of a new science',\n",
       "  'id': '2204.09378',\n",
       "  'abstract': 'cosmologists wish to explain how our universe, in all its complexity, could ever have come about. for that, we assess the number of states in our universe now. this plays the role of entropy in thermodynamics of the universe, and reveals the magnitude of the problem of initial conditions to be solved. the usual budgeting accounts for gravity, thermal motions, and finally the vacuum energy whose entropy, given by the bekenstein bound, dominates the entropy budget today. there is however one number which we have not accounted for: the number of states in our complex biosphere. what is the entropy of life and is it sizeable enough to need to be accounted for at the big bang? building on emerging ideas within theoretical biology, we show that the configuration space of living systems, unlike that of their fundamental physics counterparts, can grow rapidly in response to emerging biological complexity. a model for this expansion is provided through combinatorial innovation by the theory of the adjacent possible (tap) and its corresponding tap equation, whose solutions we investigate, confirming the possibility of rapid state-pace growth. while the results of this work remain far from being firmly established, the evidence we provide is many-fold and strong. the implications are far-reaching, and open a variety of lines for future investigation, a new scientific field we term biocosmology. in particular the relationship between the information content in life and the information content in the universe may need to be rebuilt from scratch.',\n",
       "  'categories': 'astro-ph.co q-bio.ot',\n",
       "  'doi': '',\n",
       "  'created': '2022-04-20',\n",
       "  'updated': '2024-09-27',\n",
       "  'authors': ['marina cortês',\n",
       "   'stuart a. kauffman',\n",
       "   'andrew r. liddle',\n",
       "   'lee smolin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2204.09378'},\n",
       " {'title': 'neuro-bert: rethinking masked autoencoding for self-supervised   neurological pretraining',\n",
       "  'id': '2204.12440',\n",
       "  'abstract': 'deep learning associated with neurological signals is poised to drive major advancements in diverse fields such as medical diagnostics, neurorehabilitation, and brain-computer interfaces. the challenge in harnessing the full potential of these signals lies in the dependency on extensive, high-quality annotated data, which is often scarce and expensive to acquire, requiring specialized infrastructure and domain expertise. to address the appetite for data in deep learning, we present neuro-bert, a self-supervised pre-training framework of neurological signals based on masked autoencoding in the fourier domain. the intuition behind our approach is simple: frequency and phase distribution of neurological signals can reveal intricate neurological activities. we propose a novel pre-training task dubbed fourier inversion prediction (fip), which randomly masks out a portion of the input signal and then predicts the missing information using the fourier inversion theorem. pre-trained models can be potentially used for various downstream tasks such as sleep stage classification and gesture recognition. unlike contrastive-based methods, which strongly rely on carefully hand-crafted augmentations and siamese structure, our approach works reasonably well with a simple transformer encoder with no augmentation requirements. by evaluating our method on several benchmark datasets, we show that neuro-bert improves downstream neurological-related tasks by a large margin.',\n",
       "  'categories': 'eess.sp cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-04-20',\n",
       "  'updated': '2024-07-03',\n",
       "  'authors': ['di wu', 'siyuan li', 'jie yang', 'mohamad sawan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2204.12440'},\n",
       " {'title': 'the tap equation: evaluating combinatorial innovation in biocosmology',\n",
       "  'id': '2204.14115',\n",
       "  'abstract': 'we investigate solutions to the tap equation, a phenomenological implementation of the theory of the adjacent possible. several implementations of tap are studied, with potential applications in a range of topics including economics, social sciences, environmental change, evolutionary biological systems, and the nature of physical laws. the generic behaviour is an extended plateau followed by a sharp explosive divergence. we find accurate analytic approximations for the blow-up time that we validate against numerical simulations, and explore the properties of the equation in the vicinity of equilibrium between innovation and extinction. a particular variant, the two-scale tap model, replaces the initial plateau with a phase of exponential growth, a widening of the tap equation phenomenology that may enable it to be applied in a wider range of contexts.',\n",
       "  'categories': 'q-bio.pe astro-ph.co',\n",
       "  'doi': '',\n",
       "  'created': '2022-04-24',\n",
       "  'updated': '2024-10-14',\n",
       "  'authors': ['marina cortês',\n",
       "   'stuart a. kauffman',\n",
       "   'andrew r. liddle',\n",
       "   'lee smolin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2204.14115'},\n",
       " {'title': 'discovering stochastic dynamical equations from biological time series   data',\n",
       "  'id': '2205.02645',\n",
       "  'abstract': 'theoretical studies have shown that stochasticity can affect the dynamics of ecosystems in counter-intuitive ways. however, without knowing the equations governing the dynamics of populations or ecosystems, it is difficult to ascertain the role of stochasticity in real datasets. therefore, the inverse problem of inferring the governing stochastic equations from datasets is important. here, we present an equation discovery methodology that takes time series data of state variables as input and outputs a stochastic differential equation. we achieve this by combining traditional approaches from stochastic calculus with the equation-discovery techniques. we demonstrate the generality of the method via several applications. first, we deliberately choose various stochastic models with fundamentally different governing equations; yet they produce nearly identical steady-state distributions. we show that we can recover the correct underlying equations, and thus infer the structure of their stability, accurately from the analysis of time series data alone. we demonstrate our method on two real-world datasets -- fish schooling and single-cell migration -- which have vastly different spatiotemporal scales and dynamics. we illustrate various limitations and potential pitfalls of the method and how to overcome them via diagnostic measures. finally, we provide our open-source codes via a package named pydaddy (python library for data driven dynamics).',\n",
       "  'categories': 'q-bio.qm cs.lg math.ds',\n",
       "  'doi': '',\n",
       "  'created': '2022-05-05',\n",
       "  'updated': '2024-09-22',\n",
       "  'authors': ['arshed nabeel',\n",
       "   'ashwin karichannavar',\n",
       "   'shuaib palathingal',\n",
       "   'jitesh jhawar',\n",
       "   'david b. brückner',\n",
       "   'danny raj m.',\n",
       "   'vishwesha guttal'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.02645'},\n",
       " {'title': 'single-species population models with age structure and psychological   effect in a polluted environment',\n",
       "  'id': '2205.03905',\n",
       "  'abstract': 'this paper considers a single-population model with age structure and psychological effects in a polluted environment. we divide the single population into two stages of larval and adult structure. the model uses logistic input, and the larvae are converted into adult bodies by constant ratio. we only consider adulthood. the role of psychological effects makes the contact between adult and environmental toxins a functional form, while the contact between larvae and environmental toxins is linear.   for the deterministic model embodied as a nonlinear time-varying system, we discuss the asymptotic stability of the system by lyapunov one-time approximation theory, and give a sufficient condition for stability to be established.   considering that the contact rate between biological and environmental toxins in nature is not always constant, we make the contact rate interfere with white noise, and then modify the contact rate into a stochastic process, thus establishing a corresponding random single-population model. according to it\\\\^o formula and lyapunov in the function method, we first prove the existence of globally unique positive solutions for stochastic models under arbitrary initial conditions, and then give sufficient conditions for weak average long-term survival and random long-term survival for single populations in the expected sense.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2022-05-08',\n",
       "  'updated': '2024-02-20',\n",
       "  'authors': ['jiawei wang', 'ruiyang zhou', 'fengying wei'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.03905'},\n",
       " {'title': 'an age-structured epidemic model with vaccination',\n",
       "  'id': '2205.03912',\n",
       "  'abstract': 'in this article, we construct an age-structured model for covid-19 with vaccination and analyze it from multiple perspectives. we derive the unique disease-free equilibrium point and the basic reproduction number $ \\\\mathscr{r}_0 $, then we show that the disease-free equilibrium is locally asymptotically stable when $ \\\\mathscr{r}_0 < 1 $, while is unstable when $ \\\\mathscr{r}_0 > 1 $. we also work out endemic equilibrium points and reveal the stability. we use sensitivity analysis to explore how parameters influence $ \\\\mathscr{r}_0 $. sensitivity analysis helps us develop more targeted strategies to control epidemics. finally, this model is used to discuss the cases in shijiazhuang, hebei province at the beginning of 2021. we compare reported cases with the simulation to evaluate the measures taken by shijiazhuang government. our study shows how age structure, vaccination and drastic containment measures can affect the epidemic.',\n",
       "  'categories': 'q-bio.pe physics.soc-ph',\n",
       "  'doi': '10.1371/journal.pone.0306554',\n",
       "  'created': '2022-05-08',\n",
       "  'updated': '2022-08-30',\n",
       "  'authors': ['ruiyang zhou', 'fengying wei'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.03912'},\n",
       " {'title': 'approaches to the classification of complex systems: words, texts, and   more',\n",
       "  'id': '2205.04060',\n",
       "  'abstract': 'the chapter starts with introductory information about quantitative linguistics notions, like rank--frequency dependence, zipf\\'s law, frequency spectra, etc. similarities in distributions of words in texts with level occupation in quantum ensembles hint at a superficial analogy with statistical physics. this enables one to define various parameters for texts based on this physical analogy, including \"temperature\", \"chemical potential\", entropy, and some others. such parameters provide a set of variables to classify texts serving as an example of complex systems. moreover, texts are perhaps the easiest complex systems to collect and analyze.   similar approaches can be developed to study, for instance, genomes due to well-known linguistic analogies. we consider a couple of approaches to define nucleotide sequences in mitochondrial dnas and viral rnas and demonstrate their possible application as an auxiliary tool for comparative analysis of genomes.   finally, we discuss entropy as one of the parameters, which can be easily computed from rank--frequency dependences. being a discriminating parameter in some problems of classification of complex systems, entropy can be given a proper interpretation only in a limited class of problems. its overall role and significance remain an open issue so far.',\n",
       "  'categories': 'physics.data-an cond-mat.stat-mech cs.cl q-bio.ot',\n",
       "  'doi': '10.1142/9789811260438_0005',\n",
       "  'created': '2022-05-09',\n",
       "  'updated': '',\n",
       "  'authors': ['andrij rovenchak'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.04060'},\n",
       " {'title': 'pre-frontal cortex guides dimension-reducing transformations in the   occipito-ventral pathway for categorization behaviors',\n",
       "  'id': '2205.04393',\n",
       "  'abstract': 'to interpret our surroundings, the brain uses a visual categorization process. current theories and models suggest that this process comprises a hierarchy of different computations that transforms complex, high-dimensional inputs into lower-dimensional representations (i.e. manifolds) in support of multiple categorization behaviors. here, we tested this hypothesis by analyzing these transformations reflected in dynamic meg source activity while individual participants actively categorized the same stimuli according to different tasks: face expression, face gender, pedestrian gender, vehicle type. results reveal three transformation stages guided by pre-frontal cortex. at stage 1 (high-dimensional, 50-120ms), occipital sources represent both task-relevant and task-irrelevant stimulus features; task-relevant features advance into higher ventral/dorsal regions whereas task-irrelevant features halt at the occipital-temporal junction. at stage 2 (121-150ms), stimulus feature representations reduce to lower-dimensional manifolds, which then transform into the task-relevant features underlying categorization behavior over stage 3 (161-350ms).',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-05-09',\n",
       "  'updated': '2024-06-20',\n",
       "  'authors': ['y. duan',\n",
       "   'j. zhan',\n",
       "   'j. gross',\n",
       "   'r. a. a. ince',\n",
       "   'p. g. schyns'],\n",
       "  'affiliation': ['school of psychology and neuroscience, university of glasgow, united kingdom',\n",
       "   'school of psychology and neuroscience, university of glasgow, united kingdom',\n",
       "   'institute for biomagnetism and biosignalanalysis, university of münster, germany',\n",
       "   'school of psychology and neuroscience, university of glasgow, united kingdom',\n",
       "   'school of psychology and neuroscience, university of glasgow, united kingdom'],\n",
       "  'url': 'https://arxiv.org/abs/2205.04393'},\n",
       " {'title': 'cagi, the critical assessment of genome interpretation, establishes   progress and prospects for computational genetic variant interpretation   methods',\n",
       "  'id': '2205.05897',\n",
       "  'abstract': 'the critical assessment of genome interpretation (cagi) aims to advance the state of the art for computational prediction of genetic variant impact, particularly those relevant to disease. the five complete editions of the cagi community experiment comprised 50 challenges, in which participants made blind predictions of phenotypes from genetic data, and these were evaluated by independent assessors. overall, results show that while current methods are imperfect, they have major utility for research and clinical applications. missense variant interpretation methods are able to estimate biochemical effects with increasing accuracy. performance is particularly strong for clinical pathogenic variants, including some difficult-to-diagnose cases, and extends to interpretation of cancer-related variants. assessment of methods for regulatory variants and complex trait disease risk is less definitive, and indicates performance potentially suitable for auxiliary use in the clinic. emerging methods and increasingly large, robust datasets for training and assessment promise further progress ahead.',\n",
       "  'categories': 'q-bio.gn',\n",
       "  'doi': '10.1186/s13059-023-03113-6',\n",
       "  'created': '2022-05-12',\n",
       "  'updated': '',\n",
       "  'authors': ['the critical assessment of genome interpretation consortium'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.05897'},\n",
       " {'title': 'an automatic pipeline for atlas-based fetal and neonatal brain   segmentation and analysis',\n",
       "  'id': '2205.07575',\n",
       "  'abstract': 'the automatic segmentation of perinatal brain structures in magnetic resonance imaging (mri) is of utmost importance for the study of brain growth and related complications. while different methods exist for adult and pediatric mri data, there is a lack for automatic tools for the analysis of perinatal imaging. in this work, a new pipeline for fetal and neonatal segmentation has been developed. we also report the creation of two new fetal atlases, and their use within the pipeline for atlas-based segmentation, based on novel registration methods. the pipeline is also able to extract cortical and pial surfaces and compute features, such as curvature, thickness, sulcal depth, and local gyrification index. results show that the introduction of the new templates together with our segmentation strategy leads to accurate results when compared to expert annotations, as well as better performances when compared to a reference pipeline (developing human connectome project (dhcp)), for both early and late-onset fetal brains.',\n",
       "  'categories': 'cs.cv q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-05-16',\n",
       "  'updated': '',\n",
       "  'authors': ['andrea urru',\n",
       "   'ayako nakaki',\n",
       "   'oualid benkarim',\n",
       "   'francesca crovetto',\n",
       "   'laura segales',\n",
       "   'valentin comte',\n",
       "   'nadine hahner',\n",
       "   'elisenda eixarch',\n",
       "   'eduard gratacós',\n",
       "   'fàtima crispi',\n",
       "   'gemma piella',\n",
       "   'miguel a gonzález ballester'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.07575'},\n",
       " {'title': 'brain waves are a repetition of a pause and an activity',\n",
       "  'id': '2205.08734',\n",
       "  'abstract': 'brain waves still cannot reliably distinguish between awake and asleep states. here, i present new original indices, voltage subthreshold wave $\\\\tau$ and abovethreshold wave burst, for advanced lfp/eeg readings. assuming that $\\\\tau$ is a microwave that fluctuates every sample such as the equipotential, the total number of $\\\\tau$ ($n\\\\tau$) is inferred to be the maximum, and the amplitude of burst (abst) is inferred to be the minimum. in fact, they invariably had a mean $\\\\tau$ duration ($m\\\\tau$) of 2-3 sample intervals in any case. in addition, $\\\\tau$ and burst exhibited self-similarity for sample frequency while occupying approximately 30% and 70% of lfp in the natural state, respectively. its threshold and abst were correlated with the vigilance state and decreased to 70% by doubling the sample frequency. the dose of sevoflurane, which inhibits and synchronizes neural activity, was linearly correlated with decreases in the threshold and $n\\\\tau$. thus, $\\\\tau$ could reflect the uncertainty of the membrane potential. i propose that $\\\\tau$ and burst represent a pause and an activity such as the rhythm of the brain.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-05-18',\n",
       "  'updated': '2024-11-16',\n",
       "  'authors': ['chika koyama'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.08734'},\n",
       " {'title': 'mathematical characterization of private and public immune repertoire   sequences',\n",
       "  'id': '2205.08750',\n",
       "  'abstract': 'diverse t and b cell repertoires play an important role in mounting effective immune responses against a wide range of pathogens and malignant cells. the number of unique t and b cell clones is characterized by t and b cell receptors (tcrs and bcrs), respectively. although receptor sequences are generated probabilistically by recombination processes, clinical studies found a high degree of sharing of tcrs and bcrs among different individuals. in this work, we use a general probabilistic model for t/b cell receptor clone abundances to define \"publicness\" or \"privateness\" and information-theoretic measures for comparing the frequency of sampled sequences observed across different individuals. we derive mathematical formulae to quantify the mean and the variances of clone richness and overlap. our results can be used to evaluate the effect of different sampling protocols on abundances of clones within an individual as well as the commonality of clones across individuals. using synthetic and empirical tcr amino acid sequence data, we perform simulations to study expected clonal commonalities across multiple individuals. based on our formulae, we compare these simulated results with the analytically predicted mean and variances of the repertoire overlap. complementing the results on simulated repertoires, we derive explicit expressions for the richness and its uncertainty for specific, single-parameter truncated power-law probability distributions. finally, the information loss associated with grouping together certain receptor sequences, as is done in spectratyping, is also evaluated. our approach can be, in principle, applied under more general and mechanistically realistic clone generation models.',\n",
       "  'categories': 'q-bio.qm cond-mat.stat-mech q-bio.pe',\n",
       "  'doi': '10.1007/s11538-023-01190-z',\n",
       "  'created': '2022-05-18',\n",
       "  'updated': '2023-08-19',\n",
       "  'authors': ['lucas böttcher', 'sascha wald', 'tom chou'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.08750'},\n",
       " {'title': 'odbo: bayesian optimization with search space prescreening for directed   protein evolution',\n",
       "  'id': '2205.09548',\n",
       "  'abstract': 'directed evolution is a versatile technique in protein engineering that mimics the process of natural selection by iteratively alternating between mutagenesis and screening in order to search for sequences that optimize a given property of interest, such as catalytic activity and binding affinity to a specified target. however, the space of possible proteins is too large to search exhaustively in the laboratory, and functional proteins are scarce in the vast sequence space. machine learning (ml) approaches can accelerate directed evolution by learning to map protein sequences to functions without building a detailed model of the underlying physics, chemistry and biological pathways. despite the great potentials held by these ml methods, they encounter severe challenges in identifying the most suitable sequences for a targeted function. these failures can be attributed to the common practice of adopting a high-dimensional feature representation for protein sequences and inefficient search methods. to address these issues, we propose an efficient, experimental design-oriented closed-loop optimization framework for protein directed evolution, termed odbo, which employs a combination of novel low-dimensional protein encoding strategy and bayesian optimization enhanced with search space prescreening via outlier detection. we further design an initial sample selection strategy to minimize the number of experimental samples for training ml models. we conduct and report four protein directed evolution experiments that substantiate the capability of the proposed framework for finding of the variants with properties of interest. we expect the odbo framework to greatly reduce the experimental cost and time cost of directed evolution, and can be further generalized as a powerful tool for adaptive experimental design in a broader context.',\n",
       "  'categories': 'q-bio.bm cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-05-19',\n",
       "  'updated': '2024-05-01',\n",
       "  'authors': ['lixue cheng',\n",
       "   'ziyi yang',\n",
       "   'changyu hsieh',\n",
       "   'benben liao',\n",
       "   'shengyu zhang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.09548'},\n",
       " {'title': 'complex behavior from intrinsic motivation to occupy action-state path   space',\n",
       "  'id': '2205.10316',\n",
       "  'abstract': \"most theories of behavior posit that agents tend to maximize some form of reward or utility. however, animals very often move with curiosity and seem to be motivated in a reward-free manner. here we abandon the idea of reward maximization, and propose that the goal of behavior is maximizing occupancy of future paths of actions and states. according to this maximum occupancy principle, rewards are the means to occupy path space, not the goal per se; goal-directedness simply emerges as rational ways of searching for resources so that movement, understood amply, never ends. we find that action-state path entropy is the only measure consistent with additivity and other intuitive properties of expected future action-state path occupancy. we provide analytical expressions that relate the optimal policy and state-value function, and prove convergence of our value iteration algorithm. using discrete and continuous state tasks, including a high--dimensional controller, we show that complex behaviors such as `dancing', hide-and-seek and a basic form of altruistic behavior naturally result from the intrinsic motivation to occupy path space. all in all, we present a theory of behavior that generates both variability and goal-directedness in the absence of reward maximization.\",\n",
       "  'categories': 'cs.ai cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-05-20',\n",
       "  'updated': '2024-02-24',\n",
       "  'authors': ['jorge ramírez-ruiz',\n",
       "   'dmytro grytskyy',\n",
       "   'chiara mastrogiuseppe',\n",
       "   'yamen habib',\n",
       "   'rubén moreno-bote'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.10316'},\n",
       " {'title': 'a ssim guided cgan architecture for clinically driven generative image   synthesis of multiplexed spatial proteomics channels',\n",
       "  'id': '2205.10373',\n",
       "  'abstract': 'here we present a structural similarity index measure (ssim) guided conditional generative adversarial network (cgan) that generatively performs image-to-image (i2i) synthesis to generate photo-accurate protein channels in multiplexed spatial proteomics images. this approach can be utilized to accurately generate missing spatial proteomics channels that were not included during experimental data collection either at the bench or the clinic. experimental spatial proteomic data from the human biomolecular atlas program (hubmap) was used to generate spatial representations of missing proteins through a u-net based image synthesis pipeline. hubmap channels were hierarchically clustered by the (ssim) as a heuristic to obtain the minimal set needed to recapitulate the underlying biology represented by the spatial landscape of proteins. we subsequently prove that our ssim based architecture allows for scaling of generative image synthesis to slides with up to 100 channels, which is better than current state of the art algorithms which are limited to data with 11 channels. we validate these claims by generating a new experimental spatial proteomics data set from human lung adenocarcinoma tissue sections and show that a model trained on hubmap can accurately synthesize channels from our new data set. the ability to recapitulate experimental data from sparsely stained multiplexed histological slides containing spatial proteomic will have tremendous impact on medical diagnostics and drug development, and also raises important questions on the medical ethics of utilizing data produced by generative image synthesis in the clinical setting. the algorithm that we present in this paper will allow researchers and clinicians to save time and costs in proteomics based histological staining while also increasing the amount of data that they can generate through their experiments.',\n",
       "  'categories': 'eess.iv cs.cv q-bio.qm q-bio.to',\n",
       "  'doi': '10.1109/cibcb56990.2023.10264899',\n",
       "  'created': '2022-05-20',\n",
       "  'updated': '2023-06-11',\n",
       "  'authors': ['jillur rahman saurav',\n",
       "   'mohammad sadegh nasr',\n",
       "   'paul koomey',\n",
       "   'michael robben',\n",
       "   'manfred huber',\n",
       "   'jon weidanz',\n",
       "   'bríd ryan',\n",
       "   'eytan ruppin',\n",
       "   'peng jiang',\n",
       "   'jacob m. luber'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.10373'},\n",
       " {'title': 'extended molt phenology models improve inferences about molt duration   and timing',\n",
       "  'id': '2205.12120',\n",
       "  'abstract': 'molt is an essential life-history event in birds and many mammals, as maintenance of feathers and fur is critical for survival. despite this molt remains an understudied life-history event. non-standard statistical techniques are required to estimate the phenology of molt from observations of plumage or pelage state, and existing molt phenology models have strict sampling requirements which can be difficult to meet under real-world conditions. we present an extended modelling framework that can accommodate features of real-world molt datasets, such re-encounters of individuals, misclassified molt states, and/or molt state-dependent sampling bias.   we demonstrate that such features can lead to biased inferences when using existing molt phenology models, and show that our model extensions can improve inferences about molt phenology under a wide range of sampling conditions. we hope that our novel modelling framework removes barriers for modelling molt phenology data from realworld datasets and thereby further facilitates the uptake of appropriate statistical methods for such data. although we focus on molt, the modelling framework is applicable to other phenological processes which can be recorded using either ordered categories or approximately linear progress scores.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.1093/ornithology/ukae003',\n",
       "  'created': '2022-05-24',\n",
       "  'updated': '2023-12-19',\n",
       "  'authors': ['philipp h. boersch-supan',\n",
       "   'hugh j. hanmer',\n",
       "   'robert a. robinson'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.12120'},\n",
       " {'title': 'cellular gradient flow structure connects single-cell-level rules and   population-level dynamics',\n",
       "  'id': '2205.13143',\n",
       "  'abstract': 'in multicellular systems, the single-cell behaviors should be coordinated consistently with the overall population dynamics and functions. however, the interrelation between single-cell rules and the population-level goal is still elusive. in this work, we reveal that these two levels are naturally connected via a gradient flow structure of the heterogeneous cellular population and that biologically prevalent single-cell rules such as unidirectional type-switching and hierarchical order in types emerge from this structure. we also demonstrate the gradient flow structure in a standard model of the t-cell immune response. this theoretical framework works as a basis for understanding multicellular dynamics and functions.',\n",
       "  'categories': 'q-bio.pe q-bio.cb',\n",
       "  'doi': '10.1103/physrevresearch.5.l022052',\n",
       "  'created': '2022-05-26',\n",
       "  'updated': '',\n",
       "  'authors': ['shuhei a. horiguchi', 'tetsuya j. kobayashi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.13143'},\n",
       " {'title': 'dima: sequence diversity dynamics analyser for viruses',\n",
       "  'id': '2205.13915',\n",
       "  'abstract': \"sequence diversity is one of the major challenges in the design of diagnostic, prophylactic and therapeutic interventions against viruses. dima is a novel tool that is big data-ready and designed to facilitate the dissection of sequence diversity dynamics for viruses. dima stands out from other diversity analysis tools by offering various unique features. dima provides a quantitative overview of sequence (nucleotide/protein) diversity by use of shannon's entropy corrected for size bias, applied via a user-defined k-mer sliding window to an input alignment file, and each k-mer position is dissected to various diversity motifs. the motifs are defined based on the probability of distinct sequences at a given k-mer position, whereby an index is the predominant sequence, while all the others are (total) variants to the index. the total variants are sub-classified into the major (most common) variant, minor variants (occurring more than once and of frequency lower than the major), and the unique (singleton) variants. dima allows user-defined, sequence metadata enrichment for analyses of the motifs. the application of dima was demonstrated for the alignment data of the relatively conserved spike protein (2,106,985 sequences) of severe acute respiratory syndrome coronavirus 2 (sars-cov-2) and the relatively highly diverse pol protein (3,874) of human immunodeficiency virus-1 (hiv-1). the tool is publicly available as a web server (https://dima.bezmialem.edu.tr), as a python library (via pypi) and as a command line client (via github).\",\n",
       "  'categories': 'q-bio.gn q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-05-27',\n",
       "  'updated': '2024-07-27',\n",
       "  'authors': ['shan tharanga',\n",
       "   'eyyub selim unlu',\n",
       "   'yongli hu',\n",
       "   'muhammad farhan sjaugi',\n",
       "   'muhammet a. celik',\n",
       "   'hilal hekimoglu',\n",
       "   'olivo miotto',\n",
       "   'muhammed miran oncel',\n",
       "   'asif m. khan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.13915'},\n",
       " {'title': 'group-level brain decoding with deep learning',\n",
       "  'id': '2205.14102',\n",
       "  'abstract': 'decoding brain imaging data are gaining popularity, with applications in brain-computer interfaces and the study of neural representations. decoding is typicallysubject-specific and does not generalise well over subjects, due to high amounts ofbetween subject variability. techniques that overcome this will not only providericher neuroscientific insights but also make it possible for group-level models to out-perform subject-specific models. here, we propose a method that uses subjectembedding, analogous to word embedding in natural language processing, to learnand exploit the structure in between-subject variability as part of a decoding model,our adaptation of the wavenet architecture for classification. we apply this to mag-netoencephalography data, where 15 subjects viewed 118 different images, with30 examples per image; to classify images using the entire 1 s window followingimage presentation. we show that the combination of deep learning and subjectembedding is crucial to closing the performance gap between subject- and group-level decoding models. importantly, group models outperform subject models onlow-accuracy subjects (although slightly impair high-accuracy subjects) and can behelpful for initialising subject models. while we have not generally found group-levelmodels to perform better than subject-level models, the performance of groupmodelling is expected to be even higher with bigger datasets. in order to providephysiological interpretation at the group level, we make use of permutation featureimportance. this provides insights into the spatiotemporal and spectral informationencoded in the models. all code is available on github (https://github.com/ricsinaruto/meg-group-decode).',\n",
       "  'categories': 'cs.lg eess.sp q-bio.nc',\n",
       "  'doi': '10.1002/hbm.26500',\n",
       "  'created': '2022-05-27',\n",
       "  'updated': '2024-01-19',\n",
       "  'authors': ['richard csaky',\n",
       "   'mats van es',\n",
       "   'oiwi parker jones',\n",
       "   'mark woolrich'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2205.14102'},\n",
       " {'title': 'kinetic chemotaxis tumbling kernel determined from macroscopic   quantities',\n",
       "  'id': '2206.01629',\n",
       "  'abstract': 'chemotaxis is the physical phenomenon that bacteria adjust their motions according to chemical stimulus. a classical model for this phenomenon is a kinetic equation that describes the velocity jump process whose tumbling/transition kernel uniquely determines the effect of chemical stimulus on bacteria. the model has been shown to be an accurate model that matches with bacteria motion qualitatively. for a quantitative modeling, biophysicists and practitioners are also highly interested in determining the explicit value of the tumbling kernel. due to the experimental limitations, measurements are typically macroscopic in nature. do macroscopic quantities contain enough information to recover microscopic behavior? in this paper, we give a positive answer. we show that when given a special design of initial data, the population density, one specific macroscopic quantity as a function of time, contains sufficient information to recover the tumbling kernel and its associated damping coefficient. moreover, we can read off the chemotaxis tumbling kernel using the values of population density directly from this specific experimental design. this theoretical result using kinetic theory sheds light on how practitioners may conduct experiments in laboratories.',\n",
       "  'categories': 'math.ap q-bio.qm',\n",
       "  'doi': '10.1137/22m1499911',\n",
       "  'created': '2022-06-03',\n",
       "  'updated': '2023-10-04',\n",
       "  'authors': ['kathrin hellmuth',\n",
       "   'christian klingenberg',\n",
       "   'qin li',\n",
       "   'min tang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2206.01629'},\n",
       " {'title': 'total controllability analysis discovers explainable drugs for covid-19   treatment',\n",
       "  'id': '2206.02970',\n",
       "  'abstract': 'network medicine has been pursued for covid-19 drug repurposing. one such approach adopts structural controllability, a theory for controlling a network (the cell). motivated to protect the cell from viral infections, we extended this theory to total controllability and introduced a new concept of control hubs. perturbation to any control hub renders the cell uncontrollable by exogenous stimuli, e.g., viral infections, so control hubs are ideal drug targets. we developed an efficient algorithm for finding all control hubs and applied it to the largest homogenous human protein-protein interaction network. our new method outperforms several popular gene-selection methods, including that based on structural controllability. the final 65 druggable control hubs are enriched with functions of cell proliferation, regulation of apoptosis, and responses to cellular stress and nutrient levels, revealing critical pathways induced by sars-cov-2. these druggable control hubs led to drugs in 4 major categories: antiviral and anti-inflammatory agents, drugs on central nerve systems, and dietary supplements and hormones that boost immunity. their functions also provided deep insights into the therapeutic mechanisms of the drugs for covid-19 therapy, making the new approach an explainable drug repurposing method. a remarkable example is fostamatinib that has been shown to lower mortality, shorten the length of icu stay, and reduce disease severity of hospitalized covid-19 patients. the drug targets 10 control hubs, 9 of which are kinases that play key roles in cell differentiation and programmed death. one such kinase is ripk1 that directly interacts with viral protein nsp12, the rdrp of the virus. the study produced many control hubs that were not targets of existing drugs but were enriched with proteins on membranes and the nf-$\\\\kappa$b pathway, so are excellent candidate targets for new drugs.',\n",
       "  'categories': 'q-bio.mn cs.sy eess.sy',\n",
       "  'doi': '',\n",
       "  'created': '2022-06-06',\n",
       "  'updated': '2023-06-01',\n",
       "  'authors': ['xinru wei', 'chunyu pan', 'xizhe zhang', 'weixiong zhang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2206.02970'},\n",
       " {'title': 'atp-dependent mismatch recognition in dna replication mismatch repair',\n",
       "  'id': '2206.03026',\n",
       "  'abstract': 'mismatch repair is a critical step in dna replication that occurs after base selection and proofreading, significantly increasing fidelity. however, the mechanism of mismatch recognition has not been established for any repair enzyme. speculations in this area mainly focus on exploiting thermodynamic equilibrium and free energy. nevertheless, non-equilibrium processes may play a more significant role in enhancing mismatch recognition accuracy by utilizing adenosine triphosphate (atp). this study aimed to investigate this possibility. considering our limited knowledge of actual mismatch repair enzymes, we proposed a hypothetical enzyme that operates as a quantum system with three discrete energy levels. when the enzyme is raised to its highest energy level, a quantum transition occurs, leading to one of two low-energy levels representing potential recognition outcomes: a correct match or a mismatch. the probabilities of the two outcomes are exponentially different, determined by the energy gap between the two low energy levels. by flipping the energy gap, discrimination between mismatches and correct matches can be achieved. within a framework that combines quantum mechanics with thermodynamics, we established a relationship between energy cost and the recognition error.',\n",
       "  'categories': 'physics.bio-ph cond-mat.stat-mech q-bio.bm quant-ph',\n",
       "  'doi': '10.3390/quantum5030037',\n",
       "  'created': '2022-06-07',\n",
       "  'updated': '2023-08-30',\n",
       "  'authors': ['nianqin zhang', 'yongjun zhang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2206.03026'},\n",
       " {'title': 'petri nets in epidemiology',\n",
       "  'id': '2206.03269',\n",
       "  'abstract': 'this work provides a geometric version of the next-generation method for obtaining the basic reproduction number of an epidemiological model. more precisely, we generalize the concept of the basic reproduction number for the theory of petri nets. the motivation comes from observing that any epidemiological model has the basic structures found in the sir model of kermack-mckendrick. these are three substructures, also given by three petri nets inside, representing the susceptible population, the infection process, and the infected individuals. the five assumptions of the next-generation matrix given by van den driessche-watmough can be described geometrically using petri nets. thus, the next-generation matrix results in a matrix of flows between the infection compartments. besides that, we construct a procedure that associates with a system of ordinary differential equations under certain assumptions, a petri net that is minimal. in addition, we explain how petri nets extend compartmental models to include vertical transmission.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2022-05-30',\n",
       "  'updated': '2024-10-26',\n",
       "  'authors': ['carlos segovia'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2206.03269'},\n",
       " {'title': 'on the analysis of a time varying noise-modulated heterogeneous coupled   network of chialvo neurons under the influence of electromagnetic flux',\n",
       "  'id': '2206.04234',\n",
       "  'abstract': 'we perform a numerical study on the application of electromagnetic flux on a heterogeneous network of chialvo neurons represented by a ring-star topology. heterogeneities are realized by introducing additive noise modulations on both the central-peripheral and the peripheral-peripheral coupling links in the topology that not only vary in space but also in time. the variation in time is understood by two coupling probabilities, one for the central-peripheral connections and the other for the peripheral-peripheral connections respectively, that updates the network topology with each iteration in time. we have further reported the rich spatiotemporal patterns like two-cluster states, chimera states, traveling waves, coherent, and asynchronized states that arise throughout the network dynamics. we have also investigated the appearance of a special kind of asynchronization behavior called \"solitary nodes\" that have wide range of applications pertaining to real-world nervous systems. in order to characterize the behavior of the nodes under the influence of these heterogeneities, we have studied two different metrics called the \"cross-correlation coefficient\" and the \"synchronization error\". additionally, to capture the statistical property of the network, for example, how complex the system behaves, we have also studied a measure called \"sample entropy\". various two-dimensional color-coded plots are presented in the study to exhibit how these metrics/measures behave with the variation of parameters. finally, how the nodes synchronize or asynchronize is shown via one-dimensional bifurcation diagrams of the last instance of the main dynamical variable, i.e., the state variable associated with the membrane potential, against different network parameters.',\n",
       "  'categories': 'math.ds q-bio.nc',\n",
       "  'doi': '10.1007/s11071-023-08717-y',\n",
       "  'created': '2022-06-08',\n",
       "  'updated': '',\n",
       "  'authors': ['indranil ghosh',\n",
       "   'sishu shankar muni',\n",
       "   'hammed olawale fatoyinbo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2206.04234'},\n",
       " {'title': 'compression-induced continuous phase transition in the buckling of a   semiflexible filament for two and three dimensions',\n",
       "  'id': '2206.07921',\n",
       "  'abstract': 'the ability of biomolecules to exert forces on their surroundings or resist compression from the environment is essential in a variety of biologically relevant contexts. as has been understood for centuries, slender rods can only be compressed so far until they buckle, adopting an intrinsically bent state that may be unable to bear a compressive load. in the low-temperature limit and for a constant compressive force, euler buckling theory predicts a sudden transition from a compressed to a bent state in these slender rods. in this paper, we use a mean-field theory to show that if a semiflexible chain is compressed at a finite temperature with a fixed end-to-end distance (permitting fluctuations in the compressive forces), it exhibits a continuous phase transition to a buckled state at a critical level of compression, and we determine a quantitatively accurate prediction of the transverse position distribution function of the midpoint of the chain that indicates the transition. we find the mean compressive forces are non-monotonic as the extension of the filament varies, consistent with the observation that strongly buckled filaments are less able to bear an external load. we also find that for the fixed extension (isometric) ensemble that the buckling transition does not coincide with the local minimum of the mean force (in contrast to euler buckling). we also show the theory is highly sensitive to fluctuations in length, and that the buckling transition can still be accurately recovered by accounting for those fluctuations. these predictions may be useful in understanding the behavior of filamentous biomolecules compressed by fluctuating forces, relevant in a variety of biological contexts.',\n",
       "  'categories': 'cond-mat.soft cond-mat.stat-mech physics.bio-ph q-bio.bm',\n",
       "  'doi': '10.1063/5.0104910',\n",
       "  'created': '2022-06-16',\n",
       "  'updated': '',\n",
       "  'authors': ['ananya mondal', 'greg morrison'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2206.07921'},\n",
       " {'title': 'a quantitative analysis of dynamic mechanisms regulating hiv latency and   activation',\n",
       "  'id': '2206.10501',\n",
       "  'abstract': 'objective: the reservoir of human immunodeficiency virus (hiv) latently infected cells is the major obstacle for eradication of acquired immunodeficiency syndrome (aids). due to the noisy environment and multiple influencing factors in the organism, current dynamical models cannot reach a common understanding of the molecular mechanism of hiv latency. in this work, through a new dynamical structure decomposition, the deterministic part of the equation can be separated from the stochastic noise. thus, the fixed-point analysis of ordinary differential equation is enough to obtain the different steady states of the system. methods: we established a dynamical model of hiv transcription process by using continuous stochastic differential equations, which simplifies the dimensions of equations needed to describe the system and increases the explorable space of the model. different states between latency and activation of virus and their relations can be intuitively represented by potential functions and probability distribution functions. results: based on our model, the influence of different dynamical parameters on stability is quantitatively analyzed, the parameter ranges of the system in bistable and monostable states are obtained respectively. the theoretical basis of this work is verified by comparing the effects of different factors on dynamic bifurcation with the results of biological experiments. conclusion: this paper goes beyond previous discrete stochastic methods, and can quantitatively analyze the dynamic mechanism of hiv transcriptional regulation through ordinary differential equations, which is beneficial to the promotion to deal with the high-dimensional situation, and further study the occurrence and development of aids in vivo, so as to guide the design of experiments and search for clinical treatment.',\n",
       "  'categories': 'physics.bio-ph q-bio.mn',\n",
       "  'doi': '10.16476/j.pibb.2022.0053',\n",
       "  'created': '2022-06-21',\n",
       "  'updated': '2022-06-22',\n",
       "  'authors': ['ruiqi xiong', 'yang su', 'ping ao'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2206.10501'},\n",
       " {'title': 'frequency domain identifiability and sloppiness of descriptor systems   with an lft structure',\n",
       "  'id': '2206.10814',\n",
       "  'abstract': 'identifiability and sloppiness are investigated in this paper for the parameters of a descriptor system based on its frequency response samples. two metrics are suggested respectively for measuring absolute and relative sloppiness of the parameter vector at a prescribed value. in this descriptor system, system matrices are assumed to depend on its parameters through a linear fractional transformation (lft). when an associated transfer function matrix (tfm) is of full normal row rank, a matrix rank based necessary and sufficient condition is derived for parameter identifiability with a set of finitely many frequency responses. this condition can be verified recursively which is computationally quite appealing, especially when the system is of a large scale. from this condition, an algorithm is suggested to find a set of frequencies with which the frequency responses of the system are capable to uniquely determine its parameters. an ellipsoid approximation is given for the set consisting of all the parameter values with which the associated descriptor system has a frequency response that deviates within a prescribed distance, from that corresponding to a globally identifiable parameter vector value. explicit formulas are also derived for the suggested absolute and relative sloppiness metrics.',\n",
       "  'categories': 'eess.sy cs.sy math.oc q-bio.mn',\n",
       "  'doi': '10.1016/j.automatica.2023.111362',\n",
       "  'created': '2022-06-21',\n",
       "  'updated': '',\n",
       "  'authors': ['tong zhou'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2206.10814'},\n",
       " {'title': 'the temporal evolution of modality-independent representations of   conceptual categories',\n",
       "  'id': '2206.12508',\n",
       "  'abstract': \"to what extent does language production activate amodal conceptual representations? in picture naming, we view specific exemplars of concepts and then name them with a category label, like 'dog'. in contrast, in overt reading, the written word expresses the category (dog), not an exemplar.   here we used a decoding approach with magnetoencephalography to address whether picture naming and overt word reading involve shared representations of semantic categories. this addresses a fundamental question about the modality-generality of conceptual representations and their temporal evolution. crucially, we do this using a language production task that does not require explicit categorization judgment and that controls for word form properties across semantic categories. we trained our models to classify the animal/tool distinction using meg data of one modality at each time point and then tested the generalization of those models on the other modality. we obtained evidence for the automatic activation of modality-independent semantic category representations for both pictures and words starting at ~150ms and lasting until about 500ms. the time course of lexical activation was also assessed revealing that semantic category is represented before lexical access for pictures but after lexical access for words. notably, this earlier activation of semantic category in pictures occurred simultaneously with visual representations.   we thus show evidence for the spontaneous activation of modality-independent semantic categories in picture naming and word reading, supporting theories in which amodal conceptual representations exist. together, these results serve to anchor a more comprehensive spatio-temporal delineation of the semantic feature space during production planning.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1016/j.neuroimage.2023.120254',\n",
       "  'created': '2022-06-24',\n",
       "  'updated': '2022-06-30',\n",
       "  'authors': ['julien dirani', 'liina pylkkänen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2206.12508'},\n",
       " {'title': 'how large should be the redundant numbers of copy to make a rare event   probable',\n",
       "  'id': '2206.12687',\n",
       "  'abstract': 'the redundancy principle provides the framework to study how rare events are made possible with probability 1 in accelerated time, by making many copies of similar random searchers. but what is $n$ large? to estimate large $n$ with respect to the geometrical properties of a domain and the dynamics, we present here a criteria based on splitting probabilities between a small fraction of the exploration space associated to an activation process and other absorbing regions where trajectories can be terminated. we obtain explicit computations especially when there is a killing region located inside the domain that we compare with stochastic simulations. we present also examples of extreme trajectories with killing in dimension 2. for a large $n$, the optimal trajectories avoid penetrating inside the killing region. finally we discuss some applications to cell biology.',\n",
       "  'categories': 'cond-mat.soft math.ap q-bio.qm',\n",
       "  'doi': '10.1103/physreve.106.064402',\n",
       "  'created': '2022-06-25',\n",
       "  'updated': '',\n",
       "  'authors': ['fred paquin-lefebvre', 'suney toste', 'david holcman'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2206.12687'},\n",
       " {'title': 'linearalifold: linear-time consensus structure prediction for rna   alignments',\n",
       "  'id': '2206.14794',\n",
       "  'abstract': \"predicting the consensus structure of a set of aligned rna homologs is a convenient method to find conserved structures in an rna genome, which has many applications including viral diagnostics and therapeutics. however, the most commonly used tool for this task, rnaalifold, is prohibitively slow for long sequences, due to a cubic scaling with the sequence length, taking over a day on 400 sars-cov-2 and sars-related genomes (~30,000nt). we present linearalifold, a much faster alternative that scales linearly with both the sequence length and the number of sequences, based on our work linearfold that folds a single rna in linear time. our work is orders of magnitude faster than rnaalifold (0.7 hours on the above 400 genomes, or ~36$\\\\times$ speedup) and achieves higher accuracies when compared to a database of known structures. more interestingly, linearalifold's prediction on sars-cov-2 correlates well with experimentally determined structures, substantially outperforming rnaalifold. finally, linearalifold supports two energy models (vienna and bl*) and four modes: minimum free energy (mfe), maximum expected accuracy (mea), threshknot, and stochastic sampling, each of which takes under an hour for hundreds of sars-cov variants. our resource is at: https://github.com/linearfold/linearalifold (code) and http://linearfold.org/linear-alifold (server).\",\n",
       "  'categories': 'q-bio.bm cs.ds physics.bio-ph q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-06-29',\n",
       "  'updated': '2024-07-05',\n",
       "  'authors': ['apoorv malik',\n",
       "   'liang zhang',\n",
       "   'milan gautam',\n",
       "   'ning dai',\n",
       "   'sizhen li',\n",
       "   'he zhang',\n",
       "   'david h. mathews',\n",
       "   'liang huang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2206.14794'},\n",
       " {'title': 'runtime analysis of competitive co-evolutionary algorithms for maximin   optimisation of a bilinear function',\n",
       "  'id': '2206.15238',\n",
       "  'abstract': 'co-evolutionary algorithms have a wide range of applications, such as in hardware design, evolution of strategies for board games, and patching software bugs. however, these algorithms are poorly understood and applications are often limited by pathological behaviour, such as loss of gradient, relative over-generalisation, and mediocre objective stasis. it is an open challenge to develop a theory that can predict when co-evolutionary algorithms find solutions efficiently and reliable.   this paper provides a first step in developing runtime analysis for population-based competitive co-evolutionary algorithms. we provide a mathematical framework for describing and reasoning about the performance of co-evolutionary processes. an example application of the framework shows a scenario where a simple co-evolutionary algorithm obtains a solution in polynomial expected time. finally, we describe settings where the co-evolutionary algorithm needs exponential time with overwhelmingly high probability to obtain a solution.',\n",
       "  'categories': 'cs.ne q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2022-06-30',\n",
       "  'updated': '2024-03-16',\n",
       "  'authors': ['per kristian lehre'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2206.15238'},\n",
       " {'title': 'embedding phylogenetic trees in networks of low treewidth',\n",
       "  'id': '2207.00574',\n",
       "  'abstract': 'given a rooted, binary phylogenetic network and a rooted, binary phylogenetic tree, can the tree be embedded into the network? this problem, called \\\\textsc{tree containment}, arises when validating networks constructed by phylogenetic inference methods.we present the first algorithm for (rooted) \\\\textsc{tree containment} using the treewidth $t$ of the input network $n$ as parameter, showing that the problem can be solved in $2^{o(t^2)}\\\\cdot|n|$ time and space.',\n",
       "  'categories': 'q-bio.pe cs.dm',\n",
       "  'doi': '10.46298/dmtcs.10116',\n",
       "  'created': '2022-07-01',\n",
       "  'updated': '2023-09-19',\n",
       "  'authors': ['leo van iersel', 'mark jones', 'mathias weller'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.00574'},\n",
       " {'title': 'linguistically inspired roadmap for building biologically reliable   protein language models',\n",
       "  'id': '2207.00982',\n",
       "  'abstract': 'deep neural-network-based language models (lms) are increasingly applied to large-scale protein sequence data to predict protein function. however, being largely black-box models and thus challenging to interpret, current protein lm approaches do not contribute to a fundamental understanding of sequence-function mappings, hindering rule-based biotherapeutic drug development. we argue that guidance drawn from linguistics, a field specialized in analytical rule extraction from natural language data, can aid with building more interpretable protein lms that are more likely to learn relevant domain-specific rules. differences between protein sequence data and linguistic sequence data require the integration of more domain-specific knowledge in protein lms compared to natural language lms. here, we provide a linguistics-based roadmap for protein lm pipeline choices with regard to training data, tokenization, token embedding, sequence embedding, and model interpretation. incorporating linguistic ideas into protein lms enables the development of next-generation interpretable machine-learning models with the potential of uncovering the biological mechanisms underlying sequence-function relationships.',\n",
       "  'categories': 'q-bio.qm cs.lg',\n",
       "  'doi': '10.1038/s42256-023-00637-1',\n",
       "  'created': '2022-07-03',\n",
       "  'updated': '2023-04-28',\n",
       "  'authors': ['mai ha vu',\n",
       "   'rahmad akbar',\n",
       "   'philippe a. robert',\n",
       "   'bartlomiej swiatczak',\n",
       "   'victor greiff',\n",
       "   'geir kjetil sandve',\n",
       "   'dag trygve truslew haug'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.00982'},\n",
       " {'title': 'interpretable fusion analytics framework for fmri connectivity:   self-attention mechanism and latent space item-response model',\n",
       "  'id': '2207.01581',\n",
       "  'abstract': 'there have been several attempts to use deep learning based on brain fmri signals to classify cognitive impairment diseases. however, deep learning is a hidden black box model that makes it difficult to interpret the process of classification. to address this issue, we propose a novel analytical framework that interprets the classification result from deep learning processes. we first derive the region of interest (roi) functional connectivity network (fcn) by embedding functions based on their similar signal patterns. then, using the self-attention equipped deep learning model, we classify diseases based on their fcn. finally, in order to interpret the classification results, we employ a latent space item-response interaction network model to identify the significant functions that exhibit distinct connectivity patterns when compared to other diseases. the application of this proposed framework to the four types of cognitive impairment shows that our approach is valid for determining the significant roi functions.',\n",
       "  'categories': 'cs.lg cs.ai eess.sp q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-07-04',\n",
       "  'updated': '2024-11-19',\n",
       "  'authors': ['jeong-jae kim',\n",
       "   'yeseul jeon',\n",
       "   'sumin yu',\n",
       "   'junggu choi',\n",
       "   'sanghoon han'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.01581'},\n",
       " {'title': 'tricritical behavior in a neural model with excitatory and inhibitory   units',\n",
       "  'id': '2207.02320',\n",
       "  'abstract': 'while the support for the relevance of critical dynamics to brain function is increasing, there is much less agreement on the exact nature of the advocated critical point. thus, a considerable number of theoretical efforts are currently concentrated on which mechanisms and what type/s of transition can be exhibited by neuronal networks models. in that direction, the present work describes the effect of incorporating a fraction of inhibitory neurons on the collective dynamics. as we show, this results in the appearence of a tricritical point for highly connected networks and non-zero fraction of inhibitory neurons. we discuss the relation of the present results with relevant experimental evidence.',\n",
       "  'categories': 'cond-mat.dis-nn q-bio.nc',\n",
       "  'doi': '10.1103/physreve.106.054140',\n",
       "  'created': '2022-07-05',\n",
       "  'updated': '',\n",
       "  'authors': ['joaquin almeira',\n",
       "   'tomas s. grigera',\n",
       "   'dante r. chialvo',\n",
       "   'sergio a. cannas'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.02320'},\n",
       " {'title': 'the coalescent in finite populations with arbitrary, fixed structure',\n",
       "  'id': '2207.02880',\n",
       "  'abstract': \"the coalescent is a stochastic process representing ancestral lineages in a population undergoing neutral genetic drift. originally defined for a well-mixed population, the coalescent has been adapted in various ways to accommodate spatial, age, and class structure, along with other features of real-world populations. to further extend the range of population structures to which coalescent theory applies, we formulate a coalescent process for a broad class of neutral drift models with arbitrary -- but fixed -- spatial, age, sex, and class structure, haploid or diploid genetics, and any fixed mating pattern. here, the coalescent is represented as a random sequence of mappings $\\\\mathcal{c} = \\\\left(c_t\\\\right)_{t=0}^\\\\infty$ from a finite set $g$ to itself. the set $g$ represents the ``sites'' (in individuals, in particular locations and/or classes) at which these alleles can live. the state of the coalescent, $c_t:g \\\\to g$, maps each site $g \\\\in g$ to the site containing $g$'s ancestor, $t$ time-steps into the past. using this representation, we define and analyze coalescence time, coalescence branch length, mutations prior to coalescence, and stationary probabilities of identity-by-descent and identity-by-state. for low mutation, we provide a recipe for computing identity-by-descent and identity-by-state probabilities via the coalescent. applying our results to a diploid population with arbitrary sex ratio $r$, we find that measures of genetic dissimilarity, among any set of sites, are scaled by $4r(1-r)$ relative to the even sex ratio case.\",\n",
       "  'categories': 'q-bio.pe math.pr',\n",
       "  'doi': '10.1016/j.tpb.2024.06.004',\n",
       "  'created': '2022-07-06',\n",
       "  'updated': '2024-06-28',\n",
       "  'authors': ['benjamin allen', 'alex mcavoy'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.02880'},\n",
       " {'title': 'a stochastic mobility-driven spatially explicit seiqrd covid-19 model   with vocs, seasonality, and vaccines',\n",
       "  'id': '2207.03717',\n",
       "  'abstract': \"in this work, we extend our previously developed compartmental seiqrd model for sars-cov-2 in belgium. we introduce sars-cov-2 variants of concern, vaccines, and seasonality in our model, as their addition has proven necessary for modelling sars-cov-2 transmission dynamics during the 2020-2021 covid-19 pandemic in belgium. the model is geographically stratified into eleven spatial patches (provinces), and a telecommunication dataset provided by belgium's biggest operator is used to incorporate interprovincial mobility. we calibrate the model using the daily number of hospitalisations in each province and serological data. we find the model adequately describes these data, but the addition of interprovincial mobility was not necessary to obtain an accurate description of the 2020-2021 sars-cov-2 pandemic in belgium. we further demonstrate how our model can be used to help policymakers decide on the optimal timing of the release of social restrictions. we find that adding spatial heterogeneity by geographically stratifying the model results in more uncertain model projections as compared to an equivalent nation-level model, which has both communicative advantages and disadvantages. we finally discuss the impact of imposing local mobility or social contact restrictions to contain an epidemic in a given province and find that lowering social contact is a more effective strategy than lowering mobility.\",\n",
       "  'categories': 'physics.soc-ph physics.bio-ph q-bio.pe',\n",
       "  'doi': '10.1016/j.apm.2023.06.027',\n",
       "  'created': '2022-07-08',\n",
       "  'updated': '2023-06-26',\n",
       "  'authors': ['tijs w. alleman',\n",
       "   'michiel rollier',\n",
       "   'jenna vergeynst',\n",
       "   'jan m. baetens'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.03717'},\n",
       " {'title': 'fast solver for diffusive transport times on dynamic intracellular   networks',\n",
       "  'id': '2207.07682',\n",
       "  'abstract': 'the transport of particles in cells is influenced by the properties of intracellular networks they traverse while searching for localized target regions or reaction partners. moreover, given the rapid turnover in many intracellular structures, it is crucial to understand how temporal changes in the network structure affect diffusive transport. in this work, we use network theory to characterize complex intracellular biological environments across scales. we develop an efficient computational method to compute the mean first passage times for simulating a particle diffusing along two-dimensional planar networks extracted from fluorescence microscopy imaging. we first benchmark this methodology in the context of synthetic networks, and subsequently apply it to live-cell data from endoplasmic reticulum tubular networks.',\n",
       "  'categories': 'physics.bio-ph cond-mat.soft q-bio.qm q-bio.sc',\n",
       "  'doi': '10.1137/22m1509308',\n",
       "  'created': '2022-07-15',\n",
       "  'updated': '2023-11-14',\n",
       "  'authors': ['lachlan elam',\n",
       "   'mónica c. quiñones-frías',\n",
       "   'ying zhang',\n",
       "   'avital a. rodal',\n",
       "   'thomas g. fai'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.07682'},\n",
       " {'title': \"kirchhoff's current law with displacement current\",\n",
       "  'id': '2207.08277',\n",
       "  'abstract': \"kirchhoff's current law is an essential tool in the design of circuits that operate very quickly, faster than nanoseconds. but kirchhoff's current is often identified as the flow of particles. the continuity equation or the maxwell-ampere law shows that the sum of displacement current $\\\\textbf{plus}$ particle current is conserved by maxwell's equations and kirchhoff's law. kirchoff included the displacement current in the current of his law, from early on. this kirchhoff current (including the displacement current) does not vary with spatial location in the ionic channels of biology. electronic circuits switching in nanoseconds are analyzed using the bohm representation of quantum mechanics including particle and displacement current.\",\n",
       "  'categories': 'physics.class-ph q-bio.qm quant-ph',\n",
       "  'doi': '',\n",
       "  'created': '2022-07-17',\n",
       "  'updated': '',\n",
       "  'authors': ['robert eisenberg', 'xavier oriols', 'david k. ferry'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.08277'},\n",
       " {'title': 'population dynamics under demographic and environmental stochasticity',\n",
       "  'id': '2207.08883',\n",
       "  'abstract': 'the present paper is devoted to the study of the long term dynamics of diffusion processes modelling a single species that experiences both demographic and environmental stochasticity. in our setting, the long term dynamics of the diffusion process in the absence of demographic stochasticity is determined by the sign of $\\\\lambda_0$, the external lyapunov exponent, as follows: $\\\\lambda_0<0$ implies (asymptotic) extinction and $\\\\lambda_0>0$ implies convergence to a unique positive stationary distribution $\\\\mu_0$. if the system is of size $\\\\frac{1}{\\\\epsilon^{2}}$ for small $\\\\epsilon>0$ (the intensity of demographic stochasticity), demographic effects will make the extinction time finite almost surely. this suggests that to understand the dynamics one should analyze the quasi-stationary distribution (qsd) $\\\\mu_\\\\epsilon$ of the system. the existence and uniqueness of the qsd is well-known under mild assumptions.   we look at what happens when the population size is sent to infinity, i.e., when $\\\\epsilon\\\\to 0$. we show that the external lyapunov exponent still plays a key role: 1) if $\\\\lambda_0<0$, then $\\\\mu_\\\\epsilon\\\\to \\\\delta_0$, the mean extinction time is of order $|\\\\ln \\\\epsilon|$ and the extinction rate associated with the qsd $\\\\mu_{\\\\epsilon}$ has a lower bound of order $\\\\frac{1}{|\\\\ln\\\\epsilon|}$; 2) if $\\\\lambda_0>0$, then $\\\\mu_\\\\epsilon\\\\to \\\\mu_0$, the mean extinction time is polynomial in $\\\\frac{1}{\\\\epsilon^{2}}$ and the extinction rate is polynomial in $\\\\epsilon^{2}$. furthermore, when $\\\\lambda_0>0$ we are able to show that the system exhibits multiscale dynamics: at first the process quickly approaches the qsd $\\\\mu_\\\\epsilon$ and then, after spending a polynomially long time there, it relaxes to the extinction state. we give sharp asymptotics in $\\\\epsilon$ for the time spent close to $\\\\mu_\\\\epsilon$.',\n",
       "  'categories': 'math.pr math.ds math.sp q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2022-07-18',\n",
       "  'updated': '2024-07-08',\n",
       "  'authors': ['alexandru hening', 'weiwei qi', 'zhongwei shen', 'yingfei yi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.08883'},\n",
       " {'title': 'a mechanistic model to assess the effectiveness of   test-trace-isolate-and-quarantine under limited capacities',\n",
       "  'id': '2207.09551',\n",
       "  'abstract': 'diagnostic testing followed by isolation of identified cases with subsequent tracing and quarantine of close contacts - often referred to as test-trace-isolate-and-quarantine (ttiq) strategy - is one of the cornerstone measures of infectious disease control. the covid-19 pandemic has highlighted that an appropriate response to outbreaks requires us to be aware about the effectiveness of such containment strategies. this can be evaluated using mathematical models. we present a delay differential equation model of ttiq interventions for infectious disease control. our model incorporates a detailed mechanistic description of the state-dependent dynamics induced by limited ttiq capacities. in addition, we account for transmission during the early phase of sars-cov-2 infection, including presymptomatic transmission, which may be particularly adverse to a ttiq based control. numerical experiments, inspired by the early spread of covid-19 in germany, reveal the effectiveness of ttiq in a scenario where immunity within the population is low and pharmaceutical interventions are absent - representative of a typical situation during the (re-)emergence of infectious diseases for which therapeutic drugs or vaccines are not yet available. stability and sensitivity analyses emphasize factors, partially related to the specific disease, which impede or enhance the success of ttiq. studying the diminishing effectiveness of ttiq along simulations of an epidemic wave we highlight consequences for intervention strategies.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1371/journal.pone.0299880',\n",
       "  'created': '2022-07-19',\n",
       "  'updated': '2022-11-23',\n",
       "  'authors': ['julian heidecke', 'jan fuhrmann', 'maria vittoria barbarossa'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.09551'},\n",
       " {'title': 'neuromorphic networks as revealed by features similarity',\n",
       "  'id': '2207.10571',\n",
       "  'abstract': 'the study of neuronal morphology is important not only for its potential relationship with neuronal dynamics, but also as a means to classify diverse types of cells and compare than among species, organs, and conditions. in the present work, we approach this interesting problem by using the concept of coincidence similarity, as well as a respectively derived method for mapping datasets into networks. the coincidence similarity has been found to allow some specific interesting properties which have allowed enhanced performance (selectivity and sensitivity) concerning several pattern recognition tasks. several combinations of 20 morphological features were considered, and the respective networks were obtained by maximizing the literal modularity (in supervised manner) respectively to the involved parameters. well-separated groups were obtained that provide a rich representation of the main similarity interrelationships between the 735 considered neuronal cells. a sequence of network configurations illustrating the progressive merging between cells and groups was also obtained by varying one of the coincidence parameters.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-06-13',\n",
       "  'updated': '2024-03-08',\n",
       "  'authors': ['alexandre benatti',\n",
       "   'henrique f. de arruda',\n",
       "   'luciano da f. costa'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.10571'},\n",
       " {'title': 'communities in c.elegans connectome through the prism of   non-backtracking walks',\n",
       "  'id': '2207.11282',\n",
       "  'abstract': 'the fundamental relationship between the mesoscopic structure of neuronal circuits and organismic functions they subserve is one of the major challenges in contemporary neuroscience. formation of structurally connected modules of neurons enacts the conversion from single-cell firing to large-scale behaviour of an organism, highlighting the importance of their accurate profiling in the data. while connectomes are typically characterized by significant sparsity of neuronal connections, recent advances in network theory and machine learning have revealed fundamental limitations of traditionally used community detection approaches in cases where the network is sparse. here we studied the optimal community structure in the structural connectome of c.elegans, for which we exploited a non-conventional approach that is based on non-backtracking random walks, virtually eliminating the sparsity issue. in full agreement with the previous asymptotic results, we demonstrated that non-backtracking walks resolve the ground truth annotation into clusters on stochastic block models (sbm) with the size and density of the connectome better than the spectral methods related to simple random walks. based on the cluster detectability threshold, we determined that the optimal number of modules in a recently mapped connectome of c.elegans is 10, which precisely corresponds to the number of isolated eigenvalues in the spectrum of the non-backtracking flow matrix. broadly, our work provides a robust network-based framework to reveal mesoscopic structures in sparse connectomic datasets, paving way to further investigation of connectome mechanisms for different functions.',\n",
       "  'categories': 'q-bio.nc physics.bio-ph physics.data-an q-bio.qm',\n",
       "  'doi': '10.1038/s41598-023-49503-5',\n",
       "  'created': '2022-07-22',\n",
       "  'updated': '2024-01-10',\n",
       "  'authors': ['arsenii onuchin',\n",
       "   'alina chernizova',\n",
       "   'mikhail lebedev',\n",
       "   'kirill polovnikov'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.11282'},\n",
       " {'title': 'modeling and analysis of a coupled sis bi-virus model',\n",
       "  'id': '2207.11414',\n",
       "  'abstract': 'the paper deals with the setting where two viruses (say virus 1 and virus 2) coexist in a population, and they are not necessarily mutually exclusive, in the sense that infection due to one virus does not preclude the possibility of simultaneous infection due to the other. we develop a coupled bi-virus susceptible-infected-susceptible (sis) model from a 4n-state markov chain model, where n is the number of agents (i.e., individuals or subpopulation) in the population. we identify a sufficient condition for both viruses to eventually die out, and a sufficient condition for the existence, uniqueness and asymptotic stability of the endemic equilibrium of each virus. we establish a sufficient condition and multiple necessary conditions for local exponential convergence to the boundary equilibrium (i.e., one virus persists, the other one dies out) of each virus. under mild assumptions on the healing rate, we show that there cannot exist a coexisting equilibrium where for each node there is a nonzero fraction infected only by virus 1; a nonzero fraction infected only by virus 2; but no fraction that is infected by both viruses 1 and 2. likewise, assuming that healing rates are strictly positive, a coexisting equilibrium where for each node there is a nonzero fraction infected by both viruses 1 and 2, but no fraction is infected only by virus 1 (resp. virus 2) does not exist. further, we provide a necessary condition for the existence of certain other kinds of coexisting equilibria. we show that, unlike the competitive bivirus model, the coupled bivirus model is not monotone. finally, we illustrate our theoretical findings using an extensive set of in-depth simulations.',\n",
       "  'categories': 'q-bio.pe cs.sy eess.sy',\n",
       "  'doi': '',\n",
       "  'created': '2022-07-23',\n",
       "  'updated': '2024-07-10',\n",
       "  'authors': ['sebin gracy',\n",
       "   'philip e. paré',\n",
       "   'ji liu',\n",
       "   'henrik sandberg',\n",
       "   'carolyn l. beck',\n",
       "   'karl henrik johansson',\n",
       "   'tamer başar'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.11414'},\n",
       " {'title': 'projecting social contact matrices to populations stratified by binary   attributes with known homophily',\n",
       "  'id': '2207.12328',\n",
       "  'abstract': 'contact networks are heterogeneous. people with similar characteristics are more likely to interact, a phenomenon called assortative mixing or homophily. while age-assortativity is well-established and social contact matrices for populations stratified by age have been derived through extensive survey work, we lack empirical studies that describe contact patterns of a population stratified by other attributes such as gender, sexual orientation, ethnicity, etc. accounting for heterogeneities with respect to these attributes can have a profound effect on the dynamics of epidemiological forecasting models.   here, we introduce a new methodology to expand a given e.g. age-based contact matrix to populations stratified by binary attributes with a known level of homophily. we describe a set of linear conditions any meaningful social contact matrix must satisfy and find the optimal matrix by solving a non-linear optimization problem. we show the effect homophily can have on disease dynamics and conclude by briefly describing more complicated extensions.   the available python source code enables any modeler to account for the presence of homophily with respect to binary attributes in contact patterns, ultimately yielding more accurate predictive models.',\n",
       "  'categories': 'physics.soc-ph math.ds q-bio.qm',\n",
       "  'doi': '10.3934/mbe.2023154',\n",
       "  'created': '2022-07-25',\n",
       "  'updated': '',\n",
       "  'authors': ['claus kadelka'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2207.12328'},\n",
       " {'title': 'stochastic failure of cell infection post viral entry: implications for   infection outcomes and antiviral therapy',\n",
       "  'id': '2208.00637',\n",
       "  'abstract': \"a virus infection can be initiated with very few or even a single infectious virion, and as such can become extinct, i.e. stochastically fail to take hold or spread significantly. there are many ways that a fully competent infectious virion, having successfully entered a cell, can fail to cause a productive infection, i.e. one that yields infectious virus progeny. though many discrete, stochastic mathematical models (dsms) have been developed and used to estimate a virus infection's extinction probability, these typically neglect infection failure post viral entry. the dsm presented herein introduces parameter $\\\\gamma\\\\in(0,1]$ which corresponds to the probability that a virion's entry into a cell will result in a productive cell infection. we derive an expression for the likelihood of infection extinction in this new dsm, and find that prophylactic therapy with an antiviral acting to reduce $\\\\gamma$ is best at increasing an infection's extinction probability, compared to antivirals acting on the rates of virus production or virus entry into cells. using the dsm, we investigate the difference in the fraction of cells consumed by so-called extinct versus established virus infections, and find that this distinction becomes biologically meaningless as the probability of extinction approaches 100%. we show that infections wherein virus is release by an infected cell as a single burst, rather than at a constant rate over the cell's infectious lifespan, has the same probability of infection extinction, despite previous claims to this effect [pearson 2011, doi:10.1371/journal.pcbi.1001058]. instead, extending previous work by others [yan 2016, doi:10.1007/s00285-015-0961-5], we show how the assumed distribution for the stochastic virus burst size, affects the extinction probability and associated critical antiviral efficacy.\",\n",
       "  'categories': 'q-bio.cb q-bio.qm',\n",
       "  'doi': '10.1038/s41598-023-44180-w',\n",
       "  'created': '2022-08-01',\n",
       "  'updated': '',\n",
       "  'authors': ['christian quirouette',\n",
       "   'daniel cresta',\n",
       "   'jizhou li',\n",
       "   'kathleen p. wilkie',\n",
       "   'haozhao liang',\n",
       "   'catherine a. a. beauchemin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2208.00637'},\n",
       " {'title': 'sfiles 2.0: an extended text-based flowsheet representation',\n",
       "  'id': '2208.00778',\n",
       "  'abstract': \"sfiles is a text-based notation for chemical process flowsheets. it was originally proposed by d'anterroches (2006) who was inspired by the text-based smiles notation for molecules. the text-based format has several advantages compared to flowsheet images regarding the storage format, computational accessibility, and eventually for data analysis and processing. however, the original sfiles version cannot describe essential flowsheet configurations unambiguously, such as the distinction between top and bottom products. neither is it capable of describing the control structure required for the safe and reliable operation of chemical processes. also, there is no publicly available software for decoding or encoding chemical process topologies to sfiles. we propose the sfiles 2.0 with a complete description of the extended notation and naming conventions. additionally, we provide open-source software for the automated conversion between flowsheet graphs and sfiles 2.0 strings. this way, we hope to encourage researchers and engineers to publish their flowsheet topologies as sfiles 2.0 strings. the ultimate goal is to set the standards for creating a fair database of chemical process flowsheets, which would be of great value for future data analysis and processing.\",\n",
       "  'categories': 'cs.db cs.lg q-bio.qm',\n",
       "  'doi': '10.1007/s11081-023-09798-9',\n",
       "  'created': '2022-07-25',\n",
       "  'updated': '',\n",
       "  'authors': ['gabriel vogel',\n",
       "   'lukas schulze balhorn',\n",
       "   'edwin hirtreiter',\n",
       "   'artur m. schweidtmann'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2208.00778'},\n",
       " {'title': 'using instruments for selection to adjust for selection bias in   mendelian randomization',\n",
       "  'id': '2208.02657',\n",
       "  'abstract': \"selection bias is a common concern in epidemiologic studies. in the literature, selection bias is often viewed as a missing data problem. popular approaches to adjust for bias due to missing data, such as inverse probability weighting, rely on the assumption that data are missing at random and can yield biased results if this assumption is violated. in observational studies with outcome data missing not at random, heckman's sample selection model can be used to adjust for bias due to missing data. in this paper, we review heckman's method and a similar approach proposed by tchetgen tchetgen and wirth (2017). we then discuss how to apply these methods to mendelian randomization analyses using individual-level data, with missing data for either the exposure or outcome or both. we explore whether genetic variants associated with participation can be used as instruments for selection. we then describe how to obtain missingness-adjusted wald ratio, two-stage least squares and inverse variance weighted estimates. the two methods are evaluated and compared in simulations, with results suggesting that they can both mitigate selection bias but may yield parameter estimates with large standard errors in some settings. in an illustrative real-data application, we investigate the effects of body mass index on smoking using data from the avon longitudinal study of parents and children.\",\n",
       "  'categories': 'stat.me q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-08-04',\n",
       "  'updated': '2024-04-13',\n",
       "  'authors': ['apostolos gkatzionis',\n",
       "   'eric j. tchetgen tchetgen',\n",
       "   'jon heron',\n",
       "   'kate northstone',\n",
       "   'kate tilling'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2208.02657'},\n",
       " {'title': 'algebraic experimental design: theory and computation',\n",
       "  'id': '2208.02726',\n",
       "  'abstract': 'over the past several decades, algebraic geometry has provided innovative approaches to biological experimental design that resolved theoretical questions and improved computational efficiency. however, guaranteeing uniqueness and perfect recovery of models are still open problems. in this work we study the problem of uniqueness of wiring diagrams. we use as a modeling framework polynomial dynamical systems and utilize the correspondence between simplicial complexes and square-free monomial ideals from stanley-reisner theory to develop theory and construct an algorithm for identifying input data sets $v\\\\subset \\\\mathbb f_p^n$ that are guaranteed to correspond to a unique minimal wiring diagram regardless of the experimental output. we apply the results on a tumor-suppression network mediated by epidermal derived growth factor receptor and demonstrate how careful experimental design decisions can lead to a unique minimal wiring diagram identification. one of the insights of the theoretical work is the connection between the uniqueness of a wiring diagram for a given $v\\\\subset \\\\mathbb f_p^n$ and the uniqueness of the reduced gr\\\\\"obner basis of the polynomial ideal $i(v)\\\\subset \\\\mathbb f_p[x_1,\\\\ldots, x_n]$. we discuss existing results and introduce a new necessary condition on the points in $v$ for uniqueness of the reduced gr\\\\\"obner basis of $i(v)$. these results also point to the importance of the relative proximity of the experimental input points on the number of minimal wiring diagrams, which we then study computationally. we find that there is a concrete heuristic way to generate data that tends to result in fewer minimal wiring diagrams.',\n",
       "  'categories': 'math.ag q-bio.mn',\n",
       "  'doi': '10.1137/22m1513241',\n",
       "  'created': '2022-08-04',\n",
       "  'updated': '',\n",
       "  'authors': ['elena s. dimitrova',\n",
       "   'cameron h. fredrickson',\n",
       "   'nicholas a. rondoni',\n",
       "   'brandilyn stigler',\n",
       "   'alan veliz-cuba'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2208.02726'},\n",
       " {'title': 'graph neural networks for the prediction of molecular structure-property   relationships',\n",
       "  'id': '2208.04852',\n",
       "  'abstract': 'molecular property prediction is of crucial importance in many disciplines such as drug discovery, molecular biology, or material and process design. the frequently employed quantitative structure-property/activity relationships (qsprs/qsars) characterize molecules by descriptors which are then mapped to the properties of interest via a linear or nonlinear model. in contrast, graph neural networks, a novel machine learning method, directly work on the molecular graph, i.e., a graph representation where atoms correspond to nodes and bonds correspond to edges. gnns allow to learn properties in an end-to-end fashion, thereby avoiding the need for informative descriptors as in qsprs/qsars. gnns have been shown to achieve state-of-the-art prediction performance on various property predictions tasks and represent an active field of research. we describe the fundamentals of gnns and demonstrate the application of gnns via two examples for molecular property prediction.',\n",
       "  'categories': 'q-bio.bm cs.lg math.oc',\n",
       "  'doi': '10.1039/bk9781837670178-00159',\n",
       "  'created': '2022-07-25',\n",
       "  'updated': '',\n",
       "  'authors': ['jan g. rittig',\n",
       "   'qinghe gao',\n",
       "   'manuel dahmen',\n",
       "   'alexander mitsos',\n",
       "   'artur m. schweidtmann'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2208.04852'},\n",
       " {'title': 'exact low-dimensional description for fast neural oscillations with low   firing rates',\n",
       "  'id': '2208.05515',\n",
       "  'abstract': 'recently, low-dimensional models of neuronal activity have been exactly derived for large networks of deterministic, quadratic integrate-and-fire (qif) neurons. such firing rate models (frm) describe the emergence of fast collective oscillations (>30~hz) via the frequency-locking of a subset of neurons to the global oscillation frequency. however, the suitability of such models to describe realistic neuronal states is seriously challenged by fact that during episodes of fast collective oscillations, neuronal discharges are often very irregular and have low firing rates compared to the global oscillation frequency. here we extend the theory to derive exact frm for qif neurons to include noise, and show that networks of stochastic neurons displaying irregular discharges at low firing rates during episodes of fast oscillations, are governed by exactly the same evolution equations as deterministic networks. our results reconcile two traditionally confronted views on neuronal synchronization, and upgrade the applicability of exact frm to describe a broad range of biologically realistic neuronal states.',\n",
       "  'categories': 'q-bio.nc nlin.ao',\n",
       "  'doi': '10.1103/physreve.109.014229',\n",
       "  'created': '2022-08-10',\n",
       "  'updated': '2024-02-01',\n",
       "  'authors': ['pau clusella', 'ernest montbrió'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2208.05515'},\n",
       " {'title': 'can brain signals reveal inner alignment with human languages?',\n",
       "  'id': '2208.06348',\n",
       "  'abstract': 'brain signals, such as electroencephalography (eeg), and human languages have been widely explored independently for many downstream tasks, however, the connection between them has not been well explored. in this study, we explore the relationship and dependency between eeg and language. to study at the representation level, we introduced \\\\textbf{mtam}, a \\\\textbf{m}ultimodal \\\\textbf{t}ransformer \\\\textbf{a}lignment \\\\textbf{m}odel, to observe coordinated representations between the two modalities. we used various relationship alignment-seeking techniques, such as canonical correlation analysis and wasserstein distance, as loss functions to transfigure features. on downstream applications, sentiment analysis and relation detection, we achieved new state-of-the-art results on two datasets, zuco and k-emocon. our method achieved an f1-score improvement of 1.7% on k-emocon and 9.3% on zuco datasets for sentiment analysis, and 7.4% on zuco for relation detection. in addition, we provide interpretations of the performance improvement: (1) feature distribution shows the effectiveness of the alignment module for discovering and encoding the relationship between eeg and language; (2) alignment weights show the influence of different language semantics as well as eeg frequency features; (3) brain topographical maps provide an intuitive demonstration of the connectivity in the brain regions. our code is available at \\\\url{https://github.com/jason-qiu/eeg_language_alignment}.',\n",
       "  'categories': 'q-bio.nc cs.ai cs.cl cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2022-08-10',\n",
       "  'updated': '2024-05-04',\n",
       "  'authors': ['william han',\n",
       "   'jielin qiu',\n",
       "   'jiacheng zhu',\n",
       "   'mengdi xu',\n",
       "   'douglas weber',\n",
       "   'bo li',\n",
       "   'ding zhao'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2208.06348'},\n",
       " {'title': 'modeling thermal regulation in thin vascular systems: a mathematical   analysis',\n",
       "  'id': '2208.07397',\n",
       "  'abstract': 'mimicking vascular systems in living beings, designers have realized microvascular composites to achieve thermal regulation and other functionalities, such as electromagnetic modulation, sensing, and healing. such material systems avail circulating fluids through embedded vasculatures to accomplish the mentioned functionalities that benefit various aerospace, military, and civilian applications. although heat transfer is a mature field, control of thermal characteristics in synthetic microvascular systems via circulating fluids is new, and a theoretical underpinning is lacking. what will benefit designers are predictive mathematical models and an in-depth qualitative understanding of vascular-based active cooling/heating. so, the central focus of this paper is to address the remarked knowledge gap. \\\\emph{first}, we present a reduced-order model with broad applicability, allowing the inlet temperature to differ from the ambient temperature. \\\\emph{second}, we apply mathematical analysis tools to this reduced-order model and reveal many heat transfer properties of fluid-sequestered vascular systems. we derive point-wise properties (minimum, maximum, and comparison principles) and global properties (e.g., bounds on performance metrics such as the mean surface temperature and thermal efficiency). these newfound results deepen our understanding of active cooling/heating and propel the perfecting of thermal regulation systems.',\n",
       "  'categories': 'math.na cs.na q-bio.to',\n",
       "  'doi': '10.4208/cicp.oa-2022-0240',\n",
       "  'created': '2022-08-15',\n",
       "  'updated': '2024-01-28',\n",
       "  'authors': ['kalyana b. nakshatrala'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2208.07397'},\n",
       " {'title': 'from static to dynamic structures: improving binding affinity prediction   with graph-based deep learning',\n",
       "  'id': '2208.10230',\n",
       "  'abstract': 'accurate prediction of protein-ligand binding affinities is an essential challenge in structure-based drug design. despite recent advances in data-driven methods for affinity prediction, their accuracy is still limited, partially because they only take advantage of static crystal structures while the actual binding affinities are generally determined by the thermodynamic ensembles between proteins and ligands. one effective way to approximate such a thermodynamic ensemble is to use molecular dynamics (md) simulation. here, an md dataset containing 3,218 different protein-ligand complexes is curated, and dynaformer, a graph-based deep learning model is further developed to predict the binding affinities by learning the geometric characteristics of the protein-ligand interactions from the md trajectories. in silico experiments demonstrated that the model exhibits state-of-the-art scoring and ranking power on the casf-2016 benchmark dataset, outperforming the methods hitherto reported. moreover, in a virtual screening on heat shock protein 90 (hsp90) using dynaformer, 20 candidates are identified and their binding affinities are further experimentally validated. dynaformer displayed promising results in virtual drug screening, revealing 12 hit compounds (two are in the submicromolar range), including several novel scaffolds. overall, these results demonstrated that the approach offer a promising avenue for accelerating the early drug discovery process.',\n",
       "  'categories': 'q-bio.bm cs.lg physics.chem-ph q-bio.qm',\n",
       "  'doi': '10.1002/advs.202405404',\n",
       "  'created': '2022-08-19',\n",
       "  'updated': '2024-09-02',\n",
       "  'authors': ['yaosen min',\n",
       "   'ye wei',\n",
       "   'peizhuo wang',\n",
       "   'xiaoting wang',\n",
       "   'han li',\n",
       "   'nian wu',\n",
       "   'stefan bauer',\n",
       "   'shuxin zheng',\n",
       "   'yu shi',\n",
       "   'yingheng wang',\n",
       "   'ji wu',\n",
       "   'dan zhao',\n",
       "   'jianyang zeng'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2208.10230'},\n",
       " {'title': 'prediction of good reaction coordinates and future evolution of md   trajectories using regularized sparse autoencoders: a novel deep learning   approach',\n",
       "  'id': '2208.10962',\n",
       "  'abstract': 'identifying reaction coordinates(rcs) is an active area of research, given the crucial role rcs play in determining the progress of a chemical reaction. the choice of the reaction coordinate is often based on heuristic knowledge. however, an essential criterion for the choice is that the coordinate should capture both the reactant and product states unequivocally. also, the coordinate should be the slowest one so that all the other degrees of freedom can easily equilibrate along the reaction coordinate. also, the coordinate should be the slowest one so that all the other degrees of freedom can easily equilibrate along the reaction coordinate. we used a regularised sparse autoencoder, an energy-based model, to discover a crucial set of reaction coordinates. along with discovering reaction coordinates, our model also predicts the evolution of a molecular dynamics(md) trajectory. we showcased that including sparsity enforcing regularisation helps in choosing a small but important set of reaction coordinates. we used two model systems to demonstrate our approach: alanine dipeptide system and proflavine and dna system, which exhibited intercalation of proflavine into dna minor groove in an aqueous environment. we model md trajectory as a multivariate time series, and our latent variable model performs the task of multi-step time series prediction. this idea is inspired by the popular sparse coding approach - to represent each input sample as a linear combination of few elements taken from a set of representative patterns.',\n",
       "  'categories': 'physics.chem-ph cs.lg q-bio.qm stat.me stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2022-08-22',\n",
       "  'updated': '2024-01-13',\n",
       "  'authors': ['abhijit gupta'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2208.10962'},\n",
       " {'title': 'neural coding as a statistical testing problem',\n",
       "  'id': '2209.00950',\n",
       "  'abstract': 'we take the testing perspective to understand what the minimal discrimination time between two stimuli is for different types of rate coding neurons. our main goal is to describe the testing abilities of two different encoding systems: place cells and grid cells. in particular, we show, through the notion of adaptation, that a fixed place cell system can have a minimum discrimination time that decreases when the stimuli are further away. this could be a considerable advantage for the place cell system that could complement the grid cell system, which is able to discriminate stimuli that are much closer than place cells.',\n",
       "  'categories': 'math.st q-bio.nc stat.th',\n",
       "  'doi': '10.46298/mna.10002',\n",
       "  'created': '2022-09-02',\n",
       "  'updated': '2023-12-27',\n",
       "  'authors': ['guilherme ost', 'patricia reynaud-bouret'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.00950'},\n",
       " {'title': 'stability of linear boolean networks',\n",
       "  'id': '2209.02044',\n",
       "  'abstract': 'stability is an important characteristic of network models that has implications for other desirable aspects such as controllability. the stability of a boolean network depends on various factors, such as the topology of its wiring diagram and the type of the functions describing its dynamics. in this paper, we study the stability of linear boolean networks by computing derrida curves and quantifying the number of attractors and cycle lengths imposed by their network topologies. derrida curves are commonly used to measure the stability of boolean networks and several parameters such as the average in-degree k and the output bias p can indicate if a network is stable, critical, or unstable. for random unbiased boolean networks there is a critical connectivity value kc=2 such that if k<kc networks operate in the ordered regime, and if k>kc networks operate in the chaotic regime. here, we show that for linear networks, which are the least canalizing and most unstable, the phase transition from order to chaos already happens at an average in-degree of kc=1. consistently, we also show that unstable networks exhibit a large number of attractors with very long limit cycles while stable and critical networks exhibit fewer attractors with shorter limit cycles. additionally, we present theoretical results to quantify important dynamical properties of linear networks. first, we present a formula for the proportion of attractor states in linear systems. second, we show that the expected number of fixed points in linear systems is 2, while general boolean networks possess on average one fixed point. third, we present a formula to quantify the number of bijective linear boolean networks and provide a lower bound for the percentage of this type of network.',\n",
       "  'categories': 'nlin.cd cs.dm math.co q-bio.mn',\n",
       "  'doi': '10.1016/j.physd.2023.133775',\n",
       "  'created': '2022-09-05',\n",
       "  'updated': '2023-05-09',\n",
       "  'authors': ['karthik chandrasekhar',\n",
       "   'claus kadelka',\n",
       "   'reinhard laubenbacher',\n",
       "   'david murrugarra'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.02044'},\n",
       " {'title': 'geometric signatures of switching behavior in mechanobiology',\n",
       "  'id': '2209.03324',\n",
       "  'abstract': \"the proteins involved in cells' mechanobiological processes have evolved specialized and surprising responses to applied forces. biochemical transformations that show catch-to-slip switching and force-induced pathway switching serve important functions in cell adhesion, mechano-sensing and signaling, and protein folding. we show that these switching behaviors are generated by singularities in the flow field that describes force-induced deformation of bound and transition states. these singularities allow for a complete characterization of switching mechanisms in 2-dimensional (2d) free energy landscapes, and provide a path toward elucidating novel forms of switching in higher dimensional models. remarkably, the singularity that generates a catch-slip switch occurs in almost every 2d free energy landscape, implying that almost any bond admitting a 2d model will exhibit catch-slip behavior under appropriate force. we apply our analysis to models of p-selectin and antigen extraction to illustrate how these singularities provide an intuitive framework for explaining known behaviors and predicting new behaviors.\",\n",
       "  'categories': 'q-bio.bm',\n",
       "  'doi': '10.1103/physrevresearch.5.023161',\n",
       "  'created': '2022-09-07',\n",
       "  'updated': '2023-02-26',\n",
       "  'authors': ['casey o. barkan', 'robijn f. bruinsma'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.03324'},\n",
       " {'title': 'correlation between gyral size, brain size, and head impact risk across   mammalian species',\n",
       "  'id': '2209.03924',\n",
       "  'abstract': \"a study on primates has established that gyral size is largely independent of overall brain size. building on this, and other research suggesting that brain gyrification may mitigate the effects of head impacts, our study aims to explore potential correlations between gyral size and the risk of head impact across a diverse range of mammalian species. our findings corroborate the idea that gyral sizes are largely independent of brain sizes, especially among species with larger brains, thus extending this observation beyond primates. preliminary evidence also suggests a correlation between an animal's gyral size and its lifestyle, particularly in terms of head-impact risk. for instance, goats, known for their headbutting behaviors, exhibit smaller gyral sizes. in contrast, species such as manatees and dugongs, which typically face lower risks of head impact, have lissencephalic brains. additionally, we explore mechanisms that may explain how narrower gyral sizes could offer protective advantages against head impact. finally, we discuss a possible trade-off associated with gyrencephaly.\",\n",
       "  'categories': 'physics.bio-ph q-bio.qm',\n",
       "  'doi': '10.1016/j.brainres.2024.148768',\n",
       "  'created': '2022-09-08',\n",
       "  'updated': '2024-04-04',\n",
       "  'authors': ['nianqin zhang', 'yongjun zhang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.03924'},\n",
       " {'title': 'variational causal inference',\n",
       "  'id': '2209.05935',\n",
       "  'abstract': \"estimating an individual's potential outcomes under counterfactual treatments is a challenging task for traditional causal inference and supervised learning approaches when the outcome is high-dimensional (e.g. gene expressions, impulse responses, human faces) and covariates are relatively limited. in this case, to construct one's outcome under a counterfactual treatment, it is crucial to leverage individual information contained in its observed factual outcome on top of the covariates. we propose a deep variational bayesian framework that rigorously integrates two main sources of information for outcome construction under a counterfactual treatment: one source is the individual features embedded in the high-dimensional factual outcome; the other source is the response distribution of similar subjects (subjects with the same covariates) that factually received this treatment of interest.\",\n",
       "  'categories': 'stat.ml cs.ai cs.lg math.st q-bio.gn stat.th',\n",
       "  'doi': '',\n",
       "  'created': '2022-09-13',\n",
       "  'updated': '2024-10-22',\n",
       "  'authors': ['yulun wu',\n",
       "   'layne c. price',\n",
       "   'zichen wang',\n",
       "   'vassilis n. ioannidis',\n",
       "   'robert a. barton',\n",
       "   'george karypis'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.05935'},\n",
       " {'title': 'deep learning in a bilateral brain with hemispheric specialization',\n",
       "  'id': '2209.06862',\n",
       "  'abstract': 'the brains of all bilaterally symmetric animals on earth are divided into left and right hemispheres. the anatomy and functionality of the hemispheres have a large degree of overlap, but there are asymmetries, and they specialise in possesses different attributes. other authors have used computational models to mimic hemispheric asymmetries with a focus on reproducing human data on semantic and visual processing tasks. we took a different approach and aimed to understand how dual hemispheres in a bilateral architecture interact to perform well in a given task. we propose a bilateral artificial neural network that imitates lateralisation observed in nature: that the left hemisphere specialises in local features and the right in global features. we used different training objectives to achieve the desired specialisation and tested it on an image classification task with two different cnn backbones: resnet and vgg. our analysis found that the hemispheres represent complementary features that are exploited by a network head that implements a type of weighted attention. the bilateral architecture outperformed a range of baselines of similar representational capacity that do not exploit differential specialisation, with the exception of a conventional ensemble of unilateral networks trained on dual training objectives for local and global features. the results demonstrate the efficacy of bilateralism, contribute to the discussion of bilateralism in biological brains, and the principle may serve as an inductive bias for new ai systems.',\n",
       "  'categories': 'q-bio.nc cs.ai cs.lg cs.ne',\n",
       "  'doi': '',\n",
       "  'created': '2022-09-08',\n",
       "  'updated': '2024-07-10',\n",
       "  'authors': ['chandramouli rajagopalan',\n",
       "   'david rawlinson',\n",
       "   'elkhonon goldberg',\n",
       "   'gideon kowadlo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.06862'},\n",
       " {'title': 'automated 2d and 3d finite element overclosure adjustment and mesh   morphing using generalized regression neural networks',\n",
       "  'id': '2209.06948',\n",
       "  'abstract': 'computer representations of three-dimensional (3d) geometries are crucial for simulating systems and processes in engineering and science. in medicine, and more specifically, biomechanics and orthopaedics, obtaining and using 3d geometries is critical to many workflows. however, while many tools exist to obtain 3d geometries of organic structures, little has been done to make them usable for their intended medical purposes. furthermore, many of the proposed tools are proprietary, limiting their use. this work introduces two novel algorithms based on generalized regression neural networks (grnn) and 4 processes to perform mesh morphing and overclosure adjustment. these algorithms were implemented, and test cases were used to validate them against existing algorithms to demonstrate improved performance. the resulting algorithms demonstrate improvements to existing techniques based on radial basis function (rbf) networks by converting to grnn-based implementations. implementations in matlab of these algorithms and the source code are publicly available at the following locations:   https://github.com/thor-andreassen/femors   https://simtk.org/projects/femors-rbf   https://www.mathworks.com/matlabcentral/fileexchange/120353-finite-element-morphing-overclosure-reduction-and-slicing',\n",
       "  'categories': 'q-bio.qm cs.na math.na',\n",
       "  'doi': '10.1016/j.medengphy.2024.104136',\n",
       "  'created': '2022-09-14',\n",
       "  'updated': '2023-10-09',\n",
       "  'authors': ['thor e. andreassen',\n",
       "   'donald r. hume',\n",
       "   'landon d. hamilton',\n",
       "   'sean e. higinbotham',\n",
       "   'kevin b. shelburne'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.06948'},\n",
       " {'title': 'neuro-evolutionary evidence for a universal fractal primate brain shape',\n",
       "  'id': '2209.08066',\n",
       "  'abstract': 'the cerebral cortex displays a bewildering diversity of shapes and sizes across and within species. despite this diversity, we present a universal multi-scale description of primate cortices. we show that all cortical shapes can be described as a set of nested folds of different sizes. as neighbouring folds are gradually merged, the cortices of 11 primate species follow a common scale-free morphometric trajectory, that also overlaps with over 70 other mammalian species. our results indicate that all cerebral cortices are approximations of the same archetypal fractal shape with a fractal dimension of $d_f=2.5$. importantly, this new understanding enables a more precise quantification of brain morphology as a function of scale. to demonstrate the importance of this new understanding, we show a scale-dependent effect of ageing on brain morphology. we observe a more than four-fold increase in effect size (from 2 standard deviations to 8 standard deviations) at a spatial scale of approximately 2 mm compared to standard morphological analyses. our new understanding may therefore generate superior biomarkers for a range of conditions in the future.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-09-16',\n",
       "  'updated': '2024-07-25',\n",
       "  'authors': ['yujiang wang',\n",
       "   'karoline leiberg',\n",
       "   'nathan kindred',\n",
       "   'christopher r. madan',\n",
       "   'colline poirier',\n",
       "   'christopher i. petkov',\n",
       "   'peter n. taylor',\n",
       "   'bruno c. c. mota'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.08066'},\n",
       " {'title': 'self-organized criticality explains readiness potential',\n",
       "  'id': '2209.09075',\n",
       "  'abstract': 'readiness potential is a widely observed brain activity in several species including crayfish before the spontaneous behavioral initiation. however, it is poorly understood how this spontaneous activity is generated. the hypothesis that some specific, dedicated site is responsible for the spontaneity has been questioned. here, by using intracellular recording and staining of the brain neurons in crayfish and modeling using the sandpile, which is the original model of self-organized criticality (soc), we show that readiness potential can emerge everywhere in the brain because it is a soc system. despite the diversity in neurons and their morphology, brain neurons showed signatures of criticality and readiness potential. we find that the previously known readiness potential in a neuron is a consequence of the critical behavior of the entire network. indeed, seemingly unrelated membrane potential activity in neurons in different animals can shape readiness potential when its time series are averaged after their alignment with respect to the spontaneous behavioral initiation. we show that the sandpile model not made for the potential, can form the premovement buildup activity similar to readiness potential. scaling properties of the synaptic avalanches are in line with those of vertebrate species; thus, not only is the critical brain hypothesis supported in crayfish, but our findings might also provide a unified view of the basis of spontaneity in animal behavior.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-09-19',\n",
       "  'updated': '2024-08-02',\n",
       "  'authors': ['katsushi kagaya', 'tomoyuki kubota', 'kohei nakajima'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.09075'},\n",
       " {'title': 'sgc: a semi-supervised pipeline for gene clustering using self-training   approach in gene co-expression networks',\n",
       "  'id': '2209.10545',\n",
       "  'abstract': 'a widely used approach for extracting information from gene expression data employ the construction of a gene co-expression network and the subsequent application of algorithms that discover network structure. in particular, a common goal is the computational discovery of gene clusters, commonly called modules. when applied on a novel gene expression dataset, the quality of the computed modules can be evaluated automatically, using gene ontology enrichment, a method that measures the frequencies of gene ontology terms in the computed modules and evaluates their statistical likelihood. in this work we propose sgc a novel pipeline for gene clustering based on relatively recent seminal work in the mathematics of spectral network theory. sgc consists of multiple novel steps that enable the computation of highly enriched modules in an unsupervised manner. but unlike all existing frameworks, it further incorporates a novel step that leverages gene ontology information in a semi-supervised clustering method that further improves the quality of the computed modules. comparing with already well-known existing frameworks, we show that sgc results in higher enrichment in real data. in particular, in 12 real gene expression datasets, sgc outperforms in all except one.',\n",
       "  'categories': 'q-bio.gn cs.lg',\n",
       "  'doi': '10.1186/s12859-024-05848-w',\n",
       "  'created': '2022-09-21',\n",
       "  'updated': '',\n",
       "  'authors': ['niloofar aghaieabiane', 'ioannis koutis'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.10545'},\n",
       " {'title': 'a spatial-channel-temporal-fused attention for spiking neural networks',\n",
       "  'id': '2209.10837',\n",
       "  'abstract': 'spiking neural networks (snns) mimic brain computational strategies, and exhibit substantial capabilities in spatiotemporal information processing. as an essential factor for human perception, visual attention refers to the dynamic process for selecting salient regions in biological vision systems. although visual attention mechanisms have achieved great success in computer vision applications, they are rarely introduced into snns. inspired by experimental observations on predictive attentional remapping, we propose a new spatial-channel-temporal-fused attention (sctfa) module that can guide snns to efficiently capture underlying target regions by utilizing accumulated historical spatial-channel information in the present study. through a systematic evaluation on three event stream datasets (dvs gesture, sl-animals-dvs and mnist-dvs), we demonstrate that the snn with the sctfa module (sctfa-snn) not only significantly outperforms the baseline snn (bl-snn) and two other snn models with degenerated attention modules, but also achieves competitive accuracy with existing state-of-the-art methods. additionally, our detailed analysis shows that the proposed sctfa-snn model has strong robustness to noise and outstanding stability when faced with incomplete data, while maintaining acceptable complexity and efficiency. overall, these findings indicate that incorporating appropriate cognitive mechanisms of the brain may provide a promising approach to elevate the capabilities of snns.',\n",
       "  'categories': 'cs.cv cs.ne q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-09-22',\n",
       "  'updated': '2023-05-28',\n",
       "  'authors': ['wuque cai',\n",
       "   'hongze sun',\n",
       "   'rui liu',\n",
       "   'yan cui',\n",
       "   'jun wang',\n",
       "   'yang xia',\n",
       "   'dezhong yao',\n",
       "   'daqing guo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.10837'},\n",
       " {'title': 'visual representations in the human brain are aligned with large   language models',\n",
       "  'id': '2209.11737',\n",
       "  'abstract': 'the human brain extracts complex information from visual inputs, including objects, their spatial and semantic interrelations, and their interactions with the environment. however, a quantitative approach for studying this information remains elusive. here, we test whether the contextual information encoded in large language models (llms) is beneficial for modelling the complex visual information extracted by the brain from natural scenes. we show that llm embeddings of scene captions successfully characterise brain activity evoked by viewing the natural scenes. this mapping captures selectivities of different brain areas, and is sufficiently robust that accurate scene captions can be reconstructed from brain activity. using carefully controlled model comparisons, we then proceed to show that the accuracy with which llm representations match brain representations derives from the ability of llms to integrate complex information contained in scene captions beyond that conveyed by individual words. finally, we train deep neural network models to transform image inputs into llm representations. remarkably, these networks learn representations that are better aligned with brain representations than a large number of state-of-the-art alternative models, despite being trained on orders-of-magnitude less data. overall, our results suggest that llm embeddings of scene captions provide a representational format that accounts for complex information extracted by the brain from visual inputs.',\n",
       "  'categories': 'cs.cv cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-09-23',\n",
       "  'updated': '2024-07-06',\n",
       "  'authors': ['adrien doerig',\n",
       "   'tim c kietzmann',\n",
       "   'emily allen',\n",
       "   'yihan wu',\n",
       "   'thomas naselaris',\n",
       "   'kendrick kay',\n",
       "   'ian charest'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.11737'},\n",
       " {'title': 'immunolingo: linguistics-based formalization of the antibody language',\n",
       "  'id': '2209.12635',\n",
       "  'abstract': 'apparent parallels between natural language and biological sequence have led to a recent surge in the application of deep language models (lms) to the analysis of antibody and other biological sequences. however, a lack of a rigorous linguistic formalization of biological sequence languages, which would define basic components, such as lexicon (i.e., the discrete units of the language) and grammar (i.e., the rules that link sequence well-formedness, structure, and meaning) has led to largely domain-unspecific applications of lms, which do not take into account the underlying structure of the biological sequences studied. a linguistic formalization, on the other hand, establishes linguistically-informed and thus domain-adapted components for lm applications. it would facilitate a better understanding of how differences and similarities between natural language and biological sequences influence the quality of lms, which is crucial for the design of interpretable models with extractable sequence-functions relationship rules, such as the ones underlying the antibody specificity prediction problem. deciphering the rules of antibody specificity is crucial to accelerating rational and in silico biotherapeutic drug design. here, we formalize the properties of the antibody language and thereby establish not only a foundation for the application of linguistic tools in adaptive immune receptor analysis but also for the systematic immunolinguistic studies of immune receptor specificity in general.',\n",
       "  'categories': 'q-bio.qm cs.lg',\n",
       "  'doi': '10.1038/s43588-024-00642-3',\n",
       "  'created': '2022-09-26',\n",
       "  'updated': '2022-11-29',\n",
       "  'authors': ['mai ha vu',\n",
       "   'philippe a. robert',\n",
       "   'rahmad akbar',\n",
       "   'bartlomiej swiatczak',\n",
       "   'geir kjetil sandve',\n",
       "   'dag trygve truslew haug',\n",
       "   'victor greiff'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.12635'},\n",
       " {'title': 'considerations and recommendations from the ismrm diffusion study group   for preclinical diffusion mri: part 1 -- in vivo small-animal imaging',\n",
       "  'id': '2209.12994',\n",
       "  'abstract': 'small-animal diffusion mri (dmri) has been used for methodological development and validation, characterizing the biological basis of diffusion phenomena, and comparative anatomy. the steps from animal setup and monitoring, to acquisition, analysis, and interpretation are complex, with many decisions that may ultimately affect what questions can be answered using the resultant data. this work aims to present selected recommendations and guidelines from the diffusion community, on best practices for preclinical dmri of in vivo animals. we describe the general considerations and foundational knowledge that must be considered when designing experiments. we briefly describe differences in animal species and disease models and discuss why some may be more or less appropriate for different studies. we then give guidelines for in vivo acquisition protocols, including decisions on hardware, animal preparation, and imaging sequences, followed by advice for data processing including pre-processing, model-fitting, and tractography. finally, we provide an online resource which lists publicly available preclinical dmri datasets and software packages, to promote responsible and reproducible research. in each section, we attempt to provide guides and recommendations, but also highlight areas for which no guidelines exist (and why), and where future work should focus. while we mainly cover the central nervous system (on which most preclinical dmri studies are focused), we also provide, where possible and applicable, recommendations for other organs of interest. an overarching goal herein is to enhance the rigor and reproducibility of small animal dmri acquisitions and analyses, and thereby advance biomedical knowledge.',\n",
       "  'categories': 'physics.med-ph q-bio.to',\n",
       "  'doi': '',\n",
       "  'created': '2022-09-26',\n",
       "  'updated': '2024-11-21',\n",
       "  'authors': ['ileana o jelescu',\n",
       "   'francesco grussu',\n",
       "   'andrada ianus',\n",
       "   'brian hansen',\n",
       "   'rachel l c barrett',\n",
       "   'manisha aggarwal',\n",
       "   'stijn michielse',\n",
       "   'fatima nasrallah',\n",
       "   'warda syeda',\n",
       "   'nian wang',\n",
       "   'jelle veraart',\n",
       "   'alard roebroeck',\n",
       "   'andrew f bagdasarian',\n",
       "   'cornelius eichner',\n",
       "   'farshid sepehrband',\n",
       "   'jan zimmermann',\n",
       "   'lucas soustelle',\n",
       "   'christien bowman',\n",
       "   'benjamin c tendler',\n",
       "   'andreea hertanu',\n",
       "   'ben jeurissen',\n",
       "   'marleen verhoye',\n",
       "   'lucio frydman',\n",
       "   'yohan van de looij',\n",
       "   'david hike',\n",
       "   'jeff f dunn',\n",
       "   'karla miller',\n",
       "   'bennett a landman',\n",
       "   'noam shemesh',\n",
       "   'adam anderson',\n",
       "   'emilie mckinnon',\n",
       "   'shawna farquharson',\n",
       "   \"flavio dell' acqua\",\n",
       "   'carlo pierpaoli',\n",
       "   'ivana drobnjak',\n",
       "   'alexander leemans',\n",
       "   'kevin d harkins',\n",
       "   'maxime descoteaux',\n",
       "   'duan xu',\n",
       "   'hao huang',\n",
       "   'mathieu d santin',\n",
       "   'samuel c. grant',\n",
       "   'andre obenaus',\n",
       "   'gene s kim',\n",
       "   'dan wu',\n",
       "   'denis le bihan',\n",
       "   'stephen j blackband',\n",
       "   'luisa ciobanu',\n",
       "   'els fieremans',\n",
       "   'ruiliang bai',\n",
       "   'trygve b leergaard',\n",
       "   'jiangyang zhang',\n",
       "   'tim b dyrby',\n",
       "   'g allan johnson',\n",
       "   'julien cohen-adad',\n",
       "   'matthew d budde',\n",
       "   'kurt g schilling'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.12994'},\n",
       " {'title': 'considerations and recommendations from the ismrm diffusion study group   for preclinical diffusion mri: part 2 -- ex vivo imaging: added value and   acquisition',\n",
       "  'id': '2209.13371',\n",
       "  'abstract': 'the value of preclinical diffusion mri (dmri) is substantial. while dmri enables in vivo non-invasive characterization of tissue, ex vivo dmri is increasingly used to probe tissue microstructure and brain connectivity. ex vivo dmri has several experimental advantages including higher signal-to-noise ratio and spatial resolution compared to in vivo studies, and enabling more advanced diffusion contrasts. another major advantage of ex vivo dmri is the direct comparison with histological data as a methodological validation. however, there are a number of considerations that must be made when performing ex vivo experiments. the steps from tissue preparation, image acquisition and processing, and interpretation of results are complex, with decisions that not only differ dramatically from in vivo imaging of small animals, but ultimately affect what questions can be answered using the data. this work represents \"part 2\" of a 3-part series of recommendations and considerations for preclinical dmri. we describe best practices for dmri of ex vivo tissue, with a focus on the value that ex vivo imaging adds to the field of dmri and considerations in ex vivo image acquisition. we give general considerations and foundational knowledge that must be considered when designing experiments. we describe differences in specimens and models and discuss why some may be more or less appropriate for different studies. we then give guidelines for ex vivo protocols, including tissue fixation, sample preparation, and mr scanning. in each section, we attempt to provide guidelines and recommendations, but also highlight areas for which no guidelines exist (and why), and where future work should lie. an overarching goal herein is to enhance the rigor and reproducibility of ex vivo dmri acquisitions and analyses, and thereby advance biomedical knowledge.',\n",
       "  'categories': 'physics.med-ph q-bio.to',\n",
       "  'doi': '',\n",
       "  'created': '2022-09-27',\n",
       "  'updated': '2024-10-24',\n",
       "  'authors': ['kurt g schilling',\n",
       "   'francesco grussu',\n",
       "   'andrada ianus',\n",
       "   'brian hansen',\n",
       "   'amy fd howard',\n",
       "   'rachel l c barrett',\n",
       "   'manisha aggarwal',\n",
       "   'stijn michielse',\n",
       "   'fatima nasrallah',\n",
       "   'warda syeda',\n",
       "   'nian wang',\n",
       "   'jelle veraart',\n",
       "   'alard roebroeck',\n",
       "   'andrew f bagdasarian',\n",
       "   'cornelius eichner',\n",
       "   'farshid sepehrband',\n",
       "   'jan zimmermann',\n",
       "   'lucas soustelle',\n",
       "   'christien bowman',\n",
       "   'benjamin c tendler',\n",
       "   'andreea hertanu',\n",
       "   'ben jeurissen',\n",
       "   'lucio frydman',\n",
       "   'yohan van de looij',\n",
       "   'david hike',\n",
       "   'jeff f dunn',\n",
       "   'karla miller',\n",
       "   'bennett a landman',\n",
       "   'noam shemesh',\n",
       "   'adam anderson',\n",
       "   'emilie mckinnon',\n",
       "   'shawna farquharson',\n",
       "   \"flavio dell' acqua\",\n",
       "   'carlo pierpaoli',\n",
       "   'ivana drobnjak',\n",
       "   'alexander leemans',\n",
       "   'kevin d harkins',\n",
       "   'maxime descoteaux',\n",
       "   'duan xu',\n",
       "   'hao huang',\n",
       "   'mathieu d santin',\n",
       "   'samuel c. grant',\n",
       "   'andre obenaus',\n",
       "   'gene s kim',\n",
       "   'dan wu',\n",
       "   'denis le bihan',\n",
       "   'stephen j blackband',\n",
       "   'luisa ciobanu',\n",
       "   'els fieremans',\n",
       "   'ruiliang bai',\n",
       "   'trygve leergaard',\n",
       "   'jiangyang zhang',\n",
       "   'tim b dyrby',\n",
       "   'g allan johnson',\n",
       "   'julien cohen-adad',\n",
       "   'matthew d budde',\n",
       "   'ileana o jelescu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.13371'},\n",
       " {'title': 'zero-shot 3d drug design by sketching and generating',\n",
       "  'id': '2209.13865',\n",
       "  'abstract': 'drug design is a crucial step in the drug discovery cycle. recently, various deep learning-based methods design drugs by generating novel molecules from scratch, avoiding traversing large-scale drug libraries. however, they depend on scarce experimental data or time-consuming docking simulation, leading to overfitting issues with limited training data and slow generation speed. in this study, we propose the zero-shot drug design method desert (drug design by sketching and generating). specifically, desert splits the design process into two stages: sketching and generating, and bridges them with the molecular shape. the two-stage fashion enables our method to utilize the large-scale molecular database to reduce the need for experimental data and docking simulation. experiments show that desert achieves a new state-of-the-art at a fast speed.',\n",
       "  'categories': 'cs.ce q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2022-09-28',\n",
       "  'updated': '2022-10-04',\n",
       "  'authors': ['siyu long', 'yi zhou', 'xinyu dai', 'hao zhou'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.13865'},\n",
       " {'title': 'resource competition in three-gene-motif & emergence of feed-forward   response: a spatiotemporal study',\n",
       "  'id': '2209.13886',\n",
       "  'abstract': 'feed-forward dynamics, which is well-known to have several important implications in nonlinear dynamical systems, frequently occurs in gene expression motifs, and has been well explored experimentally and mathematically. however, dependency of the components of a genetic circuit upon its host, due to the requirement for resources like ribosome, atp, transcription factors, trna, etc., and related effects are of utmost importance, which is commonly ignored in mathematical models. in a resource-limited environment, two apparently unconnected genes can compete for resources for their respective expression and may exhibit indirect regulatory connection; an emergent response thus arises in the system completely because of resource competition. in this work, we have shown how the responses of the feed-forward loop (ffl), a well-studied regulatory genetic motif, can be recreated considering the resource competition in a three-gene pathway. exploring the genetic system with temporal as well as spatiotemporal stability analysis, interesting transient and steady-state responses have been observed. the genetic motifs explored in this paper show many of the characteristic features of the conventional ffl structure, like response delay and pulse generation. most interestingly, in a two-dimensional cellular arrangement, characteristic pattern formation under a concentration gradient of input signals have also been observed. this study pinpoints a larger area of research and exploration in synthetic and cellular systems, which will reveal novel controlling ideas and unique behavioral changes in the system for its context dependencies.',\n",
       "  'categories': 'q-bio.mn q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-09-28',\n",
       "  'updated': '2024-01-07',\n",
       "  'authors': ['priya chakraborty', 'ushasi roy', 'sayantari ghosh'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.13886'},\n",
       " {'title': 'using multivariate linear regression for biochemical oxygen demand   prediction in waste water',\n",
       "  'id': '2209.14297',\n",
       "  'abstract': 'there exist opportunities for multivariate linear regression (mlr) in the prediction of biochemical oxygen demand (bod) in waste water, using the diverse water quality parameters as the input variables. the goal of this work is to examine the capability of mlr in prediction of bod in waste water through four input variables: dissolved oxygen (do), nitrogen, fecal coliform and total coliform. the four input variables have higher correlation strength to bod out of the seven parameters examined for the strength of correlation. machine learning (ml) was done with both 80% and 90% of the data as the training set and 20% and 10% as the test set respectively. mlr performance was evaluated through the coefficient of correlation (r), root mean square error (rmse) and the percentage accuracy in prediction of bod. the performance indices for the input variables of dissolved oxygen, nitrogen, fecal coliform and total coliform in prediction of bod are: rmse=6.77mg/l, r=0.60 and accuracy 70.3% for training dataset of 80% and rmse=6.74mg/l, r=0.60 and accuracy of 87.5% for training set of 90% of the dataset. it was found that increasing the percentage of the training set above 80% of the dataset improved the accuracy of the model only but did not have a significant impact on the prediction capacity of the model. the results showed that mlr model could be successfully employed in the estimation of bod in waste water using appropriately selected input parameters.',\n",
       "  'categories': 'q-bio.ot cs.lg stat.ap',\n",
       "  'doi': '10.3934/aci.2024008',\n",
       "  'created': '2022-09-08',\n",
       "  'updated': '',\n",
       "  'authors': ['isaiah k. mutai',\n",
       "   'kristof van laerhoven',\n",
       "   'nancy w. karuri',\n",
       "   'robert k. tewo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.14297'},\n",
       " {'title': 'asymptotic decoupling of population growth rate and cell size   distribution',\n",
       "  'id': '2209.14683',\n",
       "  'abstract': 'the rate at which individual bacterial cells grow depends on the concentrations of cellular components such as ribosomes and proteins. these concentrations continuously fluctuate over time and are inherited from mother to daughter cells, leading to correlations between the growth rates of cells across generations. division sizes of cells are also stochastic and correlated across generations due to a phenomenon known as cell size regulation. fluctuations and correlations from both growth and division processes affect the population dynamics of an exponentially growing culture. here, we provide analytic solutions for the population dynamics of cells with continuously fluctuating growth rates coupled with a generic model of cell-size regulation. we show that in balanced growth, the effects of growth and division processes decouple; the population growth rate only depends on the single-cell growth rate process, and the population cell size distribution only depends on the model of division and cell size regulation. the population growth rate is always higher than the average single-cell growth rate, and the difference increases with growth rate variability and its correlation time. this difference also sets the timescale for the population to reach its steady state. we provide analytical solutions for oscillations in population growth rate and traveling waves in size distribution during this approach to the steady state.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2022-09-29',\n",
       "  'updated': '2024-08-20',\n",
       "  'authors': ['yaïr hein', 'farshid jafarpour'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2209.14683'},\n",
       " {'title': 'physical computing: a category theoretic perspective on physical   computation and system compositionality',\n",
       "  'id': '2210.00392',\n",
       "  'abstract': 'this paper introduces a category theory-based framework to redefine physical computing in light of advancements in quantum computing and non-standard computing systems. by integrating classical definitions within this broader perspective, the paper rigorously recontextualizes what constitutes physical computing devices and processes. it demonstrates how the compositional nature and relational structures of physical computing systems can be coherently formalized using category theory. this approach not only encapsulates recent formalisms in physical computing but also offers a structured method to explore the dynamic interactions within these systems.',\n",
       "  'categories': 'quant-ph cs.ai cs.cl physics.comp-ph q-bio.ot',\n",
       "  'doi': '10.1088/2632-072x/ad6051',\n",
       "  'created': '2022-10-01',\n",
       "  'updated': '2023-12-31',\n",
       "  'authors': ['nima dehghani', 'gianluca caterina'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.00392'},\n",
       " {'title': 'ddos: a graph neural network based drug synergy prediction algorithm',\n",
       "  'id': '2210.00802',\n",
       "  'abstract': 'drug synergy arises when the combined impact of two drugs exceeds the sum of their individual effects. while single-drug effects on cell lines are well-documented, the scarcity of data on drug synergy, considering the vast array of potential drug combinations, prompts a growing interest in computational approaches for predicting synergies in untested drug pairs. we introduce a graph neural network (\\\\textit{gnn}) based model for drug synergy prediction, which utilizes drug chemical structures and cell line gene expression data. we extract data from the largest available drug combination database (drugcomb) and generate multiple synergy scores (commonly used in the literature) to create seven datasets that serve as a reliable benchmark with high confidence. in contrast to conventional models relying on pre-computed chemical features, our gnn-based approach learns task-specific drug representations directly from the graph structure of the drugs, providing superior performance in predicting drug synergies. our work suggests that learning task-specific drug representations and leveraging a diverse dataset is a promising approach to advancing our understanding of drug-drug interaction and synergy.',\n",
       "  'categories': 'q-bio.qm cs.lg q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2022-10-03',\n",
       "  'updated': '2024-04-26',\n",
       "  'authors': ['kyriakos schwarz',\n",
       "   'alicia pliego-mendieta',\n",
       "   'amina mollaysa',\n",
       "   'lara planas-paz',\n",
       "   'chantal pauli',\n",
       "   'ahmed allam',\n",
       "   'michael krauthammer'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.00802'},\n",
       " {'title': 'computing algebraic degrees of phylogenetic varieties',\n",
       "  'id': '2210.02116',\n",
       "  'abstract': 'a phylogenetic variety is an algebraic variety parameterized by a statistical model of the evolution of biological sequences along a tree. understanding this variety is an important problem in the area of algebraic statistics with applications in phylogeny reconstruction. in the broader area of algebra statistics, there have been important theoretical advances in computing certain invariants associated to algebraic varieties arising in applications. beyond the dimension and degree of a variety, one is interested in computing other algebraic degrees, such as the maximum likelihood degree and the euclidean distance degree. despite these efforts, the current literature lacks explicit computations of these invariants for the particular case of phylogenetic varieties. in our work, we fill this gap by computing these invariants for phylogenetic varieties arising from the simplest group-based models of nucleotide substitution cavender-farris-neyman model, jukes-cantor model, kimura 2-parameter model, and the kimura 3-parameter model on small phylogenetic trees with at most 5 leaves.',\n",
       "  'categories': 'q-bio.pe math.ag',\n",
       "  'doi': '10.2140/astat.2023.14.215',\n",
       "  'created': '2022-10-05',\n",
       "  'updated': '2024-02-09',\n",
       "  'authors': ['luis david garcia puente',\n",
       "   'marina garrote-lópez',\n",
       "   'elima shehu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.02116'},\n",
       " {'title': 'optimal classification and generalized prevalence estimates for   diagnostic settings with more than two classes',\n",
       "  'id': '2210.02366',\n",
       "  'abstract': 'an accurate multiclass classification strategy is crucial to interpreting antibody tests. however, traditional methods based on confidence intervals or receiver operating characteristics lack clear extensions to settings with more than two classes. we address this problem by developing a multiclass classification based on probabilistic modeling and optimal decision theory that minimizes the convex combination of false classification rates. the classification process is challenging when the relative fraction of the population in each class, or generalized prevalence, is unknown. thus, we also develop a method for estimating the generalized prevalence of test data that is independent of classification. we validate our approach on serological data with severe acute respiratory syndrome coronavirus 2 (sars-cov-2) na\\\\\"ive, previously infected, and vaccinated classes. synthetic data are used to demonstrate that (i) prevalence estimates are unbiased and converge to true values and (ii) our procedure applies to arbitrary measurement dimensions. in contrast to the binary problem, the multiclass setting offers wide-reaching utility as the most general framework and provides new insight into prevalence estimation best practices.',\n",
       "  'categories': 'q-bio.qm math.oc math.pr physics.bio-ph stat.me',\n",
       "  'doi': '10.1016/j.mbs.2023.108982',\n",
       "  'created': '2022-10-05',\n",
       "  'updated': '',\n",
       "  'authors': ['rayanne a. luke', 'anthony j. kearsley', 'paul n. patrone'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.02366'},\n",
       " {'title': 'quality of life and the experience of context',\n",
       "  'id': '2210.03639',\n",
       "  'abstract': \"i propose that quality of life can be compared despite the difference in values across cultures when it is experienced at the sensory and perceptual level. i argue that an approach to assessing quality of life which focuses on an individual's ability to organize his or her context by perceiving positive constellations of factors in the environment and his or her ability to achieve valuable acts and realize valuable states of being is more meaningful than the approaches of metrics which focus directly, and often solely, on the means of living and the means of freedom. because the felt experience of quality of life is derived from a constellation of factors which make up the indivisible structure of a milieu, the experience of quality of life cannot be regarded as a subjective experience. through the example of how different frequencies, and mixtures of frequencies, of light are perceived as colour by the eye, i demonstrate that the human cognitive apparatus, because of its relation to the object that is measured, apprehends different scales of quantity as degrees of quality. i show that lived experience is the result of a selective relationality with one's environment and that the experience of quality has something to do with the perception of entities in their interrelated and networked nature as wholes.\",\n",
       "  'categories': 'q-bio.nc econ.gn q-fin.ec',\n",
       "  'doi': '',\n",
       "  'created': '2022-09-21',\n",
       "  'updated': '2024-02-04',\n",
       "  'authors': ['ankur betageri'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.03639'},\n",
       " {'title': 'systematizing cellular complexity: a hilbertian approach to biological   problems',\n",
       "  'id': '2210.04627',\n",
       "  'abstract': 'examining individual components of cellular systems has been successful in uncovering molecular reactions and interactions. however, the challenge lies in integrating these components into a comprehensive system-scale map. this difficulty arises due to factors such as missing links (unknown variables), overlooked nonlinearities in high-dimensional parameter space, downplayed natural noisiness and stochasticity, and a lack of focus on causal influence and temporal dynamics. composite static and phenomenological descriptions, while appearing complicated, lack the essence of what makes the biological systems truly \"complex\".   the formalization of system-level problems is therefore important in constructing a meta-theory of biology. addressing fundamental aspects of cellular regulation, adaptability, and noise management is vital for understanding the robustness and functionality of biological systems. these aspects encapsulate the challenges that cells face in maintaining stability, responding to environmental changes, and harnessing noise for functionality. this work examines these key problems that cells must solve, serving as a template for such formalization and as a step towards the axiomatization of biological investigations. through a detailed exploration of cellular mechanisms, particularly homeostatic configuration, ion channels and harnessing noise, this paper aims to illustrate complex concepts and theories in a tangible context, providing a bridge between abstract theoretical frameworks and concrete biological phenomena.',\n",
       "  'categories': 'q-bio.ot physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2022-10-05',\n",
       "  'updated': '2024-07-01',\n",
       "  'authors': ['nima dehghani'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.04627'},\n",
       " {'title': 'cleegn: a convolutional neural network for plug-and-play automatic eeg   reconstruction',\n",
       "  'id': '2210.05988',\n",
       "  'abstract': 'human electroencephalography (eeg) is a brain monitoring modality that senses cortical neuroelectrophysiological activity in high-temporal resolution. one of the greatest challenges posed in applications of eeg is the unstable signal quality susceptible to inevitable artifacts during recordings. to date, most existing techniques for eeg artifact removal and reconstruction are applicable to offline analysis solely, or require individualized training data to facilitate online reconstruction. we have proposed cleegn, a novel convolutional neural network for plug-and-play automatic eeg reconstruction. cleegn is based on a subject-independent pre-trained model using existing data and can operate on a new user without any further calibration. the performance of cleegn was validated using multiple evaluations including waveform observation, reconstruction error assessment, and decoding accuracy on well-studied labeled datasets. the results of simulated online validation suggest that, even without any calibration, cleegn can largely preserve inherent brain activity and outperforms leading online/offline artifact removal methods in the decoding accuracy of reconstructed eeg data. in addition, visualization of model parameters and latent features exhibit the model behavior and reveal explainable insights related to existing knowledge of neuroscience. we foresee pervasive applications of cleegn in prospective works of online plug-and-play eeg decoding and analysis.',\n",
       "  'categories': 'eess.sp cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-10-12',\n",
       "  'updated': '2024-02-20',\n",
       "  'authors': ['pin-hua lai',\n",
       "   'bo-shan wang',\n",
       "   'wei-chun yang',\n",
       "   'hsiang-chieh tsou',\n",
       "   'chun-shu wei'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.05988'},\n",
       " {'title': 'experimental design for multi-channel imaging via task-driven feature   selection',\n",
       "  'id': '2210.06891',\n",
       "  'abstract': 'this paper presents a data-driven, task-specific paradigm for experimental design, to shorten acquisition time, reduce costs, and accelerate the deployment of imaging devices. current approaches in experimental design focus on model-parameter estimation and require specification of a particular model, whereas in imaging, other tasks may drive the design. furthermore, such approaches often lead to intractable optimization problems in real-world imaging applications. here we present a new paradigm for experimental design that simultaneously optimizes the design (set of image channels) and trains a machine-learning model to execute a user-specified image-analysis task. the approach obtains data densely-sampled over the measurement space (many image channels) for a small number of acquisitions, then identifies a subset of channels of prespecified size that best supports the task. we propose a method: tadred for task-driven experimental design in imaging, to identify the most informative channel-subset whilst simultaneously training a network to execute the task given the subset. experiments demonstrate the potential of tadred in diverse imaging applications: several clinically-relevant tasks in magnetic resonance imaging; and remote sensing and physiological applications of hyperspectral imaging. results show substantial improvement over classical experimental design, two recent application-specific methods within the new paradigm, and state-of-the-art approaches in supervised feature selection. we anticipate further applications of our approach. code is available: https://github.com/sbb-gh/experimental-design-multichannel',\n",
       "  'categories': 'cs.lg cs.ai q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-10-13',\n",
       "  'updated': '2024-03-17',\n",
       "  'authors': ['stefano b. blumberg', 'paddy j. slator', 'daniel c. alexander'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.06891'},\n",
       " {'title': 'transition from susceptible-infected to susceptible-infected-recovered   dynamics in a susceptible-cleric-zombie-recovered active matter model',\n",
       "  'id': '2210.07310',\n",
       "  'abstract': \"the susceptible-infected (si) and susceptible-infected-recovered (sir) models provide two distinct representations of epidemic evolution, distinguished by the lack of spontaneous recovery in the si model. here we introduce a new active matter epidemic model, the ``susceptible-cleric-zombie-recovered'' (sczr) model, in which spontaneous recovery is absent but zombies can recover with probability $\\\\gamma$ via interaction with a cleric. upon interacting with a zombie, both susceptibles and clerics can enter the zombie state with probability $\\\\beta$ and $\\\\alpha$, respectively. by changing the intial fraction of clerics or their healing ability rate $\\\\gamma$, we can tune the sczr model between si dynamics, in which no susceptibles or clerics remain at long times, and sir dynamics, in which no zombies remain at long times. the model is relevant to certain real world diseases such as hiv where spontaneous recovery is impossible but where medical interventions by a limited number of caregivers can reduce or eliminate the spread of infection.\",\n",
       "  'categories': 'cond-mat.soft cond-mat.stat-mech q-bio.pe q-bio.qm',\n",
       "  'doi': '10.1103/physreve.107.024604',\n",
       "  'created': '2022-10-13',\n",
       "  'updated': '',\n",
       "  'authors': ['a. libal',\n",
       "   'p. forgacs',\n",
       "   'a. neda',\n",
       "   'c. reichhardt',\n",
       "   'n. hengartner',\n",
       "   'c. j. o. reichhardt'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.07310'},\n",
       " {'title': 'integrative pan-cancer analysis of rnmt: a potential prognostic and   immunological biomarker',\n",
       "  'id': '2210.09574',\n",
       "  'abstract': 'background: rna guanine-7 methyltransferase (rnmt) is one of the main regulators of n7-methylguanosine, and the deregulation of rnmt correlated with tumor development and immune metabolism. however, the specific function of rnmt in pan-cancer remains unclear.   methods: rnmt expression in different cancers was analyzed using multiple databases, including cancer cell line encyclopedia (ccle), genotype-tissue expression project (gtex), and the cancer genome atlas (tcga). cox regression analysis and kaplan-meier analysis were used to estimate the correlation of rnmt expression to prognosis. the data was also used to research the relationship between rnmt expression and common immunoregulators, tumor mutation burden (tmb), microsatellite instability (msi), mismatch repair (mmr), and dna methyltransferase (dnmt). additionally, the cbioportal website was used to evaluate the characteristics of rnmt alteration. the tisdb database was used to obtain the expression of different subtypes. the tumor immune estimation resource (timer) database was used to analyze the association between rnmt and tumor immune infiltration. gene set enrichment analysis (gsea) was used to identify the relevant pathways.   results: rnmt was ubiquitously highly expressed across cancers and survival analysis revealed that its expression was highly associated with the clinical prognosis of various cancer types. remarkably, rnmt participates in immune regulation and plays a crucial part in the tumor microenvironment. a positive association was found between rnmt expression and six immune cell types expression in colon adenocarcinoma, kidney renal clear cell carcinoma, and liver hepatocellular carcinoma. moreover, rnmt expression was highly associated with immunoregulators in most cancer types, and correlated to tmb, msi, mmr, and dnmt. finally, gsea indicated that rnmt may correlate with tumor immunity.',\n",
       "  'categories': 'q-bio.gn',\n",
       "  'doi': '',\n",
       "  'created': '2022-10-18',\n",
       "  'updated': '2024-03-21',\n",
       "  'authors': ['shuqiang huang',\n",
       "   'cuiyu tan',\n",
       "   'jinzhen zheng',\n",
       "   'zhugu huang',\n",
       "   'zhihong li',\n",
       "   'ziyin lv',\n",
       "   'wanru chen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.09574'},\n",
       " {'title': 'phase transition in the computational complexity of the shortest common   superstring and genome assembly',\n",
       "  'id': '2210.09986',\n",
       "  'abstract': \"genome assembly, the process of reconstructing a long genetic sequence by aligning and merging short fragments, or reads, is known to be np-hard, either as a version of the shortest common superstring problem or in a hamiltonian-cycle formulation. that is, the computing time is believed to grow exponentially with the the problem size in the worst case. despite this fact, high-throughput technologies and modern algorithms currently allow bioinformaticians to handle datasets of billions of reads. using methods from statistical mechanics, we address this conundrum by demonstrating the existence of a phase transition in the computational complexity of the problem and showing that practical instances always fall in the 'easy' phase (solvable by polynomial-time algorithms). in addition, we propose a markov-chain monte carlo method that outperforms common deterministic algorithms in the hard regime.\",\n",
       "  'categories': 'cond-mat.stat-mech cs.cc physics.bio-ph q-bio.gn',\n",
       "  'doi': '10.1103/physreve.109.014133',\n",
       "  'created': '2022-10-18',\n",
       "  'updated': '2024-03-11',\n",
       "  'authors': ['l. a. fernandez', 'v. martin-mayor', 'd. yllanes'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.09986'},\n",
       " {'title': 'local intraspecific aggregation in phytoplankton model communities:   spatial scales of occurrence and implications for coexistence',\n",
       "  'id': '2210.11364',\n",
       "  'abstract': \"the coexistence of multiple phytoplankton species despite their reliance on similar resources is often explained with mean-field models assuming mixed populations. in reality, observations of phytoplankton indicate spatial aggregation at all scales, including at the scale of a few individuals. local spatial aggregation can hinder competitive exclusion since individuals then interact mostly with other individuals of their own species, rather than competitors from different species. to evaluate how microscale spatial aggregation might explain phytoplankton diversity maintenance, an individual-based, multispecies representation of cells in a hydrodynamic environment is required. we formulate a three-dimensional and multispecies individual-based model of phytoplankton population dynamics at the kolmogorov scale. the model is studied through both simulations and the derivation of spatial moment equations, in connection with point process theory. the spatial moment equations show a good match between theory and simulations. we parameterized the model based on phytoplankters' ecological and physical characteristics, for both large and small phytoplankton. defining a zone of potential interactions as the overlap between nutrient depletion volumes, we show that local species composition -- within the range of possible interactions -- depends on the size class of phytoplankton. in small phytoplankton, individuals remain in mostly monospecific clusters. spatial structure therefore favours intra- over inter-specific interactions for small phytoplankton, contributing to coexistence. large phytoplankton cell neighbourhoods appear more mixed. although some small-scale self-organizing spatial structure remains and could influence coexistence mechanisms, other factors may need to be explored to explain diversity maintenance in large phytoplankton.\",\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1007/s00285-024-02067-y',\n",
       "  'created': '2022-10-20',\n",
       "  'updated': '2024-04-22',\n",
       "  'authors': ['coralie picoche', 'william r. young', 'frédéric barraquand'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.11364'},\n",
       " {'title': 'the idea of exo-brain',\n",
       "  'id': '2210.11474',\n",
       "  'abstract': 'in this paper i examine the process of getting affected by and the process of making sense of non-language sounds and propose the idea of the contextual cognitive apparatus or exo-brain. we are affected by a singing voice even when we do not fully understand the sounds produced by it. when the listener starts hearing words in the sung song and/or starts interpreting the heard words he recreates the sense and meaning of the song based on a context which either preexists in a culture or gets formed due to his extended engagement with a nebulous auditory stimulus that is perceived as meaningful. this is what i call the work of the creative ear. the affective power of the song comes from an affective reaction between the singing voice, especially the sung non-language sounds, and the linguistic and cultural contexts of the singer and the listener and out this affective reaction something emerges which constitutes the meaning, sensible or affective, of the non-language sound. i show how the cognitive contexts of the production and the reception of the non-language sounds, and the song, play a central role in our apprehension of the sounds and propose the existence of an extra-bodily sense making organ of our cognitive system, the extension of our brain and sense organs, which i call contextual cognitive apparatus or exo-brain.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-10-16',\n",
       "  'updated': '2024-03-28',\n",
       "  'authors': ['ankur betageri'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.11474'},\n",
       " {'title': 'influence of density-dependent diffusion on pattern formation in a   bounded habitat',\n",
       "  'id': '2210.11638',\n",
       "  'abstract': 'considering a nonlocal version of the fisher-kpp equation, we explore the impact of heterogeneous diffusion on pattern formation within a bounded region (refuge). under homogeneous diffusion, it is established that, nonlocality can lead to spontaneous pattern formation under certain parameter conditions, otherwise, when the homogeneous state is stable, spatial perturbations such as the existence of a refuge (high quality region within a hostile environment) can induce patterns. to understand how density-dependent diffusivity influences these forms of pattern formation, we examine diffusivity reactions to both rarefaction or overcrowding. additionally, for comparison, we investigate the scenario where diffusivity levels vary spatially, inside and outside the refuge. we find that state-dependent diffusivity affects the shape and stability of patterns, potentially triggering either explosive growth or fragmentation of the population distribution, depending on how diffusion responses to density changes.',\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech nlin.ps',\n",
       "  'doi': '',\n",
       "  'created': '2022-10-20',\n",
       "  'updated': '2024-07-03',\n",
       "  'authors': ['g. g. piva', 'c. anteneodo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.11638'},\n",
       " {'title': 'embedded silicon-organic integrated neuromorphic system',\n",
       "  'id': '2210.12064',\n",
       "  'abstract': 'the development of artificial intelligence (ai) and robotics are both based on the tenet of \"science and technology are people-oriented\", and both need to achieve efficient communication with the human brain. based on multi-disciplinary research in systems neuroscience, computer architecture, and functional organic materials, we proposed the concept of using ai to simulate the operating principles and materials of the brain in hardware to develop brain-inspired intelligence technology, and realized the preparation of neuromorphic computing devices and basic materials. we simulated neurons and neural networks in terms of material and morphology, using a variety of organic polymers as the base materials for neuroelectronic devices, for building neural interfaces as well as organic neural devices and silicon neural computational modules. we assemble organic artificial synapses with simulated neurons from silicon-based field-programmable gate array (fpga) into organic artificial neurons, the basic components of neural networks, and later construct biological neural network models based on the interpreted neural circuits. finally, we also discuss how to further build neuromorphic devices based on these organic artificial neurons, which have both a neural interface friendly to nervous tissue and interact with information from real biological neural networks.',\n",
       "  'categories': 'q-bio.nc cs.ne',\n",
       "  'doi': '',\n",
       "  'created': '2022-10-17',\n",
       "  'updated': '2024-06-25',\n",
       "  'authors': ['shengjie zheng',\n",
       "   'ling liu',\n",
       "   'junjie yang',\n",
       "   'jianwei zhang',\n",
       "   'tao su',\n",
       "   'bin yue',\n",
       "   'xiaojian li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.12064'},\n",
       " {'title': 'the quest for the definition of life',\n",
       "  'id': '2210.12072',\n",
       "  'abstract': 'the intricacy and diversity inherent in living organisms present a formidable obstacle to the establishment of a universally accepted definition. life manifests in a multitude of forms, exhibiting various attributes such as growth, reproduction, responsiveness to stimuli, adaptation, and homeostasis. however, each of these characteristics can also be observed to some degree within certain non-living systems, leading to a blurring of boundaries and generating conceptual complexities. in this manuscript, i demonstrate that the transformation of a non-living entity into a living organism does not adhere to a specific temporal boundary that unequivocally designates the onset of life. through mathematical analysis, i have demonstrated that a comprehensive definition of living beings does not exist, which means that there are no clear boundaries in the chemical processes that turn non-living entities into living ones. in other words, living organisms do not possess unique characteristics that can completely set them apart from non-living entities. therefore, no definitive definition exists that unequivocally distinguishes living things from non-living things.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-10-10',\n",
       "  'updated': '2024-01-08',\n",
       "  'authors': ['kazem haghnejad azar'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.12072'},\n",
       " {'title': 'physical limits to membrane curvature sensing by a single protein',\n",
       "  'id': '2210.12204',\n",
       "  'abstract': \"membrane curvature sensing is essential for a diverse range of biological processes. recent experiments have revealed that a single nanometer-sized septin protein can distinguish between membrane-coated glass beads of one micron and three micron diameters, even though the septin is orders of magnitude smaller than the beads. this sensing ability is especially surprising since curvature-sensing proteins must deal with persistent thermal fluctuations of the membrane, leading to discrepancies between the bead's curvature and the local membrane curvature sensed instantaneously by a protein. using continuum models of fluctuating membranes, we investigate whether it is feasible for a protein acting as a perfect observer of the membrane to sense micron-scale curvature either by measuring local membrane curvature or by using bilayer lipid densities as a proxy. to do this, we develop algorithms to simulate lipid density and membrane shape fluctuations. we derive physical limits to the sensing efficacy of a protein in terms of protein size, membrane thickness, membrane bending modulus, membrane-substrate adhesion strength, and bead size. to explain the experimental protein-bead association rates, we develop two classes of predictive models: i) for proteins that maximally associate to a preferred curvature, and ii) for proteins with enhanced association rates above a threshold curvature. we find that the experimentally observed sensing efficacy is close to the theoretical sensing limits imposed on a septin-sized protein, and that the variance in membrane curvature fluctuations sensed by a protein determines how sharply the association rate depends on the curvature of the bead.\",\n",
       "  'categories': 'physics.bio-ph cond-mat.soft q-bio.sc',\n",
       "  'doi': '10.1103/physreve.108.064407',\n",
       "  'created': '2022-10-21',\n",
       "  'updated': '',\n",
       "  'authors': ['indrajit badvaram', 'brian a. camley'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.12204'},\n",
       " {'title': 'efficient representation of natural image patches',\n",
       "  'id': '2210.13004',\n",
       "  'abstract': \"utilizing an abstract information processing model based on minimal yet realistic assumptions inspired by biological systems, we study how to achieve the early visual system's two ultimate objectives: efficient information transmission and accurate sensor probability distribution modeling. we prove that optimizing for information transmission does not guarantee optimal probability distribution modeling in general. we illustrate, using a two-pixel (2d) system and image patches, that an efficient representation can be realized through a nonlinear population code driven by two types of biologically plausible loss functions that depend solely on output. after unsupervised learning, our abstract information processing model bears remarkable resemblances to biological systems, despite not mimicking many features of real neurons, such as spiking activity. a preliminary comparison with a contemporary deep learning model suggests that our model offers a significant efficiency advantage. our model provides novel insights into the computational theory of early visual systems as well as a potential new approach to enhance the efficiency of deep learning models.\",\n",
       "  'categories': 'cs.cv cs.lg eess.iv q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-10-24',\n",
       "  'updated': '2024-04-11',\n",
       "  'authors': ['cheng guo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.13004'},\n",
       " {'title': 'interactive inference: a multi-agent model of cooperative joint actions',\n",
       "  'id': '2210.13113',\n",
       "  'abstract': 'we advance a novel computational model of multi-agent, cooperative joint actions that is grounded in the cognitive framework of active inference. the model assumes that to solve a joint task, such as pressing together a red or blue button, two (or more) agents engage in a process of interactive inference. each agent maintains probabilistic beliefs about the goal of the joint task (e.g., should we press the red or blue button?) and updates them by observing the other agent\\'s movements, while in turn selecting movements that make his own intentions legible and easy to infer by the other agent (i.e., sensorimotor communication). over time, the interactive inference aligns both the beliefs and the behavioral strategies of the agents, hence ensuring the success of the joint action. we exemplify the functioning of the model in two simulations. the first simulation illustrates a \\'\\'leaderless\\'\\' joint action. it shows that when two agents lack a strong preference about their joint task goal, they jointly infer it by observing each other\\'s movements. in turn, this helps the interactive alignment of their beliefs and behavioral strategies. the second simulation illustrates a \"leader-follower\" joint action. it shows that when one agent (\"leader\") knows the true joint goal, it uses sensorimotor communication to help the other agent (\"follower\") infer it, even if doing this requires selecting a more costly individual plan. these simulations illustrate that interactive inference supports successful multi-agent joint actions and reproduces key cognitive and behavioral dynamics of \"leaderless\" and \"leader-follower\" joint actions observed in human-human experiments. in sum, interactive inference provides a cognitively inspired, formal framework to realize cooperative joint actions and consensus in multi-agent systems.',\n",
       "  'categories': 'cs.ai cs.it cs.ma math.it q-bio.nc',\n",
       "  'doi': '10.1109/tsmc.2023.3312585',\n",
       "  'created': '2022-10-24',\n",
       "  'updated': '2022-12-10',\n",
       "  'authors': ['domenico maisto', 'francesco donnarumma', 'giovanni pezzulo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.13113'},\n",
       " {'title': 'structure-based drug design with equivariant diffusion models',\n",
       "  'id': '2210.13695',\n",
       "  'abstract': 'structure-based drug design (sbdd) aims to design small-molecule ligands that bind with high affinity and specificity to pre-determined protein targets. generative sbdd methods leverage structural data of drugs in complex with their protein targets to propose new drug candidates. these approaches typically place one atom at a time in an autoregressive fashion using the binding pocket as well as previously added ligand atoms as context in each step. recently a surge of diffusion generative models has entered this domain which hold promise to capture the statistical properties of natural ligands more faithfully. however, most existing methods focus exclusively on bottom-up de novo design of compounds or tackle other drug development challenges with task-specific models. the latter requires curation of suitable datasets, careful engineering of the models and retraining from scratch for each task. here we show how a single pre-trained diffusion model can be applied to a broader range of problems, such as off-the-shelf property optimization, explicit negative design, and partial molecular design with inpainting. we formulate sbdd as a 3d-conditional generation problem and present diffsbdd, an se(3)-equivariant diffusion model that generates novel ligands conditioned on protein pockets. our in silico experiments demonstrate that diffsbdd captures the statistics of the ground truth data effectively. furthermore, we show how additional constraints can be used to improve the generated drug candidates according to a variety of computational metrics. these results support the assumption that diffusion models represent the complex distribution of structural data more accurately than previous methods, and are able to incorporate additional design objectives and constraints changing nothing but the sampling strategy.',\n",
       "  'categories': 'q-bio.bm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2022-10-24',\n",
       "  'updated': '2024-09-23',\n",
       "  'authors': ['arne schneuing',\n",
       "   'charles harris',\n",
       "   'yuanqi du',\n",
       "   'kieran didi',\n",
       "   'arian jamasb',\n",
       "   'ilia igashov',\n",
       "   'weitao du',\n",
       "   'carla gomes',\n",
       "   'tom blundell',\n",
       "   'pietro lio',\n",
       "   'max welling',\n",
       "   'michael bronstein',\n",
       "   'bruno correia'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.13695'},\n",
       " {'title': 'critical growth of cerebral tissue in organoids: theory and experiments',\n",
       "  'id': '2210.17270',\n",
       "  'abstract': 'we develop a fokker-planck theory of tissue growth with three types of cells (symmetrically dividing, asymmetrically dividing and non-dividing) as main agents to study the growth dynamics of human cerebral organoids. fitting the theory to lineage tracing data obtained in next generation sequencing experiments, we show that the growth of cerebral organoids is a critical process. we derive analytical expressions describing the time evolution of clonal lineage sizes and show how power-law distributions arise in the limit of long times due to the vanishing of a characteristic growth scale. we discuss that the independence of critical growth on initial conditions could be biologically advantageous.',\n",
       "  'categories': 'q-bio.nc cond-mat.soft q-bio.to',\n",
       "  'doi': '10.1103/physrevlett.131.178402',\n",
       "  'created': '2022-10-31',\n",
       "  'updated': '2023-08-25',\n",
       "  'authors': ['egor i. kiselev', 'florian g. pflug', 'arndt von haeseler'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.17270'},\n",
       " {'title': 'spontaneous emergence of groups and signaling diversity in dynamic   networks',\n",
       "  'id': '2210.17309',\n",
       "  'abstract': 'we study the coevolution of network structure and signaling behavior. we model agents who can preferentially associate with others in a dynamic network while they also learn to play a simple sender-receiver game. we have four major findings. first, signaling interactions in dynamic networks are sufficient to cause the endogenous formation of distinct signaling groups, even in an initially homogeneous population. second, dynamic networks allow the emergence of novel {\\\\em hybrid} signaling groups that do not converge on a single common signaling system but are instead composed of different yet complementary signaling strategies. we show that the presence of these hybrid groups promotes stable diversity in signaling among other groups in the population. third, we find important distinctions in information processing capacity of different groups: hybrid groups diffuse information more quickly initially but at the cost of taking longer to reach all group members. fourth, our findings pertain to all common interest signaling games, are robust across many parameters, and mitigate known problems of inefficient communication.',\n",
       "  'categories': 'cs.si econ.th physics.soc-ph q-bio.mn q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2022-10-22',\n",
       "  'updated': '2024-01-12',\n",
       "  'authors': ['zachary fulker',\n",
       "   'patrick forber',\n",
       "   'rory smead',\n",
       "   'christoph riedl'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2210.17309'},\n",
       " {'title': 'evolutionary innovation by polyploidy',\n",
       "  'id': '2211.00581',\n",
       "  'abstract': 'the preferred conditions for evolutionary innovation is a fundamental question, but little is known, in part because the question involves rare events. we focused on the potential role of polyploidy in the evolution of novel traits. there are two hypotheses regarding the effects of polyploidy on evolution: polyploidy reduces the effect of a single mutation and slows evolution. in contrast, the gene redundancy introduced by polyploidy will promote neofunctionalization and accelerate evolution. does polyploidy speed up or slow down evolution? in this study, we proposed a simple model of polyploid cells and showed that the evolutionary rate of polyploids is similar to or much slower than that of haploids under neutral selection or during gradual evolution. however, on a fitness landscape where cells should jump over a lethal valley to increase their fitness, the probability of evolution in polyploidy could be drastically increased, and the optimal number of chromosomes was identified. we theoretically discussed the existence of this optimal chromosome number from the large deviation theory. furthermore, we proposed that the optimization for achieving evolutionary innovation could determine the range of chromosome numbers in polyploid bacteria.',\n",
       "  'categories': 'q-bio.pe physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2022-11-01',\n",
       "  'updated': '2024-04-15',\n",
       "  'authors': ['tetsuhiro s. hatakeyama', 'ryudo ohbayashi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.00581'},\n",
       " {'title': 'learning melanocytic cell masks from adjacent stained tissue',\n",
       "  'id': '2211.00646',\n",
       "  'abstract': 'melanoma is one of the most aggressive forms of skin cancer, causing a large proportion of skin cancer deaths. however, melanoma diagnoses by pathologists shows low interrater reliability. as melanoma is a cancer of the melanocyte, there is a clear need to develop a melanocytic cell segmentation tool that is agnostic to pathologist variability and automates pixel-level annotation. gigapixel-level pathologist labeling, however, is impractical. herein, we propose a means to train deep neural networks for melanocytic cell segmentation from hematoxylin and eosin (h&e) stained sections and paired immunohistochemistry (ihc) of adjacent tissue sections, achieving a mean iou of 0.64 despite imperfect ground-truth labels.',\n",
       "  'categories': 'q-bio.qm cs.ai cs.cv cs.lg eess.iv',\n",
       "  'doi': '',\n",
       "  'created': '2022-10-31',\n",
       "  'updated': '2024-03-13',\n",
       "  'authors': ['mikio tada',\n",
       "   'ursula e. lang',\n",
       "   'iwei yeh',\n",
       "   'elizabeth s. keiser',\n",
       "   'maria l. wei',\n",
       "   'michael j. keiser'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.00646'},\n",
       " {'title': 'macrophage anti-inflammatory behaviour in a multiphase model of   atherosclerotic plaque development',\n",
       "  'id': '2211.01867',\n",
       "  'abstract': 'atherosclerosis is an inflammatory disease characterised by the formation of plaques, which are deposits of lipids and cholesterol-laden macrophages that form in the artery wall. the inflammation is often non-resolving, due in large part to changes in normal macrophage anti-inflammatory behaviour that are induced by the toxic plaque microenvironment. these changes include higher death rates, defective efferocytic uptake of dead cells, and reduced rates of emigration. we develop a free boundary multiphase model for early atherosclerotic plaques, and we use it to investigate the effects of impaired macrophage anti-inflammatory behaviour on plaque structure and growth. we find that high rates of cell death relative to efferocytic uptake results in a plaque populated mostly by dead cells. we also find that emigration can potentially slow or halt plaque growth by allowing material to exit the plaque, but this is contingent on the availability of live macrophage foam cells in the deep plaque. finally, we introduce an additional bead species to model macrophage tagging via microspheres, and we use the extended model to explore how high rates of cell death and low rates of efferocytosis and emigration prevent the clearance of macrophages from the plaque.',\n",
       "  'categories': 'q-bio.cb q-bio.to',\n",
       "  'doi': '10.1007/s11538-023-01142-7',\n",
       "  'created': '2022-11-03',\n",
       "  'updated': '',\n",
       "  'authors': ['ishraq u. ahmed', 'helen m. byrne', 'mary r. myerscough'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.01867'},\n",
       " {'title': 'a branching process with coalescence to model random phylogenetic   networks',\n",
       "  'id': '2211.02407',\n",
       "  'abstract': 'we introduce a biologically natural, mathematically tractable model of random phylogenetic network to describe evolution in the presence of hybridization. one of the features of this model is that the hybridization rate of the lineages correlates negatively with their phylogenetic distance. we give formulas / characterizations for quantities of biological interest that make them straightforward to compute in practice. we show that the appropriately rescaled network, seen as a metric space, converges to the brownian continuum random tree, and that the uniformly rooted network has a local weak limit, which we describe explicitly.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '10.1214/24-ejp1088',\n",
       "  'created': '2022-11-04',\n",
       "  'updated': '2023-10-10',\n",
       "  'authors': ['françois bienvenu', 'jean-jil duchamps'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.02407'},\n",
       " {'title': 'topological biomarkers for real-time detection of epileptic seizures',\n",
       "  'id': '2211.02523',\n",
       "  'abstract': \"real time seizure detection is a fundamental problem in computational neuroscience towards diagnosis and treatment's improvement of epileptic disease. we propose a real-time computational method for tracking and detection of epileptic seizures from raw neurophysiological recordings. our mechanism is based on the topological analysis of the sliding-window embedding of the time series derived from simultaneously recorded channels. we extract topological biomarkers from the signals via the computation of the persistent homology of time-evolving topological spaces. remarkably, the proposed biomarkers robustly captures the change in the brain dynamics during the ictal state. we apply our methods in different types of signals including scalp and intracranial electroencephalograms and magnetoencephalograms, in patients during interictal and ictal states, showing high accuracy in a range of clinical situations.\",\n",
       "  'categories': 'q-bio.qm math.at',\n",
       "  'doi': '',\n",
       "  'created': '2022-11-04',\n",
       "  'updated': '2024-06-14',\n",
       "  'authors': ['ximena fernández', 'diego mateos'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.02523'},\n",
       " {'title': 'rubicon: a framework for designing efficient deep learning-based genomic   basecallers',\n",
       "  'id': '2211.03079',\n",
       "  'abstract': 'nanopore sequencing generates noisy electrical signals that need to be converted into a standard string of dna nucleotide bases using a computational step called basecalling. the accuracy and speed of basecalling have critical implications for all later steps in genome analysis. many researchers adopt complex deep learning-based models to perform basecalling without considering the compute demands of such models, which leads to slow, inefficient, and memory-hungry basecallers. therefore, there is a need to reduce the computation and memory cost of basecalling while maintaining accuracy. our goal is to develop a comprehensive framework for creating deep learning-based basecallers that provide high efficiency and performance. we introduce rubicon, a framework to develop hardware-optimized basecallers. rubicon consists of two novel machine-learning techniques that are specifically designed for basecalling. first, we introduce the first quantization-aware basecalling neural architecture search (qabas) framework to specialize the basecalling neural network architecture for a given hardware acceleration platform while jointly exploring and finding the best bit-width precision for each neural network layer. second, we develop skipclip, the first technique to remove the skip connections present in modern basecallers to greatly reduce resource and storage requirements without any loss in basecalling accuracy. we demonstrate the benefits of rubicon by developing rubicall, the first hardware-optimized basecaller that performs fast and accurate basecalling. compared to the fastest state-of-the-art basecaller, rubicall provides a 3.96x speedup with 2.97% higher accuracy. we show that rubicon helps researchers develop hardware-optimized basecallers that are superior to expert-designed models.',\n",
       "  'categories': 'cs.ar cs.dc q-bio.gn',\n",
       "  'doi': '',\n",
       "  'created': '2022-11-06',\n",
       "  'updated': '2024-02-05',\n",
       "  'authors': ['gagandeep singh',\n",
       "   'mohammed alser',\n",
       "   'kristof denolf',\n",
       "   'can firtina',\n",
       "   'alireza khodamoradi',\n",
       "   'meryem banu cavlak',\n",
       "   'henk corporaal',\n",
       "   'onur mutlu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.03079'},\n",
       " {'title': 'efficacy of mri data harmonization in the age of machine learning. a   multicenter study across 36 datasets',\n",
       "  'id': '2211.04125',\n",
       "  'abstract': 'pooling publicly-available mri data from multiple sites allows to assemble extensive groups of subjects, increase statistical power, and promote data reuse with machine learning techniques. the harmonization of multicenter data is necessary to reduce the confounding effect associated with non-biological sources of variability in the data. however, when applied to the entire dataset before machine learning, the harmonization leads to data leakage, because information outside the training set may affect model building, and potentially falsely overestimate performance. we propose a 1) measurement of the efficacy of data harmonization; 2) harmonizer transformer, i.e., an implementation of the combat harmonization allowing its encapsulation among the preprocessing steps of a machine learning pipeline, avoiding data leakage. we tested these tools using brain t1-weighted mri data from 1740 healthy subjects acquired at 36 sites. after harmonization, the site effect was removed or reduced, and we showed the data leakage effect in predicting individual age from mri data, highlighting that introducing the harmonizer transformer into a machine learning pipeline allows for avoiding data leakage.',\n",
       "  'categories': 'cs.lg eess.iv q-bio.qm',\n",
       "  'doi': '10.1038/s41597-023-02421-7',\n",
       "  'created': '2022-11-08',\n",
       "  'updated': '2024-02-01',\n",
       "  'authors': ['chiara marzi',\n",
       "   'marco giannelli',\n",
       "   'andrea barucci',\n",
       "   'carlo tessa',\n",
       "   'mario mascalchi',\n",
       "   'stefano diciotti'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.04125'},\n",
       " {'title': 'gcondnet: a novel method for improving neural networks on small   high-dimensional tabular data',\n",
       "  'id': '2211.06302',\n",
       "  'abstract': \"neural networks often struggle with high-dimensional but small sample-size tabular datasets. one reason is that current weight initialisation methods assume independence between weights, which can be problematic when there are insufficient samples to estimate the model's parameters accurately. in such small data scenarios, leveraging additional structures can improve the model's performance and training stability. to address this, we propose gcondnet, a general approach to enhance neural networks by leveraging implicit structures present in tabular data. we create a graph between samples for each data dimension, and utilise graph neural networks (gnns) to extract this implicit structure, and for conditioning the parameters of the first layer of an underlying predictor network. by creating many small graphs, gcondnet exploits the data's high-dimensionality, and thus improves the performance of an underlying predictor network. we demonstrate gcondnet's effectiveness on 12 real-world datasets, where it outperforms 14 standard and state-of-the-art methods. the results show that gcondnet is a versatile framework for injecting graph-regularisation into various types of neural networks, including mlps and tabular transformers. code is available at https://github.com/andreimargeloiu/gcondnet.\",\n",
       "  'categories': 'cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-11-11',\n",
       "  'updated': '2024-08-17',\n",
       "  'authors': ['andrei margeloiu',\n",
       "   'nikola simidjievski',\n",
       "   'pietro lio',\n",
       "   'mateja jamnik'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.06302'},\n",
       " {'title': 'quantum versus population dynamics over cayley graphs',\n",
       "  'id': '2211.06928',\n",
       "  'abstract': 'consider a graph whose vertices are populated by identical objects, together with an algorithm for the time-evolution of the number of objects placed at each of the vertices. the discrete dynamics of these objects can be observed and studied using simple and inexpensive laboratory settings. there are many similarities but also many differences between such population dynamics and the quantum dynamics of a particle hopping on the same graph. in this work, we show that a specific decoration of the original graph enables an exact mapping between the models of population and quantum dynamics. as such, population dynamics over graphs is yet another classical platform that can simulate quantum effects. several examples are used to demonstrate this claim.',\n",
       "  'categories': 'quant-ph cond-mat.other q-bio.pe',\n",
       "  'doi': '10.1016/j.aop.2023.169430',\n",
       "  'created': '2022-11-13',\n",
       "  'updated': '',\n",
       "  'authors': ['emil prodan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.06928'},\n",
       " {'title': 'temporal patterns in insulin needs for type 1 diabetes',\n",
       "  'id': '2211.07393',\n",
       "  'abstract': 'type 1 diabetes (t1d) is a chronic condition where the body produces little or no insulin, a hormone required for the cells to use blood glucose (bg) for energy and to regulate bg levels in the body. finding the right insulin dose and time remains a complex, challenging and as yet unsolved control task. in this study, we use the openaps data commons dataset, which is an extensive dataset collected in real-life conditions, to discover temporal patterns in insulin need driven by well-known factors such as carbohydrates as well as potentially novel factors. we utilised various time series techniques to spot such patterns using matrix profile and multi-variate clustering. the better we understand t1d and the factors impacting insulin needs, the more we can contribute to building data-driven technology for t1d treatments.',\n",
       "  'categories': 'cs.lg q-bio.qm stat.ml',\n",
       "  'doi': '10.2196/44384',\n",
       "  'created': '2022-11-14',\n",
       "  'updated': '2022-11-17',\n",
       "  'authors': ['isabella degen', 'zahraa s. abdallah'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.07393'},\n",
       " {'title': 'harmonization benchmarking tool for neuroimaging datasets',\n",
       "  'id': '2211.07869',\n",
       "  'abstract': 'a major data pre-processing step for large, multi-site studies is to handle site effects by harmonizing data, generating a dataset that enables more powerful analyses and more robust algorithms. there is a wide variety of data harmonization techniques, but there are few tools that streamline the process of harmonizing data, comparing across techniques, and benchmarking new techniques. in this paper, we introduce harmonization benchmarking tool (habet), an open source tool for generating harmonized images and evaluating the performance of different harmonization algorithms. to demonstrate the capabilities of habet, we harmonize diffusion mri images from the adolescent brain and cognitive development (abcd) study using two different approaches, and we compare their performance.',\n",
       "  'categories': 'eess.iv q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-11-14',\n",
       "  'updated': '',\n",
       "  'authors': ['tom osika',\n",
       "   'ebrahim ebrahim',\n",
       "   'martin styner',\n",
       "   'marc niethammer',\n",
       "   'thomas sawyer',\n",
       "   'andinet enquobahrie'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.07869'},\n",
       " {'title': 'genome-on-diet: taming large-scale genomic analyses via sparsified   genomics',\n",
       "  'id': '2211.08157',\n",
       "  'abstract': 'searching for similar genomic sequences is an essential and fundamental step in biomedical research and an overwhelming majority of genomic analyses. state-of-the-art computational methods performing such comparisons fail to cope with the exponential growth of genomic sequencing data. we introduce the concept of sparsified genomics where we systematically exclude a large number of bases from genomic sequences and enable much faster and more memory-efficient processing of the sparsified, shorter genomic sequences, while providing similar or even higher accuracy compared to processing non-sparsified sequences. sparsified genomics provides significant benefits to many genomic analyses and has broad applicability. we show that sparsifying genomic sequences greatly accelerates the state-of-the-art read mapper (minimap2) by 2.57-5.38x, 1.13-2.78x, and 3.52-6.28x using real illumina, hifi, and ont reads, respectively, while providing up to 2.1x smaller memory footprint, 2x smaller index size, and more truly detected small and structural variations compared to minimap2. sparsifying genomic sequences makes containment search through very large genomes and large databases 72.7-75.88x faster and 723.3x more storage-efficient than searching through non-sparsified genomic sequences (with cmash and kmc3). sparsifying genomic sequences enables robust microbiome discovery by providing 54.15-61.88x faster and 720x more storage-efficient taxonomic profiling of metagenomic samples over the state-of-the-art tool (metalign). we design and open-source a framework called genome-on-diet as an example tool for sparsified genomics, which can be freely downloaded from https://github.com/cmu-safari/genome-on-diet.',\n",
       "  'categories': 'cs.ds q-bio.gn q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-11-15',\n",
       "  'updated': '2024-07-09',\n",
       "  'authors': ['mohammed alser', 'julien eudine', 'onur mutlu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.08157'},\n",
       " {'title': 'comparing apples to apples -- using a modular and adaptable analysis   pipeline to compare slow cerebral rhythms across heterogeneous datasets',\n",
       "  'id': '2211.08527',\n",
       "  'abstract': 'neuroscience is moving towards a more integrative discipline, where understanding brain function requires consolidating the accumulated evidence seen across experiments, species, and measurement techniques. a remaining challenge on that path is integrating such heterogeneous data into analysis workflows such that consistent and comparable conclusions can be distilled as an experimental basis for models and theories. here, we propose a solution in the context of slow wave activity ($<1$ hz), which occurs during unconscious brain states like sleep and general anesthesia, and is observed across diverse experimental approaches. we address the issue of integrating and comparing heterogeneous data by conceptualizing a general pipeline design that is adaptable to a variety of inputs and applications. furthermore, we present the collaborative brain wave analysis pipeline (cobrawap) as a concrete, reusable software implementation to perform broad, detailed, and rigorous comparisons of slow wave characteristics across multiple, openly available ecog and calcium imaging datasets.',\n",
       "  'categories': 'q-bio.nc q-bio.qm',\n",
       "  'doi': '10.1016/j.crmeth.2023.100681',\n",
       "  'created': '2022-11-15',\n",
       "  'updated': '2023-02-07',\n",
       "  'authors': ['robin gutzen',\n",
       "   'giulia de bonis',\n",
       "   'chiara de luca',\n",
       "   'elena pastorelli',\n",
       "   'cristiano capone',\n",
       "   'anna letizia allegra mascaro',\n",
       "   'francesco resta',\n",
       "   'arnau manasanch',\n",
       "   'francesco saverio pavone',\n",
       "   'maria v. sanchez-vives',\n",
       "   'maurizio mattia',\n",
       "   'sonja grün',\n",
       "   'pier stanislao paolucci',\n",
       "   'michael denker'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.08527'},\n",
       " {'title': 'the combinatorial code and the graph rules of dale networks',\n",
       "  'id': '2211.08618',\n",
       "  'abstract': \"we describe the combinatorics of equilibria and steady states of neurons in threshold-linear networks that satisfy dale's law. the combinatorial code of a dale network is characterized in terms of two conditions: (i) a condition on the network connectivity graph, and (ii) a spectral condition on the synaptic matrix. we find that in the weak coupling regime the combinatorial code depends only on the connectivity graph, and not on the particulars of the synaptic strengths. moreover, we prove that the combinatorial code of a weakly coupled network is a sublattice, and we provide a learning rule for encoding a sublattice in a weakly coupled excitatory network. in the strong coupling regime we prove that the combinatorial code of a generic dale network is intersection-complete and is therefore a convex code, as is common in some sensory systems in the brain.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-11-15',\n",
       "  'updated': '2024-06-05',\n",
       "  'authors': ['nikola milićević', 'vladimir itskov'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.08618'},\n",
       " {'title': 'endemic oscillations for sars-cov-2 omicron -- a sirs model analysis',\n",
       "  'id': '2211.09005',\n",
       "  'abstract': \"the sirs model with constant vaccination and immunity waning rates is well known to show a transition from a disease-free to an endemic equilibrium as the basic reproduction number $r_0$ is raised above threshold. it is shown that this model maps to hethcote's classic endemic model originally published in 1973. in this way one obtains unifying formulas for a whole class of models showing endemic bifurcation. in particular, if the vaccination rate is smaller than the recovery rate and $r_- < r_0 < r_+$ for certain upper and lower bounds $r_\\\\pm$, then trajectories spiral into the endemic equilibrium via damped infection waves. latest data of the sars-cov-2 omicron variant suggest that according to this simplified model continuous vaccination programs will not be capable to escape the oscillating endemic phase. however, in view of the strong damping factors predicted by the model, in reality these oscillations will certainly be overruled by time-dependent contact behaviors.\",\n",
       "  'categories': 'q-bio.pe math.ds physics.soc-ph',\n",
       "  'doi': '10.1016/j.chaos.2023.113678',\n",
       "  'created': '2022-11-16',\n",
       "  'updated': '2023-05-30',\n",
       "  'authors': ['florian nill'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.09005'},\n",
       " {'title': 'hodge-decomposition of brain networks',\n",
       "  'id': '2211.10542',\n",
       "  'abstract': 'we analyze brain networks by decomposing them into three orthogonal components: gradient, curl, and harmonic flows, through the hodge decomposition, a technique advantageous for capturing complex topological features. a wasserstein distance based topological inference is developed to determine the statistical significance of each component. the hodge decomposition is applied to human brain networks obtained from a resting-state fmri study. our results indicate statistically significant differences in the topological features between male and female brain networks.',\n",
       "  'categories': 'q-bio.nc q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-11-18',\n",
       "  'updated': '2024-04-01',\n",
       "  'authors': ['d. vijay anand', 'moo k. chung'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.10542'},\n",
       " {'title': 'learning biological neuronal networks with artificial neural networks:   neural oscillations',\n",
       "  'id': '2211.11169',\n",
       "  'abstract': 'first-principles-based modelings have been extremely successful in providing crucial insights and predictions for complex biological functions and phenomena. however, they can be hard to build and expensive to simulate for complex living systems. on the other hand, modern data-driven methods thrive at modeling many types of high-dimensional and noisy data. still, the training and interpretation of these data-driven models remain challenging. here, we combine the two types of methods to model stochastic neuronal network oscillations. specifically, we develop a class of first-principles-based artificial neural networks to provide faithful surrogates to the high-dimensional, nonlinear oscillatory dynamics produced by neural circuits in the brain. furthermore, when the training data set is enlarged within a range of parameter choices, the artificial neural networks become generalizable to these parameters, covering cases in distinctly different dynamical regimes. in all, our work opens a new avenue for modeling complex neuronal network dynamics with artificial neural networks.',\n",
       "  'categories': 'nlin.ao q-bio.nc',\n",
       "  'doi': '10.1007/s00285-024-02081-0',\n",
       "  'created': '2022-11-20',\n",
       "  'updated': '',\n",
       "  'authors': ['ruilin zhang',\n",
       "   'zhongyi wang',\n",
       "   'tianyi wu',\n",
       "   'yuhang cai',\n",
       "   'louis tao',\n",
       "   'zhuo-cheng xiao',\n",
       "   'yao li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.11169'},\n",
       " {'title': 'diffbp: generative diffusion of 3d molecules for target protein binding',\n",
       "  'id': '2211.11214',\n",
       "  'abstract': 'generating molecules that bind to specific proteins is an important but challenging task in drug discovery. previous works usually generate atoms in an auto-regressive way, where element types and 3d coordinates of atoms are generated one by one. however, in real-world molecular systems, the interactions among atoms in an entire molecule are global, leading to the energy function pair-coupled among atoms. with such energy-based consideration, the modeling of probability should be based on joint distributions, rather than sequentially conditional ones. thus, the unnatural sequentially auto-regressive modeling of molecule generation is likely to violate the physical rules, thus resulting in poor properties of the generated molecules. in this work, a generative diffusion model for molecular 3d structures based on target proteins as contextual constraints is established, at a full-atom level in a non-autoregressive way. given a designated 3d protein binding site, our model learns the generative process that denoises both element types and 3d coordinates of an entire molecule, with an equivariant network. experimentally, the proposed method shows competitive performance compared with prevailing works in terms of high affinity with proteins and appropriate molecule sizes as well as other drug properties such as drug-likeness of the generated molecules.',\n",
       "  'categories': 'q-bio.bm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2022-11-21',\n",
       "  'updated': '2024-07-14',\n",
       "  'authors': ['haitao lin',\n",
       "   'yufei huang',\n",
       "   'odin zhang',\n",
       "   'siqi ma',\n",
       "   'meng liu',\n",
       "   'xuanjing li',\n",
       "   'lirong wu',\n",
       "   'jishui wang',\n",
       "   'tingjun hou',\n",
       "   'stan z. li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.11214'},\n",
       " {'title': 'predicting adverse outcomes following catheter ablation treatment for   atrial fibrillation',\n",
       "  'id': '2211.11965',\n",
       "  'abstract': 'objective: to develop prognostic survival models for predicting adverse outcomes after catheter ablation treatment for non-valvular atrial fibrillation (af).   methods: we used a linked dataset including hospital administrative data, prescription medicine claims, emergency department presentations, and death registrations of patients in new south wales, australia. the cohort included patients who received catheter ablation for af. traditional and deep survival models were trained to predict major bleeding events and a composite of heart failure, stroke, cardiac arrest, and death.   results: out of a total of 3285 patients in the cohort, 177 (5.3%) experienced the composite outcome (heart failure, stroke, cardiac arrest, death) and 167 (5.1%) experienced major bleeding events after catheter ablation treatment. models predicting the composite outcome had high risk discrimination accuracy, with the best model having a concordance index > 0.79 at the evaluated time horizons. models for predicting major bleeding events had poor risk discrimination performance, with all models having a concordance index < 0.66. the most impactful features for the models predicting higher risk were comorbidities indicative of poor health, older age, and therapies commonly used in sicker patients to treat heart failure and af.   conclusions: diagnosis and medication history did not contain sufficient information for precise risk prediction of experiencing major bleeding events. the models for predicting the composite outcome have the potential to enable clinicians to identify and manage high-risk patients following catheter ablation proactively. future research is needed to validate the usefulness of these models in clinical practice.',\n",
       "  'categories': 'cs.lg q-bio.qm stat.ot',\n",
       "  'doi': '10.1016/j.hlc.2023.12.016',\n",
       "  'created': '2022-11-21',\n",
       "  'updated': '2023-06-04',\n",
       "  'authors': ['juan c. quiroz',\n",
       "   'david brieger',\n",
       "   'louisa jorm',\n",
       "   'raymond w sy',\n",
       "   'benjumin hsu',\n",
       "   'blanca gallego'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.11965'},\n",
       " {'title': 'identifiability of nonlinear ode models with time-varying parameters:   the general analytical solution and applications in viral dynamics',\n",
       "  'id': '2211.13507',\n",
       "  'abstract': 'identifiability is a structural property of any ode model characterized by a set of unknown parameters. it describes the possibility of determining the values of these parameters from fusing the observations of the system inputs and outputs. this paper finds the general analytical solution of this fundamental problem and, based on this, provides a general and automated analytical method to determine the identifiability of the unknown parameters. in particular, the method can handle any model, regardless of its complexity and type of non-linearity, and provides the identifiability of the parameters even when they are time-varying. in addition, it is automatic as it simply needs to follow the steps of a systematic procedure that only requires to perform the calculation of derivatives and matrix ranks. time-varying parameters are treated as unknown inputs and their identification is based on the very recent analytical solution of the unknown input observability problem [1, 2]. the method is used to determine the identifiability of the unknown time-varying parameters that characterize two non-linear models in the field of viral dynamics (hiv and covid-19) and a non-linear model that characterizes the genetic toggle switch. new fundamental properties that characterize these models are determined and discussed in detail through a comparison with the state-of-the-art results. in particular, regarding the very popular hiv ode model and the genetic toggle switch model, the method automatically finds new important results that are in contrast with the results in the current literature.',\n",
       "  'categories': 'eess.sy cs.sy math.oc q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-11-24',\n",
       "  'updated': '2024-09-11',\n",
       "  'authors': ['agostino martinelli'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.13507'},\n",
       " {'title': 'modelling plausible scenarios for the omicron sars-cov-2 variant from   early-stage surveillance',\n",
       "  'id': '2211.13704',\n",
       "  'abstract': \"we used a spatially explicit agent-based model of sars-cov-2 transmission combined with spatially fine-grained covid-19 observation data from public health scotland to investigate the initial rise of the omicron (ba.1) variant of concern. we evaluated plausible scenarios for transmission rate advantage and vaccine immune escape relative to the delta variant based on the data that would have been available at that time. we also explored possible outcomes of different levels of imposed non-pharmaceutical intervention. the initial results of these scenarios were used to inform the scottish government in the early outbreak stages of the omicron variant.   using the model with parameters fit over the delta variant epidemic, some initial assumptions about omicron transmission rate advantage and vaccine escape, and a simple growth rate fitting procedure, we were able to capture the initial outbreak dynamics for omicron. we found that the modelled dynamics hold up to retrospective scrutiny. the modelled imposition of extra non-pharmaceutical interventions planned by the scottish government at the time would likely have little effect in light of the transmission rate advantage held by the omicron variant and the fact that the planned interventions would have occurred too late in the outbreak's trajectory. finally, we found that any assumptions made about the projected distribution of vaccines in the model population had little bearing on the outcome, in terms of outbreak size and timing. instead, it was the landscape of prior immunity that was most important.\",\n",
       "  'categories': 'q-bio.pe physics.soc-ph',\n",
       "  'doi': '10.1016/j.epidem.2024.100800',\n",
       "  'created': '2022-11-24',\n",
       "  'updated': '2024-11-29',\n",
       "  'authors': ['christopher j. banks',\n",
       "   'ewan colman',\n",
       "   'anthony wood',\n",
       "   'thomas doherty',\n",
       "   'rowland r. kao'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.13704'},\n",
       " {'title': 'simple bots breed social punishment in humans',\n",
       "  'id': '2211.13943',\n",
       "  'abstract': 'costly punishment has been suggested as a key mechanism for stabilizing cooperation in one-shot games. however, recent studies have revealed that the effectiveness of costly punishment can be diminished by second-order free riders (i.e., cooperators who never punish defectors) and antisocial punishers (i.e., defectors who punish cooperators). in a two-stage prisoner\\'s dilemma game, players not only need to choose between cooperation and defection in the first stage, but also need to decide whether to punish their opponent in the second stage. here, we extend the theory of punishment in one-shot games by introducing simple bots, who consistently choose prosocial punishment and do not change their actions over time. we find that this simple extension of the game allows prosocial punishment to dominate in well-mixed and networked populations, and that the minimum fraction of bots required for the dominance of prosocial punishment monotonically increases with increasing dilemma strength. furthermore, if humans possess a learning bias toward a \"copy the majority\" rule or if bots are present at higher degree nodes in scale-free networks, the fully dominance of prosocial punishment is still possible at a high dilemma strength. these results indicate that introducing bots can be a significant factor in establishing prosocial punishment. we therefore, provide a novel explanation for the evolution of prosocial punishment, and note that the contrasting results that emerge from the introduction of different types of bots also imply that the design of the bots matters.',\n",
       "  'categories': 'physics.soc-ph q-bio.pe',\n",
       "  'doi': '10.1098/rsif.2024.0019',\n",
       "  'created': '2022-11-25',\n",
       "  'updated': '2022-11-28',\n",
       "  'authors': ['chen shen',\n",
       "   'zhixue he',\n",
       "   'lei shi',\n",
       "   'zhen wang',\n",
       "   'jun tanimoto'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.13943'},\n",
       " {'title': 'non-invasive liver fibrosis screening on ct images using radiomics',\n",
       "  'id': '2211.14396',\n",
       "  'abstract': 'objectives: to develop and evaluate a radiomics machine learning model for detecting liver fibrosis on ct of the liver.   methods: for this retrospective, single-centre study, radiomic features were extracted from regions of interest (rois) on ct images of patients who underwent simultaneous liver biopsy and ct examinations. combinations of contrast, normalization, machine learning model, and feature selection method were determined based on their mean test area under the receiver operating characteristic curve (auc) on randomly placed rois. the combination and selected features with the highest auc were used to develop a final liver fibrosis screening model.   results: the study included 101 male and 68 female patients (mean age = 51.2 years $\\\\pm$ 14.7 [sd]). when averaging the auc across all combinations, non-contrast enhanced (nc) ct (auc, 0.6100; 95% ci: 0.5897, 0.6303) outperformed contrast-enhanced ct (auc, 0.5680; 95% ci: 0.5471, 0.5890). the combination of hyperparameters and features that yielded the highest auc was a logistic regression model with inputs features of maximum, energy, kurtosis, skewness, and small area high gray level emphasis extracted from non-contrast enhanced nc ct normalized using gamma correction with $\\\\gamma$ = 1.5 (auc, 0.7833; 95% ci: 0.7821, 0.7845), (sensitivity, 0.9091; 95% ci: 0.9091, 0.9091).   conclusions: radiomics-based machine learning models allow for the detection of liver fibrosis with reasonable accuracy and high sensitivity on nc ct. thus, these models can be used to non-invasively screen for liver fibrosis, contributing to earlier detection of the disease at a potentially curable stage.',\n",
       "  'categories': 'cs.cv cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-11-25',\n",
       "  'updated': '2024-02-26',\n",
       "  'authors': ['jay j. yoo',\n",
       "   'khashayar namdar',\n",
       "   'sean carey',\n",
       "   'sandra e. fischer',\n",
       "   'chris mcintosh',\n",
       "   'farzad khalvati',\n",
       "   'patrik rogalla'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.14396'},\n",
       " {'title': 'discontinuous shear thickening in biological tissue rheology',\n",
       "  'id': '2211.15015',\n",
       "  'abstract': 'during embryonic morphogenesis, tissues undergo dramatic deformations in order to form functional organs. similarly, in adult animals, living cells and tissues are continually subjected to forces and deformations. therefore, the success of embryonic development and the proper maintenance of physiological functions rely on the ability of cells to withstand mechanical stresses as well as their ability to flow in a collective manner. during these events, mechanical perturbations can originate from active processes at the single cell level, competing with external stresses exerted by surrounding tissues and organs. however, the study of tissue mechanics has been somewhat limited to either the response to external forces or to intrinsic ones. in this work, we use an active vertex model of a 2d confluent tissue to study the interplay of external deformations that are applied globally to a tissue with internal active stresses that arise locally at the cellular level due to cell motility. we elucidate in particular the way in which this interplay between globally external and locally internal active driving determines the emergent mechanical properties of the tissue as a whole. for a tissue in the vicinity of a solid-fluid jamming/unjamming transition, we uncover a host of fascinating rheological phenomena, including yielding, shear thinning, continuous shear thickening (cst) and discontinuous shear thickening (dst). these model predictions provide a framework for understanding the recently observed nonlinear rheological behaviors {\\\\it in vivo}.',\n",
       "  'categories': 'cond-mat.soft q-bio.to',\n",
       "  'doi': '10.1103/physrevx.14.011027',\n",
       "  'created': '2022-11-27',\n",
       "  'updated': '2024-01-18',\n",
       "  'authors': ['michael j. hertaeg', 'suzanne m. fielding', 'dapeng bi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2211.15015'},\n",
       " {'title': 'simulation and assimilation of the digital human brain',\n",
       "  'id': '2211.15963',\n",
       "  'abstract': 'here, we present the digital brain (db), a platform for simulating spiking neuronal networks at the large neuron scale of the human brain based on personalized magnetic-resonance-imaging data and biological constraints. the db aims to reproduce both the resting state and certain aspects of the action of the human brain. an architecture with up to 86 billion neurons and 14,012 gpus, including a two-level routing scheme between gpus to accelerate spike transmission up to 47.8 trillion neuronal synapses, was implemented as part of the simulations. we show that the db can reproduce blood-oxygen-level-dependent signals of the resting-state of the human brain with a high correlation coefficient, as well as interact with its perceptual input, as demonstrated in a visual task. these results indicate the feasibility of implementing a digital representation of the human brain, which can open the door to a broad range of potential applications.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-11-29',\n",
       "  'updated': '2024-10-24',\n",
       "  'authors': ['wenlian lu',\n",
       "   'xin du',\n",
       "   'jiexiang wang',\n",
       "   'longbin zeng',\n",
       "   'leijun ye',\n",
       "   'shitong xiang',\n",
       "   'qibao zheng',\n",
       "   'jie zhang',\n",
       "   'ningsheng xu',\n",
       "   'jianfeng feng'],\n",
       "  'affiliation': ['on behalf of the dtb consortium',\n",
       "   'on behalf of the dtb consortium',\n",
       "   'on behalf of the dtb consortium',\n",
       "   'on behalf of the dtb consortium',\n",
       "   'on behalf of the dtb consortium',\n",
       "   'on behalf of the dtb consortium',\n",
       "   'on behalf of the dtb consortium',\n",
       "   'on behalf of the dtb consortium',\n",
       "   'on behalf of the dtb consortium',\n",
       "   'on behalf of the dtb consortium'],\n",
       "  'url': 'https://arxiv.org/abs/2211.15963'},\n",
       " {'title': 'automated coronary arteries labeling via geometric deep learning',\n",
       "  'id': '2212.00386',\n",
       "  'abstract': 'automatic labelling of anatomical structures, such as coronary arteries, is critical for diagnosis, yet existing (non-deep learning) methods are limited by a reliance on prior topological knowledge of the expected tree-like structures. as the structure such vascular systems is often difficult to conceptualize, graph-based representations have become popular due to their ability to capture the geometric and topological properties of the morphology in an orientation-independent and abstract manner. however, graph-based learning for automated labeling of tree-like anatomical structures has received limited attention in the literature. the majority of prior studies have limitations in the entity graph construction, are dependent on topological structures, and have limited accuracy due to the anatomical variability between subjects. in this paper, we propose an intuitive graph representation method, well suited to use with 3d coordinate data obtained from angiography scans. we subsequently seek to analyze subject-specific graphs using geometric deep learning. the proposed models leverage expert annotated labels from 141 patients to learn representations of each coronary segment, while capturing the effects of anatomical variability within the training data. we investigate different variants of so-called message passing neural networks. through extensive evaluations, our pipeline achieves a promising weighted f1-score of 0.805 for labeling coronary artery (13 classes) for a five-fold cross-validation. considering the ability of graph models in dealing with irregular data, and their scalability for data segmentation, this work highlights the potential of such methods to provide quantitative evidence to support the decisions of medical experts.',\n",
       "  'categories': 'eess.iv cs.cv q-bio.to',\n",
       "  'doi': '10.1109/isbi53787.2023.10230357',\n",
       "  'created': '2022-12-01',\n",
       "  'updated': '',\n",
       "  'authors': ['yadan li',\n",
       "   'mohammad ali armin',\n",
       "   'simon denman',\n",
       "   'david ahmedt-aristizabal'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.00386'},\n",
       " {'title': 'language models and brains align due to more than next-word prediction   and word-level information',\n",
       "  'id': '2212.00596',\n",
       "  'abstract': 'pretrained language models have been shown to significantly predict brain recordings of people comprehending language. recent work suggests that the prediction of the next word is a key mechanism that contributes to this alignment. what is not yet understood is whether prediction of the next word is necessary for this observed alignment or simply sufficient, and whether there are other shared mechanisms or information that are similarly important. in this work, we take a step towards understanding the reasons for brain alignment via two simple perturbations in popular pretrained language models. these perturbations help us design contrasts that can control for different types of information. by contrasting the brain alignment of these differently perturbed models, we show that improvements in alignment with brain recordings are due to more than improvements in next-word prediction and word-level information.',\n",
       "  'categories': 'cs.cl q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-01',\n",
       "  'updated': '2024-10-03',\n",
       "  'authors': ['gabriele merlin', 'mariya toneva'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.00596'},\n",
       " {'title': 'viral rna as a branched polymer',\n",
       "  'id': '2212.00829',\n",
       "  'abstract': 'myriad viruses use positive-strand rna molecules as their genomes. far from being only a repository of genetic material, viral rna performs numerous other functions mediated by its physical structure and chemical properties. in this chapter, we focus on its structure and discuss how long rna molecules can be treated as branched polymers through planar graphs. we describe the major results that can be obtained by this approach, in particular the observation that viral rna genomes have a characteristic compactness that sets them aside from similar random rnas. we also discuss how different parameters used in the current rna folding software influence the resulting structures and how they can be related to experimentally observable quantities. finally, we show how the connection to branched polymers can be extended to take advantage of known results from polymer physics and can be further moulded to include additional interactions, such as excluded volume or electrostatics.',\n",
       "  'categories': 'physics.bio-ph cond-mat.soft q-bio.bm',\n",
       "  'doi': '10.1007/978-3-031-36815-8_1',\n",
       "  'created': '2022-12-01',\n",
       "  'updated': '2023-03-27',\n",
       "  'authors': ['domen vaupotič',\n",
       "   'angelo rosa',\n",
       "   'rudolf podgornik',\n",
       "   'luca tubiana',\n",
       "   'anže božič'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.00829'},\n",
       " {'title': 'martinize2 and vermouth: unified framework for topology generation',\n",
       "  'id': '2212.01191',\n",
       "  'abstract': 'ongoing advances in force field and computer hardware development enable the use of molecular dynamics (md) to simulate increasingly complex systems with the ultimate goal of reaching cellular complexity. at the same time, rational design by high-throughput (ht) simulations is another forefront of md. in these areas, the martini coarse-grained force field, especially the latest version (i.e. v3), is being actively explored because it offers enhanced spatial-temporal resolution. however, the automation tools for preparing simulations with the martini force field, accompanying the previous version, were not designed for ht simulations or studies of complex cellular systems. therefore, they become a major limiting factor. to address these shortcomings, we present the open-source vermouth python library. vermouth is designed to become the unified framework for developing programs, which prepare, run, and analyze martini simulations of complex systems. to demonstrate the power of the vermouth library, the martinize2 program is showcased as a generalization of the martinize script, originally aimed to set up simulations of proteins. in contrast to the previous version, martinize2 automatically handles protonation states in proteins and post-translation modifications, offers more options to fine-tune structural biases such as the elastic network, and can convert non-protein molecules such as ligands. finally, martinize2 is used in two high-complexity benchmarks. the entire i-tasser protein template database as well as a subset of 200,000 structures from the alphafold protein structure database are converted to cg resolution and we illustrate how the checks on input structure quality can safeguard ht applications.',\n",
       "  'categories': 'q-bio.qm cond-mat.other cs.ce',\n",
       "  'doi': '',\n",
       "  'created': '2022-11-29',\n",
       "  'updated': '2024-04-09',\n",
       "  'authors': ['peter c. kroon',\n",
       "   'fabian grünewald',\n",
       "   'jonathan barnoud',\n",
       "   'marco van tilburg',\n",
       "   'paulo c. t. souza',\n",
       "   'tsjerk a. wassenaar',\n",
       "   'siewert-jan marrink'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.01191'},\n",
       " {'title': 'exchangeable coalescents beyond the cannings class',\n",
       "  'id': '2212.02154',\n",
       "  'abstract': \"we propose a general framework for the study of the genealogy of neutral discrete-time populations. we remove the standard assumption of exchangeability of offspring distributions appearing in cannings' models, and replace it by a less restrictive condition of non-heritability of reproductive success. we provide a general criterion for the weak convergence of their genealogies to $\\\\xi$-coalescents, and apply it to a simple parametrization of our scenario (which, under mild conditions, we also prove to essentially include the general case). we provide examples for such populations, including models with highly-asymmetric offspring distributions and populations undergoing random but recurrent bottlenecks. finally we study the limit genealogy of a new exponential model which, as previously shown for related models and in spite of its built in (fitness) inheritance mechanism, can be brought into our setting.\",\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-05',\n",
       "  'updated': '2024-04-17',\n",
       "  'authors': ['arno siri-jégousse', 'alejandro h. wences'],\n",
       "  'affiliation': ['iimas - unam', 'laas-cnrs'],\n",
       "  'url': 'https://arxiv.org/abs/2212.02154'},\n",
       " {'title': 'systematic analysis of biomolecular conformational ensembles with pensa',\n",
       "  'id': '2212.02714',\n",
       "  'abstract': 'atomic-level simulations are widely used to study biomolecules and their dynamics. a common goal in such studies is to compare simulations of a molecular system under several conditions -- for example, with various mutations or bound ligands -- in order to identify differences between the molecular conformations adopted under these conditions. however, the large amount of data produced by simulations of ever larger and more complex systems often renders it difficult to identify the structural features that are relevant for a particular biochemical phenomenon. we present a flexible software package named pensa that enables a comprehensive and thorough investigation into biomolecular conformational ensembles. it provides featurizations and feature transformations that allow for a complete representation of biomolecules like proteins and nucleic acids, including water and ion binding sites, thus avoiding bias that would come with manual feature selection. pensa implements methods to systematically compare the distributions of molecular features across ensembles to find the significant differences between them and identify regions of interest. it also includes a novel approach to quantify the state-specific information between two regions of a biomolecule, which allows, e.g., tracing information flow to identify allosteric pathways. pensa also comes with convenient tools for loading data and visualizing results, making them quick to process and easy to interpret. pensa is an open-source python library maintained at https://github.com/drorlab/pensa along with an example workflow and a tutorial. we demonstrate its usefulness in real-world examples by showing how it helps to determine molecular mechanisms efficiently.',\n",
       "  'categories': 'q-bio.bm physics.bio-ph physics.comp-ph',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-05',\n",
       "  'updated': '2024-08-18',\n",
       "  'authors': ['martin vögele',\n",
       "   'neil j. thomson',\n",
       "   'sang t. truong',\n",
       "   'jasper mcavity',\n",
       "   'ulrich zachariae',\n",
       "   'ron o. dror'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.02714'},\n",
       " {'title': 'bipmap: a toolbox for predictions of perceived motion artifacts on   modern displays',\n",
       "  'id': '2212.03854',\n",
       "  'abstract': 'presenting dynamic scenes without incurring motion artifacts visible to observers requires sustained effort from the display industry. a tool that predicts motion artifacts and simulates artifact elimination through optimizing the display configuration is highly desired to guide the design and manufacture of modern displays. despite the popular demands, there is no such tool available in the market. in this study, we deliver an interactive toolkit, binocular perceived motion artifact predictor (bipmap), as an executable file with gpu acceleration. bipmap accounts for an extensive collection of user-defined parameters and directly visualizes a variety of motion artifacts by presenting the perceived continuous and sampled moving stimuli side-by-side. for accurate artifact predictions, bipmap utilizes a novel model of the human contrast sensitivity function to effectively imitate the frequency modulation of the human visual system. in addition, bipmap is capable of deriving various in-plane motion artifacts for 2d displays and depth distortion in 3d stereoscopic displays.',\n",
       "  'categories': 'eess.sp cs.hc q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-07',\n",
       "  'updated': '2024-01-03',\n",
       "  'authors': ['guanghan meng',\n",
       "   'dekel galor',\n",
       "   'laura waller',\n",
       "   'martin s. banks'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.03854'},\n",
       " {'title': 'motion-invariant variational auto-encoding of brain structural   connectomes',\n",
       "  'id': '2212.04535',\n",
       "  'abstract': 'mapping of human brain structural connectomes via diffusion mri offers a unique opportunity to understand brain structural connectivity and relate it to various human traits, such as cognition. however, head displacement during image acquisition can compromise the accuracy of connectome reconstructions and subsequent inference results. we develop a generative model to learn low-dimensional representations of structural connectomes invariant to motion-induced artifacts, so that we can link brain networks and human traits more accurately, and generate motion-adjusted connectomes. we apply the proposed model to data from the adolescent brain cognitive development (abcd) study and the human connectome project (hcp) to investigate how our motion-invariant connectomes facilitate understanding of the brain network and its relationship with cognition. empirical results demonstrate that the proposed motion-invariant variational auto-encoder (inv-vae) outperforms its competitors in various aspects. in particular, motion-adjusted structural connectomes are more strongly associated with a wide array of cognition-related traits than other approaches without motion adjustment.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-08',\n",
       "  'updated': '2024-09-11',\n",
       "  'authors': ['yizi zhang', 'meimei liu', 'zhengwu zhang', 'david dunson'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.04535'},\n",
       " {'title': 'predicting shape development: a riemannian method',\n",
       "  'id': '2212.04740',\n",
       "  'abstract': \"predicting the future development of an anatomical shape from a single baseline observation is a challenging task. but it can be essential for clinical decision-making. research has shown that it should be tackled in curved shape spaces, as (e.g., disease-related) shape changes frequently expose nonlinear characteristics. we thus propose a novel prediction method that encodes the whole shape in a riemannian shape space. it then learns a simple prediction technique founded on hierarchical statistical modeling of longitudinal training data. when applied to predict the future development of the shape of the right hippocampus under alzheimer's disease and to human body motion, it outperforms deep learning-supported variants as well as state-of-the-art.\",\n",
       "  'categories': 'cs.cv math.dg q-bio.to',\n",
       "  'doi': '10.1007/978-3-031-46914-5_17',\n",
       "  'created': '2022-12-09',\n",
       "  'updated': '2023-08-28',\n",
       "  'authors': ['doğa türkseven',\n",
       "   'islem rekik',\n",
       "   'christoph von tycowicz',\n",
       "   'martin hanik'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.04740'},\n",
       " {'title': 'epidemiological agent-based modelling software (epiabm)',\n",
       "  'id': '2212.04937',\n",
       "  'abstract': 'epiabm is a fully tested, open-source software package for epidemiological agent-based modelling, re-implementing the well-known covidsim model from the mrc centre for global infectious disease analysis at imperial college london. it has been developed as part of the first-year training programme in the epsrc sabs:r3 centre for doctoral training at the university of oxford. the model builds an age-stratified, spatially heterogeneous population and offers a modular approach to configure and run epidemic scenarios, allowing for a broad scope of investigative and comparative studies. two simulation backends are provided: a pedagogical python backend (with full functionality) and a high performance c++ backend for use with larger population simulations. both are highly modular, with comprehensive testing and documentation for ease of understanding and extensibility. epiabm is publicly available through github at https://github.com/sabs-r3-epidemiology/epiabm.',\n",
       "  'categories': 'q-bio.pe q-bio.qm',\n",
       "  'doi': '10.5334/jors.449',\n",
       "  'created': '2022-12-07',\n",
       "  'updated': '',\n",
       "  'authors': ['kit gallagher',\n",
       "   'ioana bouros',\n",
       "   'nicholas fan',\n",
       "   'elizabeth hayman',\n",
       "   'luke heirene',\n",
       "   'patricia lamirande',\n",
       "   'annabelle lemenuel-diot',\n",
       "   'ben lambert',\n",
       "   'david gavaghan',\n",
       "   'richard creswell'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.04937'},\n",
       " {'title': 'targetcall: eliminating the wasted computation in basecalling via   pre-basecalling filtering',\n",
       "  'id': '2212.04953',\n",
       "  'abstract': \"basecalling is an essential step in nanopore sequencing analysis where the raw signals of nanopore sequencers are converted into nucleotide sequences, i.e., reads. state-of-the-art basecallers employ complex deep learning models to achieve high basecalling accuracy. this makes basecalling computationally inefficient and memory-hungry, bottlenecking the entire genome analysis pipeline. however, for many applications, the majority of reads do no match the reference genome of interest (i.e., target reference) and thus are discarded in later steps in the genomics pipeline, wasting the basecalling computation. to overcome this issue, we propose targetcall, the first pre-basecalling filter to eliminate the wasted computation in basecalling. targetcall's key idea is to discard reads that will not match the target reference (i.e., off-target reads) prior to basecalling. targetcall consists of two main components: (1) lightcall, a lightweight neural network basecaller that produces noisy reads; and (2) similarity check, which labels each of these noisy reads as on-target or off-target by matching them to the target reference. our thorough experimental evaluations show that targetcall 1) improves the end-to-end basecalling runtime performance of the state-of-the-art basecaller by 3.31x while maintaining high (98.88%) recall in keeping on-target reads, 2) maintains high accuracy in downstream analysis, and 3) achieves better runtime performance, throughput, recall, precision, and generality compared to prior works. targetcall is available at https://github.com/cmu-safari/targetcall.\",\n",
       "  'categories': 'q-bio.gn cs.ai cs.lg',\n",
       "  'doi': '10.3389/fgene.2024.1429306',\n",
       "  'created': '2022-12-09',\n",
       "  'updated': '2024-10-23',\n",
       "  'authors': ['meryem banu cavlak',\n",
       "   'gagandeep singh',\n",
       "   'mohammed alser',\n",
       "   'can firtina',\n",
       "   'joël lindegger',\n",
       "   'mohammad sadrosadati',\n",
       "   'nika mansouri ghiasi',\n",
       "   'can alkan',\n",
       "   'onur mutlu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.04953'},\n",
       " {'title': 'phenomenological modeling of diverse and heterogeneous synaptic dynamics   at natural density',\n",
       "  'id': '2212.05354',\n",
       "  'abstract': 'this chapter sheds light on the synaptic organization of the brain from the perspective of computational neuroscience. it provides an introductory overview on how to account for empirical data in mathematical models, implement such models in software, and perform simulations reflecting experiments. this path is demonstrated with respect to four key aspects of synaptic signaling: the connectivity of brain networks, synaptic transmission, synaptic plasticity, and the heterogeneity across synapses. each step and aspect of the modeling and simulation workflow comes with its own challenges and pitfalls, which are highlighted and addressed.',\n",
       "  'categories': 'q-bio.nc cs.ne',\n",
       "  'doi': '10.1007/978-1-0716-4019-7_15',\n",
       "  'created': '2022-12-10',\n",
       "  'updated': '2023-02-19',\n",
       "  'authors': ['agnes korcsak-gorzo',\n",
       "   'charl linssen',\n",
       "   'jasper albers',\n",
       "   'stefan dasbach',\n",
       "   'renato duarte',\n",
       "   'susanne kunkel',\n",
       "   'abigail morrison',\n",
       "   'johanna senk',\n",
       "   'jonas stapmanns',\n",
       "   'tom tetzlaff',\n",
       "   'markus diesmann',\n",
       "   'sacha j. van albada'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.05354'},\n",
       " {'title': 'numerical stability of deepgoplus inference',\n",
       "  'id': '2212.06361',\n",
       "  'abstract': \"convolutional neural networks (cnns) are currently among the most widely-used deep neural network (dnn) architectures available and achieve state-of-the-art performance for many problems. originally applied to computer vision tasks, cnns work well with any data with a spatial relationship, besides images, and have been applied to different fields. however, recent works have highlighted numerical stability challenges in dnns, which also relates to their known sensitivity to noise injection. these challenges can jeopardise their performance and reliability. this paper investigates deepgoplus, a cnn that predicts protein function. deepgoplus has achieved state-of-the-art performance and can successfully take advantage and annotate the abounding protein sequences emerging in proteomics. we determine the numerical stability of the model's inference stage by quantifying the numerical uncertainty resulting from perturbations of the underlying floating-point data. in addition, we explore the opportunity to use reduced-precision floating point formats for deepgoplus inference, to reduce memory consumption and latency. this is achieved by instrumenting deepgoplus' execution using monte carlo arithmetic, a technique that experimentally quantifies floating point operation errors and vprec, a tool that emulates results with customizable floating point precision formats. focus is placed on the inference stage as it is the primary deliverable of the deepgoplus model, widely applicable across different environments. all in all, our results show that although the deepgoplus cnn is very stable numerically, it can only be selectively implemented with lower-precision floating-point formats. we conclude that predictions obtained from the pre-trained deepgoplus model are very reliable numerically, and use existing floating-point formats efficiently.\",\n",
       "  'categories': 'cs.lg q-bio.qm',\n",
       "  'doi': '10.1371/journal.pone.0296725',\n",
       "  'created': '2022-12-12',\n",
       "  'updated': '2024-02-28',\n",
       "  'authors': ['inés gonzalez pepe',\n",
       "   'yohan chatelain',\n",
       "   'gregory kiar',\n",
       "   'tristan glatard'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.06361'},\n",
       " {'title': 'mechanistic description of spontaneous loss of memory persistent   activity based on neuronal synaptic strength',\n",
       "  'id': '2212.07225',\n",
       "  'abstract': 'persistent neural activity associated with working memory (wm) lasts for a limited time duration. current theories suggest that its termination is \\\\textit{actively} obtained via inhibitory currents, and there is currently no theory regarding the possibility of a \\\\textit{passive} memory-loss mechanism that terminates memory persistent activity. here, we develop an analytical-framework, based on synaptic strength, and show via simulations and fitting to wet-lab experiments, that passive memory-loss might be a result of an ionic-current long-term plateau, i.e., very slow reduction of memory followed by abrupt loss. we describe analytically the plateau, when the memory state is just below criticality. these results, including the plateau, are supported by experiments performed on rats. moreover, we show that even just above criticality, forgetfulness can occur due to neuronal noise with ionic-current fluctuations, yielding a plateau, representing memory with very slow decay, and eventually a fast memory decay. our results could have implications for developing new medications, targeted against memory impairments, through modifying neuronal noise.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-14',\n",
       "  'updated': '2024-02-01',\n",
       "  'authors': ['hillel sanhedrai', 'shlomo havlin', 'hila dvir'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.07225'},\n",
       " {'title': 'cherry picking in forests: a new characterization for the unrooted   hybrid number of two phylogenetic trees',\n",
       "  'id': '2212.08145',\n",
       "  'abstract': 'phylogenetic networks are a special type of graph which generalize phylogenetic trees and that are used to model non-treelike evolutionary processes such as recombination and hybridization. in this paper, we consider {\\\\em unrooted} phylogenetic networks, i.e. simple, connected graphs $\\\\mathcal{n}=(v,e)$ with leaf set $x$, for $x$ some set of species, in which every internal vertex in $\\\\mathcal{n}$ has degree three. one approach used to construct such phylogenetic networks is to take as input a collection $\\\\mathcal{p}$ of phylogenetic trees and to look for a network $\\\\mathcal{n}$ that contains each tree in $\\\\mathcal{p}$ and that minimizes the quantity $r(\\\\mathcal{n}) = |e|-(|v|-1)$ over all such networks. such a network always exists, and the quantity $r(\\\\mathcal{n})$ for an optimal network $\\\\mathcal{n}$ is called the hybrid number of $\\\\mathcal{p}$. in this paper, we give a new characterization for the hybrid number in case $\\\\mathcal{p}$ consists of two trees. this characterization is given in terms of a cherry picking sequence for the two trees, although to prove that our characterization holds we need to define the sequence more generally for two forests. cherry picking sequences have been intensively studied for collections of rooted phylogenetic trees, but our new sequences are the first variant of this concept that can be applied in the unrooted setting. since the hybrid number of two trees is equal to the well-known tree bisection and reconnection distance between the two trees, our new characterization also provides an alternative way to understand this important tree distance.',\n",
       "  'categories': 'math.co q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-15',\n",
       "  'updated': '2024-11-29',\n",
       "  'authors': ['katharina t. huber', 'simone linz', 'vincent moulton'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.08145'},\n",
       " {'title': 'narrow escape in composite domains forming heterogeneous networks',\n",
       "  'id': '2212.09179',\n",
       "  'abstract': 'cellular networks are often composed of thin tubules connecting much larger node compartments. these structures serve for active or diffusion transport of proteins. examples are glial networks in the brain, the endoplasmic reticulum in cells or dendritic spines located on dendrites. in this latter case, a large ball forming the head is connected by a narrow passage. in all cases, how the transport of molecules, ions or proteins is regulated determines the time scale of chemical reactions or signal transduction. in the present study, based on modeling diffusion in three dimensions, we compute the mean time for a brownian particle to reach a narrow target inside such a composite network made of tubules connected to spherical nodes. we derive asymptotic formulas by solving a mixed neumann-dirichlet boundary value problem with small dirichlet part. we first consider the case of a network domain organized in a 2-d lattice structure that consists of spherical ball compartments connected via narrow cylindrical passages. when there is a single target we derive a matrix equation for each mean first passage time (mfpt) averaged over each spherical compartment. we then consider a composite domain consisting of a spherical head-like domain connected to a large cylinder via a narrow cylindrical neck. for brownian particles starting within the narrow neck, we derive formulas for the mfpt to reach a target on the spherical head. when diffusing particles can be absorbed upon hitting additional absorbing boundaries of the large cylinder, we compute the probability and conditional mfpt to reach a target. we compare these formulas with numerical solutions of the mixed boundary value problem and with brownian simulations. to conclude, the present analysis reveals that the mean arrival time, driven by diffusion in heterogeneous networks, is controlled by the target and narrow passage sizes.',\n",
       "  'categories': 'cond-mat.soft math.ap q-bio.qm',\n",
       "  'doi': '10.1016/j.physd.2023.133837',\n",
       "  'created': '2022-12-18',\n",
       "  'updated': '',\n",
       "  'authors': ['frédéric paquin-lefebvre',\n",
       "   'kanishka basnayake',\n",
       "   'david holcman'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.09179'},\n",
       " {'title': 'denoising instrumented mouthguard measurements of head impact kinematics   with a convolutional neural network',\n",
       "  'id': '2212.09832',\n",
       "  'abstract': 'wearable sensors for measuring head kinematics can be noisy due to imperfect interfaces with the body. mouthguards are used to measure head kinematics during impacts in traumatic brain injury (tbi) studies, but deviations from reference kinematics can still occur due to potential looseness. in this study, deep learning is used to compensate for the imperfect interface and improve measurement accuracy. a set of one-dimensional convolutional neural network (1d-cnn) models was developed to denoise mouthguard kinematics measurements along three spatial axes of linear acceleration and angular velocity. the denoised kinematics had significantly reduced errors compared to reference kinematics, and reduced errors in brain injury criteria and tissue strain and strain rate calculated via finite element modeling. the 1d-cnn models were also tested on an on-field dataset of college football impacts and a post-mortem human subject dataset, with similar denoising effects observed. the models can be used to improve detection of head impacts and tbi risk evaluation, and potentially extended to other sensors measuring kinematics.',\n",
       "  'categories': 'cs.lg eess.sp q-bio.qm q-bio.to',\n",
       "  'doi': '10.1109/tbme.2024.3392537',\n",
       "  'created': '2022-12-19',\n",
       "  'updated': '',\n",
       "  'authors': ['xianghao zhan',\n",
       "   'yuzhe liu',\n",
       "   'nicholas j. cecchi',\n",
       "   'ashlyn a. callan',\n",
       "   'enora le flao',\n",
       "   'olivier gevaert',\n",
       "   'michael m. zeineh',\n",
       "   'gerald a. grant',\n",
       "   'david b. camarillo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.09832'},\n",
       " {'title': 'learning efficient backprojections across cortical hierarchies in real   time',\n",
       "  'id': '2212.10249',\n",
       "  'abstract': 'models of sensory processing and learning in the cortex need to efficiently assign credit to synapses in all areas. in deep learning, a known solution is error backpropagation, which however requires biologically implausible weight transport from feed-forward to feedback paths.   we introduce phaseless alignment learning (pal), a bio-plausible method to learn efficient feedback weights in layered cortical hierarchies. this is achieved by exploiting the noise naturally found in biophysical systems as an additional carrier of information. in our dynamical system, all weights are learned simultaneously with always-on plasticity and using only information locally available to the synapses. our method is completely phase-free (no forward and backward passes or phased learning) and allows for efficient error propagation across multi-layer cortical hierarchies, while maintaining biologically plausible signal transport and learning.   our method is applicable to a wide class of models and improves on previously known biologically plausible ways of credit assignment: compared to random synaptic feedback, it can solve complex tasks with less neurons and learn more useful latent representations. we demonstrate this on various classification tasks using a cortical microcircuit model with prospective coding.',\n",
       "  'categories': 'q-bio.nc cs.lg cs.ne',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-20',\n",
       "  'updated': '2024-02-02',\n",
       "  'authors': ['kevin max',\n",
       "   'laura kriener',\n",
       "   'garibaldi pineda garcía',\n",
       "   'thomas nowotny',\n",
       "   'ismael jaras',\n",
       "   'walter senn',\n",
       "   'mihai a. petrovici'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.10249'},\n",
       " {'title': 'multi-modal molecule structure-text model for text-based retrieval and   editing',\n",
       "  'id': '2212.10789',\n",
       "  'abstract': \"there is increasing adoption of artificial intelligence in drug discovery. however, existing studies use machine learning to mainly utilize the chemical structures of molecules but ignore the vast textual knowledge available in chemistry. incorporating textual knowledge enables us to realize new drug design objectives, adapt to text-based instructions and predict complex biological activities. here we present a multi-modal molecule structure-text model, moleculestm, by jointly learning molecules' chemical structures and textual descriptions via a contrastive learning strategy. to train moleculestm, we construct a large multi-modal dataset, namely, pubchemstm, with over 280,000 chemical structure-text pairs. to demonstrate the effectiveness and utility of moleculestm, we design two challenging zero-shot tasks based on text instructions, including structure-text retrieval and molecule editing. moleculestm has two main properties: open vocabulary and compositionality via natural language. in experiments, moleculestm obtains the state-of-the-art generalization ability to novel biochemical concepts across various benchmarks.\",\n",
       "  'categories': 'cs.lg cs.cl q-bio.qm stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-21',\n",
       "  'updated': '2024-01-29',\n",
       "  'authors': ['shengchao liu',\n",
       "   'weili nie',\n",
       "   'chengpeng wang',\n",
       "   'jiarui lu',\n",
       "   'zhuoran qiao',\n",
       "   'ling liu',\n",
       "   'jian tang',\n",
       "   'chaowei xiao',\n",
       "   'anima anandkumar'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.10789'},\n",
       " {'title': 'forecasting west nile virus with graph neural networks: harnessing   spatial dependence in irregularly sampled geospatial data',\n",
       "  'id': '2212.11367',\n",
       "  'abstract': 'machine learning methods have seen increased application to geospatial environmental problems, such as precipitation nowcasting, haze forecasting, and crop yield prediction. however, many of the machine learning methods applied to mosquito population and disease forecasting do not inherently take into account the underlying spatial structure of the given data. in our work, we apply a spatially aware graph neural network model consisting of graphsage layers to forecast the presence of west nile virus in illinois, to aid mosquito surveillance and abatement efforts within the state. more generally, we show that graph neural networks applied to irregularly sampled geospatial data can exceed the performance of a range of baseline methods including logistic regression, xgboost, and fully-connected neural networks.',\n",
       "  'categories': 'q-bio.pe cs.lg q-bio.qm stat.ap',\n",
       "  'doi': '10.1029/2023gh000784',\n",
       "  'created': '2022-12-21',\n",
       "  'updated': '',\n",
       "  'authors': ['adam tonks',\n",
       "   'trevor harris',\n",
       "   'bo li',\n",
       "   'william brown',\n",
       "   'rebecca smith'],\n",
       "  'affiliation': ['department of statistics, university of illinois at urbana-champaign',\n",
       "   'department of statistics, texas a&m university',\n",
       "   'department of statistics, university of illinois at urbana-champaign',\n",
       "   'department of pathobiology, university of illinois at urbana-champaign',\n",
       "   'department of pathobiology, university of illinois at urbana-champaign'],\n",
       "  'url': 'https://arxiv.org/abs/2212.11367'},\n",
       " {'title': 'combining mutation and recombination statistics to infer clonal families   in antibody repertoires',\n",
       "  'id': '2212.11997',\n",
       "  'abstract': 'b-cell repertoires are characterized by a diverse set of receptors of distinct specificities generated through two processes of somatic diversification: v(d)j recombination and somatic hypermutations. b cell clonal families stem from the same v(d)j recombination event, but differ in their hypermutations. clonal families identification is key to understanding b-cell repertoire function, evolution and dynamics. we present hilary (high-precision inference of lineages in antibody repertoires), an efficient, fast and precise method to identify clonal families from single- or paired-chain repertoire sequencing datasets. hilary combines probabilistic models that capture the receptor generation and selection statistics with adapted clustering methods to achieve consistently high inference accuracy. it automatically leverages the phylogenetic signal of shared mutations in difficult repertoire subsets. exploiting the high sensitivity of the method, we find the statistics of evolutionary properties such as the site frequency spectrum and dn/ds ratio do not depend on the junction length. we also identify a broad range of selection pressures spanning two orders of magnitude.',\n",
       "  'categories': 'q-bio.pe q-bio.gn',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-22',\n",
       "  'updated': '2024-03-15',\n",
       "  'authors': ['natanael spisak',\n",
       "   'gabriel athènes',\n",
       "   'thomas dupic',\n",
       "   'thierry mora',\n",
       "   'aleksandra m. walczak'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.11997'},\n",
       " {'title': 'a filtering approach for statistical inference in a stochastic sir model   with an application to covid-19 data',\n",
       "  'id': '2212.13443',\n",
       "  'abstract': 'in this paper, we consider a discrete-time stochastic sir model, where the transmission rate and the true number of infectious individuals are random and unobservable. an advantage of this model is that it permits us to account for random fluctuations in infectiousness and for non-detected infections. however, a difficulty arises because statistical inference has to be done in a partial information setting. we adopt a nested particle filtering approach to estimate the reproduction rate and the model parameters. as a case study, we apply our methodology to austrian covid-19 infection data. moreover, we discuss forecasts and model tests.',\n",
       "  'categories': 'physics.soc-ph math.pr q-bio.pe stat.ap',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-27',\n",
       "  'updated': '2024-01-28',\n",
       "  'authors': ['katia colaneri', 'camilla damian', 'rüdiger frey'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.13443'},\n",
       " {'title': 'deciphering rna secondary structure prediction: a probabilistic k-rook   matching perspective',\n",
       "  'id': '2212.14041',\n",
       "  'abstract': 'the secondary structure of ribonucleic acid (rna) is more stable and accessible in the cell than its tertiary structure, making it essential for functional prediction. although deep learning has shown promising results in this field, current methods suffer from poor generalization and high complexity. in this work, we reformulate the rna secondary structure prediction as a k-rook problem, thereby simplifying the prediction process into probabilistic matching within a finite solution space. building on this innovative perspective, we introduce rfold, a simple yet effective method that learns to predict the most matching k-rook solution from the given sequence. rfold employs a bi-dimensional optimization strategy that decomposes the probabilistic matching problem into row-wise and column-wise components to reduce the matching complexity, simplifying the solving process while guaranteeing the validity of the output. extensive experiments demonstrate that rfold achieves competitive performance and about eight times faster inference efficiency than the state-of-the-art approaches. the code and colab demo are available in (http://github.com/a4bio/rfold).',\n",
       "  'categories': 'q-bio.bm cs.ai cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-02',\n",
       "  'updated': '2024-06-19',\n",
       "  'authors': ['cheng tan',\n",
       "   'zhangyang gao',\n",
       "   'hanqun cao',\n",
       "   'xingran chen',\n",
       "   'ge wang',\n",
       "   'lirong wu',\n",
       "   'jun xia',\n",
       "   'jiangbin zheng',\n",
       "   'stan z. li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.14041'},\n",
       " {'title': 'spatial structure supports diversity in prebiotic autocatalytic chemical   ecosystems',\n",
       "  'id': '2212.14445',\n",
       "  'abstract': 'autocatalysis is thought to have played an important role in the earliest stages of the origin of life. an autocatalytic cycle (ac) is a set of reactions that results in stoichiometric increase in its constituent chemicals. when the reactions of multiple interacting acs are active in a region of space, they can have interactions analogous to those between species in biological ecosystems. prior studies of autocatalytic chemical ecosystems (aces) have suggested avenues for accumulating complexity, such as ecological succession, as well as obstacles such as competitive exclusion. we extend this ecological framework to investigate the effects of surface adsorption, desorption, and diffusion on ace ecology. simulating aces as particle-based stochastic reaction-diffusion systems in spatial environments-including open, two-dimensional reaction-diffusion systems and adsorptive mineral surfaces-we demonstrate that spatial structure can enhance ace diversity by i) permitting otherwise mutually exclusive acs to coexist and ii) subjecting new ac traits to selection.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-29',\n",
       "  'updated': '2024-07-25',\n",
       "  'authors': ['alex m. plum',\n",
       "   'christopher p. kempes',\n",
       "   'zhen peng',\n",
       "   'david a. baum'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2212.14445'},\n",
       " {'title': 'mapping effective connectivity by virtually perturbing a surrogate brain',\n",
       "  'id': '2301.00148',\n",
       "  'abstract': \"effective connectivity (ec), indicative of the causal interactions between brain regions, is fundamental to understanding information processing in the brain. traditional approaches, which infer ec from neural responses to stimulations, are not suited for mapping whole-brain ec in humans due to being invasive and having limited spatial coverage of stimulations. to address this gap, we present neural perturbational inference (npi), a data-driven framework designed to map ec across the entire brain. npi employs an artificial neural network trained to learn large-scale neural dynamics as a computational surrogate of the brain. npi maps ec by perturbing each region of the surrogate brain and observing the resulting responses in all other regions. npi captures the directionality, strength, and excitatory/inhibitory properties of brain-wide ec. our validation of npi, using models having ground-truth ec, shows its superiority over granger causality and dynamic causal modeling. applying npi to resting-state fmri data from diverse datasets reveals consistent and structurally supported ec. further validation using a cortico-cortical evoked potentials dataset reveals a significant correlation between npi-inferred ec and real stimulation propagation pathways. by transitioning from correlational to causal understandings of brain functionality, npi marks a stride in decoding the brain's functional architecture and facilitating both neuroscience research and clinical applications.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-31',\n",
       "  'updated': '2024-09-26',\n",
       "  'authors': ['zixiang luo',\n",
       "   'kaining peng',\n",
       "   'zhichao liang',\n",
       "   'shengyuan cai',\n",
       "   'chenyu xu',\n",
       "   'dan li',\n",
       "   'yu hu',\n",
       "   'changsong zhou',\n",
       "   'quanying liu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.00148'},\n",
       " {'title': 'a compositional account of motifs, mechanisms, and dynamics in   biochemical regulatory networks',\n",
       "  'id': '2301.01445',\n",
       "  'abstract': 'regulatory networks depict promoting or inhibiting interactions between molecules in a biochemical system. we introduce a category-theoretic formalism for regulatory networks, using signed graphs to model the networks and signed functors to describe occurrences of one network in another, especially occurrences of network motifs. with this foundation, we establish functorial mappings between regulatory networks and other mathematical models in biochemistry. we construct a functor from reaction networks, modeled as petri nets with signed links, to regulatory networks, enabling us to precisely define when a reaction network could be a physical mechanism underlying a regulatory network. turning to quantitative models, we associate a regulatory network with a lotka-volterra system of differential equations, defining a functor from the category of signed graphs to a category of parameterized dynamical systems. we extend this result from closed to open systems, demonstrating that lotka-volterra dynamics respects not only inclusions and collapsings of regulatory networks, but also the process of building up complex regulatory networks by gluing together simpler pieces. formally, we use the theory of structured cospans to produce a lax double functor from the double category of open signed graphs to that of open parameterized dynamical systems. throughout the paper, we ground the categorical formalism in examples inspired by systems biology.',\n",
       "  'categories': 'q-bio.mn math.ct',\n",
       "  'doi': '10.32408/compositionality-6-2',\n",
       "  'created': '2023-01-03',\n",
       "  'updated': '2024-05-05',\n",
       "  'authors': ['rebekah aduddell',\n",
       "   'james fairbanks',\n",
       "   'amit kumar',\n",
       "   'pablo s. ocal',\n",
       "   'evan patterson',\n",
       "   'brandon t. shapiro'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.01445'},\n",
       " {'title': 'ci-gnn: a granger causality-inspired graph neural network for   interpretable brain network-based psychiatric diagnosis',\n",
       "  'id': '2301.01642',\n",
       "  'abstract': 'there is a recent trend to leverage the power of graph neural networks (gnns) for brain-network based psychiatric diagnosis, which,in turn, also motivates an urgent need for psychiatrists to fully understand the decision behavior of the used gnns. however, most of the existing gnn explainers are either post-hoc in which another interpretive model needs to be created to explain a well-trained gnn, or do not consider the causal relationship between the extracted explanation and the decision, such that the explanation itself contains spurious correlations and suffers from weak faithfulness. in this work, we propose a granger causality-inspired graph neural network (ci-gnn), a built-in interpretable model that is able to identify the most influential subgraph (i.e., functional connectivity within brain regions) that is causally related to the decision (e.g., major depressive disorder patients or healthy controls), without the training of an auxillary interpretive network. ci-gnn learns disentangled subgraph-level representations {\\\\alpha} and \\\\b{eta} that encode, respectively, the causal and noncausal aspects of original graph under a graph variational autoencoder framework, regularized by a conditional mutual information (cmi) constraint. we theoretically justify the validity of the cmi regulation in capturing the causal relationship. we also empirically evaluate the performance of ci-gnn against three baseline gnns and four state-of-the-art gnn explainers on synthetic data and three large-scale brain disease datasets. we observe that ci-gnn achieves the best performance in a wide range of metrics and provides more reliable and concise explanations which have clinical evidence.the source code and implementation details of ci-gnn are freely available at github repository (https://github.com/zkz-brain/ci-gnn/).',\n",
       "  'categories': 'stat.ml cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-01-04',\n",
       "  'updated': '2024-01-28',\n",
       "  'authors': ['kaizhong zheng', 'shujian yu', 'badong chen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.01642'},\n",
       " {'title': 't-smiles: a scalable fragment-based molecular representation framework   for de novo molecule generation',\n",
       "  'id': '2301.01829',\n",
       "  'abstract': 'effective representation of molecules is a crucial factor affecting the performance of artificial intelligence models. this study introduces a flexible, fragment-based, multiscale molecular representation framework called t-smiles (tree-based smiles) with three code algorithms: tssa, tsdy and tsid. it describes molecules using smiles-type strings obtained by performing a breadth-first search on a full binary tree formed from a fragmented molecular graph. systematic evaluations using jtvae, brics, mmpa, and scaffold show the feasibility of constructing a multi-code molecular description system, where various descriptions complement each other, enhancing the overall performance. in addition, it can avoid overfitting and achieve higher novelty scores while maintaining reasonable similarity on labeled low-resource datasets, regardless of whether the model is original, data-augmented, or pre-trained then fine-tuned. furthermore, it significantly outperforms classical smiles, deepsmiles, selfies and baseline models in goal-directed tasks. and it surpasses state-of-the-art fragment, graph and smiles based approaches on chembl, zinc, and qm9.',\n",
       "  'categories': 'cs.lg cs.ai q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-01-04',\n",
       "  'updated': '2024-05-20',\n",
       "  'authors': ['juan-ni wu',\n",
       "   'tong wang',\n",
       "   'yue chen',\n",
       "   'li-juan tang',\n",
       "   'hai-long wu',\n",
       "   'ru-qin yu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.01829'},\n",
       " {'title': 'probabilistic genotype-phenotype maps reveal mutational robustness of   rna folding, spin glasses, and quantum circuits',\n",
       "  'id': '2301.01847',\n",
       "  'abstract': 'recent studies of genotype-phenotype (gp) maps have reported universally enhanced phenotypic robustness to genotype mutations, a feature essential to evolution. virtually all of these studies make a simplifying assumption that each genotype -- represented as a sequence -- maps deterministically to a single phenotype, such as a discrete structure. here, we introduce probabilistic genotype-phenotype (prgp) maps, where each genotype maps to a vector of phenotype probabilities, as a more realistic and universal language for investigating robustness in a variety of physical, biological, and computational systems. we study three model systems to show that prgp maps offer a generalized framework which can handle uncertainty emerging from various physical sources: (1) thermal fluctuation in rna folding, (2) external field disorder in spin glass ground state finding, and (3) superposition and entanglement in quantum circuits, which are realized experimentally on ibm quantum computers. in all three cases, we observe a novel biphasic robustness scaling which is enhanced relative to random expectation for more frequent phenotypes and approaches random expectation for less frequent phenotypes. we derive an analytical theory for the behavior of prgp robustness, and we demonstrate that the theory is highly predictive of empirical robustness.',\n",
       "  'categories': 'cond-mat.stat-mech cond-mat.dis-nn physics.bio-ph q-bio.pe quant-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-01-04',\n",
       "  'updated': '2024-08-22',\n",
       "  'authors': ['anna sappington', 'vaibhav mohanty'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.01847'},\n",
       " {'title': 'a data-driven gaussian process filter for electrocardiogram denoising',\n",
       "  'id': '2301.02607',\n",
       "  'abstract': 'objective: gaussian processes (gp)-based filters, which have been effectively used for various applications including electrocardiogram (ecg) filtering can be computationally demanding and the choice of their hyperparameters is typically ad hoc. methods: we develop a data-driven gp filter to address both issues, using the notion of the ecg phase domain -- a time-warped representation of the ecg beats onto a fixed number of samples and aligned r-peaks, which is assumed to follow a gaussian distribution. under this assumption, the computation of the sample mean and covariance matrix is simplified, enabling an efficient implementation of the gp filter in a data-driven manner, with no ad hoc hyperparameters. the proposed filter is evaluated and compared with a state-of-the-art wavelet-based filter, on the physionet qt database. the performance is evaluated by measuring the signal-to-noise ratio (snr) improvement of the filter at snr levels ranging from -5 to 30db, in 5db steps, using additive noise. for a clinical evaluation, the error between the estimated qt-intervals of the original and filtered signals is measured and compared with the benchmark filter. results: it is shown that the proposed gp filter outperforms the benchmark filter for all the tested noise levels. it also outperforms the state-of-the-art filter in terms of qt-interval estimation error bias and variance. conclusion: the proposed gp filter is a versatile technique for preprocessing the ecg in clinical and research applications, is applicable to ecg of arbitrary lengths and sampling frequencies, and provides confidence intervals for its performance.',\n",
       "  'categories': 'eess.sp cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-01-06',\n",
       "  'updated': '2024-01-09',\n",
       "  'authors': ['mircea dumitru',\n",
       "   'qiao li',\n",
       "   'erick andres perez alday',\n",
       "   'ali bahrami rad',\n",
       "   'gari d. clifford',\n",
       "   'reza sameni'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.02607'},\n",
       " {'title': 'does gaia play dice? : simple models of non-darwinian selection',\n",
       "  'id': '2301.02623',\n",
       "  'abstract': \"in this paper we introduce some simple models, based on rolling dice, to explore mechanisms proposed to explain planetary habitability. the idea is to study these selection mechanisms in an analytically tractable setting, isolating their consequences from other details which can confound or obscure their effect in more realistic models. we find that while the observable of interest, the face value shown on the die, `improves' over time in all models, for two of the more popular ideas, selection by survival and sequential selection, this is down to sampling effects. a modified version of sequential selection, sequential selection with memory, implies a statistical tendency for systems to improve over time. we discuss the implications of this and its relationship to the ideas of the `inhabitance paradox' and the `gaian bottleneck'.\",\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-01-06',\n",
       "  'updated': '2024-07-27',\n",
       "  'authors': ['rudy arthur', 'arwen nicholson'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.02623'},\n",
       " {'title': 'complex balanced distributions for chemical reaction networks',\n",
       "  'id': '2301.04091',\n",
       "  'abstract': 'reaction networks have become a major modelling framework in the biological sciences from epidemiology and population biology to genetics and cellular biology. in recent years, much progress has been made on stochastic reaction networks (srns) that in particular are applicable when copy numbers are low (as in cellular systems) or when noise and drift are ubiquitous (as in population genetic models). often a main interest is the long term behaviour of a system: does the system settle towards stochastic equilibrium in some sense? in that context, we are concerned with srns, modelled as continuous time markov chains (ctmcs) and their stationary distributions. in particular, we are interested in complex balanced stationary distributions, where the probability flow out of a complex equals the flow into the complex. we characterise the existence and the form of complex balanced distributions of srns with arbitrary transition functions through conditions on the cycles of a corresponding reaction graph (a digraph). furthermore, we give a sufficient condition for the existence of a complex balanced distribution and give precise conditions for when it is also necessary. the sufficient condition is also necessary for mass-action kinetics (and certain generalisations of that) or if the connected components of the digraph are cycles. moreover, we state a deficiency theorem, a generalisation of the deficiency theorem for stochastic mass-action kinetics to arbitrary stochastic genetics. the theorem gives the co-dimension of the parameter space for which a complex balanced distribution exists.',\n",
       "  'categories': 'math.pr q-bio.mn',\n",
       "  'doi': '',\n",
       "  'created': '2023-01-10',\n",
       "  'updated': '2024-11-22',\n",
       "  'authors': ['linard hoessly', 'carsten wiuf', 'panqiu xia'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.04091'},\n",
       " {'title': 'sir-model for households',\n",
       "  'id': '2301.04355',\n",
       "  'abstract': 'households play an important role in disease dynamics. many infections happening there due to the close contact, while mitigation measures mainly target the transmission between households. therefore, one can see households as boosting the transmission depending on household size. to study the effect of household size and size distribution, we differentiated the within and between household reproduction rate. there are basically no preventive measures, and thus the close contacts can boost the spread. we explicitly incorporated that typically only a fraction of all household members are infected. thus, viewing the infection of a household of a given size as a splitting process generating a new, small fully infected sub-household and a remaining still susceptible sub-household we derive a compartmental ode-model for the dynamics of the sub-households. in this setting, the basic reproduction number as well as prevalence and the peak of an infection wave in a population with given households size distribution can be computed analytically. we compare numerical simulation results of this novel household-ode model with results from an agent--based model using data for realistic household size distributions of different countries. we find good agreement of both models showing the catalytic effect of large households on the overall disease dynamics.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1137/23m1556861',\n",
       "  'created': '2023-01-11',\n",
       "  'updated': '',\n",
       "  'authors': ['philipp doenges',\n",
       "   'thomas götz',\n",
       "   'tyll krueger',\n",
       "   'karol niedzielewski',\n",
       "   'viola priesemann',\n",
       "   'moritz schaefer'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.04355'},\n",
       " {'title': 'evidence of scale-free clusters of vegetation in tropical rainforests',\n",
       "  'id': '2301.05917',\n",
       "  'abstract': \"tropical rainforests exhibit a rich repertoire of spatial patterns emerging from the intricate relationship between the microscopic interaction between species. in particular, the distribution of vegetation clusters can shed much light on the underlying process regulating the ecosystem. analyzing the distribution of vegetation clusters at different resolution scales, we show the first robust evidence of scale-invariant clusters of vegetation, suggesting the coexistence of multiple intertwined scales in the collective dynamics of tropical rainforests. we use field data and computational simulations to confirm our hypothesis, proposing a predictor that could be particularly interesting to monitor the ecological resilience of the world's 'green lungs'.\",\n",
       "  'categories': 'cond-mat.stat-mech cond-mat.dis-nn nlin.ao q-bio.pe',\n",
       "  'doi': '10.1103/physreve.109.l042402',\n",
       "  'created': '2023-01-14',\n",
       "  'updated': '2024-02-26',\n",
       "  'authors': ['pablo villegas',\n",
       "   'tommaso gili',\n",
       "   'guido caldarelli',\n",
       "   'andrea gabrielli'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.05917'},\n",
       " {'title': 'consciousness is entailed by compositional learning of new causal   structures in deep predictive processing systems',\n",
       "  'id': '2301.07016',\n",
       "  'abstract': \"machine learning algorithms have achieved superhuman performance in specific complex domains. however, learning online from few examples and compositional learning for efficient generalization across domains remain elusive. in humans, such learning includes specific declarative memory formation and is closely associated with consciousness. predictive processing has been advanced as a principled bayesian framework for understanding the cortex as implementing deep generative models for both sensory perception and action control. however, predictive processing offers little direct insight into fast compositional learning or of the separation between conscious and unconscious contents. here, propose that access consciousness arises as a consequence of a particular learning mechanism operating within a predictive processing system. we extend predictive processing by adding online, single-example new structure learning via hierarchical binding of unpredicted inferences. this system learns new causes by quickly connecting together novel combinations of perceptions, which manifests as working memories that can become short- and long-term declarative memories retrievable by associative recall. the contents of such bound representations are unified yet differentiated, can be maintained by selective attention and are globally available. the proposed learning process explains contrast and masking manipulations, postdictive perceptual integration, and other paradigm cases of consciousness research. 'phenomenal conscious experience' is how the learning system transparently models its own functioning, giving rise to perceptual illusions underlying the meta-problem of consciousness. our proposal naturally unifies the feature binding, recurrent processing, predictive processing, and global workspace theories of consciousness.\",\n",
       "  'categories': 'q-bio.nc cs.ai cs.lg cs.ne cs.ro',\n",
       "  'doi': '',\n",
       "  'created': '2023-01-17',\n",
       "  'updated': '2024-11-07',\n",
       "  'authors': ['v. a. aksyuk'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.07016'},\n",
       " {'title': 'scaffold-based multi-objective drug candidate optimization',\n",
       "  'id': '2301.07175',\n",
       "  'abstract': 'in therapeutic design, balancing various physiochemical properties is crucial for molecule development, similar to how multiparameter optimization (mpo) evaluates multiple variables to meet a primary goal. while many molecular features can now be predicted using \\\\textit{in silico} methods, aiding early drug development, the vast data generated from high throughput virtual screening challenges the practicality of traditional mpo approaches. addressing this, we introduce a scaffold focused graph-based markov chain monte carlo framework (scamars) built to generate molecules with optimal properties. this innovative framework is capable of self-training and handling a wider array of properties, sampling different chemical spaces according to the starting scaffold. the benchmark analysis on several properties shows that scamars has a diversity score of 84.6\\\\% and has a much higher success rate of 99.5\\\\% compared to conditional models. the integration of new features into mpo significantly enhances its adaptability and effectiveness in therapeutic design, facilitating the discovery of candidates that efficiently optimize multiple properties.',\n",
       "  'categories': 'q-bio.bm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2022-12-15',\n",
       "  'updated': '2024-01-02',\n",
       "  'authors': ['agustin kruel', 'andrew d. mcnaughton', 'neeraj kumar'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.07175'},\n",
       " {'title': \"beyond the exome: what's next in diagnostic testing for mendelian   conditions\",\n",
       "  'id': '2301.07363',\n",
       "  'abstract': 'despite advances in clinical genetic testing, including the introduction of exome sequencing (es), more than 50% of individuals with a suspected mendelian condition lack a precise molecular diagnosis. clinical evaluation is increasingly undertaken by specialists outside of clinical genetics, often occurring in a tiered fashion and typically ending after es. the current diagnostic rate reflects multiple factors, including technical limitations, incomplete understanding of variant pathogenicity, missing genotype-phenotype associations, complex gene-environment interactions, and reporting differences between clinical labs. maintaining a clear understanding of the rapidly evolving landscape of diagnostic tests beyond es, and their limitations, presents a challenge for non-genetics professionals. newer tests, such as short-read genome or rna sequencing, can be challenging to order and emerging technologies, such as optical genome mapping and long-read dna or rna sequencing, are not available clinically. furthermore, there is no clear guidance on the next best steps after inconclusive evaluation. here, we review why a clinical genetic evaluation may be negative, discuss questions to be asked in this setting, and provide a framework for further investigation, including the advantages and disadvantages of new approaches that are nascent in the clinical sphere. we present a guide for the next best steps after inconclusive molecular testing based upon phenotype and prior evaluation, including when to consider referral to a consortium such as gregor, which is focused on elucidating the underlying cause of rare unsolved genetic disorders.',\n",
       "  'categories': 'q-bio.gn',\n",
       "  'doi': '10.1016/j.ajhg.2023.06.009',\n",
       "  'created': '2023-01-18',\n",
       "  'updated': '',\n",
       "  'authors': ['monica h. wojcik',\n",
       "   'chloe m. reuter',\n",
       "   'shruti marwaha',\n",
       "   'medhat mahmoud',\n",
       "   'michael h. duyzend',\n",
       "   'hayk barseghyan',\n",
       "   'bo yuan',\n",
       "   'philip m. boone',\n",
       "   'emily e. groopman',\n",
       "   'emmanuèle c. délot',\n",
       "   'deepti jain',\n",
       "   'alba sanchis-juan',\n",
       "   'genomics research to elucidate the genetics of rare diseases',\n",
       "   'n/a consortium',\n",
       "   'lea m. starita',\n",
       "   'michael talkowski',\n",
       "   'stephen b. montgomery',\n",
       "   'michael j. bamshad',\n",
       "   'jessica x. chong',\n",
       "   'matthew t. wheeler',\n",
       "   'seth i. berger',\n",
       "   \"anne o'donnell-luria\",\n",
       "   'fritz j. sedlazeck',\n",
       "   'danny e. miller'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.07363'},\n",
       " {'title': 'hierarchical bayesian inference for community detection and connectivity   of functional brain networks',\n",
       "  'id': '2301.07386',\n",
       "  'abstract': 'many functional magnetic resonance imaging (fmri) studies rely on estimates of hierarchically organised brain networks whose segregation and integration reflect the dynamic transitions of latent cognitive states. however, most existing methods for estimating the community structure of networks from both individual and group-level analysis neglect the variability between subjects and lack validation. in this paper, we develop a new multilayer community detection method based on bayesian latent block modelling. the method can robustly detect the group-level community structure of weighted functional networks that give rise to hidden brain states with an unknown number of communities and retain the variability of individual networks. for validation, we propose a new community structure-based multivariate gaussian generative model to simulate synthetic signal. our result shows that the inferred community memberships using hierarchical bayesian analysis are consistent with the predefined node labels in the generative model. the method is also tested using real working memory task-fmri data of 100 unrelated healthy subjects from the human connectome project. the results show distinctive community structure patterns between 2-back, 0-back, and fixation conditions, which may reflect cognitive and behavioural states under working memory task conditions.',\n",
       "  'categories': 'q-bio.nc stat.ap',\n",
       "  'doi': '',\n",
       "  'created': '2023-01-18',\n",
       "  'updated': '2024-05-26',\n",
       "  'authors': ['lingbin bian',\n",
       "   'nizhuan wang',\n",
       "   'leonardo novelli',\n",
       "   'jonathan keith',\n",
       "   'adeel razi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.07386'},\n",
       " {'title': 'asymptotic sampling distributions made easy: loose linkage in the   ancestral recombination graph',\n",
       "  'id': '2301.07394',\n",
       "  'abstract': 'understanding the interplay between recombination and resampling is a significant challenge in mathematical population genetics and of great practical relevance. asymptotic results about the distribution of samples when recombination is strong compared to resampling are often based on the approximate solution of certain recursions, which is technically hard and offers little conceptual insight. this work generalises an elegant probabilistic argument, based on the coupling of ancestral processes but so far only available in the case of two sites, to the multilocus setting. this offers an alternative route to, and slightly generalises, a classical result of bhaskar and song.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-01-18',\n",
       "  'updated': '2024-11-04',\n",
       "  'authors': ['frederic alberti'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.07394'},\n",
       " {'title': 'evolutionary adaptation is facilitated by the presence of lethal   genotypes',\n",
       "  'id': '2301.07599',\n",
       "  'abstract': 'the adaptation rate in theoretical models of biological evolution increases with the mutation rate but only to a point when mutations into lethal states cause extinction. one would expect that removing such states should be beneficial for evolution. we show here that, counter-intuitively, lethal mutations speed up adaptation on rugged fitness landscapes with many fitness maxima and minima, if strong competition for resources exist. we consider a modified stochastic version of the quasispecies model with two types of genotypes, viable and lethal, and show that increasing the rate of lethal mutations decreases the time to evolve the best-fit genotype. this can be explained by an increased rate of crossing fitness valleys, facilitated by reduced selection against less-fit variants.',\n",
       "  'categories': 'q-bio.pe physics.bio-ph',\n",
       "  'doi': '10.1103/physrevresearch.6.013286',\n",
       "  'created': '2023-01-18',\n",
       "  'updated': '2023-02-07',\n",
       "  'authors': ['viktoria blavatska', 'bartlomiej waclaw'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.07599'},\n",
       " {'title': 'evolutionary dynamics of glucose-deprived cancer cells: insights from   experimentally-informed mathematical modelling',\n",
       "  'id': '2301.08023',\n",
       "  'abstract': 'glucose is a primary energy source for cancer cells. several lines of evidence support the idea that monocarboxylate transporters, such as mct1, elicit metabolic reprogramming of cancer cells in glucose-poor environments, allowing them to reuse lactate, a byproduct of glucose metabolism, as an alternative energy source with serious consequences for disease progression. we employ a synergistic experimental and mathematical modelling approach to explore the evolutionary processes at the root of cancer cell adaptation to glucose deprivation, with particular focus on the mechanisms underlying the increase in mct1 expression observed in glucose-deprived aggressive cancer cells. data from in vitro experiments on breast cancer cells are used to inform and calibrate a mathematical model that comprises a partial integro-differential equation for the dynamics of a population of cancer cells structured by the level of mct1 expression. analytical and numerical results of this model suggest that environment-induced changes in mct1 expression mediated by lactate-associated signalling pathways enable a prompt adaptive response of glucose-deprived cancer cells, whilst fluctuations in mct1 expression due to epigenetic changes create the substrate for environmental selection to act upon, speeding up the selective sweep underlying cancer cell adaptation to glucose deprivation, and may constitute a long-term bet-hedging mechanism.',\n",
       "  'categories': 'q-bio.cb',\n",
       "  'doi': '10.1098/rsif.2023.0587',\n",
       "  'created': '2023-01-19',\n",
       "  'updated': '2024-01-24',\n",
       "  'authors': ['luis almeida',\n",
       "   'jérôme denis',\n",
       "   'nathalie ferrand',\n",
       "   'tommaso lorenzi',\n",
       "   'antonin prunet',\n",
       "   'michéle sabbah',\n",
       "   'chiara villa'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.08023'},\n",
       " {'title': 'individualized prescriptive inference in ischaemic stroke',\n",
       "  'id': '2301.10748',\n",
       "  'abstract': \"the gold standard in the treatment of ischaemic stroke is set by evidence from randomized controlled trials, based on simple descriptions of presumptively homogeneous populations. yet the manifest complexity of the brain's functional, connective, and vascular architectures introduces heterogeneities that violate the underlying statistical premisses, potentially leading to substantial errors at both individual and population levels. the counterfactual nature of interventional inference renders quantifying the impact of this defect difficult. here we conduct a comprehensive series of semi-synthetic, biologically plausible, virtual interventional trials across 100m+ distinct simulations. we generate empirically grounded virtual trial data from large-scale meta-analytic connective, functional, genetic expression, and receptor distribution data, with high-resolution maps of 4k+ acute ischaemic lesions. within each trial, we estimate treatment effects using models varying in complexity, in the presence of increasingly confounded outcomes and noisy treatment responses. individualized prescriptions inferred from simple models, fitted to unconfounded data, were less accurate than those from complex models, fitted to confounded data. our results indicate that complex modelling with richly represented lesion data is critical to individualized prescriptive inference in ischaemic stroke.\",\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-01-25',\n",
       "  'updated': '2024-11-26',\n",
       "  'authors': ['dominic giles',\n",
       "   'robert gray',\n",
       "   'chris foulon',\n",
       "   'guilherme pombo',\n",
       "   'james k. ruffle',\n",
       "   'tianbo xu',\n",
       "   'h. rolf jäger',\n",
       "   'jorge cardoso',\n",
       "   'sebastien ourselin',\n",
       "   'geraint rees',\n",
       "   'ashwani jha',\n",
       "   'parashkev nachev'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.10748'},\n",
       " {'title': 'rdesign: hierarchical data-efficient representation learning for   tertiary structure-based rna design',\n",
       "  'id': '2301.10774',\n",
       "  'abstract': \"while artificial intelligence has made remarkable strides in revealing the relationship between biological macromolecules' primary sequence and tertiary structure, designing rna sequences based on specified tertiary structures remains challenging. though existing approaches in protein design have thoroughly explored structure-to-sequence dependencies in proteins, rna design still confronts difficulties due to structural complexity and data scarcity. moreover, direct transplantation of protein design methodologies into rna design fails to achieve satisfactory outcomes although sharing similar structural components. in this study, we aim to systematically construct a data-driven rna design pipeline. we crafted a large, well-curated benchmark dataset and designed a comprehensive structural modeling approach to represent the complex rna tertiary structure. more importantly, we proposed a hierarchical data-efficient representation learning framework that learns structural representations through contrastive learning at both cluster-level and sample-level to fully leverage the limited data. by constraining data representations within a limited hyperspherical space, the intrinsic relationships between data points could be explicitly imposed. moreover, we incorporated extracted secondary structures with base pairs as prior knowledge to facilitate the rna design process. extensive experiments demonstrate the effectiveness of our proposed method, providing a reliable baseline for future rna design tasks. the source code and benchmark dataset are available at https://github.com/a4bio/rdesign.\",\n",
       "  'categories': 'q-bio.bm cs.ai cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-01-25',\n",
       "  'updated': '2024-03-06',\n",
       "  'authors': ['cheng tan',\n",
       "   'yijie zhang',\n",
       "   'zhangyang gao',\n",
       "   'bozhen hu',\n",
       "   'siyuan li',\n",
       "   'zicheng liu',\n",
       "   'stan z. li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.10774'},\n",
       " {'title': 'predator extinction arose from chaos of the prey: the chaotic behavior   of a homomorphic two-dimensional logistic map in the form of lotka-volterra   equations',\n",
       "  'id': '2301.11669',\n",
       "  'abstract': 'a two-dimensional homomorphic logistic map that preserves features of the lotka-volterra equations was proposed. to examine chaos, iteration plots of the population, lyapunov exponents calculated from jacobian eigenvalues of the $2$d logistic mapping, and from time series algorithms of rosenstein and eckmann et al. were calculated. bifurcation diagrams may be divided into four categories depending on topological shapes. our model not only recovered the $1$d logistic map, which exhibits flip bifurcation, for the prey when there is a nonzero initial predator population, but it can also simulate normal competition between two species with equal initial populations. despite the possibility for two species to go into chaos simultaneously, where the neimark-sacker bifurcation was observed, it is also possible that with the same interspecies parameters as normal but with a predator population $10$ times more than that of the prey, the latter becomes chaotic, while the former dramatically reduces to zero with only a few iterations, indicating total annihilation of the predator species. interpreting humans as predators and natural resources as preys in the ecological system, the above-mentioned conclusion may imply that not only excessive consumption of natural resources, but its chaotic state triggered by an overpopulation of humans may backfire in a manner of total extinction of the human species. fortunately, there is little chance for the survival of the human race, as isolated fixed points in the bifurcation diagram of the predator reveal. finally, two possible applications of the phenomenon of chaotic extinction are proposed: one is to inhibit viruses or pests by initiating the chaotic states of the prey on which the viruses or pests rely for existence, and the other is to achieve the superconducting state with the chaotic state of the applied magnetic field.',\n",
       "  'categories': 'nlin.cd physics.soc-ph q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-01-27',\n",
       "  'updated': '2024-02-24',\n",
       "  'authors': ['wei shan lee',\n",
       "   'hou fai chan',\n",
       "   'ka ian im',\n",
       "   'kuan ieong chan',\n",
       "   'u hin cheang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.11669'},\n",
       " {'title': 'robust statistical properties of t1 transitions in confluent cell   tissues',\n",
       "  'id': '2301.11758',\n",
       "  'abstract': 'large-scale tissue deformation which is fundamental to tissue development hinges on local cellular rearrangements, such as t1 transitions. in the realm of the multi-phase field model, we analyse the statistical and dynamical properties of t1 transitions in a confluent tissue. we identify an energy profile that is robust to changes in several model parameters. it is characterized by an asymmetric profile with a fast increase in energy before the t1 transition and a sudden drop after the t1 transition, followed by a slow relaxation. the latter being a signature of the fluidity of the cell tissue. we show that t1 transitions are sources of localised large deformation of the cells undergoing the neighbour exchange and induce other t1 transitions in the nearby cells through a chaining of events that propagate local cell deformation to large scale tissue flows.',\n",
       "  'categories': 'physics.bio-ph cond-mat.soft physics.comp-ph q-bio.to',\n",
       "  'doi': '10.1038/s41598-023-37064-6',\n",
       "  'created': '2023-01-27',\n",
       "  'updated': '',\n",
       "  'authors': ['harish p jain', 'axel voigt', 'luiza angheluta'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.11758'},\n",
       " {'title': '3d genome reconstruction from partially phased hi-c data',\n",
       "  'id': '2301.11764',\n",
       "  'abstract': 'the 3-dimensional (3d) structure of the genome is of significant importance for many cellular processes. in this paper, we study the problem of reconstructing the 3d structure of chromosomes from hi-c data of diploid organisms, which poses additional challenges compared to the better-studied haploid setting. with the help of techniques from algebraic geometry, we prove that a small amount of phased data is sufficient to ensure finite identifiability, both for noiseless and noisy data. in the light of these results, we propose a new 3d reconstruction method based on semidefinite programming, paired with numerical algebraic geometry and local optimization. the performance of this method is tested on several simulated datasets under different noise levels and with different amounts of phased data. we also apply it to a real dataset from mouse x chromosomes, and we are then able to recover previously known structural features.',\n",
       "  'categories': 'q-bio.gn math.ag math.oc',\n",
       "  'doi': '10.1007/s11538-024-01263-7',\n",
       "  'created': '2023-01-27',\n",
       "  'updated': '2024-05-29',\n",
       "  'authors': ['diego cifuentes',\n",
       "   'jan draisma',\n",
       "   'oskar henriksson',\n",
       "   'annachiara korchmaros',\n",
       "   'kaie kubjas'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2301.11764'},\n",
       " {'title': 'clash: contrastive learning through alignment shifting to extract   stimulus information from eeg',\n",
       "  'id': '2302.01924',\n",
       "  'abstract': 'stimulus-evoked eeg data has a notoriously low signal-to-noise ratio and high inter-subject variability. we propose a novel paradigm for the self-supervised extraction of stimulus-related brain response data: a model is trained to extract similar information between two time-aligned segments of eeg in response to the same stimulus. the extracted information can subsequently be used to obtain better results in downstream tasks that utilize the response to the stimulus. we show the efficacy of our method for a downstream task of decoding the speech envelope from auditory eeg. our method outperforms other state-of-the-art denoising techniques, improving reconstruction scores by 45\\\\%. additionally, we show that in contrast to the baseline denoising techniques, our method can be used with data of unseen subjects and stimuli without retraining, improving decoding performance by 19\\\\% and 34\\\\% over raw eeg for two holdout datasets. finally, the last experiment reveals that the accuracies obtained in the clash paradigm are significantly correlated with the percentile of obtained reconstruction correlation on the null distribution. in general, we showed that the proposed paradigm is suitable to train deep learning models to extract stimulus information from eeg while being stimulus feature agnostic.',\n",
       "  'categories': 'q-bio.nc eess.sp',\n",
       "  'doi': '',\n",
       "  'created': '2023-01-09',\n",
       "  'updated': '2024-05-14',\n",
       "  'authors': ['bernd accou', 'hugo van hamme', 'tom francart'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.01924'},\n",
       " {'title': 'a three-state coupled markov switching model for covid-19 outbreaks   across quebec based on hospital admissions',\n",
       "  'id': '2302.02488',\n",
       "  'abstract': 'recurrent covid-19 outbreaks have placed immense strain on the hospital system in quebec. we develop a bayesian three-state coupled markov switching model to analyze covid-19 outbreaks across quebec based on admissions in the 30 largest hospitals. within each catchment area, we assume the existence of three states for the disease: absence, a new state meant to account for many zeroes in some of the smaller areas, endemic and outbreak. then we assume the disease switches between the three states in each area through a series of coupled nonhomogeneous hidden markov chains. unlike previous approaches, the transition probabilities may depend on covariates and the occurrence of outbreaks in neighboring areas, to account for geographical outbreak spread. additionally, to prevent rapid switching between endemic and outbreak periods we introduce clone states into the model which enforce minimum endemic and outbreak durations. we make some interesting findings, such as that mobility in retail and recreation venues had a positive association with the development and persistence of new covid-19 outbreaks in quebec. based on model comparison our contributions show promise in improving state estimation retrospectively and in real-time, especially when there are smaller areas and highly spatially synchronized outbreaks. furthermore, our approach offers new and interesting epidemiological interpretations, such as being able to estimate the effect of covariates on disease extinction.',\n",
       "  'categories': 'stat.ap q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-05',\n",
       "  'updated': '2024-09-22',\n",
       "  'authors': ['dirk douwes-schultz',\n",
       "   'alexandra m. schmidt',\n",
       "   'yannan shen',\n",
       "   'david buckeridge'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.02488'},\n",
       " {'title': 'pgcn: pyramidal graph convolutional network for eeg emotion recognition',\n",
       "  'id': '2302.02520',\n",
       "  'abstract': 'emotion recognition is essential in the diagnosis and rehabilitation of various mental diseases. in the last decade, electroencephalogram (eeg)-based emotion recognition has been intensively investigated due to its prominative accuracy and reliability, and graph convolutional network (gcn) has become a mainstream model to decode emotions from eeg signals. however, the electrode relationship, especially long-range electrode dependencies across the scalp, may be underutilized by gcns, although such relationships have been proven to be important in emotion recognition. the small receptive field makes shallow gcns only aggregate local nodes. on the other hand, stacking too many layers leads to over-smoothing. to solve these problems, we propose the pyramidal graph convolutional network (pgcn), which aggregates features at three levels: local, mesoscopic, and global. first, we construct a vanilla gcn based on the 3d topological relationships of electrodes, which is used to integrate two-order local features; second, we construct several mesoscopic brain regions based on priori knowledge and employ mesoscopic attention to sequentially calculate the virtual mesoscopic centers to focus on the functional connections of mesoscopic brain regions; finally, we fuse the node features and their 3d positions to construct a numerical relationship adjacency matrix to integrate structural and functional connections from the global perspective. experimental results on three public datasets indicate that pgcn enhances the relationship modelling across the scalp and achieves state-of-the-art performance in both subject-dependent and subject-independent scenarios. meanwhile, pgcn makes an effective trade-off between enhancing network depth and receptive fields while suppressing the ensuing over-smoothing. our codes are publicly accessible at https://github.com/jinminbox/pgcn.',\n",
       "  'categories': 'cs.hc q-bio.nc',\n",
       "  'doi': '10.1109/tmm.2024.3385676',\n",
       "  'created': '2023-02-05',\n",
       "  'updated': '',\n",
       "  'authors': ['ming jin',\n",
       "   'enwei zhu',\n",
       "   'changde du',\n",
       "   'huiguang he',\n",
       "   'jinpeng li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.02520'},\n",
       " {'title': 'empirical quantification of predictive uncertainty due to model   discrepancy by training with an ensemble of experimental designs: an   application to ion channel kinetics',\n",
       "  'id': '2302.02942',\n",
       "  'abstract': \"when mathematical biology models are used to make quantitative predictions for clinical or industrial use, it is important that these predictions come with a reliable estimate of their accuracy (uncertainty quantification). because models of complex biological systems are always large simplifications, model discrepancy arises - where a mathematical model fails to recapitulate the true data generating process. this presents a particular challenge for making accurate predictions, and especially for making accurate estimates of uncertainty in these predictions. experimentalists and modellers must choose which experimental procedures (protocols) are used to produce data to train their models. we propose to characterise uncertainty owing to model discrepancy with an ensemble of parameter sets, each of which results from training to data from a different protocol. the variability in predictions from this ensemble provides an empirical estimate of predictive uncertainty owing to model discrepancy, even for unseen protocols. we use the example of electrophysiology experiments, which are used to investigate the kinetics of the herg potassium ion channel. here, 'information-rich' protocols allow mathematical models to be trained using numerous short experiments performed on the same cell. typically, assuming independent observational errors and training a model to an individual experiment results in parameter estimates with very little dependence on observational noise. moreover, parameter sets arising from the same model applied to different experiments often conflict - indicative of model discrepancy. our methods will help select more suitable mathematical models of herg for future studies, and will be widely applicable to a range of biological modelling problems.\",\n",
       "  'categories': 'stat.co math.ds math.oc q-bio.qm',\n",
       "  'doi': '10.1007/s11538-023-01224-6',\n",
       "  'created': '2023-02-06',\n",
       "  'updated': '2024-02-19',\n",
       "  'authors': ['joseph g. shuttleworth',\n",
       "   'chon lok lei',\n",
       "   'dominic g. whittaker',\n",
       "   'monique j. windley',\n",
       "   'adam p. hill',\n",
       "   'simon p. preston',\n",
       "   'gary r. mirams'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.02942'},\n",
       " {'title': 'single cells are spatial tokens: transformers for spatial transcriptomic   data imputation',\n",
       "  'id': '2302.03038',\n",
       "  'abstract': 'spatially resolved transcriptomics brings exciting breakthroughs to single-cell analysis by providing physical locations along with gene expression. however, as a cost of the extremely high spatial resolution, the cellular level spatial transcriptomic data suffer significantly from missing values. while a standard solution is to perform imputation on the missing values, most existing methods either overlook spatial information or only incorporate localized spatial context without the ability to capture long-range spatial information. using multi-head self-attention mechanisms and positional encoding, transformer models can readily grasp the relationship between tokens and encode location information. in this paper, by treating single cells as spatial tokens, we study how to leverage transformers to facilitate spatial tanscriptomics imputation. in particular, investigate the following two key questions: (1) $\\\\textit{how to encode spatial information of cells in transformers}$, and (2) $\\\\textit{ how to train a transformer for transcriptomic imputation}$. by answering these two questions, we present a transformer-based imputation framework, spaformer, for cellular-level spatial transcriptomic data. extensive experiments demonstrate that spaformer outperforms existing state-of-the-art imputation algorithms on three large-scale datasets while maintaining superior computational efficiency.',\n",
       "  'categories': 'q-bio.gn cs.ai cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-05',\n",
       "  'updated': '2024-02-16',\n",
       "  'authors': ['hongzhi wen',\n",
       "   'wenzhuo tang',\n",
       "   'wei jin',\n",
       "   'jiayuan ding',\n",
       "   'renming liu',\n",
       "   'xinnan dai',\n",
       "   'feng shi',\n",
       "   'lulu shang',\n",
       "   'hui liu',\n",
       "   'yuying xie'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.03038'},\n",
       " {'title': 'temporal and probabilistic comparisons of epidemic interventions',\n",
       "  'id': '2302.03210',\n",
       "  'abstract': \"forecasting disease spread is a critical tool to help public health officials design and plan public health interventions.however, the expected future state of an epidemic is not necessarily well defined as disease spread is inherently stochastic, contact patterns within a population are heterogeneous, and behaviors change. in this work, we use time-dependent probability generating functions (pgfs) to capture these characteristics by modeling a stochastic branching process of the spread of a disease over a network of contacts in which public health interventions are introduced over time. to achieve this, we define a general transmissibility equation to account for varying transmission rates (e.g. masking), recovery rates (e.g. treatment), contact patterns (e.g. social distancing) and percentage of the population immunized (e.g. vaccination). the resulting framework allows for a temporal and probabilistic analysis of an intervention's impact on disease spread, which match continuous-time stochastic simulations that are much more computationally expensive.to aid policy making, we then define several metrics over which temporal and probabilistic intervention forecasts can be compared: looking at the expected number of cases and the worst-case scenario over time, as well as the probability of reaching a critical level of cases and of not seeing any improvement following an intervention.given that epidemics do not always follow their average expected trajectories and that the underlying dynamics can change over time, our work paves the way for more detailed short-term forecasts of disease spread and more informed comparison of intervention strategies.\",\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1007/s11538-023-01220-w',\n",
       "  'created': '2023-02-06',\n",
       "  'updated': '2024-01-18',\n",
       "  'authors': ['mariah c. boudreau',\n",
       "   'andrea j. allen',\n",
       "   'nicholas j. roberts',\n",
       "   'antoine allard',\n",
       "   'laurent hébert-dufresne'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.03210'},\n",
       " {'title': 'geometry-complete diffusion for 3d molecule generation and optimization',\n",
       "  'id': '2302.04313',\n",
       "  'abstract': \"denoising diffusion probabilistic models (ddpms) have pioneered new state-of-the-art results in disciplines such as computer vision and computational biology for diverse tasks ranging from text-guided image generation to structure-guided protein design. along this latter line of research, methods have recently been proposed for generating 3d molecules using equivariant graph neural networks (gnns) within a ddpm framework. however, such methods are unable to learn important geometric properties of 3d molecules, as they adopt molecule-agnostic and non-geometric gnns as their 3d graph denoising networks, which notably hinders their ability to generate valid large 3d molecules. in this work, we address these gaps by introducing the geometry-complete diffusion model (gcdm) for 3d molecule generation, which outperforms existing 3d molecular diffusion models by significant margins across conditional and unconditional settings for the qm9 dataset and the larger geom-drugs dataset, respectively, and generates more novel and unique unconditional 3d molecules for the qm9 dataset compared to previous methods. importantly, we demonstrate that the geometry-complete denoising process of gcdm learned for 3d molecule generation enables the model to generate a significant proportion of valid and energetically-stable large molecules at the scale of geom-drugs, whereas previous methods fail to do so with the features they learn. additionally, we show that extensions of gcdm can not only effectively design 3d molecules for specific protein pockets but also that gcdm's geometric features can be repurposed to consistently optimize the geometry and chemical composition of existing 3d molecules for molecular stability and property specificity, demonstrating new versatility of molecular diffusion models. our source code and data are freely available at https://github.com/bioinfomachinelearning/bio-diffusion.\",\n",
       "  'categories': 'cs.lg cs.ai q-bio.bm q-bio.qm stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-08',\n",
       "  'updated': '2024-05-24',\n",
       "  'authors': ['alex morehead', 'jianlin cheng'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.04313'},\n",
       " {'title': 'a text-guided protein design framework',\n",
       "  'id': '2302.04611',\n",
       "  'abstract': \"current ai-assisted protein design mainly utilizes protein sequential and structural information. meanwhile, there exists tremendous knowledge curated by humans in the text format describing proteins' high-level functionalities. yet, whether the incorporation of such text data can help protein design tasks has not been explored. to bridge this gap, we propose proteindt, a multi-modal framework that leverages textual descriptions for protein design. proteindt consists of three subsequent steps: proteinclap which aligns the representation of two modalities, a facilitator that generates the protein representation from the text modality, and a decoder that creates the protein sequences from the representation. to train proteindt, we construct a large dataset, swissprotclap, with 441k text and protein pairs. we quantitatively verify the effectiveness of proteindt on three challenging tasks: (1) over 90\\\\% accuracy for text-guided protein generation; (2) best hit ratio on 12 zero-shot text-guided protein editing tasks; (3) superior performance on four out of six protein property prediction benchmarks.\",\n",
       "  'categories': 'cs.lg cs.ai q-bio.qm stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-09',\n",
       "  'updated': '2024-08-12',\n",
       "  'authors': ['shengchao liu',\n",
       "   'yanjing li',\n",
       "   'zhuoxinran li',\n",
       "   'anthony gitter',\n",
       "   'yutao zhu',\n",
       "   'jiarui lu',\n",
       "   'zhao xu',\n",
       "   'weili nie',\n",
       "   'arvind ramanathan',\n",
       "   'chaowei xiao',\n",
       "   'jian tang',\n",
       "   'hongyu guo',\n",
       "   'anima anandkumar'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.04611'},\n",
       " {'title': 'the effects of leaflet material properties on the simulated function of   regurgitant mitral valves',\n",
       "  'id': '2302.04939',\n",
       "  'abstract': 'advances in three-dimensional imaging provide the ability to construct and analyze finite element (fe) models to evaluate the biomechanical behavior and function of atrioventricular valves. however, while obtaining patient-specific valve geometry is now possible, non-invasive measurement of patient-specific leaflet material properties remains nearly impossible. both valve geometry and tissue properties play a significant role in governing valve dynamics, leading to the central question of whether clinically relevant insights can be attained from fe analysis of atrioventricular valves without precise knowledge of tissue properties. as such we investigated 1) the influence of tissue extensibility and 2) the effects of constitutive model parameters and leaflet thickness on simulated valve function and mechanics. we compared metrics of valve function (e.g., leaflet coaptation and regurgitant orifice area) and mechanics (e.g., stress and strain) across one normal and three regurgitant mitral valve (mv) models with common mechanisms of regurgitation (annular dilation, leaflet prolapse, leaflet tethering) of both moderate and severe degree. we developed a novel fully-automated approach to accurately quantify regurgitant orifice areas of complex valve geometries. we found that the relative ordering of the mechanical and functional metrics was maintained across a group of valves using material properties up to 15% softer than the representative adult mitral constitutive model. our findings suggest that fe simulations can be used to qualitatively compare how differences and alterations in valve structure affect relative atrioventricular valve function even in populations where material properties are not precisely known.',\n",
       "  'categories': 'physics.med-ph cs.na math.na q-bio.to',\n",
       "  'doi': '10.1016/j.jmbbm.2023.105858',\n",
       "  'created': '2023-02-09',\n",
       "  'updated': '2023-04-25',\n",
       "  'authors': ['wensi wu',\n",
       "   'stephen ching',\n",
       "   'patricia sabin',\n",
       "   'devin w. laurence',\n",
       "   'steve a. maas',\n",
       "   'andras lasso',\n",
       "   'jeffrey a. weiss',\n",
       "   'matthew a. jolley'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.04939'},\n",
       " {'title': 'boundary integral formulation of the cell-by-cell model of cardiac   electrophysiology',\n",
       "  'id': '2302.05281',\n",
       "  'abstract': 'we propose a boundary element method for the accurate solution of the cell-by-cell bidomain model of electrophysiology. the cell-by-cell model, also called extracellular-membrane-intracellular (emi) model, is a system of reaction-diffusion equations describing the evolution of the electric potential within each domain: intra- and extra-cellular space and the cellular membrane. the system is parabolic but degenerate because the time derivative is only in the membrane domain. in this work, we adopt a boundary-integral formulation for removing the degeneracy in the system and recast it to a parabolic equation on the membrane. the formulation is also numerically advantageous since the number of degrees of freedom is sensibly reduced compared to the original model. specifically, we prove that the boundary-element discretization of the emi model is equivalent to a system of ordinary differential equations, and we consider a time discretization based on the multirate explicit stabilized runge-kutta method. we numerically show that our scheme convergences exponentially in space for the single-cell case. we finally provide several numerical experiments of biological interest.',\n",
       "  'categories': 'math.na cs.na q-bio.to',\n",
       "  'doi': '10.1016/j.enganabound.2023.10.021',\n",
       "  'created': '2023-02-10',\n",
       "  'updated': '',\n",
       "  'authors': ['giacomo rosilho de souza', 'rolf krause', 'simone pezzuto'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.05281'},\n",
       " {'title': 'improvement of attention in subjects diagnosed with hyperkinetic   syndrome using biovit simulator',\n",
       "  'id': '2302.05562',\n",
       "  'abstract': \"this study aimed to stimulate the brain's executive function through a series of tasks and rules based on dynamic perceptual stimuli using the biotechnology virtual immersion simulator (biovit) and thus evaluate its usefulness to maintain and increase attention levels in subjects diagnosed with hyperkinetic syndrome. with a quantitative methodology framed in a longitudinal trend design, the cause of the exposure-outcome relationships was studied using the biovit simulator. exploratory analysis of oscillatory brain activity was measured using a graphical recording of brain electrical activity and attention levels. data consisted of 77,566 observations from n = 18 separately studied participants. the findings established that the biovit simulator maintained and increased the attention levels of the participants by 77.8%. furthermore, the hypothesis was tested that virtual reality immersion technologies significantly affect attention levels in participants aged 8 to 12. the evidence shows that the biovit simulator is an alternative to developing learning methodologies in vulnerable populations. the low implementation costs and the diversity of academic applications may allow schools in developing countries to solve this problem that afflicts thousands of children with attention deficit and hyperactivity disorder.\",\n",
       "  'categories': 'cs.hc cs.cy q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-10',\n",
       "  'updated': '',\n",
       "  'authors': ['cesar r salas-guerra'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.05562'},\n",
       " {'title': 'federated attention consistent learning models for prostate cancer   diagnosis and gleason grading',\n",
       "  'id': '2302.06089',\n",
       "  'abstract': 'artificial intelligence (ai) holds significant promise in transforming medical imaging, enhancing diagnostics, and refining treatment strategies. however, the reliance on extensive multicenter datasets for training ai models poses challenges due to privacy concerns. federated learning provides a solution by facilitating collaborative model training across multiple centers without sharing raw data. this study introduces a federated attention-consistent learning (facl) framework to address challenges associated with large-scale pathological images and data heterogeneity. facl enhances model generalization by maximizing attention consistency between local clients and the server model. to ensure privacy and validate robustness, we incorporated differential privacy by introducing noise during parameter transfer. we assessed the effectiveness of facl in cancer diagnosis and gleason grading tasks using 19,461 whole-slide images of prostate cancer from multiple centers. in the diagnosis task, facl achieved an area under the curve (auc) of 0.9718, outperforming seven centers with an average auc of 0.9499 when categories are relatively balanced. for the gleason grading task, facl attained a kappa score of 0.8463, surpassing the average kappa score of 0.7379 from six centers. in conclusion, facl offers a robust, accurate, and cost-effective ai training model for prostate cancer pathology while maintaining effective data safeguards.',\n",
       "  'categories': 'cs.cv cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-12',\n",
       "  'updated': '2024-03-28',\n",
       "  'authors': ['fei kong',\n",
       "   'xiyue wang',\n",
       "   'jinxi xiang',\n",
       "   'sen yang',\n",
       "   'xinran wang',\n",
       "   'meng yue',\n",
       "   'jun zhang',\n",
       "   'junhan zhao',\n",
       "   'xiao han',\n",
       "   'yuhan dong',\n",
       "   'biyue zhu',\n",
       "   'fang wang',\n",
       "   'yueping liu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.06089'},\n",
       " {'title': 'knowledge from large-scale protein contact prediction models can be   transferred to the data-scarce rna contact prediction task',\n",
       "  'id': '2302.06120',\n",
       "  'abstract': 'rna, whose functionality is largely determined by its structure, plays an important role in many biological activities. the prediction of pairwise structural proximity between each nucleotide of an rna sequence can characterize the structural information of the rna. historically, this problem has been tackled by machine learning models using expert-engineered features and trained on scarce labeled datasets. here, we find that the knowledge learned by a protein-coevolution transformer-based deep neural network can be transferred to the rna contact prediction task. as protein datasets are orders of magnitude larger than those for rna contact prediction, our findings and the subsequent framework greatly reduce the data scarcity bottleneck. experiments confirm that rna contact prediction through transfer learning using a publicly available protein model is greatly improved. our findings indicate that the learned structural patterns of proteins can be transferred to rnas, opening up potential new avenues for research.',\n",
       "  'categories': 'q-bio.qm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-13',\n",
       "  'updated': '2024-01-18',\n",
       "  'authors': ['yiren jian',\n",
       "   'chongyang gao',\n",
       "   'chen zeng',\n",
       "   'yunjie zhao',\n",
       "   'soroush vosoughi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.06120'},\n",
       " {'title': 'propagation of weakly advantageous mutations in cancer cell population',\n",
       "  'id': '2302.06573',\n",
       "  'abstract': 'research into somatic mutations in cancer cell dna and their role in tumour growth and progression between successive stages is crucial for improving our understanding of cancer evolution. mathematical and computer modelling can provide valuable insights into the scenarios of cancer growth, the roles of somatic mutations, and the types and strengths of evolutionary forces they introduce. previous studies have developed mathematical models of cancer evolution, incorporating driver and passenger somatic mutations. driver mutations were assumed to have a strong advantageous effect on the growth of the cancer cell population, while passenger mutations were considered fully neutral or mildly deleterious.   however, according to several studies, passenger mutations may have a weakly advantageous effect on tumour growth. in this paper, we develop models of cancer evolution with somatic mutations that introduce a weakly advantageous force to the evolution of cancer cells. the models used in this study can be classified into two categories: deterministic and stochastic. deterministic models are based on systems of differential equations that balance the average number of cells and mutations during evolution. to verify the results of our deterministic modelling, we use a stochastic model based on the gillespie algorithm. we compare the predictions of our modelling with some observational data on cancer evolution.',\n",
       "  'categories': 'q-bio.pe cs.na math.na',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-13',\n",
       "  'updated': '2024-02-23',\n",
       "  'authors': ['andrzej polanski',\n",
       "   'mateusz kania',\n",
       "   'jarosław gil',\n",
       "   'wojciech łabaj',\n",
       "   'ewa lach',\n",
       "   'agnieszka szczęsna'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.06573'},\n",
       " {'title': 'random boundaries: quantifying segmentation uncertainty in solutions to   boundary-value problems',\n",
       "  'id': '2302.06842',\n",
       "  'abstract': \"engineering simulations using boundary-value partial differential equations often implicitly assume that the uncertainty in the location of the boundary has a negligible impact on the output of the simulation. in this work, we develop a novel method for describing the geometric uncertainty in image-derived models and use a naive method for subsequently quantifying a simulation's sensitivity to that uncertainty. a gaussian random field is constructed to represent the space of possible geometries, based on image-derived quantities such as pixel size, which can then be used to probe the simulation's output space. the algorithm is demonstrated with examples from biomechanics where patient-specific geometries are often segmented from low-resolution, three-dimensional images. these examples show the method's wide applicability with examples using linear elasticity and fluid dynamics. we show that important biomechanical outputs of these example simulations, namely maximum principal stress and wall shear stress, can be highly sensitive to realistic uncertainties in geometry.\",\n",
       "  'categories': 'q-bio.to',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-14',\n",
       "  'updated': '2024-06-09',\n",
       "  'authors': ['s. gerry gralton',\n",
       "   'farah alkhatib',\n",
       "   'ben zwick',\n",
       "   'george bourantas',\n",
       "   'adam wittek',\n",
       "   'karol miller'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.06842'},\n",
       " {'title': 'community-developed checklists for publishing images and image analysis',\n",
       "  'id': '2302.07005',\n",
       "  'abstract': 'images document scientific discoveries and are prevalent in modern biomedical research. microscopy imaging in particular is currently undergoing rapid technological advancements. however for scientists wishing to publish the obtained images and image analyses results, there are to date no unified guidelines. consequently, microscopy images and image data in publications may be unclear or difficult to interpret. here we present community-developed checklists for preparing light microscopy images and image analysis for publications. these checklists offer authors, readers, and publishers key recommendations for image formatting and annotation, color selection, data availability, and for reporting image analysis workflows. the goal of our guidelines is to increase the clarity and reproducibility of image figures and thereby heighten the quality of microscopy data is in publications.',\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '10.1038/s41592-023-01987-9',\n",
       "  'created': '2023-02-14',\n",
       "  'updated': '2023-09-14',\n",
       "  'authors': ['christopher schmied',\n",
       "   'michael nelson',\n",
       "   'sergiy avilov',\n",
       "   'gert-jan bakker',\n",
       "   'cristina bertocchi',\n",
       "   'johanna bischof',\n",
       "   'ulrike boehm',\n",
       "   'jan brocher',\n",
       "   'mariana carvalho',\n",
       "   'catalin chiritescu',\n",
       "   'jana christopher',\n",
       "   'beth cimini',\n",
       "   'eduardo conde-sousa',\n",
       "   'michael ebner',\n",
       "   'rupert ecker',\n",
       "   'kevin eliceiri',\n",
       "   'julia fernandez-rodriguez',\n",
       "   'nathalie gaudreault',\n",
       "   'laurent gelman',\n",
       "   'david grunwald',\n",
       "   'tingting gu',\n",
       "   'nadia halidi',\n",
       "   'mathias hammer',\n",
       "   'matthew hartley',\n",
       "   'marie held',\n",
       "   'florian jug',\n",
       "   'varun kapoor',\n",
       "   'ayse aslihan koksoy',\n",
       "   'judith lacoste',\n",
       "   'sylvia le dévédec',\n",
       "   'sylvie le guyader',\n",
       "   'penghuan liu',\n",
       "   'gabriel martins',\n",
       "   'aastha mathur',\n",
       "   'kota miura',\n",
       "   'paula montero llopis',\n",
       "   'roland nitschke',\n",
       "   'alison north',\n",
       "   'adam parslow',\n",
       "   'alex payne-dwyer',\n",
       "   'laure plantard',\n",
       "   'ali rizwan',\n",
       "   'britta schroth-diez',\n",
       "   'lucas schütz',\n",
       "   'ryan t. scott',\n",
       "   'arne seitz',\n",
       "   'olaf selchow',\n",
       "   'ved sharma',\n",
       "   'martin spitaler',\n",
       "   'sathya srinivasan',\n",
       "   'caterina strambio de castillia',\n",
       "   'douglas taatjes',\n",
       "   'christian tischer',\n",
       "   'helena klara jambor'],\n",
       "  'affiliation': ['fondazione human technopole, milano, italy',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'centre for bioimage analysis, embl, heidelberg, germany',\n",
       "   'nct-ucc, medizinische fakultät tu dresden, dresden, germany'],\n",
       "  'url': 'https://arxiv.org/abs/2302.07005'},\n",
       " {'title': 'target specific de novo design of drug candidate molecules with graph   transformer-based generative adversarial networks',\n",
       "  'id': '2302.07868',\n",
       "  'abstract': 'discovering novel drug candidate molecules is one of the most fundamental and critical steps in drug development. generative deep learning models, which create synthetic data given a probability distribution, offer a high potential for designing de novo molecules. however, for them to be useful in real-life drug development pipelines, these models should be able to design drug-like and target-centric molecules. in this study, we propose an end-to-end generative system, druggen, for the de novo design of drug candidate molecules that interact with intended target proteins. the proposed method represents molecules as graphs and processes them via a generative adversarial network comprising graph transformer layers. the system is trained using a large dataset of drug-like compounds and target-specific bioactive molecules to design effective inhibitory molecules against the akt1 protein, which is critically important in developing treatments for various types of cancer. we conducted molecular docking and dynamics to assess the target-centric generation performance of the model, as well as attention score visualisation to examine model interpretability. results indicate that our de novo molecules have a high potential for interacting with the akt1 protein at the level of its native ligands. using the open-access druggen codebase, it is possible to easily train models for other druggable proteins, given a dataset of experimentally known bioactive molecules.',\n",
       "  'categories': 'cs.lg q-bio.bm q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-15',\n",
       "  'updated': '2024-07-26',\n",
       "  'authors': ['atabey ünlü',\n",
       "   'elif çevrim',\n",
       "   'ahmet sarıgün',\n",
       "   'melih gökay yiğit',\n",
       "   'hayriye çelikbilek',\n",
       "   'osman bayram',\n",
       "   'heval ataş güvenilir',\n",
       "   'altay koyaş',\n",
       "   'deniz cansen kahraman',\n",
       "   'abdurrahman olğaç',\n",
       "   'ahmet rifaioğlu',\n",
       "   'erden banoğlu',\n",
       "   'tunca doğan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.07868'},\n",
       " {'title': 'fossil image identification using deep learning ensembles of data   augmented multiviews',\n",
       "  'id': '2302.08062',\n",
       "  'abstract': 'identification of fossil species is crucial to evolutionary studies. recent advances from deep learning have shown promising prospects in fossil image identification. however, the quantity and quality of labeled fossil images are often limited due to fossil preservation, conditioned sampling, and expensive and inconsistent label annotation by domain experts, which pose great challenges to training deep learning based image classification models. to address these challenges, we follow the idea of the wisdom of crowds and propose a multiview ensemble framework, which collects original (o), gray (g), and skeleton (s) views of each fossil image reflecting its different characteristics to train multiple base models, and then makes the final decision via soft voting. experiments on the largest fusulinid dataset with 2400 images show that the proposed ogs consistently outperforms baselines (using a single model for each view), and obtains superior or comparable performance compared to ooo (using three base models for three the same original views). besides, as the training data decreases, the proposed framework achieves more gains. while considering the identification consistency estimation with respect to human experts, ogs receives the highest agreement with the original labels of dataset and with the re-identifications of two human experts. the validation performance provides a quantitative estimation of consistency across different experts and genera. we conclude that the proposed framework can present state-of-the-art performance in the fusulinid fossil identification case study. this framework is designed for general fossil identification and it is expected to see applications to other fossil datasets in future work. the source code is publicly available at https://github.com/houchengbin/fossil-image-identification to benefit future research in fossil image identification.',\n",
       "  'categories': 'cs.cv cs.ai q-bio.pe',\n",
       "  'doi': '10.1111/2041-210x.14229',\n",
       "  'created': '2023-02-15',\n",
       "  'updated': '2024-02-01',\n",
       "  'authors': ['chengbin hou',\n",
       "   'xinyu lin',\n",
       "   'hanhui huang',\n",
       "   'sheng xu',\n",
       "   'junxuan fan',\n",
       "   'yukun shi',\n",
       "   'hairong lv'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.08062'},\n",
       " {'title': 'a countable-type branching process model for the tug-of-war cancer cell   dynamics',\n",
       "  'id': '2302.08695',\n",
       "  'abstract': 'we consider a time-continuous markov branching process of proliferating cells with a countable collection of types. among-type transitions are inspired by the tug-of-war process introduced in mcfarland et al. as a mathematical model for competition of advantageous driver mutations and deleterious passenger mutations in cancer cells. we introduce a version of the model in which a driver mutation pushes the type of the cell $l$-units up, while a passenger mutation pulls it $1$-unit down. the distribution of time to divisions depends on the type (fitness) of cell, which is an integer. the extinction probability given any initial cell type is strictly less than $1$, which allows us to investigate the transition between types (type transition) in an infinitely long cell lineage of cells. the analysis leads to the result that under driver dominance, the type transition process escapes to infinity, while under passenger dominance, it leads to a limit distribution. implications in cancer cell dynamics and population genetics are discussed.',\n",
       "  'categories': 'q-bio.pe math.pr',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-16',\n",
       "  'updated': '2024-02-04',\n",
       "  'authors': ['ren-yi wang', 'marek kimmel'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.08695'},\n",
       " {'title': 'theory of coupled neuronal-synaptic dynamics',\n",
       "  'id': '2302.08985',\n",
       "  'abstract': 'in neural circuits, synaptic strengths influence neuronal activity by shaping network dynamics, and neuronal activity influences synaptic strengths through activity-dependent plasticity. motivated by this fact, we study a recurrent-network model in which neuronal units and synaptic couplings are interacting dynamic variables, with couplings subject to hebbian modification with decay around quenched random strengths. rather than assigning a specific role to the plasticity, we use dynamical mean-field theory and other techniques to systematically characterize the neuronal-synaptic dynamics, revealing a rich phase diagram. adding hebbian plasticity slows activity in chaotic networks and can induce chaos in otherwise quiescent networks. anti-hebbian plasticity quickens activity and produces an oscillatory component. analysis of the jacobian shows that hebbian and anti-hebbian plasticity push locally unstable modes toward the real and imaginary axes, explaining these behaviors. both random-matrix and lyapunov analysis show that strong hebbian plasticity segregates network timescales into two bands with a slow, synapse-dominated band driving the dynamics, suggesting a flipped view of the network as synapses connected by neurons. for increasing strength, hebbian plasticity initially raises the complexity of the dynamics, measured by the maximum lyapunov exponent and attractor dimension, but then decreases these metrics, likely due to the proliferation of stable fixed points. we compute the marginally stable spectra of such fixed points as well as their number, showing exponential growth with network size. in chaotic states with strong hebbian plasticity, a stable fixed point of neuronal dynamics is destabilized by synaptic dynamics, allowing any neuronal state to be stored as a stable fixed point by halting the plasticity. this phase of freezable chaos offers a new mechanism for working memory.',\n",
       "  'categories': 'q-bio.nc cond-mat.dis-nn cs.ne',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-17',\n",
       "  'updated': '2024-01-10',\n",
       "  'authors': ['david g. clark', 'l. f. abbott'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.08985'},\n",
       " {'title': \"hairygami: analysis of dna nanostructures' conformational change driven   by functionalizable overhangs\",\n",
       "  'id': '2302.09109',\n",
       "  'abstract': 'dna origami is a widely used method to construct nanostructures by self-assembling designed dna strands. these structures are often used as \"pegboards\" for templated assembly of proteins, gold nanoparticles, aptamers, and other molecules, with applications ranging from therapeutics and diagnostics to plasmonics and photonics.   imaging these structures using afm or tem does not capture their full conformation ensemble as they only show their shape flattened on a surface. however, certain conformations of the nanostructure can position guest molecules into distances unaccounted for in their intended design, thus leading to spurious interactions between guest molecules that are designed to be separated. here, we use molecular dynamics simulations to capture conformational ensemble of 2d dna origami tiles and show that introducing single-stranded overhangs, which are typically used for functionalization of the origami with guest molecules, induces a curvature of the tile structure in the bulk. we show that the shape deformation is of entropic origin, with implications for design of robust dna origami breadboards as well as potential approach to modulate structure shape by introducing overhangs. we then verify experimentally that the dna overhangs introduce curvature into the dna origami tiles in divalent as well as monovalent salt buffer conditions. we further experimentally verify that dna origami functionalized with attached proteins also experience such induced curvature. we provide the developed simulation code implementing the enhanced sampling to characterize conformational space of dna origami as open source software.',\n",
       "  'categories': 'cond-mat.soft cond-mat.stat-mech physics.comp-ph q-bio.bm',\n",
       "  'doi': '10.1021/acsnano.4c10796',\n",
       "  'created': '2023-02-17',\n",
       "  'updated': '2024-08-22',\n",
       "  'authors': ['matthew sample',\n",
       "   'hao liu',\n",
       "   'thong diep',\n",
       "   'michael matthies',\n",
       "   'petr šulc'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.09109'},\n",
       " {'title': 'sbcoyote: an extensible python-based reaction editor and viewer',\n",
       "  'id': '2302.09151',\n",
       "  'abstract': 'sbcoyote is an open-source cross-platform biochemical reaction viewer and editor released under the liberal mit license. it is written in python and uses wxpython to implement the gui and the drawing canvas. it supports the visualization and editing of compartments, species, and reactions. it includes many options to stylize each of these components. for instance, species can be in different colors and shapes. other core features include the ability to create alias nodes, alignment of groups of nodes, network zooming, as well as an interactive bird-eye view of the network to allow easy navigation on large networks. a unique feature of the tool is the extensive python plugin api, where third-party developers can include new functionality. to assist third-party plugin developers, we provide a variety of sample plugins, including, random network generation, a simple auto layout tool, export to antimony, export sbml, import sbml, etc. of particular interest are the export and import sbml plugins since these support the sbml level 3 layout and render standard, which is exchangeable with other software packages. plugins are stored in a github repository, and an included plugin manager can retrieve and install new plugins from the repository on demand. plugins have version metadata associated with them to make it install plugin updates. availability: https://github.com/sys-bio/sbcoyote.',\n",
       "  'categories': 'q-bio.qm q-bio.mn',\n",
       "  'doi': '10.1016/j.biosystems.2023.105001',\n",
       "  'created': '2023-02-17',\n",
       "  'updated': '2023-08-14',\n",
       "  'authors': ['jin xu',\n",
       "   'gary geng',\n",
       "   'nhan d. nguyen',\n",
       "   'carmen perena-cortes',\n",
       "   'claire samuels',\n",
       "   'herbert m. sauro'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.09151'},\n",
       " {'title': 'crp-tree: a phylogenetic association test for binary traits',\n",
       "  'id': '2302.09402',\n",
       "  'abstract': 'an important problem in evolutionary genomics is to investigate whether a certain trait measured on each sample is associated with the sample phylogenetic tree. the phylogenetic tree represents the shared evolutionary history of the samples and it is usually estimated from molecular sequence data at a locus or from other type of genetic data. we propose a model for trait evolution inspired by the chinese restaurant process that includes a parameter that controls the degree of preferential attachment, that is, the tendency of nodes in the tree to subtend from nodes of the same type. this model with no preferential attachment is equivalent to a structured coalescent model with simultaneous migration and coalescence events and serves as a null model. we derive a test for phylogenetic binary trait association with linear computational complexity and empirically demonstrate that it is more powerful than some other methods. we apply our test to study the phylogenetic association of some traits in swordtail fish, breast cancer, yellow fever virus and influenza a h1n1 virus. r package implementation of our methods is available at $\\\\texttt{https://github.com/jyzhang27/crptree}$.',\n",
       "  'categories': 'q-bio.pe stat.ap',\n",
       "  'doi': '10.1093/jrsssc/qlad098',\n",
       "  'created': '2023-02-18',\n",
       "  'updated': '',\n",
       "  'authors': ['julie zhang',\n",
       "   'gabriel a. preising',\n",
       "   'molly schumer',\n",
       "   'julia a. palacios'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.09402'},\n",
       " {'title': 'inference of weak-form partial differential equations describing   migration and proliferation mechanisms in wound healing experiments on cancer   cells',\n",
       "  'id': '2302.09445',\n",
       "  'abstract': 'targeting signaling pathways that drive cancer cell migration or proliferation is a common therapeutic approach. a popular experimental technique, the scratch assay, measures the migration and proliferation-driven cell closure of a defect in a confluent cell monolayer. these assays do not measure dynamic effects. to improve analysis of scratch assays, we combine high-throughput scratch assays, video microscopy, and system identification to infer partial differential equation (pde) models of cell migration and proliferation. we capture the evolution of cell density fields over time using live cell microscopy and automated image processing. we employ weak form-based system identification techniques for cell density dynamics modeled with first-order kinetics of advection-diffusion-reaction systems. we present a comparison of our methods to results obtained using traditional inference approaches on previously analyzed 1-dimensional scratch assay data. we demonstrate the application of this pipeline on high throughput 2-dimensional scratch assays and find that low levels of trametinib inhibit wound closure primarily by decreasing random cell migration by approximately 20%. our integrated experimental and computational pipeline can be adapted for quantitatively inferring the effect of biological perturbations on cell migration and proliferation in various cell lines.',\n",
       "  'categories': 'q-bio.cb',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-18',\n",
       "  'updated': '2024-10-23',\n",
       "  'authors': ['patrick c. kinnunen',\n",
       "   'siddhartha srivastava',\n",
       "   'zhenlin wang',\n",
       "   'kenneth k. y. ho',\n",
       "   'brock a. humphries',\n",
       "   'siyi chen',\n",
       "   'jennifer j. linderman',\n",
       "   'gary d. luker',\n",
       "   'kathryn e. luker',\n",
       "   'krishna garikipati'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.09445'},\n",
       " {'title': 'learning temporal relationships between symbols with laplace neural   manifolds',\n",
       "  'id': '2302.10163',\n",
       "  'abstract': 'firing across populations of neurons in many regions of the mammalian brain maintains a temporal memory, a neural timeline of the recent past. behavioral results demonstrate that people can both remember the past and anticipate the future over an analogous internal timeline. this paper presents a mathematical framework for building this timeline of the future. we assume that the input to the system is a time series of symbols--sparse tokenized representations of the present--in continuous time. the goal is to record pairwise temporal relationships between symbols over a wide range of time scales. we assume that the brain has access to a temporal memory in the form of the real laplace transform. hebbian associations with a diversity of synaptic time scales are formed between the past timeline and the present symbol. the associative memory stores the convolution between the past and the present. knowing the temporal relationship between the past and the present allows one to infer relationships between the present and the future. with appropriate normalization, this hebbian associative matrix can store a laplace successor representation and a laplace predecessor representation from which measures of temporal contingency can be evaluated. the diversity of synaptic time constants allows for learning of non-stationary statistics as well as joint statistics between triplets of symbols. this framework synthesizes a number of recent neuroscientific findings including results from dopamine neurons in the mesolimbic forebrain.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-20',\n",
       "  'updated': '2024-09-21',\n",
       "  'authors': ['marc w. howard',\n",
       "   'zahra g. esfahani',\n",
       "   'bao le',\n",
       "   'per b. sederberg'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.10163'},\n",
       " {'title': 'snakes and ladders: a treewidth story',\n",
       "  'id': '2302.10662',\n",
       "  'abstract': 'let $g$ be an undirected graph. we say that $g$ contains a ladder of length $k$ if the $2 \\\\times (k+1)$ grid graph is an induced subgraph of $g$ that is only connected to the rest of $g$ via its four cornerpoints. we prove that if all the ladders contained in $g$ are reduced to length 4, the treewidth remains unchanged (and that this bound is tight). our result indicates that, when computing the treewidth of a graph, long ladders can simply be reduced, and that minimal forbidden minors for bounded treewidth graphs cannot contain long ladders. our result also settles an open problem from algorithmic phylogenetics: the common chain reduction rule, used to simplify the comparison of two evolutionary trees, is treewidth-preserving in the display graph of the two trees.',\n",
       "  'categories': 'math.co cs.ds q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-21',\n",
       "  'updated': '2024-01-30',\n",
       "  'authors': ['steven chaplick',\n",
       "   'steven kelk',\n",
       "   'ruben meuwese',\n",
       "   'matus mihalak',\n",
       "   'georgios stamoulis'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.10662'},\n",
       " {'title': 'abrupt and spontaneous strategy switches emerge in simple regularised   neural networks',\n",
       "  'id': '2302.11351',\n",
       "  'abstract': 'humans sometimes have an insight that leads to a sudden and drastic performance improvement on the task they are working on. sudden strategy adaptations are often linked to insights, considered to be a unique aspect of human cognition tied to complex processes such as creativity or meta-cognitive reasoning. here, we take a learning perspective and ask whether insight-like behaviour can occur in simple artificial neural networks, even when the models only learn to form input-output associations through gradual gradient descent. we compared learning dynamics in humans and regularised neural networks in a perceptual decision task that included a hidden regularity to solve the task more efficiently. our results show that only some humans discover this regularity, whose behaviour was marked by a sudden and abrupt strategy switch that reflects an aha-moment. notably, we find that simple neural networks with a gradual learning rule and a constant learning rate closely mimicked behavioural characteristics of human insight-like switches, exhibiting delay of insight, suddenness and selective occurrence in only some networks. analyses of network architectures and learning dynamics revealed that insight-like behaviour crucially depended on a regularised gating mechanism and noise added to gradient updates, which allowed the networks to accumulate \"silent knowledge\" that is initially suppressed by regularised (attentional) gating. this suggests that insight-like behaviour can arise naturally from gradual learning in simple neural networks, where it reflects the combined influences of noise, gating and regularisation.',\n",
       "  'categories': 'cs.ai q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-22',\n",
       "  'updated': '2024-03-01',\n",
       "  'authors': ['anika t. löwe',\n",
       "   'léo touzo',\n",
       "   'paul s. muhle-karbe',\n",
       "   'andrew m. saxe',\n",
       "   'christopher summerfield',\n",
       "   'nicolas w. schuck'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.11351'},\n",
       " {'title': 'aligned diffusion schr\\\\\"odinger bridges',\n",
       "  'id': '2302.11419',\n",
       "  'abstract': 'diffusion schr\\\\\"odinger bridges (dsb) have recently emerged as a powerful framework for recovering stochastic dynamics via their marginal observations at different time points. despite numerous successful applications, existing algorithms for solving dsbs have so far failed to utilize the structure of aligned data, which naturally arises in many biological phenomena. in this paper, we propose a novel algorithmic framework that, for the first time, solves dsbs while respecting the data alignment. our approach hinges on a combination of two decades-old ideas: the classical schr\\\\\"odinger bridge theory and doob\\'s $h$-transform. compared to prior methods, our approach leads to a simpler training procedure with lower variance, which we further augment with principled regularization schemes. this ultimately leads to sizeable improvements across experiments on synthetic and real data, including the tasks of predicting conformational changes in proteins and temporal evolution of cellular differentiation processes.',\n",
       "  'categories': 'cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-22',\n",
       "  'updated': '2024-04-28',\n",
       "  'authors': ['vignesh ram somnath',\n",
       "   'matteo pariset',\n",
       "   'ya-ping hsieh',\n",
       "   'maria rodriguez martinez',\n",
       "   'andreas krause',\n",
       "   'charlotte bunne'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.11419'},\n",
       " {'title': 'energy--information trade-off induces continuous and discontinuous phase   transitions in lateral predictive coding',\n",
       "  'id': '2302.11681',\n",
       "  'abstract': 'lateral predictive coding is a recurrent neural network which creates energy-efficient internal representations by exploiting statistical regularity in sensory inputs. here we investigate the trade-off between information robustness and energy in a linear model of lateral predictive coding analytically and by numerical minimization of a free energy. we observe several phase transitions in the synaptic weight matrix, especially a continuous transition which breaks reciprocity and permutation symmetry and builds cyclic dominance and a discontinuous transition with the associated sudden emergence of tight balance between excitatory and inhibitory interactions. the optimal network follows an ideal-gas law in an extended temperature range and saturates the efficiency upper-bound of energy utilization. these results bring theoretical insights on the emergence and evolution of complex internal models in predictive processing systems.',\n",
       "  'categories': 'q-bio.nc cond-mat.dis-nn',\n",
       "  'doi': '10.1007/s11433-024-2341-2',\n",
       "  'created': '2023-02-22',\n",
       "  'updated': '2024-01-10',\n",
       "  'authors': ['zhen-ye huang', 'ruyi zhou', 'miao huang', 'hai-jun zhou'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.11681'},\n",
       " {'title': 'equipocket: an e(3)-equivariant geometric graph neural network for   ligand binding site prediction',\n",
       "  'id': '2302.12177',\n",
       "  'abstract': 'predicting the binding sites of target proteins plays a fundamental role in drug discovery. most existing deep-learning methods consider a protein as a 3d image by spatially clustering its atoms into voxels and then feed the voxelized protein into a 3d cnn for prediction. however, the cnn-based methods encounter several critical issues: 1) defective in representing irregular protein structures; 2) sensitive to rotations; 3) insufficient to characterize the protein surface; 4) unaware of protein size shift. to address the above issues, this work proposes equipocket, an e(3)-equivariant graph neural network (gnn) for binding site prediction, which comprises three modules: the first one to extract local geometric information for each surface atom, the second one to model both the chemical and spatial structure of protein and the last one to capture the geometry of the surface via equivariant message passing over the surface atoms. we further propose a dense attention output layer to alleviate the effect incurred by variable protein size. extensive experiments on several representative benchmarks demonstrate the superiority of our framework to the state-of-the-art methods.',\n",
       "  'categories': 'q-bio.bm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-23',\n",
       "  'updated': '2024-07-22',\n",
       "  'authors': ['yang zhang',\n",
       "   'zhewei wei',\n",
       "   'ye yuan',\n",
       "   'chongxuan li',\n",
       "   'wenbing huang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.12177'},\n",
       " {'title': 'tutorial on survival modeling with applications to omics data',\n",
       "  'id': '2302.12542',\n",
       "  'abstract': \"motivation: identification of genomic, molecular and clinical markers prognostic of patient survival is important for developing personalized disease prevention, diagnostic and treatment approaches. modern omics technologies have made it possible to investigate the prognostic impact of markers at multiple molecular levels, including genomics, epigenomics, transcriptomics, proteomics and metabolomics, and how these potential risk factors complement clinical characterization of patient outcomes for survival prognosis. however, the massive sizes of the omics data sets, along with their correlation structures, pose challenges for studying relationships between the molecular information and patients' survival outcomes. results: we present a general workflow for survival analysis that is applicable to high-dimensional omics data as inputs when identifying survival-associated features and validating survival models. in particular, we focus on the commonly used cox-type penalized regressions and hierarchical bayesian models for feature selection in survival analysis, which are are especially useful for high-dimensional data, but the framework is applicable more generally. availability and implementation: a step-by-step r tutorial using the cancer genome atlas survival and omics data for the execution and evaluation of survival models has been made available at https://ocbe-uio.github.io/survomics/survomics.html.\",\n",
       "  'categories': 'stat.ap q-bio.gn',\n",
       "  'doi': '10.1093/bioinformatics/btae132',\n",
       "  'created': '2023-02-24',\n",
       "  'updated': '2024-03-04',\n",
       "  'authors': ['zhi zhao',\n",
       "   'john zobolas',\n",
       "   'manuela zucknick',\n",
       "   'tero aittokallio'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.12542'},\n",
       " {'title': 'a multimodal graph neural network framework of cancer molecular subtype   classification',\n",
       "  'id': '2302.12838',\n",
       "  'abstract': 'the recent development of high-throughput sequencing creates a large collection of multi-omics data, which enables researchers to better investigate cancer molecular profiles and cancer taxonomy based on molecular subtypes. integrating multi-omics data has been proven to be effective for building more precise classification models. current multi-omics integrative models mainly use early fusion by concatenation or late fusion based on deep neural networks. due to the nature of biological systems, graphs are a better representation of bio-medical data. although few graph neural network (gnn) based multi-omics integrative methods have been proposed, they suffer from three common disadvantages. one is most of them use only one type of connection, either inter-omics or intra-omic connection; second, they only consider one kind of gnn layer, either graph convolution network (gcn) or graph attention network (gat); and third, most of these methods lack testing on a more complex cancer classification task. we propose a novel end-to-end multi-omics gnn framework for accurate and robust cancer subtype classification. the proposed model utilizes multi-omics data in the form of heterogeneous multi-layer graphs that combines both inter-omics and intra-omic connections from established biological knowledge. the proposed model incorporates learned graph features and global genome features for accurate classification. we test the proposed model on tcga pan-cancer dataset and tcga breast cancer dataset for molecular subtype and cancer subtype classification, respectively. the proposed model outperforms four current state-of-the-art baseline models in multiple evaluation metrics. the comparative analysis of gat-based models and gcn-based models reveals that gat-based models are preferred for smaller graphs with less information and gcn-based models are preferred for larger graphs with extra information.',\n",
       "  'categories': 'q-bio.gn cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-24',\n",
       "  'updated': '2024-01-23',\n",
       "  'authors': ['bingjun li', 'sheida nabavi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.12838'},\n",
       " {'title': 'comparison and analysis of cognitive load under 2d/3d visual stimuli',\n",
       "  'id': '2302.12968',\n",
       "  'abstract': 'with the increasing prevalence of 3d videos, investigating the differences of viewing experiences between 2d and 3d videos has become an important issue. in this study, we explored the cognitive load induced by 2d and 3d video stimuli under various cognitive tasks utilizing electroencephalogram (eeg) data. we also introduced the cognitive load index (cli), a metric which combines {\\\\theta} and {\\\\alpha} oscillations to evaluate the cognitive differences. four video stimuli, each associated with typical cognitive tasks were adopted in our experiments. subjects were exposed to both 2d and 3d video stimuli, and the corresponding eeg data were recorded. then, we analyzed the power within the 0.5-45 hz frequency of eeg data, and cli was utilized to evaluate the brain activity of different subjects. according to our experiments and analysis, videos that involve simple observational tasks (p <0.05) consistently induced a higher cognitive load in subjects when they were viewing 3d videos. however, for videos that involve calculation tasks (p >0.05), the differences in cognitive load induced by 2d and 3d video were not obvious. thus, we concluded that 3d videos could generally induce a higher cognitive load, but the extent of the differences also depended on the contents of the video stimuli and the viewing purpose.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-24',\n",
       "  'updated': '2024-05-13',\n",
       "  'authors': ['yu liu',\n",
       "   'chen song',\n",
       "   'yunpeng yin',\n",
       "   'herui shi',\n",
       "   'jinglin sun',\n",
       "   'han wang',\n",
       "   'peiguang jing'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.12968'},\n",
       " {'title': 'a survey of machine learning techniques in medical applications',\n",
       "  'id': '2302.13268',\n",
       "  'abstract': 'in recent years, machine learning (ml) has emerged as a powerful tool for solving a wide range of problems, including medical decision-making. the exponential growth of medical data over the past two decades has surpassed the capacity for manual analysis, prompting increased interest in automated data analysis and processing. ml algorithms, capable of learning from data with minimal human intervention, are particularly well-suited for medical data analysis and interpretation. one significant advantage of ml is the reduced cost of collecting labeled training data necessary for supervised learning. while numerous studies have explored the applications of ml in medicine, this survey specifically focuses on the use of ml across various medical research fields. we provide a comprehensive technical overview of existing studies on ml applications in medicine, highlighting the strengths and limitations of these approaches. additionally, we discuss potential research directions for future exploration. these include the development of more sophisticated reward functions, as the accuracy of the reward function is crucial for ml performance, the integration of ml with other techniques, and the application of ml to new and emerging areas in genomics research. finally, we summarize our findings and present the current state of the field and the future outlook for ml in medical application.',\n",
       "  'categories': 'q-bio.gn cs.ai cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-26',\n",
       "  'updated': '2024-07-30',\n",
       "  'authors': ['m. keramy',\n",
       "   'k. jahanian',\n",
       "   'r. sani',\n",
       "   'a. agha',\n",
       "   'i. dehzangy',\n",
       "   'm. yan',\n",
       "   'h. rokni'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.13268'},\n",
       " {'title': 'nonlinear bias toward complex contagion in uncertain transmission   settings',\n",
       "  'id': '2302.13358',\n",
       "  'abstract': 'current epidemics in the biological and social domains are challenging the standard assumptions of mathematical contagion models. chief among them are the complex patterns of transmission caused by heterogeneous group sizes and infection risk varying by orders of magnitude in different settings, like indoor versus outdoor gatherings in the covid-19 pandemic or different moderation practices in social media communities. however, quantifying these heterogeneous levels of risk is difficult and most models typically ignore them. here, we include these novel features in an epidemic model on weighted hypergraphs to capture group-specific transmission rates. we study analytically the consequences of ignoring the heterogeneous transmissibility and find an induced superlinear infection rate during the emergence of a new outbreak, even though the underlying mechanism is a simple, linear contagion. the dynamics produced at the individual and group levels are therefore more similar to complex, nonlinear contagions, thus blurring the line between simple and complex contagions in realistic settings. we support this claim by introducing a bayesian inference framework to quantify the nonlinearity of contagion processes. we show that simple contagions on real weighted hypergraphs are systematically biased toward the superlinear regime if the heterogeneity of the weights is ignored, greatly increasing the risk of erroneous classification as complex contagions. our results provide an important cautionary tale for the challenging task of inferring transmission mechanisms from incidence data. yet, it also paves the way for effective models that capture complex features of epidemics through nonlinear infection rates.',\n",
       "  'categories': 'physics.soc-ph q-bio.pe',\n",
       "  'doi': '10.1073/pnas.2312202121',\n",
       "  'created': '2023-02-26',\n",
       "  'updated': '2023-12-30',\n",
       "  'authors': ['guillaume st-onge',\n",
       "   'laurent hébert-dufresne',\n",
       "   'antoine allard'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.13358'},\n",
       " {'title': 'individual based sis models on (not so) dense large random networks',\n",
       "  'id': '2302.13385',\n",
       "  'abstract': 'starting from a stochastic individual-based description of an sis epidemic spreading on a random network, we study the dynamics when the size $n$ of the network tends to infinity. we recover in the limit an infinite-dimensional integro-differential equation studied by delmas, dronnier and zitt (2022) for an sis epidemic propagating on a graphon. our work covers the case of dense and sparse graphs, provided that the number of edges grows faster than $n$, but not the case of very sparse graphs with $o(n)$ edges. in order to establish our limit theorem, we have to deal with both the convergence of the random graphs to the graphon and the convergence of the stochastic process spreading on top of these random structures: in particular, we propose a coupling between the process of interest and an epidemic that spreads on the complete graph but with a modified infection rate.   keywords: random graph, mathematical models of epidemics, measure-valued process, large network limit, limit theorem, graphon.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '10.30757/alea.v21-52',\n",
       "  'created': '2023-02-26',\n",
       "  'updated': '2024-09-09',\n",
       "  'authors': ['jean-françois delmas',\n",
       "   'paolo frasca',\n",
       "   'federica garin',\n",
       "   'viet chi tran',\n",
       "   'aurélien velleret',\n",
       "   'pierre-andré zitt'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.13385'},\n",
       " {'title': 'new go-based measures and their statistical significance in multiple   network alignment',\n",
       "  'id': '2302.13440',\n",
       "  'abstract': 'protein-protein interaction (ppi) networks provide valuable insights into the function of biological systems, and aligning multiple ppi networks can reveal important functional relationships between different species. however, assessing the quality of multiple network alignments is a challenging problem. in this paper, we propose two new measures, the squared go score (sgs) and the exposed g score, to evaluate the quality of multiple network alignments while using functional information from gene ontology (go) terms. we also introduce a $p$-value measure, the statistical exposed g score, to compute the exact significance of a multiple network alignment based on its revealed go terms. we also show that our measures are highly correlated with the recovered ortholog count, providing further evidence for their effectiveness. our work contributes to the development of more reliable and accurate measures for evaluating multiple network alignments and has potential applications in predicting gene function and identifying evolutionary relationships between different species using multiple network alignment.',\n",
       "  'categories': 'q-bio.mn',\n",
       "  'doi': '10.1093/bioinformatics/btae476',\n",
       "  'created': '2023-02-26',\n",
       "  'updated': '',\n",
       "  'authors': ['reza mousapour', 'kimia yazdani', 'wayne b. hayes'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.13440'},\n",
       " {'title': 'internal-coordinate density modelling of protein structure: covariance   matters',\n",
       "  'id': '2302.13711',\n",
       "  'abstract': 'after the recent ground-breaking advances in protein structure prediction, one of the remaining challenges in protein machine learning is to reliably predict distributions of structural states. parametric models of fluctuations are difficult to fit due to complex covariance structures between degrees of freedom in the protein chain, often causing models to either violate local or global structural constraints. in this paper, we present a new strategy for modelling protein densities in internal coordinates, which uses constraints in 3d space to induce covariance structure between the internal degrees of freedom. we illustrate the potential of the procedure by constructing a variational autoencoder with full covariance output induced by the constraints implied by the conditional mean in 3d, and demonstrate that our approach makes it possible to scale density models of internal coordinates to full protein backbones in two settings: 1) a unimodal setting for proteins exhibiting small fluctuations and limited amounts of available data, and 2) a multimodal setting for larger conformational changes in a high data regime.',\n",
       "  'categories': 'cs.lg q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-02-27',\n",
       "  'updated': '2024-01-24',\n",
       "  'authors': ['marloes arts', 'jes frellsen', 'wouter boomsma'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.13711'},\n",
       " {'title': 'nonlinear social evolution and the emergence of collective action',\n",
       "  'id': '2302.14700',\n",
       "  'abstract': 'organisms from microbes to humans engage in a variety of social behaviors, which affect fitness in complex, often nonlinear ways. the question of how these behaviors evolve has consequences ranging from antibiotic resistance to human origins. however, evolution with nonlinear social interactions is challenging to model mathematically, especially in combination with spatial, group, and/or kin assortment. we derive a mathematical condition for natural selection with synergistic interactions among any number of individuals. this result applies to populations with arbitrary (but fixed) spatial or network structure, group subdivision, and/or mating patterns. in this condition, nonlinear fitness effects are ascribed to collectives, and weighted by a new measure of collective relatedness. for weak selection, this condition can be systematically evaluated by computing branch lengths of ancestral trees. we apply this condition to pairwise games between diploid relatives, and to dilemmas of collective help or harm among siblings and on spatial networks. our work provides a rigorous basis for extending the notion of ``actor\", in the study of social evolution, from individuals to collectives.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1093/pnasnexus/pgae131',\n",
       "  'created': '2023-02-28',\n",
       "  'updated': '2024-06-13',\n",
       "  'authors': ['benjamin allen',\n",
       "   'abdur-rahman khwaja',\n",
       "   'james l. donahue',\n",
       "   'cassidy lattanzio',\n",
       "   'yulia a. dementieva',\n",
       "   'christine sample'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2302.14700'},\n",
       " {'title': 'overcompensation of transient and permanent death rate increases in   age-structured models with cannibalistic interactions',\n",
       "  'id': '2303.00864',\n",
       "  'abstract': 'there has been renewed interest in understanding the mathematical structure of ecological population models that lead to overcompensation, the process by which a population recovers to a higher level after suffering a permanent increase in predation or harvesting. here, we apply a recently formulated kinetic population theory to formally construct an age-structured single-species population model that includes a cannibalistic interaction in which older individuals prey on younger ones. depending on the age-dependent structure of this interaction, our model can exhibit transient or steady-state overcompensation of an increased death rate as well as oscillations of the total population, both phenomena that have been observed in ecological systems. analytic and numerical analysis of our model reveals sufficient conditions for overcompensation and oscillations. we also show how our structured population partial integrodifferential equation (pide) model can be reduced to coupled ode models representing piecewise constant parameter domains, providing additional mathematical insight into the emergence of overcompensation.',\n",
       "  'categories': 'q-bio.pe q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-01',\n",
       "  'updated': '2024-03-14',\n",
       "  'authors': ['mingtao xia', 'xiangting li', 'tom chou'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.00864'},\n",
       " {'title': 'a neuro-symbolic ai approach to personal health risk assessment and   immune age characterisation using common blood markers',\n",
       "  'id': '2303.01444',\n",
       "  'abstract': \"we introduce a simulated digital model that learns a person's baseline blood health over time. using an adaptive learning algorithm, the model provides a risk assessment score that compares an individual's chronological age with an estimation of biological age based on common immune-relevant markers used in current clinical practice. we demonstrate its efficacy on real and synthetic data from medically relevant cases, extreme cases, and empirical blood cell count data from 100k data records in the centers for disease control and prevention's national health and nutrition examination survey (cdc nhanes) that spans 13 years. we find that the score is informative in distinguishing healthy individuals from those with diseases, both self-reported and as manifested via abnormal blood test results, providing an entry-level score for patient triaging. the risk assessment score is not a machine learning black-box approach but can interact with ml and dl approaches to help guide, control the attention given to specific features, and assign proper explainable weight to an otherwise transparent adaptive learning algorithm. this approach may allow fast and scalable deployment to personalised, sensitive, and predictive derivative indexes within digital medicine, without the need for a new test, assay, or prospective sampling, unlike other biological ageing-related scores and methods. it demonstrates the potential of clinical informatics and deep medicine in digital healthcare as drivers of innovation in preventive patient care.\",\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-02',\n",
       "  'updated': '2024-11-11',\n",
       "  'authors': ['santiago hernández-orozco',\n",
       "   'abicumaran uthamacumaran',\n",
       "   'francisco hernández-quiroz',\n",
       "   'kourosh saeb-parsy',\n",
       "   'hector zenil'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.01444'},\n",
       " {'title': 'additive invariants of open petri nets',\n",
       "  'id': '2303.01643',\n",
       "  'abstract': \"we classify all additive invariants of open petri nets: these are $\\\\mathbb{n}$-valued invariants which are additive with respect to sequential and parallel composition of open petri nets. in particular, we prove two classification theorems: one for open petri nets and one for monically open petri nets (i.e. open petri nets whose interfaces are specified by monic maps). our results can be summarized as follows. the additive invariants of open petri nets are completely determined by their values on a particular class of single-transition petri nets. however, for monically open petri nets, the additive invariants are determined by their values on transitionless petri nets and all single-transition petri nets. our results confirm a conjecture of john baez (stated during the ams' 2022 mathematical research communities workshop).\",\n",
       "  'categories': 'math.ct q-bio.mn',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-02',\n",
       "  'updated': '2024-02-22',\n",
       "  'authors': ['benjamin merlin bumpus',\n",
       "   'sophie libkind',\n",
       "   'jordy lopez garcia',\n",
       "   'layla sorkatti',\n",
       "   'samuel tenka'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.01643'},\n",
       " {'title': 'fundamental bound on epidemic overshoot in the sir model',\n",
       "  'id': '2303.01650',\n",
       "  'abstract': 'we derive an exact upper bound on the epidemic overshoot for the kermack-mckendrick sir model. this maximal overshoot value of 0.2984... occurs at $r_0^*$ = 2.151... . in considering the utility of the notion of overshoot, a rudimentary analysis of data from the first wave of the covid-19 pandemic in manaus, brazil highlights the public health hazard posed by overshoot for epidemics with $r_0$ near 2. using the general analysis framework presented within, we then consider more complex sir models that incorporate vaccination.',\n",
       "  'categories': 'q-bio.pe physics.bio-ph physics.soc-ph',\n",
       "  'doi': '10.1098/rsif.2023.0322',\n",
       "  'created': '2023-03-02',\n",
       "  'updated': '2023-11-03',\n",
       "  'authors': ['maximilian nguyen',\n",
       "   'ari freedman',\n",
       "   'sinan ozbay',\n",
       "   'simon levin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.01650'},\n",
       " {'title': 'upper bounds on overshoot in sir models with nonlinear incidence',\n",
       "  'id': '2303.01653',\n",
       "  'abstract': 'we expand the calculation of the upper bound on epidemic overshoot in sir models to account for nonlinear incidence. we lay out the general procedure and restrictions to perform the calculation analytically for nonlinear functions in the number of susceptibles. we demonstrate the procedure by working through several examples and also numerically study what happens to the upper bound on overshoot when nonlinear incidence manifests in the form of epidemic dynamics over a contact network. we find that both steeper incidence terms and larger contact heterogeneity can increase the range of communicable diseases at which the overshoot remains a relatively large public health hazard.',\n",
       "  'categories': 'q-bio.pe math.ds physics.bio-ph physics.soc-ph',\n",
       "  'doi': '10.1038/s44260-024-00010-2',\n",
       "  'created': '2023-03-02',\n",
       "  'updated': '2024-08-04',\n",
       "  'authors': ['maximilian nguyen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.01653'},\n",
       " {'title': 'insights from number theory into the critical kauffman model with   connectivity one',\n",
       "  'id': '2303.02079',\n",
       "  'abstract': 'the kauffman model of genetic computation highlights the importance of criticality at the border of order and chaos. the model with connectivity one is of special interest because it is exactly solvable. but our understanding of its behavior is incomplete, and much of what we do know relies on heuristic arguments. here, we show that the key quantities in the model are intimately related to aspects of number theory. using these links, we derive improved bounds for the number of attractors as well as the mean attractor length, which is harder to compute. our work suggests that number theory is the natural language for deducing many properties of the critical kauffman model with connectivity one, and opens the door to further insight into this deceptively simple model.',\n",
       "  'categories': 'q-bio.mn cond-mat.dis-nn math.pr',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-03',\n",
       "  'updated': '2024-04-24',\n",
       "  'authors': ['f. c. sheldon', 't. m. a. fink'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.02079'},\n",
       " {'title': 'estimating the parameters of epidemic spread on two-layer random graphs:   a classical and a neural network approach',\n",
       "  'id': '2303.02195',\n",
       "  'abstract': \"in this paper, we study the spread of a classical sir process on a two-layer random network, where the first layer represents the households, while the second layer models the contacts outside the households by a random scale-free graph. we build a three-parameter graph, called polynomial model, where the new vertices are connected to the existing ones either uniformly, or preferentially, or by forming random triangles. we examine the effect of the graph's properties on the goodness of the estimation of the infection rate $\\\\tau$, which is the most important parameter, determining the reproduction rate of the epidemic.   in the classical maximum likelihood approach, to estimate $\\\\tau$ one needs to approximate the number of si edges between households, since the graph itself is supposed to be unobservable. our simulation study reveals that the estimation is poorer at the beginning of the epidemic, for larger preferential attachment parameter of the graph, and for larger $\\\\tau$. we present two heuristic improvement algorithms and establish our method to be robust to changes in average clustering of the graph model.   we also extend a graph neural network (gnn) approach for estimating contagion dynamics for our two-layered graphs. we find that dense networks offer better training datasets. moreover, gnn perfomance is measured better using the $l_2$ loss function rather than cross-entropy.\",\n",
       "  'categories': 'q-bio.pe physics.soc-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-03',\n",
       "  'updated': '2024-02-29',\n",
       "  'authors': ['ágnes backhausz',\n",
       "   'edit bognár',\n",
       "   'villő csiszár',\n",
       "   'damján tárkányi',\n",
       "   'andrás zempléni'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.02195'},\n",
       " {'title': 'the critical beta-splitting random tree ii: overview and open problems',\n",
       "  'id': '2303.02529',\n",
       "  'abstract': 'in the critical beta-splitting model of a random $n$-leaf rooted tree, clades are recursively (from the root) split into sub-clades, and a clade of $m$ leaves is split into sub-clades containing $i$ and $m-i$ leaves with probabilities $\\\\propto 1/(i(m-i))$. study of structure theory and explicit quantitative aspects of the model is an active research topic, and this article provides an extensive overview what is currently known. for many results there are different proofs, probabilistic or analytic, so the model provides a testbed for a ``compare and contrast\" discussion of techniques. we give some proofs that are not currently available elsewhere, and also give heuristics for some proven results and for some open problems. our discussion is centered around three ``foundational\" results.   (i) there is a canonical embedding into a continuous-time model, that is a random tree $\\\\mbox{ctcs}(n)$ on $n$ leaves with real-valued edge lengths, and this model turns out more convenient to study. the family $(\\\\mbox{ctcs}(n), n \\\\ge 2)$ is consistent under a ``delete random leaf and prune\" operation. that leads to an explicit inductive construction of $(\\\\mbox{ctcs}(n), n \\\\ge 2)$ as $n$ increases, and then to a limit structure $\\\\mbox{ctcs}(\\\\infty)$ formalized via exchangeable partitions, in some ways analogous to the brownian continuum random tree.   (ii) there is a clt for leaf heights, and the analytic proof can be extended to provide surprisingly precise analysis of other height-related aspects.   (iii) there is an explicit description of the limit {\\\\em fringe distribution} relative to a random leaf, whose graphical representation is essentially the format of the cladogram representation of biological phylogenies.   many open problems remain.',\n",
       "  'categories': 'math.pr math.co q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-04',\n",
       "  'updated': '2024-07-05',\n",
       "  'authors': ['david j. aldous', 'svante janson'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.02529'},\n",
       " {'title': 'on competition through growth reduction',\n",
       "  'id': '2303.02981',\n",
       "  'abstract': 'we consider a population organised hierarchically with respect to size in such a way that the growth rate of each individual depends only on the presence of larger individuals. as a concrete example one might think of a forest, in which the incidence of light on a tree (and hence how fast it grows) is affected by shading of taller trees. the model is formulated as a delay equation, more specifically a scalar renewal equation, for the population birth rate. after discussing the well-posedness of the model, we analyse how many stationary birth rates the equation can have in terms of the functional parameters of the model. in particular we show that, under reasonable and rather general assumptions, only one stationary birth rate can exist besides the trivial one (associated to the state in which there are no individuals and the population birth rate is zero). we give conditions for this non-trivial stationary birth rate to exist and we analyse its stability using the principle of linearised stability for delay equations. finally we relate the results to an alternative formulation of the model taking the form of a quasilinear partial differential equation for the population size-density.',\n",
       "  'categories': 'q-bio.pe math.ap',\n",
       "  'doi': '10.1007/s00285-024-02084-x',\n",
       "  'created': '2023-03-06',\n",
       "  'updated': '2024-04-22',\n",
       "  'authors': ['carles barril',\n",
       "   'àngel calsina',\n",
       "   'odo diekmann',\n",
       "   'józsef z. farkas'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.02981'},\n",
       " {'title': 'ecg classification system for arrhythmia detection using convolutional   neural networks',\n",
       "  'id': '2303.03660',\n",
       "  'abstract': 'arrhythmia is just one of the many cardiovascular illnesses that have been extensively studied throughout the years. using multi-lead ecg data, this research describes a deep learning (dl) pipeline technique based on convolutional neural network (cnn) algorithms to detect cardiovascular lar arrhythmia in patients. the suggested model architecture has hidden layers with a residual block in addition to the input and output layers. in this study, the classification of the ecg signals into five main groups, namely: left bundle branch block (lbbb), right bundle branch block (rbbb), atrial premature contraction (apc), premature ventricular contraction (pvc), and normal beat (n), are performed. using the mit-bih arrhythmia dataset, we assessed the suggested technique. the findings show that our suggested strategy classified 15,000 cases with a high accuracy of 98.2%',\n",
       "  'categories': 'eess.sp cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-07',\n",
       "  'updated': '2024-06-12',\n",
       "  'authors': ['aryan odugoudar', 'jaskaran singh walia'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.03660'},\n",
       " {'title': 'on the connectedness of multistationarity regions of small reaction   networks',\n",
       "  'id': '2303.03960',\n",
       "  'abstract': \"a multistationarity region is the part of a reaction network's parameter space that gives rise to multiple steady states. mathematically, this region consists of the positive parameters for which a parametrized family of polynomial equations admits two or more positive roots. much recent work has focused on analyzing multistationarity regions of biologically significant reaction networks and determining whether such regions are connected; indeed, a better understanding of the topology and geometry of such regions may help elucidate how robust multistationarity is to perturbations. here we focus on the multistationarity regions of small networks, those with few species and few reactions. for two families of such networks -- those with one species and up to three reactions, and those with two species and up to two reactions -- we prove that the resulting multistationarity regions are connected. we also give an example of a network with one species and six reactions for which the multistationarity region is disconnected. our proofs rely on the formula for the discriminant of a trinomial, a classification of small multistationary networks, and a recent result of feliu and telek that partially generalizes descartes' rule of signs.\",\n",
       "  'categories': 'math.ds q-bio.mn',\n",
       "  'doi': '10.1016/j.jsc.2024.102323',\n",
       "  'created': '2023-03-07',\n",
       "  'updated': '2024-04-05',\n",
       "  'authors': ['allison mcclure', 'anne shiu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.03960'},\n",
       " {'title': 'the evolution of cooperation and diversity by integrated indirect   reciprocity',\n",
       "  'id': '2303.04467',\n",
       "  'abstract': \"indirect reciprocity is one of the major mechanisms for the evolution of cooperation in human societies. there are two types of indirect reciprocity: upstream and downstream. cooperation in downstream reciprocity follows the pattern, 'you helped someone, and i will help you'. the direction of cooperation is reversed in upstream reciprocity, which instead follows the pattern, 'you helped me, and i will help someone else'. in reality, these two types of indirect reciprocity often occur in combination. however, upstream and downstream reciprocity have mostly been studied theoretically in isolation. here, we propose a new model that integrates both types. we apply the standard giving-game framework of indirect reciprocity and analyze the model by means of evolutionary game theory. we show that the model can result in the stable coexistence of altruistic reciprocators and free riders in well-mixed populations. we also found that considering inattention in the assessment rule can strengthen the stability of this mixed equilibrium, even resulting in a global attractor. our results indicate that the cycles of forwarding help and rewarding help need to be established for creating and maintaining diversity and inclusion in a society.\",\n",
       "  'categories': 'q-bio.pe cs.cy nlin.ao physics.soc-ph',\n",
       "  'doi': '10.3390/g15020015',\n",
       "  'created': '2023-03-08',\n",
       "  'updated': '',\n",
       "  'authors': ['tatsuya sasaki',\n",
       "   'satoshi uchida',\n",
       "   'isamu okada',\n",
       "   'hitoshi yamamoto'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.04467'},\n",
       " {'title': 'inter-brain substrates of role switching during mother-child interaction',\n",
       "  'id': '2303.04902',\n",
       "  'abstract': \"mother-child interaction is highly dynamic and reciprocal. switching roles in these back-and-forth interactions serves as a crucial feature of reciprocal behaviors while the underlying neural entrainment is still not well-studied. here, we designed a role-controlled cooperative task with dual eeg recording to study how differently two brains interact when mothers and children hold different roles. when children were actors and mothers were observers, mother-child inter-brain synchrony emerged within the theta oscillations and the frontal lobe, which highly correlated with children's attachment to their mothers. when their roles were reversed, this synchrony was shifted to the alpha oscillations and the central area and associated with mothers' perception of their relationship with their children. the results suggested an observer-actor neural alignment within the actor's oscillations, which was modulated by the actor-toward-observer emotional bonding. our findings contribute to the understanding of how inter-brain synchrony is established and dynamically changed during mother-child reciprocal interaction.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1002/hbm.26672',\n",
       "  'created': '2023-03-08',\n",
       "  'updated': '',\n",
       "  'authors': ['yamin li',\n",
       "   'saishuang wu',\n",
       "   'jiayang xu',\n",
       "   'haiwa wang',\n",
       "   'qi zhu',\n",
       "   'wen shi',\n",
       "   'yue fang',\n",
       "   'fan jiang',\n",
       "   'shanbao tong',\n",
       "   'yunting zhang',\n",
       "   'xiaoli guo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.04902'},\n",
       " {'title': 'emergent rate-based dynamics in duplicate-free populations of spiking   neurons',\n",
       "  'id': '2303.05174',\n",
       "  'abstract': 'can spiking neural networks (snns) approximate the dynamics of recurrent neural networks (rnns)? arguments in classical mean-field theory based on laws of large numbers provide a positive answer when each neuron in the network has many \"duplicates\", i.e. other neurons with almost perfectly correlated inputs. using a disordered network model that guarantees the absence of duplicates, we show that duplicate-free snns can converge to rnns, thanks to the concentration of measure phenomenon. this result reveals a general mechanism underlying the emergence of rate-based dynamics in large snns.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-09',\n",
       "  'updated': '2024-11-07',\n",
       "  'authors': ['valentin schmutz', 'johanni brea', 'wulfram gerstner'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.05174'},\n",
       " {'title': 'the epidemiological footprint of contact structures in models with two   levels of mixing',\n",
       "  'id': '2303.05287',\n",
       "  'abstract': 'models with several levels of mixing (households, workplaces), as well as various corresponding formulations for r0, have been proposed in the literature. however, little attention has been paid to the impact of the distribution of the population size within social structures, effect that can help plan effective interventions. we focus on the influence on the model outcomes of teleworking strategies, consisting in reshaping the distribution of workplace sizes. we consider a stochastic sir model with two levels of mixing, accounting for a uniformly mixing general population, each individual belonging also to a household and a workplace. the variance of the workplace size distribution appears to be a good proxy for the impact of this distribution on key outcomes of the epidemic, such as epidemic size and peak. in particular, our findings suggest that strategies where the proportion of individuals teleworking depends sublinearly on the size of the workplace outperform the strategy with linear dependence. besides, one drawback of the model with multiple levels of mixing is its complexity, raising interest in a reduced model. we propose a homogeneously mixing sir ode-based model, whose infection rate is chosen as to observe the growth rate of the initial model. this reduced model yields a generally satisfying approximation of the epidemic. these results, robust to various changes in model structure, are very promising from the perspective of implementing effective strategies based on social distancing of specific contacts. furthermore, they contribute to the effort of building relevant approximations of individual based models at intermediate scales.',\n",
       "  'categories': 'physics.soc-ph q-bio.pe',\n",
       "  'doi': '10.1007/s00285-024-02147-z',\n",
       "  'created': '2023-03-08',\n",
       "  'updated': '2024-11-20',\n",
       "  'authors': ['vincent bansaye',\n",
       "   'françois deslandes',\n",
       "   'madeleine kubasch',\n",
       "   'elisabeta vergu'],\n",
       "  'affiliation': ['cmap, merge', 'maiage', 'cmap, maiage, merge', 'maiage'],\n",
       "  'url': 'https://arxiv.org/abs/2303.05287'},\n",
       " {'title': 'unbiased likelihood estimation of wright-fisher diffusion processes',\n",
       "  'id': '2303.05390',\n",
       "  'abstract': 'in this paper we propose a monte carlo maximum likelihood estimation strategy for discretely observed wright-fisher diffusions. our approach provides an unbiased estimator of the likelihood function and is based on exact simulation techniques that are of special interest for diffusion processes defined on a bounded domain, where numerical methods typically fail to remain within the required boundaries. we start by building unbiased likelihood estimators for scalar diffusions and later present an extension to the multidimensional case. consistency results of our proposed estimator are also presented and the performance of our method is illustrated through numerical examples.',\n",
       "  'categories': 'math.st q-bio.pe stat.th',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-08',\n",
       "  'updated': '2024-06-10',\n",
       "  'authors': ['celia garcía-pareja', 'fabio nobile'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.05390'},\n",
       " {'title': 'exploration of the search space of gaussian graphical models for paired   data',\n",
       "  'id': '2303.05561',\n",
       "  'abstract': 'we consider the problem of learning a gaussian graphical model in the case where the observations come from two dependent groups sharing the same variables. we focus on a family of coloured gaussian graphical models specifically suited for the paired data problem. commonly, graphical models are ordered by the submodel relationship so that the search space is a lattice, called the model inclusion lattice. we introduce a novel order between models, named the twin order. we show that, embedded with this order, the model space is a lattice that, unlike the model inclusion lattice, is distributive. furthermore, we provide the relevant rules for the computation of the neighbours of a model. the latter are more efficient than the same operations in the model inclusion lattice, and are then exploited to achieve a more efficient exploration of the search space. these results can be applied to improve the efficiency of both greedy and bayesian model search procedures. here we implement a stepwise backward elimination procedure and evaluate its performance by means of simulations. finally, the procedure is applied to learn a brain network from fmri data where the two groups correspond to the left and right hemispheres, respectively.',\n",
       "  'categories': 'stat.ml cs.lg q-bio.nc stat.me',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-09',\n",
       "  'updated': '2024-04-15',\n",
       "  'authors': ['alberto roverato', 'dung ngoc nguyen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.05561'},\n",
       " {'title': 'chemical kinetic theory of aging',\n",
       "  'id': '2303.05585',\n",
       "  'abstract': 'a theory of aging based on the principles of the kinetics of chemical reactions and the rules of natural selection of organisms is proposed. the theory is based on a hypothesis that the biochemical processes in the organism can be described in the terms of chemical reaction kinetics. the evolutionary process of organisms is determined by the goal of continuing life, and the natural selection forced organisms to develop in an optimized way for survival and reproduction, after which any further development of the organisms did not matter for natural selection and therefore was not regulated by it. accordingly, the ongoing biochemical processes in the organism after giving birth were not stabilised and continued casual, which, in the case of complex systems of biochemical processes, led to imbalances and biochemical processes that did not contribute to health in the organism, resulting in aging. based on this view, it is necessary to look for the key biochemical processes that regulate the vital activity of the organism. balancing of these processes in the period after giving birth can artificially lead to the stabilization of the kinetics of biochemical reactions, and consequently, the continuation of life almost indefinitely.',\n",
       "  'categories': 'q-bio.pe physics.chem-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-09',\n",
       "  'updated': '2024-01-06',\n",
       "  'authors': ['alexey kondyurin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.05585'},\n",
       " {'title': 'international vaccine allocation: an optimization framework',\n",
       "  'id': '2303.05917',\n",
       "  'abstract': \"as observed during the covid-19 pandemic, high-income countries, such as the u.s., may exhibit vaccine nationalism during a pandemic: stockpiling doses of vaccine for their own citizens and being reluctant to distribute doses of the vaccine to lower-income countries. while many cite moral objections to vaccine nationalism, vaccine inequity during a pandemic could possibly worsen the global effects of the pandemic, including in the high-income countries themselves, through the evolution of new variants of the virus. this paper uses the covid-19 pandemic as a case study to identify scenarios under which it might be in a high-income nation's own interest to donate vaccine doses to another country before its own population has been fully vaccinated. we develop an extended seir (susceptible-exposed-infectious-recovered) epidemiological model embedded in an optimization framework and examine scenarios involving a single donor and multiple recipient (nondonor) geographic areas. we find that policies other than donor-first can delay the emergence of a more-contagious variant compared to donor-first, sometimes reducing donor-country deaths in addition to total deaths. thus, vaccine distribution is not a zero-sum game between donor and nondonor countries: an optimization approach can achieve a dramatic reduction in total deaths with only a small increase in donor-country deaths. the iterative linear programming approximation approach we develop can help confirm those instances when a priority policy is optimal and, when not optimal, can identify superior policies. this optimization framework can be used to guide equitable vaccine distribution in future pandemics.\",\n",
       "  'categories': 'q-bio.pe math.oc',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-08',\n",
       "  'updated': '2024-08-20',\n",
       "  'authors': ['abraham holleran', 'susan e. martonosi', 'michael veatch'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.05917'},\n",
       " {'title': 'optimal foraging strategies can be learned',\n",
       "  'id': '2303.06050',\n",
       "  'abstract': \"the foraging behavior of animals is a paradigm of target search in nature. understanding which foraging strategies are optimal and how animals learn them are central challenges in modeling animal foraging. while the question of optimality has wide-ranging implications across fields such as economy, physics, and ecology, the question of learnability is a topic of ongoing debate in evolutionary biology. recognizing the interconnected nature of these challenges, this work addresses them simultaneously by exploring optimal foraging strategies through a reinforcement learning framework. to this end, we model foragers as learning agents. we first prove theoretically that maximizing rewards in our reinforcement learning model is equivalent to optimizing foraging efficiency. we then show with numerical experiments that, in the paradigmatic model of non-destructive search, our agents learn foraging strategies which outperform the efficiency of some of the best known strategies such as l\\\\'evy walks. these findings highlight the potential of reinforcement learning as a versatile framework not only for optimizing search strategies but also to model the learning process, thus shedding light on the role of learning in natural optimization processes.\",\n",
       "  'categories': 'cond-mat.stat-mech cs.ai physics.bio-ph q-bio.pe',\n",
       "  'doi': '10.1088/1367-2630/ad19a8',\n",
       "  'created': '2023-03-10',\n",
       "  'updated': '2023-08-03',\n",
       "  'authors': ['gorka muñoz-gil',\n",
       "   'andrea lópez-incera',\n",
       "   'lukas j. fiderer',\n",
       "   'hans j. briegel'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.06050'},\n",
       " {'title': 'efficient simulation of individual-based population models: the r   package ibmpopsim',\n",
       "  'id': '2303.06183',\n",
       "  'abstract': 'the r package ibmpopsim aims to simulate the random evolution of heterogeneous populations using stochastic individual-based models (ibms).   the package enables users to simulate population evolution, in which individuals are characterized by their age and some characteristics, and the population is modified by different types of events, including births/arrivals, death/exit events, or changes of characteristics. the frequency at which an event can occur to an individual can depend on their age and characteristics, but also on the characteristics of other individuals (interactions). such models have a wide range of applications in fields including actuarial science, biology, ecology or epidemiology.   ibmpopsim overcomes the limitations of time-consuming ibms simulations by implementing new efficient algorithms based on thinning methods, which are compiled using the rcpp package while providing a user-friendly interface.',\n",
       "  'categories': 'q-bio.pe cs.ms math.pr',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-10',\n",
       "  'updated': '2024-02-27',\n",
       "  'authors': ['daphné giorgi', 'sarah kaakai', 'vincent lemaire'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.06183'},\n",
       " {'title': 'bridging the gap between chemical reaction pretraining and conditional   molecule generation with a unified model',\n",
       "  'id': '2303.06965',\n",
       "  'abstract': 'chemical reactions are the fundamental building blocks of drug design and organic chemistry research. in recent years, there has been a growing need for a large-scale deep-learning framework that can efficiently capture the basic rules of chemical reactions. in this paper, we have proposed a unified framework that addresses both the reaction representation learning and molecule generation tasks, which allows for a more holistic approach. inspired by the organic chemistry mechanism, we develop a novel pretraining framework that enables us to incorporate inductive biases into the model. our framework achieves state-of-the-art results on challenging downstream tasks. by possessing chemical knowledge, our generative framework overcome the limitations of current molecule generation models that rely on a small number of reaction templates. in the extensive experiments, our model generates synthesizable drug-like structures of high quality. overall, our work presents a significant step toward a large-scale deep-learning framework for a variety of reaction-based applications.',\n",
       "  'categories': 'cs.lg q-bio.bm',\n",
       "  'doi': '10.1038/s42256-023-00764-9',\n",
       "  'created': '2023-03-13',\n",
       "  'updated': '2024-03-07',\n",
       "  'authors': ['bo qiang',\n",
       "   'yiran zhou',\n",
       "   'yuheng ding',\n",
       "   'ningfeng liu',\n",
       "   'song song',\n",
       "   'liangren zhang',\n",
       "   'bo huang',\n",
       "   'zhenming liu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.06965'},\n",
       " {'title': 'stochastic nutrient-plankton models',\n",
       "  'id': '2303.07534',\n",
       "  'abstract': 'we analyze plankton-nutrient food chain models composed of phytoplankton, herbivorous zooplankton and a limiting nutrient. these models have played a key role in understanding the dynamics of plankton in the oceanic layer. given the strong environmental and seasonal fluctuations that are present in the oceanic layer, we propose a stochastic model for which we are able to fully classify the longterm behavior of the dynamics. in order to achieve this we had to develop new analytical techniques, as the system does not satisfy the regular dissipativity conditions and the analysis is more subtle than in other population dynamics models.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '10.1016/j.jde.2023.08.017',\n",
       "  'created': '2023-03-13',\n",
       "  'updated': '',\n",
       "  'authors': ['alexandru hening',\n",
       "   'nguyen trong hieu',\n",
       "   'dang hai nguyen',\n",
       "   'nhu ngoc nguyen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.07534'},\n",
       " {'title': 'tensor-based multimodal learning for prediction of pulmonary arterial   wedge pressure from cardiac mri',\n",
       "  'id': '2303.07540',\n",
       "  'abstract': 'heart failure is a serious and life-threatening condition that can lead to elevated pressure in the left ventricle. pulmonary arterial wedge pressure (pawp) is an important surrogate marker indicating high pressure in the left ventricle. pawp is determined by right heart catheterization (rhc) but it is an invasive procedure. a non-invasive method is useful in quickly identifying high-risk patients from a large population. in this work, we develop a tensor learning-based pipeline for identifying pawp from multimodal cardiac magnetic resonance imaging (mri). this pipeline extracts spatial and temporal features from high-dimensional scans. for quality control, we incorporate an epistemic uncertainty-based binning strategy to identify poor-quality training samples. to improve the performance, we learn complementary information by integrating features from multimodal data: cardiac mri with short-axis and four-chamber views, and electronic health records. the experimental analysis on a large cohort of $1346$ subjects who underwent the rhc procedure for pawp estimation indicates that the proposed pipeline has a diagnostic value and can produce promising performance with significant improvement over the baseline in clinical practice (i.e., $\\\\delta$auc $=0.10$, $\\\\delta$accuracy $=0.06$, and $\\\\delta$mcc $=0.39$). the decision curve analysis further confirms the clinical utility of our method.',\n",
       "  'categories': 'cs.lg cs.cv q-bio.qm',\n",
       "  'doi': '10.1007/978-3-031-43990-2_20',\n",
       "  'created': '2023-03-13',\n",
       "  'updated': '2024-04-06',\n",
       "  'authors': ['prasun c. tripathi',\n",
       "   'mohammod n. i. suvon',\n",
       "   'lawrence schobs',\n",
       "   'shuo zhou',\n",
       "   'samer alabed',\n",
       "   'andrew j. swift',\n",
       "   'haiping lu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.07540'},\n",
       " {'title': 'far-reaching consequences of trait-based social preferences for the   structure and function of animal social networks',\n",
       "  'id': '2303.08107',\n",
       "  'abstract': 'social network structures play an important role in the lives of animals by affecting individual fitness, and the spread of disease and information. nevertheless, we still lack a good understanding of how these structures emerge from the behaviour of individuals. generative network models based on empirical knowledge about animal social systems provide a powerful approach that can help close this gap. in this study: 1) we develop a general model for the emergence of social structures based on a key generative process of real animal social networks, namely social preferences for traits (such as the age, sex, etc. of social partners); 2) we use this model to investigate how different trait preferences affect social network structure and function. we find that the preferences used in a population can have far-reaching consequences for the population, via effects on the transmission of disease and information and the robustness of the social network against fragmentation when individuals disappear. the study thus shows that social preferences can have consequences that go far beyond direct benefits individuals gain from social partner selection. it also shows that these consequences depend both on the preference types, and on the types of traits they are used with. we discuss the implications of the results for social evolution.',\n",
       "  'categories': 'physics.soc-ph cs.si q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-14',\n",
       "  'updated': '2024-01-30',\n",
       "  'authors': ['josefine bohr brask',\n",
       "   'andreas koher',\n",
       "   'darren p. croft',\n",
       "   'sune lehmann'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.08107'},\n",
       " {'title': 'eco-evolutionary cyclic dominance among predators, prey, and parasites',\n",
       "  'id': '2303.08158',\n",
       "  'abstract': \"predator prey interactions are one of ecology's central research themes, but with many interdisciplinary implications across the social and natural sciences. here we consider an often-overlooked species in these interactions, namely parasites. we first show that a simple predator prey parasite model, inspired by the classical lotka volterra equations, fails to produce a stable coexistence of all three species, thus failing to provide a biologically realistic outcome. to improve this, we introduce free space as a relevant eco-evolutionary component in a new mathematical model that uses a game-theoretical payoff matrix to describe a more realistic setup. we then show that the consideration of free space stabilizes the dynamics by means of cyclic dominance that emerges between the three species. we determine the parameter regions of coexistence as well as the types of bifurcations leading to it by means of analytical derivations as well as by means of numerical simulations. we conclude that the consideration of free space as a finite resource reveals the limits of biodiversity in predator prey parasite interactions, and it may also help us in the determination of factors that promote a healthy biota.\",\n",
       "  'categories': 'q-bio.pe nlin.ao',\n",
       "  'doi': '10.1016/j.jtbi.2023.111446',\n",
       "  'created': '2023-03-14',\n",
       "  'updated': '',\n",
       "  'authors': ['sayantan nag chowdhury',\n",
       "   'jeet banerjee',\n",
       "   'matjaž perc',\n",
       "   'dibakar ghosh'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.08158'},\n",
       " {'title': 'the measurement of bovine pericardium density and its implications on   leaflet stress distribution in bioprosthetic heart valves',\n",
       "  'id': '2303.09094',\n",
       "  'abstract': 'purpose: bioprosthetic heart valves (bhvs) are currently in widespread use with promising outcomes. computational modeling provides a framework for quantitatively describing bhvs in the preclinical phase. to obtain reliable solutions in computational modeling, it is essential to consider accurate leaflet properties such as mechanical properties and density. bovine pericardium (bp) is widely used as bhv leaflets. previous computational studies assume bp density to be close to the density of water or blood. however, bp leaflets undergo multiple treatments such as fixation and anti-calcification. the present study aims to measure the density of the bp used in bhvs and determine its effect on leaflet stress distribution.   methods: we determined the density of eight square bp samples laser cut from edwards bp patches. the weight of specimens was measured using an a&d analytical balance, and volume was measured by high-resolution imaging. finite element models of a bhv similar to perimount magna were developed in abaqus.   results: the average density value of the bp samples was 1410 kg/m3. in the acceleration phase of a cardiac cycle, the maximum stress value reached 1.89 mpa for a density value of 1410 kg/m3 , and 2.47 mpa for a density of 1000 kg/m3(30.7% difference). in the deceleration, the maximum stress value reached 713 and 669 kpa, respectively.   conclusion: stress distribution and deformation of bhv leaflets are dependent upon the magnitude of density. ascertaining an accurate value for the density of bhv leaflets is essential for computational models.',\n",
       "  'categories': 'q-bio.to',\n",
       "  'doi': '10.1007/s13239-023-00692-0',\n",
       "  'created': '2023-03-16',\n",
       "  'updated': '',\n",
       "  'authors': ['masod sadipour', 'ali n. azadani'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.09094'},\n",
       " {'title': 'initiation of motility on a compliant substrate',\n",
       "  'id': '2303.09666',\n",
       "  'abstract': 'the conditions under which biological cells switch from a static to a motile state are fundamental to the understanding of many healthy and pathological processes. in this paper, we show that even in the presence of a fully symmetric protrusive activity at the cell edges, such a spontaneous transition can result solely from the mechanical interaction of the cell traction forces with an elastic substrate. the loss of symmetry of the traction forces leading to the cell propulsion is rooted in the fact that the surface loading follows the substrate deformation. the bifurcation between the static and motile states is characterized analytically and, considering the measurements performed on two cell types, we show that such an instability can realistically occur on soft in vivo substrates.',\n",
       "  'categories': 'physics.bio-ph cond-mat.soft q-bio.cb',\n",
       "  'doi': '10.1016/j.jmps.2023.105526',\n",
       "  'created': '2023-03-16',\n",
       "  'updated': '2023-12-21',\n",
       "  'authors': ['jocelyn étienne', 'pierre recho'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.09666'},\n",
       " {'title': 'predicting discrete-time bifurcations with deep learning',\n",
       "  'id': '2303.09669',\n",
       "  'abstract': 'many natural and man-made systems are prone to critical transitions -- abrupt and potentially devastating changes in dynamics. deep learning classifiers can provide an early warning signal (ews) for critical transitions by learning generic features of bifurcations (dynamical instabilities) from large simulated training data sets. so far, classifiers have only been trained to predict continuous-time bifurcations, ignoring rich dynamics unique to discrete-time bifurcations. here, we train a deep learning classifier to provide an ews for the five local discrete-time bifurcations of codimension-1. we test the classifier on simulation data from discrete-time models used in physiology, economics and ecology, as well as experimental data of spontaneously beating chick-heart aggregates that undergo a period-doubling bifurcation. the classifier outperforms commonly used ews under a wide range of noise intensities and rates of approach to the bifurcation. it also predicts the correct bifurcation in most cases, with particularly high accuracy for the period-doubling, neimark-sacker and fold bifurcations. deep learning as a tool for bifurcation prediction is still in its nascence and has the potential to transform the way we monitor systems for critical transitions.',\n",
       "  'categories': 'q-bio.qm cs.lg math.ds',\n",
       "  'doi': '10.1038/s41467-023-42020-z',\n",
       "  'created': '2023-03-16',\n",
       "  'updated': '2024-02-08',\n",
       "  'authors': ['thomas m. bury',\n",
       "   'daniel dylewsky',\n",
       "   'chris t. bauch',\n",
       "   'madhur anand',\n",
       "   'leon glass',\n",
       "   'alvin shrier',\n",
       "   'gil bub'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.09669'},\n",
       " {'title': 'image statistics predict the sensitivity of perceptual quality metrics',\n",
       "  'id': '2303.09874',\n",
       "  'abstract': 'previously, barlow and attneave hypothesised a link between biological vision and information maximisation. following shannon, information was defined using the probability of natural images. several physiological and psychophysical phenomena have been derived from principles like info-max, efficient coding, or optimal denoising. however, it remains unclear how this link is expressed in mathematical terms from image probability. classical derivations were subjected to strong assumptions on the probability models and on the behaviour of the sensors. moreover, the direct evaluation of the hypothesis was limited by the inability of classical image models to deliver accurate estimates of the probability. here, we directly evaluate image probabilities using a generative model for natural images, and analyse how probability-related factors can be combined to predict the sensitivity of state-of-the-art subjective image quality metrics, a proxy for human perception. we use information theory and regression analysis to find a simple model that when combining just two probability-related factors achieves 0.77 correlation with subjective metrics. this probability-based model is validated in two ways: through direct comparison with the opinion of real observers in a subjective quality experiment, and by reproducing basic trends of classical psychophysical facts such as the contrast sensitivity function, the weber-law, and contrast masking.',\n",
       "  'categories': 'cs.cv cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-17',\n",
       "  'updated': '2024-12-02',\n",
       "  'authors': ['alexander hepburn',\n",
       "   'valero laparra',\n",
       "   'raúl santos-rodriguez',\n",
       "   'jesús malo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.09874'},\n",
       " {'title': 'discovering mesoscopic descriptions of collective movement with neural   stochastic modelling',\n",
       "  'id': '2303.09906',\n",
       "  'abstract': \"collective motion is an ubiquitous phenomenon in nature, inspiring engineers, physicists and mathematicians to develop mathematical models and bio-inspired designs. collective motion at small to medium group sizes ($\\\\sim$10-1000 individuals, also called the `mesoscale'), can show nontrivial features due to stochasticity. therefore, characterizing both the deterministic and stochastic aspects of the dynamics is crucial in the study of mesoscale collective phenomena. here, we use a physics-inspired, neural-network based approach to characterize the stochastic group dynamics of interacting individuals, through a stochastic differential equation (sde) that governs the collective dynamics of the group. we apply this technique on both synthetic and real-world datasets, and identify the deterministic and stochastic aspects of the dynamics using drift and diffusion fields, enabling us to make novel inferences about the nature of order in these systems.\",\n",
       "  'categories': 'cs.lg physics.soc-ph q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-17',\n",
       "  'updated': '2024-01-18',\n",
       "  'authors': ['utkarsh pratiush',\n",
       "   'arshed nabeel',\n",
       "   'vishwesha guttal',\n",
       "   'prathosh ap'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.09906'},\n",
       " {'title': 'mobility restrictions in response to local epidemic outbreaks in   rock-paper-scissors models',\n",
       "  'id': '2303.10724',\n",
       "  'abstract': \"we study a three-species cyclic model whose organisms are vulnerable to contamination with an infectious disease which propagates person-to-person. we consider that individuals of one species perform an evolutionary self-preservation strategy by reducing the mobility rate to minimise infection risk whenever an epidemic outbreak reaches the neighbourhood. running stochastic simulations, we quantify the changes in spatial patterns induced by unevenness in the cyclic game introduced by the mobility restriction strategy of organisms of one out of the species. our findings show that variations in disease virulence impact the benefits of dispersal limitation reaction, with the relative reduction of the organisms' infection risk accentuating in surges of less contagious or deadlier diseases. the effectiveness of the mobility restriction tactic depends on the tolerable fraction of infected neighbours used as a trigger of the defensive strategy and the deceleration level. if each organism promptly reacts to the arrival of the first viral vectors in its surroundings with strict mobility reduction, contamination risk decreases significantly. our conclusions may help biologists understand the impact of evolutionary defensive strategies in ecosystems during an epidemic.\",\n",
       "  'categories': 'q-bio.pe nlin.ao nlin.ps physics.bio-ph q-bio.qm',\n",
       "  'doi': '10.1088/2632-072x/ad2d5b',\n",
       "  'created': '2023-03-19',\n",
       "  'updated': '',\n",
       "  'authors': ['j. menezes'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.10724'},\n",
       " {'title': 'an analysis of $\\\\mathbb{p}$-invariance and dynamical compensation   properties from a control perspective',\n",
       "  'id': '2303.10996',\n",
       "  'abstract': \"dynamical compensation (dc) provides robustness to parameter fluctuations. as an example, dc enable control of the functional mass of endocrine or neuronal tissue essential for controlling blood glucose by insulin through a nonlinear feedback loop. researchers have shown that dc is related to structural unidentifiability and $\\\\mathbb{p}$-invariance property, and $\\\\mathbb{p}$-invariance property is a sufficient and necessary condition for the dc property. in this article, we discuss dc and $\\\\mathbb{p}$-invariancy from an adaptive control perspective. an adaptive controller is a self-tuning controller used to compensate for changes in a dynamical system. to design an adaptive controller with the dc property, it is easier to start with a two-dimensional dynamical model. we introduce a simplified system of ordinary differential equations (odes) with the dc property and extend it to a general form. the value of the ideal adaptive control lies in developing methods to synthesize dc to variations in multiple parameters. then we investigate the stability of the system with time-varying input and disturbance signals, with a focus on the system's $\\\\mathbb{p}$-invariance properties. this study provides phase portraits and step-like response graphs to visualize the system's behavior and stability properties.\",\n",
       "  'categories': 'eess.sy cs.sy math.oc physics.bio-ph q-bio.qm',\n",
       "  'doi': '10.1186/s12859-024-05718-5',\n",
       "  'created': '2023-03-20',\n",
       "  'updated': '2023-11-09',\n",
       "  'authors': ['akram ashyani',\n",
       "   'yu-heng wu',\n",
       "   'huan-wei hsu',\n",
       "   'torbjörn e. m. nordling'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.10996'},\n",
       " {'title': 'an agent-based modelling framework to study growth mechanisms in   egfr-l858r mutant alveolar type ii cells',\n",
       "  'id': '2303.11342',\n",
       "  'abstract': \"mutations in the epidermal growth factor receptor (egfr) are common in non-small cell lung cancer (nsclc), particularly in never-smoker patients. however, these mutations are not always carcinogenic, and have recently been reported in histologically normal lung tissue from patients with and without lung cancer. to investigate the outcome of egfr mutation in healthy lung stem cells, we grew murine alveolar type-ii organoids monoclonally in a 3d matrigel. our experiments showed that the \\\\textit{egfr-l858r} mutation induced a change in organoid structure: mutated organoids displayed more `budding', in comparison to non-mutant controls, which were nearly spherical. we perform on-lattice computational simulations, which suggest that this can be explained by the concentration of division amongst a small number of cells on the surface of the organoid, which may arise from several possible biological mechanisms. these results suggest that the l858r mutation produces structures which expand quickly from surface protrusions. we are currently unable to distinguish the cell-based mechanisms that lead to this spatial heterogeneity in growth, but suggest a number of future experiments which could be used to do so. we suggest that the likelihood of l858r-fuelled tumorigenesis is affected not just by random fluctuations in cell fitness, but by whether the mutation arises in a spatial environment that allows mutant cells to reproduce without being forced to encounter each other. these data may have implications for cancer prevention strategies and for understanding nsclc progression.\",\n",
       "  'categories': 'q-bio.cb q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-20',\n",
       "  'updated': '2024-03-11',\n",
       "  'authors': ['helena coggan',\n",
       "   'clare e. weeden',\n",
       "   'philip pearce',\n",
       "   'mohit p. dalwadi',\n",
       "   'alastair magness',\n",
       "   'charles swanton',\n",
       "   'karen m. page'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.11342'},\n",
       " {'title': 'explosive cooperation in social dilemmas on higher-order networks',\n",
       "  'id': '2303.11475',\n",
       "  'abstract': 'understanding how cooperative behaviours can emerge from competitive interactions is an open problem in biology and social sciences. while interactions are usually modelled as pairwise networks, the units of many real-world systems can also interact in groups of three or more. here, we introduce a general framework to extend pairwise games to higher-order networks. by studying social dilemmas on hypergraphs with a tunable structure, we find an explosive transition to cooperation triggered by a critical number of higher-order games. the associated bistable regime implies that an initial critical mass of cooperators is also required for the emergence of prosocial behavior. our results show that higher-order interactions provide a novel explanation for the survival of cooperation.',\n",
       "  'categories': 'physics.soc-ph cs.gt cs.si math.ds q-bio.pe',\n",
       "  'doi': '10.1103/physrevlett.132.167401',\n",
       "  'created': '2023-03-20',\n",
       "  'updated': '',\n",
       "  'authors': ['andrea civilini',\n",
       "   'onkar sadekar',\n",
       "   'federico battiston',\n",
       "   'jesús gómez-gardeñes',\n",
       "   'vito latora'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.11475'},\n",
       " {'title': 'difficulty in chirality recognition for transformer architectures   learning chemical structures from string',\n",
       "  'id': '2303.11593',\n",
       "  'abstract': 'recent years have seen rapid development of descriptor generation based on representation learning of extremely diverse molecules, especially those that apply natural language processing (nlp) models to smiles, a literal representation of molecular structure. however, little research has been done on how these models understand chemical structure. to address this black box, we investigated the relationship between the learning progress of smiles and chemical structure using a representative nlp model, the transformer. we show that while the transformer learns partial structures of molecules quickly, it requires extended training to understand overall structures. consistently, the accuracy of molecular property predictions using descriptors generated from models at different learning steps was similar from the beginning to the end of training. furthermore, we found that the transformer requires particularly long training to learn chirality and sometimes stagnates with low performance due to misunderstanding of enantiomers. these findings are expected to deepen the understanding of nlp models in chemistry.',\n",
       "  'categories': 'cs.lg cs.cl physics.chem-ph q-bio.bm',\n",
       "  'doi': '10.1038/s41467-024-45102-8',\n",
       "  'created': '2023-03-21',\n",
       "  'updated': '2024-01-13',\n",
       "  'authors': ['yasuhiro yoshikai',\n",
       "   'tadahaya mizuno',\n",
       "   'shumpei nemoto',\n",
       "   'hiroyuki kusuhara'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.11593'},\n",
       " {'title': 'ccpl: cross-modal contrastive protein learning',\n",
       "  'id': '2303.11783',\n",
       "  'abstract': \"effective protein representation learning is crucial for predicting protein functions. traditional methods often pretrain protein language models on large, unlabeled amino acid sequences, followed by finetuning on labeled data. while effective, these methods underutilize the potential of protein structures, which are vital for function determination. common structural representation techniques rely heavily on annotated data, limiting their generalizability. moreover, structural pretraining methods, similar to natural language pretraining, can distort actual protein structures. in this work, we introduce a novel unsupervised protein structure representation pretraining method, cross-modal contrastive protein learning (ccpl). ccpl leverages a robust protein language model and uses unsupervised contrastive alignment to enhance structure learning, incorporating self-supervised structural constraints to maintain intrinsic structural information. we evaluated our model across various benchmarks, demonstrating the framework's superiority.\",\n",
       "  'categories': 'q-bio.bm cs.ai cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-19',\n",
       "  'updated': '2024-09-03',\n",
       "  'authors': ['jiangbin zheng', 'stan z. li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.11783'},\n",
       " {'title': 'materials discovery with extreme properties via reinforcement   learning-guided combinatorial chemistry',\n",
       "  'id': '2303.11833',\n",
       "  'abstract': 'the goal of most materials discovery is to discover materials that are superior to those currently known. fundamentally, this is close to extrapolation, which is a weak point for most machine learning models that learn the probability distribution of data. herein, we develop reinforcement learning-guided combinatorial chemistry, which is a rule-based molecular designer driven by trained policy for selecting subsequent molecular fragments to get a target molecule. since our model has the potential to generate all possible molecular structures that can be obtained from combinations of molecular fragments, unknown molecules with superior properties can be discovered. we theoretically and empirically demonstrate that our model is more suitable for discovering better compounds than probability distribution-learning models. in an experiment aimed at discovering molecules that hit seven extreme target properties, our model discovered 1,315 of all target-hitting molecules and 7,629 of five target-hitting molecules out of 100,000 trials, whereas the probability distribution-learning models failed. moreover, it has been confirmed that every molecule generated under the binding rules of molecular fragments is 100% chemically valid. to illustrate the performance in actual problems, we also demonstrate that our models work well on two practical applications: discovering protein docking molecules and hiv inhibitors.',\n",
       "  'categories': 'q-bio.bm cs.lg',\n",
       "  'doi': '10.1039/d3sc05281h',\n",
       "  'created': '2023-03-21',\n",
       "  'updated': '2024-05-07',\n",
       "  'authors': ['hyunseung kim',\n",
       "   'haeyeon choi',\n",
       "   'dongju kang',\n",
       "   'won bo lee',\n",
       "   'jonggeol na'],\n",
       "  'affiliation': ['seoul national university',\n",
       "   'ewha womans university',\n",
       "   'seoul national university',\n",
       "   'seoul national university',\n",
       "   'ewha womans university'],\n",
       "  'url': 'https://arxiv.org/abs/2303.11833'},\n",
       " {'title': 'the multiscale surface vision transformer',\n",
       "  'id': '2303.11909',\n",
       "  'abstract': 'surface meshes are a favoured domain for representing structural and functional information on the human cortex, but their complex topology and geometry pose significant challenges for deep learning analysis. while transformers have excelled as domain-agnostic architectures for sequence-to-sequence learning, the quadratic cost of the self-attention operation remains an obstacle for many dense prediction tasks. inspired by some of the latest advances in hierarchical modelling with vision transformers, we introduce the multiscale surface vision transformer (ms-sit) as a backbone architecture for surface deep learning. the self-attention mechanism is applied within local-mesh-windows to allow for high-resolution sampling of the underlying data, while a shifted-window strategy improves the sharing of information between windows. neighbouring patches are successively merged, allowing the ms-sit to learn hierarchical representations suitable for any prediction task. results demonstrate that the ms-sit outperforms existing surface deep learning methods for neonatal phenotyping prediction tasks using the developing human connectome project (dhcp) dataset. furthermore, building the ms-sit backbone into a u-shaped architecture for surface segmentation demonstrates competitive results on cortical parcellation using the uk biobank (ukb) and manually-annotated mindboggle datasets. code and trained models are publicly available at https://github.com/metrics-lab/surface-vision-transformers.',\n",
       "  'categories': 'eess.iv cs.cv q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-21',\n",
       "  'updated': '2024-06-11',\n",
       "  'authors': ['simon dahan',\n",
       "   'logan z. j. williams',\n",
       "   'daniel rueckert',\n",
       "   'emma c. robinson'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.11909'},\n",
       " {'title': 'informational rescaling of pca maps with application to genetic distance',\n",
       "  'id': '2303.12654',\n",
       "  'abstract': \"we discuss the inadequacy of covariances/correlations and other measures in l2 as relative distance metrics under some conditions. we propose a computationally simple heuristic to transform a map based on standard principal component analysis (pca) (when the variables are asymptotically gaussian) into an entropy-based map where distances are based on mutual information (mi). rescaling principal component based distances using mi allows a representation of relative statistical associations when, as in genetics, it is applied on bit measurements between individuals' genomic mutual information.   this entropy rescaled pca, while preserving order relationships (along a dimension), changes the relative distances to make them linear to information. we show the effect on the entire world population and some subsamples, which leads to significant differences with the results of current research.\",\n",
       "  'categories': 'cs.it math.it q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-14',\n",
       "  'updated': '2024-03-04',\n",
       "  'authors': ['nassim nicholas taleb',\n",
       "   'pierre zalloua',\n",
       "   'khaled elbassioni',\n",
       "   'andreas henschel',\n",
       "   'daniel e. platt'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.12654'},\n",
       " {'title': 'dermatologist-like explainable ai enhances trust and confidence in   diagnosing melanoma',\n",
       "  'id': '2303.12806',\n",
       "  'abstract': \"although artificial intelligence (ai) systems have been shown to improve the accuracy of initial melanoma diagnosis, the lack of transparency in how these systems identify melanoma poses severe obstacles to user acceptance. explainable artificial intelligence (xai) methods can help to increase transparency, but most xai methods are unable to produce precisely located domain-specific explanations, making the explanations difficult to interpret. moreover, the impact of xai methods on dermatologists has not yet been evaluated. extending on two existing classifiers, we developed an xai system that produces text and region based explanations that are easily interpretable by dermatologists alongside its differential diagnoses of melanomas and nevi. to evaluate this system, we conducted a three-part reader study to assess its impact on clinicians' diagnostic accuracy, confidence, and trust in the xai-support. we showed that our xai's explanations were highly aligned with clinicians' explanations and that both the clinicians' trust in the support system and their confidence in their diagnoses were significantly increased when using our xai compared to using a conventional ai system. the clinicians' diagnostic accuracy was numerically, albeit not significantly, increased. this work demonstrates that clinicians are willing to adopt such an xai system, motivating their future use in the clinic.\",\n",
       "  'categories': 'q-bio.qm cs.cv cs.lg eess.iv',\n",
       "  'doi': '10.1038/s41467-023-43095-4',\n",
       "  'created': '2023-03-17',\n",
       "  'updated': '',\n",
       "  'authors': ['tirtha chanda',\n",
       "   'katja hauser',\n",
       "   'sarah hobelsberger',\n",
       "   'tabea-clara bucher',\n",
       "   'carina nogueira garcia',\n",
       "   'christoph wies',\n",
       "   'harald kittler',\n",
       "   'philipp tschandl',\n",
       "   'cristian navarrete-dechent',\n",
       "   'sebastian podlipnik',\n",
       "   'emmanouil chousakos',\n",
       "   'iva crnaric',\n",
       "   'jovana majstorovic',\n",
       "   'linda alhajwan',\n",
       "   'tanya foreman',\n",
       "   'sandra peternel',\n",
       "   'sergei sarap',\n",
       "   'i̇rem özdemir',\n",
       "   'raymond l. barnhill',\n",
       "   'mar llamas velasco',\n",
       "   'gabriela poch',\n",
       "   'sören korsing',\n",
       "   'wiebke sondermann',\n",
       "   'frank friedrich gellrich',\n",
       "   'markus v. heppt',\n",
       "   'michael erdmann',\n",
       "   'sebastian haferkamp',\n",
       "   'konstantin drexler',\n",
       "   'matthias goebeler',\n",
       "   'bastian schilling',\n",
       "   'jochen s. utikal',\n",
       "   'kamran ghoreschi',\n",
       "   'stefan fröhling',\n",
       "   'eva krieghoff-henning',\n",
       "   'titus j. brinker'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.12806'},\n",
       " {'title': 'clinically relevant latent space embedding of cancer histopathology   slides through variational autoencoder based image compression',\n",
       "  'id': '2303.13332',\n",
       "  'abstract': 'in this paper, we introduce a variational autoencoder (vae) based training approach that can compress and decompress cancer pathology slides at a compression ratio of 1:512, which is better than the previously reported state of the art (sota) in the literature, while still maintaining accuracy in clinical validation tasks. the compression approach was tested on more common computer vision datasets such as cifar10, and we explore which image characteristics enable this compression ratio on cancer imaging data but not generic images. we generate and visualize embeddings from the compressed latent space and demonstrate how they are useful for clinical interpretation of data, and how in the future such latent embeddings can be used to accelerate search of clinical imaging data.',\n",
       "  'categories': 'eess.iv cs.cv q-bio.qm',\n",
       "  'doi': '10.1109/isbi53787.2023.10230343',\n",
       "  'created': '2023-03-23',\n",
       "  'updated': '',\n",
       "  'authors': ['mohammad sadegh nasr',\n",
       "   'amir hajighasemi',\n",
       "   'paul koomey',\n",
       "   'parisa boodaghi malidarreh',\n",
       "   'michael robben',\n",
       "   'jillur rahman saurav',\n",
       "   'helen h. shang',\n",
       "   'manfred huber',\n",
       "   'jacob m. luber'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.13332'},\n",
       " {'title': 'many bioinformatics programming tasks can be automated with chatgpt',\n",
       "  'id': '2303.13528',\n",
       "  'abstract': \"computer programming is a fundamental tool for life scientists, allowing them to carry out many essential research tasks. however, despite a variety of educational efforts, learning to write code can be a challenging endeavor for both researchers and students in life science disciplines. recent advances in artificial intelligence have made it possible to translate human-language prompts to functional code, raising questions about whether these technologies can aid (or replace) life scientists' efforts to write code. using 184 programming exercises from an introductory-bioinformatics course, we evaluated the extent to which one such model -- openai's chatgpt -- can successfully complete basic- to moderate-level programming tasks. on its first attempt, chatgpt solved 139 (75.5%) of the exercises. for the remaining exercises, we provided natural-language feedback to the model, prompting it to try different approaches. within 7 or fewer attempts, chatgpt solved 179 (97.3%) of the exercises. these findings have important implications for life-sciences research and education. for many programming tasks, researchers no longer need to write code from scratch. instead, machine-learning models may produce usable solutions. instructors may need to adapt their pedagogical approaches and assessment techniques to account for these new capabilities that are available to the general public.\",\n",
       "  'categories': 'q-bio.ot cs.cy',\n",
       "  'doi': '10.1371/journal.pcbi.1011511',\n",
       "  'created': '2023-03-07',\n",
       "  'updated': '',\n",
       "  'authors': ['stephen r. piccolo',\n",
       "   'paul denny',\n",
       "   'andrew luxton-reilly',\n",
       "   'samuel payne',\n",
       "   'perry g. ridge'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.13528'},\n",
       " {'title': 'a biological sequence comparison algorithm using quantum computers',\n",
       "  'id': '2303.13608',\n",
       "  'abstract': 'genetic information is encoded in a linear sequence of nucleotides, represented by letters ranging from thousands to billions. mutations refer to changes in the dna or rna nucleotide sequence. thus, mutation detection is vital in all areas of biology and medicine. careful monitoring of virulence-enhancing mutations is essential. however, an enormous amount of classical computing power is required to analyze genetic sequences of this size. inspired by human perception of vision and pixel representation of images on quantum computers, we leverage these techniques to implement a pairwise sequence analysis. the methodology has a potential advantage over classical approaches and can be further applied to identify mutations and other modifications in genetic sequences. we present a method to display and analyze the similarity between two genome sequences on a quantum computer where a similarity score is calculated to determine the similarity between nucleotides.',\n",
       "  'categories': 'quant-ph q-bio.gn',\n",
       "  'doi': '10.1038/s41598-023-41086-5',\n",
       "  'created': '2023-03-23',\n",
       "  'updated': '2023-07-20',\n",
       "  'authors': ['büsra kösoglu-kind',\n",
       "   'robert loredo',\n",
       "   'michele grossi',\n",
       "   'christian bernecker',\n",
       "   'jody m burks',\n",
       "   'rudiger buchkremer'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.13608'},\n",
       " {'title': 'differential co-abundance network analyses for microbiome data adjusted   for clinical covariates using jackknife pseudo-values',\n",
       "  'id': '2303.13702',\n",
       "  'abstract': 'a recent breakthrough in differential network (dn) analysis of microbiome data has been realized with the advent of next-generation sequencing technologies. the dn analysis disentangles the microbial co-abundance among taxa by comparing the network properties between two or more graphs under different biological conditions. however, the existing methods to the dn analysis for microbiome data do not adjust for other clinical differences between subjects. we propose a statistical approach via pseudo-value information and estimation for differential network analysis (sohpie-dna) that incorporates additional covariates such as continuous age and categorical bmi. sohpie-dna is a regression technique adopting jackknife pseudo-values that can be implemented readily for the analysis. we demonstrate through simulations that sohpie-dna consistently reaches higher recall and f1-score, while maintaining similar precision and accuracy to existing methods (netcomi and mdine). lastly, we apply sohpie-dna on two real datasets from the american gut project and the diet exchange study to showcase the utility. the analysis of the diet exchange study is to showcase that sohpie-dna can also be used to incorporate the temporal change of connectivity of taxa with the inclusion of additional covariates. as a result, our method has found taxa that are related to the prevention of intestinal inflammation and severity of fatigue in advanced metastatic cancer patients.',\n",
       "  'categories': 'stat.me q-bio.qm stat.ap',\n",
       "  'doi': '10.1186/s12859-024-05689-7',\n",
       "  'created': '2023-03-23',\n",
       "  'updated': '',\n",
       "  'authors': ['seungjun ahn', 'somnath datta'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.13702'},\n",
       " {'title': 'patch formation driven by stochastic effects of interaction between   viruses and defective interfering particles',\n",
       "  'id': '2303.13848',\n",
       "  'abstract': 'defective interfering particles (dips) are virus-like particles that occur naturally during virus infections. these particles are defective, lacking essential genetic materials for replication, but they can interact with the wild-type virus and potentially be used as therapeutic agents. however, the effect of dips on infection spread is still unclear due to complicated stochastic effects and nonlinear spatial dynamics. in this work, we develop a model with a new hybrid method to study the spatial-temporal dynamics of viruses and dips co-infections within hosts. we present two different scenarios of virus production and compare the results from deterministic and stochastic models to demonstrate how the stochastic effect is involved in the spatial dynamics of virus transmission. we quantitatively study the spread features of the virus, including the formation and the speed of virus spread and the emergence of stochastic patchy patterns of virus distribution. our simulations simultaneously capture observed spatial spread features in the experimental data, including the spread rate of the virus and its patchiness. the results demonstrate that dips can slow down the growth of virus particles and make the spread of the virus more patchy.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.1371/journal.pcbi.1011513',\n",
       "  'created': '2023-03-24',\n",
       "  'updated': '',\n",
       "  'authors': ['qiantong liang',\n",
       "   'johnny yang',\n",
       "   'wai-tong louis fan',\n",
       "   'wing-cheong lo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.13848'},\n",
       " {'title': 'overflow metabolism originates from growth optimization and cell   heterogeneity',\n",
       "  'id': '2303.14159',\n",
       "  'abstract': 'a classic problem in metabolism is that fast-proliferating cells use seemingly wasteful fermentation for energy biogenesis in the presence of sufficient oxygen. this counterintuitive phenomenon, known as overflow metabolism or the warburg effect, is universal across various organisms. despite extensive research, its origin and function remain unclear. here, we show that overflow metabolism can be understood through growth optimization combined with cell heterogeneity. a model of optimal protein allocation, coupled with heterogeneity in enzyme catalytic rates among cells, quantitatively explains why and how cells choose between respiration and fermentation under different nutrient conditions. our model quantitatively illustrates the growth rate dependence of fermentation flux and enzyme allocation under various perturbations and is fully validated by experimental results in escherichia coli. our work provides a quantitative explanation for the crabtree effect in yeast and the warburg effect in cancer cells and can be broadly used to address heterogeneity-related challenges in metabolism.',\n",
       "  'categories': 'physics.bio-ph cond-mat.stat-mech q-bio.cb',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-24',\n",
       "  'updated': '2024-10-14',\n",
       "  'authors': ['xin wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.14159'},\n",
       " {'title': 'reference standard analysis of multiple new and old plasma clearance   models and renal clearance with special attention to measurement of reduced   glomerular filtration rate',\n",
       "  'id': '2303.14995',\n",
       "  'abstract': 'nine models were evaluated as candidate glomerular filtration rate (gfr) reference standards in three datasets using [$^{51}$cr(edta)]$^-$ or [$^{169}$yb(dtpa)]$^{2-}$ anions in 98 studies. noncompartmental methods formed an upper limit for estimating mass excreted and voluntary urine collection formed a lower limit. for current models and methods, reduced gfr in adults resulted in inflated clearance estimates. two different logarithmic models with exponential tails were created and may have underestimated reduced clearance. the logarithmic formulae can be used with only two plasma samples, and fit 13 studies totalling 162 plasma samples drawn from 5 min to 24 h with an 8% standard deviation of residuals compared to 20% error for monoexponentials. for shorter times (4 or 5 h) the fit errors decreased but the ratio of errors remained at circa 2.5 times lesser for the logarithmic versus monoexponential models. adaptively regularised gamma variate, tk-gv, models that are well documented, but not in common use, were largely contained within the reference extreme values, were unbiased for different levels of clearance and were the only models to be uncorrelated to volume of distribution from mean residence time divided by weight. using tk-gv as a candidate reference standard, potentially better methods for routine clinical usage were discussed. prospective clinical testing, and metabolic scaling of decreased renal function is advised for potential changes to patient triage.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.1080/00365513.2023.2275278',\n",
       "  'created': '2023-03-27',\n",
       "  'updated': '2023-11-02',\n",
       "  'authors': ['carl a. wesolowski'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.14995'},\n",
       " {'title': 'interpretable machine learning of amino acid patterns in proteins: a   statistical ensemble approach',\n",
       "  'id': '2303.15228',\n",
       "  'abstract': 'explainable and interpretable unsupervised machine learning helps understand the underlying structure of data. we introduce an ensemble analysis of machine learning models to consolidate their interpretation. its application shows that restricted boltzmann machines compress consistently into a few bits the information stored in a sequence of five amino acids at the start or end of $\\\\alpha$-helices or $\\\\beta$-sheets. the weights learned by the machines reveal unexpected properties of the amino acids and the secondary structure of proteins: (i) his and thr have a negligible contribution to the amphiphilic pattern of $\\\\alpha$-helices; (ii) there is a class of $\\\\alpha$-helices particularly rich in ala at their end; (iii) pro occupies most often slots otherwise occupied by polar or charged amino acids, and its presence at the start of helices is relevant; (iv) glu and especially asp on one side, and val, leu, iso, and phe on the other, display the strongest tendency to mark amphiphilic patterns, i.e., extreme values of an \"effective hydrophobicity\", though they are not the most powerful (non) hydrophobic amino acids.',\n",
       "  'categories': 'q-bio.bm cond-mat.stat-mech cs.lg',\n",
       "  'doi': '10.1021/acs.jctc.3c00383',\n",
       "  'created': '2023-03-27',\n",
       "  'updated': '',\n",
       "  'authors': ['anna braghetto', 'enzo orlandini', 'marco baiesi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.15228'},\n",
       " {'title': 'modeling and forecasting age-specific drug overdose mortality in the   united states',\n",
       "  'id': '2303.16172',\n",
       "  'abstract': 'drug overdose deaths continue to increase in the united states for all major drug categories. over the past two decades the total number of overdose fatalities has increased more than five-fold; since 2013 the surge in overdose rates is primarily driven by fentanyl and methamphetamines. different drug categories and factors such as age, gender, and ethnicity are associated with different overdose mortality characteristics that may also change in time. for example, the average age at death from a drug overdose has decreased from 1940 to 1990 while the overall mortality rate has steadily increased. to provide insight into the population-level dynamics of drug-overdose mortality, we develop an age-structured model for drug addiction. using an augmented ensemble kalman filter (enkf), we show through a simple example how our model can be combined with synthetic observation data to estimate mortality rate and an age-distribution parameter. finally, we use an enkf to combine our model with observation data on overdose fatalities in the united states from 1999 to 2020 to forecast the evolution of overdose trends and estimate model parameters.',\n",
       "  'categories': 'stat.ap q-bio.pe q-bio.qm',\n",
       "  'doi': '10.1140/epjs/s11734-023-00801-z',\n",
       "  'created': '2023-03-28',\n",
       "  'updated': '',\n",
       "  'authors': ['lucas böttcher', 'tom chou', \"maria r. d'orsogna\"],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.16172'},\n",
       " {'title': 'neural spikes as rare events',\n",
       "  'id': '2303.16829',\n",
       "  'abstract': 'we consider the information transmission problem in neurons and its possible implications for learning in neural networks. our approach is based on recent developments in statistical physics and complexity science. we use branching process simulations to model neural activity. this would support temporal coding theory as a model for neural coding.',\n",
       "  'categories': 'q-bio.nc cond-mat.stat-mech',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-29',\n",
       "  'updated': '2024-08-21',\n",
       "  'authors': ['siddharth kackar'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.16829'},\n",
       " {'title': 'utilizing reinforcement learning for de novo drug design',\n",
       "  'id': '2303.17615',\n",
       "  'abstract': 'deep learning-based approaches for generating novel drug molecules with specific properties have gained a lot of interest in the last few years. recent studies have demonstrated promising performance for string-based generation of novel molecules utilizing reinforcement learning. in this paper, we develop a unified framework for using reinforcement learning for de novo drug design, wherein we systematically study various on- and off-policy reinforcement learning algorithms and replay buffers to learn an rnn-based policy to generate novel molecules predicted to be active against the dopamine receptor drd2. our findings suggest that it is advantageous to use at least both top-scoring and low-scoring molecules for updating the policy when structural diversity is essential. using all generated molecules at an iteration seems to enhance performance stability for on-policy algorithms. in addition, when replaying high, intermediate, and low-scoring molecules, off-policy algorithms display the potential of improving the structural diversity and number of active molecules generated, but possibly at the cost of a longer exploration phase. our work provides an open-source framework enabling researchers to investigate various reinforcement learning methods for de novo drug design.',\n",
       "  'categories': 'q-bio.bm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-03-30',\n",
       "  'updated': '2024-01-30',\n",
       "  'authors': ['hampus gummesson svensson',\n",
       "   'christian tyrchan',\n",
       "   'ola engkvist',\n",
       "   'morteza haghir chehreghani'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2303.17615'},\n",
       " {'title': 'hypergraph animals',\n",
       "  'id': '2304.00841',\n",
       "  'abstract': 'here we introduce simple structures for the analysis of complex hypergraphs, hypergraph animals. these structures are designed to describe the local node neighbourhoods of nodes in hypergraphs. we establish their relationships to lattice animals and network motifs, and we develop their combinatorial properties for sparse and uncorrelated hypergraphs. we make use of the tight link of hypergraph animals to partition numbers, which opens up a vast mathematical framework for the analysis of hypergraph animals. we then study their abundances in random hypergraphs. two transferable insights result from this analysis: (i) it establishes the importance of high-cardinality edges in ensembles of random hypergraphs that are inspired by the classical erd\\\\\"os-reny\\\\\\'i random graphs; and (ii) there is a close connection between degree and hyperedge cardinality in random hypergraphs that shapes animal abundances and spectra profoundly. both findings imply that hypergraph animals can have the potential to affect information flow and processing in complex systems. our analysis of also suggests that we need to spend more effort on investigating and developing suitable conditional ensembles of random hypergraphs that can capture real-world structures and their complex dependency structures.',\n",
       "  'categories': 'q-bio.mn cond-mat.dis-nn math.co',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-03',\n",
       "  'updated': '2024-07-03',\n",
       "  'authors': ['michael p. h. stumpf'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.00841'},\n",
       " {'title': 'faket: simulating cryo-electron tomograms with neural style transfer',\n",
       "  'id': '2304.02011',\n",
       "  'abstract': 'in cryo-electron microscopy, accurate particle localization and classification are imperative. recent deep learning solutions, though successful, require extensive training data sets. the protracted generation time of physics-based models, often employed to produce these data sets, limits their broad applicability. we introduce faket, a method based on neural style transfer, capable of simulating the forward operator of any cryo transmission electron microscope. it can be used to adapt a synthetic training data set according to reference data producing high-quality simulated micrographs or tilt-series. to assess the quality of our generated data, we used it to train a state-of-the-art localization and classification architecture and compared its performance with a counterpart trained on benchmark data. remarkably, our technique matches the performance, boosts data generation speed 750 times, uses 33 times less memory, and scales well to typical transmission electron microscope detector sizes. it leverages gpu acceleration and parallel processing. the source code is available at https://github.com/paloha/faket.',\n",
       "  'categories': 'cs.lg eess.iv q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-04',\n",
       "  'updated': '2024-07-05',\n",
       "  'authors': ['pavol harar',\n",
       "   'lukas herrmann',\n",
       "   'philipp grohs',\n",
       "   'david haselbach'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.02011'},\n",
       " {'title': 'generic consistency and nondegeneracy of vertically parametrized systems',\n",
       "  'id': '2304.02302',\n",
       "  'abstract': 'we determine the generic consistency, dimension and nondegeneracy of the zero locus over $\\\\mathbb{c}^*$, $\\\\mathbb{r}^*$ and $\\\\mathbb{r}_{>0}$ of vertically parametrized systems: parametric polynomial systems consisting of linear combinations of monomials scaled by free parameters. these systems generalize sparse systems with fixed monomial support and freely varying parametric coefficients. as our main result, we establish the equivalence among three key properties: the existence of nondegenerate zeros, the zero set having generically the expected dimension, and the system being generically consistent. importantly, we prove that checking whether a vertically parametrized system has these properties amounts to an easily computed matrix rank condition.',\n",
       "  'categories': 'math.ag q-bio.mn q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-05',\n",
       "  'updated': '2024-08-16',\n",
       "  'authors': ['elisenda feliu',\n",
       "   'oskar henriksson',\n",
       "   'beatriz pascual-escudero'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.02302'},\n",
       " {'title': 'a two-phase model of early atherosclerotic plaque development with ldl   toxicity effects',\n",
       "  'id': '2304.02587',\n",
       "  'abstract': 'atherosclerosis is a chronic inflammatory cardiovascular disease in which fatty plaque is built inside an artery wall. early atherosclerotic plaque development is typically characterised by inflammatory tissues primarily consisting of foam cells and macrophages. we present a multiphase model that explores early plaque growth to emphasise the role of cytokines (in particular, monocyte chemoattractant protein-1) and oxidised low-density lipoprotein (oxldl) in monocyte recruitment and foam cell production, respectively. the plaque boundary is assumed to move at the same speed as the inflammatory tissues close to the periphery. this study discusses oxldl-induced toxic environment towards macrophages and foam cells such that they start to die beyond a threshold limit of oxldl concentration owing to excessive consumption. our findings reveal that initially, the plaque evolves rapidly, and the growth rate subsequently reduces because of oxldl-induced toxicity. in addition, it is observed that the saturation of inflammatory cell volume fraction becomes independent of oxldl concentration beyond the threshold limit, referred to as the oxldl toxicity limit. our study manifests that increased oxldl flux promotes flattened plaque growth through oxldl-induced toxicity, while elevated cytokines flux disrupts the corresponding flattened behaviour. detailed analysis of the model presented in this article unfolds critical insights into the various biochemical and cellular mechanisms behind early plaque development.',\n",
       "  'categories': 'q-bio.to physics.flu-dyn',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-05',\n",
       "  'updated': '2024-04-29',\n",
       "  'authors': ['abdush salam pramanik', 'bibaswan dey', 'g. p. raja sekhar'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.02587'},\n",
       " {'title': 'the expected sample allele frequencies from populations of changing size   via orthogonal polynomials',\n",
       "  'id': '2304.02593',\n",
       "  'abstract': 'in this article, discrete and stochastic changes in (effective) population size are incorporated into the spectral representation of a biallelic diffusion process for drift and small mutation rates. a forward algorithm inspired by hidden-markov-model (hmm) literature is used to compute exact sample allele frequency spectra for three demographic scenarios: single changes in (effective) population size, boom-bust dynamics, and stochastic fluctuations in (effective) population size. an approach for fully agnostic demographic inference from these sample allele spectra is explored, and sufficient statistics for step-wise changes in population size are found. further, convergence behaviours of the polymorphic sample spectra for population size changes on different time scales are examined and discussed within the context of inference of the effective population size. joint visual assessment of the sample spectra and the temporal coefficients of the spectral decomposition of the forward diffusion process is found to be important in determining departure from equilibrium. stochastic changes in (effective) population size are shown to shape sample spectra particularly strongly.',\n",
       "  'categories': 'q-bio.pe stat.ap',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-05',\n",
       "  'updated': '2024-03-08',\n",
       "  'authors': ['lynette caitlin mikula', 'claus vogl'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.02593'},\n",
       " {'title': 'bifunctional enzyme provides absolute concentration robustness in   multisite covalent modification networks',\n",
       "  'id': '2304.02679',\n",
       "  'abstract': 'biochemical covalent modification networks exhibit a remarkable suite of steady state and dynamical properties such as multistationarity, oscillations, ultrasensitivity and absolute concentration robustness. this paper focuses on conditions required for a network to have a species with absolute concentration robustness. we find that the robustness in a substrate is endowed by its interaction with a bifunctional enzyme, which is an enzyme that has different roles when isolated versus when bound as a substrate-enzyme complex. when isolated, the bifunctional enzyme promotes production of more molecules of the robust species while when bound, the same enzyme facilitates degradation of the robust species. these dual actions produce robustness in the large class of covalent modification networks. for each network of this type, we find the network conditions for the presence of robustness, the species that has robustness, and its robustness value. the unified approach of simultaneously analyzing a large class of networks for a single property, i.e. absolute concentration robustness, reveals the underlying mechanism of the action of bifunctional enzyme while simultaneously providing a precise mathematical description of bifunctionality.',\n",
       "  'categories': 'q-bio.mn math.ds',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-05',\n",
       "  'updated': '2024-02-27',\n",
       "  'authors': ['badal joshi', 'tung d. nguyen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.02679'},\n",
       " {'title': 'nonequilibrium antigen recognition during infections and vaccinations',\n",
       "  'id': '2304.02794',\n",
       "  'abstract': 'the immune response to an acute primary infection is a coupled process of antigen proliferation, molecular recognition by naive b cells, and their subsequent proliferation and antibody shedding. this process contains a fundamental problem: the recognition of an exponentially time-dependent antigen signal. here we show that b cells can efficiently recognise new antigens by a tuned kinetic proofreading mechanism, where the molecular recognition machinery is adapted to the complexity of the immune repertoire. this process produces potent, specific and fast recognition of antigens, maintaining a spectrum of genetically distinct b cell lineages as input for affinity maturation. we show that the proliferation-recognition dynamics of a primary infection is a generalised luria-delbr\\\\\"uck process, akin to the dynamics of the classic fluctuation experiment. this map establishes a link between signal recognition dynamics and evolution. we derive the resulting statistics of the activated immune repertoire: antigen binding affinity, expected size, and frequency of active b cell clones are related by power laws, which define the class of generalised luria-delbr\\\\\"uck processes. their exponents depend on the antigen and b cell proliferation rate, the number of proofreading steps, and the lineage density of the naive repertoire. we extend the model to include spatio-temporal processes, including the diffusion-recognition dynamics of a vaccination. empirical data of activated mouse immune repertoires are found to be consistent with activation involving about three proofreading steps. the model predicts key clinical characteristics of acute infections and vaccinations, including the emergence of elite neutralisers and the effects of immune ageing. more broadly, our results establish infections and vaccinations as a new probe into the global architecture and functional principles of immune repertoires.',\n",
       "  'categories': 'q-bio.pe q-bio.sc',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-05',\n",
       "  'updated': '2024-02-27',\n",
       "  'authors': ['roberto morán-tovar', 'michael lässig'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.02794'},\n",
       " {'title': \"phase transitions in the prisoner's dilemma game on scale-free networks\",\n",
       "  'id': '2304.02896',\n",
       "  'abstract': 'we study stochastic dynamics of the prisoner\\'s dilemma game on random erd\\\\\"{o}s-r\\\\\\'{e}nyi and barab\\\\\\'{a}si-albert networks with a cost of maintaining a link between interacting players. stochastic simulations show that when the cost increases, the population of players located on barab\\\\\\'{a}si-albert network undergoes a sharp transition from an ordered state, where almost all players cooperate, to a state in which both cooperators and defectors coexist. at the critical cost, the population oscillates in time between these two states. such a situation is not present in the erd\\\\\"{o}s-r\\\\\\'{e}nyi network. we provide some heuristic analytical arguments for the phase transition and the value of the critical cost in the barab\\\\\\'{a}si-albert network.',\n",
       "  'categories': 'q-bio.pe physics.soc-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-06',\n",
       "  'updated': '2024-02-12',\n",
       "  'authors': ['jacek miękisz', 'javad mohamadichamgavi', 'jakub łącki'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.02896'},\n",
       " {'title': 'interpretable statistical representations of neural population dynamics   and geometry',\n",
       "  'id': '2304.03376',\n",
       "  'abstract': 'the dynamics of neuron populations commonly evolve on low-dimensional manifolds. thus, we need methods that learn the dynamical processes over neural manifolds to infer interpretable and consistent latent representations. we introduce a representation learning method, marble, that decomposes on-manifold dynamics into local flow fields and maps them into a common latent space using unsupervised geometric deep learning. in simulated non-linear dynamical systems, recurrent neural networks, and experimental single-neuron recordings from primates and rodents, we discover emergent low-dimensional latent representations that parametrise high-dimensional neural dynamics during gain modulation, decision-making, and changes in the internal state. these representations are consistent across neural networks and animals, enabling the robust comparison of cognitive computations. extensive benchmarking demonstrates state-of-the-art within- and across-animal decoding accuracy of marble compared with current representation learning approaches, with minimal user input. our results suggest that manifold structure provides a powerful inductive bias to develop powerful decoding algorithms and assimilate data across experiments.',\n",
       "  'categories': 'cs.lg math.ds q-bio.nc q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-06',\n",
       "  'updated': '2024-09-24',\n",
       "  'authors': ['adam gosztolai',\n",
       "   'robert l. peach',\n",
       "   'alexis arnaudon',\n",
       "   'mauricio barahona',\n",
       "   'pierre vandergheynst'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.03376'},\n",
       " {'title': 'templ: a novel deep learning model for zero-shot prediction of protein   stability and activity based on temperature-guided language modeling',\n",
       "  'id': '2304.03780',\n",
       "  'abstract': \"we introduce templ, a novel deep learning approach for zero-shot prediction of protein stability and activity, harnessing temperature-guided language modeling. by assembling an extensive dataset of 96 million sequence-host bacterial strain optimal growth temperatures (ogts) and {\\\\delta}tm data for point mutations under consistent experimental conditions, we effectively compared templ with state-of-the-art models. notably, templ demonstrated superior performance in predicting protein stability. an ablation study was conducted to elucidate the influence of ogt prediction and language modeling modules on templ's performance, revealing the importance of integrating both components. consequently, templ offers considerable promise for protein engineering applications, facilitating the design of mutation sequences with enhanced stability and activity.\",\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-07',\n",
       "  'updated': '2024-05-13',\n",
       "  'authors': ['pan tan',\n",
       "   'mingchen li',\n",
       "   'liang zhang',\n",
       "   'zhiqiang hu',\n",
       "   'liang hong'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.03780'},\n",
       " {'title': 'a geometric analysis of the sirs model with secondary infections',\n",
       "  'id': '2304.03793',\n",
       "  'abstract': 'we propose a compartmental model for a disease with temporary immunity and secondary infections. from our assumptions on the parameters involved in the model, the system naturally evolves in three time scales. we characterize the equilibria of the system and analyze their stability. we find conditions for the existence of two endemic equilibria, for some cases in which $\\\\mathcal{r}_0 < 1$. then, we unravel the interplay of the three time scales, providing conditions to foresee whether the system evolves in all three scales, or only in the fast and the intermediate ones. we conclude with numerical simulations and bifurcation analysis, to complement our analytical results.',\n",
       "  'categories': 'math.ds physics.soc-ph q-bio.pe',\n",
       "  'doi': '10.1137/23m1565632',\n",
       "  'created': '2023-04-07',\n",
       "  'updated': '2023-04-11',\n",
       "  'authors': ['panagiotis kaklamanos',\n",
       "   'andrea pugliese',\n",
       "   'mattia sensi',\n",
       "   'sara sottile'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.03793'},\n",
       " {'title': 'precision oncology, signaling pathways reprogramming and targeted   therapy: a holistic approach to molecular cancer therapeutics',\n",
       "  'id': '2304.05411',\n",
       "  'abstract': \"cancer is a fatal genetic disease involving unregulated cell growth and proliferation with varying underlying complexities including immune evasion, treatment resistance and recurrence, and optimized treatment is required for proper cure. molecular studies have revealed that tumors are extremely heterogeneous in nature, leading to the complexity of cancer development, which is ultimately linked to its genetic machinery. it would require effective targeting of dysregulated molecular mechanisms involving growth factors, regulatory proteins, cell adhesion molecules, and molecules of immune system mainly driven by alterations in tumor suppressor genes and oncogenes that may vary among different cancer types. importantly, patients with the same type of cancer respond differently to available cancer treatments, indicating the need for patient-specific treatment options. thus, in-depth genomic studies of patients' tumors are needed to fully understand the determinants of cancer initiation and progression for effective targeted therapy. precision oncology has evolved as a form of cancer therapy focused on genetic profiling of tumors to identify molecular alterations involved in cancer manifestation for tailored individualized treatment of the disease. accordingly, there have been great developments in the formulation and production of anticancer agents in recent years owing to advances in molecular technologies enabling precise targeting of oncogenic pathways involved in disease progression. this article aims to briefly explain the foundations and frontiers of precision oncology in the context of advancements in the tools and techniques associated with the process to assess its scope and importance in realizing the intended goals.\",\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '10.17632/j33xfgvyyz.5',\n",
       "  'created': '2023-04-11',\n",
       "  'updated': '2024-11-22',\n",
       "  'authors': ['manish kumar'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.05411'},\n",
       " {'title': 'energy-positive soaring using transient turbulent fluctuations',\n",
       "  'id': '2304.05983',\n",
       "  'abstract': 'soaring birds gain energy from stable ascending currents or shear. however, it remains unclear whether energy loss due to drag can be overcome by extracting work from transient turbulent fluctuations. we designed numerical simulations of gliders navigating in a kinematic model that captures the spatio-temporal correlations of atmospheric turbulence. energy extraction is enabled by an adaptive algorithm based on monte carlo tree search that dynamically filters acquired information about the flow to plan future paths. we show that net energy gain is feasible under realistic constraints. glider paths reflect patterns of foraging, where exploration of the flow is interspersed with bouts of energy extraction through localized spirals.',\n",
       "  'categories': 'physics.flu-dyn physics.app-ph q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-12',\n",
       "  'updated': '2024-01-09',\n",
       "  'authors': ['danyun he', 'gautam reddy', 'chris h. rycroft'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.05983'},\n",
       " {'title': 'bayesian mixture models for phylogenetic source attribution from   consensus sequences and time since infection estimates',\n",
       "  'id': '2304.06353',\n",
       "  'abstract': 'in stopping the spread of infectious diseases, pathogen genomic data can be used to reconstruct transmission events and characterize population-level sources of infection. most approaches for identifying transmission pairs do not account for the time passing since divergence of pathogen variants in individuals, which is problematic in viruses with high within-host evolutionary rates. this prompted us to consider possible transmission pairs in terms of phylogenetic data and additional estimates of time since infection derived from clinical biomarkers. we develop bayesian mixture models with an evolutionary clock as signal component and additional mixed effects or covariate random functions describing the mixing weights to classify potential pairs into likely and unlikely transmission pairs. we demonstrate that although sources cannot be identified at the individual level with certainty, even with the additional data on time elapsed, inferences into the population-level sources of transmission are possible, and more accurate than using only phylogenetic data without time since infection estimates. we apply the approach to estimate age-specific sources of hiv infection in amsterdam msm transmission networks between 2010-2021. this study demonstrates that infection time estimates provide informative data to characterize transmission sources, and shows how phylogenetic source attribution can then be done with multi-dimensional mixture models.',\n",
       "  'categories': 'q-bio.pe stat.me',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-13',\n",
       "  'updated': '2024-08-22',\n",
       "  'authors': ['alexandra blenkinsop',\n",
       "   'lysandros sofocleous',\n",
       "   'francesco di lauro',\n",
       "   'evangelia georgia kostaki',\n",
       "   'ard van sighem',\n",
       "   'daniela bezemer',\n",
       "   'thijs van de laar',\n",
       "   'peter reiss',\n",
       "   'godelieve de bree',\n",
       "   'nikos pantazis',\n",
       "   'oliver ratmann'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.06353'},\n",
       " {'title': 'deterministic epidemic models overestimate the basic reproduction number   of observed outbreaks',\n",
       "  'id': '2304.06661',\n",
       "  'abstract': \"the basic reproduction number, $r_0$, is a well-known quantifier of epidemic spread. however, a class of existing methods for estimating $r_0$ from incidence data early in the epidemic can lead to an over-estimation of this quantity. in particular, when fitting deterministic models to estimate the rate of spread, we do not account for the stochastic nature of epidemics and that, given the same system, some outbreaks may lead to epidemics and some may not. typically, an observed epidemic that we wish to control is a major outbreak. this amounts to implicit selection for major outbreaks which leads to the over-estimation problem. we formally characterised the split between major and minor outbreaks by using otsu's method which provides us with a working definition. we show that by conditioning a `deterministic' model on major outbreaks, we can more reliably estimate the basic reproduction number from an observed epidemic trajectory.\",\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1016/j.idm.2024.02.007',\n",
       "  'created': '2023-04-13',\n",
       "  'updated': '2024-03-26',\n",
       "  'authors': ['wajid ali',\n",
       "   'christopher e. overton',\n",
       "   'robert r. wilkinson',\n",
       "   'kieran j. sharkey'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.06661'},\n",
       " {'title': 'modeling dense multimodal interactions between biological pathways and   histology for survival prediction',\n",
       "  'id': '2304.06819',\n",
       "  'abstract': 'integrating whole-slide images (wsis) and bulk transcriptomics for predicting patient survival can improve our understanding of patient prognosis. however, this multimodal task is particularly challenging due to the different nature of these data: wsis represent a very high-dimensional spatial description of a tumor, while bulk transcriptomics represent a global description of gene expression levels within that tumor. in this context, our work aims to address two key challenges: (1) how can we tokenize transcriptomics in a semantically meaningful and interpretable way?, and (2) how can we capture dense multimodal interactions between these two modalities? specifically, we propose to learn biological pathway tokens from transcriptomics that can encode specific cellular functions. together with histology patch tokens that encode the different morphological patterns in the wsi, we argue that they form appropriate reasoning units for downstream interpretability analyses. we propose fusing both modalities using a memory-efficient multimodal transformer that can model interactions between pathway and histology patch tokens. our proposed model, survpath, achieves state-of-the-art performance when evaluated against both unimodal and multimodal baselines on five datasets from the cancer genome atlas. our interpretability framework identifies key multimodal prognostic factors, and, as such, can provide valuable insights into the interaction between genotype and phenotype, enabling a deeper understanding of the underlying biological mechanisms at play. we make our code public at: https://github.com/ajv012/survpath.',\n",
       "  'categories': 'cs.cv cs.ai q-bio.gn q-bio.qm q-bio.to',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-13',\n",
       "  'updated': '2024-04-15',\n",
       "  'authors': ['guillaume jaume',\n",
       "   'anurag vaidya',\n",
       "   'richard chen',\n",
       "   'drew williamson',\n",
       "   'paul liang',\n",
       "   'faisal mahmood'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.06819'},\n",
       " {'title': 'shu: visualization of high dimensional biological pathways',\n",
       "  'id': '2304.07178',\n",
       "  'abstract': 'summary: shu is a visualization tool that integrates diverse data types into a metabolic map, with a focus on supporting multiple conditions and visualizing distributions. the goal is to provide a unified platform for handling the growing volume of multi-omics data, leveraging the metabolic maps developed by the metabolic modeling community. additionally, shu offers a streamlined python api, based on the grammar of graphics, for easy integration with data pipelines.   availability and implementation: freely available at https://github.com/biosustain/shu under mit/apache 2.0 license. binaries are available in the release page of the repository and the web app is deployed at https://biosustain.github.io/shu.',\n",
       "  'categories': 'q-bio.mn',\n",
       "  'doi': '10.1093/bioinformatics/btae140',\n",
       "  'created': '2023-04-14',\n",
       "  'updated': '',\n",
       "  'authors': ['jorge carrasco muriel',\n",
       "   'nicholas cowie',\n",
       "   'marjan mansouvar',\n",
       "   'teddy groves',\n",
       "   'lars keld nielsen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.07178'},\n",
       " {'title': 'easyner: a customizable easy-to-use pipeline for deep learning- and   dictionary-based named entity recognition from medical text',\n",
       "  'id': '2304.07805',\n",
       "  'abstract': 'background medical research generates millions of publications and it is a great challenge for researchers to utilize this information in full since its scale and complexity greatly surpasses human reading capabilities. automated text mining can help extract and connect information spread across this large body of literature but this technology is not easily accessible to life scientists. results here, we developed an easy-to-use end-to-end pipeline for deep learning- and dictionary-based named entity recognition (ner) of typical entities found in medical research articles, including diseases, cells, chemicals, genes/proteins, and species. the pipeline can access and process large medical research article collections (pubmed, cord-19) or raw text and incorporates a series of deep learning models fine-tuned on the huner corpora collection. in addition, the pipeline can perform dictionary-based ner related to covid-19 and other medical topics. users can also load their own ner models and dictionaries to include additional entities. the output consists of publication-ready ranked lists and graphs of detected entities and files containing the annotated texts. an associated script allows rapid inspection of the results for specific entities of interest. as model use cases, the pipeline was deployed on two collections of autophagy-related abstracts from pubmed and on the cord19 dataset, a collection of 764 398 research article abstracts related to covid-19. conclusions the ner pipeline we present is applicable in a variety of medical research settings and makes customizable text mining accessible to life scientists.',\n",
       "  'categories': 'q-bio.qm cs.cl',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-16',\n",
       "  'updated': '2024-03-07',\n",
       "  'authors': ['rafsan ahmed',\n",
       "   'petter berntsson',\n",
       "   'alexander skafte',\n",
       "   'salma kazemi rashed',\n",
       "   'marcus klang',\n",
       "   'adam barvesten',\n",
       "   'ola olde',\n",
       "   'william lindholm',\n",
       "   'antton lamarca arrizabalaga',\n",
       "   'pierre nugues',\n",
       "   'sonja aits'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.07805'},\n",
       " {'title': 'a discussion of bisexual populations with wolbachia infection as an   evolution algebra',\n",
       "  'id': '2304.07826',\n",
       "  'abstract': 'in this paper, wolbachia infection in a bisexual and diploid population with a fixed cytoplasmic incompatibility rate $w$ and maternal transmission rate $d$ is studied as an evolution algebra. as the cytoplasmic incompatibility (ci) of the population causes deaths in the offspring, the evolution algebra of this model is not baric, and is a dibaric algebra if and only if the cytoplasmic incompatibility rate $w$ is 1 and $d=1$. the idempotent elements are given in terms of $d$ and $w$. moreover, this algebra has no absolute nilpotent elements when ci expression $w \\\\neq 1$.',\n",
       "  'categories': 'math.ra q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-16',\n",
       "  'updated': '2024-01-20',\n",
       "  'authors': ['songül esin', 'müge kanuni', 'barış özdinç'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.07826'},\n",
       " {'title': 'gene activity fully predicts transcriptional bursting dynamics',\n",
       "  'id': '2304.08770',\n",
       "  'abstract': \"transcription commonly occurs in bursts, with alternating productive (on) and quiescent (off) periods, governing mrna production rates. yet, how transcription is regulated through bursting dynamics remains unresolved. here, we conduct real-time measurements of endogenous transcriptional bursting with single-mrna sensitivity. leveraging the diverse transcriptional activities in early fly embryos, we uncover stringent relationships between bursting parameters. specifically, we find that the durations of on and off periods are linked. regardless of the developmental stage or body-axis position, gene activity levels predict individual alleles' average on and off periods. lowly transcribing alleles predominantly modulate off periods (burst frequency), while highly transcribing alleles primarily tune on periods (burst size). these relationships persist even under perturbations of cis-regulatory elements or trans-factors and account for bursting dynamics measured in other species. our results suggest a novel mechanistic constraint governing bursting dynamics rather than a modular control of distinct parameters by distinct regulatory processes.\",\n",
       "  'categories': 'q-bio.mn physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-18',\n",
       "  'updated': '2024-06-28',\n",
       "  'authors': ['po-ta chen', 'michal levo', 'benjamin zoller', 'thomas gregor'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.08770'},\n",
       " {'title': 'tuna and billfish larval distributions in a warming ocean',\n",
       "  'id': '2304.09442',\n",
       "  'abstract': 'tuna and billfish are charismatic pelagic fishes attracting considerable scientific attention due to their ecophysiological and socioeconomic importance. however, the knowledge of their basin-wide spawning and larval habitats, especially in a warming ocean, is limited. this knowledge gap undermines effective fishery management by introducing recruitment uncertainty, which makes population dynamics unpredictable. we fill the gap with a parsimonious geostatistical species-distribution model trained on the largest available dataset on tuna and billfish larvae in the pacific ocean. the model reveals (i) the basin-wide seasonal larval distributions over the reference period 1960-85, (ii) the expected impact of ongoing ocean warming on these distributions, and (iii) the biogeochemical factors, such as ph, phosphate concentration, and sea-surface height, that shape the larval habitat. our findings make a quantum leap in understanding the ecophysiology of tuna and billfish, providing valuable information for future conservation efforts.',\n",
       "  'categories': 'physics.ao-ph q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-19',\n",
       "  'updated': '2024-10-22',\n",
       "  'authors': ['hirotaka ijima', 'marko jusup'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.09442'},\n",
       " {'title': 'long time scales, individual differences, and scale invariance in animal   behavior',\n",
       "  'id': '2304.09608',\n",
       "  'abstract': 'the explosion of data on animal behavior in more natural contexts highlights the fact that these behaviors exhibit correlations across many time scales. but there are major challenges in analyzing these data: records of behavior in single animals have fewer independent samples than one might expect; in pooling data from multiple animals, individual differences can mimic long-ranged temporal correlations; conversely long-ranged correlations can lead to an over-estimate of individual differences. we suggest an analysis scheme that addresses these problems directly, apply this approach to data on the spontaneous behavior of walking flies, and find evidence for scale invariant correlations over nearly three decades in time, from seconds to one hour. three different measures of correlation are consistent with a single underlying scaling field of dimension $\\\\delta = 0.180\\\\pm 0.005$.',\n",
       "  'categories': 'q-bio.nc cond-mat.dis-nn q-bio.qm',\n",
       "  'doi': '10.1103/physrevlett.132.048401',\n",
       "  'created': '2023-04-19',\n",
       "  'updated': '',\n",
       "  'authors': ['william bialek', 'joshua w. shaevitz'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.09608'},\n",
       " {'title': 'genegpt: augmenting large language models with domain tools for improved   access to biomedical information',\n",
       "  'id': '2304.09667',\n",
       "  'abstract': 'while large language models (llms) have been successfully applied to various tasks, they still face challenges with hallucinations. augmenting llms with domain-specific tools such as database utilities can facilitate easier and more precise access to specialized knowledge. in this paper, we present genegpt, a novel method for teaching llms to use the web apis of the national center for biotechnology information (ncbi) for answering genomics questions. specifically, we prompt codex to solve the geneturing tests with ncbi web apis by in-context learning and an augmented decoding algorithm that can detect and execute api calls. experimental results show that genegpt achieves state-of-the-art performance on eight tasks in the geneturing benchmark with an average score of 0.83, largely surpassing retrieval-augmented llms such as the new bing (0.44), biomedical llms such as biomedlm (0.08) and biogpt (0.04), as well as gpt-3 (0.16) and chatgpt (0.12). our further analyses suggest that: (1) api demonstrations have good cross-task generalizability and are more useful than documentations for in-context learning; (2) genegpt can generalize to longer chains of api calls and answer multi-hop questions in genehop, a novel dataset introduced in this work; (3) different types of errors are enriched in different tasks, providing valuable insights for future improvements.',\n",
       "  'categories': 'cs.cl cs.ai q-bio.gn',\n",
       "  'doi': '10.1093/bioinformatics/btae075',\n",
       "  'created': '2023-04-19',\n",
       "  'updated': '2023-05-16',\n",
       "  'authors': ['qiao jin', 'yifan yang', 'qingyu chen', 'zhiyong lu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.09667'},\n",
       " {'title': 'pulse shape and voltage-dependent synchronization in spiking neuron   networks',\n",
       "  'id': '2304.09813',\n",
       "  'abstract': 'pulse-coupled spiking neural networks are a powerful tool to gain mechanistic insights into how neurons self-organize to produce coherent collective behavior. these networks use simple spiking neuron models, such as the $\\\\theta$-neuron or the quadratic integrate-and-fire (qif) neuron, that replicate the essential features of real neural dynamics. interactions between neurons are modeled with infinitely narrow pulses, or spikes, rather than the more complex dynamics of real synapses. to make these networks biologically more plausible, it has been proposed that they must also account for the finite width of the pulses, which can have a significant impact on the network dynamics. however, the derivation and interpretation of these pulses is contradictory and the impact of the pulse shape on the network dynamics is largely unexplored. here, i take a comprehensive approach to pulse-coupling in networks of qif and $\\\\theta$-neurons. i argue that narrow pulses activate voltage-dependent synaptic conductances and show how to implement them in qif neurons such that their effect can last through the phase after the spike. using an exact low-dimensional description for networks of globally coupled spiking neurons, i prove for instantaneous interactions that collective oscillations emerge due to an effective coupling through the mean voltage. i analyze the impact of the pulse shape by means of a family of smooth pulse functions with arbitrary finite width and symmetric or asymmetric shapes. for symmetric pulses, the resulting voltage-coupling is not very effective in synchronizing neurons, but pulses that are slightly skewed to the phase after the spike readily generate collective oscillations. the results unveil a voltage-dependent spike synchronization mechanism in neural networks, which is facilitated by pulses of finite width and complementary to traditional synaptic transmission.',\n",
       "  'categories': 'q-bio.nc nlin.ao physics.bio-ph',\n",
       "  'doi': '10.1162/neco_a_01680',\n",
       "  'created': '2023-04-19',\n",
       "  'updated': '2024-01-19',\n",
       "  'authors': ['bastian pietras'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.09813'},\n",
       " {'title': \"revealing the unseen: likely half of the americans relied on others'   experience when deciding on taking the covid-19 vaccine\",\n",
       "  'id': '2304.09931',\n",
       "  'abstract': \"efficient coverage for newly developed vaccines requires knowing which groups of individuals will accept the vaccine immediately and which will take longer to accept or never accept. of those who may eventually accept the vaccine, there are two main types: success-based learners, basing their decisions on others' satisfaction, and myopic rationalists, attending to their own immediate perceived benefit. we used covid-19 vaccination data to fit a mechanistic model capturing the distinct effects of the two types on the vaccination progress. we estimated that 47 percent of americans behaved as myopic rationalist with a high variations across the jurisdictions, from 31 percent in mississippi to 76 percent in vermont. the proportion was correlated with the vaccination coverage, proportion of votes in favor of democrats in 2020 presidential election, and education score.\",\n",
       "  'categories': 'q-bio.pe physics.soc-ph',\n",
       "  'doi': '10.1007/s11538-024-01290-4',\n",
       "  'created': '2023-04-19',\n",
       "  'updated': '2023-08-11',\n",
       "  'authors': ['azadeh aghaeeyan', 'pouria ramazi', 'mark a. lewis'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.09931'},\n",
       " {'title': 'interpretable (not just posthoc-explainable) heterogeneous survivor   bias-corrected treatment effects for assignment of postdischarge   interventions to prevent readmissions',\n",
       "  'id': '2304.09981',\n",
       "  'abstract': 'we used survival analysis to quantify the impact of postdischarge evaluation and management (e/m) services in preventing hospital readmission or death. our approach avoids a specific pitfall of applying machine learning to this problem, which is an inflated estimate of the effect of interventions, due to survivors bias -- where the magnitude of inflation may be conditional on heterogeneous confounders in the population. this bias arises simply because in order to receive an intervention after discharge, a person must not have been readmitted in the intervening period. after deriving an expression for this phantom effect, we controlled for this and other biases within an inherently interpretable bayesian survival framework. we identified case management services as being the most impactful for reducing readmissions overall.',\n",
       "  'categories': 'stat.me cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-19',\n",
       "  'updated': '2023-08-03',\n",
       "  'authors': ['hongjing xia',\n",
       "   'joshua c. chang',\n",
       "   'sarah nowak',\n",
       "   'sonya mahajan',\n",
       "   'rohit mahajan',\n",
       "   'ted l. chang',\n",
       "   'carson c. chow'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.09981'},\n",
       " {'title': 'universal slow dynamics of chemical reaction networks',\n",
       "  'id': '2304.10072',\n",
       "  'abstract': 'understanding the emergent behavior of chemical reaction networks (crns) is a fundamental aspect of biology and its origin from inanimate matter. a closed crn monotonically tends to thermal equilibrium, but when it is opened to external reservoirs, a range of behaviors is possible, including transition to a new equilibrium state, a non-equilibrium state, or indefinite growth. this study shows that slowly driven crns are governed by the conserved quantities of the closed system, which are generally far fewer in number than the species. considering both deterministic and stochastic dynamics, a universal slow dynamics equation is derived with singular perturbation methods, and is shown to be thermodynamically consistent. the slow dynamics is highly robust against microscopic details of the network, which may be unknown in practical situations. in particular, non-equilibrium states of realistic large crns can be sought without knowledge of bulk reaction rates. the framework is successfully tested against a suite of networks of increasing complexity and argued to be relevant in the treatment of open crns as chemical machines.',\n",
       "  'categories': 'q-bio.mn cond-mat.stat-mech nlin.ao',\n",
       "  'doi': '10.1103/physreve.109.044105',\n",
       "  'created': '2023-04-19',\n",
       "  'updated': '2024-03-13',\n",
       "  'authors': ['masanari shimada', 'pegah behrad', 'eric de giuli'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.10072'},\n",
       " {'title': 'stomanager1: an enhanced, automated, and high-throughput tool to measure   leaf stomata and guard cell metrics using empirical and theoretical   algorithms',\n",
       "  'id': '2304.10450',\n",
       "  'abstract': \"automated stomata detection and measuring are vital for understanding plant physiological performance and ecological functioning in global water and carbon cycles. current methods are laborious, time-consuming, prone to bias, and limited in scale. we developed stomanager1, a high-throughput tool utilizing empirical and theoretical algorithms and convolutional neural networks to automatically detect, count, and measure over 30 stomatal and guard cell metrics, including stomata and guard cell area, length, width, and orientation, stomatal evenness, divergence, and aggregation index. these metrics, combined with leaf functional traits, explained 78% and 93% of productivity and intrinsic water use efficiency (iwue) variances in hardwoods, making them significant factors in leaf physiology and tree growth. stomanager1 demonstrates exceptional precision and recall (map@0.5 over 0.993), effectively capturing diverse stomatal properties across various species. stomanager1facilitates the automation of measuring leaf stomata, enabling broader exploration of stomatal control in plant growth and adaptation to environmental stress and climate change. this has implications for global gross primary productivity (gpp) modeling and estimation, as integrating stomatal metrics can enhance comprehension and predictions of plant growth and resource usage worldwide. stomanager1's source code and an online demonstration are available on github (https://github.com/jiaxinwang123/stomanager.git), along with a user-friendly windows application on zenodo (https://doi.org/10.5281/zenodo.7686022).\",\n",
       "  'categories': 'q-bio.to',\n",
       "  'doi': '10.1093/plphys/kiae049',\n",
       "  'created': '2023-04-20',\n",
       "  'updated': '2023-05-25',\n",
       "  'authors': ['jiaxin wang', 'heidi j. renninger', 'qin ma', 'shichao jin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.10450'},\n",
       " {'title': 'orientation selectivity properties for the affine gaussian derivative   and the affine gabor models for visual receptive fields',\n",
       "  'id': '2304.11920',\n",
       "  'abstract': 'this paper presents a theoretical analysis of the orientation selectivity of simple and complex cells that can be well modelled by the generalized gaussian derivative model for visual receptive fields, with the purely spatial component of the receptive fields determined by oriented affine gaussian derivatives for different orders of spatial differentiation.   a detailed mathematical analysis is presented for the three different cases of either: (i) purely spatial receptive fields, (ii) space-time separable spatio-temporal receptive fields and (iii) velocity-adapted spatio-temporal receptive fields. closed-form theoretical expressions for the orientation selectivity curves for idealized models of simple and complex cells are derived for all these main cases, and it is shown that the orientation selectivity of the receptive fields becomes more narrow, as a scale parameter ratio $\\\\kappa$, defined as the ratio between the scale parameters in the directions perpendicular to vs. parallel with the preferred orientation of the receptive field, increases. it is also shown that the orientation selectivity becomes more narrow with increasing order of spatial differentiation in the underlying affine gaussian derivative operators over the spatial domain.   for comparison, we also present a corresponding theoretical orientation selectivity analysis for purely spatial receptive fields according to an affine gabor model. the results from that analysis are consistent with the results obtained from the affine gaussian derivative model,in the respect that the orientation selectivity becomes more narrow when making the receptive fields wider in the direction perpendicular to the preferred orientation of the receptive field.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-24',\n",
       "  'updated': '2024-11-08',\n",
       "  'authors': ['tony lindeberg'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.11920'},\n",
       " {'title': 'information theory for complex systems scientists',\n",
       "  'id': '2304.12482',\n",
       "  'abstract': 'in the 21st century, many of the crucial scientific and technical issues facing humanity can be understood as problems associated with understanding, modelling, and ultimately controlling complex systems: systems comprised of a large number of non-trivially interacting components whose collective behaviour can be difficult to predict. information theory, a branch of mathematics historically associated with questions about encoding and decoding messages, has emerged as something of a lingua franca for those studying complex systems, far exceeding its original narrow domain of communication systems engineering. in the context of complexity science, information theory provides a set of tools which allow researchers to uncover the statistical and effective dependencies between interacting components; relationships between systems and their environment; mereological whole-part relationships; and is sensitive to non-linearities missed by commonly parametric statistical models.   in this review, we aim to provide an accessible introduction to the core of modern information theory, aimed specifically at aspiring (and established) complex systems scientists. this includes standard measures, such as shannon entropy, relative entropy, and mutual information, before building to more advanced topics, including: information dynamics, measures of statistical complexity, information decomposition, and effective network inference. in addition to detailing the formal definitions, in this review we make an effort to discuss how information theory can be interpreted and develop the intuition behind abstract concepts like \"entropy,\" in the hope that this will enable interested readers to understand what information is, and how it is used, at a more fundamental level.',\n",
       "  'categories': 'cs.it math.it physics.data-an q-bio.qm stat.ot',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-24',\n",
       "  'updated': '2024-08-13',\n",
       "  'authors': ['thomas f. varley'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.12482'},\n",
       " {'title': 'phylo2vec: a vector representation for binary trees',\n",
       "  'id': '2304.12693',\n",
       "  'abstract': 'binary phylogenetic trees inferred from biological data are central to understanding the shared history among evolutionary units. however, inferring the placement of latent nodes in a tree is computationally expensive. state-of-the-art methods rely on carefully designed heuristics for tree search, using different data structures for easy manipulation (e.g., classes in object-oriented programming languages) and readable representation of trees (e.g., newick-format strings). here, we present phylo2vec, a parsimonious encoding for phylogenetic trees that serves as a unified approach for both manipulating and representing phylogenetic trees. phylo2vec maps any binary tree with $n$ leaves to a unique integer vector of length $n-1$. the advantages of phylo2vec are fourfold: i) fast tree sampling, (ii) compressed tree representation compared to a newick string, iii) quick and unambiguous verification if two binary trees are identical topologically, and iv) systematic ability to traverse tree space in very large or small jumps. as a proof of concept, we use phylo2vec for maximum likelihood inference on five real-world datasets and show that a simple hill-climbing-based optimisation scheme can efficiently traverse the vastness of tree space from a random to an optimal tree.',\n",
       "  'categories': 'q-bio.pe cs.lg q-bio.qm',\n",
       "  'doi': '10.1093/sysbio/syae030',\n",
       "  'created': '2023-04-25',\n",
       "  'updated': '2024-11-04',\n",
       "  'authors': ['matthew j penn',\n",
       "   'neil scheidwasser',\n",
       "   'mark p khurana',\n",
       "   'david a duchêne',\n",
       "   'christl a donnelly',\n",
       "   'samir bhatt'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.12693'},\n",
       " {'title': 'adaptive representations of sound for automatic insect recognition',\n",
       "  'id': '2304.12739',\n",
       "  'abstract': 'insect population numbers and biodiversity have been rapidly declining with time, and monitoring these trends has become increasingly important for conservation measures to be effectively implemented. but monitoring methods are often invasive, time and resource intense, and prone to various biases. many insect species produce characteristic sounds that can easily be detected and recorded without large cost or effort. using deep learning methods, insect sounds from field recordings could be automatically detected and classified to monitor biodiversity and species distribution ranges. we implement this using recently published datasets of insect sounds (orthoptera and cicadidae) and machine learning methods and evaluate their potential for acoustic insect monitoring. we compare the performance of the conventional spectrogram-based audio representation against leaf, a new adaptive and waveform-based frontend. leaf achieved better classification performance than the mel-spectrogram frontend by adapting its feature extraction parameters during training. this result is encouraging for future implementations of deep learning technology for automatic insect sound recognition, especially as larger datasets become available.',\n",
       "  'categories': 'cs.sd eess.as q-bio.qm',\n",
       "  'doi': '10.1371/journal.pcbi.1011541',\n",
       "  'created': '2023-04-25',\n",
       "  'updated': '',\n",
       "  'authors': ['marius faiß', 'dan stowell'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.12739'},\n",
       " {'title': 'generating molecular fragmentation graphs with autoregressive neural   networks',\n",
       "  'id': '2304.13136',\n",
       "  'abstract': 'the accurate prediction of tandem mass spectra from molecular structures has the potential to unlock new metabolomic discoveries by augmenting the community\\'s libraries of experimental reference standards. cheminformatic spectrum prediction strategies use a \"bond-breaking\" framework to iteratively simulate mass spectrum fragmentations, but these methods are (a) slow, due to the need to exhaustively and combinatorially break molecules and (b) inaccurate, as they often rely upon heuristics to predict the intensity of each resulting fragment; neural network alternatives mitigate computational cost but are black-box and not inherently more accurate. we introduce a physically-grounded neural approach that learns to predict each breakage event and score the most relevant subset of molecular fragments quickly and accurately. we evaluate our model by predicting spectra from both public and private standard libraries, demonstrating that our hybrid approach offers state of the art prediction accuracy, improved metabolite identification from a database of candidates, and higher interpretability when compared to previous breakage methods and black box neural networks. the grounding of our approach in physical fragmentation events shows especially high promise for elucidating natural product molecules with more complex scaffolds.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-25',\n",
       "  'updated': '2024-01-08',\n",
       "  'authors': ['samuel goldman', 'janet li', 'connor w. coley'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.13136'},\n",
       " {'title': 'random evolutionary games and random polynomials',\n",
       "  'id': '2304.13831',\n",
       "  'abstract': 'in this paper, we discover that the class of random polynomials arising from the equilibrium analysis of random asymmetric evolutionary games is \\\\textit{exactly} the kostlan-shub-smale system of random polynomials, revealing an intriguing connection between evolutionary game theory and the theory of random polynomials. through this connection, we analytically characterize the statistics of the number of internal equilibria of random asymmetric evolutionary games, namely its mean value, probability distribution, central limit theorem and universality phenomena. biologically, these quantities enable prediction of the levels of social and biological diversity as well as the overall complexity in a dynamical system. by comparing symmetric and asymmetric random games, we establish that symmetry in group interactions increases the expected number of internal equilibria. our research establishes new theoretical understanding of asymmetric evolutionary games and highlights the significance of symmetry and asymmetry in group interactions.',\n",
       "  'categories': 'math.pr q-bio.pe stat.me',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-26',\n",
       "  'updated': '2024-10-02',\n",
       "  'authors': ['manh hong duong', 'the anh han'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.13831'},\n",
       " {'title': 'mudiff: unified diffusion for complete molecule generation',\n",
       "  'id': '2304.14621',\n",
       "  'abstract': 'molecule generation is a very important practical problem, with uses in drug discovery and material design, and ai methods promise to provide useful solutions. however, existing methods for molecule generation focus either on 2d graph structure or on 3d geometric structure, which is not sufficient to represent a complete molecule as 2d graph captures mainly topology while 3d geometry captures mainly spatial atom arrangements. combining these representations is essential to better represent a molecule. in this paper, we present a new model for generating a comprehensive representation of molecules, including atom features, 2d discrete molecule structures, and 3d continuous molecule coordinates, by combining discrete and continuous diffusion processes. the use of diffusion processes allows for capturing the probabilistic nature of molecular processes and exploring the effect of different factors on molecular structures. additionally, we propose a novel graph transformer architecture to denoise the diffusion process. the transformer adheres to 3d roto-translation equivariance constraints, allowing it to learn invariant atom and edge representations while preserving the equivariance of atom coordinates. this transformer can be used to learn molecular representations robust to geometric transformations. we evaluate the performance of our model through experiments and comparisons with existing methods, showing its ability to generate more stable and valid molecules. our model is a promising approach for designing stable and diverse molecules and can be applied to a wide range of tasks in molecular modeling.',\n",
       "  'categories': 'cs.lg q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-28',\n",
       "  'updated': '2024-02-05',\n",
       "  'authors': ['chenqing hua',\n",
       "   'sitao luan',\n",
       "   'minkai xu',\n",
       "   'rex ying',\n",
       "   'jie fu',\n",
       "   'stefano ermon',\n",
       "   'doina precup'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.14621'},\n",
       " {'title': 'in silico design, in vitro construction and in vivo application of   synthetic small regulatory rnas in bacteria',\n",
       "  'id': '2304.14932',\n",
       "  'abstract': 'small regulatory rnas (srnas) are short non-coding rnas in bacteria capable of post-transcriptional regulation. srnas have recently gained attention as tools in basic and applied sciences for example to fine-tune genetic circuits or biotechnological processes. even though srnas often have a rather simple and modular structure, the design of functional synthetic srnas is not necessarily trivial. this protocol outlines how to use computational predictions and synthetic biology approaches to design, construct and validate synthetic srna functionality for their application in bacteria. the computational tool, seedling, matches the optimal seed region with the user-selected srna scaffold for repression of target mrnas. the synthetic srnas are assembled using golden gate cloning and their functionality is subsequently validated. the protocol uses the acra mrna as an exemplary proof-of-concept target in escherichia coli. since acra is part of a multidrug efflux pump, acra repression can be revealed by assessing oxacillin susceptibility in a phenotypic screen. however, in case target repression does not result in a screenable phenotype, an alternative validation of synthetic srna functionality based on a fluorescence reporter is described.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.1007/978-1-0716-3658-9_27',\n",
       "  'created': '2023-04-28',\n",
       "  'updated': '',\n",
       "  'authors': ['michel brueck', 'bork a. berghoff', 'daniel schindler'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2304.14932'},\n",
       " {'title': 'using neural ordinary differential equations to predict complex   ecological dynamics from population density data',\n",
       "  'id': '2305.00338',\n",
       "  'abstract': 'simple models have been used to describe ecological processes for over a century. however, the complexity of ecological systems makes simple models subject to modeling bias due to simplifying assumptions or unaccounted factors, limiting their predictive power. neural ordinary differential equations (nodes) have surged as a machine-learning algorithm that preserves the dynamic nature of the data \\\\cite{chen_neural_2018}. although preserving the dynamics in the data is an advantage, the question of how nodes perform as a forecasting tool of ecological communities is unanswered. here we explore this question using simulated time series of competing species in a time-varying environment. we find that nodes provide more precise forecasts than arima models. {we also find that untuned nodes have a similar forecasting accuracy as untuned long-short term memory neural networks (lstms) and both are outperformed in accuracy and precision by edm models. however, we also find nodes generally outperform all other methods when evaluating with the interval score, which evaluates precision and accuracy in terms of prediction intervals rather than pointwise accuracy.} we also discuss ways to improve the forecasting performance {of nodes}. the power of a forecasting tool such as nodes is that it can provide insights into population dynamics and should thus broaden the approaches to studying time series of ecological communities.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-29',\n",
       "  'updated': '2024-01-23',\n",
       "  'authors': ['jorge arroyo-esquivel',\n",
       "   'christopher a klausmeier',\n",
       "   'elena litchman'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.00338'},\n",
       " {'title': 'dynein-driven self-organization of microtubules: an entropy- and   network-based analysis',\n",
       "  'id': '2305.00539',\n",
       "  'abstract': \"microtubules self-organize to form part of the cellular cytoskeleton. they give cells their shape and play a crucial role in cell division and intracellular transport. strikingly, microtubules driven by motor proteins reorganize into stable mitotic/meiotic spindles with high spatial and temporal precision during successive cell division cycles. although the topic has been extensively studied, the question remains: what defines such microtubule networks' spatial order and robustness? here, we aim to approach this problem by analyzing a simplified computational model of radial microtubule self-organization driven by a single type of motor protein -- dyneins. we establish that the spatial order of the steady-state pattern is likely associated with the dynein-driven microtubule motility. at the same time, the structure of the microtubule network is likely linked to its connectivity at the beginning of self-organization. using the continuous variation of dynein concentration, we reveal hysteresis in microtubule self-organization, ensuring the stability of radial filament structures.\",\n",
       "  'categories': 'nlin.ao nlin.ps q-bio.sc',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-30',\n",
       "  'updated': '2024-05-15',\n",
       "  'authors': ['nikita frolov',\n",
       "   'bram bijnens',\n",
       "   'daniel ruiz-reynés',\n",
       "   'lendert gelens'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.00539'},\n",
       " {'title': 'spatial dynamics of synergistic coinfection in rock-paper-scissors   models',\n",
       "  'id': '2305.00590',\n",
       "  'abstract': \"we investigate the spatial dynamics of two disease epidemics reaching a three-species cyclic model. regardless of their species, all individuals are susceptible to being infected with two different pathogens, which spread through person-to-person contact. the occurrence of coinfection leads to a synergistic increase in the risk of hosts dying due to complications from either disease. our stochastic simulations show that departed areas inhabited by hosts of a single pathogen arise from random initial conditions. the single-disease spatial domains are bordered by interfaces of coinfected hosts whose dynamics are curvature-driven. our findings show that the coarsening dynamics of the interface network are controlled by the fluctuations of coinfection waves invading the single-disease territories. as the coinfection mortality grows, the dynamics of the interface network attain the scaling regime. we discover that organisms' infection risk is maximised if the coinfection increases the death due to disease in $30\\\\%$, and minimised as the network dynamics reach the scaling regime, with species populations being maximum. our conclusions may help ecologists understand the dynamics of epidemics and their impact on the stability of ecosystems.\",\n",
       "  'categories': 'q-bio.pe nlin.ps physics.bio-ph physics.soc-ph q-bio.qm',\n",
       "  'doi': '10.1063/5.0160753',\n",
       "  'created': '2023-04-30',\n",
       "  'updated': '',\n",
       "  'authors': ['j. menezes', 'e. rangel'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.00590'},\n",
       " {'title': 'multi-species prey-predator dynamics during a multi-strain pandemic',\n",
       "  'id': '2305.00704',\n",
       "  'abstract': \"small and large scale pandemics are a natural phenomenon repeatably appearing throughout history, causing ecological and biological shifts in ecosystems and a wide range of their habitats. these pandemics usually start with a single strain but shortly become multi-strain due to a mutation process of the pathogen causing the epidemic. in this study, we propose a novel eco-epidemiological model that captures multi-species prey-predator dynamics with a multi-strain pandemic. the proposed model extends and combines the lotka-volterra prey-predator model and the susceptible-infectious-recovered (sir) epidemiological model. we investigate the ecosystem's sensitivity and stability during such a multi-strain pandemic through extensive simulation relying on both synthetic cases as well as two real-world configurations. our results are aligned with known ecological and epidemiological findings, thus supporting the adequacy of the proposed model in realistically capturing the complex eco-epidemiological properties of the multi-species multi-strain pandemic dynamics.\",\n",
       "  'categories': 'q-bio.pe physics.soc-ph',\n",
       "  'doi': '10.1063/5.0154968',\n",
       "  'created': '2023-05-01',\n",
       "  'updated': '',\n",
       "  'authors': ['ariel alexi', 'ariel rosenfeld', 'teddy lazebnik'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.00704'},\n",
       " {'title': 'attractor identification in asynchronous boolean dynamics with network   reduction',\n",
       "  'id': '2305.01327',\n",
       "  'abstract': 'identification of attractors, that is, stable states and sustained oscillations, is an important step in the analysis of boolean models and exploration of potential variants. we describe an approach to the search for asynchronous cyclic attractors of boolean networks that exploits, in a novel way, the established technique of elimination of components. computation of attractors of simplified networks allows the identification of a limited number of candidate attractor states, which are then screened with techniques of reachability analysis combined with trap space computation. an implementation that brings together recently developed boolean network analysis tools, tested on biological models and random benchmark networks, shows the potential to significantly reduce running times.',\n",
       "  'categories': 'cs.dm q-bio.qm',\n",
       "  'doi': '10.1007/978-3-031-42697-1_14',\n",
       "  'created': '2023-05-02',\n",
       "  'updated': '',\n",
       "  'authors': ['elisa tonello', 'loïc paulevé'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.01327'},\n",
       " {'title': 'systematic review of methods and prognostic value of mitotic activity.   part 1: feline tumors',\n",
       "  'id': '2305.01403',\n",
       "  'abstract': 'increased proliferation is a key driver of tumorigenesis, and quantification of mitotic activity is a standard task for prognostication. the goal of this systematic review is scholarly analysis of all available references on mitotic activity in feline tumors, and to provide an overview of the measuring methods and prognostic value. a systematic literature search in pubmed and scopus and a manual search in google scholar was conducted. all articles on feline tumors that correlated mitotic activity with patient outcome were identified. data analysis revealed that of the eligible 42 articles, the mitotic count (mc, mitotic figures per tumor area) was evaluated in 39 instances and the mitotic index (mi, mitotic figures per tumor cells) in three instances. the risk of bias was considered high for most studies (26/42, 62%) based on small study populations, insufficient details of the mc/mi methods, and lack of statistical measures for diagnostic accuracy or effect on outcome. the mc/mi methods varied markedly between studies. a significant association of the mc with survival was determined in 21/29 (72%) studies, while one study found an inverse effect. there were three tumor types with at least four studies and a prognostic association was found in 5/6 studies on mast cell tumors, 5/5 on mammary tumors and 3/4 on soft tissue sarcomas. the mi was shown to correlate with survival by two research groups, however a comparison to the mc was not conducted. an updated systematic review will be needed with of new literature for different tumor types.',\n",
       "  'categories': 'q-bio.sc',\n",
       "  'doi': '10.1177/03009858241239566',\n",
       "  'created': '2023-05-02',\n",
       "  'updated': '',\n",
       "  'authors': ['christof a. bertram', 'taryn a. donovan', 'alexander bartel'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.01403'},\n",
       " {'title': 'cancer-inspired genomics mapper model for the generation of synthetic   dna sequences with desired genomics signatures',\n",
       "  'id': '2305.01475',\n",
       "  'abstract': 'genome data are crucial in modern medicine, offering significant potential for diagnosis and treatment. thanks to technological advancements, many millions of healthy and diseased genomes have already been sequenced; however, obtaining the most suitable data for a specific study, and specifically for validation studies, remains challenging with respect to scale and access. therefore, in silico genomics sequence generators have been proposed as a possible solution. however, the current generators produce inferior data using mostly shallow (stochastic) connections, detected with limited computational complexity in the training data. this means they do not take the appropriate biological relations and constraints, that originally caused the observed connections, into consideration. to address this issue, we propose cancer-inspired genomics mapper model (cgmm), that combines genetic algorithm (ga) and deep learning (dl) methods to tackle this challenge. cgmm mimics processes that generate genetic variations and mutations to transform readily available control genomes into genomes with the desired phenotypes. we demonstrate that cgmm can generate synthetic genomes of selected phenotypes such as ancestry and cancer that are indistinguishable from real genomes of such phenotypes, based on unsupervised clustering. our results show that cgmm outperforms four current state-of-the-art genomics generators on two different tasks, suggesting that cgmm will be suitable for a wide range of purposes in genomic medicine, especially for much-needed validation studies.',\n",
       "  'categories': 'q-bio.gn cs.lg cs.ne',\n",
       "  'doi': '10.1016/j.compbiomed.2023.107221',\n",
       "  'created': '2023-05-01',\n",
       "  'updated': '',\n",
       "  'authors': ['teddy lazebnik', 'liron simon-keren'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.01475'},\n",
       " {'title': 'brainnpt: pre-training of transformer networks for brain network   classification',\n",
       "  'id': '2305.01666',\n",
       "  'abstract': 'deep learning methods have advanced quickly in brain imaging analysis over the past few years, but they are usually restricted by the limited labeled data. pre-trained model on unlabeled data has presented promising improvement in feature learning in many domains, including natural language processing and computer vision. however, this technique is under-explored in brain network analysis. in this paper, we focused on pre-training methods with transformer networks to leverage existing unlabeled data for brain functional network classification. first, we proposed a transformer-based neural network, named as brainnpt, for brain functional network classification. the proposed method leveraged <cls> token as a classification embedding vector for the transformer model to effectively capture the representation of brain network. second, we proposed a pre-training framework for brainnpt model to leverage unlabeled brain network data to learn the structure information of brain networks. the results of classification experiments demonstrated the brainnpt model without pre-training achieved the best performance with the state-of-the-art models, and the brainnpt model with pre-training strongly outperformed the state-of-the-art models. the pre-training brainnpt model improved 8.75% of accuracy compared with the model without pre-training. we further compared the pre-training strategies, analyzed the influence of the parameters of the model, and interpreted the trained model.',\n",
       "  'categories': 'q-bio.nc cs.lg cs.ne',\n",
       "  'doi': '10.1109/tnsre.2024.3434343',\n",
       "  'created': '2023-05-02',\n",
       "  'updated': '2023-08-02',\n",
       "  'authors': ['jinlong hu', 'yangmin huang', 'nan wang', 'shoubin dong'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.01666'},\n",
       " {'title': 'complexity and enumeration in models of genome rearrangement',\n",
       "  'id': '2305.01851',\n",
       "  'abstract': \"in this paper, we examine the computational complexity of enumeration in certain genome rearrangement models. we first show that the pairwise rearrangement problem in the single cut-and-join model (bergeron, medvedev, & stoye, j. comput. biol. 2010) is $\\\\#\\\\textsf{p}$-complete under polynomial-time turing reductions. next, we show that in the single cut or join model (feijao & meidanis, ieee acm trans. comp. biol. bioinf. 2011), the problem of enumerating all medians ($\\\\#$median) is logspace-computable ($\\\\textsf{fl}$), improving upon the previous polynomial-time ($\\\\textsf{fp}$) bound of mikl\\\\'os & smith (recomb 2015).\",\n",
       "  'categories': 'q-bio.gn cs.cc math.co',\n",
       "  'doi': '10.1016/j.tcs.2024.114880',\n",
       "  'created': '2023-05-02',\n",
       "  'updated': '2024-09-08',\n",
       "  'authors': ['lora bailey',\n",
       "   'heather smith blake',\n",
       "   'garner cochran',\n",
       "   'nathan fox',\n",
       "   'michael levet',\n",
       "   'reem mahmoud',\n",
       "   'elizabeth matson',\n",
       "   'inne singgih',\n",
       "   'grace stadnyk',\n",
       "   'xinyi wang',\n",
       "   'alexander wiedemann'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.01851'},\n",
       " {'title': 'the inner screen model of consciousness: applying the free energy   principle directly to the study of conscious experience',\n",
       "  'id': '2305.02205',\n",
       "  'abstract': 'this paper presents a model of consciousness that follows directly from the free-energy principle (fep). we first rehearse the classical and quantum formulations of the fep. in particular, we consider the inner screen hypothesis that follows from the quantum information theoretic version of the fep. we then review applications of the fep to the known sparse (nested and hierarchical) neuro-anatomy of the brain. we focus on the holographic structure of the brain, and how this structure supports (overt and covert) action.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-15',\n",
       "  'updated': '2024-01-02',\n",
       "  'authors': ['maxwell j. d. ramstead',\n",
       "   'mahault albarracin',\n",
       "   'alex kiefer',\n",
       "   'brennan klein',\n",
       "   'chris fields',\n",
       "   'karl friston',\n",
       "   'adam safron'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.02205'},\n",
       " {'title': 'diffuse and localized functional dysconnectivity in schizophrenia: a   bootstrapped top-down approach',\n",
       "  'id': '2305.02369',\n",
       "  'abstract': \"schizophrenia (sz) is a brain disorder leading to detached mind's normally integrated processes. hence, the exploration of the symptoms in relation to functional connectivity (fc) had great relevance in the field. fc can be investigated on different levels, going from global features to single edges between regions, revealing diffuse and localized dysconnection patterns. in this context, sz is characterized by a diverse global integration with reduced connectivity in specific areas of the default mode network (dmn). however, the assessment of fc presents various sources of uncertainty. this study proposes a multi-level approach for more robust group-comparison. fc between 74 aal brain areas of 15 healthy controls (hc) and 12 sz subjects were used. multi-level analyses and graph topological indexes evaluation were carried out by the previously published spider-net tool. robustness was augmented by bootstrapped (boot) data and the stability was evaluated by removing one (rst1) or two subjects (rst2). the dmn subgraph was evaluated, toegether with overall local indexes and connection weights to enhance common activations/deactivations. at a global level, expected trends were found. the robustness assessment tests highlighted more stable results for boot compared to the direct data testing. conversely, significant results were found in the analysis at lower levels. the dmn highlighted reduced connectivity and strength as well as increased deactivation in the sz group. at local level, 13 areas were found to be significantly different ($p<0.05$), highlighting a greater divergence in the frontal lobe. these results were confirmed analyzing the negative edges, suggesting inverted connectivity between prefronto-temporal areas. in conclusion, multi-level analysis supported by boot is highly recommended, especially when diffuse and localized dysconnections must be investigated in limited samples.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.3233/fi-222157',\n",
       "  'created': '2023-05-03',\n",
       "  'updated': '2023-07-18',\n",
       "  'authors': ['davide coluzzi', 'giuseppe baselli'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.02369'},\n",
       " {'title': 'contrastive losses as generalized models of global epistasis',\n",
       "  'id': '2305.03136',\n",
       "  'abstract': 'fitness functions map large combinatorial spaces of biological sequences to properties of interest. inferring these multimodal functions from experimental data is a central task in modern protein engineering. global epistasis models are an effective and physically-grounded class of models for estimating fitness functions from observed data. these models assume that a sparse latent function is transformed by a monotonic nonlinearity to emit measurable fitness. here we demonstrate that minimizing supervised contrastive loss functions, such as the bradley-terry loss, is a simple and flexible technique for extracting the sparse latent function implied by global epistasis. we argue by way of a fitness-epistasis uncertainty principle that the nonlinearities in global epistasis models can produce observed fitness functions that do not admit sparse representations, and thus may be inefficient to learn from observations when using a mean squared error (mse) loss (a common practice). we show that contrastive losses are able to accurately estimate a ranking function from limited data even in regimes where mse is ineffective and validate the practical utility of this insight by demonstrating that contrastive loss functions result in consistently improved performance on benchmark tasks.',\n",
       "  'categories': 'q-bio.pe cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-04',\n",
       "  'updated': '2024-10-15',\n",
       "  'authors': ['david h. brookes', 'jakub otwinowski', 'sam sinai'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.03136'},\n",
       " {'title': 'impact of reproduction-mobility trade-off on biodiversity in   rock-paper-scissors models in changing environmental conditions',\n",
       "  'id': '2305.03199',\n",
       "  'abstract': \"we investigate a tritrophic system in which organisms' energy depletion, resulting from failed selection attempts, leads to a partial loss of capacity to win the cyclic spatial game. the energy required to maintain optimal organism fitness may be impacted by changes in environmental conditions, increasing the death risk due to accelerated deterioration of health. we simulate the evolutionary behavioural strategy performed by individuals of one species, which consists in balancing efforts dedicated to reproduction and mobility to minimise the chances of death by lack of energy. we show that the unevenness introduced by the trade-off strategy unbalances the rock-paper-scissors model, with the predominant species profiting from enemies' lower birth rate of enemies. quantifying the spatial patterns, we demonstrate that the characteristic length scale of single-species domains decreases as energy loss accelerates due to environmental changes. the erosion in the spatial patterns provoked by the reproduction-mobility trade-off benefits biodiversity, with coexistence probability rising for faster energy depletion and higher trade-off factors. the findings have implications for ecologists seeking to understand the impact of survival behaviour on biodiversity promotion.\",\n",
       "  'categories': 'q-bio.pe nlin.ao nlin.ps physics.bio-ph',\n",
       "  'doi': '10.1088/1402-4896/ad3177',\n",
       "  'created': '2023-05-04',\n",
       "  'updated': '',\n",
       "  'authors': ['j. menezes', 'e. rangel'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.03199'},\n",
       " {'title': 'working memory capacity of chatgpt: an empirical study',\n",
       "  'id': '2305.03731',\n",
       "  'abstract': \"working memory is a critical aspect of both human intelligence and artificial intelligence, serving as a workspace for the temporary storage and manipulation of information. in this paper, we systematically assess the working memory capacity of chatgpt, a large language model developed by openai, by examining its performance in verbal and spatial n-back tasks under various conditions. our experiments reveal that chatgpt has a working memory capacity limit strikingly similar to that of humans. furthermore, we investigate the impact of different instruction strategies on chatgpt's performance and observe that the fundamental patterns of a capacity limit persist. from our empirical findings, we propose that n-back tasks may serve as tools for benchmarking the working memory capacity of large language models and hold potential for informing future efforts aimed at enhancing ai working memory.\",\n",
       "  'categories': 'cs.ai cs.cl q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-30',\n",
       "  'updated': '2024-02-01',\n",
       "  'authors': ['dongyu gong', 'xingchen wan', 'dingmin wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.03731'},\n",
       " {'title': 'mechanisms of dna-mediated allostery',\n",
       "  'id': '2305.03962',\n",
       "  'abstract': 'proteins often regulate their activities via allostery - or action at a distance - in which the binding of a ligand at one binding site influences the affinity for another ligand at a distal site. although less studied than in proteins, allosteric effects have been observed in experiments with dna as well. in these experiments two or more proteins bind at distinct dna sites and interact indirectly with each other, via a mechanism mediated by the linker dna molecule. we develop a mechanical model of dna/protein interactions which predicts three distinct mechanisms of allostery. two of these involve an enthalpy-mediated allostery, while a third mechanism is entropy driven. we analyze experiments of dna allostery and highlight the distinctive signatures allowing one to identify which of the proposed mechanisms best fits the data.',\n",
       "  'categories': 'cond-mat.stat-mech q-bio.bm',\n",
       "  'doi': '10.1103/physrevlett.131.238402',\n",
       "  'created': '2023-05-06',\n",
       "  'updated': '2024-01-10',\n",
       "  'authors': ['midas segers',\n",
       "   'aderik voorspoels',\n",
       "   'takahiro sakaue',\n",
       "   'enrico carlon'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.03962'},\n",
       " {'title': 'combinatorial topological models for phylogenetic networks and the   mergegram invariant',\n",
       "  'id': '2305.04860',\n",
       "  'abstract': \"mutations of genetic sequences are often accompanied by their recombinations, known as phylogenetic networks. these networks are typically reconstructed from coalescent processes that may arise from optimal merging or fitting together a given set of phylogenetic trees. nakhleh formulated the phylogenetic network reconstruction problem (pnrp): given a family of phylogenetic trees over a common set of taxa, is there a unique minimal phylogenetic network whose set of spanning trees contains the family?   inspired by ideas from topological data analysis (tda), we devise lattice-diagram models for phylogenetic networks and filtrations, the cliquegram and the facegram, both generalizing the dendrogram (filtered partition) model of phylogenetic trees. both models allow us to solve the pnrp rigorously. the solutions are obtained by taking the join of the dendrograms on the lattice of cliquegrams or facegrams. furthermore, computing the join-facegram is polynomial in the size and number of the input trees.   cliquegrams and facegrams can be challenging to work with when the number of taxa is large. we propose a topological invariant of facegrams and filtrations, called the mergegram, by extending a construction by elkin and kurlin defined on dendrograms. we show that the mergegram is invariant of weak equivalences of filtrations which, in turn, implies that it is a 1-lipschitz stable invariant with respect to m\\\\'emoli's tripod distance. the mergegram, can be used as a computable proxy for phylogenetic networks and filtrations of datasets. we illustrate the utility of these new tda-concepts to phylogenetics, by performing experiments with artificial and biological data.\",\n",
       "  'categories': 'math.at q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-08',\n",
       "  'updated': '2024-08-27',\n",
       "  'authors': ['paweł dłotko', 'jan felix senge', 'anastasios stefanou'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.04860'},\n",
       " {'title': 'dynamical properties and mechanisms of metastability: a perspective in   neuroscience',\n",
       "  'id': '2305.05328',\n",
       "  'abstract': 'metastability, characterized by a variability of regimes in time, is a ubiquitous type of neural dynamics. it has been formulated in many different ways in the neuroscience literature, however, which may cause some confusion. in this perspective, we discuss metastability from the point of view of dynamical systems theory. we extract from the literature a very simple but general definition through the concept of metastable regimes as long-lived but transient epochs of activity with unique dynamical properties. this definition serves as an umbrella term that encompasses formulations from other works, and readily connects to concepts from dynamical systems theory. this allows us to examine general dynamical properties of metastable regimes, propose in a didactic manner several dynamics-based mechanisms that generate them, and discuss a theoretical tool to characterize them quantitatively. this perspective leads to insights that help to address issues debated in the literature and also suggest pathways for future research.',\n",
       "  'categories': 'q-bio.nc math.ds nlin.cd',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-09',\n",
       "  'updated': '2024-05-21',\n",
       "  'authors': ['kalel l. rossi',\n",
       "   'roberto c. budzinski',\n",
       "   'everton s. medeiros',\n",
       "   'bruno r. r. boaretto',\n",
       "   'lyle muller',\n",
       "   'ulrike feudel'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.05328'},\n",
       " {'title': 'chromatin remodeling due to transient-link-and-pass activity enhances   subnuclear dynamics',\n",
       "  'id': '2305.05521',\n",
       "  'abstract': 'spatiotemporal coordination of chromatin and subnuclear compartments is crucial for cells. numerous enzymes act inside nucleus\\\\textemdash some of those transiently link and pass two chromatin segments. here we study how such an active perturbation affects fluctuating dynamics of an inclusion in the chromatic medium. using numerical simulations and a versatile effective model, we categorize inclusion dynamics into three distinct modes. the transient-link-and-pass activity speeds up inclusion dynamics by affecting a slow mode related to chromatin remodeling, viz., size and shape of the chromatin meshes.',\n",
       "  'categories': 'physics.bio-ph cond-mat.soft q-bio.sc',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-09',\n",
       "  'updated': '2024-01-17',\n",
       "  'authors': ['rakesh das',\n",
       "   'takahiro sakaue',\n",
       "   'g. v. shivashankar',\n",
       "   'jacques prost',\n",
       "   'tetsuya hiraiwa'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.05521'},\n",
       " {'title': 'slicertms: real-time visualization of transcranial magnetic stimulation   for mental health treatment',\n",
       "  'id': '2305.06459',\n",
       "  'abstract': \"we present a real-time visualization system for transcranial magnetic stimulation (tms), a non-invasive neuromodulation technique for treating various brain disorders and mental health diseases. our solution targets the current challenges of slow and labor-intensive practices in treatment planning. integrating deep learning (dl), our system rapidly predicts electric field (e-field) distributions in 0.2 seconds for precise and effective brain stimulation. the core advancement lies in our tool's real-time neuronavigation visualization capabilities, which support clinicians in making more informed decisions quickly and effectively. we assess our system's performance through three studies: first, a real-world use case scenario in a clinical setting, providing concrete feedback on applicability and usability in a practical environment. second, a comparative analysis with another tms tool focusing on computational efficiency across various hardware platforms. lastly, we conducted an expert user study to measure usability and influence in optimizing tms treatment planning. the system is openly available for community use and further development on github: \\\\url{https://github.com/lorifranke/slicertms}.\",\n",
       "  'categories': 'eess.sp cs.gr cs.hc eess.iv q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-10',\n",
       "  'updated': '2024-03-12',\n",
       "  'authors': ['loraine franke',\n",
       "   'tae young park',\n",
       "   'jie luo',\n",
       "   'yogesh rathi',\n",
       "   'steve pieper',\n",
       "   'lipeng ning',\n",
       "   'daniel haehn'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.06459'},\n",
       " {'title': 'a platform for the biomedical application of large language models',\n",
       "  'id': '2305.06488',\n",
       "  'abstract': 'current-generation large language models (llms) have stirred enormous interest in recent months, yielding great potential for accessibility and automation, while simultaneously posing significant challenges and risk of misuse. to facilitate interfacing with llms in the biomedical space, while at the same time safeguarding their functionalities through sensible constraints, we propose a dedicated, open-source framework: biochatter. based on open-source software packages, we synergise the many functionalities that are currently developing around llms, such as knowledge integration / retrieval-augmented generation, model chaining, and benchmarking, resulting in an easy-to-use and inclusive framework for application in many use cases of biomedicine. we focus on robust and user-friendly implementation, including ways to deploy privacy-preserving local open-source llms. we demonstrate use cases via two multi-purpose web apps (https://chat.biocypher.org), and provide documentation, support, and an open community.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-10',\n",
       "  'updated': '2024-02-17',\n",
       "  'authors': ['sebastian lobentanzer',\n",
       "   'shaohong feng',\n",
       "   'the biochatter consortium',\n",
       "   'andreas maier',\n",
       "   'cankun wang',\n",
       "   'jan baumbach',\n",
       "   'nils krehl',\n",
       "   'qin ma',\n",
       "   'julio saez-rodriguez'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.06488'},\n",
       " {'title': 'parasite infection in a cell population: role of the partitioning kernel',\n",
       "  'id': '2305.06962',\n",
       "  'abstract': 'we consider a cell population subject to a parasite infection. cells divide at a constant rate and, at division, share the parasites they contain between their two daughter cells. the sharing may be asymmetrical, and its law may depend on the quantity of parasites in the mother. cells die at a rate which may depend on the quantity of parasites they carry, and are also killed when this quantity explodes. we study the survival of the cell population as well as the mean quantity of parasites in the cells, and focus on the role of the parasites partitioning kernel at division.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-11',\n",
       "  'updated': '2024-06-17',\n",
       "  'authors': ['aline marguet', 'charline smadi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.06962'},\n",
       " {'title': 'cremp: conformer-rotamer ensembles of macrocyclic peptides for machine   learning',\n",
       "  'id': '2305.08057',\n",
       "  'abstract': 'computational and machine learning approaches to model the conformational landscape of macrocyclic peptides have the potential to enable rational design and optimization. however, accurate, fast, and scalable methods for modeling macrocycle geometries remain elusive. recent deep learning approaches have significantly accelerated protein structure prediction and the generation of small-molecule conformational ensembles, yet similar progress has not been made for macrocyclic peptides due to their unique properties. here, we introduce cremp, a resource generated for the rapid development and evaluation of machine learning models for macrocyclic peptides. cremp contains 36,198 unique macrocyclic peptides and their high-quality structural ensembles generated using the conformer-rotamer ensemble sampling tool (crest). altogether, this new dataset contains nearly 31.3 million unique macrocycle geometries, each annotated with energies derived from semi-empirical extended tight-binding (xtb) dft calculations. additionally, we include 3,258 macrocycles with reported passive permeability data to couple conformational ensembles to experiment. we anticipate that this dataset will enable the development of machine learning models that can improve peptide design and optimization for novel therapeutics.',\n",
       "  'categories': 'q-bio.bm cs.lg physics.chem-ph',\n",
       "  'doi': '10.1038/s41597-024-03698-y',\n",
       "  'created': '2023-05-13',\n",
       "  'updated': '2024-08-09',\n",
       "  'authors': ['colin a. grambow',\n",
       "   'hayley weir',\n",
       "   'christian n. cunningham',\n",
       "   'tommaso biancalani',\n",
       "   'kangway v. chuang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.08057'},\n",
       " {'title': 'introduction to dynamical mean-field theory of randomly connected neural   networks with bidirectionally correlated couplings',\n",
       "  'id': '2305.08459',\n",
       "  'abstract': 'dynamical mean-field theory is a powerful physics tool used to analyze the typical behavior of neural networks, where neurons can be recurrently connected, or multiple layers of neurons can be stacked. however, it is not easy for beginners to access the essence of this tool and the underlying physics. here, we give a pedagogical introduction of this method in a particular example of random neural networks, where neurons are randomly and fully connected by correlated synapses and therefore the network exhibits rich emergent collective dynamics. we also review related past and recent important works applying this tool. in addition, a physically transparent and alternative method, namely the dynamical cavity method, is also introduced to derive exactly the same results. the numerical implementation of solving the integro-differential mean-field equations is also detailed, with an illustration of exploring the fluctuation dissipation theorem.',\n",
       "  'categories': 'cond-mat.dis-nn cond-mat.stat-mech cs.lg q-bio.nc',\n",
       "  'doi': '10.21468/scipostphyslectnotes.79',\n",
       "  'created': '2023-05-15',\n",
       "  'updated': '2023-10-07',\n",
       "  'authors': ['wenxuan zou', 'haiping huang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.08459'},\n",
       " {'title': 'smoothing in linear multicompartment biological processes subject to   stochastic input',\n",
       "  'id': '2305.09004',\n",
       "  'abstract': 'many physical and biological systems rely on the progression of material through multiple independent stages. in viral replication, for example, virions enter a cell to undergo a complex process comprising several disparate stages before the eventual accumulation and release of replicated virions. while such systems may have some control over the internal dynamics that make up this progression, a challenge for many is to regulate behaviour under what are often highly variable external environments acting as system inputs. in this work, we study a simple analogue of this problem through a linear multicompartment model subject to a stochastic input in the form of a mean-reverting ornstein-uhlenbeck process, a type of gaussian process. by expressing the system as a multidimensional gaussian process, we derive several closed-form analytical results relating to the covariances and autocorrelations of the system, quantifying the smoothing effect discrete compartments afford multicompartment systems. semi-analytical results demonstrate that feedback and feedforward loops can enhance system robustness, and simulation results probe the intractable problem of the first passage time distribution, which has specific relevance to eventual cell lysis in the viral replication cycle. finally, we demonstrate that the smoothing seen in the process is a consequence of the discreteness of the system, and does not manifest in system with continuous transport. while we make progress through analysis of a simple linear problem, many of our insights are applicable more generally, and our work enables future analysis into multicompartment processes subject to stochastic inputs.',\n",
       "  'categories': 'q-bio.mn math.pr q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-03',\n",
       "  'updated': '2024-04-02',\n",
       "  'authors': ['alexander p browning',\n",
       "   'adrianne l jenner',\n",
       "   'ruth e baker',\n",
       "   'philip k maini'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.09004'},\n",
       " {'title': 'intrinsic statistical separation of subpopulations in heterogeneous   collective motion via dimensionality reduction',\n",
       "  'id': '2305.09189',\n",
       "  'abstract': \"collective motion of locally interacting agents is found ubiquitously throughout nature. the inability to probe individuals has driven longstanding interest in the development of methods for inferring the underlying interactions. in the context of heterogeneous collectives, where the population consists of individuals driven by different interactions, existing approaches require some knowledge about the heterogeneities or underlying interactions. here, we investigate the feasibility of identifying the identities in a heterogeneous collective without such prior knowledge. we numerically explore the behavior of a heterogeneous vicsek model and find sufficiently long trajectories intrinsically cluster in a pca-based dimensionally reduced model-agnostic description of the data. we identify how heterogeneities in each parameter in the model (interaction radius, noise, population proportions) dictate this clustering. finally, we show the generality of this phenomenon by finding similar behavior in a heterogeneous d'orsogna model. altogether, our results establish and quantify the intrinsic model-agnostic statistical disentanglement of identities in heterogeneous collectives.\",\n",
       "  'categories': 'q-bio.qm nlin.ao physics.bio-ph',\n",
       "  'doi': '10.1103/physreve.109.014403',\n",
       "  'created': '2023-05-16',\n",
       "  'updated': '2023-11-08',\n",
       "  'authors': ['pei tan', 'christopher e miles'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.09189'},\n",
       " {'title': 'cross-gate mlp with protein complex invariant embedding is a one-shot   antibody designer',\n",
       "  'id': '2305.09480',\n",
       "  'abstract': 'antibodies are crucial proteins produced by the immune system in response to foreign substances or antigens. the specificity of an antibody is determined by its complementarity-determining regions (cdrs), which are located in the variable domains of the antibody chains and form the antigen-binding site. previous studies have utilized complex techniques to generate cdrs, but they suffer from inadequate geometric modeling. moreover, the common iterative refinement strategies lead to an inefficient inference. in this paper, we propose a \\\\textit{simple yet effective} model that can co-design 1d sequences and 3d structures of cdrs in a one-shot manner. to achieve this, we decouple the antibody cdr design problem into two stages: (i) geometric modeling of protein complex structures and (ii) sequence-structure co-learning. we develop a novel macromolecular structure invariant embedding, typically for protein complexes, that captures both intra- and inter-component interactions among the backbone atoms, including c$\\\\alpha$, n, c, and o atoms, to achieve comprehensive geometric modeling. then, we introduce a simple cross-gate mlp for sequence-structure co-learning, allowing sequence and structure representations to implicitly refine each other. this enables our model to design desired sequences and structures in a one-shot manner. extensive experiments are conducted to evaluate our results at both the sequence and structure levels, which demonstrate that our model achieves superior performance compared to the state-of-the-art antibody cdr design methods.',\n",
       "  'categories': 'q-bio.bm cs.ai cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-04-21',\n",
       "  'updated': '2024-01-10',\n",
       "  'authors': ['cheng tan',\n",
       "   'zhangyang gao',\n",
       "   'lirong wu',\n",
       "   'jun xia',\n",
       "   'jiangbin zheng',\n",
       "   'xihong yang',\n",
       "   'yue liu',\n",
       "   'bozhen hu',\n",
       "   'stan z. li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.09480'},\n",
       " {'title': 'upper bounds for integrated information',\n",
       "  'id': '2305.09826',\n",
       "  'abstract': 'originally developed as a theory of consciousness, integrated information theory provides a mathematical framework to quantify the causal irreducibility of systems and subsets of units in the system. specifically, mechanism integrated information quantifies how much of the causal powers of a subset of units in a state, also referred to as a mechanism, cannot be accounted for by its parts. if the causal powers of the mechanism can be fully explained by its parts, it is reducible and its integrated information is zero. here, we study the upper bound of this measure and how it is achieved. we study mechanisms in isolation, groups of mechanisms, and groups of causal relations among mechanisms. we put forward new theoretical results that show mechanisms that share parts with each other cannot all achieve their maximum. we also introduce techniques to design systems that can maximize the integrated information of a subset of their mechanisms or relations. our results can potentially be used to exploit the symmetries and constraints to reduce the computations significantly and to compare different connectivity profiles in terms of their maximal achievable integrated information.',\n",
       "  'categories': 'q-bio.nc math.ds math.pr',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-16',\n",
       "  'updated': '2024-04-21',\n",
       "  'authors': ['alireza zaeemzadeh', 'giulio tononi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.09826'},\n",
       " {'title': 'co-mapping cellular content and extracellular matrix with hemodynamics   in intact arterial tissues using scanning immunofluorescent multiphoton   microscopy',\n",
       "  'id': '2305.10253',\n",
       "  'abstract': 'deviation of blood flow from an optimal range is known to be associated with the initiation and progression of vascular pathologies. important open questions remain about how the abnormal flow drives specific wall changes in pathologies such as cerebral aneurysms where the flow is highly heterogeneous and complex. this knowledge gap precludes the clinical use of readily available flow data to predict outcomes and improve treatment of these diseases. as both flow and the pathological wall changes are spatially heterogeneous, a crucial requirement for progress in this area is a methodology for co-mapping local data from vascular wall biology with local hemodynamic data. in this study, we developed an imaging pipeline to address this pressing need. a protocol that employs scanning multiphoton microscopy was designed to obtain 3d data sets for smooth muscle actin, collagen and elastin in intact vascular specimens. a cluster analysis was developed to objectively categorize the smooth muscle cells (smc) across the vascular specimen based on smc density. in the final step in this pipeline, the location specific categorization of smc, along with wall thickness was co-mapped with patient specific hemodynamic results, enabling direct quantitative comparison of local flow and wall biology in 3d intact specimens.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-17',\n",
       "  'updated': '2024-01-17',\n",
       "  'authors': ['yasutaka tobe',\n",
       "   'anne robertson',\n",
       "   'mehdi ramezanpour',\n",
       "   'juan cebral',\n",
       "   'simon watkins',\n",
       "   'fady charbel',\n",
       "   'sepideh amin-hanjani',\n",
       "   'alexander yu',\n",
       "   'boyle cheng',\n",
       "   'henry woo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.10253'},\n",
       " {'title': 'predicting side effect of drug molecules using recurrent neural networks',\n",
       "  'id': '2305.10473',\n",
       "  'abstract': 'identification and verification of molecular properties such as side effects is one of the most important and time-consuming steps in the process of molecule synthesis. for example, failure to identify side effects before submission to regulatory groups can cost millions of dollars and months of additional research to the companies. failure to identify side effects during the regulatory review can also cost lives. the complexity and expense of this task have made it a candidate for a machine learning-based solution. prior approaches rely on complex model designs and excessive parameter counts for side effect predictions. we believe reliance on complex models only shifts the difficulty away from chemists rather than alleviating the issue. implementing large models is also expensive without prior access to high-performance computers. we propose a heuristic approach that allows for the utilization of simple neural networks, specifically the recurrent neural network, with a 98+% reduction in the number of required parameters compared to available large language models while still obtaining near identical results as top-performing models.',\n",
       "  'categories': 'q-bio.qm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-17',\n",
       "  'updated': '2024-04-10',\n",
       "  'authors': ['collin beaudoin', 'koustubh phalak', 'swaroop ghosh'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.10473'},\n",
       " {'title': 'a benchmark for computational analysis of animal behavior, using   animal-borne tags',\n",
       "  'id': '2305.10740',\n",
       "  'abstract': \"animal-borne sensors (`bio-loggers') can record a suite of kinematic and environmental data, which are used to elucidate animal ecophysiology and improve conservation efforts. machine learning techniques are used for interpreting the large amounts of data recorded by bio-loggers, but there exists no common framework for comparing the different machine learning techniques in this domain. this makes it difficult to, for example, identify patterns in what works well for machine learning-based analysis of bio-logger data. it also makes it difficult to evaluate the effectiveness of novel methods developed by the machine learning community.   to address this, we present the bio-logger ethogram benchmark (bebe), a collection of datasets with behavioral annotations, as well as a modeling task and evaluation metrics. bebe is to date the largest, most taxonomically diverse, publicly available benchmark of this type. using bebe, we compare the performance of deep and classical machine learning methods for identifying animal behaviors based on bio-logger data. as an example usage of bebe, we test an approach based on self-supervised learning. to apply this approach to animal behavior classification, we adapt a deep neural network pre-trained with 700,000 hours of data collected from human wrist-worn accelerometers.   we find that deep neural networks out-perform the classical machine learning methods we tested across all nine datasets in bebe. we additionally find that the approach based on self-supervised learning out-performs the alternatives we tested, especially in settings when there is a low amount of training data available. in light of this, we are able to make concrete suggestions for designing studies that rely on machine learning to infer behavior from bio-logger data. datasets and code are available at https://github.com/earthspecies/bebe.\",\n",
       "  'categories': 'cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-18',\n",
       "  'updated': '2024-09-27',\n",
       "  'authors': ['benjamin hoffman',\n",
       "   'maddie cusimano',\n",
       "   'vittorio baglione',\n",
       "   'daniela canestrari',\n",
       "   'damien chevallier',\n",
       "   'dominic l. desantis',\n",
       "   'lorène jeantet',\n",
       "   'monique a. ladds',\n",
       "   'takuya maekawa',\n",
       "   'vicente mata-silva',\n",
       "   'víctor moreno-gonzález',\n",
       "   'anthony pagano',\n",
       "   'eva trapote',\n",
       "   'outi vainio',\n",
       "   'antti vehkaoja',\n",
       "   'ken yoda',\n",
       "   'katherine zacarian',\n",
       "   'ari friedlaender'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.10740'},\n",
       " {'title': 'cs-trd: a cross sections tree ring detection method',\n",
       "  'id': '2305.10809',\n",
       "  'abstract': \"this work describes a tree ring detection method for complete cross-sections of trees (cs-trd) that detects, processes and connects edges corresponding to the tree's growth rings. the method depends on the parameters for the canny devernay edge detector (sigma), a resize factor, the number of rays, and the pith location. the first five are fixed by default. the pith location can be marked manually or using an automatic pith detection algorithm. besides the pith localization, cs-trd is fully automated and achieves an f-score of 89% in the urudendro dataset (of pinus taeda) and 97% in the kennel dataset (of abies alba) without specialized hardware requirements.\",\n",
       "  'categories': 'cs.cv q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-18',\n",
       "  'updated': '2024-08-13',\n",
       "  'authors': ['henry marichal', 'diego passarella', 'gregory randall'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.10809'},\n",
       " {'title': 'the generalized hierarchical gaussian filter',\n",
       "  'id': '2305.10937',\n",
       "  'abstract': 'hierarchical bayesian models of perception and learning feature prominently in contemporary cognitive neuroscience where, for example, they inform computational concepts of mental disorders. this includes predictive coding and hierarchical gaussian filtering (hgf), which differ in the nature of hierarchical representations. predictive coding assumes that higher levels in a given hierarchy influence the state (value) of lower levels. in hgf, however, higher levels determine the rate of change at lower levels. here, we extend the space of generative models underlying hgf to include a form of nonlinear hierarchical coupling between state values akin to predictive coding and artificial neural networks in general. we derive the update equations corresponding to this generalization of hgf and conceptualize them as connecting a network of (belief) nodes where parent nodes either predict the state of child nodes or their rate of change. this enables us to (1) create modular architectures with generic computational steps in each node of the network, and (2) disclose the hierarchical message passing implied by generalized hgf models and to compare this to comparable schemes under predictive coding. we find that the algorithmic architecture instantiated by the generalized hgf is largely compatible with that of predictive coding but extends it with some unique predictions which arise from precision and volatility related computations. our developments enable highly flexible implementations of hierarchical bayesian models for empirical data analysis and are available as open source software.',\n",
       "  'categories': 'cs.ne q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-18',\n",
       "  'updated': '2024-09-04',\n",
       "  'authors': ['lilian aline weber',\n",
       "   'peter thestrup waade',\n",
       "   'nicolas legrand',\n",
       "   'anna hedvig møller',\n",
       "   'klaas enno stephan',\n",
       "   'christoph mathys'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.10937'},\n",
       " {'title': 'multi-relevance: coexisting but distinct notions of scale in large   systems',\n",
       "  'id': '2305.11009',\n",
       "  'abstract': \"renormalization group (rg) methods are emerging as tools in biology and computer science to support the search for simplifying structure in distributions over high-dimensional spaces. we show that mixture models can be thought of as having multiple coexisting, exactly independent rg flows, each with its own notion of scale. we define this property as ``multi-relevance''. as an example, we construct a model that has two distinct notions of scale, each corresponding to the state of an unobserved categorical variable. in the regime where this latent variable can be inferred using a linear classifier, the vertex expansion approach in non-perturbative rg can be applied successfully but will give different answers depending the choice of expansion point in state space. in the regime where linear estimation of the latent state fails, we show that the vertex expansion predicts a decrease in the total number of relevant couplings from four to three and does not admit a good polynomial truncation scheme. this indicates oversimplification. one consequence of this is that principal component analysis (pca) may be a poor choice of coarse-graining scheme in multi-relevant systems, since it imposes a notion of scale which is incorrect from the rg perspective. taken together, our results indicate that rg and pca can lead to oversimplification when multi-relevance is present and not accounted for.\",\n",
       "  'categories': 'cond-mat.stat-mech q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-18',\n",
       "  'updated': '2024-02-07',\n",
       "  'authors': ['adam g. kline', 'stephanie e. palmer'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.11009'},\n",
       " {'title': 'quantifying stimulus-relevant representational drift using   cross-modality contrastive learning',\n",
       "  'id': '2305.11953',\n",
       "  'abstract': 'previous works investigating representational drift from sensory to central nervous systems converged to show that neural coding, especially at the population level, readily overcomes these session-to-session fluctuations. however, representational drift in the primary visual cortex is more prominent when presenting naturalistic stimuli than artificial stimuli. animals continuously navigate natural environments during the evolutionary timescale. why did evolution not get rid of representational drift if it was just an inconvenience? here, we investigate how representational drift simultaneously influences the encoding of multiple behaviorally relevant features in a natural movie stimulus. because natural environments contain multiple interacting spatio-temporal features, previous works only provided incomplete understanding of representational drift because of such simplification. here, we use cross modality contrastive learning to learn an embedding of neural activity that retains only those relevant components of the natural movie stimulus. we also observe that our learned embedding is near-optimal in decoding a whole suite of natural features (scene, optic flow, complex spatio-temporal features, and time) and generalizable to decode those features from single-trial or novel hold-out data. using this embedding as a surrogate model, we observe that representational drift perturbs the local geometry of the embedding, and this results in various changes in performance when we decode from a different session (90 min later) even at the population level. our work further suggests that a separate compensation mechanism may be necessary for the optic flow features, as their autocorrelation scale is shorter than the minimum time needed to discriminate scene texture features. thus, representational drift may encourage neural processing flexibility rather than be a mere nuisance.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-19',\n",
       "  'updated': '2024-06-26',\n",
       "  'authors': ['siwei wang',\n",
       "   'elizabeth a de laittre',\n",
       "   'jason maclean',\n",
       "   'stephanie e palmer'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.11953'},\n",
       " {'title': 'alignment of density maps in wasserstein distance',\n",
       "  'id': '2305.12310',\n",
       "  'abstract': 'in this paper we propose an algorithm for aligning three-dimensional objects when represented as density maps, motivated by applications in cryogenic electron microscopy. the algorithm is based on minimizing the 1-wasserstein distance between the density maps after a rigid transformation. the induced loss function enjoys a more benign landscape than its euclidean counterpart and bayesian optimization is employed for computation. numerical experiments show improved accuracy and efficiency over existing algorithms on the alignment of real protein molecules. in the context of aligning heterogeneous pairs, we illustrate a potential need for new distance functions.',\n",
       "  'categories': 'eess.iv q-bio.qm stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-20',\n",
       "  'updated': '2024-03-11',\n",
       "  'authors': ['amit singer', 'ruiyi yang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.12310'},\n",
       " {'title': 'enhancing biodiversity through intraspecific suppression in large   ecosystems',\n",
       "  'id': '2305.12341',\n",
       "  'abstract': \"the competitive exclusion principle (cep) is a fundamental concept in the niche theory, which posits that the number of available resources constrains the coexistence of species. while the cep offers an intuitive explanation on coexistence, it has been challenged by counterexamples observed in nature. one prominent counterexample is the phytoplankton community, known as the paradox of the plankton. diverse phytoplankton species coexist in the ocean even though they demand a limited number of resources. to shed light on this remarkable biodiversity in large ecosystems quantitatively, we consider \\\\textit{intraspecific suppression} into the generalized macarthur's consumer-resource model and study the relative diversity, the number ratio between coexisting consumers and resource kinds. by employing the cavity method and generating functional analysis, we demonstrate that, under intraspecific suppression, the number of consumer species can surpass the available resources. this phenomenon stems from the fact that intraspecific suppression prevents the emergence of dominant species, thereby fostering high biodiversity. furthermore, our study highlights that the impact of this competition on biodiversity is contingent upon environmental conditions. our work presents a comprehensive framework that encompasses the cep and its counterexamples by introducing intraspecific suppression.\",\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-21',\n",
       "  'updated': '2024-04-01',\n",
       "  'authors': ['seong-gyu yang', 'hye jin park'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.12341'},\n",
       " {'title': 'transition from time-variant to static networks: timescale separation in   nimfa sis epidemics',\n",
       "  'id': '2305.12446',\n",
       "  'abstract': 'we extend the n-intertwined mean-field approximation (nimfa) for the susceptible-infectious-susceptible (sis) epidemiological process to time-varying networks. processes on time-varying networks are often analysed under the assumption that the process and network evolution happen on different timescales. this approximation is called timescale separation. we investigate timescale separation between disease spreading and topology updates of the network. we introduce the transition times $\\\\mathrm{\\\\underline{t}}(r)$ and $\\\\mathrm{\\\\overline{t}}(r)$ as the boundaries between the intermediate regime and the annealed (fast changing network) and quenched (static network) regimes, respectively, for a fixed accuracy tolerance $r$. by analysing the convergence of static nimfa processes, we analytically derive upper and lower bounds for $\\\\mathrm{\\\\overline{t}}(r)$. our results provide insights/bounds on the time of convergence to the steady state of the static nimfa sis process. we show that, under our assumptions, the upper-transition time $\\\\mathrm{\\\\overline{t}}(r)$ is almost entirely determined by the basic reproduction number $r_0$ of the network. the value of the upper-transition time $\\\\mathrm{\\\\overline{t}}(r)$ around the epidemic threshold is large, which agrees with the current understanding that some real-world epidemics cannot be approximated with the aforementioned timescale separation.',\n",
       "  'categories': 'math.ds q-bio.pe',\n",
       "  'doi': '10.1103/physreve.109.034308',\n",
       "  'created': '2023-05-21',\n",
       "  'updated': '2024-02-14',\n",
       "  'authors': ['robin persoons',\n",
       "   'mattia sensi',\n",
       "   'bastian prasse',\n",
       "   'piet van mieghem'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.12446'},\n",
       " {'title': 'topological data analysis suggests human brain networks reconfiguration   in the transition from a resting state to cognitive load',\n",
       "  'id': '2305.12491',\n",
       "  'abstract': 'the functional network of the brain continually adapts to changing environmental demands. the environmental changes closely connect with changes of active cognitive processes. in recent years, the network approach has emerged as a promising method for analyzing the neurophysiological mechanisms that underlie psychological functions. the present study examines topological characteristics of functional brain networks in resting state and in cognitive load, provided by the execution of the sternberg item recognition paradigm (sirp) based on electroencephalographic data. we propose that the topological properties of the functional networks in the human brain are distinct between cognitive load and resting state with higher integration in the networks during cognitive load. it was shown that topological features of functional connectomes strongly depend on the type of cognitive process performed by the subject and change in accordance with task change. the analysis also demonstrated that functional connectivity during working memory tasks showed a faster emergence of homology groups generators, supporting the idea of a relationship between the initial stages of working memory execution and an increase in faster network integration, with connector hubs playing a crucial role.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.23868/gc562843',\n",
       "  'created': '2023-05-21',\n",
       "  'updated': '2023-09-18',\n",
       "  'authors': ['ilya ernston', 'arsenii onuchin', 'timofey adamovich'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.12491'},\n",
       " {'title': 'rigorous estimates for the quasi-steady state approximation of the   michaelis-menten reaction mechanism at low enzyme concentrations',\n",
       "  'id': '2305.12602',\n",
       "  'abstract': 'there is a vast amount of literature concerning the appropriateness of various perturbation parameters for the standard quasi-steady state approximation in the michaelis-menten reaction mechanism, and also concerning the relevance of these parameters for the accuracy of the approximation by the familiar michaelis-menten equation. typically, the arguments in the literature are based on (heuristic) timescale estimates, from which one cannot obtain reliable quantitative estimates for the error of the quasi-steady state approximation. we take a different approach. by combining phase plane analysis with differential inequalities, we derive sharp explicit upper and lower estimates for the duration of the initial transient and substrate depletion during this transitory phase. in addition, we obtain rigorous bounds on the accuracy of the standard quasi-steady state approximation in the slow dynamics regime. notably, under the assumption that the quasi-steady state approximation is valid over the entire time course of the reaction, our error estimate is of order one in the segel-slemrod parameter.',\n",
       "  'categories': 'math.ds physics.chem-ph q-bio.mn',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-21',\n",
       "  'updated': '2024-01-30',\n",
       "  'authors': ['justin eilertsen', 'santiago schnell', 'sebastian walcher'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.12602'},\n",
       " {'title': 'learning to detect an animal sound from five examples',\n",
       "  'id': '2305.13210',\n",
       "  'abstract': \"automatic detection and classification of animal sounds has many applications in biodiversity monitoring and animal behaviour. in the past twenty years, the volume of digitised wildlife sound available has massively increased, and automatic classification through deep learning now shows strong results. however, bioacoustics is not a single task but a vast range of small-scale tasks (such as individual id, call type, emotional indication) with wide variety in data characteristics, and most bioacoustic tasks do not come with strongly-labelled training data. the standard paradigm of supervised learning, focussed on a single large-scale dataset and/or a generic pre-trained algorithm, is insufficient. in this work we recast bioacoustic sound event detection within the ai framework of few-shot learning. we adapt this framework to sound event detection, such that a system can be given the annotated start/end times of as few as 5 events, and can then detect events in long-duration audio -- even when the sound category was not known at the time of algorithm training. we introduce a collection of open datasets designed to strongly test a system's ability to perform few-shot sound event detections, and we present the results of a public contest to address the task. we show that prototypical networks are a strong-performing method, when enhanced with adaptations for general characteristics of animal sounds. we demonstrate that widely-varying sound event durations are an important factor in performance, as well as non-stationarity, i.e. gradual changes in conditions throughout the duration of a recording. for fine-grained bioacoustic recognition tasks without massive annotated training data, our results demonstrate that few-shot sound event detection is a powerful new method, strongly outperforming traditional signal-processing detection methods in the fully automated scenario.\",\n",
       "  'categories': 'cs.sd eess.as q-bio.qm',\n",
       "  'doi': '10.1016/j.ecoinf.2023.102258',\n",
       "  'created': '2023-05-22',\n",
       "  'updated': '',\n",
       "  'authors': ['inês nolasco',\n",
       "   'shubhr singh',\n",
       "   'veronica morfi',\n",
       "   'vincent lostanlen',\n",
       "   'ariana strandburg-peshkin',\n",
       "   'ester vidaña-vila',\n",
       "   'lisa gill',\n",
       "   'hanna pamuła',\n",
       "   'helen whitehead',\n",
       "   'ivan kiskin',\n",
       "   'frants h. jensen',\n",
       "   'joe morford',\n",
       "   'michael g. emmerson',\n",
       "   'elisabetta versace',\n",
       "   'emily grout',\n",
       "   'haohe liu',\n",
       "   'dan stowell'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.13210'},\n",
       " {'title': 'gene set summarization using large language models',\n",
       "  'id': '2305.13338',\n",
       "  'abstract': 'molecular biologists frequently interpret gene lists derived from high-throughput experiments and computational analysis. this is typically done as a statistical enrichment analysis that measures the over- or under-representation of biological function terms associated with genes or their properties, based on curated assertions from a knowledge base (kb) such as the gene ontology (go). interpreting gene lists can also be framed as a textual summarization task, enabling the use of large language models (llms), potentially utilizing scientific texts directly and avoiding reliance on a kb.   we developed spindoctor (structured prompt interpolation of natural language descriptions of controlled terms for ontology reporting), a method that uses gpt models to perform gene set function summarization as a complement to standard enrichment analysis. this method can use different sources of gene functional information: (1) structured text derived from curated ontological kb annotations, (2) ontology-free narrative gene summaries, or (3) direct model retrieval.   we demonstrate that these methods are able to generate plausible and biologically valid summary go term lists for gene sets. however, gpt-based approaches are unable to deliver reliable scores or p-values and often return terms that are not statistically significant. crucially, these methods were rarely able to recapitulate the most precise and informative term from standard enrichment, likely due to an inability to generalize and reason using an ontology. results are highly nondeterministic, with minor variations in prompt resulting in radically different term lists. our results show that at this point, llm-based methods are unsuitable as a replacement for standard term enrichment analysis and that manual curation of ontological assertions remains necessary.',\n",
       "  'categories': 'q-bio.gn cs.ai cs.cl q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-20',\n",
       "  'updated': '2024-07-03',\n",
       "  'authors': ['marcin p. joachimiak',\n",
       "   'j. harry caufield',\n",
       "   'nomi l. harris',\n",
       "   'hyeongsik kim',\n",
       "   'christopher j. mungall'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.13338'},\n",
       " {'title': 'signatures of hierarchical temporal processing in the mouse visual   system',\n",
       "  'id': '2305.13427',\n",
       "  'abstract': 'a core challenge for the brain is to process information across various timescales. this could be achieved by a hierarchical organization of temporal processing through intrinsic mechanisms (e.g., recurrent coupling or adaptation), but recent evidence from spike recordings of the rodent visual system seems to conflict with this hypothesis. here, we used an optimized information-theoretic and classical autocorrelation analysis to show that information- and intrinsic timescales of spiking activity increase along the anatomical hierarchy of the mouse visual system, while information-theoretic predictability decreases. moreover, the timescale hierarchy was invariant to the stimulus condition, whereas the decrease in predictability was strongest under natural movie stimulation. we could reproduce this effect in a basic recurrent network model with correlated sensory input. our findings suggest that the rodent visual system indeed employs intrinsic mechanisms to achieve longer integration for higher cortical areas, while simultaneously reducing predictability for an efficient neural code.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-22',\n",
       "  'updated': '2024-01-17',\n",
       "  'authors': ['lucas rudelt',\n",
       "   'daniel gonzález marx',\n",
       "   'f. paul spitzner',\n",
       "   'benjamin cramer',\n",
       "   'johannes zierenberg',\n",
       "   'viola priesemann'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.13427'},\n",
       " {'title': 'parameter estimation from an ornstein-uhlenbeck process with measurement   noise',\n",
       "  'id': '2305.13498',\n",
       "  'abstract': 'this article aims to investigate the impact of noise on parameter fitting for an ornstein-uhlenbeck process, focusing on the effects of multiplicative and thermal noise on the accuracy of signal separation. to address these issues, we propose algorithms and methods that can effectively distinguish between thermal and multiplicative noise and improve the precision of parameter estimation for optimal data analysis. specifically, we explore the impact of both multiplicative and thermal noise on the obfuscation of the actual signal and propose methods to resolve them. first, we present an algorithm that can effectively separate thermal noise with comparable performance to hamilton monte carlo (hmc) but with significantly improved speed. we then analyze multiplicative noise and demonstrate that hmc is insufficient for isolating thermal and multiplicative noise. however, we show that, with additional knowledge of the ratio between thermal and multiplicative noise, we can accurately distinguish between the two types of noise when provided with a sufficiently large sampling rate or an amplitude of multiplicative noise smaller than thermal noise. thus, we demonstrate the mechanism underlying an otherwise counterintuitive phenomenon: when multiplicative noise dominates the noise spectrum, one can successfully estimate the parameters for such systems after adding additional white noise to shift the noise balance.',\n",
       "  'categories': 'stat.ml cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-22',\n",
       "  'updated': '2024-07-10',\n",
       "  'authors': ['simon carter', 'lilianne mujica-parodi', 'helmut h. strey'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.13498'},\n",
       " {'title': 'toward stochastic neural computing',\n",
       "  'id': '2305.13982',\n",
       "  'abstract': \"the highly irregular spiking activity of cortical neurons and behavioral variability suggest that the brain could operate in a fundamentally probabilistic way. mimicking how the brain implements and learns probabilistic computation could be a key to developing machine intelligence that can think more like humans. in this work, we propose a theory of stochastic neural computing (snc) in which streams of noisy inputs are transformed and processed through populations of nonlinearly coupled spiking neurons. to account for the propagation of correlated neural variability, we derive from first principles a moment embedding for spiking neural network (snn). this leads to a new class of deep learning model called the moment neural network (mnn) which naturally generalizes rate-based neural networks to second order. as the mnn faithfully captures the stationary statistics of spiking neural activity, it can serve as a powerful proxy for training snn with zero free parameters. through joint manipulation of mean firing rate and noise correlations in a task-driven way, the model is able to learn inference tasks while simultaneously minimizing prediction uncertainty, resulting in enhanced inference speed. we further demonstrate the application of our method to intel's loihi neuromorphic hardware. the proposed theory of snc may open up new opportunities for developing machine intelligence capable of computing uncertainty and for designing unconventional computing architectures.\",\n",
       "  'categories': 'cs.ne physics.bio-ph q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-23',\n",
       "  'updated': '2024-04-21',\n",
       "  'authors': ['yang qi',\n",
       "   'zhichao zhu',\n",
       "   'yiming wei',\n",
       "   'lu cao',\n",
       "   'zhigang wang',\n",
       "   'jie zhang',\n",
       "   'wenlian lu',\n",
       "   'jianfeng feng'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.13982'},\n",
       " {'title': 'brain memory working. optimal control behavior for improved   hopfield-like models',\n",
       "  'id': '2305.14360',\n",
       "  'abstract': 'recent works have highlighted the need for a new dynamical paradigm in the modeling of brain function and evolution. specifically, these models should incorporate non-constant and asymmetric synaptic weights \\\\(t_{ij}\\\\) in the neuron-neuron interaction matrix, moving beyond the classical hopfield framework. krotov and hopfield proposed a non-constant yet symmetric model, resulting in a vector field that describes gradient-type dynamics, which includes a lyapunov-like energy function. firstly, we will outline the general conditions for generating a hopfield-like vector field of gradient type, recovering the krotov-hopfield condition as a particular case. secondly, we address the issue of symmetry, which we abandon for two key physiological reasons: (1) actual neural connections have a distinctly directional character (axons and dendrites), and (2) the gradient structure derived from symmetry forces the dynamics towards stationary points, leading to the recognition of every pattern. we propose a novel model that incorporates a set of limited but variable controls \\\\(|\\\\xi_{ij}|\\\\leq k\\\\), which are used to adjust an initially constant interaction matrix, \\\\(t_{ij}=a_{ij}+\\\\xi_{ij}\\\\). additionally, we introduce a reasonable controlled variational functional for optimization. this allows us to simulate three potential outcomes when a pattern is submitted to the learning system: (1) if the dynamics converges to an existing stationary point without activating controls, the system has \\\\emph{recognized} an existing pattern; (2) if a new stationary point is reached through control activation, the system has \\\\emph{learned} a new pattern; and (3) if the dynamics \\\\emph{wanders} without reaching any stationary point, the system is unable to recognize or learn the submitted pattern. an additional feature (4) models the processes of \\\\emph{forgetting and restoring} memory.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-11',\n",
       "  'updated': '2024-11-11',\n",
       "  'authors': ['franco cardin',\n",
       "   'alberto lovison',\n",
       "   'amos maritan',\n",
       "   'aram megighian'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.14360'},\n",
       " {'title': 'criticality analysis: bio-inspired nonlinear data representation',\n",
       "  'id': '2305.14361',\n",
       "  'abstract': 'the representation of arbitrary data in a biological system is one of the most elusive elements of biological information processing. the often logarithmic nature of information in amplitude and frequency presented to biosystems prevents simple encapsulation of the information contained in the input. criticality analysis (ca) is a bio-inspired method of information representation within a controlled self-organised critical system that allows scale-free representation. this is based on the concept of a reservoir of dynamic behaviour in which self-similar data will create dynamic nonlinear representations. this unique projection of data preserves the similarity of data within a multidimensional neighbourhood. the input can be reduced dimensionally to a projection output that retains the features of the overall data, yet has much simpler dynamic response. the method depends only on the rate control of chaos applied to the underlying controlled models, that allows the encoding of arbitrary data, and promises optimal encoding of data given biological relevant networks of oscillators. the ca method allows for a biologically relevant encoding mechanism of arbitrary input to biosystems, creating a suitable model for information processing in varying complexity of organisms and scale-free data representation for machine learning.',\n",
       "  'categories': 'q-bio.nc cs.lg',\n",
       "  'doi': '10.3390/e25121660',\n",
       "  'created': '2023-05-11',\n",
       "  'updated': '',\n",
       "  'authors': ['tjeerd v. olde scheper'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.14361'},\n",
       " {'title': 'looking forwards and backwards: dynamics and genealogies of locally   regulated populations',\n",
       "  'id': '2305.14488',\n",
       "  'abstract': 'we introduce a broad class of spatial models to describe how spatially heterogeneous populations live, die, and reproduce. individuals are represented by points of a point measure, whose birth and death rates can depend both on spatial position and local population density, defined via the convolution of the point measure with a nonnegative kernel. we pass to three different scaling limits: an interacting superprocess, a nonlocal partial differential equation (pde), and a classical pde. the classical pde is obtained both by first scaling time and population size to pass to the nonlocal pde, and then scaling the kernel that determines local population density; and also (when the limit is a reaction-diffusion equation) by simultaneously scaling the kernel width, timescale and population size in our individual based model. a novelty of our model is that we explicitly model a juvenile phase: offspring are thrown off in a gaussian distribution around the location of the parent, and reach (instant) maturity with a probability that can depend on the population density at the location at which they land. although we only record mature individuals, a trace of this two-step description remains in our population models, resulting in novel limits governed by a nonlinear diffusion. using a lookdown representation, we retain information about genealogies and, in the case of deterministic limiting models, use this to deduce the backwards in time motion of the ancestral lineage of a sampled individual. we observe that knowing the history of the population density is not enough to determine the motion of ancestral lineages in our model. we also investigate the behaviour of lineages for three different deterministic models of a population expanding its range as a travelling wave: the fisher-kpp equation, the allen-cahn equation, and a porous medium equation with logistic growth.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-23',\n",
       "  'updated': '2023-12-30',\n",
       "  'authors': ['alison m. etheridge',\n",
       "   'thomas g. kurtz',\n",
       "   'ian letter',\n",
       "   'peter l. ralph',\n",
       "   'terence tsui ho lung'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.14488'},\n",
       " {'title': 'grnade: geometric deep learning for 3d rna inverse design',\n",
       "  'id': '2305.14749',\n",
       "  'abstract': 'computational rna design tasks are often posed as inverse problems, where sequences are designed based on adopting a single desired secondary structure without considering 3d geometry and conformational diversity. we introduce grnade, a geometric rna design pipeline operating on 3d rna backbones to design sequences that explicitly account for structure and dynamics. under the hood, grnade is a multi-state graph neural network that generates candidate rna sequences conditioned on one or more 3d backbone structures where the identities of the bases are unknown. on a single-state fixed backbone re-design benchmark of 14 rna structures from the pdb identified by das et al. [2010], grnade obtains higher native sequence recovery rates (56% on average) compared to rosetta (45% on average), taking under a second to produce designs compared to the reported hours for rosetta. we further demonstrate the utility of grnade on a new benchmark of multi-state design for structurally flexible rnas, as well as zero-shot ranking of mutational fitness landscapes in a retrospective analysis of a recent ribozyme. open source code: https://github.com/chaitjo/geometric-rna-design',\n",
       "  'categories': 'cs.lg q-bio.bm q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-24',\n",
       "  'updated': '2024-10-06',\n",
       "  'authors': ['chaitanya k. joshi',\n",
       "   'arian r. jamasb',\n",
       "   'ramon viñas',\n",
       "   'charles harris',\n",
       "   'simon v. mathis',\n",
       "   'alex morehead',\n",
       "   'rishabh anand',\n",
       "   'pietro liò'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.14749'},\n",
       " {'title': 'automated discovery of interpretable hyperelastic material models for   human brain tissue with euclid',\n",
       "  'id': '2305.16362',\n",
       "  'abstract': 'we propose an automated computational algorithm for simultaneous model selection and parameter identification for the hyperelastic mechanical characterization of human brain tissue. following the motive of the recently proposed computational framework euclid (efficient unsupervised constitutive law identitication and discovery) and in contrast to conventional parameter calibration methods, we construct an extensive set of candidate hyperelastic models, i.e., a model library including popular models known from the literature, and develop a computational strategy for automatically selecting a model from the library that conforms to the available experimental data while being represented as an interpretable symbolic mathematical expression. this computational strategy comprises sparse regression, i.e., a regression problem that is regularized by a sparsity promoting penalty term that filters out irrelevant models from the model library, and a clustering method for grouping together highly correlated and thus redundant features in the model library. the model selection procedure is driven by labelled data pairs stemming from mechanical tests under different deformation modes, i.e., uniaxial compression/tension and simple torsion, and can thus be interpreted as a supervised counterpart to the originally proposed euclid that is informed by full-field displacement data and global reaction forces. the proposed method is verified on synthetical data with artificial noise and validated on experimental data acquired through mechanical tests of human brain specimens, proving that the method is capable of discovering hyperelastic models that exhibit both high fitting accuracy to the data as well as concise and thus interpretable mathematical representations.',\n",
       "  'categories': 'q-bio.qm cs.ce',\n",
       "  'doi': '10.1016/j.jmps.2023.105404',\n",
       "  'created': '2023-05-25',\n",
       "  'updated': '',\n",
       "  'authors': ['moritz flaschel',\n",
       "   'huitian yu',\n",
       "   'nina reiter',\n",
       "   'jan hinrichsen',\n",
       "   'silvia budday',\n",
       "   'paul steinmann',\n",
       "   'siddhant kumar',\n",
       "   'laura de lorenzis'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.16362'},\n",
       " {'title': 'epidemic transmission modeling with fractional derivatives and   environmental pathogens',\n",
       "  'id': '2305.16689',\n",
       "  'abstract': \"this research presents an advanced fractional-order compartmental model designed to delve into the complexities of covid-19 transmission dynamics, specifically accounting for the influence of environmental pathogens on disease spread. by enhancing the classical compartmental framework, our model distinctively incorporates the effects of order derivatives and environmental shedding mechanisms on the basic reproduction numbers, thus offering a holistic perspective on transmission dynamics. leveraging fractional calculus, the model adeptly captures the memory effect associated with disease spread, providing an authentic depiction of the virus's real-world propagation patterns. a thorough mathematical analysis confirming the existence, uniqueness, and stability of the model's solutions emphasizes its robustness. furthermore, the numerical simulations, meticulously calibrated with real covid-19 case data, affirm the model's capacity to emulate observed transmission trends, demonstrating the pivotal role of environmental transmission vectors in shaping public health strategies. the study highlights the critical role of environmental sanitation and targeted interventions in controlling the pandemic's spread, suggesting new insights for research and policy-making in infectious disease management.\",\n",
       "  'categories': 'q-bio.pe cs.na math.na',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-26',\n",
       "  'updated': '2024-07-08',\n",
       "  'authors': ['moein khalighi', 'faïçal ndaïrou', 'leo lahti'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.16689'},\n",
       " {'title': 'the brain tumor segmentation (brats) challenge 2023: focus on pediatrics   (cbtn-connect-dipgr-asnr-miccai brats-peds)',\n",
       "  'id': '2305.17033',\n",
       "  'abstract': 'pediatric tumors of the central nervous system are the most common cause of cancer-related death in children. the five-year survival rate for high-grade gliomas in children is less than 20\\\\%. due to their rarity, the diagnosis of these entities is often delayed, their treatment is mainly based on historic treatment concepts, and clinical trials require multi-institutional collaborations. the miccai brain tumor segmentation (brats) challenge is a landmark community benchmark event with a successful history of 12 years of resource creation for the segmentation and analysis of adult glioma. here we present the cbtn-connect-dipgr-asnr-miccai brats-peds 2023 challenge, which represents the first brats challenge focused on pediatric brain tumors with data acquired across multiple international consortia dedicated to pediatric neuro-oncology and clinical trials. the brats-peds 2023 challenge focuses on benchmarking the development of volumentric segmentation algorithms for pediatric brain glioma through standardized quantitative performance evaluation metrics utilized across the brats 2023 cluster of challenges. models gaining knowledge from the brats-peds multi-parametric structural mri (mpmri) training data will be evaluated on separate validation and unseen test mpmri dataof high-grade pediatric glioma. the cbtn-connect-dipgr-asnr-miccai brats-peds 2023 challenge brings together clinicians and ai/imaging scientists to lead to faster development of automated segmentation techniques that could benefit clinical trials, and ultimately the care of children with brain tumors.',\n",
       "  'categories': 'eess.iv cs.cv cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-26',\n",
       "  'updated': '2024-05-23',\n",
       "  'authors': ['anahita fathi kazerooni',\n",
       "   'nastaran khalili',\n",
       "   'xinyang liu',\n",
       "   'debanjan haldar',\n",
       "   'zhifan jiang',\n",
       "   'syed muhammed anwar',\n",
       "   'jake albrecht',\n",
       "   'maruf adewole',\n",
       "   'udunna anazodo',\n",
       "   'hannah anderson',\n",
       "   'sina bagheri',\n",
       "   'ujjwal baid',\n",
       "   'timothy bergquist',\n",
       "   'austin j. borja',\n",
       "   'evan calabrese',\n",
       "   'verena chung',\n",
       "   'gian-marco conte',\n",
       "   'farouk dako',\n",
       "   'james eddy',\n",
       "   'ivan ezhov',\n",
       "   'ariana familiar',\n",
       "   'keyvan farahani',\n",
       "   'shuvanjan haldar',\n",
       "   'juan eugenio iglesias',\n",
       "   'anastasia janas',\n",
       "   'elaine johansen',\n",
       "   'blaise v jones',\n",
       "   'florian kofler',\n",
       "   'dominic labella',\n",
       "   'hollie anne lai',\n",
       "   'koen van leemput',\n",
       "   'hongwei bran li',\n",
       "   'nazanin maleki',\n",
       "   'aaron s mcallister',\n",
       "   'zeke meier',\n",
       "   'bjoern menze',\n",
       "   'ahmed w moawad',\n",
       "   'khanak k nandolia',\n",
       "   'julija pavaine',\n",
       "   'marie piraud',\n",
       "   'tina poussaint',\n",
       "   'sanjay p prabhu',\n",
       "   'zachary reitman',\n",
       "   'andres rodriguez',\n",
       "   'jeffrey d rudie',\n",
       "   'mariana sanchez-montano',\n",
       "   'ibraheem salman shaikh',\n",
       "   'lubdha m. shah',\n",
       "   'nakul sheth',\n",
       "   'russel taki shinohara',\n",
       "   'wenxin tu',\n",
       "   'karthik viswanathan',\n",
       "   'chunhao wang',\n",
       "   'jeffrey b ware',\n",
       "   'benedikt wiestler',\n",
       "   'walter wiggins',\n",
       "   'anna zapaishchykova',\n",
       "   'mariam aboian',\n",
       "   'miriam bornhorst',\n",
       "   'peter de blank',\n",
       "   'michelle deutsch',\n",
       "   'maryam fouladi',\n",
       "   'lindsey hoffman',\n",
       "   'benjamin kann',\n",
       "   'margot lazow',\n",
       "   'leonie mikael',\n",
       "   'ali nabavizadeh',\n",
       "   'roger packer',\n",
       "   'adam resnick',\n",
       "   'brian rood',\n",
       "   'arastoo vossough',\n",
       "   'spyridon bakas',\n",
       "   'marius george linguraru'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.17033'},\n",
       " {'title': 'ai-based analysis of super-resolution microscopy: biological discovery   in the absence of ground truth',\n",
       "  'id': '2305.17193',\n",
       "  'abstract': 'super-resolution microscopy, or nanoscopy, enables the use of fluorescent-based molecular localization tools to study molecular structure at the nanoscale level in the intact cell, bridging the mesoscale gap to classical structural biology methodologies. analysis of super-resolution data by artificial intelligence (ai), such as machine learning, offers tremendous potential for discovery of new biology, that, by definition, is not known and lacks ground truth. herein, we describe the application of weakly supervised paradigms to super-resolution microscopy and its potential to enable the accelerated exploration of the nanoscale architecture of subcellular macromolecules and organelles.',\n",
       "  'categories': 'q-bio.sc cs.ai cs.cv cs.lg physics.bio-ph q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-26',\n",
       "  'updated': '2024-05-27',\n",
       "  'authors': ['ivan r. nabi',\n",
       "   'ben cardoen',\n",
       "   'ismail m. khater',\n",
       "   'guang gao',\n",
       "   'timothy h. wong',\n",
       "   'ghassan hamarneh'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.17193'},\n",
       " {'title': 'generating visual information for motion sickness reduction using a   computational model based on svc theory',\n",
       "  'id': '2305.17832',\n",
       "  'abstract': 'with the advancements in automated driving, there is concern that motion sickness will increase as non-driving-related tasks increase. therefore, techniques to reduce motion sickness have drawn much attention. research studies have attempted to estimate motion sickness using computational models for controlling it. among them, a computational model for estimating motion sickness incidence (msi) with visual information as input based on subjective vertical conflict theories was developed. in addition, some studies attempt to mitigate motion sickness by controlling visual information. in particular, it has been confirmed that motion sickness is suppressed by matching head movement and visual information. however, there has been no research on optimal visual information control that suppresses motion sickness in vehicles by utilizing mathematical models. we, therefore, propose a method for generating optimal visual information to suppress motion sickness caused from vehicle motion by utilizing a motion sickness model with vestibular and visual inputs. to confirm the effectiveness of the proposed method, we investigated changes in the motion sickness experienced by the participants according to the visual information displayed on the head-mounted display. the experimental results suggested that the proposed method mitigates the motion sickness of the participants.',\n",
       "  'categories': 'cs.hc q-bio.nc',\n",
       "  'doi': '10.1109/itsc57777.2023.10422244',\n",
       "  'created': '2023-05-28',\n",
       "  'updated': '',\n",
       "  'authors': ['yujiro tamura', 'takahiro wada', 'hailong liu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.17832'},\n",
       " {'title': 'drug repurposing targeting covid-19 3cl protease using molecular docking   and machine learning regression approach',\n",
       "  'id': '2305.18088',\n",
       "  'abstract': 'the covid-19 pandemic has initiated a global health emergency, with an exigent need for effective cure. progressively, drug repurposing is emerging a promise solution as it saves the time, cost and labor. however, the number of drug candidates that have been identified as being repurposed for the treatment of covid-19 are still insufficient, so more effective and thorough drug exploring strategies are required. in this study, we joint the molecular docking with machine learning regression approaches to find some prospective therapeutic candidates for covid-19 treatment. we screened the 5903 approved drugs for their inhibition by targeting the main protease 3cl of sars-cov-2, which is responsible to replicate the virus. molecular docking is used to calculate the binding affinities of these drugs to the main protease 3cl. we employed several machine learning regression approaches for qsar modeling to find out some potential drugs with high binding affinities. our outcomes demonstrated that the decision tree regression (dtr) model with best scores of r2 and rmse, is the most suitable model to explore the potential drugs. we shortlisted six favorable drugs. these drugs have novel repurposing potential, except for one antiviral zinc203757351 compound that has already been identified in other studies. we further examined the physiochemical and pharmacokinetic properties of these most potent drugs and their best binding interaction to specific target protease 3clpro. our verdicts contribute to the larger goal of finding effective cures for covid-19, which is an acute global health challenge. the outcomes of our study provide valuable insights into potential therapeutic candidates for covid-19 treatment.',\n",
       "  'categories': 'q-bio.bm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-25',\n",
       "  'updated': '2024-06-23',\n",
       "  'authors': ['imra aqeel', 'abdul majid'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.18088'},\n",
       " {'title': 'determinantal point process attention over grid cell code supports out   of distribution generalization',\n",
       "  'id': '2305.18417',\n",
       "  'abstract': 'deep neural networks have made tremendous gains in emulating human-like intelligence, and have been used increasingly as ways of understanding how the brain may solve the complex computational problems on which this relies. however, these still fall short of, and therefore fail to provide insight into how the brain supports strong forms of generalization of which humans are capable. one such case is out-of-distribution (ood) generalization-successful performance on test examples that lie outside the distribution of the training set. here, we identify properties of processing in the brain that may contribute to this ability. we describe a two-part algorithm that draws on specific features of neural computation to achieve ood generalization, and provide a proof of concept by evaluating performance on two challenging cognitive tasks. first we draw on the fact that the mammalian brain represents metric spaces using grid cell code (e.g., in the entorhinal cortex): abstract representations of relational structure, organized in recurring motifs that cover the representational space. second, we propose an attentional mechanism that operates over the grid cell code using determinantal point process (dpp), that we call dpp attention (dpp-a) -- a transformation that ensures maximum sparseness in the coverage of that space. we show that a loss function that combines standard task-optimized error with dpp-a can exploit the recurring motifs in the grid cell code, and can be integrated with common architectures to achieve strong ood generalization performance on analogy and arithmetic tasks. this provides both an interpretation of how the grid cell code in the mammalian brain may contribute to generalization performance, and at the same time a potential means for improving such capabilities in artificial neural networks.',\n",
       "  'categories': 'cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-28',\n",
       "  'updated': '2024-01-23',\n",
       "  'authors': ['shanka subhra mondal',\n",
       "   'steven frankland',\n",
       "   'taylor webb',\n",
       "   'jonathan d. cohen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.18417'},\n",
       " {'title': 'identification of novel diagnostic neuroimaging biomarkers for autism   spectrum disorder through convolutional neural network-based analysis of   functional, structural, and diffusion tensor imaging data towards enhanced   autism diagnosis',\n",
       "  'id': '2305.18841',\n",
       "  'abstract': 'autism spectrum disorder is one of the leading neurodevelopmental disorders in our world, present in over 1% of the population and rapidly increasing in prevalence, yet the condition lacks a robust, objective, and efficient diagnostic. clinical diagnostic criteria rely on subjective behavioral assessments, which are prone to misdiagnosis as they face limitations in terms of their heterogeneity, specificity, and biases. this study proposes a novel convolutional neural network-based classification tool that aims to identify the potential of different neuroimaging features as autism biomarkers. the model is constructed using a set of sequential layers specifically designed to extract relevant features from brain imaging data. trained and tested on over 300,000 distinct features across three imaging types, the model shows promise in classifying individuals with autism from typical controls, outperforming metrics of current gold standard diagnostics by achieving an accuracy of 95.4% on a dataset of 1,111 samples with 521 autistic subjects (260 male and 261 female) and 590 controls (297 male and 293 female). 32 optimal features from the training data were identified and classified as candidate biomarkers using an independent samples t-test, in which functional features such as connectivity and the time series of signal intensity from each voxel exhibited the highest mean value differences between individuals with autism and typical control subjects. the p-values of these biomarkers were < 0.001, proving the statistical significance of the results and indicating that this research could pave the way towards the usage of neuroimaging in conjunction with behavioral criteria in clinics.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-30',\n",
       "  'updated': '2024-02-29',\n",
       "  'authors': ['annie adhikary'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.18841'},\n",
       " {'title': 'closed ecosystems extract energy through self-organized nutrient cycles',\n",
       "  'id': '2305.19102',\n",
       "  'abstract': 'our planet is roughly closed to matter, but open to energy input from the sun. however, to harness this energy, organisms must transform matter from one chemical (redox) state to another. for example, photosynthetic organisms can capture light energy by carrying out a pair of electron donor and acceptor transformations (e.g., water to oxygen, co$_2$ to organic carbon). closure of ecosystems to matter requires that all such transformations are ultimately balanced, i.e., other organisms must carry out corresponding reverse transformations, resulting in cycles that are coupled to each other. a sustainable closed ecosystem thus requires self-organized cycles of matter, in which every transformation has sufficient thermodynamic favorability to maintain an adequate number of organisms carrying out that process. here, we propose a new conceptual model that explains the self-organization and emergent features of closed ecosystems. we study this model with varying levels of metabolic diversity and energy input, finding that several thermodynamic features converge across ecosystems. specifically, irrespective of their species composition, large and metabolically diverse communities self-organize to extract roughly 10% of the maximum extractable energy, or 100 fold more than randomized communities. moreover, distinct communities implement energy extraction in convergent ways, as indicated by strongly correlated fluxes through nutrient cycles. as the driving force from light increases, however, these features -- fluxes and total energy extraction -- become more variable across communities, indicating that energy limitation imposes tight thermodynamic constraints on collective metabolism.',\n",
       "  'categories': 'q-bio.pe nlin.ao q-bio.mn',\n",
       "  'doi': '10.1073/pnas.2309387120',\n",
       "  'created': '2023-05-30',\n",
       "  'updated': '',\n",
       "  'authors': ['akshit goyal',\n",
       "   'avi i. flamholz',\n",
       "   'alexander p. petroff',\n",
       "   'arvind murugan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.19102'},\n",
       " {'title': 'sparse species interactions reproduce abundance correlation patterns in   microbial communities',\n",
       "  'id': '2305.19154',\n",
       "  'abstract': 'during the last decades macroecology has identified broad-scale patterns of abundances and diversity of microbial communities and put forward some potential explanations for them. however, these advances are not paralleled by a full understanding of the dynamical processes behind them. in particular, abundance fluctuations of different species are found to be correlated, both across time and across communities in metagenomic samples. reproducing such correlations through appropriate population models remains an open challenge. the present paper tackles this problem and points to sparse species interactions as a necessary mechanism to account for them. specifically, we discuss several possibilities to include interactions in population models and recognize lotka-volterra constants as a successful ansatz. for this, we design a bayesian inference algorithm to extract sets of interaction constants able to reproduce empirical probability distributions of pairwise correlations for diverse biomes. importantly, the inferred models still reproduce well-known single-species macroecological patterns concerning abundance fluctuations across both species and communities. endorsed by the agreement with the empirically observed phenomenology, our analyses provide insights on the properties of the networks of microbial interactions, revealing that sparsity is a crucial feature.',\n",
       "  'categories': 'q-bio.pe math.st q-bio.qm stat.th',\n",
       "  'doi': '10.1073/pnas.2309575121',\n",
       "  'created': '2023-05-30',\n",
       "  'updated': '2023-12-06',\n",
       "  'authors': ['josé camacho-mateu',\n",
       "   'aniello lampo',\n",
       "   'matteo sireci',\n",
       "   'miguel ángel muñoz',\n",
       "   'josé a. cuesta'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.19154'},\n",
       " {'title': 'covid-19 detection from exhaled breath',\n",
       "  'id': '2305.19211',\n",
       "  'abstract': 'the sars-cov-2 coronavirus emerged in 2019, causing a covid-19 pandemic that resulted in 7 million deaths out of 770 million reported cases over the next four years. the global health emergency called for unprecedented efforts to monitor and reduce the rate of infection, pushing the study of new diagnostic methods. in this paper, we introduce a cheap, fast, and non-invasive detection system, which exploits only the exhaled breath. specifically, provided an air sample, the mass spectra in the 10--351 mass-to-charge range are measured using an original nano-sampling device coupled with a high-precision spectrometer; then, the raw spectra are processed by custom software algorithms; the clean and augmented data are eventually classified using state-of-the-art machine-learning algorithms. an uncontrolled clinical trial was conducted between 2021 and 2022 on some 300 subjects who were concerned about being infected, either due to exhibiting symptoms or having quite recently recovered from illness. despite the simplicity of use, our system showed a performance comparable to the traditional polymerase-chain-reaction and antigen testing in identifying cases of covid-19 (that is, 0.95 accuracy, 0.94 recall, 0.96 specificity, and 0.92 f1-score). in light of these outcomes, we think that the proposed system holds the potential for substantial contributions to routine screenings and expedited responses during future epidemics, as it yields results comparable to state-of-the-art methods, providing them in a more rapid and less invasive manner.',\n",
       "  'categories': 'cs.lg q-bio.qm',\n",
       "  'doi': '10.1038/s41598-024-74104-1',\n",
       "  'created': '2023-05-30',\n",
       "  'updated': '2024-04-25',\n",
       "  'authors': ['nicolo bellarmino',\n",
       "   'giorgio bozzini',\n",
       "   'riccardo cantoro',\n",
       "   'francesco castelletti',\n",
       "   'michele castelluzzo',\n",
       "   'carla ciricugno',\n",
       "   'raffaele correale',\n",
       "   'daniela dalla gasperina',\n",
       "   'francesco dentali',\n",
       "   'giovanni poggialini',\n",
       "   'piergiorgio salerno',\n",
       "   'giovanni squillero',\n",
       "   'stefano taborelli'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.19211'},\n",
       " {'title': 'fibration symmetries and cluster synchronization in the caenorhabditis   elegans connectome',\n",
       "  'id': '2305.19367',\n",
       "  'abstract': 'capturing how the caenorhabditis elegans connectome structure gives rise to its neuron functionality remains unclear. it is through fiber symmetries found in its neuronal connectivity that synchronization of a group of neurons can be determined. to understand these we investigate graph symmetries and search for such in the symmetrized versions of the forward and backward locomotive sub-networks of the caenorhabditi elegans worm neuron network. the use of ordinarily differential equations simulations admissible to these graphs are used to validate the predictions of these fiber symmetries and are compared to the more restrictive orbit symmetries. additionally fibration symmetries are used to decompose these graphs into their fundamental building blocks which reveal units formed by nested loops or multilayered fibers. it is found that fiber symmetries of the connectome can accurately predict neuronal synchronization even under not idealized connectivity as long as the dynamics are within stable regimes of simulations.',\n",
       "  'categories': 'q-bio.nc q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-30',\n",
       "  'updated': '2024-05-03',\n",
       "  'authors': ['bryant avila',\n",
       "   'pedro augusto',\n",
       "   'manuel zimmer',\n",
       "   'matteo serafino',\n",
       "   'hernán a. makse'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.19367'},\n",
       " {'title': 'synaptic weight distributions depend on the geometry of plasticity',\n",
       "  'id': '2305.19394',\n",
       "  'abstract': 'a growing literature in computational neuroscience leverages gradient descent and learning algorithms that approximate it to study synaptic plasticity in the brain. however, the vast majority of this work ignores a critical underlying assumption: the choice of distance for synaptic changes - i.e. the geometry of synaptic plasticity. gradient descent assumes that the distance is euclidean, but many other distances are possible, and there is no reason that biology necessarily uses euclidean geometry. here, using the theoretical tools provided by mirror descent, we show that the distribution of synaptic weights will depend on the geometry of synaptic plasticity. we use these results to show that experimentally-observed log-normal weight distributions found in several brain areas are not consistent with standard gradient descent (i.e. a euclidean geometry), but rather with non-euclidean distances. finally, we show that it should be possible to experimentally test for different synaptic geometries by comparing synaptic weight distributions before and after learning. overall, our work shows that the current paradigm in theoretical work on synaptic plasticity that assumes euclidean synaptic geometry may be misguided and that it should be possible to experimentally determine the true geometry of synaptic plasticity in the brain.',\n",
       "  'categories': 'q-bio.nc cs.lg cs.ne',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-30',\n",
       "  'updated': '2024-03-04',\n",
       "  'authors': ['roman pogodin',\n",
       "   'jonathan cornford',\n",
       "   'arna ghosh',\n",
       "   'gauthier gidel',\n",
       "   'guillaume lajoie',\n",
       "   'blake richards'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.19394'},\n",
       " {'title': 'reliability of energy landscape analysis of resting-state functional mri   data',\n",
       "  'id': '2305.19573',\n",
       "  'abstract': 'energy landscape analysis is a data-driven method to analyze multidimensional time series, including functional magnetic resonance imaging (fmri) data. it has been shown to be a useful characterization of fmri data in health and disease. it fits an ising model to the data and captures the dynamics of the data as movement of a noisy ball constrained on the energy landscape derived from the estimated ising model. in the present study, we examine test-retest reliability of the energy landscape analysis. to this end, we construct a permutation test that assesses whether or not indices characterizing the energy landscape are more consistent across different sets of scanning sessions from the same participant (i.e., within-participant reliability) than across different sets of sessions from different participants (i.e., between-participant reliability). we show that the energy landscape analysis has significantly higher within-participant than between-participant test-retest reliability with respect to four commonly used indices. we also show that a variational bayesian method, which enables us to estimate energy landscapes tailored to each participant, displays comparable test-retest reliability to that using the conventional likelihood maximization method. the proposed methodology paves the way to perform individual-level energy landscape analysis for given data sets with a statistically controlled reliability.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1111/ejn.16390',\n",
       "  'created': '2023-05-31',\n",
       "  'updated': '2024-08-20',\n",
       "  'authors': ['pitambar khanra',\n",
       "   'johan nakuci',\n",
       "   'sarah muldoon',\n",
       "   'takamitsu watanabe',\n",
       "   'naoki masuda'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.19573'},\n",
       " {'title': 'the dynamic sensorium competition for predicting large-scale mouse   visual cortex activity from videos',\n",
       "  'id': '2305.19654',\n",
       "  'abstract': 'understanding how biological visual systems process information is challenging due to the complex nonlinear relationship between neuronal responses and high-dimensional visual input. artificial neural networks have already improved our understanding of this system by allowing computational neuroscientists to create predictive models and bridge biological and machine vision. during the sensorium 2022, we introduced benchmarks for vision models with static input. however, animals operate and excel in dynamic environments, making it crucial to study and understand how the brain functions under these conditions. moreover, many biological theories, such as predictive coding, suggest that previous input is crucial for current input processing. currently, there is no standardized benchmark to identify state-of-the-art dynamic models of the mouse visual system. to address this gap, we propose the sensorium 2023 benchmark competition with dynamic input. it includes the collection of a new large-scale dataset from the primary visual cortex of ten mice, containing responses from over 78,000 neurons to over 2 hours of dynamic stimuli per neuron. participants in the main benchmark track will compete to identify the best predictive models of neuronal responses for dynamic input. we will also host a bonus track in which submission performance will be evaluated on out-of-domain input, using withheld neuronal responses to dynamic input stimuli whose statistics differ from the training set. both tracks will offer behavioral data along with video stimuli. as before, we will provide code, tutorials, and strong pre-trained baseline models to encourage participation. we hope this competition will continue to strengthen the accompanying sensorium benchmarks collection as a standard tool to measure progress in large-scale neural system identification models of the entire mouse visual hierarchy and beyond.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-31',\n",
       "  'updated': '2024-07-12',\n",
       "  'authors': ['polina turishcheva',\n",
       "   'paul g. fahey',\n",
       "   'laura hansel',\n",
       "   'rachel froebe',\n",
       "   'kayla ponder',\n",
       "   'michaela vystrčilová',\n",
       "   'konstantin f. willeke',\n",
       "   'mohammad bashiri',\n",
       "   'eric wang',\n",
       "   'zhiwei ding',\n",
       "   'andreas s. tolias',\n",
       "   'fabian h. sinz',\n",
       "   'alexander s. ecker'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.19654'},\n",
       " {'title': 'systematic review of methods and prognostic value of mitotic activity.   part 2: canine tumors',\n",
       "  'id': '2305.19667',\n",
       "  'abstract': 'one of the most relevant prognostication tests for tumors is cellular proliferation, which is most commonly measured by the mitotic activity in routine tumor sections. the goal of this systematic review is to scholarly analyze the methods and prognostic relevance of histologically measuring mitotic activity in canine tumors. a total of 137 articles that correlated the mitotic activity in canine tumors with patient outcome were identified through a systematic (pubmed and scopus) and manual (google scholar) literature search and eligibility screening process. these studies determined the mitotic count (mc, number of mitotic figures per tumor area) in 126 instances, presumably the mitotic count (method not specified) in 6 instances and the mitotic index (mi, proportion of mitotic figures per tumor cells) in 5 instances. a particularly high risk of bias was identified in the available details of the mc methods and statistical analysis, which often did not quantify the prognostic discriminative ability of the mc and only reported p-values. a significant association of the mc with survival was found in 72/109 (66%) studies. however, survival was evaluated by at least three studies in only 7 tumor types/groups, of which a prognostic relevance is apparent for mast cell tumors of the skin, cutaneous melanoma and soft tissue sarcoma of the skin. none of the studies on the mi found a prognostic relevance. further studies on the mc and mi with standardized methods are needed to prove the prognostic benefit of this test for further tumor types.',\n",
       "  'categories': 'q-bio.sc q-bio.to',\n",
       "  'doi': '10.1177/03009858241239565',\n",
       "  'created': '2023-05-31',\n",
       "  'updated': '',\n",
       "  'authors': ['christof a. bertram', 'taryn a. donovan', 'alexander bartel'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.19667'},\n",
       " {'title': 'accurate and efficient structural ensemble generation of macrocyclic   peptides using internal coordinate diffusion',\n",
       "  'id': '2305.19800',\n",
       "  'abstract': 'macrocyclic peptides are an emerging therapeutic modality, yet computational approaches for accurately sampling their diverse 3d ensembles remain challenging due to their conformational diversity and geometric constraints. here, we introduce ringer, a diffusion-based transformer model using a redundant internal coordinate representation that generates three-dimensional conformational ensembles of macrocyclic peptides from their 2d representations. ringer provides fast backbone and side-chain sampling while respecting key structural invariances of cyclic peptides. through extensive benchmarking and analysis against gold-standard conformer ensembles of cyclic peptides generated with metadynamics, we demonstrate how ringer generates both high-quality and diverse geometries at a fraction of the computational cost. our work lays the foundation for improved sampling of cyclic geometries and the development of geometric learning methods for peptides.',\n",
       "  'categories': 'q-bio.bm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-30',\n",
       "  'updated': '2024-08-13',\n",
       "  'authors': ['colin a. grambow',\n",
       "   'hayley weir',\n",
       "   'nathaniel l. diamant',\n",
       "   'gabriele scalia',\n",
       "   'tommaso biancalani',\n",
       "   'kangway v. chuang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.19800'},\n",
       " {'title': 'element-wise and recursive solutions for the power spectral density of   biological stochastic dynamical systems at fixed points',\n",
       "  'id': '2305.19890',\n",
       "  'abstract': \"stochasticity plays a central role in nearly every biological process, and the noise power spectral density (psd) is a critical tool for understanding variability and information processing in living systems. in steady-state, many such processes can be described by stochastic linear time-invariant (lti) systems driven by gaussian white noise, whose psd is a complex rational function of the frequency that can be concisely expressed in terms of their jacobian, dispersion, and diffusion matrices, fully defining the statistical properties of the system's dynamics at steady-state. here, we arrive at compact element-wise solutions of the rational function coefficients for the auto- and cross-spectrum that enable the explicit analytical computation of the psd in dimensions n=2,3,4. we further present a recursive leverrier-faddeev-type algorithm for the exact computation of the rational function coefficients. crucially, both solutions are free of matrix inverses. we illustrate our element-wise and recursive solutions by considering the stochastic dynamics of neural systems models, namely fitzhugh-nagumo (n=2), hindmarsh-rose (n=3), wilson-cowan (n=4), and the stabilized supralinear network (n=22), as well as an evolutionary game-theoretic model with mutations (n=5, 31). we extend our approach to derive a recursive method for calculating the coefficients in the power series expansion of the integrated covariance matrix for interacting spiking neurons modeled as hawkes processes on arbitrary directed graphs.\",\n",
       "  'categories': 'math.ds cond-mat.stat-mech q-bio.nc q-bio.pe',\n",
       "  'doi': '10.1103/physrevresearch.6.043179',\n",
       "  'created': '2023-05-31',\n",
       "  'updated': '2024-09-05',\n",
       "  'authors': ['shivang rawat', 'stefano martiniani'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2305.19890'},\n",
       " {'title': 'the ancestral selection graph for a $\\\\lambda$-asymmetric moran model',\n",
       "  'id': '2306.00130',\n",
       "  'abstract': 'motivated by the question of the impact of selective advantage in populations with skewed reproduction mechanims, we study a moran model with selection. we assume that there are two types of individuals, where the reproductive success of one type is larger than the other. the higher reproductive success may stem from either more frequent reproduction, or from larger numbers of offspring, and is encoded in a measure $\\\\lambda$ for each of the two types. our approach consists of constructing a $\\\\lambda$-asymmetric moran model in which individuals of the two populations compete, rather than considering a moran model for each population. under certain conditions, that we call the \"partial order of adaptation\", we can couple these measures. this allows us to construct the central object of this paper, the $\\\\lambda-$asymmetric ancestral selection graph, leading to a pathwise duality of the forward in time $\\\\lambda$-asymmetric moran model with its ancestral process. interestingly, the construction also provides a connection to the theory of optimal transport. we apply the ancestral selection graph in order to obtain scaling limits of the forward and backward processes, and note that the frequency process converges to the solution of an sde with discontinous paths. finally, we derive a griffiths representation for the generator of the sde and use it to find a semi-explicit formula for the probability of fixation of the less beneficial of the two types.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-31',\n",
       "  'updated': '2024-01-05',\n",
       "  'authors': ['adrián gonzález casanova', 'noemi kurt', 'josé luis pérez'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.00130'},\n",
       " {'title': \"muller's ratchet in a near-critical regime: tournament versus fitness   proportional selection\",\n",
       "  'id': '2306.00471',\n",
       "  'abstract': \"muller's ratchet, in its prototype version, models a haploid, asexual population whose size~$n$ is constant over the generations. slightly deleterious mutations are acquired along the lineages at a constant rate, and individuals carrying less mutations have a selective advantage. the classical variant considers {\\\\it fitness proportional} selection, but other fitness schemes are conceivable as well. inspired by the work of etheridge et al. ([epw09]) we propose a parameter scaling which fits well to the ``near-critical'' regime that was in the focus of [epw09] (and in which the mutation-selection ratio diverges logarithmically as $n\\\\to \\\\infty$). using a moran model, we investigate the``rule of thumb'' given in [epw09] for the click rate of the ``classical ratchet'' by putting it into the context of new results on the long-time evolution of the size of the best class of the ratchet with (binary) tournament selection, which (other than that of the classical ratchet) follows an autonomous dynamics up to the time of its extinction.   in [gsw23] it was discovered that the tournament ratchet has a hierarchy of dual processes which can be constructed on top of an ancestral selection graph with a poisson decoration. for a regime in which the mutation/selection-ratio remains bounded away from 1, this was used in [gsw23] to reveal the asymptotics of the click rates as well as that of the type frequency profile between clicks. we will describe how these ideas can be extended to the near-critical regime in which the mutation-selection ratio of the tournament ratchet converges to 1 as $n\\\\to \\\\infty$.\",\n",
       "  'categories': 'q-bio.pe math.pr',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-01',\n",
       "  'updated': '2024-06-18',\n",
       "  'authors': ['jan lukas igelbrink',\n",
       "   'adrián gonzález casanova',\n",
       "   'charline smadi',\n",
       "   'anton wakolbinger'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.00471'},\n",
       " {'title': 'stable developmental patterns of gene expression without morphogen   gradients',\n",
       "  'id': '2306.00537',\n",
       "  'abstract': 'gene expression patterns (geps) are established by cross-regulating target genes that interpret morphogen gradients. however, as development progresses, morphogen activity is reduced, leaving the emergent gep without stabilizing positional cues. the gep then can be deteriorated by the intrinsically noisy biochemical processes acting at the cellular level. however, the established geps remain spatio-temporally stable in many biological systems. here we combine spatial-stochastic simulations with an enhanced sampling method (non-stationary forward flux sampling) and a recently developed stability theory to address how spatiotemporal integrity of a gep is maintained without morphogen gradients. using a minimal embryo model consisting of spatially coupled biochemical reactor volumes, we study a stripe pattern in which weak cross-repression between nearest neighbor domians alternates with strong repression between next-nearest neighbor domains, inspired by the gap gene system in the drosophila embryo. we find that fine-tuning of the weak repressive interactions to an optimal level increases temporal stability of geps by orders of magnitude, providing stability over developmentally relevant times, without morphogen gradients. the numerically determined optimal parameter regime closely agrees with the predictions of the stability theory. by analizing the dynamics of gep asymmetry factors, we trace back the pattern stability enhancement to the emergence of a metastable basin and restoring forces that counteract pattern perturbations. the origin of these forces is further explained by the effective model, describing the emergent deterministic dynamics of the system. altogether, we show that metastable attractors can emerge as a property of stochastic geps even without system-wide positional cues, provided that the gene regulatory interactions shaping the pattern are optimally tuned.',\n",
       "  'categories': 'physics.bio-ph cond-mat.stat-mech nlin.ps q-bio.cb q-bio.mn',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-01',\n",
       "  'updated': '2024-08-27',\n",
       "  'authors': ['maciej majka',\n",
       "   'nils b. becker',\n",
       "   'pieter rein ten wolde',\n",
       "   'marcin zagorski',\n",
       "   'thomas r. sokolowski'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.00537'},\n",
       " {'title': 'demixing fluorescence time traces transmitted by multimode fibers',\n",
       "  'id': '2306.00695',\n",
       "  'abstract': 'optical methods based on thin multimode fibers (mmfs) are promising tools for measuring neuronal activity in deep brain regions of freely moving mice thanks to their small diameter. however, current methods are limited: while fiber photometry provides only ensemble activity, imaging techniques using of long multimode fibers are very sensitive to bending and have not been applied to unrestrained rodents yet. here, we demonstrate the fundamentals of a new approach using a short mmf coupled to a miniscope. in proof-of-principle in vitro experiments, we disentangled spatio-temporal fluorescence signals from multiple fluorescent sources transmitted by a thin (200 \\\\mu m) and short (8 mm) mmf, using a general unconstrained non-negative matrix factorization algorithm directly on the raw video data. furthermore, we show that low-cost open-source miniscopes have sufficient sensitivity to image the same fluorescence patterns seen in our proof-of-principle experiment, suggesting a new avenue for novel minimally invasive deep brain studies using multimode fibers in freely behaving mice.',\n",
       "  'categories': 'physics.optics physics.app-ph physics.bio-ph q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-01',\n",
       "  'updated': '2024-07-01',\n",
       "  'authors': ['caio vaz rimoli',\n",
       "   'claudio moretti',\n",
       "   'fernando soldevila',\n",
       "   'enora brémont',\n",
       "   'cathie ventalon',\n",
       "   'sylvain gigan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.00695'},\n",
       " {'title': 'the brain tumor segmentation (brats-mets) challenge 2023: brain   metastasis segmentation on pre-treatment mri',\n",
       "  'id': '2306.00838',\n",
       "  'abstract': 'the translation of ai-generated brain metastases (bm) segmentation into clinical practice relies heavily on diverse, high-quality annotated medical imaging datasets. the brats-mets 2023 challenge has gained momentum for testing and benchmarking algorithms using rigorously annotated internationally compiled real-world datasets. this study presents the results of the segmentation challenge and characterizes the challenging cases that impacted the performance of the winning algorithms. untreated brain metastases on standard anatomic mri sequences (t1, t2, flair, t1pg) from eight contributed international datasets were annotated in stepwise method: published unet algorithms, student, neuroradiologist, final approver neuroradiologist. segmentations were ranked based on lesion-wise dice and hausdorff distance (hd95) scores. false positives (fp) and false negatives (fn) were rigorously penalized, receiving a score of 0 for dice and a fixed penalty of 374 for hd95. eight datasets comprising 1303 studies were annotated, with 402 studies (3076 lesions) released on synapse as publicly available datasets to challenge competitors. additionally, 31 studies (139 lesions) were held out for validation, and 59 studies (218 lesions) were used for testing. segmentation accuracy was measured as rank across subjects, with the winning team achieving a lesionwise mean score of 7.9. common errors among the leading teams included false negatives for small lesions and misregistration of masks in space.the brats-mets 2023 challenge successfully curated well-annotated, diverse datasets and identified common errors, facilitating the translation of bm segmentation across varied clinical environments and providing personalized volumetric reports to patients undergoing bm treatment.',\n",
       "  'categories': 'q-bio.ot eess.iv',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-01',\n",
       "  'updated': '2024-06-17',\n",
       "  'authors': ['ahmed w. moawad',\n",
       "   'anastasia janas',\n",
       "   'ujjwal baid',\n",
       "   'divya ramakrishnan',\n",
       "   'rachit saluja',\n",
       "   'nader ashraf',\n",
       "   'leon jekel',\n",
       "   'raisa amiruddin',\n",
       "   'maruf adewole',\n",
       "   'jake albrecht',\n",
       "   'udunna anazodo',\n",
       "   'sanjay aneja',\n",
       "   'syed muhammad anwar',\n",
       "   'timothy bergquist',\n",
       "   'evan calabrese',\n",
       "   'veronica chiang',\n",
       "   'verena chung',\n",
       "   'gian marco marco conte',\n",
       "   'farouk dako',\n",
       "   'james eddy',\n",
       "   'ivan ezhov',\n",
       "   'ariana familiar',\n",
       "   'keyvan farahani',\n",
       "   'juan eugenio iglesias',\n",
       "   'zhifan jiang',\n",
       "   'elaine johanson',\n",
       "   'anahita fathi kazerooni',\n",
       "   'florian kofler',\n",
       "   'kiril krantchev',\n",
       "   'dominic labella',\n",
       "   'koen van leemput',\n",
       "   'hongwei bran li',\n",
       "   'marius george linguraru',\n",
       "   'katherine e. link',\n",
       "   'xinyang liu',\n",
       "   'nazanin maleki',\n",
       "   'zeke meier',\n",
       "   'bjoern h menze',\n",
       "   'harrison moy',\n",
       "   'klara osenberg',\n",
       "   'marie piraud',\n",
       "   'zachary reitman',\n",
       "   'russel takeshi shinohara',\n",
       "   'nourel hoda tahon',\n",
       "   'ayman nada',\n",
       "   'yuri s. velichko',\n",
       "   'chunhao wang',\n",
       "   'benedikt wiestler',\n",
       "   'walter wiggins',\n",
       "   'umber shafique',\n",
       "   'klara willms',\n",
       "   'arman avesta',\n",
       "   'khaled bousabarah',\n",
       "   'satrajit chakrabarty',\n",
       "   'nicolo gennaro',\n",
       "   'wolfgang holler',\n",
       "   'manpreet kaur',\n",
       "   'pamela lamontagne',\n",
       "   'mingde lin',\n",
       "   'jan lost',\n",
       "   'daniel s. marcus',\n",
       "   'ryan maresca',\n",
       "   'sarah merkaj',\n",
       "   'ayaman nada',\n",
       "   'gabriel cassinelli pedersen',\n",
       "   'marc von reppert',\n",
       "   'aristeidis sotiras',\n",
       "   'oleg teytelboym',\n",
       "   'niklas tillmans',\n",
       "   'malte westerhoff',\n",
       "   'ayda youssef',\n",
       "   'devon godfrey',\n",
       "   'scott floyd',\n",
       "   'andreas rauschecker',\n",
       "   'javier villanueva-meyer',\n",
       "   'irada pflüger',\n",
       "   'jaeyoung cho',\n",
       "   'martin bendszus',\n",
       "   'gianluca brugnara',\n",
       "   'justin cramer',\n",
       "   'gloria j. guzman perez-carillo',\n",
       "   'derek r. johnson',\n",
       "   'anthony kam',\n",
       "   'benjamin yin ming kwan',\n",
       "   'lillian lai',\n",
       "   'neil u. lall',\n",
       "   'fatima memon',\n",
       "   'satya narayana patro',\n",
       "   'bojan petrovic',\n",
       "   'tiffany y. so',\n",
       "   'gerard thompson',\n",
       "   'lei wu',\n",
       "   'e. brooke schrickel',\n",
       "   'anu bansal',\n",
       "   'frederik barkhof',\n",
       "   'cristina besada',\n",
       "   'sammy chu',\n",
       "   'jason druzgal',\n",
       "   'alexandru dusoi',\n",
       "   'luciano farage',\n",
       "   'fabricio feltrin',\n",
       "   'amy fong',\n",
       "   'steve h. fung',\n",
       "   'r. ian gray',\n",
       "   'ichiro ikuta',\n",
       "   'michael iv',\n",
       "   'alida a. postma',\n",
       "   'amit mahajan',\n",
       "   'david joyner',\n",
       "   'chase krumpelman',\n",
       "   'laurent letourneau-guillon',\n",
       "   'christie m. lincoln',\n",
       "   'mate e. maros',\n",
       "   'elka miller',\n",
       "   'fanny morón',\n",
       "   'esther a. nimchinsky',\n",
       "   'ozkan ozsarlak',\n",
       "   'uresh patel',\n",
       "   'saurabh rohatgi',\n",
       "   'atin saha',\n",
       "   'anousheh sayah',\n",
       "   'eric d. schwartz',\n",
       "   'robert shih',\n",
       "   'mark s. shiroishi',\n",
       "   'juan e. small',\n",
       "   'manoj tanwar',\n",
       "   'jewels valerie',\n",
       "   'brent d. weinberg',\n",
       "   'matthew l. white',\n",
       "   'robert young',\n",
       "   'vahe m. zohrabian',\n",
       "   'aynur azizova',\n",
       "   'melanie maria theresa brüßeler',\n",
       "   'pascal fehringer',\n",
       "   'mohanad ghonim',\n",
       "   'mohamed ghonim',\n",
       "   'athanasios gkampenis',\n",
       "   'abdullah okar',\n",
       "   'luca pasquini',\n",
       "   'yasaman sharifi',\n",
       "   'gagandeep singh',\n",
       "   'nico sollmann',\n",
       "   'theodora soumala',\n",
       "   'mahsa taherzadeh',\n",
       "   'nikolay yordanov',\n",
       "   'philipp vollmuth',\n",
       "   'martha foltyn-dumitru',\n",
       "   'ajay malhotra',\n",
       "   'aly h. abayazeed',\n",
       "   'francesco dellepiane',\n",
       "   'philipp lohmann',\n",
       "   'víctor m. pérez-garcía',\n",
       "   'hesham elhalawani',\n",
       "   'sanaria al-rubaiey',\n",
       "   'rui duarte armindo',\n",
       "   'kholod ashraf',\n",
       "   'moamen m. asla',\n",
       "   'mohamed badawy',\n",
       "   'jeroen bisschop',\n",
       "   'nima broomand lomer',\n",
       "   'jan bukatz',\n",
       "   'jim chen',\n",
       "   'petra cimflova',\n",
       "   'felix corr',\n",
       "   'alexis crawley',\n",
       "   'lisa deptula',\n",
       "   'tasneem elakhdar',\n",
       "   'islam h. shawali',\n",
       "   'shahriar faghani',\n",
       "   'alexandra frick',\n",
       "   'vaibhav gulati',\n",
       "   'muhammad ammar haider',\n",
       "   'fátima hierro',\n",
       "   'rasmus holmboe dahl',\n",
       "   'sarah maria jacobs',\n",
       "   'kuang-chun jim hsieh',\n",
       "   'sedat g. kandemirli',\n",
       "   'katharina kersting',\n",
       "   'laura kida',\n",
       "   'sofia kollia',\n",
       "   'ioannis koukoulithras',\n",
       "   'xiao li',\n",
       "   'ahmed abouelatta',\n",
       "   'aya mansour',\n",
       "   'ruxandra-catrinel maria-zamfirescu',\n",
       "   'marcela marsiglia',\n",
       "   'yohana sarahi mateo-camacho',\n",
       "   'mark mcarthur',\n",
       "   'olivia mcdonnell',\n",
       "   'maire mchugh',\n",
       "   'mana moassefi',\n",
       "   'samah mostafa morsi',\n",
       "   'alexander muntenu',\n",
       "   'khanak k. nandolia',\n",
       "   'syed raza naqvi',\n",
       "   'yalda nikanpour',\n",
       "   'mostafa alnoury',\n",
       "   'abdullah mohamed aly nouh',\n",
       "   'francesca pappafava',\n",
       "   'markand d. patel',\n",
       "   'samantha petrucci',\n",
       "   'eric rawie',\n",
       "   'scott raymond',\n",
       "   'borna roohani',\n",
       "   'sadeq sabouhi',\n",
       "   'laura m. sanchez-garcia',\n",
       "   'zoe shaked',\n",
       "   'pokhraj p. suthar',\n",
       "   'talissa altes',\n",
       "   'edvin isufi',\n",
       "   'yaseen dhermesh',\n",
       "   'jaime gass',\n",
       "   'jonathan thacker',\n",
       "   'abdul rahman tarabishy',\n",
       "   'benjamin turner',\n",
       "   'sebastiano vacca',\n",
       "   'george k. vilanilam',\n",
       "   'daniel warren',\n",
       "   'david weiss',\n",
       "   'klara willms',\n",
       "   'fikadu worede',\n",
       "   'sara yousry',\n",
       "   'wondwossen lerebo',\n",
       "   'alejandro aristizabal',\n",
       "   'alexandros karargyris',\n",
       "   'hasan kassem',\n",
       "   'sarthak pati',\n",
       "   'micah sheller',\n",
       "   'spyridon bakas',\n",
       "   'jeffrey d. rudie',\n",
       "   'mariam aboian'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.00838'},\n",
       " {'title': 'suppression of chaos in a partially driven recurrent neural network',\n",
       "  'id': '2306.00900',\n",
       "  'abstract': 'the dynamics of recurrent neural networks (rnns), and particularly their response to inputs, play a critical role in information processing. in many applications of rnns, only a specific subset of the neurons generally receive inputs. however, it remains to be theoretically clarified how the restriction of the input to a specific subset of neurons affects the network dynamics. considering rnns with such restricted input, we investigate how the proportion, $p$, of the neurons receiving inputs (the \"inputs neurons\") and the strength of the input signals affect the dynamics by analytically deriving the conditional maximum lyapunov exponent. our results show that for sufficiently large $p$, the maximum lyapunov exponent decreases monotonically as a function of the input strength, indicating the suppression of chaos, but if $p$ is smaller than a critical threshold, $p_c$, even significantly amplified inputs cannot suppress spontaneous chaotic dynamics. furthermore, although the value of $p_c$ is seemingly dependent on several model parameters, such as the sparseness and strength of recurrent connections, it is proved to be intrinsically determined solely by the strength of chaos in spontaneous activity of the rnn. this is to say, despite changes in these model parameters, it is possible to represent the value of $p_c$ as a common invariant function by appropriately scaling these parameters to yield the same strength of spontaneous chaos. our study suggests that if $p$ is above $p_c$, we can bring the neural network to the edge of chaos, thereby maximizing its information processing capacity, by amplifying inputs.',\n",
       "  'categories': 'q-bio.nc cond-mat.dis-nn nlin.cd',\n",
       "  'doi': '10.1103/physrevresearch.6.013172',\n",
       "  'created': '2023-06-01',\n",
       "  'updated': '2024-01-23',\n",
       "  'authors': ['shotaro takasu', 'toshio aoyagi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.00900'},\n",
       " {'title': 'long-range feedback spiking network captures dynamic and static   representations of the visual cortex under movie stimuli',\n",
       "  'id': '2306.01354',\n",
       "  'abstract': 'deep neural networks (dnns) are widely used models for investigating biological visual representations. however, existing dnns are mostly designed to analyze neural responses to static images, relying on feedforward structures and lacking physiological neuronal mechanisms. there is limited insight into how the visual cortex represents natural movie stimuli that contain context-rich information. to address these problems, this work proposes the long-range feedback spiking network (lorafb-snet), which mimics top-down connections between cortical regions and incorporates spike information processing mechanisms inherent to biological neurons. taking into account the temporal dependence of representations under movie stimuli, we present time-series representational similarity analysis (tsrsa) to measure the similarity between model representations and visual cortical representations of mice. lorafb-snet exhibits the highest level of representational similarity, outperforming other well-known and leading alternatives across various experimental paradigms, especially when representing long movie stimuli. we further conduct experiments to quantify how temporal structures (dynamic information) and static textures (static information) of the movie stimuli influence representational similarity, suggesting that our model benefits from long-range feedback to encode context-dependent representations just like the brain. altogether, lorafb-snet is highly competent in capturing both dynamic and static representations of the mouse visual cortex and contributes to the understanding of movie processing mechanisms of the visual system. our codes are available at https://github.com/grasshlw/snn-neural-similarity-movie.',\n",
       "  'categories': 'cs.ne q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-02',\n",
       "  'updated': '2024-11-01',\n",
       "  'authors': ['liwei huang',\n",
       "   'zhengyu ma',\n",
       "   'liutao yu',\n",
       "   'huihui zhou',\n",
       "   'yonghong tian'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.01354'},\n",
       " {'title': 'dynamical theory for adaptive systems',\n",
       "  'id': '2306.01403',\n",
       "  'abstract': \"the study of adaptive dynamics, involving many degrees of freedom on two separated timescales, one for fast changes of state variables and another for the slow adaptation of parameters controlling the former's dynamics is crucial for understanding feedback mechanisms underlying evolution and learning. we present a path-integral approach \\\\`a la martin-siggia-rose-de dominicis-janssen (msrdj) to analyse nonequilibrium phase transitions in such dynamical systems. as an illustration, we apply our framework to the adaptation of gene-regulatory networks under a dynamic genotype-phenotype map: phenotypic variations are shaped by the fast stochastic gene-expression dynamics and are coupled to the slowly evolving distribution of genotypes, each encoded by a network structure. we establish that under this map, genotypes corresponding to reciprocal networks of coherent feedback loops are selected within an intermediate range of environmental noise, leading to phenotypic robustness.\",\n",
       "  'categories': 'q-bio.pe cond-mat.dis-nn nlin.ao physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-02',\n",
       "  'updated': '2024-08-05',\n",
       "  'authors': ['tuan minh pham', 'kunihiko kaneko'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.01403'},\n",
       " {'title': 'generalist equivariant transformer towards 3d molecular interaction   learning',\n",
       "  'id': '2306.01474',\n",
       "  'abstract': 'many processes in biology and drug discovery involve various 3d interactions between molecules, such as protein and protein, protein and small molecule, etc. given that different molecules are usually represented in different granularity, existing methods usually encode each type of molecules independently with different models, leaving it defective to learn the various underlying interaction physics. in this paper, we first propose to universally represent an arbitrary 3d complex as a geometric graph of sets, shedding light on encoding all types of molecules with one model. we then propose a generalist equivariant transformer (get) to effectively capture both domain-specific hierarchies and domain-agnostic interaction physics. to be specific, get consists of a bilevel attention module, a feed-forward module and a layer normalization module, where each module is e(3) equivariant and specialized for handling sets of variable sizes. notably, in contrast to conventional pooling-based hierarchical models, our get is able to retain fine-grained information of all levels. extensive experiments on the interactions between proteins, small molecules and rna/dnas verify the effectiveness and generalization capability of our proposed method across different domains.',\n",
       "  'categories': 'cs.lg q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-02',\n",
       "  'updated': '2024-05-29',\n",
       "  'authors': ['xiangzhe kong', 'wenbing huang', 'yang liu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.01474'},\n",
       " {'title': 'bi-level contrastive learning for knowledge-enhanced molecule   representations',\n",
       "  'id': '2306.01631',\n",
       "  'abstract': 'molecule representation learning is crucial for various downstream applications, such as understanding and predicting molecular properties and side effects. in this paper, we propose a novel method called gode, which takes into account the two-level structure of individual molecules. we recognize that molecules have an intrinsic graph structure as well as being a node in a larger molecule knowledge graph. gode integrates graph representations of individual molecules with multidomain biochemical data from knowledge graphs. by pre-training two graph neural networks (gnns) on different graph structures, combined with contrastive learning, gode fuses molecular structures with their corresponding knowledge graph substructures. this fusion results in a more robust and informative representation, which enhances molecular property prediction by harnessing both chemical and biological information. when fine-tuned across 11 chemical property tasks, our model outperforms existing benchmarks, registering an average roc-auc uplift of 13.8% for classification tasks and an average rmse/mae enhancement of 35.1% for regression tasks. impressively, it surpasses the current leading model in molecule property predictions with average advancements of 2.1% in classification and 6.4% in regression tasks.',\n",
       "  'categories': 'cs.lg cs.ai q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-02',\n",
       "  'updated': '2024-01-19',\n",
       "  'authors': ['pengcheng jiang', 'cao xiao', 'tianfan fu', 'jimeng sun'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.01631'},\n",
       " {'title': 'biomarker discovery with quantum neural networks: a case-study in   ctla4-activation pathways',\n",
       "  'id': '2306.01745',\n",
       "  'abstract': 'biomarker discovery is a challenging task due to the massive search space. quantum computing and quantum artificial intelligence (quantum ai) can be used to address the computational problem of biomarker discovery tasks. we propose a quantum neural networks (qnns) architecture to discover biomarkers for input activation pathways. the maximum relevance, minimum redundancy (mrmr) criteria is used to score biomarker candidate sets. our proposed model is economical since the neural solution can be delivered on constrained hardware. we demonstrate the proof of concept on four activation pathways associated with ctla4, including (1) ctla4-activation stand-alone, (2) ctla4-cd8a-cd8b co-activation, (3) ctla4-cd2 co-activation, and (4) ctla4-cd2-cd48-cd53-cd58-cd84 co-activation. the model indicates new biomarkers associated with the mutational activation of clta4-associated pathways, including 20 genes: clic4, cpe, ets2, fam107a, gpr116, hyou1, lcn2, macf1, mt1g, napa, ndufs5, pak1, pfn1, pgap3, ppm1g, psmd8, rnf213, slc25a3, uba1, and wls. we open source the implementation at: https://github.com/namnguyen0510/biomarker-discovery-with-quantum-neural-networks.',\n",
       "  'categories': 'q-bio.qm cs.ai',\n",
       "  'doi': '10.1186/s12859-024-05755-0',\n",
       "  'created': '2023-05-15',\n",
       "  'updated': '2024-02-12',\n",
       "  'authors': ['nam nguyen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.01745'},\n",
       " {'title': 'diffpack: a torsional diffusion model for autoregressive protein   side-chain packing',\n",
       "  'id': '2306.01794',\n",
       "  'abstract': 'proteins play a critical role in carrying out biological functions, and their 3d structures are essential in determining their functions. accurately predicting the conformation of protein side-chains given their backbones is important for applications in protein structure prediction, design and protein-protein interactions. traditional methods are computationally intensive and have limited accuracy, while existing machine learning methods treat the problem as a regression task and overlook the restrictions imposed by the constant covalent bond lengths and angles. in this work, we present diffpack, a torsional diffusion model that learns the joint distribution of side-chain torsional angles, the only degrees of freedom in side-chain packing, by diffusing and denoising on the torsional space. to avoid issues arising from simultaneous perturbation of all four torsional angles, we propose autoregressively generating the four torsional angles from $\\\\chi_1$ to $\\\\chi_4$ and training diffusion models for each torsional angle. we evaluate the method on several benchmarks for protein side-chain packing and show that our method achieves improvements of $11.9\\\\%$ and $13.5\\\\%$ in angle accuracy on casp13 and casp14, respectively, with a significantly smaller model size ($60\\\\times$ fewer parameters). additionally, we show the effectiveness of our method in enhancing side-chain predictions in the alphafold2 model. code is available at https://github.com/deepgraphlearning/diffpack.',\n",
       "  'categories': 'q-bio.qm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-01',\n",
       "  'updated': '2024-02-15',\n",
       "  'authors': ['yangtian zhang',\n",
       "   'zuobai zhang',\n",
       "   'bozitao zhong',\n",
       "   'sanchit misra',\n",
       "   'jian tang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.01794'},\n",
       " {'title': 'cellprofiler plugins -- an easy image analysis platform integration for   containers and python tools',\n",
       "  'id': '2306.01915',\n",
       "  'abstract': 'cellprofiler is a widely used software for creating reproducible, reusable image analysis workflows without needing to code. in addition to the >90 modules that make up the main cellprofiler program, cellprofiler has a plugins system that allows for the creation of new modules which integrate with other python tools or tools that are packaged in software containers. the cellprofiler-plugins repository contains a number of these cellprofiler modules, especially modules that are experimental and/or dependency-heavy. here, we present an upgraded cellprofiler-plugins repository, an example of accessing containerized tools, improved documentation, and added citation/reference tools to facilitate the use and contribution of the community.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.1111/jmi.13223',\n",
       "  'created': '2023-06-02',\n",
       "  'updated': '2023-08-15',\n",
       "  'authors': ['erin weisbart',\n",
       "   'callum tromans-coia',\n",
       "   'barbara diaz-rohrer',\n",
       "   'david r stirling',\n",
       "   'fernanda garcia-fossa',\n",
       "   'rebecca a senft',\n",
       "   'mark c hiner',\n",
       "   'marcelo b de jesus',\n",
       "   'kevin w eliceiri',\n",
       "   'beth a cimini'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.01915'},\n",
       " {'title': 'brainlife.io: a decentralized and open source cloud platform to support   neuroscience research',\n",
       "  'id': '2306.02183',\n",
       "  'abstract': \"neuroscience research has expanded dramatically over the past 30 years by advancing standardization and tool development to support rigor and transparency. consequently, the complexity of the data pipeline has also increased, hindering access to fair (findable, accessible, interoperabile, and reusable) data analysis to portions of the worldwide research community. brainlife.io was developed to reduce these burdens and democratize modern neuroscience research across institutions and career levels. using community software and hardware infrastructure, the platform provides open-source data standardization, management, visualization, and processing and simplifies the data pipeline. brainlife.io automatically tracks the provenance history of thousands of data objects, supporting simplicity, efficiency, and transparency in neuroscience research. here brainlife.io's technology and data services are described and evaluated for validity, reliability, reproducibility, replicability, and scientific utility. using data from 4 modalities and 3,200 participants, we demonstrate that brainlife.io's services produce outputs that adhere to best practices in modern neuroscience research.\",\n",
       "  'categories': 'cs.dc q-bio.nc q-bio.qm',\n",
       "  'doi': '10.1038/s41592-024-02237-2',\n",
       "  'created': '2023-06-03',\n",
       "  'updated': '2023-08-11',\n",
       "  'authors': ['soichi hayashi',\n",
       "   'bradley a. caron',\n",
       "   'anibal sólon heinsfeld',\n",
       "   'sophia vinci-booher',\n",
       "   'brent mcpherson',\n",
       "   'daniel n. bullock',\n",
       "   'giulia bertò',\n",
       "   'guiomar niso',\n",
       "   'sandra hanekamp',\n",
       "   'daniel levitas',\n",
       "   'kimberly ray',\n",
       "   'anne mackenzie',\n",
       "   'lindsey kitchell',\n",
       "   'josiah k. leong',\n",
       "   'filipi nascimento-silva',\n",
       "   'serge koudoro',\n",
       "   'hanna willis',\n",
       "   'jasleen k. jolly',\n",
       "   'derek pisner',\n",
       "   'taylor r. zuidema',\n",
       "   'jan w. kurzawski',\n",
       "   'kyriaki mikellidou',\n",
       "   'aurore bussalb',\n",
       "   'christopher rorden',\n",
       "   'conner victory',\n",
       "   'dheeraj bhatia',\n",
       "   'dogu baran aydogan',\n",
       "   'fang-cheng f. yeh',\n",
       "   'franco delogu',\n",
       "   'javier guaje',\n",
       "   'jelle veraart',\n",
       "   'steffen bollman',\n",
       "   'ashley stewart',\n",
       "   'jeremy fischer',\n",
       "   'joshua faskowitz',\n",
       "   'maximilien chaumon',\n",
       "   'ricardo fabrega',\n",
       "   'david hunt',\n",
       "   'shawn mckee',\n",
       "   'shawn t. brown',\n",
       "   'stephanie heyman',\n",
       "   'vittorio iacovella',\n",
       "   'amanda f. mejia',\n",
       "   'daniele marinazzo',\n",
       "   'r. cameron craddock',\n",
       "   'emanuele olivetti',\n",
       "   'jamie l. hanson',\n",
       "   'paolo avesani',\n",
       "   'eleftherios garyfallidis',\n",
       "   'dan stanzione',\n",
       "   'james carson',\n",
       "   'robert henschel',\n",
       "   'david y. hancock',\n",
       "   'craig a. stewart',\n",
       "   'david schnyer',\n",
       "   'damian o. eke',\n",
       "   'russell a. poldrack',\n",
       "   'nathalie george',\n",
       "   'holly bridge',\n",
       "   'ilaria sani',\n",
       "   'winrich a. freiwald',\n",
       "   'aina puce',\n",
       "   'nicholas l. port',\n",
       "   'franco pestilli'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.02183'},\n",
       " {'title': 'intermittent migration can induce pulses of speciation in a two-island   system',\n",
       "  'id': '2306.02510',\n",
       "  'abstract': 'geographic barriers prevent migration between populations, thereby facilitating speciation through allopatry. however, these barriers can exhibit dynamic behavior in nature, promoting cycles of expansion and contraction of populations. such oscillations cause temporal variations in migration that do not necessarily prevent speciation; on the contrary, they have been suggested as a driving force for diversification. here we present a study on a two-island neutral speciation model in scenarios with intermittent migration driven by sea-level fluctuations. mating is constrained to genetically compatible individuals inhabiting the same island, and offspring inherit nuclear genomes from both parents with recombination. we observe pulses of speciation that would not occur in strict isolation or continuous migration. according to the seabed height, which modulates the duration of the isolation and connection periods, the maximum richness occurs at different times and in an ephemeral fashion. the expansion-contraction dynamics can accelerate diversification, but a long time in isolation can reduce the richness to one species per island, resembling patterns described by the taxon pulse hypothesis of diversification. together with other studies, our results support the relevance of research on the impact of variable migration on diversification, suggested to be related to regions of high diversity.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1093/evolut/qpad210',\n",
       "  'created': '2023-06-04',\n",
       "  'updated': '',\n",
       "  'authors': ['débora princepe',\n",
       "   'simone czarnobai',\n",
       "   'rodrigo a. caetano',\n",
       "   'flavia m. d. marquitti',\n",
       "   'marcus a. m. de aguiar',\n",
       "   'sabrina b. l. araujo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.02510'},\n",
       " {'title': 'automated importance sampling via optimal control for stochastic   reaction networks: a markovian projection-based approach',\n",
       "  'id': '2306.02660',\n",
       "  'abstract': 'we propose a novel alternative approach to our previous work (ben hammouda et al., 2023) to improve the efficiency of monte carlo (mc) estimators for rare event probabilities for stochastic reaction networks (srns). in the same spirit of (ben hammouda et al., 2023), an efficient path-dependent measure change is derived based on a connection between determining optimal importance sampling (is) parameters within a class of probability measures and a stochastic optimal control formulation, corresponding to solving a variance minimization problem. in this work, we propose a novel approach to address the encountered curse of dimensionality by mapping the problem to a significantly lower-dimensional space via a markovian projection (mp) idea. the output of this model reduction technique is a low-dimensional srn (potentially even one dimensional) that preserves the marginal distribution of the original high-dimensional srn system. the dynamics of the projected process are obtained by solving a related optimization problem via a discrete $l^2$ regression. by solving the resulting projected hamilton-jacobi-bellman (hjb) equations for the reduced-dimensional srn, we obtain projected is parameters, which are then mapped back to the original full-dimensional srn system, resulting in an efficient is-mc estimator for rare events probabilities of the full-dimensional srn. our analysis and numerical experiments reveal that the proposed mp-hjb-is approach substantially reduces the mc estimator variance, resulting in a lower computational complexity in the rare event regime than standard mc estimators.',\n",
       "  'categories': 'math.na cs.na math.oc q-bio.mn q-bio.qm stat.co',\n",
       "  'doi': '10.1016/j.cam.2024.115853',\n",
       "  'created': '2023-06-05',\n",
       "  'updated': '2024-03-11',\n",
       "  'authors': ['chiheb ben hammouda',\n",
       "   'nadhir ben rached',\n",
       "   'raúl tempone',\n",
       "   'sophia wiechert'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.02660'},\n",
       " {'title': 'bootstrapped training of score-conditioned generator for offline design   of biological sequences',\n",
       "  'id': '2306.03111',\n",
       "  'abstract': 'we study the problem of optimizing biological sequences, e.g., proteins, dna, and rna, to maximize a black-box score function that is only evaluated in an offline dataset. we propose a novel solution, bootstrapped training of score-conditioned generator (bootgen) algorithm. our algorithm repeats a two-stage process. in the first stage, our algorithm trains the biological sequence generator with rank-based weights to enhance the accuracy of sequence generation based on high scores. the subsequent stage involves bootstrapping, which augments the training dataset with self-generated data labeled by a proxy score function. our key idea is to align the score-based generation with a proxy score function, which distills the knowledge of the proxy score function to the generator. after training, we aggregate samples from multiple bootstrapped generators and proxies to produce a diverse design. extensive experiments show that our method outperforms competitive baselines on biological sequential design tasks. we provide reproducible source code: \\\\href{https://github.com/kaist-silab/bootgen}{https://github.com/kaist-silab/bootgen}.',\n",
       "  'categories': 'q-bio.qm cs.lg stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-05',\n",
       "  'updated': '2024-03-22',\n",
       "  'authors': ['minsu kim', 'federico berto', 'sungsoo ahn', 'jinkyoo park'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.03111'},\n",
       " {'title': 'str2str: a score-based framework for zero-shot protein conformation   sampling',\n",
       "  'id': '2306.03117',\n",
       "  'abstract': 'the dynamic nature of proteins is crucial for determining their biological functions and properties, for which monte carlo (mc) and molecular dynamics (md) simulations stand as predominant tools to study such phenomena. by utilizing empirically derived force fields, mc or md simulations explore the conformational space through numerically evolving the system via markov chain or newtonian mechanics. however, the high-energy barrier of the force fields can hamper the exploration of both methods by the rare event, resulting in inadequately sampled ensemble without exhaustive running. existing learning-based approaches perform direct sampling yet heavily rely on target-specific simulation data for training, which suffers from high data acquisition cost and poor generalizability. inspired by simulated annealing, we propose str2str, a novel structure-to-structure translation framework capable of zero-shot conformation sampling with roto-translation equivariant property. our method leverages an amortized denoising score matching objective trained on general crystal structures and has no reliance on simulation data during both training and inference. experimental results across several benchmarking protein systems demonstrate that str2str outperforms previous state-of-the-art generative structure prediction models and can be orders of magnitude faster compared to long md simulations. our open-source implementation is available at https://github.com/lujiarui/str2str',\n",
       "  'categories': 'q-bio.qm cs.lg q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-05',\n",
       "  'updated': '2024-03-11',\n",
       "  'authors': ['jiarui lu', 'bozitao zhong', 'zuobai zhang', 'jian tang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.03117'},\n",
       " {'title': 'optimal transport for automatic alignment of untargeted metabolomic data',\n",
       "  'id': '2306.03218',\n",
       "  'abstract': 'untargeted metabolomic profiling through liquid chromatography-mass spectrometry (lc-ms) measures a vast array of metabolites within biospecimens, advancing drug development, disease diagnosis, and risk prediction. however, the low throughput of lc-ms poses a major challenge for biomarker discovery, annotation, and experimental comparison, necessitating the merging of multiple datasets. current data pooling methods encounter practical limitations due to their vulnerability to data variations and hyperparameter dependence. here we introduce gromovmatcher, a flexible and user-friendly algorithm that automatically combines lc-ms datasets using optimal transport. by capitalizing on feature intensity correlation structures, gromovmatcher delivers superior alignment accuracy and robustness compared to existing approaches. this algorithm scales to thousands of features requiring minimal hyperparameter tuning. manually curated datasets for validating alignment algorithms are limited in the field of untargeted metabolomics, and hence we develop a dataset split procedure to generate pairs of validation datasets to test the alignments produced by gromovmatcher and other methods. applying our method to experimental patient studies of liver and pancreatic cancer, we discover shared metabolic features related to patient alcohol intake, demonstrating how gromovmatcher facilitates the search for biomarkers associated with lifestyle risk factors linked to several cancer types.',\n",
       "  'categories': 'q-bio.qm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-05',\n",
       "  'updated': '2024-05-24',\n",
       "  'authors': ['marie breeur',\n",
       "   'george stepaniants',\n",
       "   'pekka keski-rahkonen',\n",
       "   'philippe rigollet',\n",
       "   'vivian viallon'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.03218'},\n",
       " {'title': 'form follows function: a different approach to neuron connectivity',\n",
       "  'id': '2306.03337',\n",
       "  'abstract': 'a different method of discovering how neurons are connected to process information is presented here: design a simple logic circuit that can perform a single, biologically advantageous function. engineering concepts can be helpful in choosing the function and in designing the logic circuit. several implementations of the method are reviewed to demonstrate how a biologically advantageous function can be chosen, how one simple network can generate major phenomena that are widely considered unrelated, and how one network design can lead to others that explain entirely different aspects of the brain. these results show that the method can benefit neuromorphic engineering as well as neuroscience, and that some brain functions can be carried out remarkably simply, at least in principle if not in the details.',\n",
       "  'categories': 'q-bio.nc eess.sp',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-05',\n",
       "  'updated': '2024-11-25',\n",
       "  'authors': ['lane yoder'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.03337'},\n",
       " {'title': 'the mutation process on the ancestral line under selection',\n",
       "  'id': '2306.03512',\n",
       "  'abstract': 'we consider the moran model of population genetics with two types, mutation, and selection, and investigate the line of descent of a randomly-sampled individual from a contemporary population. we trace this ancestral line back into the distant past, far beyond the most recent common ancestor of the population (thus connecting population genetics to phylogeny), and analyse the mutation process along this line.   to this end, we use the pruned lookdown ancestral selection graph, which consists of a set of potential ancestors of the sampled individual at any given time. relative to the neutral case (that is, without selection), we obtain a general bias towards the beneficial type, an increase in the beneficial mutation rate, and a decrease in the deleterious mutation rate. this sheds new light on previous analytical results. we discuss our findings in the light of a well-known observation at the interface of phylogeny and population genetics, namely, the difference in the mutation rates (or, more precisely, mutation fluxes) estimated via phylogenetic methods relative to those observed in pedigree studies.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '10.1016/j.tpb.2024.04.004',\n",
       "  'created': '2023-06-06',\n",
       "  'updated': '2024-05-21',\n",
       "  'authors': ['ellen baake', 'enrico di gaspero', 'fernando cordero'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.03512'},\n",
       " {'title': 'bernoulli factories and duality in wright-fisher and allen-cahn models   of population genetics',\n",
       "  'id': '2306.03539',\n",
       "  'abstract': 'mathematical models of genetic evolution often come in pairs, connected by a so-called duality relation. the most seminal example are the wright-fisher diffusion and the kingman coalescent, where the former describes the stochastic evolution of neutral allele frequencies in a large population forwards in time, and the latter describes the genetic ancestry of randomly sampled individuals from the population backwards in time. as well as providing a richer description than either model in isolation, duality often yields equations satisfied by quantities of interest. we employ the so-called bernoulli factory - a celebrated tool in simulation-based computing - to derive duality relations for broad classes of genetics models. as concrete examples, we present wright-fisher diffusions with general drift functions, and allen-cahn equations with general, nonlinear forcing terms. the drift and forcing functions can be interpreted as the action of frequency-dependent selection. to our knowledge, this work is the first time a connection has been drawn between bernoulli factories and duality in models of population genetics.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-06',\n",
       "  'updated': '2024-02-01',\n",
       "  'authors': ['jere koskela', 'krzysztof łatuszyński', 'dario spanò'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.03539'},\n",
       " {'title': 'active learning of the thermodynamics-dynamics tradeoff in protein   condensates',\n",
       "  'id': '2306.03696',\n",
       "  'abstract': 'phase-separated biomolecular condensates exhibit a wide range of dynamical properties, which depend on the sequences of the constituent proteins and rnas. however, it is unclear to what extent condensate dynamics can be tuned without also changing the thermodynamic properties that govern phase separation. using coarse-grained simulations of intrinsically disordered proteins, we show that the dynamics and thermodynamics of homopolymer condensates are strongly correlated, with increased condensate stability being coincident with low mobilities and high viscosities. we then apply an \"active learning\" strategy to identify heteropolymer sequences that break this correlation. this data-driven approach and accompanying analysis reveal how heterogeneous amino-acid compositions and non-uniform sequence patterning map to a range of independently tunable dynamical and thermodynamic properties of biomolecular condensates. our results highlight key molecular determinants governing the physical properties of biomolecular condensates and establish design rules for the development of stimuli-responsive biomaterials.',\n",
       "  'categories': 'q-bio.bm cond-mat.soft physics.bio-ph physics.comp-ph',\n",
       "  'doi': '10.1126/sciadv.adj2448',\n",
       "  'created': '2023-06-06',\n",
       "  'updated': '2023-12-09',\n",
       "  'authors': ['yaxin an', 'michael a. webb', 'william m. jacobs'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.03696'},\n",
       " {'title': 'a mathematical model for co-infection dynamics of pneumocystis pneumonia   and hiv/aids with treatment',\n",
       "  'id': '2306.04407',\n",
       "  'abstract': 'the control of opportunistic infections among hiv infected individuals should be one of the major public health concerns in reducing mortality rate of individuals living with hiv/aids. in this study a deterministic co-infection mathematical model is employed to provide a quantification of treatment at each contagious stage against pneumocystis pneumonia (pcp) among hiv infected individuals on art. the disease-free equilibrium for the hiv/aids sub model, pcp sub model and the co-infection model are shown to be locally asymptotically stable when their associated disease threshold parameter is less than a unity. by use of suitable lyapunov functions, the endemic equilibrium corresponding to hiv/aids and pcp sub models are globally asymptotically stable whenever $\\\\mathcal{r}_{0h}>1$ and $\\\\mathcal{r}_{0p}>1$ respectively. the sensitivity analysis results implicate that the effective contact rates are the main mechanisms fueling the proliferation of the two diseases and on the other hand treatment efforts play an important role in reducing the incidence. numerical simulations show that treatment of pcp at all contagious stages reduces its burden on hiv/aids patients and dual treatment of the co-infected individuals significantly reduces the burden of the co-infection.',\n",
       "  'categories': 'math.ds physics.soc-ph q-bio.pe',\n",
       "  'doi': '10.11648/j.sjams.20241204.11',\n",
       "  'created': '2023-06-07',\n",
       "  'updated': '2024-08-01',\n",
       "  'authors': ['michael byamukama', 'damian kajunguri', 'martin karuhanga'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.04407'},\n",
       " {'title': 'statistical thermodynamics of self-organization of the binding energy   super-landscape in the adaptive immune system',\n",
       "  'id': '2306.04665',\n",
       "  'abstract': 'the humoral adaptive immune system, from the physical perspective, can be regarded as a self-organized antibody binding energy landscape. in biological terms it means the system organizes its repertoire of antigen binding molecules so as to maintain its integrity. in this article i reason that the super-landscape created by the fusion of binding energy landscapes can be described by the distribution of interaction energies and a deformation parameter of thermodynamic potentials in the system. this deformation parameter characterizes the adaptive network of interactions in the system and the asymmetry of generalized logistic distributions in immunoassays. overall, statistical thermodynamics approaches could provide a deeper theoretical insight into the dynamical self-organization of the adaptive immune system and into the interpretation of experimental results.',\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-07',\n",
       "  'updated': '2024-07-30',\n",
       "  'authors': ['jozsef prechl'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.04665'},\n",
       " {'title': \"don't trust your eyes: on the (un)reliability of feature visualizations\",\n",
       "  'id': '2306.04719',\n",
       "  'abstract': 'how do neural networks extract patterns from pixels? feature visualizations attempt to answer this important question by visualizing highly activating patterns through optimization. today, visualization methods form the foundation of our knowledge about the internal workings of neural networks, as a type of mechanistic interpretability. here we ask: how reliable are feature visualizations? we start our investigation by developing network circuits that trick feature visualizations into showing arbitrary patterns that are completely disconnected from normal network behavior on natural input. we then provide evidence for a similar phenomenon occurring in standard, unmanipulated networks: feature visualizations are processed very differently from standard input, casting doubt on their ability to \"explain\" how neural networks process natural images. this can be used as a sanity check for feature visualizations. we underpin our empirical findings by theory proving that the set of functions that can be reliably understood by feature visualization is extremely small and does not include general black-box neural networks. therefore, a promising way forward could be the development of networks that enforce certain structures in order to ensure more reliable feature visualizations.',\n",
       "  'categories': 'cs.cv cs.ai cs.hc cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-07',\n",
       "  'updated': '2024-06-06',\n",
       "  'authors': ['robert geirhos',\n",
       "   'roland s. zimmermann',\n",
       "   'blair bilodeau',\n",
       "   'wieland brendel',\n",
       "   'been kim'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.04719'},\n",
       " {'title': 'toward more accurate and generalizable brain deformation estimators for   traumatic brain injury detection with unsupervised domain adaptation',\n",
       "  'id': '2306.05255',\n",
       "  'abstract': 'machine learning head models (mlhms) are developed to estimate brain deformation for early detection of traumatic brain injury (tbi). however, the overfitting to simulated impacts and the lack of generalizability caused by distributional shift of different head impact datasets hinders the broad clinical applications of current mlhms. we propose brain deformation estimators that integrates unsupervised domain adaptation with a deep neural network to predict whole-brain maximum principal strain (mps) and mps rate (mpsr). with 12,780 simulated head impacts, we performed unsupervised domain adaptation on on-field head impacts from 302 college football (cf) impacts and 457 mixed martial arts (mma) impacts using domain regularized component analysis (drca) and cycle-gan-based methods. the new model improved the mps/mpsr estimation accuracy, with the drca method significantly outperforming other domain adaptation methods in prediction accuracy (p<0.001): mps rmse: 0.027 (cf) and 0.037 (mma); mpsr rmse: 7.159 (cf) and 13.022 (mma). on another two hold-out test sets with 195 college football impacts and 260 boxing impacts, the drca model significantly outperformed the baseline model without domain adaptation in mps and mpsr estimation accuracy (p<0.001). the drca domain adaptation reduces the mps/mpsr estimation error to be well below tbi thresholds, enabling accurate brain deformation estimation to detect tbi in future clinical applications.',\n",
       "  'categories': 'cs.lg eess.sp physics.bio-ph q-bio.qm stat.ap',\n",
       "  'doi': '10.1109/jsen.2023.3349213',\n",
       "  'created': '2023-06-08',\n",
       "  'updated': '',\n",
       "  'authors': ['xianghao zhan',\n",
       "   'jiawei sun',\n",
       "   'yuzhe liu',\n",
       "   'nicholas j. cecchi',\n",
       "   'enora le flao',\n",
       "   'olivier gevaert',\n",
       "   'michael m. zeineh',\n",
       "   'david b. camarillo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.05255'},\n",
       " {'title': 'mc-nn: an end-to-end multi-channel neural network approach for   predicting influenza a virus hosts and antigenic types',\n",
       "  'id': '2306.05587',\n",
       "  'abstract': 'influenza poses a significant threat to public health, particularly among the elderly, young children, and people with underlying dis-eases. the manifestation of severe conditions, such as pneumonia, highlights the importance of preventing the spread of influenza. an accurate and cost-effective prediction of the host and antigenic sub-types of influenza a viruses is essential to addressing this issue, particularly in resource-constrained regions. in this study, we propose a multi-channel neural network model to predict the host and antigenic subtypes of influenza a viruses from hemagglutinin and neuraminidase protein sequences. our model was trained on a comprehensive data set of complete protein sequences and evaluated on various test data sets of complete and incomplete sequences. the results demonstrate the potential and practicality of using multi-channel neural networks in predicting the host and antigenic subtypes of influenza a viruses from both full and partial protein sequences.',\n",
       "  'categories': 'cs.lg q-bio.qm',\n",
       "  'doi': '10.1007/s42979-023-01839-5',\n",
       "  'created': '2023-06-08',\n",
       "  'updated': '2024-02-21',\n",
       "  'authors': ['yanhua xu', 'dominik wojtczak'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.05587'},\n",
       " {'title': 'rapid, antibiotic incubation-free determination of tuberculosis drug   resistance using machine learning and raman spectroscopy',\n",
       "  'id': '2306.05653',\n",
       "  'abstract': \"tuberculosis (tb) is the world's deadliest infectious disease, with over 1.5 million deaths annually and 10 million new cases reported each year. the causative organism, mycobacterium tuberculosis (mtb) can take nearly 40 days to culture, a required step to determine the pathogen's antibiotic susceptibility. both rapid identification of mtb and rapid antibiotic susceptibility testing (ast) are essential for effective patient treatment and combating antimicrobial resistance. here, we demonstrate a rapid, culture-free, and antibiotic incubation-free drug susceptibility test for tb using raman spectroscopy and machine learning. we collect few-to-single-cell raman spectra from over 25,000 cells of the mtb complex strain bacillus calmette guerin (bcg) resistant to one of the four mainstay anti-tb drugs, isoniazid, rifampicin, moxifloxacin and amikacin, as well as a pan susceptible wildtype strain. by training a neural network on this data, we classify the antibiotic resistance profile of each strain, both on dried samples and in patient sputum samples. on dried samples, we achieve >98% resistant versus susceptible classification accuracy across all 5 bcg strains. in patient sputum samples, we achieve ~79% average classification accuracy. we develop a feature recognition algorithm in order to verify that our machine learning model is using biologically relevant spectral features to assess the resistance profiles of our mycobacterial strains. finally, we demonstrate how this approach can be deployed in resource-limited settings by developing a low-cost, portable raman microscope that costs <$5000. we show how this instrument and our machine learning model enables combined microscopy and spectroscopy for accurate few-to-single-cell drug susceptibility testing of bcg.\",\n",
       "  'categories': 'physics.bio-ph physics.optics q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-08',\n",
       "  'updated': '2024-04-09',\n",
       "  'authors': ['babatunde ogunlade',\n",
       "   'loza f. tadesse',\n",
       "   'hongquan li',\n",
       "   'nhat vu',\n",
       "   'niaz banaei',\n",
       "   'amy k. barczak',\n",
       "   'amr. a. e. saleh',\n",
       "   'manu prakash',\n",
       "   'jennifer a. dionne'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.05653'},\n",
       " {'title': 'leaping through tree space: continuous phylogenetic inference for rooted   and unrooted trees',\n",
       "  'id': '2306.05739',\n",
       "  'abstract': 'phylogenetics is now fundamental in life sciences, providing insights into the earliest branches of life and the origins and spread of epidemics. however, finding suitable phylogenies from the vast space of possible trees remains challenging. to address this problem, for the first time, we perform both tree exploration and inference in a continuous space where the computation of gradients is possible. this continuous relaxation allows for major leaps across tree space in both rooted and unrooted trees, and is less susceptible to convergence to local minima. our approach outperforms the current best methods for inference on unrooted trees and, in simulation, accurately infers the tree and root in ultrametric cases. the approach is effective in cases of empirical data with negligible amounts of data, which we demonstrate on the phylogeny of jawed vertebrates. indeed, only a few genes with an ultrametric signal were generally sufficient for resolving the major lineages of vertebrates. optimisation is possible via automatic differentiation and our method presents an effective way forwards for exploring the most difficult, data-deficient phylogenetic questions.',\n",
       "  'categories': 'q-bio.pe cs.lg',\n",
       "  'doi': '10.1093/gbe/evad213',\n",
       "  'created': '2023-06-09',\n",
       "  'updated': '2024-01-23',\n",
       "  'authors': ['matthew j penn',\n",
       "   'neil scheidwasser',\n",
       "   'joseph penn',\n",
       "   'christl a donnelly',\n",
       "   'david a duchêne',\n",
       "   'samir bhatt'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.05739'},\n",
       " {'title': 'extraction and recovery of spatio-temporal structure in latent dynamics   alignment with diffusion models',\n",
       "  'id': '2306.06138',\n",
       "  'abstract': \"in the field of behavior-related brain computation, it is necessary to align raw neural signals against the drastic domain shift among them. a foundational framework within neuroscience research posits that trial-based neural population activities rely on low-dimensional latent dynamics, thus focusing on the latter greatly facilitates the alignment procedure. despite this field's progress, existing methods ignore the intrinsic spatio-temporal structure during the alignment phase. hence, their solutions usually lead to poor quality in latent dynamics structures and overall performance. to tackle this problem, we propose an alignment method erdiff, which leverages the expressivity of the diffusion model to preserve the spatio-temporal structure of latent dynamics. specifically, the latent dynamics structures of the source domain are first extracted by a diffusion model. then, under the guidance of this diffusion model, such structures are well-recovered through a maximum likelihood alignment procedure in the target domain. we first demonstrate the effectiveness of our proposed method on a synthetic dataset. then, when applied to neural recordings from the non-human primate motor cortex, under both cross-day and inter-subject settings, our method consistently manifests its capability of preserving the spatiotemporal structure of latent dynamics and outperforms existing approaches in alignment goodness-of-fit and neural decoding performance.\",\n",
       "  'categories': 'q-bio.nc cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-09',\n",
       "  'updated': '2024-03-08',\n",
       "  'authors': ['yule wang', 'zijing wu', 'chengrui li', 'anqi wu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.06138'},\n",
       " {'title': 'poet: a generative model of protein families as sequences-of-sequences',\n",
       "  'id': '2306.06156',\n",
       "  'abstract': \"generative protein language models are a natural way to design new proteins with desired functions. however, current models are either difficult to direct to produce a protein from a specific family of interest, or must be trained on a large multiple sequence alignment (msa) from the specific family of interest, making them unable to benefit from transfer learning across families. to address this, we propose $\\\\textbf{p}$r$\\\\textbf{o}$tein $\\\\textbf{e}$volutionary $\\\\textbf{t}$ransformer (poet), an autoregressive generative model of whole protein families that learns to generate sets of related proteins as sequences-of-sequences across tens of millions of natural protein sequence clusters. poet can be used as a retrieval-augmented language model to generate and score arbitrary modifications conditioned on any protein family of interest, and can extrapolate from short context lengths to generalize well even for small families. this is enabled by a unique transformer layer; we model tokens sequentially within sequences while attending between sequences order invariantly, allowing poet to scale to context lengths beyond those used during training. in extensive experiments on deep mutational scanning datasets, we show that poet outperforms existing protein language models and evolutionary sequence models for variant function prediction across proteins of all msa depths. we also demonstrate poet's ability to controllably generate new protein sequences.\",\n",
       "  'categories': 'q-bio.qm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-09',\n",
       "  'updated': '2023-11-01',\n",
       "  'authors': ['timothy f. truong', 'tristan bepler'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.06156'},\n",
       " {'title': 'neurograph: benchmarks for graph machine learning in brain connectomics',\n",
       "  'id': '2306.06202',\n",
       "  'abstract': 'machine learning provides a valuable tool for analyzing high-dimensional functional neuroimaging data, and is proving effective in predicting various neurological conditions, psychiatric disorders, and cognitive patterns. in functional magnetic resonance imaging (mri) research, interactions between brain regions are commonly modeled using graph-based representations. the potency of graph machine learning methods has been established across myriad domains, marking a transformative step in data interpretation and predictive modeling. yet, despite their promise, the transposition of these techniques to the neuroimaging domain has been challenging due to the expansive number of potential preprocessing pipelines and the large parameter search space for graph-based dataset construction. in this paper, we introduce neurograph, a collection of graph-based neuroimaging datasets, and demonstrated its utility for predicting multiple categories of behavioral and cognitive traits. we delve deeply into the dataset generation search space by crafting 35 datasets that encompass static and dynamic brain connectivity, running in excess of 15 baseline methods for benchmarking. additionally, we provide generic frameworks for learning on both static and dynamic graphs. our extensive experiments lead to several key observations. notably, using correlation vectors as node features, incorporating larger number of regions of interest, and employing sparser graphs lead to improved performance. to foster further advancements in graph-based data driven neuroimaging analysis, we offer a comprehensive open-source python package that includes the benchmark datasets, baseline implementations, model training, and standard evaluation.',\n",
       "  'categories': 'cs.lg cs.ai q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-09',\n",
       "  'updated': '2024-11-22',\n",
       "  'authors': ['anwar said',\n",
       "   'roza g. bayrak',\n",
       "   'tyler derr',\n",
       "   'mudassir shabbir',\n",
       "   'daniel moyer',\n",
       "   'catie chang',\n",
       "   'xenofon koutsoukos'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.06202'},\n",
       " {'title': 'movement bias in asymmetric landscapes and its impact on population   distribution and critical habitat size',\n",
       "  'id': '2306.06450',\n",
       "  'abstract': \"ecologists have long investigated how demographic and movement parameters determine the spatial distribution and critical habitat size of a population. however, most models oversimplify movement behavior, neglecting how landscape heterogeneity influences individual movement. we relax this assumption and introduce a reaction-advection-diffusion equation that describes population dynamics when individuals exhibit space-dependent movement bias toward preferred regions. our model incorporates two types of these preferred regions: a high-quality habitat patch, termed `habitat', which is included to model avoidance of degraded habitats like deforested regions; and a preferred location, such as a chemoattractant source or a watering hole, that we allow to be asymmetrically located with respect to habitat edges. in this scenario, the critical habitat size depends on both the relative position of the preferred location and the movement bias intensities. when preferred locations are near habitat edges, the critical habitat size can decrease when diffusion increases, a phenomenon called the drift paradox. also, ecological traps arise when the habitat overcrowds due to excessive attractiveness or the preferred location is near a low-quality region. our results highlight the importance of species-specific movement behavior and habitat preference as drivers of population dynamics in fragmented landscapes and, therefore, in the design of protected areas.\",\n",
       "  'categories': 'q-bio.pe physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-10',\n",
       "  'updated': '2024-03-08',\n",
       "  'authors': ['vivian dornelas',\n",
       "   'pablo de castro',\n",
       "   'justin m. calabrese',\n",
       "   'william f. fagan',\n",
       "   'ricardo martinez-garcia'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.06450'},\n",
       " {'title': 'phylogenetic network classes through the lens of expanding covers',\n",
       "  'id': '2306.06519',\n",
       "  'abstract': \"it was recently shown that a large class of phylogenetic networks, the `labellable' networks, is in bijection with the set of `expanding' covers of finite sets. in this paper, we show how several prominent classes of phylogenetic networks can be characterised purely in terms of properties of their associated covers. these classes include the tree-based, tree-child, orchard, tree-sibling, and normal networks.\",\n",
       "  'categories': 'q-bio.pe math.co',\n",
       "  'doi': '10.1007/s00285-024-02075-y',\n",
       "  'created': '2023-06-10',\n",
       "  'updated': '2023-06-24',\n",
       "  'authors': ['andrew francis', 'daniele marchei', 'mike steel'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.06519'},\n",
       " {'title': '3d molecule generation by denoising voxel grids',\n",
       "  'id': '2306.07473',\n",
       "  'abstract': 'we propose a new score-based approach to generate 3d molecules represented as atomic densities on regular grids. first, we train a denoising neural network that learns to map from a smooth distribution of noisy molecules to the distribution of real molecules. then, we follow the neural empirical bayes framework (saremi and hyvarinen, 19) and generate molecules in two steps: (i) sample noisy density grids from a smooth distribution via underdamped langevin markov chain monte carlo, and (ii) recover the \"clean\" molecule by denoising the noisy grid with a single step. our method, voxmol, generates molecules in a fundamentally different way than the current state of the art (ie, diffusion models applied to atom point clouds). it differs in terms of the data representation, the noise model, the network architecture and the generative modeling algorithm. our experiments show that voxmol captures the distribution of drug-like molecules better than state of the art, while being faster to generate samples.',\n",
       "  'categories': 'cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-12',\n",
       "  'updated': '2024-03-08',\n",
       "  'authors': ['pedro o. pinheiro',\n",
       "   'joshua rackers',\n",
       "   'joseph kleinhenz',\n",
       "   'michael maser',\n",
       "   'omar mahmood',\n",
       "   'andrew martin watkins',\n",
       "   'stephen ra',\n",
       "   'vishnu sresht',\n",
       "   'saeed saremi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.07473'},\n",
       " {'title': 'hyperbolic graph diffusion model',\n",
       "  'id': '2306.07618',\n",
       "  'abstract': 'diffusion generative models (dms) have achieved promising results in image and graph generation. however, real-world graphs, such as social networks, molecular graphs, and traffic graphs, generally share non-euclidean topologies and hidden hierarchies. for example, the degree distributions of graphs are mostly power-law distributions. the current latent diffusion model embeds the hierarchical data in a euclidean space, which leads to distortions and interferes with modeling the distribution. instead, hyperbolic space has been found to be more suitable for capturing complex hierarchical structures due to its exponential growth property. in order to simultaneously utilize the data generation capabilities of diffusion models and the ability of hyperbolic embeddings to extract latent hierarchical distributions, we propose a novel graph generation method called, hyperbolic graph diffusion model (hgdm), which consists of an auto-encoder to encode nodes into successive hyperbolic embeddings, and a dm that operates in the hyperbolic latent space. hgdm captures the crucial graph structure distributions by constructing a hyperbolic potential node space that incorporates edge information. extensive experiments show that hgdm achieves better performance in generic graph and molecule generation benchmarks, with a $48\\\\%$ improvement in the quality of graph generation with highly hierarchical structures.',\n",
       "  'categories': 'cs.lg cs.ai q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-13',\n",
       "  'updated': '2024-01-03',\n",
       "  'authors': ['lingfeng wen',\n",
       "   'xuan tang',\n",
       "   'mingjie ouyang',\n",
       "   'xiangxiang shen',\n",
       "   'jian yang',\n",
       "   'daxin zhu',\n",
       "   'mingsong chen',\n",
       "   'xian wei'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.07618'},\n",
       " {'title': 'mol-instructions: a large-scale biomolecular instruction dataset for   large language models',\n",
       "  'id': '2306.08018',\n",
       "  'abstract': \"large language models (llms), with their remarkable task-handling capabilities and innovative outputs, have catalyzed significant advancements across a spectrum of fields. however, their proficiency within specialized domains such as biomolecular studies remains limited. to address this challenge, we introduce mol-instructions, a comprehensive instruction dataset designed for the biomolecular domain. mol-instructions encompasses three key components: molecule-oriented instructions, protein-oriented instructions, and biomolecular text instructions. each component aims to improve the understanding and prediction capabilities of llms concerning biomolecular features and behaviors. through extensive instruction tuning experiments on llms, we demonstrate the effectiveness of mol-instructions in enhancing large models' performance in the intricate realm of biomolecular studies, thus fostering progress in the biomolecular research community. mol-instructions is publicly available for ongoing research and will undergo regular updates to enhance its applicability.\",\n",
       "  'categories': 'q-bio.qm cs.ai cs.ce cs.cl cs.ir cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-13',\n",
       "  'updated': '2024-03-04',\n",
       "  'authors': ['yin fang',\n",
       "   'xiaozhuan liang',\n",
       "   'ningyu zhang',\n",
       "   'kangwei liu',\n",
       "   'rui huang',\n",
       "   'zhuo chen',\n",
       "   'xiaohui fan',\n",
       "   'huajun chen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.08018'},\n",
       " {'title': 'latent mutations in the ancestries of alleles under selection',\n",
       "  'id': '2306.08142',\n",
       "  'abstract': 'we consider a single genetic locus with two alleles $a_1$ and $a_2$ in a large haploid population. the locus is subject to selection and two-way, or recurrent, mutation. assuming the allele frequencies follow a wright-fisher diffusion and have reached stationarity, we describe the asymptotic behaviors of the conditional gene genealogy and the latent mutations of a sample with known allele counts, when the count $n_1$ of allele $a_1$ is fixed, and when either or both the sample size $n$ and the selection strength $\\\\lvert\\\\alpha\\\\rvert$ tend to infinity. our study extends previous work under neutrality to the case of non-neutral rare alleles, asserting that when selection is not too strong relative to the sample size, even if it is strongly positive or strongly negative in the usual sense ($\\\\alpha\\\\to -\\\\infty$ or $\\\\alpha\\\\to +\\\\infty$), the number of latent mutations of the $n_1$ copies of allele $a_1$ follows the same distribution as the number of alleles in the ewens sampling formula. on the other hand, very strong positive selection relative to the sample size leads to neutral gene genealogies with a single ancient latent mutation. we also demonstrate robustness of our asymptotic results against changing population sizes, when one of $\\\\lvert\\\\alpha\\\\rvert$ or $n$ is large.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-13',\n",
       "  'updated': '2024-04-26',\n",
       "  'authors': ['wai-tong louis fan', 'john wakeley'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.08142'},\n",
       " {'title': 'strong regulatory graphs',\n",
       "  'id': '2306.08261',\n",
       "  'abstract': 'logical modeling is a powerful tool in biology, offering a system-level understanding of the complex interactions that govern biological processes. a gap that hinders the scalability of logical models is the need to specify the update function of every vertex in the network depending on the status of its predecessors. to address this, we introduce in this paper the concept of strong regulation, where a vertex is only updated to active/inactive if all its predecessors agree in their influences; otherwise, it is set to ambiguous. we explore the interplay between active, inactive, and ambiguous influences in a network. we discuss the existence of phenotype attractors in such networks, where the status of some of the variables is fixed to active/inactive, while the others can have an arbitrary status, including ambiguous.',\n",
       "  'categories': 'cs.dm q-bio.mn q-bio.qm',\n",
       "  'doi': '10.3233/fi-242184',\n",
       "  'created': '2023-06-14',\n",
       "  'updated': '2024-07-06',\n",
       "  'authors': ['patric gustafsson', 'ion petre'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.08261'},\n",
       " {'title': 'mixalime: mixture models for allelic imbalance estimation in   high-throughput sequencing data',\n",
       "  'id': '2306.08287',\n",
       "  'abstract': 'modern high-throughput sequencing assays efficiently capture not only gene expression and different levels of gene regulation but also a multitude of genome variants. focused analysis of alternative alleles of variable sites at homologous chromosomes of the human genome reveals allele-specific gene expression and allele-specific gene regulation by assessing allelic imbalance of read counts at individual sites. here we formally describe an advanced statistical framework for detecting the allelic imbalance in allelic read counts at single-nucleotide variants detected in diverse omics studies (chip-seq, atac-seq, dnase-seq, cage-seq, and others). mixalime accounts for copy-number variants and aneuploidy, reference read mapping bias, and provides several scoring models to balance between sensitivity and specificity when scoring data with varying levels of experimental noise-caused overdispersion.',\n",
       "  'categories': 'stat.ap q-bio.gn q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-14',\n",
       "  'updated': '2024-04-01',\n",
       "  'authors': ['georgy meshcheryakov',\n",
       "   'sergey abramov',\n",
       "   'aleksandr boytsov',\n",
       "   'andrey i. buyan',\n",
       "   'vsevolod j. makeev',\n",
       "   'ivan v. kulakovskiy'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.08287'},\n",
       " {'title': 'tropical logistic regression model on space of phylogenetic trees',\n",
       "  'id': '2306.08796',\n",
       "  'abstract': 'classification of gene trees is an important task both in the analysis of multi-locus phylogenetic data, and assessment of the convergence of markov chain monte carlo (mcmc) analyses used in bayesian phylogenetic tree reconstruction. the logistic regression model is one of the most popular classification models in statistical learning, thanks to its computational speed and interpretability. however, it is not appropriate to directly apply the standard logistic regression model to a set of phylogenetic trees, as the space of phylogenetic trees is non-euclidean and thus contradicts the standard assumptions on covariates. it is well-known in tropical geometry and phylogenetics that the space of phylogenetic trees is a tropical linear space in terms of the max-plus algebra. therefore, in this paper, we propose an analogue approach of the logistic regression model in the setting of tropical geometry. our proposed method outperforms classical logistic regression in terms of area under the roc curve (auc) in numerical examples, including with data generated by the multi-species coalescent model. theoretical properties such as statistical consistency have been proved and generalization error rates have been derived. finally, our classification algorithm is proposed as an mcmc convergence criterion for mr bayes. unlike the convergence metric used by mrbayes which is only dependent on tree topologies, our method is sensitive to branch lengths and therefore provides a more robust metric for convergence. in a test case, it is illustrated that the tropical logistic regression can differentiate between two independently run mcmc chains, even when the standard metric cannot.',\n",
       "  'categories': 'math.co q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-14',\n",
       "  'updated': '2024-06-07',\n",
       "  'authors': ['georgios aliatimis',\n",
       "   'ruriko yoshida',\n",
       "   'burak boyaci',\n",
       "   'james a. grant'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.08796'},\n",
       " {'title': 'orthogonal extended infomax algorithm',\n",
       "  'id': '2306.09180',\n",
       "  'abstract': 'the extended infomax algorithm for independent component analysis (ica) can separate sub- and super-gaussian signals but converges slowly as it uses stochastic gradient optimization. in this paper, an improved extended infomax algorithm is presented that converges much faster. accelerated convergence is achieved by replacing the natural gradient learning rule of extended infomax by a fully-multiplicative orthogonal-group based update scheme of the unmixing matrix leading to an orthogonal extended infomax algorithm (ogextinf). computational performance of ogextinf is compared with two fast ica algorithms: the popular fastica and picard, a l-bfgs algorithm belonging to the family of quasi-newton methods. our results demonstrate superior performance of the proposed method on small-size eeg data sets as used for example in online eeg processing systems, such as brain-computer interfaces or clinical systems for spike and seizure detection.',\n",
       "  'categories': 'stat.me eess.sp q-bio.qm',\n",
       "  'doi': '10.1088/1741-2552/ad38db',\n",
       "  'created': '2023-06-15',\n",
       "  'updated': '',\n",
       "  'authors': ['nicole ille'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.09180'},\n",
       " {'title': 'survival of the flattest in the quasispecies model',\n",
       "  'id': '2306.09221',\n",
       "  'abstract': 'viruses present an amazing genetic variability. an ensemble of infecting viruses, also called a viral quasispecies, is a cloud of mutants centered around a specific genotype. the simplest model of evolution, whose equilibrium state is described by the quasispecies equation, is the moran--kingman model. for the sharp peak landscape, we perform several exact computations and we derive several exact formulas. we obtain also an exact formula for the quasispecies distribution, involving a series and the mean fitness. a very simple formula for the mean hamming distance is derived, which is exact and which do not require a specific asymptotic expansion (like sending the length of the macromolecules to $\\\\infty$ or the mutation probability to $0$). we try also to extend these formulas to a general fitness landscape. we obtain an equation involving the covariance of the fitness and the hamming class number in the quasispecies distribution. with the help of these formulas, we discuss the phenomenon of the error threshold and the notion of quasispecies. we recover the limiting quasipecies distribution in the long chain regime. we go beyond the sharp peak landscape and we consider fitness landscapes having finitely many peaks and a plateau--type landscape. we finally prove rigorously within this framework the possible occurrence of the survival of the flattest, a phenomenon which has been previously discovered by wilke, wang, ofria, lenski and adami and which has been investigated in several works.',\n",
       "  'categories': 'q-bio.pe math.pr',\n",
       "  'doi': '10.1017/apr.2024.39',\n",
       "  'created': '2023-06-15',\n",
       "  'updated': '2024-08-20',\n",
       "  'authors': ['maxime berger', 'raphaël cerf'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.09221'},\n",
       " {'title': 'on selecting distance metrics in $n$-dimensional normed vector spaces of   cells: a novel criterion and similarity measure towards efficient and   accurate omics analysis',\n",
       "  'id': '2306.09243',\n",
       "  'abstract': \"single-cell omics enable the profiles of cells, which contain large numbers of biological features, to be quantified. cluster analysis, a dimensionality reduction process, is used to reduce the dimensions of the data to make it computationally tractable. in these analyses, cells are represented as vectors in $n$-dimensional space, where each dimension corresponds to a certain cell feature. the distance between cells is used as a surrogate measure of similarity, providing insight into the cell's state, function, and genetic mechanisms. however, as cell profiles are clustered in 3d or higher-dimensional space, it remains unknown which distance metric provides the most accurate spatiotemporal representation of similarity, limiting the interpretability of the data. i propose and prove a generalized proposition and set of corollaries that serve as a criterion to determine which of the standard distance measures is most accurate for conveying cell profile heterogeneity. each distance method is evaluated via statistical, geometric, and topological proofs, which are formalized into a set of criteria. in this paper, i present the putative, first-ever method to elect the most accurate and precise distance metrics with any profiling modality, which are determined to be the wasserstein distance and cosine similarity metrics, respectively, in general cases. i also identify special cases in which the criterion may select non-standard metrics. combining the metric properties selected by the criterion, i develop a novel, custom, optimal distance metric that demonstrates superior computational efficiency, peak annotation, motif identification, and footprinting for transcription factor binding sites when compared with leading methods.\",\n",
       "  'categories': 'q-bio.gn math.dg',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-12',\n",
       "  'updated': '2024-06-05',\n",
       "  'authors': ['okezue bell', 'arthur lee', 'elizabeth engle'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.09243'},\n",
       " {'title': 'multi-omics prediction from high-content cellular imaging with deep   learning',\n",
       "  'id': '2306.09391',\n",
       "  'abstract': 'high-content cellular imaging, transcriptomics, and proteomics data provide rich and complementary views on the molecular layers of biology that influence cellular states and function. however, the biological determinants through which changes in multi-omics measurements influence cellular morphology have not yet been systematically explored, and the degree to which cell imaging could potentially enable the prediction of multi-omics directly from cell imaging data is therefore currently unclear. here, we address the question of whether it is possible to predict bulk multi-omics measurements directly from cell images using image2omics - a deep learning approach that predicts multi-omics in a cell population directly from high-content images of cells stained with multiplexed fluorescent dyes. we perform an experimental evaluation in gene-edited macrophages derived from human induced pluripotent stem cells (hipsc) under multiple stimulation conditions and demonstrate that image2omics achieves significantly better performance in predicting transcriptomics and proteomics measurements directly from cell images than predictions based on the mean observed training set abundance. we observed significant predictability of abundances for 4927 (18.72%; 95% ci: 6.52%, 35.52%) and 3521 (13.38%; 95% ci: 4.10%, 32.21%) transcripts out of 26137 in m1 and m2-stimulated macrophages respectively and for 422 (8.46%; 95% ci: 0.58%, 25.83%) and 697 (13.98%; 95% ci: 2.41%, 32.83%) proteins out of 4986 in m1 and m2-stimulated macrophages respectively. our results show that some transcript and protein abundances are predictable from cell imaging and that cell imaging may potentially, in some settings and depending on the mechanisms of interest and desired performance threshold, even be a scalable and resource-efficient substitute for multi-omics measurements.',\n",
       "  'categories': 'q-bio.qm cs.cv cs.lg q-bio.gn',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-15',\n",
       "  'updated': '2024-05-21',\n",
       "  'authors': ['rahil mehrizi',\n",
       "   'arash mehrjou',\n",
       "   'maryana alegro',\n",
       "   'yi zhao',\n",
       "   'benedetta carbone',\n",
       "   'carl fishwick',\n",
       "   'johanna vappiani',\n",
       "   'jing bi',\n",
       "   'siobhan sanford',\n",
       "   'hakan keles',\n",
       "   'marcus bantscheff',\n",
       "   'cuong nguyen',\n",
       "   'patrick schwab'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.09391'},\n",
       " {'title': 'the core & periphery hypothesis: a conceptual basis for generality in   cell and developmental biology',\n",
       "  'id': '2306.09534',\n",
       "  'abstract': 'the discovery of general principles underlying the complexity and diversity of cellular and developmental systems is a central and long-standing aim of biology. whilst new technologies collect data at an ever-accelerating rate, there is growing concern that conceptual progress is not keeping pace. we contend that this is due to a paucity of appropriate conceptual frameworks to serve as a basis for general theories of mesoscale biological phenomena. in exploring this issue, we have developed a foundation for one such framework, termed the core and periphery (c&p) hypothesis, which reveals hidden generality across the diverse and complex behaviors exhibited by cells and tissues. here, we present the c&p concept, provide examples of its applicability across multiple scales, argue its consistency with evolution, and discuss key implications and open questions. we propose that the c&p hypothesis could unlock new avenues of conceptual progress in cell and developmental biology.',\n",
       "  'categories': 'q-bio.cb q-bio.pe q-bio.sc',\n",
       "  'doi': '10.1016/j.cels.2024.08.001',\n",
       "  'created': '2023-06-15',\n",
       "  'updated': '',\n",
       "  'authors': ['elisa gallo',\n",
       "   'stefano de renzis',\n",
       "   'james sharpe',\n",
       "   'roberto mayor',\n",
       "   'jonas hartmann'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.09534'},\n",
       " {'title': 'opportunities and challenges for chatgpt and large language models in   biomedicine and health',\n",
       "  'id': '2306.10070',\n",
       "  'abstract': 'chatgpt has drawn considerable attention from both the general public and domain experts with its remarkable text generation capabilities. this has subsequently led to the emergence of diverse applications in the field of biomedicine and health. in this work, we examine the diverse applications of large language models (llms), such as chatgpt, in biomedicine and health. specifically we explore the areas of biomedical information retrieval, question answering, medical text summarization, information extraction, and medical education, and investigate whether llms possess the transformative power to revolutionize these tasks or whether the distinct complexities of biomedical domain presents unique challenges. following an extensive literature survey, we find that significant advances have been made in the field of text generation tasks, surpassing the previous state-of-the-art methods. for other applications, the advances have been modest. overall, llms have not yet revolutionized biomedicine, but recent rapid progress indicates that such methods hold great potential to provide valuable means for accelerating discovery and improving health. we also find that the use of llms, like chatgpt, in the fields of biomedicine and health entails various risks and challenges, including fabricated information in its generated responses, as well as legal and privacy concerns associated with sensitive patient data. we believe this survey can provide a comprehensive and timely overview to biomedical researchers and healthcare practitioners on the opportunities and challenges associated with using chatgpt and other llms for transforming biomedicine and health.',\n",
       "  'categories': 'cs.cy cs.ai cs.cl q-bio.qm',\n",
       "  'doi': '10.1093/bib/bbad493',\n",
       "  'created': '2023-06-15',\n",
       "  'updated': '2023-10-16',\n",
       "  'authors': ['shubo tian',\n",
       "   'qiao jin',\n",
       "   'lana yeganova',\n",
       "   'po-ting lai',\n",
       "   'qingqing zhu',\n",
       "   'xiuying chen',\n",
       "   'yifan yang',\n",
       "   'qingyu chen',\n",
       "   'won kim',\n",
       "   'donald c. comeau',\n",
       "   'rezarta islamaj',\n",
       "   'aadit kapoor',\n",
       "   'xin gao',\n",
       "   'zhiyong lu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.10070'},\n",
       " {'title': 'chaotic turnover of rare and abundant species in a strongly interacting   model community',\n",
       "  'id': '2306.11031',\n",
       "  'abstract': \"the composition of ecological communities varies not only between different locations but also in time. understanding the fundamental processes that drive species towards rarity or abundance is crucial to assessing ecosystem resilience and adaptation to changing environmental conditions. in plankton communities in particular, large temporal fluctuations in species abundances have been associated with chaotic dynamics. on the other hand, microbial diversity is overwhelmingly sustained by a `rare biosphere' of species with very low abundances. we consider here the possibility that interactions within a species-rich community can relate both phenomena. we use a lotka-volterra model with weak immigration and strong, disordered, and mostly competitive interactions between hundreds of species to bridge single-species temporal fluctuations and abundance distribution patterns. we highlight a generic chaotic regime where a few species at a time achieve dominance, but are continuously overturned by the invasion of formerly rare species. we derive a focal-species model that captures the intermittent boom-and-bust dynamics that every species undergoes. although species cannot be treated as effectively uncorrelated in their abundances, the community's effect on a focal species can nonetheless be described by a time-correlated noise characterized by a few effective parameters that can be estimated from time series. the model predicts a non-unitary exponent of the power-law abundance decay, which varies weakly with ecological parameters, consistent with observation in marine protist communities. the chaotic turnover regime is thus poised to capture relevant ecological features of species-rich microbial communities.\",\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech',\n",
       "  'doi': '10.1073/pnas.2312822121',\n",
       "  'created': '2023-06-19',\n",
       "  'updated': '2024-11-21',\n",
       "  'authors': ['emil mallmin', 'arne traulsen', 'silvia de monte'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.11031'},\n",
       " {'title': 'eight challenges in developing theory of intelligence',\n",
       "  'id': '2306.11232',\n",
       "  'abstract': 'a good theory of mathematical beauty is more practical than any current observation, as new predictions of physical reality can be verified self-consistently. this belief applies to the current status of understanding deep neural networks including large language models and even the biological intelligence. toy models provide a metaphor of physical reality, allowing mathematically formulating that reality (i.e., the so-called theory), which can be updated as more conjectures are justified or refuted. one does not need to pack all details into a model, but rather, more abstract models are constructed, as complex systems like brains or deep networks have many sloppy dimensions but much less stiff dimensions that strongly impact macroscopic observables. this kind of bottom-up mechanistic modeling is still promising in the modern era of understanding the natural or artificial intelligence. here, we shed light on eight challenges in developing theory of intelligence following this theoretical paradigm. theses challenges are representation learning, generalization, adversarial robustness, continual learning, causal learning, internal model of the brain, next-token prediction, and finally the mechanics of subjective experience.',\n",
       "  'categories': 'q-bio.nc cond-mat.stat-mech cs.ai cs.cl',\n",
       "  'doi': '10.3389/fncom.2024.1388166',\n",
       "  'created': '2023-06-19',\n",
       "  'updated': '2024-06-21',\n",
       "  'authors': ['haiping huang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.11232'},\n",
       " {'title': 'error-induced extinction in a multi-type critical birth-death process',\n",
       "  'id': '2306.11609',\n",
       "  'abstract': 'extreme mutation rates in microbes and cancer cells can result in error-induced extinction (eex), where every descendant cell eventually acquires a lethal mutation. in this work, we investigate critical birth-death processes with $n$ distinct types as a birth-death model of eex in a growing population. each type-$i$ cell divides independently $(i)\\\\to(i)+(i)$ or mutates $(i)\\\\to(i+1)$ at the same rate. the total number of cells grows exponentially as a yule process until a cell of type-$n$ appears, which cell type can only die at rate one. this makes the whole process critical and hence after the exponentially growing phase eventually all cells die with probability one. we present large-time asymptotic results for the general $n$-type critical birth-death process. we find that the mass function of the number of cells of type-$k$ has algebraic and stationary tail $(\\\\text{size})^{-1-\\\\chi_k}$, with $\\\\chi_k=2^{1-k}$, for $k=2,\\\\dots,n$, in sharp contrast to the exponential tail of the first type. the same exponents describe the tail of the asymptotic survival probability $(\\\\text{time})^{-\\\\chi_n}$. we present applications of the results for studying extinction due to intolerable mutation rates in biological populations.',\n",
       "  'categories': 'q-bio.pe math.pr',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-20',\n",
       "  'updated': '2024-07-05',\n",
       "  'authors': ['meritxell brunet guasch', 'p. l. krapivsky', 'tibor antal'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.11609'},\n",
       " {'title': 'grenedalf: population genetic statistics for the next generation of pool   sequencing',\n",
       "  'id': '2306.11622',\n",
       "  'abstract': \"pool sequencing is an efficient method for capturing genome-wide allele frequencies from multiple individuals, with broad applications such as studying adaptation in evolve-and-resequence experiments, monitoring of genetic diversity in wild populations, and genotype-to-phenotype mapping. here, we present grenedalf, a command line tool written in c++ that implements common population genetic statistics such as $\\\\theta$, tajima's d, and fst for pool sequencing. it is orders of magnitude faster than current tools, and is focused on providing usability and scalability, while also offering a plethora of input file formats and convenience options.\",\n",
       "  'categories': 'q-bio.pe q-bio.gn',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-20',\n",
       "  'updated': '2024-06-07',\n",
       "  'authors': ['lucas czech', 'jeffrey p. spence', 'moisés expósito-alonso'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.11622'},\n",
       " {'title': 'multi-fidelity active learning with gflownets',\n",
       "  'id': '2306.11715',\n",
       "  'abstract': 'in the last decades, the capacity to generate large amounts of data in science and engineering applications has been growing steadily. meanwhile, machine learning has progressed to become a suitable tool to process and utilise the available data. nonetheless, many relevant scientific and engineering problems present challenges where current machine learning methods cannot yet efficiently leverage the available data and resources. for example, in scientific discovery, we are often faced with the problem of exploring very large, structured and high-dimensional spaces. moreover, the high fidelity, black-box objective function is often very expensive to evaluate. progress in machine learning methods that can efficiently tackle such challenges would help accelerate currently crucial areas such as drug and materials discovery. in this paper, we propose a multi-fidelity active learning algorithm with gflownets as a sampler, to efficiently discover diverse, high-scoring candidates where multiple approximations of the black-box function are available at lower fidelity and cost. our evaluation on molecular discovery tasks shows that multi-fidelity active learning with gflownets can discover high-scoring candidates at a fraction of the budget of its single-fidelity counterpart while maintaining diversity, unlike rl-based alternatives. these results open new avenues for multi-fidelity active learning to accelerate scientific discovery and engineering design.',\n",
       "  'categories': 'cs.lg q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-20',\n",
       "  'updated': '2024-09-01',\n",
       "  'authors': ['alex hernandez-garcia',\n",
       "   'nikita saxena',\n",
       "   'moksh jain',\n",
       "   'cheng-hao liu',\n",
       "   'yoshua bengio'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.11715'},\n",
       " {'title': 'geometric deep learning for structure-based drug design: a survey',\n",
       "  'id': '2306.11768',\n",
       "  'abstract': 'structure-based drug design (sbdd) leverages the three-dimensional geometry of proteins to identify potential drug candidates. traditional approaches, rooted in physicochemical modeling and domain expertise, are often resource-intensive. recent advancements in geometric deep learning, which effectively integrate and process 3d geometric data, alongside breakthroughs in accurate protein structure predictions from tools like alphafold, have significantly propelled the field forward. this paper systematically reviews the state-of-the-art in geometric deep learning for sbdd. we begin by outlining foundational tasks in sbdd, discussing prevalent 3d protein representations, and highlighting representative predictive and generative models. next, we provide an in-depth review of key tasks, including binding site prediction, binding pose generation, de novo molecule generation, linker design, protein pocket generation, and binding affinity prediction. for each task, we present formal problem definitions, key methods, datasets, evaluation metrics, and performance benchmarks. lastly, we explore current challenges and future opportunities in sbdd. challenges include oversimplified problem formulations, limited out-of-distribution generalization, biosecurity concerns related to the misuse of structural data, insufficient evaluation metrics and large-scale benchmarks, and the need for experimental validation and enhanced model interpretability. opportunities lie in leveraging multimodal datasets, integrating domain knowledge, developing comprehensive benchmarks, establishing criteria aligned with clinical outcomes, and designing foundation models to expand the scope of design tasks. we also curate \\\\url{https://github.com/zaixizhang/awesome-sbdd}, reflecting ongoing contributions and new datasets in sbdd.',\n",
       "  'categories': 'q-bio.qm cs.ce cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-20',\n",
       "  'updated': '2024-11-15',\n",
       "  'authors': ['zaixi zhang',\n",
       "   'jiaxian yan',\n",
       "   'yining huang',\n",
       "   'qi liu',\n",
       "   'enhong chen',\n",
       "   'mengdi wang',\n",
       "   'marinka zitnik'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.11768'},\n",
       " {'title': \"symmetry's edge in cortical dynamics: multiscale dynamics of ensemble   excitation and inhibition\",\n",
       "  'id': '2306.11965',\n",
       "  'abstract': 'creating a quantitative theory for the cortex poses several challenges and raises numerous questions. for instance, what are the significant scales of the system? are they micro, meso or macroscopic? what are the relevant interactions? are they pairwise, higher order or mean-field? and what are the control parameters? are they noisy, dissipative or emergent?   to tackle these issues, we suggest using an approach akin to what has transformed our understanding of the state of matter. this includes identifying invariances in the ensemble dynamics of various neuron functional classes, searching for order parameters that connect important degrees of freedom and distinguish macroscopic system states, and identifying broken symmetries in the order parameter space to comprehend the emerging laws when many neurons interact and coordinate their activation.   by utilizing multielectrode and multiscale neural recordings, we measure the scale-invariant balance between excitatory and inhibitory neurons at a population level, referred to as ensemble e/i balance. this differs from the input e/i balance typically studied at the single-neuron level, focusing instead on the collective behavior of large neural populations. we investigate a set of parameters that can assist us in differentiating between various functional system states (such as the wake/sleep cycle) and pinpointing broken symmetries that serve different information processing and memory functions. furthermore, we identify broken symmetries that result in pathological states like seizures.   this study provides new insights into the multiscale dynamics of excitation and inhibition in cortical networks, advancing our understanding of the underlying principles governing neural computation and dysfunction.',\n",
       "  'categories': 'q-bio.nc cond-mat.dis-nn nlin.ao physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-20',\n",
       "  'updated': '2024-07-04',\n",
       "  'authors': ['nima dehghani'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.11965'},\n",
       " {'title': 'node-layer duality in networked systems',\n",
       "  'id': '2306.12136',\n",
       "  'abstract': 'real-world networks typically exhibit several aspects, or layers, of interactions among their nodes. by permuting the role of the nodes and the layers, we establish a new criterion to construct the dual of a network. this approach allows to examine information from either a node-centric or layer-centric viewpoint. through rigorous analytical methods and extensive simulations, we demonstrate that nodewise and layerwise connectivity measure different but related aspects of the same system. leveraging node-layer duality provides complementary insights, enabling a deeper comprehension of diverse networks across social science, technology and biology. taken together, these findings reveal previously unappreciated features of complex systems and provide a fresh tool for delving into their structure and dynamics.',\n",
       "  'categories': 'physics.soc-ph q-bio.nc',\n",
       "  'doi': '10.1038/s41467-024-50176-5',\n",
       "  'created': '2023-06-21',\n",
       "  'updated': '2024-05-11',\n",
       "  'authors': ['charley presigny',\n",
       "   'marie-constance corsi',\n",
       "   'fabrizio de vico fallani'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.12136'},\n",
       " {'title': 'protein discovery with discrete walk-jump sampling',\n",
       "  'id': '2306.12360',\n",
       "  'abstract': 'we resolve difficulties in training and sampling from a discrete generative model by learning a smoothed energy function, sampling from the smoothed data manifold with langevin markov chain monte carlo (mcmc), and projecting back to the true data manifold with one-step denoising. our discrete walk-jump sampling formalism combines the contrastive divergence training of an energy-based model and improved sample quality of a score-based model, while simplifying training and sampling by requiring only a single noise level. we evaluate the robustness of our approach on generative modeling of antibody proteins and introduce the distributional conformity score to benchmark protein generative models. by optimizing and sampling from our models for the proposed distributional conformity score, 97-100% of generated samples are successfully expressed and purified and 70% of functional designs show equal or improved binding affinity compared to known functional antibodies on the first attempt in a single round of laboratory experiments. we also report the first demonstration of long-run fast-mixing mcmc chains where diverse antibody protein classes are visited in a single mcmc chain.',\n",
       "  'categories': 'q-bio.bm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-08',\n",
       "  'updated': '2024-03-15',\n",
       "  'authors': ['nathan c. frey',\n",
       "   'daniel berenberg',\n",
       "   'karina zadorozhny',\n",
       "   'joseph kleinhenz',\n",
       "   'julien lafrance-vanasse',\n",
       "   'isidro hotzel',\n",
       "   'yan wu',\n",
       "   'stephen ra',\n",
       "   'richard bonneau',\n",
       "   'kyunghyun cho',\n",
       "   'andreas loukas',\n",
       "   'vladimir gligorijevic',\n",
       "   'saeed saremi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.12360'},\n",
       " {'title': 'precision psychiatry: predicting predictability',\n",
       "  'id': '2306.12462',\n",
       "  'abstract': 'precision psychiatry is an ermerging field that aims to provide individualized approaches to mental health care. multivariate analysis and machine learning are used to create outcome prediction models based on clinical data such as demographics, symptom assessments, genetic information, and brain imaging. while much emphasis has been placed on technical innovation, the complex and varied nature of mental health presents significant challenges to the successful implementation of these models. from this perspective, i review ten challenges in the field of precision psychiatry, including the need for studies on real-world populations and realistic clinical outcome definitions, consideration of treatment-related factors such as placebo effects and non-adherence to prescriptions. fairness, prospective validation in comparison to current practice and implementation studies of prediction models are other key issues that are currently understudied. a shift is proposed from retrospective studies based on linear and static concepts of disease towards prospective research that considers the importance of contextual factors and the dynamic and complex nature of mental health.',\n",
       "  'categories': 'q-bio.qm cs.lg',\n",
       "  'doi': '10.1017/s0033291724000370',\n",
       "  'created': '2023-06-21',\n",
       "  'updated': '',\n",
       "  'authors': ['edwin van dellen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.12462'},\n",
       " {'title': 'targeted collapse regularized autoencoder for anomaly detection: black   hole at the center',\n",
       "  'id': '2306.12627',\n",
       "  'abstract': 'autoencoders have been extensively used in the development of recent anomaly detection techniques. the premise of their application is based on the notion that after training the autoencoder on normal training data, anomalous inputs will exhibit a significant reconstruction error. consequently, this enables a clear differentiation between normal and anomalous samples. in practice, however, it is observed that autoencoders can generalize beyond the normal class and achieve a small reconstruction error on some of the anomalous samples. to improve the performance, various techniques propose additional components and more sophisticated training procedures. in this work, we propose a remarkably straightforward alternative: instead of adding neural network components, involved computations, and cumbersome training, we complement the reconstruction loss with a computationally light term that regulates the norm of representations in the latent space. the simplicity of our approach minimizes the requirement for hyperparameter tuning and customization for new applications which, paired with its permissive data modality constraint, enhances the potential for successful adoption across a broad range of applications. we test the method on various visual and tabular benchmarks and demonstrate that the technique matches and frequently outperforms more complex alternatives. we further demonstrate that implementing this idea in the context of state-of-the-art methods can further improve their performance. we also provide a theoretical analysis and numerical simulations that help demonstrate the underlying process that unfolds during training and how it helps with anomaly detection. this mitigates the black-box nature of autoencoder-based anomaly detection algorithms and offers an avenue for further investigation of advantages, fail cases, and potential new directions.',\n",
       "  'categories': 'cs.lg cs.ai cs.cv q-bio.nc stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-21',\n",
       "  'updated': '2024-03-27',\n",
       "  'authors': ['amin ghafourian',\n",
       "   'huanyi shui',\n",
       "   'devesh upadhyay',\n",
       "   'rajesh gupta',\n",
       "   'dimitar filev',\n",
       "   'iman soltani bozchalooi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.12627'},\n",
       " {'title': 'ultrametric identities in glassy models of natural evolution',\n",
       "  'id': '2306.13430',\n",
       "  'abstract': 'spin-glasses constitute a well-grounded framework for evolutionary models. of particular interest for (some of) these models is the lack of self-averaging of their order parameters (e.g. the hamming distance between the genomes of two individuals), even in asymptotic limits, much as like the behavior of the overlap between the configurations of two replica in mean-field spin-glasses. in the latter, this lack of self-averaging is related to peculiar fluctuations of the overlap, known as ghirlanda-guerra identities and aizenman-contucci polynomials, that cover a pivotal role in describing the ultrametric structure of the spin-glass landscape. as for evolutionary models, such identities may therefore be related to a taxonomic classification of individuals, yet a full investigation on their validity is missing. in this paper, we study ultrametric identities in simple cases where solely random mutations take place, while selective pressure is absent, namely in {\\\\em flat landscape} models. in particular, we study three paradigmatic models in this setting: the {\\\\em one parent model} (which, by construction, is ultrametric at the level of single individuals), the {\\\\em homogeneous population model} (which is replica symmetric), and the {\\\\em species formation model} (where a broken-replica scenario emerges at the level of species). we find analytical and numerical evidence that in the first and in the third model nor the ghirlanda-guerra neither the aizenman-contucci constraints hold, rather a new class of ultrametric identities is satisfied; in the second model all these constraints hold trivially. very preliminary results on a real biological human genome derived by {\\\\em the 1000 genome project consortium} and on two artificial human genomes (generated by two different types neural networks) seem in better agreement with these new identities rather than the classic ones.',\n",
       "  'categories': 'cond-mat.dis-nn q-bio.pe',\n",
       "  'doi': '10.1088/1751-8121/acf101',\n",
       "  'created': '2023-06-23',\n",
       "  'updated': '',\n",
       "  'authors': ['elena agliari',\n",
       "   'francesco alemanno',\n",
       "   'miriam aquaro',\n",
       "   'adriano barra'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.13430'},\n",
       " {'title': 'rethinking tipping points in spatial ecosystems',\n",
       "  'id': '2306.13571',\n",
       "  'abstract': 'the theory of alternative stable states and tipping points has garnered substantial attention in the last several decades. it predicts potential critical transitions from one ecosystem state to a completely different state under increasing environmental stress. however, typically, ecosystem models that predict tipping do not resolve space explicitly. ecosystems being inherently spatial, it is important to understand the effects of spatial processes. in fact, it has been argued that spatial dynamics can actually help ecosystems evade tipping. here, using a dryland and a savanna-forest model as example systems, we provide a synthesis of several mechanisms by which spatial processes can change our predictions of tipping in ecosystems. we show that self-organized turing patterns can emerge in drylands that help evade tipping, but that (non-turing) patterns driven by environmental heterogeneity are key to evasion of tipping in humid savannas. since the ecological interactions driving the dynamics of these ecosystems differ from each other, we suggest that tipping evasion mechanisms in ecosystems may be connected to the key ecological interactions in a system. this highlights the need for further research into the link between the two in order to formulate better strategies to make ecosystems resilient to global change.',\n",
       "  'categories': 'nlin.ps q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-23',\n",
       "  'updated': '2024-09-09',\n",
       "  'authors': ['swarnendu banerjee',\n",
       "   'mara baudena',\n",
       "   'paul carter',\n",
       "   'robbin bastiaansen',\n",
       "   'arjen doelman',\n",
       "   'max rietkerk'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.13571'},\n",
       " {'title': 'optimal vaccination policy to prevent endemicity: a stochastic model',\n",
       "  'id': '2306.13633',\n",
       "  'abstract': \"we examine here the effects of recurrent vaccination and waning immunity on the establishment of an endemic equilibrium in a population. an individual-based model that incorporates memory effects for transmission rate during infection and subsequent immunity is introduced, considering stochasticity at the individual level. by letting the population size going to infinity, we derive a set of equations describing the large scale behavior of the epidemic. the analysis of the model's equilibria reveals a criterion for the existence of an endemic equilibrium, which depends on the rate of immunity loss and the distribution of time between booster doses. the outcome of a vaccination policy in this context is influenced by the efficiency of the vaccine in blocking transmissions and the distribution pattern of booster doses within the population. strategies with evenly spaced booster shots at the individual level prove to be more effective in preventing disease spread compared to irregularly spaced boosters, as longer intervals without vaccination increase susceptibility and facilitate more efficient disease transmission. we provide an expression for the critical fraction of the population required to adhere to the vaccination policy in order to eradicate the disease, that resembles a well-known threshold for preventing an outbreak with an imperfect vaccine. we also investigate the consequences of unequal vaccine access in a population and prove that, under reasonable assumptions, fair vaccine allocation is the optimal strategy to prevent endemicity.\",\n",
       "  'categories': 'q-bio.pe math.pr',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-23',\n",
       "  'updated': '2024-11-21',\n",
       "  'authors': ['félix foutel-rodier', 'arthur charpentier', 'hélène guérin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.13633'},\n",
       " {'title': 'many-species ecological fluctuations as a jump process from the brink of   extinction',\n",
       "  'id': '2306.13634',\n",
       "  'abstract': 'highly-diverse ecosystems exhibit a broad distribution of population sizes and species turnover, where species at high and low abundances are exchanged over time. we show that these two features generically emerge in the fluctuating phase of many-variable model ecosystems with disordered species interactions, when species are supported by migration from outside the system at a small rate. we show that these and other phenomena can be understood through the existence of a scaling regime in the limit of small migration, in which large fluctuations and long timescales emerge. we construct an exact analytical theory for this asymptotic regime, that provides scaling predictions on timescales and abundance distributions that are verified exactly in simulations. in this regime, a clear separation emerges between rare and abundant species at any given time, despite species moving back and forth between the rare and abundant subsets. the number of abundant species is found to lie strictly below a well-known stability bound, maintaining the system away from marginality. at the same time, other measures of diversity, which also include some of the rare species, go above this bound.   in the asymptotic limit where the migration rate goes to zero, trajectories of individual species abundances are described by non-markovian jump-diffusion processes, which proceeds as follows: a rare species remains so for some time, then experiences a jump in population sizes after which it becomes abundant (a species turnover event) and later sees its population size gradually decreasing again until rare, due to the competition with other species. the asymmetry of abundance trajectories under time-reversal is maintained at small but finite migration rate. these features may serve as fingerprints of endogenous fluctuations in highly-diverse ecosystems.',\n",
       "  'categories': 'q-bio.pe cond-mat.dis-nn cond-mat.stat-mech',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-23',\n",
       "  'updated': '2024-01-08',\n",
       "  'authors': ['thibaut arnoulx de pirey', 'guy bunin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.13634'},\n",
       " {'title': 'functional-group-based diffusion for pocket-specific molecule generation   and elaboration',\n",
       "  'id': '2306.13769',\n",
       "  'abstract': \"in recent years, ai-assisted drug design methods have been proposed to generate molecules given the pockets' structures of target proteins. most of them are atom-level-based methods, which consider atoms as basic components and generate atom positions and types. in this way, however, it is hard to generate realistic fragments with complicated structures. to solve this, we propose d3fg, a functional-group-based diffusion model for pocket-specific molecule generation and elaboration. d3fg decomposes molecules into two categories of components: functional groups defined as rigid bodies and linkers as mass points. and the two kinds of components can together form complicated fragments that enhance ligand-protein interactions.   to be specific, in the diffusion process, d3fg diffuses the data distribution of the positions, orientations, and types of the components into a prior distribution; in the generative process, the noise is gradually removed from the three variables by denoisers parameterized with designed equivariant graph neural networks. in the experiments, our method can generate molecules with more realistic 3d structures, competitive affinities toward the protein targets, and better drug properties. besides, d3fg as a solution to a new task of molecule elaboration, could generate molecules with high affinities based on existing ligands and the hotspots of target proteins.\",\n",
       "  'categories': 'q-bio.bm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-05-30',\n",
       "  'updated': '2024-03-18',\n",
       "  'authors': ['haitao lin',\n",
       "   'yufei huang',\n",
       "   'odin zhang',\n",
       "   'lirong wu',\n",
       "   'siyuan li',\n",
       "   'zhiyuan chen',\n",
       "   'stan z. li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.13769'},\n",
       " {'title': 'parametrized systems of generalized polynomial inequalitites via linear   algebra and convex geometry',\n",
       "  'id': '2306.13916',\n",
       "  'abstract': 'we provide fundamental results on positive solutions to parametrized systems of generalized polynomial $\\\\textit{inequalities}$ (with real exponents and positive parameters), including generalized polynomial $\\\\textit{equations}$. in doing so, we also offer a new perspective on fewnomials and (generalized) mass-action systems.   we find that geometric objects, rather than matrices, determine generalized polynomial systems: a bounded set/\"polytope\" $p$ (arising from the coefficient matrix) and two subspaces representing monomial differences and dependencies (arising from the exponent matrix). the dimension of the latter subspace, the monomial dependency $d$, is crucial.   as our main result, we rewrite $\\\\textit{polynomial inequalities}$ in terms of $d$ $\\\\textit{binomial equations}$ on $p$, involving $d$ monomials in the parameters. in particular, we establish an explicit bijection between the original solution set and the solution set on $p$ via exponentiation.   (i) our results apply to any generalized polynomial system. (ii) the dependency $d$ and the dimension of $p$ indicate the complexity of a system. (iii) our results are based on methods from linear algebra and convex/polyhedral geometry, and the solution set on $p$ can be further studied using methods from analysis such as sign-characteristic functions (introduced in this work).   we illustrate our results (in particular, the relevant geometric objects) through three examples from real fewnomial and reaction network theory. for two mass-action systems, we parametrize the set of equilibria and the region for multistationarity, respectively, and even for univariate trinomials, we offer new insights: we provide a \"solution formula\" involving discriminants and \"roots\".',\n",
       "  'categories': 'math.ag q-bio.mn',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-24',\n",
       "  'updated': '2024-10-04',\n",
       "  'authors': ['stefan müller', 'georg regensburger'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.13916'},\n",
       " {'title': 'causal potency of consciousness in the physical world',\n",
       "  'id': '2306.14707',\n",
       "  'abstract': 'the evolution of the human mind through natural selection mandates that our conscious experiences are causally potent in order to leave a tangible impact upon the surrounding physical world. any attempt to construct a functional theory of the conscious mind within the framework of classical physics, however, inevitably leads to causally impotent conscious experiences in direct contradiction to evolution theory. here, we derive several rigorous theorems that identify the origin of the latter impasse in the mathematical properties of ordinary differential equations employed in combination with the alleged functional production of the mind by the brain. then, we demonstrate that a mind--brain theory consistent with causally potent conscious experiences is provided by modern quantum physics, in which the unobservable conscious mind is reductively identified with the quantum state of the brain and the observable brain is constructed by the physical measurement of quantum brain observables. the resulting quantum stochastic dynamics obtained from sequential quantum measurements of the brain is governed by stochastic differential equations, which permit genuine free will exercised through sequential conscious choices of future courses of action. thus, quantum reductionism provides a solid theoretical foundation for the causal potency of consciousness, free will and cultural transmission.',\n",
       "  'categories': 'q-bio.nc quant-ph',\n",
       "  'doi': '10.1142/s0217979224502564',\n",
       "  'created': '2023-06-26',\n",
       "  'updated': '',\n",
       "  'authors': ['danko d. georgiev'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.14707'},\n",
       " {'title': 'dnabert-2: efficient foundation model and benchmark for multi-species   genome',\n",
       "  'id': '2306.15006',\n",
       "  'abstract': 'decoding the linguistic intricacies of the genome is a crucial problem in biology, and pre-trained foundational models such as dnabert and nucleotide transformer have made significant strides in this area. existing works have largely hinged on k-mer, fixed-length permutations of a, t, c, and g, as the token of the genome language due to its simplicity. however, we argue that the computation and sample inefficiencies introduced by k-mer tokenization are primary obstacles in developing large genome foundational models. we provide conceptual and empirical insights into genome tokenization, building on which we propose to replace k-mer tokenization with byte pair encoding (bpe), a statistics-based data compression algorithm that constructs tokens by iteratively merging the most frequent co-occurring genome segment in the corpus. we demonstrate that bpe not only overcomes the limitations of k-mer tokenization but also benefits from the computational efficiency of non-overlapping tokenization. based on these insights, we introduce dnabert-2, a refined genome foundation model that adapts an efficient tokenizer and employs multiple strategies to overcome input length constraints, reduce time and memory expenditure, and enhance model capability. furthermore, we identify the absence of a comprehensive and standardized benchmark for genome understanding as another significant impediment to fair comparative analysis. in response, we propose the genome understanding evaluation (gue), a comprehensive multi-species genome classification dataset that amalgamates $36$ distinct datasets across $9$ tasks, with input lengths ranging from $70$ to $10000$. through comprehensive experiments on the gue benchmark, we demonstrate that dnabert-2 achieves comparable performance to the state-of-the-art model with $21 \\\\times$ fewer parameters and approximately $92 \\\\times$ less gpu time in pre-training.',\n",
       "  'categories': 'q-bio.gn cs.ai cs.ce cs.cl',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-26',\n",
       "  'updated': '2024-03-18',\n",
       "  'authors': ['zhihan zhou',\n",
       "   'yanrong ji',\n",
       "   'weijian li',\n",
       "   'pratik dutta',\n",
       "   'ramana davuluri',\n",
       "   'han liu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.15006'},\n",
       " {'title': 'semi-supervised multimodal representation learning through a global   workspace',\n",
       "  'id': '2306.15711',\n",
       "  'abstract': 'recent deep learning models can efficiently combine inputs from different modalities (e.g., images and text) and learn to align their latent representations, or to translate signals from one domain to another (as in image captioning, or text-to-image generation). however, current approaches mainly rely on brute-force supervised training over large multimodal datasets. in contrast, humans (and other animals) can learn useful multimodal representations from only sparse experience with matched cross-modal data. here we evaluate the capabilities of a neural network architecture inspired by the cognitive notion of a \"global workspace\": a shared representation for two (or more) input modalities. each modality is processed by a specialized system (pretrained on unimodal data, and subsequently frozen). the corresponding latent representations are then encoded to and decoded from a single shared workspace. importantly, this architecture is amenable to self-supervised training via cycle-consistency: encoding-decoding sequences should approximate the identity function. for various pairings of vision-language modalities and across two datasets of varying complexity, we show that such an architecture can be trained to align and translate between two modalities with very little need for matched data (from 4 to 7 times less than a fully supervised approach). the global workspace representation can be used advantageously for downstream classification tasks and for robust transfer learning. ablation studies reveal that both the shared workspace and the self-supervised cycle-consistency training are critical to the system\\'s performance.',\n",
       "  'categories': 'cs.ai q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-27',\n",
       "  'updated': '2024-05-27',\n",
       "  'authors': ['benjamin devillers', 'léopold maytié', 'rufin vanrullen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.15711'},\n",
       " {'title': 'modelling coevolutionary dynamics in heterogeneous si epidemiological   systems across scales',\n",
       "  'id': '2306.16432',\n",
       "  'abstract': 'we develop a new structured compartmental model for the coevolutionary dynamics between susceptible and infectious individuals in heterogeneous si epidemiological systems. in this model, the susceptible compartment is structured by a continuous variable that represents the level of resistance to infection of susceptible individuals, while the infectious compartment is structured by a continuous variable that represents the viral load of infectious individuals. we first formulate an individual-based model wherein the dynamics of single individuals is described through stochastic processes, which permits a fine-grain representation of individual dynamics and captures stochastic variability in evolutionary trajectories amongst individuals. next we formally derive the mesoscopic counterpart of this model, which consists of a system of coupled integro-differential equations for the population density functions of susceptible and infectious individuals. then we consider an appropriately rescaled version of this system and we carry out formal asymptotic analysis to derive the corresponding macroscopic model, which comprises a system of coupled ordinary differential equations for the proportions of susceptible and infectious individuals, the mean level of resistance to infection of susceptible individuals, and the mean viral load of infectious individuals. overall, this leads to a coherent mathematical representation of the coevolutionary dynamics between susceptible and infectious individuals across scales. we provide well-posedness results for the mesoscopic and macroscopic models, and we show that there is excellent agreement between analytical results on the long-time behaviour of the components of the solution to the macroscopic model, the results of monte carlo simulations of the individual-based model, and numerical solutions of the macroscopic model.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.4310/cms.240918045626',\n",
       "  'created': '2023-06-28',\n",
       "  'updated': '2024-03-14',\n",
       "  'authors': ['tommaso lorenzi', 'elisa paparelli', 'andrea tosin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.16432'},\n",
       " {'title': 'nonparametric causal discovery with applications to cancer   bioinformatics',\n",
       "  'id': '2306.16520',\n",
       "  'abstract': 'many natural phenomena are intrinsically causal. the discovery of the cause-effect relationships implicit in these processes can help us to understand and describe them more effectively, which boils down to causal discovery about the data and variables that describe them. however, causal discovery is not an easy task. current methods for this are extremely complex and costly, and their usefulness is strongly compromised in contexts with large amounts of data or where the nature of the variables involved is unknown. as an alternative, this paper presents an original methodology for causal discovery, built on essential aspects of the main theories of causality, in particular probabilistic causality, with many meeting points with the inferential approach of regularity theories and others. based on this methodology, a non-parametric algorithm is developed for the discovery of causal relationships between binary variables associated to data sets, and the modeling in graphs of the causal networks they describe. this algorithm is applied to gene expression data sets in normal and cancerous prostate tissues, with the aim of discovering cause-effect relationships between gene dysregulations leading to carcinogenesis. the gene characterizations constructed from the causal relationships discovered are compared with another study based on principal component analysis (pca) on the same data, with satisfactory results.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-28',\n",
       "  'updated': '2024-01-08',\n",
       "  'authors': ['jean pierre gomez'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.16520'},\n",
       " {'title': 'ongoing eeg artifact correction using blind source separation',\n",
       "  'id': '2306.16910',\n",
       "  'abstract': 'objective: analysis of the electroencephalogram (eeg) for epileptic spike and seizure detection or brain-computer interfaces can be severely hampered by the presence of artifacts. the aim of this study is to describe and evaluate a fast automatic algorithm for ongoing correction of artifacts in continuous eeg recordings, which can be applied offline and online. methods: the automatic algorithm for ongoing correction of artifacts is based on fast blind source separation. it uses a sliding window technique with overlapping epochs and features in the spatial, temporal and frequency domain to detect and correct ocular, cardiac, muscle and powerline artifacts. results: the approach was validated in an independent evaluation study on publicly available continuous eeg data with 2035 marked artifacts. validation confirmed that 88% of the artifacts could be removed successfully (ocular: 81%, cardiac: 84%, muscle: 98%, powerline: 100%). it outperformed state-of-the-art algorithms both in terms of artifact reduction rates and computation time. conclusions: fast ongoing artifact correction successfully removed a good proportion of artifacts, while preserving most of the eeg signals. significance: the presented algorithm may be useful for ongoing correction of artifacts, e.g., in online systems for epileptic spike and seizure detection or brain-computer interfaces.',\n",
       "  'categories': 'q-bio.qm eess.sp stat.me',\n",
       "  'doi': '10.1016/j.clinph.2023.12.133',\n",
       "  'created': '2023-06-29',\n",
       "  'updated': '',\n",
       "  'authors': ['nicole ille',\n",
       "   'yoshiaki nakao',\n",
       "   'yano shumpei',\n",
       "   'toshiyuki taura',\n",
       "   'arndt ebert',\n",
       "   'harald bornfleth',\n",
       "   'suguru asagi',\n",
       "   'kanoko kozawa',\n",
       "   'izumi itabashi',\n",
       "   'takafumi sato',\n",
       "   'rie sakuraba',\n",
       "   'rie tsuda',\n",
       "   'yosuke kakisaka',\n",
       "   'kazutaka jin',\n",
       "   'nobukazu nakasato'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.16910'},\n",
       " {'title': 'the expressive leaky memory neuron: an efficient and expressive   phenomenological neuron model can solve long-horizon tasks',\n",
       "  'id': '2306.16922',\n",
       "  'abstract': \"biological cortical neurons are remarkably sophisticated computational devices, temporally integrating their vast synaptic input over an intricate dendritic tree, subject to complex, nonlinearly interacting internal biological processes. a recent study proposed to characterize this complexity by fitting accurate surrogate models to replicate the input-output relationship of a detailed biophysical cortical pyramidal neuron model and discovered it needed temporal convolutional networks (tcn) with millions of parameters. requiring these many parameters, however, could stem from a misalignment between the inductive biases of the tcn and cortical neuron's computations. in light of this, and to explore the computational implications of leaky memory units and nonlinear dendritic processing, we introduce the expressive leaky memory (elm) neuron model, a biologically inspired phenomenological model of a cortical neuron. remarkably, by exploiting such slowly decaying memory-like hidden states and two-layered nonlinear integration of synaptic input, our elm neuron can accurately match the aforementioned input-output relationship with under ten thousand trainable parameters. to further assess the computational ramifications of our neuron design, we evaluate it on various tasks with demanding temporal structures, including the long range arena (lra) datasets, as well as a novel neuromorphic dataset based on the spiking heidelberg digits dataset (shd-adding). leveraging a larger number of memory units with sufficiently long timescales, and correspondingly sophisticated synaptic integration, the elm neuron displays substantial long-range processing capabilities, reliably outperforming the classic transformer or chrono-lstm architectures on lra, and even solving the pathfinder-x task with over 70% accuracy (16k context length).\",\n",
       "  'categories': 'cs.ne cs.ai cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-14',\n",
       "  'updated': '2024-03-17',\n",
       "  'authors': ['aaron spieler',\n",
       "   'nasim rahaman',\n",
       "   'georg martius',\n",
       "   'bernhard schölkopf',\n",
       "   'anna levina'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.16922'},\n",
       " {'title': 'the state of applying artificial intelligence to tissue imaging for   cancer research and early detection',\n",
       "  'id': '2306.16989',\n",
       "  'abstract': 'artificial intelligence represents a new frontier in human medicine that could save more lives and reduce the costs, thereby increasing accessibility. as a consequence, the rate of advancement of ai in cancer medical imaging and more particularly tissue pathology has exploded, opening it to ethical and technical questions that could impede its adoption into existing systems. in order to chart the path of ai in its application to cancer tissue imaging, we review current work and identify how it can improve cancer pathology diagnostics and research. in this review, we identify 5 core tasks that models are developed for, including regression, classification, segmentation, generation, and compression tasks. we address the benefits and challenges that such methods face, and how they can be adapted for use in cancer prevention and treatment. the studies looked at in this paper represent the beginning of this field and future experiments will build on the foundations that we highlight.',\n",
       "  'categories': 'q-bio.to cs.cv eess.iv',\n",
       "  'doi': '10.12688/f1000research.139210.1',\n",
       "  'created': '2023-06-29',\n",
       "  'updated': '',\n",
       "  'authors': ['michael robben',\n",
       "   'amir hajighasemi',\n",
       "   'mohammad sadegh nasr',\n",
       "   'jai prakesh veerla',\n",
       "   'anne m. alsup',\n",
       "   'biraaj rout',\n",
       "   'helen h. shang',\n",
       "   'kelli fowlds',\n",
       "   'parisa boodaghi malidarreh',\n",
       "   'paul koomey',\n",
       "   'md jillur rahman saurav',\n",
       "   'jacob m. luber'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.16989'},\n",
       " {'title': 'histopathology slide indexing and search: are we there yet?',\n",
       "  'id': '2306.17019',\n",
       "  'abstract': \"the search and retrieval of digital histopathology slides is an important task that has yet to be solved. in this case study, we investigate the clinical readiness of three state-of-the-art histopathology slide search engines, yottixel, sish, and retccl, on three patients with solid tumors. we provide a qualitative assessment of each model's performance in providing retrieval results that are reliable and useful to pathologists. we found that all three image search engines fail to produce consistently reliable results and have difficulties in capturing granular and subtle features of malignancy, limiting their diagnostic accuracy. based on our findings, we also propose a minimal set of requirements to further advance the development of accurate and reliable histopathology image search engines for successful clinical adoption.\",\n",
       "  'categories': 'eess.iv cs.cv q-bio.to',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-29',\n",
       "  'updated': '2024-01-04',\n",
       "  'authors': ['helen h. shang',\n",
       "   'mohammad sadegh nasr',\n",
       "   'jai prakash veerla',\n",
       "   'parisa boodaghi malidarreh',\n",
       "   'md jillur rahman saurav',\n",
       "   'amir hajighasemi',\n",
       "   'manfred huber',\n",
       "   'chace moleta',\n",
       "   'jitin makker',\n",
       "   'jacob m. luber'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.17019'},\n",
       " {'title': 'the exploration-exploitation paradigm for networked biological systems',\n",
       "  'id': '2306.17300',\n",
       "  'abstract': 'the stochastic exploration of the configuration space and the exploitation of functional states underlie many biological processes. the evolutionary dynamics stands out as a remarkable example. here, we introduce a novel formalism that mimics evolution and encodes a general exploration-exploitation dynamics for biological networks. we apply it to the brain wiring problem, focusing on the maturation of that of the nematode c. elegans. we demonstrate that a parsimonious maxent description of the adult brain combined with our framework is able to track down the entire developmental trajectory.',\n",
       "  'categories': 'physics.bio-ph q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-29',\n",
       "  'updated': '2024-01-20',\n",
       "  'authors': ['vito dichio', 'fabrizio de vico fallani'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.17300'},\n",
       " {'title': 'computation of random time-shift distributions for stochastic population   models',\n",
       "  'id': '2306.17331',\n",
       "  'abstract': 'even in large systems, the effect of noise arising from when populations are initially small can persist to be measurable on the macroscale. a deterministic approximation to a stochastic model will fail to capture this effect, but it can be accurately approximated by including an additional random time-shift to the initial conditions. we present a efficient numerical method to compute this time-shift distribution for a large class of stochastic models. the method relies on differentiation of certain functional equations, which we show can be effectively automated by deriving rules for different types of model rates that arise commonly when mass-action mixing is assumed. explicit computation of the time-shift distribution can be used to build a practical tool for the efficient generation of macroscopic trajectories of stochastic population models, without the need for costly stochastic simulations. full code is provided to implement this and we demonstrate our method on an epidemic model and a model of within-host viral dynamics.',\n",
       "  'categories': 'q-bio.pe math.pr',\n",
       "  'doi': '10.1007/s00285-024-02132-6',\n",
       "  'created': '2023-06-29',\n",
       "  'updated': '2024-08-12',\n",
       "  'authors': ['dylan morris', 'john maclean', 'andrew j. black'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.17331'},\n",
       " {'title': 'practical and asymptotically exact conditional sampling in diffusion   models',\n",
       "  'id': '2306.17775',\n",
       "  'abstract': 'diffusion models have been successful on a range of conditional generation tasks including molecular design and text-to-image generation. however, these achievements have primarily depended on task-specific conditional training or error-prone heuristic approximations. ideally, a conditional generation method should provide exact samples for a broad range of conditional distributions without requiring task-specific training. to this end, we introduce the twisted diffusion sampler, or tds. tds is a sequential monte carlo (smc) algorithm that targets the conditional distributions of diffusion models through simulating a set of weighted particles. the main idea is to use twisting, an smc technique that enjoys good computational efficiency, to incorporate heuristic approximations without compromising asymptotic exactness. we first find in simulation and in conditional image generation tasks that tds provides a computational statistical trade-off, yielding more accurate approximations with many particles but with empirical improvements over heuristics with as few as two particles. we then turn to motif-scaffolding, a core task in protein design, using a tds extension to riemannian diffusion models. on benchmark test cases, tds allows flexible conditioning criteria and often outperforms the state of the art.',\n",
       "  'categories': 'stat.ml cs.lg q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-30',\n",
       "  'updated': '2024-11-22',\n",
       "  'authors': ['luhuan wu',\n",
       "   'brian l. trippe',\n",
       "   'christian a. naesseth',\n",
       "   'david m. blei',\n",
       "   'john p. cunningham'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2306.17775'},\n",
       " {'title': \"application of time-aware pc algorithm to compute causal functional   connectivity in alzheimer's disease from fmri data\",\n",
       "  'id': '2307.00253',\n",
       "  'abstract': \"functional connectivity between brain regions is known to be altered in alzheimer's disease, and promises to be a biomarker for early diagnosis of the disease. while several approaches for functional connectivity obtain an un-directed network representing stochastic associations (correlations) between brain regions, association does not necessarily imply causation. in contrast, causal functional connectivity is more informative, providing a directed network representing causal relationships between brain regions. in this paper, we obtained the causal functional connectome for the whole brain from recordings of resting-state functional magnetic resonance imaging (rs-fmri) for subjects from three clinical groups: cognitively normal, mild cognitive impairment, and alzheimer's disease. we applied the recently developed time-aware pc (tpc) algorithm to infer the causal functional connectome for the whole brain. tpc supports model-free estimation of whole brain causal functional connectivity based on directed graphical modeling in a time series setting. we then perform an exploratory analysis to identify the causal brain connections between brain regions which have altered strengths between pairs of subject groups, and over the three subject groups, based on edge-wise p-values from statistical tests. we used the altered causal brain connections thus obtained to compile a comprehensive list of brain regions impacted by alzheimer's disease according to the current data set. the brain regions thus identified are found to be in agreement with literature on brain regions impacted by alzheimer's disease, published by researchers from clinical/medical institutions.\",\n",
       "  'categories': 'q-bio.nc q-bio.qm stat.ap',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-01',\n",
       "  'updated': '2024-02-19',\n",
       "  'authors': ['rahul biswas', 'suryanarayana sripada'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.00253'},\n",
       " {'title': 'improving protein optimization with smoothed fitness landscapes',\n",
       "  'id': '2307.00494',\n",
       "  'abstract': 'the ability to engineer novel proteins with higher fitness for a desired property would be revolutionary for biotechnology and medicine. modeling the combinatorially large space of sequences is infeasible; prior methods often constrain optimization to a small mutational radius, but this drastically limits the design space. instead of heuristics, we propose smoothing the fitness landscape to facilitate protein optimization. first, we formulate protein fitness as a graph signal then use tikunov regularization to smooth the fitness landscape. we find optimizing in this smoothed landscape leads to improved performance across multiple methods in the gfp and aav benchmarks. second, we achieve state-of-the-art results utilizing discrete energy-based models and mcmc in the smoothed landscape. our method, called gibbs sampling with graph-based smoothing (ggs), demonstrates a unique ability to achieve 2.5 fold fitness improvement (with in-silico evaluation) over its training set. ggs demonstrates potential to optimize proteins in the limited data regime. code: https://github.com/kirjner/ggs',\n",
       "  'categories': 'q-bio.bm cs.lg q-bio.qm stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-02',\n",
       "  'updated': '2024-03-02',\n",
       "  'authors': ['andrew kirjner',\n",
       "   'jason yim',\n",
       "   'raman samusevich',\n",
       "   'shahar bracha',\n",
       "   'tommi jaakkola',\n",
       "   'regina barzilay',\n",
       "   'ila fiete'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.00494'},\n",
       " {'title': 'on efficient computation in active inference',\n",
       "  'id': '2307.00504',\n",
       "  'abstract': \"despite being recognized as neurobiologically plausible, active inference faces difficulties when employed to simulate intelligent behaviour in complex environments due to its computational cost and the difficulty of specifying an appropriate target distribution for the agent. this paper introduces two solutions that work in concert to address these limitations. first, we present a novel planning algorithm for finite temporal horizons with drastically lower computational complexity. second, inspired by z-learning from control theory literature, we simplify the process of setting an appropriate target distribution for new and existing active inference planning schemes. our first approach leverages the dynamic programming algorithm, known for its computational efficiency, to minimize the cost function used in planning through the bellman-optimality principle. accordingly, our algorithm recursively assesses the expected free energy of actions in the reverse temporal order. this improves computational efficiency by orders of magnitude and allows precise model learning and planning, even under uncertain conditions. our method simplifies the planning process and shows meaningful behaviour even when specifying only the agent's final goal state. the proposed solutions make defining a target distribution from a goal state straightforward compared to the more complicated task of defining a temporally informed target distribution. the effectiveness of these methods is tested and demonstrated through simulations in standard grid-world tasks. these advances create new opportunities for various applications.\",\n",
       "  'categories': 'cs.lg cs.ai q-bio.nc',\n",
       "  'doi': '10.1016/j.eswa.2024.124315',\n",
       "  'created': '2023-07-02',\n",
       "  'updated': '',\n",
       "  'authors': ['aswin paul', 'noor sajid', 'lancelot da costa', 'adeel razi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.00504'},\n",
       " {'title': 'medcpt: contrastive pre-trained transformers with large-scale pubmed   search logs for zero-shot biomedical information retrieval',\n",
       "  'id': '2307.00589',\n",
       "  'abstract': 'information retrieval (ir) is essential in biomedical knowledge acquisition and clinical decision support. while recent progress has shown that language model encoders perform better semantic retrieval, training such models requires abundant query-article annotations that are difficult to obtain in biomedicine. as a result, most biomedical ir systems only conduct lexical matching. in response, we introduce medcpt, a first-of-its-kind contrastively pre-trained transformer model for zero-shot semantic ir in biomedicine. to train medcpt, we collected an unprecedented scale of 255 million user click logs from pubmed. with such data, we use contrastive learning to train a pair of closely-integrated retriever and re-ranker. experimental results show that medcpt sets new state-of-the-art performance on six biomedical ir tasks, outperforming various baselines including much larger models such as gpt-3-sized cpt-text-xl. in addition, medcpt also generates better biomedical article and sentence representations for semantic evaluations. as such, medcpt can be readily applied to various real-world biomedical ir tasks.',\n",
       "  'categories': 'cs.ir cs.ai cs.cl q-bio.qm',\n",
       "  'doi': '10.1093/bioinformatics/btad651',\n",
       "  'created': '2023-07-02',\n",
       "  'updated': '2023-10-03',\n",
       "  'authors': ['qiao jin',\n",
       "   'won kim',\n",
       "   'qingyu chen',\n",
       "   'donald c. comeau',\n",
       "   'lana yeganova',\n",
       "   'w. john wilbur',\n",
       "   'zhiyong lu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.00589'},\n",
       " {'title': 'signature of (anti)cooperativity in the stochastic fluctuations of small   systems: application to the bacterial flagellar motor',\n",
       "  'id': '2307.00636',\n",
       "  'abstract': 'the cooperative binding of molecular agents onto a substrate is pervasive in living systems. to study whether a system shows cooperativity, one can rely on a fluctuation analysis of quantities such as the number of substrate-bound units and the residence time in an occupancy state. since the relative standard deviation from the statistical mean monotonically decreases with the number of binding sites, these techniques are only suitable for small enough systems, such as those implicated in stochastic processes inside cells. here, we present a general-purpose grand canonical hamiltonian description of a small one-dimensional (1d) lattice gas with either nearest-neighbor or long-range interactions as prototypical examples of cooperativity-influenced adsorption processes. first, we elucidate how the strength and sign of the interaction potential between neighboring bound particles on the lattice determine the intensity of the fluctuations of the mean occupancy. we then employ this relationship to compare the theoretical predictions of our model to data from single molecule experiments on bacterial flagellar motors (bfm) of e. coli. in this way, we find evidence that cooperativity controls the mechano-sensitive dynamical assembly of the torque-generating units, the so-called stator units, onto the bfm. furthermore, in an attempt to quantify fluctuations and the adaptability of the bfm, we estimate the stator-stator interaction potential. finally, we conclude that the system resides in a sweet spot of the parameter space (phase diagram) suitable for a smoothly adaptive system while minimizing fluctuations.',\n",
       "  'categories': 'physics.bio-ph cond-mat.stat-mech q-bio.sc',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-02',\n",
       "  'updated': '2024-02-21',\n",
       "  'authors': ['maría-josé franco-oñate',\n",
       "   'andrea parmeggiani',\n",
       "   'jérôme dorignac',\n",
       "   'frédéric geniet',\n",
       "   'jean-charles walter',\n",
       "   'francesco pedaci',\n",
       "   'ashley l nord',\n",
       "   'john palmeri',\n",
       "   'nils-ole walliser'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.00636'},\n",
       " {'title': 'cardigraphormer: unveiling the power of self-supervised learning in   revolutionizing drug discovery',\n",
       "  'id': '2307.00859',\n",
       "  'abstract': \"in the expansive realm of drug discovery, with approximately 15,000 known drugs and only around 4,200 approved, the combinatorial nature of the chemical space presents a formidable challenge. while artificial intelligence (ai) has emerged as a powerful ally, traditional ai frameworks face significant hurdles. this manuscript introduces cardigraphormer, a groundbreaking approach that synergizes self-supervised learning (ssl), graph neural networks (gnns), and cardinality preserving attention to revolutionize drug discovery. cardigraphormer, a novel combination of graphormer and cardinality preserving attention, leverages ssl to learn potent molecular representations and employs gnns to extract molecular fingerprints, enhancing predictive performance and interpretability while reducing computation time. it excels in handling complex data like molecular structures and performs tasks associated with nodes, pairs of nodes, subgraphs, or entire graph structures. cardigraphormer's potential applications in drug discovery and drug interactions are vast, from identifying new drug targets to predicting drug-to-drug interactions and enabling novel drug discovery. this innovative approach provides an ai-enhanced methodology in drug development, utilizing ssl combined with gnns to overcome existing limitations and pave the way for a richer exploration of the vast combinatorial chemical space in drug discovery.\",\n",
       "  'categories': 'cs.lg q-bio.qm stat.ap stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-03',\n",
       "  'updated': '2024-01-13',\n",
       "  'authors': ['abhijit gupta'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.00859'},\n",
       " {'title': 'mechanical compression regulates tumor spheroid invasion into a 3d   collagen matrix',\n",
       "  'id': '2307.01289',\n",
       "  'abstract': 'uncontrolled growth of tumor cells in confined spaces leads to the accumulation of compressive stress within the tumor. although the effects of tension within 3d extracellular matrices on tumor growth and invasion are well established, the role of compression in tumor mechanics and invasion is largely unexplored. in this study, we modified a transwell assay such that it provides constant compressive loads to spheroids embedded within a collagen matrix. we used microscopic imaging to follow the single cell dynamics of the cells within the spheroids, as well as invasion into the 3d extracellular matrices (emcs). our experimental results showed that malignant breast tumor (mda-mb-231) and non-tumorigenic epithelial (mcf10a) spheroids responded differently to a constant compression. cells within the malignant spheroids became more motile within the spheroids and invaded more into the ecm under compression; whereas cells within non-tumorigenic mcf10a spheroids became less motile within the spheroids and did not display apparent detachment from the spheroids under compression. these findings suggest that compression may play differential roles in healthy and pathogenic epithelial tissues and highlights the importance of tumor mechanics and invasion.',\n",
       "  'categories': 'q-bio.cb',\n",
       "  'doi': '10.1088/1478-3975/ad3ac5',\n",
       "  'created': '2023-07-03',\n",
       "  'updated': '',\n",
       "  'authors': ['mrinal pandey',\n",
       "   'young joon suh',\n",
       "   'minha kim',\n",
       "   'hannah jane davis',\n",
       "   'jeffrey e segall',\n",
       "   'mingming wu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.01289'},\n",
       " {'title': 'analysis of complex excitation patterns using feynman-like diagrams',\n",
       "  'id': '2307.01508',\n",
       "  'abstract': \"many extended chemical and biological systems self-organise into complex patterns that drive the medium behaviour in a non-linear fashion. an important class of such systems are excitable media, including neural and cardiac tissues. in extended excitable media, wave breaks can form rotating patterns and turbulence. however, the onset, sustaining and elimination of such complex patterns is currently incompletely understood. the classical theory of phase singularities in excitable media was recently challenged, as extended lines of conduction block were identified as phase discontinuities. here, we provide a theoretical framework that captures the rich dynamics in excitable systems in terms of three quasiparticles: heads, tails, and pivots. we propose to call these quasiparticles `cardions'. in simulations and experiments, we show that these basic building blocks combine into at least four different bound states. by representing their interactions similarly to feynman diagrams in physics, the creation and annihilation of vortex pairs are shown to be sequences of dynamical creation, annihilation, and recombination of the identified quasiparticles. we draw such diagrams for numerical simulations, as well as optical voltage mapping experiments performed on cultured human atrial myocytes (hiams). our results provide a new, unified language for a more detailed theory, analysis, and mechanistic insights of dynamical transitions in excitation patterns.\",\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-04',\n",
       "  'updated': '2024-09-06',\n",
       "  'authors': ['louise arno', 'desmond kabus', 'hans dierckx'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.01508'},\n",
       " {'title': 'single-molecule fluorescence multiplexing by multi-parameter   spectroscopic detection of nanostructured fret labels',\n",
       "  'id': '2307.01614',\n",
       "  'abstract': 'multiplexed, real-time fluorescence detection at the single-molecule level is highly desirable to reveal the stoichiometry, dynamics, and interactions of individual molecular species within complex systems. however, traditionally fluorescence sensing is limited to 3-4 concurrently detected labels, due to low signal-to-noise, high spectral overlap between labels, and the need to avoid dissimilar dye chemistries. we have engineered a palette of several dozen fluorescent labels, called fretfluors, for spectroscopic multiplexing at the single-molecule level. each fretfluor is a compact nanostructure formed from the same three chemical building blocks (dna, cy3, and cy5). the composition and dye-dye geometries create a characteristic f\\\\\"orster resonance energy transfer (fret) efficiency for each construct. in addition, we varied the local dna sequence and attachment chemistry to alter the cy3 and cy5 emission properties and thereby shift the emission signatures of an entire series of fret constructs to new sectors of the multi-parameter detection space. unique spectroscopic emission of each fretfluor is therefore conferred by a combination of fret and this site-specific tuning of individual fluorophore photophysics. we show single-molecule identification of a set of 27 fretfluors in a sample mixture using a subset of constructs statistically selected to minimize classification errors, measured using an anti-brownian electrokinetic (abel) trap which provides precise multi-parameter spectroscopic measurements. the abel trap also enables discrimination between fretfluors attached to a target (here: mrna) and unbound fretfluors, eliminating the need for washes or removal of excess label by purification. we show single-molecule identification of a set of 27 fretfluors in a sample mixture using a subset of constructs selected to minimize classification errors.',\n",
       "  'categories': 'physics.bio-ph q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-04',\n",
       "  'updated': '2024-01-25',\n",
       "  'authors': ['jiachong chu',\n",
       "   'ayesha ejaz',\n",
       "   'kyle m. lin',\n",
       "   'madeline r. joseph',\n",
       "   'aria e. coraor',\n",
       "   'd. allan drummond',\n",
       "   'allison h. squires'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.01614'},\n",
       " {'title': 'probability metrics for tropical spaces of different dimensions',\n",
       "  'id': '2307.02846',\n",
       "  'abstract': 'the problem of comparing probability distributions is at the heart of many tasks in statistics and machine learning. established comparison methods treat the standard setting that the distributions are supported in the same space. recently, a new geometric solution has been proposed to address the more challenging problem of comparing measures in euclidean spaces of differing dimensions. here, we study the same problem of comparing probability distributions of different dimensions in the tropical setting, which is becoming increasingly relevant in applications involving complex data structures such as phylogenetic trees. specifically, we construct a wasserstein distance between measures on different tropical projective tori -- the focal metric spaces in both theory and applications of tropical geometry -- via tropical mappings between probability measures. we prove equivalence of the directionality of the maps, whether mapping from a low dimensional space to a high dimensional space or vice versa. as an important practical implication, our work provides a framework for comparing probability distributions on the spaces of phylogenetic trees with different leaf sets. we demonstrate the computational feasibility of our approach using existing optimisation techniques on both simulated and real data.',\n",
       "  'categories': 'math.mg math.co math.st q-bio.pe stat.th',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-06',\n",
       "  'updated': '2024-09-29',\n",
       "  'authors': ['roan talbut',\n",
       "   'daniele tramontano',\n",
       "   'yueqi cao',\n",
       "   'mathias drton',\n",
       "   'anthea monod'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.02846'},\n",
       " {'title': 'generalized lotka-volterra systems with time correlated stochastic   interactions',\n",
       "  'id': '2307.02851',\n",
       "  'abstract': 'in this work, we explore the dynamics of species abundances within ecological communities using the generalized lotka-volterra (glv) model. at variance with previous approaches, we present an analysis of stochastic glv dynamics with temporal fluctuations in interaction strengths between species. we develop a dynamical mean field theory (dmft) tailored for scenarios with annealed colored noise and simple functional responses. we show that time-dependent interactions can be effectively modeled as environmental noise in the dmft and we obtain analytical predictions for the species abundance distribution that well matches empirical observations. our results suggest that environmental noise favors species coexistence and allows to overcome the complexity-stability paradox, especially in comparison to dynamics with quenched disorder. this study offers new insights not only into the modeling of large ecosystem dynamics, but also proposes novel methodologies for examining ecological systems.',\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-06',\n",
       "  'updated': '2024-04-23',\n",
       "  'authors': ['samir suweis',\n",
       "   'francesco ferraro',\n",
       "   'christian grilletta',\n",
       "   'sandro azaele',\n",
       "   'amos maritan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.02851'},\n",
       " {'title': 'corrected hill function in stochastic gene regulatory networks',\n",
       "  'id': '2307.03057',\n",
       "  'abstract': 'describing reaction rates in stochastic bio-circuits is commonly done by directly introducing the deterministically deduced hill function into the master equation. however, when fluctuations in enzymatic reaction rates are not neglectable, the hill function must be derived, considering all the involved stochastic reactions. in this work, we derived the stochastic version of the hill function from the master equation of the complete set of reactions that, in the macroscopic limit, lead to the hill function reaction rate. we performed a series expansion around the average values of the concentrations, which allowed us to find corrections for the deterministic hill function. this process allowed us to quantify the fluctuations of enzymatic reactions. we found that the underlying variability in propensity rates of gene regulatory networks has an important non-linear effect that reduces the intrinsic fluctuations of the mrna and protein concentrations.',\n",
       "  'categories': 'q-bio.mn q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-06',\n",
       "  'updated': '2024-08-22',\n",
       "  'authors': ['manuel eduardo hernández-garcía', 'jorge velázquez-castro'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.03057'},\n",
       " {'title': 'learning curves for noisy heterogeneous feature-subsampled ridge   ensembles',\n",
       "  'id': '2307.03176',\n",
       "  'abstract': 'feature bagging is a well-established ensembling method which aims to reduce prediction variance by combining predictions of many estimators trained on subsets or projections of features. here, we develop a theory of feature-bagging in noisy least-squares ridge ensembles and simplify the resulting learning curves in the special case of equicorrelated data. using analytical learning curves, we demonstrate that subsampling shifts the double-descent peak of a linear predictor. this leads us to introduce heterogeneous feature ensembling, with estimators built on varying numbers of feature dimensions, as a computationally efficient method to mitigate double-descent. then, we compare the performance of a feature-subsampling ensemble to a single linear predictor, describing a trade-off between noise amplification due to subsampling and noise reduction due to ensembling. our qualitative insights carry over to linear classifiers applied to image classification tasks with realistic datasets constructed using a state-of-the-art deep learning feature map.',\n",
       "  'categories': 'stat.ml cond-mat.dis-nn cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-06',\n",
       "  'updated': '2024-01-09',\n",
       "  'authors': ['benjamin s. ruben', 'cengiz pehlevan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.03176'},\n",
       " {'title': 'solving stochastic gene expression models using queueing theory: a   tutorial review',\n",
       "  'id': '2307.03253',\n",
       "  'abstract': 'stochastic models of gene expression are typically formulated using the chemical master equation, which can be solved exactly or approximately using a repertoire of analytical methods. here, we provide a tutorial review of an alternative approach based on queueing theory that has rarely been used in the literature of gene expression. we discuss the interpretation of six types of infinite server queues from the angle of stochastic single-cell biology and provide analytical expressions for the stationary and non-stationary distributions and/or moments of mrna/protein numbers, and bounds on the fano factor. this approach may enable the solution of complex models which have hitherto evaded analytical solution.',\n",
       "  'categories': 'q-bio.sc',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-06',\n",
       "  'updated': '2024-04-18',\n",
       "  'authors': ['juraj szavits-nossan', 'ramon grima'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.03253'},\n",
       " {'title': 'limit theorems for the site frequency spectrum of neutral mutations in   an exponentially growing population',\n",
       "  'id': '2307.03346',\n",
       "  'abstract': 'the site frequency spectrum (sfs) is a widely used summary statistic of genomic data. motivated by recent evidence for the role of neutral evolution in cancer, we investigate the sfs of neutral mutations in an exponentially growing population. using branching process techniques, we establish (first-order) almost sure convergence results for the sfs of a galton-watson process, evaluated either at a fixed time or at the stochastic time at which the population first reaches a certain size. we finally use our results to construct consistent estimators for the extinction probability and the effective mutation rate of a birth-death process.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-06',\n",
       "  'updated': '2024-03-12',\n",
       "  'authors': ['einar bjarki gunnarsson', 'kevin leder', 'xuanming zhang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.03346'},\n",
       " {'title': 'persistent disruption of interspecific competition after ultra-low   esfenvalerate exposure',\n",
       "  'id': '2307.03605',\n",
       "  'abstract': 'field and mesocosm studies repeatedly show that higher tier process reduce the predictive accuracy of toxicity evaluation and consequently their value for pesticide risk assessment. therefore, understanding the influence of ecological complexity on toxicant effects is crucial to improve realism of aquatic risk assessment. here we investigate the influence of repeated exposure to ecologically realistic concentrations of esfenvalerate on the similarly sensitive species daphnia magna and culex pipiens in a food limited and highly competitive environment. we show that significant perturbations in population development are only present close to the ec50. in contrast, interspecific competition between species is already reduced at concentrations 3-4 orders of magnitude below the acute ec50. we conclude that extremely low, environmentally relevant concentrations can disrupt species interactions. this toxicant mediated alteration of competitive balances in ecological communities may be the underlying mechanism for shifts in species distribution at ultra-low pesticide concentrations. a realistic risk assessment should therefore consider these processes in order to predict potential pesticide effects on the structure of communities.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1016/j.scitotenv.2023.167455',\n",
       "  'created': '2023-07-07',\n",
       "  'updated': '',\n",
       "  'authors': ['florian schunck', 'matthias liess'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.03605'},\n",
       " {'title': 'absolute concentration robustness and multistationarity in reaction   networks: conditions for coexistence',\n",
       "  'id': '2307.04186',\n",
       "  'abstract': 'many reaction networks arising in applications are multistationary, that is, they have the capacity for more than one steady state; while some networks exhibit absolute concentration robustness (acr), which means that some species concentration is the same at all steady states. both multistationarity and acr are significant in biological settings, but only recently has attention focused on the possibility for these properties to coexist. our main result states that such coexistence in at-most-bimolecular networks (which encompass most networks arising in biology) requires at least $3$ species, $5$ complexes, and $3$ reactions. we prove additional bounds on the number of reactions for general networks based on the number of linear conservation laws. finally, we prove that, outside of a few exceptional cases, acr is equivalent to non-multistationarity for bimolecular networks that are small (more precisely, one-dimensional or up to two species). our proofs involve analyses of systems of sparse polynomials, and we also use classical results from chemical reaction network theory.',\n",
       "  'categories': 'math.ds math.ag q-bio.mn',\n",
       "  'doi': '10.1017/s0956792523000335',\n",
       "  'created': '2023-07-09',\n",
       "  'updated': '',\n",
       "  'authors': ['nidhi kaihnsa', 'tung nguyen', 'anne shiu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.04186'},\n",
       " {'title': 'tropical convexity in location problems',\n",
       "  'id': '2307.04465',\n",
       "  'abstract': 'we investigate location problems whose optimum lies in the tropical convex hull of the input points. firstly, we study geodesically star-convex sets under the asymmetric tropical distance and introduce the class of tropically quasiconvex functions whose sub-level sets have this shape. the latter are related to monotonic functions. then we show that location problems whose distances are measured by tropically quasiconvex functions as before give an optimum in the tropical convex hull of the input points. we also show that a similar result holds if we replace the input points by tropically convex sets. finally, we focus on applications to phylogenetics presenting properties of consensus methods arising from our class of location problems.',\n",
       "  'categories': 'math.oc math.mg q-bio.pe',\n",
       "  'doi': '10.1007/s00186-024-00869-w',\n",
       "  'created': '2023-07-10',\n",
       "  'updated': '',\n",
       "  'authors': ['andrei comăneci'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.04465'},\n",
       " {'title': 'strength and weakness of disease-induced herd immunity in networks',\n",
       "  'id': '2307.04700',\n",
       "  'abstract': 'when a fraction of a population becomes immune to an infectious disease, the population-wide infection risk decreases nonlinearly due to collective protection known as herd immunity. studies based on mean-field models suggest that natural infection in a heterogeneous population may induce herd immunity more efficiently than homogeneous immunization. here, we use network epidemic models to show that the opposite can also be the case. we identify two competing mechanisms driving disease-induced herd immunity in networks: the high density of immunity among socially active individuals enhances the herd immunity effect, while the topological localization of immune individuals weakens it. the effect of localization is stronger in networks embedded in low-dimensional space, which can make disease-induced immunity less effective than random immunization. our results highlight the role of networks in shaping herd immunity and call for careful examination of model predictions that inform public health policies.',\n",
       "  'categories': 'physics.soc-ph q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-10',\n",
       "  'updated': '2024-07-03',\n",
       "  'authors': ['takayuki hiraoka',\n",
       "   'zahra ghadiri',\n",
       "   'abbas k. rizi',\n",
       "   'mikko kivelä',\n",
       "   'jari saramäki'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.04700'},\n",
       " {'title': 'mathematical modeling insights into improving car t cell therapy for   solid tumors: antigen heterogeneity and bystander effects',\n",
       "  'id': '2307.05606',\n",
       "  'abstract': 'as an adoptive cellular therapy, chimeric antigen receptor t-cell (car t-cell) therapy has shown remarkable success in hematological malignancies, but only limited efficacy against solid tumors. compared with blood cancers, solid tumors present a unique set of challenges that ultimately neutralize the function of car t-cells. one such barrier is antigen heterogeneity - variability in the expression of the antigen on tumor cells. success of car t-cell therapy in solid tumors is unlikely unless almost all the tumor cells express the specific antigen that car t-cells target. a critical question for solving the heterogeneity problem is whether car t therapy induces bystander effects, such as antigen spreading. antigen spreading occurs when car t-cells activate other endogenous antitumor cd8 t cells against antigens that were not originally targeted. in this work, we develop a mathematical model of car t-cell therapy for solid tumors that takes into consideration both antigen heterogeneity and bystander effects. our model is based on in vivo treatment data that includes a mixture of target antigen-positive and target antigen-negative tumor cells. we use our model to simulate large cohorts of virtual patients to gain a better understanding of the relationship between bystander killing. we also investigate several strategies for enhancing the bystander effect and thus increasing the overall efficacy of car t-cell therapy for solid tumor.',\n",
       "  'categories': 'q-bio.to',\n",
       "  'doi': '10.1038/s41540-024-00435-4',\n",
       "  'created': '2023-07-10',\n",
       "  'updated': '',\n",
       "  'authors': ['erdi kara',\n",
       "   't. l. jackson',\n",
       "   'chartese jones',\n",
       "   'reginald l. mcgee',\n",
       "   'rockford sison'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.05606'},\n",
       " {'title': 'mean-field interacting multi-type birth-death processes with a view to   applications in phylodynamics',\n",
       "  'id': '2307.06010',\n",
       "  'abstract': \"multi-type birth-death processes underlie approaches for inferring evolutionary dynamics from phylogenetic trees across biological scales, ranging from deep-time species macroevolution to rapid viral evolution and somatic cellular proliferation. a limitation of current phylogenetic birth-death models is that they require restrictive linearity assumptions that yield tractable message-passing likelihoods, but that also preclude interactions between individuals. many fundamental evolutionary processes -- such as environmental carrying capacity or frequency-dependent selection -- entail interactions, and may strongly influence the dynamics in some systems. here, we introduce a multi-type birth-death process in mean-field interaction with an ensemble of replicas of the focal process. we prove that, under quite general conditions, the ensemble's stochastically evolving interaction field converges to a deterministic trajectory in the limit of an infinite ensemble. in this limit, the replicas effectively decouple, and self-consistent interactions appear as nonlinearities in the infinitesimal generator of the focal process. we investigate a special case that is rich enough to model both carrying capacity and frequency-dependent selection while yielding tractable message-passing likelihoods in the context of a phylogenetic birth-death model.\",\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-12',\n",
       "  'updated': '2024-03-31',\n",
       "  'authors': ['william s. dewitt',\n",
       "   'steven n. evans',\n",
       "   'ella hiesmayr',\n",
       "   'sebastian hummel'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.06010'},\n",
       " {'title': 'determinants of successful disease control through voluntary quarantine   dynamics on social networks',\n",
       "  'id': '2307.06297',\n",
       "  'abstract': 'in the wake of epidemics, quarantine measures are typically recommended by health authorities or governments to help control the spread of the disease. compared with mandatory quarantine, voluntary quarantine offers individuals the liberty to decide whether to isolate themselves in case of infection exposure, driven by their personal assessment of the trade-off between economic loss and health risks as well as their own sense of social responsibility and concern for public health. to better understand self-motivated health behavior choices under these factors, here we incorporate voluntary quarantine into an endemic disease model -- the susceptible-infected-susceptible (sis) model -- and perform comprehensive agent-based simulations to characterize the resulting behavior-disease interactions in structured populations. we quantify the conditions under which voluntary quarantine will be an effective intervention measure to mitigate disease burden. furthermore, we demonstrate how individual decision-making factors, including the level of temptation to refrain from quarantine and the degree of social compassion, impact compliance levels of voluntary quarantines and the consequent collective disease mitigation efforts. we find that successful disease control requires either a sufficiently low level of temptation or a sufficiently high degree of social compassion, such that even complete containment of the epidemic is attainable. in addition to well-mixed populations, our simulation results are applicable to other more realistic social networks of contacts, including spatial lattices, small-world networks, and real social networks. our work offers new insights into the fundamental social dilemma aspect of disease control through non-pharmaceutical interventions, such as voluntary quarantine and isolation, where the collective outcome of individual decision-making is crucial.',\n",
       "  'categories': 'physics.soc-ph q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-12',\n",
       "  'updated': '2024-03-13',\n",
       "  'authors': ['simiao shi', 'zhiyuan wang', 'xingru chen', 'feng fu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.06297'},\n",
       " {'title': 'neutral diversity in experimental metapopulations',\n",
       "  'id': '2307.06393',\n",
       "  'abstract': 'new automated and high-throughput methods allow the manipulation and selection of numerous bacterial populations. in this manuscript we are interested in the neutral diversity patterns that emerge from such a setup in which many bacterial populations are grown in parallel serial transfers, in some cases with population-wide extinction and splitting events. we model bacterial growth by a birth-death process and use the theory of coalescent point processes. we show that there is a dilution factor that optimises the expected amount of neutral diversity for a given amount of cycles, and study the power law behaviour of the mutation frequency spectrum for different experimental regimes. we also explore how neutral variation diverges between two recently split populations by establishing a new formula for the expected number of shared and private mutations. finally, we show the interest of such a setup to select a phenotype of interest that requires multiple mutations.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-12',\n",
       "  'updated': '2024-03-12',\n",
       "  'authors': ['guilhem doulcier', 'amaury lambert'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.06393'},\n",
       " {'title': 'early autism diagnosis based on path signature and siamese unsupervised   feature compressor',\n",
       "  'id': '2307.06472',\n",
       "  'abstract': 'autism spectrum disorder (asd) has been emerging as a growing public health threat. early diagnosis of asd is crucial for timely, effective intervention and treatment. however, conventional diagnosis methods based on communications and behavioral patterns are unreliable for children younger than 2 years of age. given evidences of neurodevelopmental abnormalities in asd infants, we resort to a novel deep learning-based method to extract key features from the inherently scarce, class-imbalanced, and heterogeneous structural mr images for early autism diagnosis. specifically, we propose a siamese verification framework to extend the scarce data, and an unsupervised compressor to alleviate data imbalance by extracting key features. we also proposed weight constraints to cope with sample heterogeneity by giving different samples different voting weights during validation, and we used path signature to unravel meaningful developmental features from the two-time point data longitudinally. we further extracted machine learning focused brain regions for autism diagnosis. extensive experiments have shown that our method performed well under practical scenarios, transcending existing machine learning methods and providing anatomical insights for autism early diagnosis.',\n",
       "  'categories': 'cs.cv q-bio.nc',\n",
       "  'doi': '10.1093/cercor/bhae069',\n",
       "  'created': '2023-07-12',\n",
       "  'updated': '2024-05-02',\n",
       "  'authors': ['zhuowen yin',\n",
       "   'xinyao ding',\n",
       "   'xin zhang',\n",
       "   'zhengwang wu',\n",
       "   'li wang',\n",
       "   'xiangmin xu',\n",
       "   'gang li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.06472'},\n",
       " {'title': 'fast and functional structured data generators rooted in   out-of-equilibrium physics',\n",
       "  'id': '2307.06797',\n",
       "  'abstract': \"in this study, we address the challenge of using energy-based models to produce high-quality, label-specific data in complex structured datasets, such as population genetics, rna or protein sequences data. traditional training methods encounter difficulties due to inefficient markov chain monte carlo mixing, which affects the diversity of synthetic data and increases generation times. to address these issues, we use a novel training algorithm that exploits non-equilibrium effects. this approach, applied on the restricted boltzmann machine, improves the model's ability to correctly classify samples and generate high-quality synthetic data in only a few sampling steps. the effectiveness of this method is demonstrated by its successful application to four different types of data: handwritten digits, mutations of human genomes classified by continental origin, functionally characterized sequences of an enzyme protein family, and homologous rna sequences from specific taxonomies.\",\n",
       "  'categories': 'cs.lg cond-mat.dis-nn q-bio.bm q-bio.gn q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-13',\n",
       "  'updated': '2024-11-12',\n",
       "  'authors': ['alessandra carbone',\n",
       "   'aurélien decelle',\n",
       "   'lorenzo rosset',\n",
       "   'beatriz seoane'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.06797'},\n",
       " {'title': 'modeling correlated uncertainties in stochastic compartmental models',\n",
       "  'id': '2307.06866',\n",
       "  'abstract': \"we consider compartmental models of communicable disease with uncertain contact rates. stochastic fluctuations are often added to the contact rate to account for uncertainties. white noise, which is the typical choice for the fluctuations, leads to significant underestimation of the disease severity. here, starting from reasonable assumptions on the social behavior of individuals, we model the contacts as a markov process which takes into account the temporal correlations present in human social activities. consequently, we show that the mean-reverting ornstein-uhlenbeck (ou) process is the correct model for the stochastic contact rate. we demonstrate the implication of our model on two examples: a susceptibles-infected-susceptibles (sis) model and a susceptibles-exposed-infected-removed (seir) model of the covid-19 pandemic. in particular, we observe that both compartmental models with white noise uncertainties undergo transitions that lead to the systematic underestimation of the spread of the disease. in contrast, modeling the contact rate with the ou process significantly hinders such unrealistic noise-induced transitions. for the sis model, we derive its stationary probability density analytically, for both white and correlated noise. this allows us to give a complete description of the model's asymptotic behavior as a function of its bifurcation parameters, i.e., the basic reproduction number, noise intensity, and correlation time. for the seir model, where the probability density is not available in closed form, we study the transitions using monte carlo simulations. our study underscores the necessity of temporal correlations in stochastic compartmental models and the need for more empirical studies that would systematically quantify such correlations.\",\n",
       "  'categories': 'q-bio.pe math.ds math.pr',\n",
       "  'doi': '10.1016/j.mbs.2024.109226',\n",
       "  'created': '2023-07-09',\n",
       "  'updated': '',\n",
       "  'authors': ['konstantinos mamis', 'mohammad farazmand'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.06866'},\n",
       " {'title': 'aligned and oblique dynamics in recurrent neural networks',\n",
       "  'id': '2307.07654',\n",
       "  'abstract': \"the relation between neural activity and behaviorally relevant variables is at the heart of neuroscience research. when strong, this relation is termed a neural representation. there is increasing evidence, however, for partial dissociations between activity in an area and relevant external variables. while many explanations have been proposed, a theoretical framework for the relationship between external and internal variables is lacking. here, we utilize recurrent neural networks (rnns) to explore the question of when and how neural dynamics and the network's output are related from a geometrical point of view. we find that training rnns can lead to two dynamical regimes: dynamics can either be aligned with the directions that generate output variables, or oblique to them. we show that the choice of readout weight magnitude before training can serve as a control knob between the regimes, similar to recent findings in feedforward networks. these regimes are functionally distinct. oblique networks are more heterogeneous and suppress noise in their output directions. they are furthermore more robust to perturbations along the output directions. crucially, the oblique regime is specific to recurrent (but not feedforward) networks, arising from dynamical stability considerations. finally, we show that tendencies towards the aligned or the oblique regime can be dissociated in neural recordings. altogether, our results open a new perspective for interpreting neural activity by relating network dynamics and their output.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-14',\n",
       "  'updated': '2024-08-02',\n",
       "  'authors': ['friedrich schuessler',\n",
       "   'francesca mastrogiuseppe',\n",
       "   'srdjan ostojic',\n",
       "   'omri barak'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.07654'},\n",
       " {'title': 'trajectory inference for a branching sde model of cell differentiation',\n",
       "  'id': '2307.07687',\n",
       "  'abstract': 'a core challenge for modern biology is how to infer the trajectories of individual cells from population-level time courses of high-dimensional gene expression data. birth and death of cells present a particular difficulty: existing trajectory inference methods cannot distinguish variability in net proliferation from cell differentiation dynamics, and hence require accurate prior knowledge of the proliferation rate. building on global waddington-ot (gwot), which performs trajectory inference with rigorous theoretical guarantees when birth and death can be neglected, we show how to use lineage trees available with recently developed crispr-based measurement technologies to disentangle proliferation and differentiation. in particular, when there is neither death nor subsampling of cells, we show that we extend gwot to the case with proliferation with similar theoretical guarantees and computational cost, without requiring any prior information. in the case of death and/or subsampling, our method introduces a bias, that we describe explicitly and argue to be inherent to these lineage tracing data. we demonstrate in both cases the ability of this method to reliably reconstruct the landscape of a branching sde from time-courses of simulated datasets with lineage tracing, outperforming even a benchmark using the experimentally unavailable true branching rates.',\n",
       "  'categories': 'q-bio.qm math.pr',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-14',\n",
       "  'updated': '2024-03-18',\n",
       "  'authors': ['elias ventre',\n",
       "   'aden forrow',\n",
       "   'nitya gadhiwala',\n",
       "   'parijat chakraborty',\n",
       "   'omer angel',\n",
       "   'geoffrey schiebinger'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.07687'},\n",
       " {'title': 'prost: computing, storing and visualizing attributes of prokaryotic   genomes',\n",
       "  'id': '2307.08367',\n",
       "  'abstract': 'prokaryotic organisms usually possess compact genomes, which are particularly suitable to complete sequencing with existing technologies, which led to an escalating accumulation of available genome data. in response to this ever-expanding repository of information, we introduce prost, a computational system designed for the batch computation, storage, and interactive visualization of the values of attributes of prokaryotic genomes. the system allows for parallel attribute value batch computation, dynamically designed to incrementally integrate new attribute values as additional genomes become available.   prost is flexible permitting the definition of attributes by implementing attribute value computation plugins, supporting several languages (python, nim, rust and bash). this allows the system to continually evolve in accordance with changing research needs and developments. additionally, our computation and storage systems maintain comprehensive metadata, thereby enabling data provenance tracking for the computed attribute values.',\n",
       "  'categories': 'q-bio.gn',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-17',\n",
       "  'updated': '2024-03-03',\n",
       "  'authors': ['giorgio gonnella'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.08367'},\n",
       " {'title': 'the weighted total cophenetic index: a novel balance index for   phylogenetic networks',\n",
       "  'id': '2307.08654',\n",
       "  'abstract': 'phylogenetic networks play an important role in evolutionary biology as, other than phylogenetic trees, they can be used to accommodate reticulate evolutionary events such as horizontal gene transfer and hybridization. recent research has provided a lot of progress concerning the reconstruction of such networks from data as well as insight into their graph theoretical properties. however, methods and tools to quantify structural properties of networks or differences between them are still very limited. for example, for phylogenetic trees, it is common to use balance indices to draw conclusions concerning the underlying evolutionary model, and more than twenty such indices have been proposed and are used for different purposes. one of the most frequently used balance index for trees is the so-called total cophenetic index, which has several mathematically and biologically desirable properties. for networks, on the other hand, balance indices are to-date still scarce.   in this contribution, we introduce the \\\\textit{weighted} total cophenetic index as a generalization of the total cophenetic index for trees to make it applicable to general phylogenetic networks. as we shall see, this index can be determined efficiently and behaves in a mathematical sound way, i.e., it satisfies so-called locality and recursiveness conditions. in addition, we analyze its extremal properties and, in particular, we investigate its maxima and minima as well as the structure of networks that achieve these values within the space of so-called level-$1$ networks. we finally briefly compare this novel index to the two other network balance indices available so-far.',\n",
       "  'categories': 'q-bio.pe math.co',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-17',\n",
       "  'updated': '2024-03-18',\n",
       "  'authors': ['linda knüver',\n",
       "   'mareike fischer',\n",
       "   'marc hellmuth',\n",
       "   'kristina wicke'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.08654'},\n",
       " {'title': 'a formalizable proof of the no-supervenience theorem: a diagonal   limitation on the viability of physicalist theories of consciousness',\n",
       "  'id': '2307.10178',\n",
       "  'abstract': 'the no-supervenience theorem limits the capacity of physicalist theories to provide a comprehensive account of human consciousness. the proof of the theorem is difficult to formalize because it relies on both alethic and epistemic notions of possibility. this article outlines a formalizable proof using predicate modal logic in which the epistemic inferences are expressed in terms of an existing mathematical formalism, the inference device (wolpert, 2008). the resulting proof shows definitely that any physicalist theory which describes a self-aware, intelligent system must be internally inconsistent.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-06-08',\n",
       "  'updated': '2024-11-12',\n",
       "  'authors': ['cathy m reason'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.10178'},\n",
       " {'title': 'deep neural networks and brain alignment: brain encoding and decoding   (survey)',\n",
       "  'id': '2307.10246',\n",
       "  'abstract': 'can we obtain insights about the brain using ai models? how is the information in deep learning models related to brain recordings? can we improve ai models with the help of brain recordings? such questions can be tackled by studying brain recordings like functional magnetic resonance imaging (fmri). as a first step, the neuroscience community has contributed several large cognitive neuroscience datasets related to passive reading/listening/viewing of concept words, narratives, pictures, and movies. encoding and decoding models using these datasets have also been proposed in the past two decades. these models serve as additional tools for basic cognitive science and neuroscience research. encoding models aim at generating fmri brain representations given a stimulus automatically. they have several practical applications in evaluating and diagnosing neurological conditions and thus may also help design therapies for brain damage. decoding models solve the inverse problem of reconstructing the stimuli given the fmri. they are useful for designing brain-machine or brain-computer interfaces. inspired by the effectiveness of deep learning models for natural language processing, computer vision, and speech, several neural encoding and decoding models have been recently proposed. in this survey, we will first discuss popular representations of language, vision and speech stimuli, and present a summary of neuroscience datasets. further, we will review popular deep learning based encoding and decoding architectures and note their benefits and limitations. finally, we will conclude with a summary and discussion about future trends. given the large amount of recently published work in the computational cognitive neuroscience (ccn) community, we believe that this survey enables an entry point for dnn researchers to diversify into ccn research.',\n",
       "  'categories': 'q-bio.nc cs.ai cs.cl cs.cv cs.hc cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-17',\n",
       "  'updated': '2024-07-08',\n",
       "  'authors': ['subba reddy oota',\n",
       "   'zijiao chen',\n",
       "   'manish gupta',\n",
       "   'raju s. bapi',\n",
       "   'gael jobard',\n",
       "   'frederic alexandre',\n",
       "   'xavier hinaut'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.10246'},\n",
       " {'title': 'design principles of multi-map variation in biological systems',\n",
       "  'id': '2307.10251',\n",
       "  'abstract': 'complexity in biology is often described using a multi-map architecture, where the genotype, representing the encoded information, is mapped to the functional level, known as the phenotype, which is then connected to a latent phenotype we refer to as fitness. this underlying architecture governs the processes that drive evolution. moreover, natural selection, along with other neutral forces, can modify these maps. at each hierarchical level, variation is observed. here, i propose the need to establish principles that can aid in understanding the transformation of variation within this multi-map architecture. specifically, i will introduce three, related to the presence of modulators, constraints, and the modular channeling of variation. by comprehending these design principles in various biological systems, we can gain better insights into the mechanisms underlying these maps and their evolutionary dynamics.',\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-17',\n",
       "  'updated': '2024-02-04',\n",
       "  'authors': ['juan f poyatos'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.10251'},\n",
       " {'title': 'computational modelling of angiogenesis: the importance of cell   rearrangements during vascular growth',\n",
       "  'id': '2307.10297',\n",
       "  'abstract': \"angiogenesis is the process wherein endothelial cells (ecs) form sprouts that elongate from the pre-existing vasculature to create new vascular networks. in addition to its essential role in normal development, angiogenesis plays a vital role in pathologies such as cancer, diabetes and atherosclerosis. mathematical and computational modelling has contributed to unravelling its complexity. many existing theoretical models of angiogenic sprouting are based on the 'snail-trail' hypothesis. this framework assumes that leading ecs positioned at sprout tips migrate towards low-oxygen regions while other ecs in the sprout passively follow the leaders' trails and proliferate to maintain sprout integrity. however, experimental results indicate that, contrary to the snail-trail assumption, ecs exchange positions within developing vessels, and the elongation of sprouts is primarily driven by directed migration of ecs. the functional role of cell rearrangements remains unclear. this review of the theoretical modelling of angiogenesis is the first to focus on the phenomenon of cell mixing during early sprouting. we start by describing the biological processes that occur during early angiogenesis, such as phenotype specification, cell rearrangements and cell interactions with the microenvironment. next, we provide an overview of various theoretical approaches that have been employed to model angiogenesis, with particular emphasis on recent in silico models that account for the phenomenon of cell mixing. finally, we discuss when cell mixing should be incorporated into theoretical models and what essential modelling components such models should include in order to investigate its functional role.\",\n",
       "  'categories': 'q-bio.to q-bio.cb',\n",
       "  'doi': '10.1002/wsbm.1634',\n",
       "  'created': '2023-07-18',\n",
       "  'updated': '2024-01-15',\n",
       "  'authors': ['daria stepanova',\n",
       "   'helen m. byrne',\n",
       "   'philip k. maini',\n",
       "   'tomás alarcón'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.10297'},\n",
       " {'title': 'uncertainty quantification for molecular property predictions with graph   neural architecture search',\n",
       "  'id': '2307.10438',\n",
       "  'abstract': 'graph neural networks (gnns) have emerged as a prominent class of data-driven methods for molecular property prediction. however, a key limitation of typical gnn models is their inability to quantify uncertainties in the predictions. this capability is crucial for ensuring the trustworthy use and deployment of models in downstream tasks. to that end, we introduce autognnuq, an automated uncertainty quantification (uq) approach for molecular property prediction. autognnuq leverages architecture search to generate an ensemble of high-performing gnns, enabling the estimation of predictive uncertainties. our approach employs variance decomposition to separate data (aleatoric) and model (epistemic) uncertainties, providing valuable insights for reducing them. in our computational experiments, we demonstrate that autognnuq outperforms existing uq methods in terms of both prediction accuracy and uq performance on multiple benchmark datasets. additionally, we utilize t-sne visualization to explore correlations between molecular features and uncertainty, offering insight for dataset improvement. autognnuq has broad applicability in domains such as drug discovery and materials science, where accurate uncertainty quantification is crucial for decision-making.',\n",
       "  'categories': 'cs.lg physics.chem-ph q-bio.bm',\n",
       "  'doi': '10.1039/d4dd00088a',\n",
       "  'created': '2023-07-19',\n",
       "  'updated': '2024-06-28',\n",
       "  'authors': ['shengli jiang',\n",
       "   'shiyi qin',\n",
       "   'reid c. van lehn',\n",
       "   'prasanna balaprakash',\n",
       "   'victor m. zavala'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.10438'},\n",
       " {'title': 'fractional denoising for 3d molecular pre-training',\n",
       "  'id': '2307.10683',\n",
       "  'abstract': 'coordinate denoising is a promising 3d molecular pre-training method, which has achieved remarkable performance in various downstream drug discovery tasks. theoretically, the objective is equivalent to learning the force field, which is revealed helpful for downstream tasks. nevertheless, there are two challenges for coordinate denoising to learn an effective force field, i.e. low coverage samples and isotropic force field. the underlying reason is that molecular distributions assumed by existing denoising methods fail to capture the anisotropic characteristic of molecules. to tackle these challenges, we propose a novel hybrid noise strategy, including noises on both dihedral angel and coordinate. however, denoising such hybrid noise in a traditional way is no more equivalent to learning the force field. through theoretical deductions, we find that the problem is caused by the dependency of the input conformation for covariance. to this end, we propose to decouple the two types of noise and design a novel fractional denoising method (frad), which only denoises the latter coordinate part. in this way, frad enjoys both the merits of sampling more low-energy structures and the force field equivalence. extensive experiments show the effectiveness of frad in molecular representation, with a new state-of-the-art on 9 out of 12 tasks of qm9 and on 7 out of 8 targets of md17.',\n",
       "  'categories': 'q-bio.qm cs.lg physics.chem-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-20',\n",
       "  'updated': '2024-02-26',\n",
       "  'authors': ['shikun feng',\n",
       "   'yuyan ni',\n",
       "   'yanyan lan',\n",
       "   'zhi-ming ma',\n",
       "   'wei-ying ma'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.10683'},\n",
       " {'title': 'stochastic trade-offs and the emergence of diversification in e. coli   evolution experiments',\n",
       "  'id': '2307.11033',\n",
       "  'abstract': \"laboratory experiments with bacterial colonies, under well-controlled conditions often lead to evolutionary diversification, where at least two ecotypes emerge from an initially monomorphic population. empirical evidence suggests that such ''evolutionary branching'' occurs stochastically, even under fixed and stable conditions. this stochastic nature is characterized by: (i) occurrence in a significant fraction, but not all, of experimental settings, (ii) emergence at widely varying times, and (iii) variable relative abundances of the resulting subpopulations across experiments. theoretical approaches to understanding evolutionary branching under these conditions have been previously developed within the (deterministic) framework of ''adaptive dynamics''. here, we advance the understanding of the stochastic nature of evolutionary outcomes by introducing the concept of ''stochastic trade-offs'' as opposed to ''hard'' ones. the key idea is that the stochasticity of mutations occurs in a high-dimensional trait space and this translates into variability that is constrained to a flexible tradeoff curve. by incorporating this additional source of stochasticity, we are able to account for the observed empirical variability and make predictions regarding the likelihood of evolutionary branching under different conditions. this approach effectively bridges the gap between theoretical predictions and experimental observations, providing insights into when and how evolutionary branching is more likely to occur in laboratory experiments.\",\n",
       "  'categories': 'q-bio.pe physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-20',\n",
       "  'updated': '2024-11-05',\n",
       "  'authors': ['roberto corral lópez',\n",
       "   'samir suweis',\n",
       "   'sandro azaele',\n",
       "   'miguel a. muñoz'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.11033'},\n",
       " {'title': 'the nature of intelligence',\n",
       "  'id': '2307.11114',\n",
       "  'abstract': 'the human brain is the substrate for human intelligence. by simulating the human brain, artificial intelligence builds computational models that have learning capabilities and perform intelligent tasks approaching the human level. deep neural networks consist of multiple computation layers to learn representations of data and improve the state-of-the-art in many recognition domains. however, the essence of intelligence commonly represented by both humans and ai is unknown. here, we show that the nature of intelligence is a series of mathematically functional processes that minimize system entropy by establishing functional relationships between datasets over the space and time. humans and ai have achieved intelligence by implementing these entropy-reducing processes in a reinforced manner that consumes energy. with this hypothesis, we establish mathematical models of language, unconsciousness and consciousness, predicting the evidence to be found by neuroscience and achieved by ai engineering. furthermore, a conclusion is made that the total entropy of the universe is conservative, and the intelligence counters the spontaneous processes to decrease entropy by physically or informationally connecting datasets that originally exist in the universe but are separated across the space and time. this essay should be a starting point for a deeper understanding of the universe and us as human beings and for achieving sophisticated ai models that are tantamount to human intelligence or even superior. furthermore, this essay argues that more advanced intelligence than humans should exist if only it reduces entropy in a more efficient energy-consuming way.',\n",
       "  'categories': 'q-bio.nc cs.ai',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-20',\n",
       "  'updated': '2024-02-19',\n",
       "  'authors': ['barco jie you'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.11114'},\n",
       " {'title': 'contrastive graph pooling for explainable classification of brain   networks',\n",
       "  'id': '2307.11133',\n",
       "  'abstract': \"functional magnetic resonance imaging (fmri) is a commonly used technique to measure neural activation. its application has been particularly important in identifying underlying neurodegenerative conditions such as parkinson's, alzheimer's, and autism. recent analysis of fmri data models the brain as a graph and extracts features by graph neural networks (gnns). however, the unique characteristics of fmri data require a special design of gnn. tailoring gnn to generate effective and domain-explainable features remains challenging. in this paper, we propose a contrastive dual-attention block and a differentiable graph pooling method called contrastpool to better utilize gnn for brain networks, meeting fmri-specific requirements. we apply our method to 5 resting-state fmri brain network datasets of 3 diseases and demonstrate its superiority over state-of-the-art baselines. our case study confirms that the patterns extracted by our method match the domain knowledge in neuroscience literature, and disclose direct and interesting insights. our contributions underscore the potential of contrastpool for advancing the understanding of brain networks and neurodegenerative conditions. the source code is available at https://github.com/angusmonroe/contrastpool.\",\n",
       "  'categories': 'q-bio.nc cs.ai cs.lg',\n",
       "  'doi': '10.1109/tmi.2024.3392988',\n",
       "  'created': '2023-07-07',\n",
       "  'updated': '2024-09-06',\n",
       "  'authors': ['jiaxing xu',\n",
       "   'qingtian bian',\n",
       "   'xinhang li',\n",
       "   'aihu zhang',\n",
       "   'yiping ke',\n",
       "   'miao qiao',\n",
       "   'wei zhang',\n",
       "   'wei khang jeremy sim',\n",
       "   'balázs gulyás'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.11133'},\n",
       " {'title': 'unlocking ensemble ecosystem modelling for large and complex networks',\n",
       "  'id': '2307.11365',\n",
       "  'abstract': 'the potential effects of conservation actions on threatened species can be predicted using ensemble ecosystem models by forecasting populations with and without intervention. these model ensembles commonly assume stable coexistence of species in the absence of available data. however, existing ensemble-generation methods become computationally inefficient as the size of the ecosystem network increases, preventing larger networks from being studied. we present a novel sequential monte carlo sampling approach for ensemble generation that is orders of magnitude faster than existing approaches. we demonstrate that the methods produce equivalent parameter inferences, model predictions, and tightly constrained parameter combinations using a novel sensitivity analysis method. for one case study, we demonstrate a speed-up from 108 days to 6 hours, while maintaining equivalent ensembles. additionally, we demonstrate how to identify the parameter combinations that strongly drive feasibility and stability, drawing ecological insight from the ensembles. now, for the first time, larger and more realistic networks can be practically simulated and analysed.',\n",
       "  'categories': 'q-bio.pe stat.ap',\n",
       "  'doi': '10.1371/journal.pcbi.1011976',\n",
       "  'created': '2023-07-21',\n",
       "  'updated': '2024-01-24',\n",
       "  'authors': ['sarah a. vollert', 'christopher drovandi', 'matthew p. adams'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.11365'},\n",
       " {'title': 'a theoretical framework for learning through structural plasticity',\n",
       "  'id': '2307.11735',\n",
       "  'abstract': 'a growing body of research indicates that structural plasticity mechanisms are crucial for learning and memory consolidation. starting from a simple phenomenological model, we exploit a mean-field approach to develop a theoretical framework of learning through this kind of plasticity, capable of taking into account several features of the connectivity and pattern of activity of biological neural networks, including probability distributions of neuron firing rates, selectivity of the responses of single neurons to multiple stimuli, probabilistic connection rules and noisy stimuli. more importantly, it describes the effects of stabilization, pruning and reorganization of synaptic connections. this framework is used to compute the values of some relevant quantities used to characterize the learning and memory capabilities of the neuronal network in training and testing procedures as the number of training patterns and other model parameters vary. the results are then compared with those obtained through simulations with firing-rate-based neuronal network models.',\n",
       "  'categories': 'q-bio.nc physics.bio-ph physics.comp-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-21',\n",
       "  'updated': '2024-06-18',\n",
       "  'authors': ['gianmarco tiddia', 'luca sergi', 'bruno golosio'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.11735'},\n",
       " {'title': 'social and individual learning in the minority game',\n",
       "  'id': '2307.11846',\n",
       "  'abstract': 'we study the roles of social and individual learning on outcomes of the minority game model of a financial market. social learning occurs via agents adopting the strategies of their neighbours within a social network, while individual learning results in agents changing their strategies without input from other agents. in particular, we show how social learning can undermine efficiency of the market due to negative frequency dependent selection and loss of strategy diversity. the latter of which can lock the population into a maximally inefficient state. we show how individual learning can rescue a population engaged in social learning from such inefficiencies.',\n",
       "  'categories': 'physics.soc-ph q-bio.pe q-fin.st',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-21',\n",
       "  'updated': '2024-03-03',\n",
       "  'authors': ['bryce morsky', 'fuwei zhuang', 'zuojun zhou'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.11846'},\n",
       " {'title': 'kinetic description of swarming dynamics with topological interaction   and transient leaders',\n",
       "  'id': '2307.12044',\n",
       "  'abstract': \"in this paper, we present a model describing the collective motion of birds. the model introduces spontaneous changes in direction which are initialized by few agents, here referred as leaders, whose influence act on their nearest neighbors, in the following referred as followers. starting at the microscopic level, we develop a kinetic model that characterizes the behaviour of large flocks with transient leadership. one significant challenge lies in managing topological interactions, as identifying nearest neighbors in extensive systems can be computationally expensive. to address this, we propose a novel stochastic particle method to simulate the mesoscopic dynamics and reduce the computational cost of identifying closer agents from quadratic to logarithmic complexity using a $k$-nearest neighbours search algorithm with a binary tree. lastly, we conduct various numerical experiments for different scenarios to validate the algorithm's effectiveness and investigate collective dynamics in both two and three dimensions.\",\n",
       "  'categories': 'math.na cs.na q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-22',\n",
       "  'updated': '2024-06-19',\n",
       "  'authors': ['giacomo albi', 'federica ferrarese'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.12044'},\n",
       " {'title': 'an individual-based model to explore the impact of psychological stress   on immune infiltration into tumour spheroids',\n",
       "  'id': '2307.12627',\n",
       "  'abstract': 'in recent in vitro experiments on co-culture between breast tumour spheroids and activated immune cells, it was observed that the introduction of the stress hormone cortisol resulted in a decreased immune cell infiltration into the spheroids. moreover, the presence of cortisol deregulated the normal levels of the pro- and anti-inflammatory cytokines ifn-{\\\\gamma} and il-10. we present an individual-based model to explore the interaction dynamics between tumour and immune cells under psychological stress conditions. with our model, we explore the processes underlying the emergence of different levels of immune infiltration, with particular focus on the biological mechanisms regulated by ifn-{\\\\gamma} and il-10. the set-up of numerical simulations is defined to mimic the scenarios considered in the experimental study. similarly to the experimental quantitative analysis, we compute a score that quantifies the level of immune cell infiltration into the tumour. the results of numerical simulations indicate that the motility of immune cells, their capability to infiltrate through tumour cells, their growth rate and the interplay between these cell parameters can affect the level of immune cell infiltration in different ways. ultimately, numerical simulations of this model support a deeper understanding of the impact of biological stress-induced mechanisms on immune infiltration.',\n",
       "  'categories': 'q-bio.cb',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-24',\n",
       "  'updated': '2024-03-08',\n",
       "  'authors': ['emma leschiera',\n",
       "   'gheed al-hity',\n",
       "   'melanie s. flint',\n",
       "   'chandrasekhar venkataraman',\n",
       "   'tommaso lorenzi',\n",
       "   'luis almeida',\n",
       "   'chloe audebert'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.12627'},\n",
       " {'title': 'pro-prime: a general temperature-guided language model to engineer   enhanced stability and activity in proteins',\n",
       "  'id': '2307.12682',\n",
       "  'abstract': \"designing protein mutants of both high stability and activity is a critical yet challenging task in protein engineering. here, we introduce prime, a deep learning model, which can suggest protein mutants of improved stability and activity without any prior experimental mutagenesis data of the specified protein. leveraging temperature-aware language modeling, prime demonstrated superior predictive power compared to current state-of-the-art models on the public mutagenesis dataset over 283 protein assays. furthermore, we validated prime's predictions on five proteins, examining the top 30-45 single-site mutations' impact on various protein properties, including thermal stability, antigen-antibody binding affinity, and the ability to polymerize non-natural nucleic acid or resilience to extreme alkaline conditions. remarkably, over 30% of the ai-recommended mutants exhibited superior performance compared to their pre-mutation counterparts across all proteins and desired properties. moreover, we have developed an efficient, and successful method based on prime to rapidly obtain multi-site mutants with enhanced activity and stability. hence, prime demonstrates the general applicability in protein engineering.\",\n",
       "  'categories': 'q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-24',\n",
       "  'updated': '2024-10-27',\n",
       "  'authors': ['fan jiang',\n",
       "   'mingchen li',\n",
       "   'jiajun dong',\n",
       "   'yuanxi yu',\n",
       "   'xinyu sun',\n",
       "   'banghao wu',\n",
       "   'jin huang',\n",
       "   'liqi kang',\n",
       "   'yufeng pei',\n",
       "   'liang zhang',\n",
       "   'shaojie wang',\n",
       "   'wenxue xu',\n",
       "   'jingyao xin',\n",
       "   'wanli ouyang',\n",
       "   'guisheng fan',\n",
       "   'lirong zheng',\n",
       "   'yang tan',\n",
       "   'zhiqiang hu',\n",
       "   'yi xiong',\n",
       "   'yan feng',\n",
       "   'guangyu yang',\n",
       "   'qian liu',\n",
       "   'jie song',\n",
       "   'jia liu',\n",
       "   'liang hong',\n",
       "   'pan tan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.12682'},\n",
       " {'title': 'ctage: curvature-based topology-aware graph embedding for learning   molecular representations',\n",
       "  'id': '2307.13275',\n",
       "  'abstract': 'ai-driven drug design relies significantly on predicting molecular properties, which is a complex task. in current approaches, the most commonly used feature representations for training deep neural network models are based on smiles and molecular graphs. while these methods are concise and efficient, they have limitations in capturing complex spatial information. recently, researchers have recognized the importance of incorporating three-dimensional information of molecular structures into models. however, capturing spatial information requires the introduction of additional units in the generator, bringing additional design and computational costs. therefore, it is necessary to develop a method for predicting molecular properties that effectively combines spatial structural information while maintaining the simplicity and efficiency of graph neural networks. in this work, we propose an embedding approach ctage, utilizing $k$-hop discrete ricci curvature to extract structural insights from molecular graph data. this effectively integrates spatial structural information while preserving the training complexity of the network. experimental results indicate that introducing node curvature significantly improves the performance of current graph neural network frameworks, validating that the information from k-hop node curvature effectively reflects the relationship between molecular structure and function.',\n",
       "  'categories': 'cs.lg cs.ai q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-25',\n",
       "  'updated': '2024-01-18',\n",
       "  'authors': ['yili chen', 'zhengyu li', 'zheng wan', 'hui yu', 'xian wei'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.13275'},\n",
       " {'title': 'topologically regularized multiple instance learning to harness data   scarcity',\n",
       "  'id': '2307.14025',\n",
       "  'abstract': \"in biomedical data analysis, multiple instance learning (mil) models have emerged as a powerful tool to classify patients' microscopy samples. however, the data-intensive requirement of these models poses a significant challenge in scenarios with scarce data availability, e.g., in rare diseases. we introduce a topological regularization term to mil to mitigate this challenge. it provides a shape-preserving inductive bias that compels the encoder to maintain the essential geometrical-topological structure of input bags during projection into latent space. this enhances the performance and generalization of the mil classifier regardless of the aggregation function, particularly for scarce training data. the effectiveness of our method is confirmed through experiments across a range of datasets, showing an average enhancement of 2.8% for mil benchmarks, 15.3% for synthetic mil datasets, and 5.5% for real-world biomedical datasets over the current state-of-the-art.\",\n",
       "  'categories': 'cs.lg cs.cv eess.iv q-bio.qm stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-26',\n",
       "  'updated': '2024-03-11',\n",
       "  'authors': ['salome kazeminia', 'carsten marr', 'bastian rieck'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.14025'},\n",
       " {'title': 'a hybrid machine learning model for classifying gene mutations in cancer   using lstm, bilstm, cnn, gru, and glove',\n",
       "  'id': '2307.14361',\n",
       "  'abstract': \"in our study, we introduce a novel hybrid ensemble model that synergistically combines lstm, bilstm, cnn, gru, and glove embeddings for the classification of gene mutations in cancer. this model was rigorously tested using kaggle's personalized medicine: redefining cancer treatment dataset, demonstrating exceptional performance across all evaluation metrics. notably, our approach achieved a training accuracy of 80.6%, precision of 81.6%, recall of 80.6%, and an f1 score of 83.1%, alongside a significantly reduced mean squared error (mse) of 2.596. these results surpass those of advanced transformer models and their ensembles, showcasing our model's superior capability in handling the complexities of gene mutation classification. the accuracy and efficiency of gene mutation classification are paramount in the era of precision medicine, where tailored treatment plans based on individual genetic profiles can dramatically improve patient outcomes and save lives. our model's remarkable performance highlights its potential in enhancing the precision of cancer diagnoses and treatments, thereby contributing significantly to the advancement of personalized healthcare.\",\n",
       "  'categories': 'q-bio.qm cs.ai cs.cl cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-24',\n",
       "  'updated': '2024-05-20',\n",
       "  'authors': ['sanad aburass', 'osama dorgham', 'jamil al shaqsi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.14361'},\n",
       " {'title': \"prot2text: multimodal protein's function generation with gnns and   transformers\",\n",
       "  'id': '2307.14367',\n",
       "  'abstract': \"in recent years, significant progress has been made in the field of protein function prediction with the development of various machine-learning approaches. however, most existing methods formulate the task as a multi-classification problem, i.e. assigning predefined labels to proteins. in this work, we propose a novel approach, prot2text, which predicts a protein's function in a free text style, moving beyond the conventional binary or categorical classifications. by combining graph neural networks(gnns) and large language models(llms), in an encoder-decoder framework, our model effectively integrates diverse data types including protein sequence, structure, and textual annotation and description. this multimodal approach allows for a holistic representation of proteins' functions, enabling the generation of detailed and accurate functional descriptions. to evaluate our model, we extracted a multimodal protein dataset from swissprot, and demonstrate empirically the effectiveness of prot2text. these results highlight the transformative impact of multimodal models, specifically the fusion of gnns and llms, empowering researchers with powerful tools for more accurate function prediction of existing as well as first-to-see proteins.\",\n",
       "  'categories': 'q-bio.qm cs.cl cs.lg',\n",
       "  'doi': '10.1609/aaai.v38i10.28948',\n",
       "  'created': '2023-07-25',\n",
       "  'updated': '2024-04-20',\n",
       "  'authors': ['hadi abdine',\n",
       "   'michail chatzianastasis',\n",
       "   'costas bouyioukos',\n",
       "   'michalis vazirgiannis'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.14367'},\n",
       " {'title': 'collective behavior from surprise minimization',\n",
       "  'id': '2307.14804',\n",
       "  'abstract': \"collective motion is ubiquitous in nature; groups of animals, such as fish, birds, and ungulates appear to move as a whole, exhibiting a rich behavioral repertoire that ranges from directed movement to milling to disordered swarming. typically, such macroscopic patterns arise from decentralized, local interactions among constituent components (e.g., individual fish in a school). preeminent models of this process describe individuals as self-propelled particles, subject to self-generated motion and 'social forces' such as short-range repulsion and long-range attraction or alignment. however, organisms are not particles; they are probabilistic decision-makers. here, we introduce an approach to modelling collective behavior based on active inference. this cognitive framework casts behavior as the consequence of a single imperative: to minimize surprise. we demonstrate that many empirically-observed collective phenomena, including cohesion, milling and directed motion, emerge naturally when considering behavior as driven by active bayesian inference -- without explicitly building behavioral rules or goals into individual agents. furthermore, we show that active inference can recover and generalize the classical notion of social forces as agents attempt to suppress prediction errors that conflict with their expectations. by exploring the parameter space of the belief-based model, we reveal non-trivial relationships between the individual beliefs and group properties like polarization and the tendency to visit different collective states. we also explore how individual beliefs about uncertainty determine collective decision-making accuracy. finally, we show how agents can update their generative model over time, resulting in groups that are collectively more sensitive to external fluctuations and encode information more robustly.\",\n",
       "  'categories': 'nlin.ao cs.ma q-bio.nc',\n",
       "  'doi': '10.1073/pnas.2320239121',\n",
       "  'created': '2023-07-27',\n",
       "  'updated': '2024-05-14',\n",
       "  'authors': ['conor heins',\n",
       "   'beren millidge',\n",
       "   'lancelot da costa',\n",
       "   'richard mann',\n",
       "   'karl friston',\n",
       "   'iain couzin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.14804'},\n",
       " {'title': 'distinctive pupil and microsaccade-rate signatures in self-recognition',\n",
       "  'id': '2307.15239',\n",
       "  'abstract': \"visual fixation is an active process with pupil dynamics as well as fixational eye movements and microsaccades that support perception. measures of both pupil contraction and microsaccades are known to be sensitive to ongoing cognition and emotional processing. here we present experimental results from a visual fixation task demonstrating that pupil size and microsaccade rate respond differently during self-recognition (when seeing one's own face) than when seeing familiar or unfamiliar faces. first, the pupil response is characterized by an immediate pupil-constriction followed by later dilation in response to stimulus onsets. for one's own face, we observe muted constriction and greater dilation compared to other faces. second, microsaccades, which generally show an inhibitory response to incoming stimuli, are more strongly inhibited in response to one's own face compared to other faces. our results lend support to the idea that eye-related physiological measures could be useful for authentication or exposing hidden knowledge.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-27',\n",
       "  'updated': '2024-03-06',\n",
       "  'authors': ['lisa schwetlick',\n",
       "   'hendrik graupner',\n",
       "   'olaf dimigen',\n",
       "   'ralf engbert'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.15239'},\n",
       " {'title': 'quantifying the influence of climate on human mind and culture: evidence   from visual art',\n",
       "  'id': '2307.15540',\n",
       "  'abstract': \"while connections between climate and the human mind and culture are widely acknowledged, they are not thoroughly quantified. analyzing 100,000 paintings and data on 2,000 artists from the 13th to 21st centuries, the study reveals that the lightness of the paintings exhibited an interesting u-shaped pattern mirroring global temperature trends. there is a significant association between the two, even after controlling for various factors. event study analysis using the artist-level data further reveals that high-temperature shocks resulted in brighter paintings in later periods for artists who experienced them compared to the control group. the effect is particularly pronounced in art genres that rely on artists' imaginations, indicating a notable influence on artists' minds. these findings underscore the enduring impact of climate on the human mind and culture throughout history and highlight art as a valuable tool for understanding people's minds and cultures.\",\n",
       "  'categories': 'q-bio.pe econ.gn q-fin.ec',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-28',\n",
       "  'updated': '2024-04-11',\n",
       "  'authors': ['shuhei kitamura'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.15540'},\n",
       " {'title': 'bayesian time-series classifier for decoding simple visual stimuli from   intracranial neural activity',\n",
       "  'id': '2307.15672',\n",
       "  'abstract': \"understanding how external stimuli are encoded in distributed neural activity is of significant interest in clinical and basic neuroscience. to address this need, it is essential to develop analytical tools capable of handling limited data and the intrinsic stochasticity present in neural data. in this study, we propose a straightforward bayesian time series classifier (btsc) model that tackles these challenges whilst maintaining a high level of interpretability. we demonstrate the classification capabilities of this approach by utilizing neural data to decode colors in a visual task. the model exhibits consistent and reliable average performance of 75.55% on 4 patients' dataset, improving upon state-of-the-art machine learning techniques by about 3.0 percent. in addition to its high classification accuracy, the proposed btsc model provides interpretable results, making the technique a valuable tool to study neural activity in various tasks and categories. the proposed solution can be applied to neural data recorded in various tasks, where there is a need for interpretable results and accurate classification accuracy.\",\n",
       "  'categories': 'cs.lg eess.sp q-bio.nc',\n",
       "  'doi': '10.1007/978-3-031-43075-6_20',\n",
       "  'created': '2023-07-28',\n",
       "  'updated': '',\n",
       "  'authors': ['navid ziaei',\n",
       "   'reza saadatifard',\n",
       "   'ali yousefi',\n",
       "   'behzad nazari',\n",
       "   'sydney s. cash',\n",
       "   'angelique c. paulk'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.15672'},\n",
       " {'title': 'parameter identifiability in pde models of fluorescence recovery after   photobleaching',\n",
       "  'id': '2307.15857',\n",
       "  'abstract': 'identifying unique parameters for mathematical models describing biological data can be challenging and often impossible. parameter identifiability for partial differential equations models in cell biology is especially difficult given that many established \\\\textit{in vivo} measurements of protein dynamics average out the spatial dimensions. here, we are motivated by recent experiments on the binding dynamics of the rna-binding protein ptbp3 in rnp granules of frog oocytes based on fluorescence recovery after photobleaching (frap) measurements. frap is a widely-used experimental technique for probing protein dynamics in living cells, and is often modeled using simple reaction-diffusion models of the protein dynamics. we show that current methods of structural and practical parameter identifiability provide limited insights into identifiability of kinetic parameters for these pde models and spatially-averaged frap data. we thus propose a pipeline for assessing parameter identifiability and for learning parameter combinations based on re-parametrization and profile likelihoods analysis. we show that this method is able to recover parameter combinations for synthetic frap datasets and investigate its application to real experimental data.',\n",
       "  'categories': 'q-bio.qm math.ds',\n",
       "  'doi': '10.1007/s11538-024-01266-4',\n",
       "  'created': '2023-07-28',\n",
       "  'updated': '2024-03-02',\n",
       "  'authors': ['maria-veronica ciocanel',\n",
       "   'lee ding',\n",
       "   'lucas mastromatteo',\n",
       "   'sarah reichheld',\n",
       "   'sarah cabral',\n",
       "   'kimberly mowry',\n",
       "   'bjorn sandstede'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.15857'},\n",
       " {'title': 'infection-induced cascading failures -- impact and mitigation',\n",
       "  'id': '2307.16767',\n",
       "  'abstract': 'in the context of epidemic spreading, many intricate dynamical patterns can emerge due to the cooperation of different types of pathogens or the interaction between the disease spread and other failure propagation mechanism. to unravel such patterns, simulation frameworks are usually adopted, but they are computationally demanding on big networks and subject to large statistical uncertainty. here, we study the two-layer spreading processes on unidirectionally dependent networks, where the spreading infection of diseases or malware in one layer can trigger cascading failures in another layer and lead to secondary disasters, e.g., disrupting public services, supply chains, or power distribution. we utilize a dynamic message-passing method to devise efficient algorithms for inferring the system states, which allows one to investigate systematically the nature of complex intertwined spreading processes and evaluate their impact. based on such dynamic message-passing framework and optimal control, we further develop an effective optimization algorithm for mitigating network failures.',\n",
       "  'categories': 'physics.soc-ph cond-mat.dis-nn math.oc q-bio.pe',\n",
       "  'doi': '10.1038/s42005-024-01638-1',\n",
       "  'created': '2023-07-31',\n",
       "  'updated': '2024-05-05',\n",
       "  'authors': ['bo li', 'david saad'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.16767'},\n",
       " {'title': 'on the use of associative memory in hopfield networks designed to solve   propositional satisfiability problems',\n",
       "  'id': '2307.16807',\n",
       "  'abstract': 'hopfield networks are an attractive choice for solving many types of computational problems because they provide a biologically plausible mechanism. the self-optimization (so) model adds to the hopfield network by using a biologically founded hebbian learning rule, in combination with repeated network resets to arbitrary initial states, for optimizing its own behavior towards some desirable goal state encoded in the network. in order to better understand that process, we demonstrate first that the so model can solve concrete combinatorial problems in sat form, using two examples of the liars problem and the map coloring problem. in addition, we show how under some conditions critical information might get lost forever with the learned network producing seemingly optimal solutions that are in fact inappropriate for the problem it was tasked to solve. what appears to be an undesirable side-effect of the so model, can provide insight into its process for solving intractable problems.',\n",
       "  'categories': 'nlin.ao cs.ai q-bio.nc',\n",
       "  'doi': '10.1109/ssci52147.2023.10371918',\n",
       "  'created': '2023-07-31',\n",
       "  'updated': '2024-03-04',\n",
       "  'authors': ['natalya weber', 'werner koch', 'ozan erdem', 'tom froese'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2307.16807'},\n",
       " {'title': 'effect of synaptic heterogeneity on neuronal coordination',\n",
       "  'id': '2308.00421',\n",
       "  'abstract': 'recent advancements in measurement techniques have resulted in an increasing amount of data on neural activities recorded in parallel, revealing largely heterogeneous correlation patterns across neurons. yet, the mechanistic origin of this heterogeneity is largely unknown because existing theoretical approaches linking structure and dynamics in neural circuits are restricted to population-averaged connectivity and activity. here we present a systematic inclusion of heterogeneity in network connectivity to derive quantitative predictions for neuron-resolved covariances and their statistics in spiking neural networks. our study shows that the heterogeneity in covariances is not a result of variability in single-neuron firing statistics but stems from the ubiquitously observed sparsity and variability of connections in brain networks. linear-response theory maps these features to the effective connectivity between neurons, which in turn determines neuronal covariances. beyond-mean-field tools reveal that synaptic heterogeneity modulates the variability of covariances and thus the complexity of neuronal coordination across many orders of magnitude.',\n",
       "  'categories': 'cond-mat.dis-nn q-bio.nc',\n",
       "  'doi': '10.1103/prxlife.2.013013',\n",
       "  'created': '2023-08-01',\n",
       "  'updated': '2024-04-04',\n",
       "  'authors': ['moritz layer', 'moritz helias', 'david dahmen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.00421'},\n",
       " {'title': 'pdbimages: a command line tool for automated macromolecular structure   visualization',\n",
       "  'id': '2308.00563',\n",
       "  'abstract': 'summary: pdbimages is an innovative, open-source node.js package that harnesses the power of the popular macromolecule structure visualization software mol*. designed for use by the scientific community, pdbimages provides a means to generate high-quality images for pdb and alphafold db models. its unique ability to render and save images directly to files in a browserless mode sets it apart, offering users a streamlined, automated process for macromolecular structure visualization. here, we detail the implementation of pdbimages, enumerating its diverse image types and elaborating on its user-friendly setup. this powerful tool opens a new gateway for researchers to visualize, analyse, and share their work, fostering a deeper understanding of bioinformatics. availability and implementation: pdbimages is available as an npm package from https://www.npmjs.com/package/pdb-images. the source code is available from https://github.com/pdbeurope/pdb-images. contact: mvaradi@ebi.ac.uk, sameer@ebi.ac.uk',\n",
       "  'categories': 'q-bio.bm',\n",
       "  'doi': '10.1093/bioinformatics/btad744',\n",
       "  'created': '2023-08-01',\n",
       "  'updated': '',\n",
       "  'authors': ['adam midlik',\n",
       "   'sreenath nair',\n",
       "   'stephen anyango',\n",
       "   'mandar deshpande',\n",
       "   'david sehnal',\n",
       "   'mihaly varadi',\n",
       "   'sameer velankar'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.00563'},\n",
       " {'title': 'the contact process with an asymptomatic state',\n",
       "  'id': '2308.00731',\n",
       "  'abstract': 'in order to understand the cost of a potentially high infectiousness of symptomatic individuals or, on the contrary, the benefit of social distancing, quarantine, etc. in the course of an infectious disease, this paper considers a natural variant of the popular contact process that distinguishes between asymptomatic and symptomatic individuals. infected individuals all recover at rate one but infect nearby individuals at a rate that depends on whether they show the symptoms of the disease or not. newly infected individuals are always asymptomatic and may or may not show the symptoms before they recover. the analysis of the corresponding mean-field model reveals that, in the absence of local interactions, regardless of the rate at which asymptomatic individuals become symptomatic, there is an epidemic whenever at least one of the infection rates is sufficiently large. in contrast, our analysis of the interacting particle system shows that, when the rate at which asymptomatic individuals become symptomatic is small and the asymptomatic individuals are not infectious, there cannot be an epidemic even when the symptomatic individuals are highly infectious.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-01',\n",
       "  'updated': '2024-04-26',\n",
       "  'authors': ['lamia belhadji', 'nicolas lanchier', 'max mercer'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.00731'},\n",
       " {'title': 'evaluation of network-guided random forest for disease gene discovery',\n",
       "  'id': '2308.01323',\n",
       "  'abstract': 'gene network information is believed to be beneficial for disease module and pathway identification, but has not been explicitly utilized in the standard random forest (rf) algorithm for gene expression data analysis. we investigate the performance of a network-guided rf where the network information is summarized into a sampling probability of predictor variables which is further used in the construction of the rf. our results suggest that network-guided rf does not provide better disease prediction than the standard rf. in terms of disease gene discovery, if disease genes form module(s), network-guided rf identifies them more accurately. in addition, when disease status is independent from genes in the given network, spurious gene selection results can occur when using network information, especially on hub genes. our empirical analysis on two balanced microarray and rna-seq breast cancer datasets from the cancer genome atlas (tcga) for classification of progesterone receptor (pr) status also demonstrates that network-guided rf can identify genes from pgr-related pathways, which leads to a better connected module of identified genes.',\n",
       "  'categories': 'q-bio.mn cs.lg',\n",
       "  'doi': '10.1186/s13040-024-00361-5',\n",
       "  'created': '2023-08-02',\n",
       "  'updated': '',\n",
       "  'authors': ['jianchang hu', 'silke szymczak'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.01323'},\n",
       " {'title': 'a vision transformer-based framework for knowledge transfer from   multi-modal to mono-modal lymphoma subtyping models',\n",
       "  'id': '2308.01328',\n",
       "  'abstract': 'determining lymphoma subtypes is a crucial step for better patient treatment targeting to potentially increase their survival chances. in this context, the existing gold standard diagnosis method, which relies on gene expression technology, is highly expensive and time-consuming, making it less accessibility. although alternative diagnosis methods based on ihc (immunohistochemistry) technologies exist (recommended by the who), they still suffer from similar limitations and are less accurate. whole slide image (wsi) analysis using deep learning models has shown promising potential for cancer diagnosis, that could offer cost-effective and faster alternatives to existing methods. in this work, we propose a vision transformer-based framework for distinguishing dlbcl (diffuse large b-cell lymphoma) cancer subtypes from high-resolution wsis. to this end, we introduce a multi-modal architecture to train a classifier model from various wsi modalities. we then leverage this model through a knowledge distillation process to efficiently guide the learning of a mono-modal classifier. our experimental study conducted on a lymphoma dataset of 157 patients shows the promising performance of our mono-modal classification model, outperforming six recent state-of-the-art methods. in addition, the power-law curve, estimated on our experimental data, suggests that with more training data from a reasonable number of additional patients, our model could achieve competitive diagnosis accuracy with ihc technologies. furthermore, the efficiency of our framework is confirmed through an additional experimental study on an external breast cancer dataset (bci dataset).',\n",
       "  'categories': 'eess.iv cs.cv q-bio.qm',\n",
       "  'doi': '10.1109/jbhi.2024.3407878',\n",
       "  'created': '2023-08-02',\n",
       "  'updated': '2024-05-29',\n",
       "  'authors': ['bilel guetarni',\n",
       "   'feryal windal',\n",
       "   'halim benhabiles',\n",
       "   'marianne petit',\n",
       "   'romain dubois',\n",
       "   'emmanuelle leteurtre',\n",
       "   'dominique collard'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.01328'},\n",
       " {'title': \"dispersal-enhanced resilience in two-patch metapopulations: origin's   instability type matters\",\n",
       "  'id': '2308.01364',\n",
       "  'abstract': 'many populations of animals or plants, exhibit a metapopulation structure with close, spatially-separated subpopulations. the field of metapopulation theory has made significant advancements since the influential levins model. various modeling approaches have provided valuable insights to theoretical ecology. despite extensive research on metapopulation models, there are still challenging questions that are difficult to answer from ecological metapopulational data or multi-patch models. low-dimension mathematical models offer a promising avenue to address these questions, especially for global dynamics which have been scarcely investigated. in this study, we investigate a two-patch metapopulation model with logistic growth and diffusion between patches. by using analytical and numerical methods, we thoroughly analyze the impact of diffusion on the dynamics of the metapopulation. we identify the equilibrium points and assess their local and global stability. furthermore, we analytically derive the optimal diffusion rate that leads to the highest metapopulation values. our findings demonstrate that increased diffusion plays a crucial role in the preservation of both subpopulations and the full metapopulation, especially under the presence of stochastic perturbations. specifically, at low diffusion values, the origin is a repeller, causing orbits starting around it to travel closely parallel to the axes. this configuration makes the metapopulation less resilient and thus more susceptible to local and global extinctions. however, as diffusion increases, the repeller transitions to a saddle point, and orbits starting near the origin rapidly converge to the unstable manifold of the saddle. this phenomenon reduces the likelihood of stochastic extinctions and the metapopulation becomes more resilient due to these changes in the vector field of the phase space.',\n",
       "  'categories': 'q-bio.pe math.ds',\n",
       "  'doi': '10.1007/s12064-023-00411-2',\n",
       "  'created': '2023-08-02',\n",
       "  'updated': '2023-12-11',\n",
       "  'authors': ['marc jorba-cuscó',\n",
       "   'ruth i. oliva-zúniga',\n",
       "   'josep sardanyés',\n",
       "   'daniel pérez-palau'],\n",
       "  'affiliation': ['centre de recerca matemàtica',\n",
       "   'universidad nacional autónoma de honduras',\n",
       "   'centre de recerca matemàtica',\n",
       "   'universidad internacional de la rioja'],\n",
       "  'url': 'https://arxiv.org/abs/2308.01364'},\n",
       " {'title': 'is your data alignable? principled and interpretable alignability   testing and integration of single-cell data',\n",
       "  'id': '2308.01839',\n",
       "  'abstract': \"single-cell data integration can provide a comprehensive molecular view of cells, and many algorithms have been developed to remove unwanted technical or biological variations and integrate heterogeneous single-cell datasets. despite their wide usage, existing methods suffer from several fundamental limitations. in particular, we lack a rigorous statistical test for whether two high-dimensional single-cell datasets are alignable (and therefore should even be aligned). moreover, popular methods can substantially distort the data during alignment, making the aligned data and downstream analysis difficult to interpret. to overcome these limitations, we present a spectral manifold alignment and inference (smai) framework, which enables principled and interpretable alignability testing and structure-preserving integration of single-cell data with the same type of features. smai provides a statistical test to robustly assess the alignability between datasets to avoid misleading inference, and is justified by high-dimensional statistical theory. on a diverse range of real and simulated benchmark datasets, it outperforms commonly used alignment methods. moreover, we show that smai improves various downstream analyses such as identification of differentially expressed genes and imputation of single-cell spatial transcriptomics, providing further biological insights. smai's interpretability also enables quantification and a deeper understanding of the sources of technical confounders in single-cell data.\",\n",
       "  'categories': 'q-bio.qm cs.cv q-bio.gn stat.ap stat.ml',\n",
       "  'doi': '10.1073/pnas.2313719121',\n",
       "  'created': '2023-08-03',\n",
       "  'updated': '2024-02-29',\n",
       "  'authors': ['rong ma', 'eric d. sun', 'david donoho', 'james zou'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.01839'},\n",
       " {'title': 'digital twin brain: a bridge between biological intelligence and   artificial intelligence',\n",
       "  'id': '2308.01941',\n",
       "  'abstract': \"in recent years, advances in neuroscience and artificial intelligence have paved the way for unprecedented opportunities for understanding the complexity of the brain and its emulation by computational systems. cutting-edge advancements in neuroscience research have revealed the intricate relationship between brain structure and function, while the success of artificial neural networks highlights the importance of network architecture. now is the time to bring them together to better unravel how intelligence emerges from the brain's multiscale repositories. in this review, we propose the digital twin brain (dtb) as a transformative platform that bridges the gap between biological and artificial intelligence. it consists of three core elements: the brain structure that is fundamental to the twinning process, bottom-layer models to generate brain functions, and its wide spectrum of applications. crucially, brain atlases provide a vital constraint, preserving the brain's network organization within the dtb. furthermore, we highlight open questions that invite joint efforts from interdisciplinary fields and emphasize the far-reaching implications of the dtb. the dtb can offer unprecedented insights into the emergence of intelligence and neurological disorders, which holds tremendous promise for advancing our understanding of both biological and artificial intelligence, and ultimately propelling the development of artificial general intelligence and facilitating precision mental healthcare.\",\n",
       "  'categories': 'q-bio.nc cs.ai cs.ne',\n",
       "  'doi': '10.34133/icomputing.0055',\n",
       "  'created': '2023-08-02',\n",
       "  'updated': '',\n",
       "  'authors': ['hui xiong',\n",
       "   'congying chu',\n",
       "   'lingzhong fan',\n",
       "   'ming song',\n",
       "   'jiaqi zhang',\n",
       "   'yawei ma',\n",
       "   'ruonan zheng',\n",
       "   'junyang zhang',\n",
       "   'zhengyi yang',\n",
       "   'tianzi jiang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.01941'},\n",
       " {'title': 'fixation times on directed graphs',\n",
       "  'id': '2308.02762',\n",
       "  'abstract': 'computing the rate of evolution in spatially structured populations is difficult. a key quantity is the fixation time of a single mutant with relative reproduction rate $r$ which invades a population of residents. we say that the fixation time is \"fast\" if it is at most a polynomial function in terms of the population size $n$. here we study fixation times of advantageous mutants ($r>1$) and neutral mutants ($r=1$) on directed graphs, which are those graphs that have at least some one-way connections. we obtain three main results. first, we prove that for any directed graph the fixation time is fast, provided that $r$ is sufficiently large. second, we construct an efficient algorithm that gives an upper bound for the fixation time for any graph and any $r\\\\ge 1$. third, we identify a broad class of directed graphs with fast fixation times for any $r\\\\ge 1$. this class includes previously studied amplifiers of selection, such as superstars and metafunnels. we also show that on some graphs the fixation time is not a monotonically declining function of $r$; in particular, neutral fixation can occur faster than fixation for small selective advantages.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-04',\n",
       "  'updated': '2024-03-05',\n",
       "  'authors': ['david a. brewster', 'martin a. nowak', 'josef tkadlec'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.02762'},\n",
       " {'title': 'msigsdk -- private, at scale, computation of mutation signatures',\n",
       "  'id': '2308.02995',\n",
       "  'abstract': \"in our previous work, we demonstrated that it is feasible to perform analysis on mutation signature data without the need for downloads or installations and analyze individual patient data at scale without compromising privacy. building on this foundation, we developed a software development kit (sdk) called msigsdk to facilitate the orchestration of distributed data processing workflows and graphic visualization of mutational signature analysis results. we strictly adhered to modern web computing standards, particularly the modularization standards set by the ecmascript es6 framework (javascript modules). our approach allows for computation to be entirely performed by secure delegation to the computational resources of the user's own machine (in-browser), without any downloads or installations. the msigsdk was developed primarily as a companion library to the msig portal resource of the national cancer institute division of cancer epidemiology and genetics (nih/nci/dceg), with a focus on its fair extensibility as components of other researchers' computational constructs. anticipated extensions include the programmatic operation of other mutation signature api ecosystems such as signal and cosmic, advancing towards a data commons for mutational signature research (grossman et al., 2016).\",\n",
       "  'categories': 'q-bio.gn',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-05',\n",
       "  'updated': '2024-01-19',\n",
       "  'authors': ['aaron ge',\n",
       "   'yasmmin côrtes martins',\n",
       "   'tongwu zhang',\n",
       "   'kailing chen',\n",
       "   'maria teresa landi',\n",
       "   'brian park',\n",
       "   'jeya balasubramanian',\n",
       "   'jonas s almeida'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.02995'},\n",
       " {'title': 'adapting machine learning diagnostic models to new populations using a   small amount of data: results from clinical neuroscience',\n",
       "  'id': '2308.03175',\n",
       "  'abstract': \"machine learning (ml) has shown great promise for revolutionizing a number of areas, including healthcare. however, it is also facing a reproducibility crisis, especially in medicine. ml models that are carefully constructed from and evaluated on a training set might not generalize well on data from different patient populations or acquisition instrument settings and protocols. we tackle this problem in the context of neuroimaging of alzheimer's disease (ad), schizophrenia (sz) and brain aging. we develop a weighted empirical risk minimization approach that optimally combines data from a source group, e.g., subjects are stratified by attributes such as sex, age group, race and clinical cohort to make predictions on a target group, e.g., other sex, age group, etc. using a small fraction (10%) of data from the target group. we apply this method to multi-source data of 15,363 individuals from 20 neuroimaging studies to build ml models for diagnosis of ad and sz, and estimation of brain age. we found that this approach achieves substantially better accuracy than existing domain adaptation techniques: it obtains area under curve greater than 0.95 for ad classification, area under curve greater than 0.7 for sz classification and mean absolute error less than 5 years for brain age prediction on all target groups, achieving robustness to variations of scanners, protocols, and demographic or clinical characteristics. in some cases, it is even better than training on all data from the target group, because it leverages the diversity and size of a larger training set. we also demonstrate the utility of our models for prognostic tasks such as predicting disease progression in individuals with mild cognitive impairment. critically, our brain age prediction models lead to new clinical insights regarding correlations with neurophysiological tests.\",\n",
       "  'categories': 'cs.lg eess.iv q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-06',\n",
       "  'updated': '2024-09-12',\n",
       "  'authors': ['rongguang wang',\n",
       "   'guray erus',\n",
       "   'pratik chaudhari',\n",
       "   'christos davatzikos'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.03175'},\n",
       " {'title': 'enhancing cell tracking with a time-symmetric deep learning approach',\n",
       "  'id': '2308.03887',\n",
       "  'abstract': 'the accurate tracking of live cells using video microscopy recordings remains a challenging task for popular state-of-the-art image processing based object tracking methods. in recent years, several existing and new applications have attempted to integrate deep-learning based frameworks for this task, but most of them still heavily rely on consecutive frame based tracking embedded in their architecture or other premises that hinder generalized learning. to address this issue, we aimed to develop a new deep-learning based tracking method that relies solely on the assumption that cells can be tracked based on their spatio-temporal neighborhood, without restricting it to consecutive frames. the proposed method has the additional benefit that the motion patterns of the cells can be learned completely by the predictor without any prior assumptions, and it has the potential to handle a large number of video frames with heavy artifacts. the efficacy of the proposed method is demonstrated through biologically motivated validation strategies and compared against multiple state-of-the-art cell tracking methods.',\n",
       "  'categories': 'eess.iv cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-04',\n",
       "  'updated': '2024-09-03',\n",
       "  'authors': ['gergely szabó',\n",
       "   'paolo bonaiuti',\n",
       "   'andrea ciliberto',\n",
       "   'andrás horváth'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.03887'},\n",
       " {'title': 'molecular docking via quantum approximate optimization algorithm',\n",
       "  'id': '2308.04098',\n",
       "  'abstract': \"molecular docking plays a pivotal role in drug discovery and precision medicine, enabling us to understand protein functions and advance novel therapeutics. here, we introduce a potential alternative solution to this problem, the digitized-counterdiabatic quantum approximate optimization algorithm (dc-qaoa), which utilizes counterdiabatic driving and qaoa on a quantum computer. our method was applied to analyze diverse biological systems, including the sars-cov-2 mpro complex with pm-2-020b, the dpp-4 complex with piperidine fused imidazopyridine 34, and the hiv-1 gp120 complex with jp-iii-048. the dc-qaoa exhibits superior performance, providing more accurate and biologically relevant docking results, especially for larger molecular docking problems. moreover, qaoa-based algorithms demonstrate enhanced hardware compatibility in the noisy intermediate-scale quantum era, indicating their potential for efficient implementation under practical docking scenarios. our findings underscore quantum computing's potential in drug discovery and offer valuable insights for optimizing protein-ligand docking processes.\",\n",
       "  'categories': 'quant-ph physics.chem-ph q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-08',\n",
       "  'updated': '2024-05-15',\n",
       "  'authors': ['qi-ming ding', 'yi-ming huang', 'xiao yuan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.04098'},\n",
       " {'title': 'gromov-wasserstein unsupervised alignment reveals structural   correspondences between the color similarity structures of humans and large   language models',\n",
       "  'id': '2308.04381',\n",
       "  'abstract': 'large language models (llms), such as the general pre-trained transformer (gpt), have shown remarkable performance in various cognitive tasks. however, it remains unclear whether these models have the ability to accurately infer human perceptual representations. previous research has addressed this question by quantifying correlations between similarity response patterns of humans and llms. correlation provides a measure of similarity, but it relies pre-defined item labels and does not distinguish category- and item- level similarity, falling short of characterizing detailed structural correspondence between humans and llms. to assess their structural equivalence in more detail, we propose the use of an unsupervised alignment method based on gromov-wasserstein optimal transport (gwot). gwot allows for the comparison of similarity structures without relying on pre-defined label correspondences and can reveal fine-grained structural similarities and differences that may not be detected by simple correlation analysis. using a large dataset of similarity judgments of 93 colors, we compared the color similarity structures of humans (color-neurotypical and color-atypical participants) and two gpt models (gpt-3.5 and gpt-4). our results show that the similarity structure of color-neurotypical participants can be remarkably well aligned with that of gpt-4 and, to a lesser extent, to that of gpt-3.5. these results contribute to the methodological advancements of comparing llms with human perception, and highlight the potential of unsupervised alignment methods to reveal detailed structural correspondences. this work has been published in scientific reports, doi: https://doi.org/10.1038/s41598-024-65604-1.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1038/s41598-024-65604-1 10.1038/s41598-024-65604-1   10.1038/s41598-024-65604-1',\n",
       "  'created': '2023-08-08',\n",
       "  'updated': '2024-08-13',\n",
       "  'authors': ['genji kawakita',\n",
       "   'ariel zeleznikow-johnston',\n",
       "   'naotsugu tsuchiya',\n",
       "   'masafumi oizumi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.04381'},\n",
       " {'title': 'harnessing artificial intelligence to reduce phototoxicity in live   imaging',\n",
       "  'id': '2308.04387',\n",
       "  'abstract': 'fluorescence microscopy, widely used in the study of living cells, tissues, and organisms, often faces the challenge of photodamage. this is primarily caused by the interaction between light and biochemical components during the imaging process, leading to compromised accuracy and reliability of biological results. methods necessitating extended high-intensity illumination, such as super-resolution microscopy or thick sample imaging, are particularly susceptible to this issue. as part of the solution to these problems, advanced imaging approaches involving artificial intelligence (ai) have been developed. here we underscore the necessity of establishing constraints to maintain light-induced damage at levels that permit cells to sustain their live behaviour. from this perspective, data-driven live-cell imaging bears significant potential in aiding the development of ai-enhanced photodamage-aware microscopy. these technologies could streamline precise observations of natural biological dynamics while minimising phototoxicity risks.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.1242/jcs.261545',\n",
       "  'created': '2023-08-08',\n",
       "  'updated': '',\n",
       "  'authors': ['estibaliz gómez-de-mariscal',\n",
       "   'mario del rosario',\n",
       "   'joanna w pylvänäinen',\n",
       "   'guillaume jacquemet',\n",
       "   'ricardo henriques'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.04387'},\n",
       " {'title': 'microbundlecompute: automated segmentation, tracking, and analysis of   subdomain deformation in cardiac microbundles',\n",
       "  'id': '2308.04610',\n",
       "  'abstract': 'advancing human induced pluripotent stem cell derived cardiomyocyte (hipsc-cm) technology will lead to significant progress ranging from disease modeling, to drug discovery, to regenerative tissue engineering. yet, alongside these potential opportunities comes a critical challenge: attaining mature hipsc-cm tissues. at present, there are multiple techniques to promote maturity of hipsc-cms including physical platforms and cell culture protocols. however, when it comes to making quantitative comparisons of functional behavior, there are limited options for reliably and reproducibly computing functional metrics that are suitable for direct cross-system comparison. in addition, the current standard functional metrics obtained from time-lapse images of cardiac microbundle contraction reported in the field (i.e., post forces, average tissue stress) do not take full advantage of the available information present in these data (i.e., full-field tissue displacements and strains). thus, we present \"microbundlecompute,\" a computational framework for automatic quantification of morphology-based mechanical metrics from movies of cardiac microbundles. briefly, this computational framework offers tools for automatic tissue segmentation, tracking, and analysis of brightfield and phase contrast movies of beating cardiac microbundles. it is straightforward to implement, requires little to no parameter tuning, and runs quickly on a personal computer. in this paper, we describe the methods underlying this computational framework, show the results of our extensive validation studies, and demonstrate the utility of exploring heterogeneous tissue deformations and strains as functional metrics. with this manuscript, we disseminate \"microbundlecompute\" as an open-source computational tool with the aim of making automated quantitative analysis of beating cardiac microbundles more accessible to the community.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-08',\n",
       "  'updated': '2024-02-20',\n",
       "  'authors': ['hiba kobeissi',\n",
       "   'javiera jilberto',\n",
       "   'm. çağatay karakan',\n",
       "   'xining gao',\n",
       "   'samuel j. depalma',\n",
       "   'shoshana l. das',\n",
       "   'lani quach',\n",
       "   'jonathan urquia',\n",
       "   'brendon m. baker',\n",
       "   'christopher s. chen',\n",
       "   'david nordsletten',\n",
       "   'emma lejeune'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.04610'},\n",
       " {'title': 'universal scaling relation and criticality in metabolism and growth of   escherichia coli',\n",
       "  'id': '2308.04776',\n",
       "  'abstract': 'the metabolic network plays a crucial role in regulating bacterial metabolism and growth, but it is subject to inherent molecular stochasticity. previous studies have utilized flux balance analysis and the maximum entropy method to predict metabolic fluxes and growth rates, while the underlying principles governing bacterial metabolism and growth, especially the criticality hypothesis, remain unclear. in this study, we employ a maximum entropy approach to investigate the universality in various constraint-based metabolic networks of escherichia coli. our findings reveal the existence of universal scaling relations across different nutritional environments and metabolic network models, similar to the universality observed in physics. by analyzing single-cell data, we confirm that metabolism of escherichia coli operates close to the state with maximum fisher information, which serves as a signature of criticality. this critical state provides functional advantages such as high sensitivity and long-range correlation. moreover, we demonstrate that a metabolic system operating at criticality takes a compromise solution between growth and adaptation, thereby serving as a survival strategy in fluctuating environments.',\n",
       "  'categories': 'q-bio.mn cond-mat.stat-mech',\n",
       "  'doi': '10.1103/physrevresearch.6.013035',\n",
       "  'created': '2023-08-09',\n",
       "  'updated': '2024-01-11',\n",
       "  'authors': ['shaohua guan', 'zhichao zhang', 'zihan zhang', 'hualin shi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.04776'},\n",
       " {'title': 'integrating large language models and active inference to understand eye   movements in reading and dyslexia',\n",
       "  'id': '2308.04941',\n",
       "  'abstract': \"we present a novel computational model employing hierarchical active inference to simulate reading and eye movements. the model characterizes linguistic processing as inference over a hierarchical generative model, facilitating predictions and inferences at various levels of granularity, from syllables to sentences.   our approach combines the strengths of large language models for realistic textual predictions and active inference for guiding eye movements to informative textual information, enabling the testing of predictions. the model exhibits proficiency in reading both known and unknown words and sentences, adhering to the distinction between lexical and nonlexical routes in dual-route theories of reading. notably, our model permits the exploration of maladaptive inference effects on eye movements during reading, such as in dyslexia. to simulate this condition, we attenuate the contribution of priors during the reading process, leading to incorrect inferences and a more fragmented reading style, characterized by a greater number of shorter saccades. this alignment with empirical findings regarding eye movements in dyslexic individuals highlights the model's potential to aid in understanding the cognitive processes underlying reading and eye movements, as well as how reading deficits associated with dyslexia may emerge from maladaptive predictive processing.   in summary, our model represents a significant advancement in comprehending the intricate cognitive processes involved in reading and eye movements, with potential implications for understanding and addressing dyslexia through the simulation of maladaptive inference. it may offer valuable insights into this condition and contribute to the development of more effective interventions for treatment.\",\n",
       "  'categories': 'q-bio.nc cs.cl',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-09',\n",
       "  'updated': '2024-02-24',\n",
       "  'authors': ['francesco donnarumma', 'mirco frosolone', 'giovanni pezzulo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.04941'},\n",
       " {'title': 'improved cryo-em pose estimation and 3d classification through   latent-space disentanglement',\n",
       "  'id': '2308.04956',\n",
       "  'abstract': 'due to the extremely low signal-to-noise ratio (snr) and unknown poses (projection angles and image shifts) in cryo-electron microscopy (cryo-em) experiments, reconstructing 3d volumes from 2d images is very challenging. in addition to these challenges, heterogeneous cryo-em reconstruction requires conformational classification. in popular cryo-em reconstruction algorithms, poses and conformation classification labels must be predicted for every input cryo-em image, which can be computationally costly for large datasets. an emerging class of methods adopted the amortized inference approach. in these methods, only a subset of the input dataset is needed to train neural networks for the estimation of poses and conformations. once trained, these neural networks can make pose/conformation predictions and 3d reconstructions at low cost for the entire dataset during inference. unfortunately, when facing heterogeneous reconstruction tasks, it is hard for current amortized-inference-based methods to effectively estimate the conformational distribution and poses from entangled latent variables. here, we propose a self-supervised variational autoencoder architecture called \"hetacumn\" based on amortized inference. we employed an auxiliary conditional pose prediction task by inverting the order of encoder-decoder to explicitly enforce the disentanglement of conformation and pose predictions. results on simulated datasets show that hetacumn generated more accurate conformational classifications than other amortized or non-amortized methods. furthermore, we show that hetacumn is capable of performing heterogeneous 3d reconstructions of a real experimental dataset.',\n",
       "  'categories': 'eess.iv cs.cv q-bio.bm q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-09',\n",
       "  'updated': '2024-04-22',\n",
       "  'authors': ['weijie chen', 'yuhang wang', 'lin yao'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.04956'},\n",
       " {'title': 'abdiffuser: full-atom generation of in vitro functioning antibodies',\n",
       "  'id': '2308.05027',\n",
       "  'abstract': 'we introduce abdiffuser, an equivariant and physics-informed diffusion model for the joint generation of antibody 3d structures and sequences. abdiffuser is built on top of a new representation of protein structure, relies on a novel architecture for aligned proteins, and utilizes strong diffusion priors to improve the denoising process. our approach improves protein diffusion by taking advantage of domain knowledge and physics-based constraints; handles sequence-length changes; and reduces memory complexity by an order of magnitude, enabling backbone and side chain generation. we validate abdiffuser in silico and in vitro. numerical experiments showcase the ability of abdiffuser to generate antibodies that closely track the sequence and structural properties of a reference set. laboratory experiments confirm that all 16 her2 antibodies discovered were expressed at high levels and that 57.1% of the selected designs were tight binders.',\n",
       "  'categories': 'q-bio.bm cs.lg stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-07-28',\n",
       "  'updated': '2024-03-06',\n",
       "  'authors': ['karolis martinkus',\n",
       "   'jan ludwiczak',\n",
       "   'kyunghyun cho',\n",
       "   'wei-ching liang',\n",
       "   'julien lafrance-vanasse',\n",
       "   'isidro hotzel',\n",
       "   'arvind rajpal',\n",
       "   'yan wu',\n",
       "   'richard bonneau',\n",
       "   'vladimir gligorijevic',\n",
       "   'andreas loukas'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.05027'},\n",
       " {'title': 'ptransips: identification of phosphorylation sites enhanced by protein   plm embeddings',\n",
       "  'id': '2308.05115',\n",
       "  'abstract': 'phosphorylation is pivotal in numerous fundamental cellular processes and plays a significant role in the onset and progression of various diseases. the accurate identification of these phosphorylation sites is crucial for unraveling the molecular mechanisms within cells and during viral infections, potentially leading to the discovery of novel therapeutic targets. in this study, we develop ptransips, a new deep learning framework for the identification of phosphorylation sites. independent testing results demonstrate that ptransips outperforms existing state-of-the-art (sota) methods, achieving aucs of 0.9232 and 0.9660 for the identification of phosphorylated s/t and y sites, respectively. ptransips contributes from three aspects. 1) ptransips is the first to apply protein pre-trained language model (plm) embeddings to this task. it utilizes prottrans and ember2 to extract sequence and structure embeddings, respectively, as additional inputs into the model, effectively addressing issues of dataset size and overfitting, thus enhancing model performance; 2) ptransips is based on transformer architecture, optimized through the integration of convolutional neural networks and tim loss function, providing practical insights for model design and training; 3) the encoding of amino acids in ptransips enables it to serve as a universal framework for other peptide bioactivity tasks, with its excellent performance shown in extended experiments of this paper. our code, data and models are publicly available at https://github.com/statxzy7/ptransips.',\n",
       "  'categories': 'q-bio.qm cs.lg',\n",
       "  'doi': '10.1109/jbhi.2024.3377362',\n",
       "  'created': '2023-08-08',\n",
       "  'updated': '2024-03-13',\n",
       "  'authors': ['ziyang xu',\n",
       "   'haitian zhong',\n",
       "   'bingrui he',\n",
       "   'xueying wang',\n",
       "   'tianchi lu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.05115'},\n",
       " {'title': 'models matter: the impact of single-step retrosynthesis on synthesis   planning',\n",
       "  'id': '2308.05522',\n",
       "  'abstract': 'retrosynthesis consists of breaking down a chemical compound recursively step-by-step into molecular precursors until a set of commercially available molecules is found with the goal to provide a synthesis route. its two primary research directions, single-step retrosynthesis prediction, which models the chemical reaction logic, and multi-step synthesis planning, which tries to find the correct sequence of reactions, are inherently intertwined. still, this connection is not reflected in contemporary research. in this work, we combine these two major research directions by applying multiple single-step retrosynthesis models within multi-step synthesis planning and analyzing their impact using public and proprietary reaction data. we find a disconnection between high single-step performance and potential route-finding success, suggesting that single-step models must be evaluated within synthesis planning in the future. furthermore, we show that the commonly used single-step retrosynthesis benchmark dataset uspto-50k is insufficient as this evaluation task does not represent model performance and scalability on larger and more diverse datasets. for multi-step synthesis planning, we show that the choice of the single-step model can improve the overall success rate of synthesis planning by up to +28% compared to the commonly used baseline model. finally, we show that each single-step model finds unique synthesis routes, and differs in aspects such as route-finding success, the number of found synthesis routes, and chemical validity, making the combination of single-step retrosynthesis prediction and multi-step synthesis planning a crucial aspect when developing future methods.',\n",
       "  'categories': 'cs.ai cs.lg physics.chem-ph q-bio.bm',\n",
       "  'doi': '10.1039/d3dd00252g',\n",
       "  'created': '2023-08-10',\n",
       "  'updated': '',\n",
       "  'authors': ['paula torren-peraire',\n",
       "   'alan kai hassen',\n",
       "   'samuel genheden',\n",
       "   'jonas verhoeven',\n",
       "   'djork-arne clevert',\n",
       "   'mike preuss',\n",
       "   'igor tetko'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.05522'},\n",
       " {'title': 'modelling non-local cell-cell adhesion: a multiscale approach',\n",
       "  'id': '2308.05676',\n",
       "  'abstract': 'cell-cell adhesion plays a vital role in the development and maintenance of multicellular organisms. one of its functions is regulation of cell migration, such as occurs, e.g. during embryogenesis or in cancer. in this work, we develop a versatile multiscale approach to modelling a moving self-adhesive cell population that combines a careful microscopic description of a deterministic adhesion-driven motion component with an efficient mesoscopic representation of a stochastic velocity-jump process. this approach gives rise to mesoscopic models in the form of kinetic transport equations featuring multiple non-localities. subsequent parabolic and hyperbolic scalings produce general classes of equations with non-local adhesion and myopic diffusion, a special case being the classical macroscopic model proposed in [4]. our simulations show how the combination of the two motion effects can unfold. cell-cell adhesion relies on the subcellular cell adhesion molecule binding. our approach lends itself conveniently to capturing this microscopic effect. on the macroscale, this results in an additional non-linear integral equation of a novel type that is coupled to the cell density equation.',\n",
       "  'categories': 'q-bio.to math.ap',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-10',\n",
       "  'updated': '2024-03-07',\n",
       "  'authors': ['anna zhigun', 'mabel lizzy rajendran'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.05676'},\n",
       " {'title': 'the multi-modality cell segmentation challenge: towards universal   solutions',\n",
       "  'id': '2308.05864',\n",
       "  'abstract': 'cell segmentation is a critical step for quantitative single-cell analysis in microscopy images. existing cell segmentation methods are often tailored to specific modalities or require manual interventions to specify hyper-parameters in different experimental settings. here, we present a multi-modality cell segmentation benchmark, comprising over 1500 labeled images derived from more than 50 diverse biological experiments. the top participants developed a transformer-based deep-learning algorithm that not only exceeds existing methods but can also be applied to diverse microscopy images across imaging platforms and tissue types without manual parameter adjustments. this benchmark and the improved algorithm offer promising avenues for more accurate and versatile cell analysis in microscopy imaging.',\n",
       "  'categories': 'eess.iv cs.cv cs.lg q-bio.qm',\n",
       "  'doi': '10.1038/s41592-024-02233-6',\n",
       "  'created': '2023-08-10',\n",
       "  'updated': '2024-04-01',\n",
       "  'authors': ['jun ma',\n",
       "   'ronald xie',\n",
       "   'shamini ayyadhury',\n",
       "   'cheng ge',\n",
       "   'anubha gupta',\n",
       "   'ritu gupta',\n",
       "   'song gu',\n",
       "   'yao zhang',\n",
       "   'gihun lee',\n",
       "   'joonkee kim',\n",
       "   'wei lou',\n",
       "   'haofeng li',\n",
       "   'eric upschulte',\n",
       "   'timo dickscheid',\n",
       "   'josé guilherme de almeida',\n",
       "   'yixin wang',\n",
       "   'lin han',\n",
       "   'xin yang',\n",
       "   'marco labagnara',\n",
       "   'vojislav gligorovski',\n",
       "   'maxime scheder',\n",
       "   'sahand jamal rahi',\n",
       "   'carly kempster',\n",
       "   'alice pollitt',\n",
       "   'leon espinosa',\n",
       "   'tâm mignot',\n",
       "   'jan moritz middeke',\n",
       "   'jan-niklas eckardt',\n",
       "   'wangkai li',\n",
       "   'zhaoyang li',\n",
       "   'xiaochen cai',\n",
       "   'bizhe bai',\n",
       "   'noah f. greenwald',\n",
       "   'david van valen',\n",
       "   'erin weisbart',\n",
       "   'beth a. cimini',\n",
       "   'trevor cheung',\n",
       "   'oscar brück',\n",
       "   'gary d. bader',\n",
       "   'bo wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.05864'},\n",
       " {'title': 'probabilistic neural transfer function estimation with bayesian system   identification',\n",
       "  'id': '2308.05990',\n",
       "  'abstract': 'neural population responses in sensory systems are driven by external physical stimuli. this stimulus-response relationship is typically characterized by receptive fields, which have been estimated by neural system identification approaches. such models usually requires a large amount of training data, yet, the recording time for animal experiments is limited, giving rise to epistemic uncertainty for the learned neural transfer functions. while deep neural network models have demonstrated excellent power on neural prediction, they usually do not provide the uncertainty of the resulting neural representations and derived statistics, such as the stimuli driving neurons optimally, from in silico experiments. here, we present a bayesian system identification approach to predict neural responses to visual stimuli, and explore whether explicitly modeling network weight variability can be beneficial for identifying neural response properties. to this end, we use variational inference to estimate the posterior distribution of each model weight given the training data. tests with different neural datasets demonstrate that this method can achieve higher or comparable performance on neural prediction, with a much higher data efficiency compared to monte carlo dropout methods and traditional models using point estimates of the model parameters. furthermore, our approach enables to identify response properties with credible intervals and perform statistical test for the learned neural features, which avoid the idiosyncrasy of a single model. finally, in silico experiments show that our model generates stimuli driving neuronal activity significantly better than traditional models, particularly in the limited-data regime.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-11',\n",
       "  'updated': '2024-02-07',\n",
       "  'authors': ['nan wu',\n",
       "   'isabel valera',\n",
       "   'fabian sinz',\n",
       "   'alexander ecker',\n",
       "   'thomas euler',\n",
       "   'yongrong qiu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.05990'},\n",
       " {'title': 'non-markovian gene expression',\n",
       "  'id': '2308.06538',\n",
       "  'abstract': 'we study two non-markovian gene-expression models in which protein production is a stochastic process with a fat-tailed non-exponential waiting time distribution (wtd). for both models, we find two distinct scaling regimes separated by an exponentially long time, proportional to the mean first passage time (mfpt) to a ground state (with zero proteins) of the dynamics, from which the system can only exit via a non-exponential reaction. at times shorter than the mfpt the dynamics are stationary and ergodic, entailing similarity across different realizations of the same process, with an increased fano factor of the protein distribution, even when the wtd has a finite cutoff. notably, at times longer than the mfpt the dynamics are nonstationary and nonergodic, entailing significant variability across different realizations. the mfpt to the ground state is shown to directly affect the average population sizes and we postulate that the transition to nonergodicity is universal in such non-markovian models.',\n",
       "  'categories': 'cond-mat.stat-mech q-bio.mn',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-12',\n",
       "  'updated': '2024-04-05',\n",
       "  'authors': ['ohad vilk', 'ralf metzler', 'michael assaf'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.06538'},\n",
       " {'title': 'the time is ripe to reverse engineer an entire nervous system:   simulating behavior from neural interactions',\n",
       "  'id': '2308.06578',\n",
       "  'abstract': \"just like electrical engineers understand how microprocessors execute programs in terms of how transistor currents are affected by their inputs, neuroscientists want to understand behavior production in terms of how neuronal outputs are affected by their inputs and internal states. this dependency of neuronal outputs on inputs can be described by a state-dependent input-output (io)-function. however, to reliably identify these io-functions, we need to perturb each input and combinations of inputs while observing all the outputs. here, we argue that such completeness is possible in c. elegans; a complete description that goes all the way from the activity of every neuron to predict behavior. the established and growing toolkit of optophysiology can non-invasively capture and control every neuron's activity and scale to countless experiments. the information from many such experiments can be pooled while capturing the inter-individual variability because neuronal identity and function are largely conserved across individuals. just like electrical engineers use transistor io-functions to simulate program execution, we argue that neuronal io-functions could be used to simulate the impressive breadth of brain states and behaviors of c. elegans.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-12',\n",
       "  'updated': '2024-09-18',\n",
       "  'authors': ['gal haspel',\n",
       "   'ben baker',\n",
       "   'isabel beets',\n",
       "   'edward s boyden',\n",
       "   'jeffrey brown',\n",
       "   'george church',\n",
       "   'netta cohen',\n",
       "   'daniel colon-ramos',\n",
       "   'eva dyer',\n",
       "   'christopher fang-yen',\n",
       "   'steven flavell',\n",
       "   'miriam b goodman',\n",
       "   'anne c hart',\n",
       "   'eduardo j izquierdo',\n",
       "   'konstantinos kagias',\n",
       "   'shawn lockery',\n",
       "   'yangning lu',\n",
       "   'adam marblestone',\n",
       "   'jordan matelsky',\n",
       "   'brett mensh',\n",
       "   'talmo d pereira',\n",
       "   'hanspeter pfister',\n",
       "   'kanaka rajan',\n",
       "   'horacio g rotstein',\n",
       "   'monika scholz',\n",
       "   'joshua w. shaevitz',\n",
       "   'eli shlizerman',\n",
       "   'quilee simeon',\n",
       "   'michael a skuhersky',\n",
       "   'vineet tiruvadi',\n",
       "   'vivek venkatachalam',\n",
       "   'donglai wei',\n",
       "   'brock wester',\n",
       "   'guangyu robert yang',\n",
       "   'eviatar yemini',\n",
       "   'manuel zimmer',\n",
       "   'konrad p kording'],\n",
       "  'affiliation': ['njit',\n",
       "   'colby college',\n",
       "   'ku leuven',\n",
       "   'mit',\n",
       "   'mit',\n",
       "   'harvard university',\n",
       "   'university of leeds',\n",
       "   'yale university',\n",
       "   'georgia institute of technology',\n",
       "   'ohio state university',\n",
       "   'mit',\n",
       "   'stanford university',\n",
       "   'brown university',\n",
       "   'rose-hulman institute of technology',\n",
       "   'mit',\n",
       "   'university of oregon',\n",
       "   'mit',\n",
       "   'convergent research',\n",
       "   'university of pennsylvania',\n",
       "   'optimize science',\n",
       "   'salk institute',\n",
       "   'harvard university',\n",
       "   'harvard medical school',\n",
       "   'njit',\n",
       "   'max planck institute for neurobiology of behavior',\n",
       "   'princeton university',\n",
       "   'university of washington',\n",
       "   'mit',\n",
       "   'mit',\n",
       "   'hume ai',\n",
       "   'northeastern university',\n",
       "   'boston college',\n",
       "   'johns hopkins apl',\n",
       "   'mit',\n",
       "   'umass',\n",
       "   'university of vienna',\n",
       "   'university of pennsylvania'],\n",
       "  'url': 'https://arxiv.org/abs/2308.06578'},\n",
       " {'title': 'the effect of host population heterogeneity on epidemic outbreaks',\n",
       "  'id': '2308.06593',\n",
       "  'abstract': 'in the first part of this paper, we review old and new results about the influence of host population heterogeneity on (various characteristics of) epidemic outbreaks. in the second part we highlight a modelling issue that so far has received little attention: how do contact patterns, and hence transmission opportunities, depend on the size and the composition of the host population? without any claim on completeness, we offer a range of potential (quasi-mechanistic) submodels. the overall aim of the paper is to describe the state-of-the-art and to catalyse new work.',\n",
       "  'categories': 'q-bio.pe math.ds',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-12',\n",
       "  'updated': '2024-01-16',\n",
       "  'authors': ['martin bootsma', 'danny chan', 'odo diekmann', 'hisashi inaba'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.06593'},\n",
       " {'title': 'git-mol: a multi-modal large language model for molecular science with   graph, image, and text',\n",
       "  'id': '2308.06911',\n",
       "  'abstract': 'large language models have made significant strides in natural language processing, enabling innovative applications in molecular science by processing textual representations of molecules. however, most existing language models cannot capture the rich information with complex molecular structures or images. in this paper, we introduce git-mol, a multi-modal large language model that integrates the graph, image, and text information. to facilitate the integration of multi-modal molecular data, we propose git-former, a novel architecture that is capable of aligning all modalities into a unified latent space. we achieve a 5%-10% accuracy increase in properties prediction and a 20.2% boost in molecule generation validity compared to the baselines. with the any-to-language molecular translation strategy, our model has the potential to perform more downstream tasks, such as compound name recognition and chemical reaction prediction.',\n",
       "  'categories': 'cs.lg cs.cl q-bio.bm',\n",
       "  'doi': '10.1016/j.compbiomed.2024.108073',\n",
       "  'created': '2023-08-13',\n",
       "  'updated': '2024-02-06',\n",
       "  'authors': ['pengfei liu', 'yiming ren', 'jun tao', 'zhixiang ren'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.06911'},\n",
       " {'title': 'evolving division of labor in a response threshold model',\n",
       "  'id': '2308.07122',\n",
       "  'abstract': 'the response threshold model explains the emergence of division of labor (i.e., task specialization) in an unstructured population by assuming that the individuals have different propensities to work on different tasks. the incentive to attend to a particular task increases when the task is left unattended and decreases when individuals work on it. here we derive mean-field equations for the stimulus dynamics and show that they exhibit complex attractors through period-doubling bifurcation cascades when the noise disrupting the thresholds is small. in addition, we show how the fixed threshold can be set to ensure specialization in both the transient and equilibrium regimes of the stimulus dynamics. however, a complete explanation of the emergence of division of labor requires that we address the question of where the threshold variation comes from, starting from a homogeneous population. we then study a structured population scenario, where the population is divided into a large number of independent groups of equal size, and the fitness of a group is proportional to the weighted mean work performed on the tasks during a fixed period of time. using a winner-take-all strategy to model group competition and assuming an initial homogeneous metapopulation, we find that a substantial fraction of workers specialize in each task, without the need to penalize task switching.',\n",
       "  'categories': 'q-bio.pe nlin.ao nlin.cd',\n",
       "  'doi': '10.1016/j.ecocom.2024.101083',\n",
       "  'created': '2023-08-14',\n",
       "  'updated': '',\n",
       "  'authors': ['josé f. fontanari',\n",
       "   'viviane m. de oliveira',\n",
       "   'paulo r. a. campos'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.07122'},\n",
       " {'title': 'a fast algorithm for all-pairs-shortest-paths suitable for neural   networks',\n",
       "  'id': '2308.07403',\n",
       "  'abstract': 'given a directed graph of nodes and edges connecting them, a common problem is to find the shortest path between any two nodes. here we show that the shortest path distances can be found by a simple matrix inversion: if the edges are given by the adjacency matrix $a_{ij}$ then with a suitably small value of $\\\\gamma$ the shortest path distances are   $$ d_{ij} = \\\\operatorname{ceil} \\\\left( \\\\log_{\\\\gamma} {\\\\left[ {\\\\left({\\\\mathbf{i}}-\\\\gamma {\\\\mathbf{a}}\\\\right)^{-1}} \\\\right]}_{ij} \\\\right)$$   we derive several graph-theoretic bounds on the value of $\\\\gamma$, and explore its useful range with numerics on different graph types. even when the distance function is not globally accurate across the entire graph, it still works locally to instruct pursuit of the shortest path. in this mode, it also extends to weighted graphs with positive edge weights. for a wide range of dense graphs this distance function is computationally faster than the best available alternative. finally we show that this method leads naturally to a neural network solution of the all-pairs-shortest-path problem.',\n",
       "  'categories': 'cs.ds q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-06',\n",
       "  'updated': '2024-07-23',\n",
       "  'authors': ['zeyu jing', 'markus meister'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.07403'},\n",
       " {'title': 'n-body oscillator interactions of higher-order coupling functions',\n",
       "  'id': '2308.07425',\n",
       "  'abstract': 'we introduce a method to identify phase equations that include $n$-body interactions for general coupled oscillators valid far beyond the weak coupling approximation. this strategy is an extension of the theory from [park and wilson, siads 20.3 (2021)] and yields coupling functions for $n\\\\geq2$ oscillators for arbitrary types of coupling (e.g., diffusive, gap-junction, chemical synaptic). these coupling functions enable the study of oscillator networks in terms of phase-locked states, whose stability can be determined using straightforward linear stability arguments. we demonstrate the utility of our approach with two examples. first, we use $n=3$ diffusively coupled complex ginzburg-landau (cgl) model and show that the loss of stability in its splay state occurs through a hopf bifurcation \\\\yp{as a function of non-weak diffusive coupling. our reduction also captures asymptotic limit-cycle dynamics in the phase differences}. second, we use $n=3$ realistic conductance-based thalamic neuron models and show that our method correctly predicts a loss in stability of a splay state for non-weak synaptic coupling. in both examples, our theory accurately captures model behaviors that weak and recent non-weak coupling theories can not.',\n",
       "  'categories': 'q-bio.nc nlin.cd',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-14',\n",
       "  'updated': '2024-04-05',\n",
       "  'authors': ['youngmin park', 'dan wilson'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.07425'},\n",
       " {'title': 'apace: alphafold2 and advanced computing as a service for accelerated   discovery in biophysics',\n",
       "  'id': '2308.07954',\n",
       "  'abstract': 'the prediction of protein 3d structure from amino acid sequence is a computational grand challenge in biophysics, and plays a key role in robust protein structure prediction algorithms, from drug discovery to genome interpretation. the advent of ai models, such as alphafold, is revolutionizing applications that depend on robust protein structure prediction algorithms. to maximize the impact, and ease the usability, of these novel ai tools we introduce apace, alphafold2 and advanced computing as a service, a novel computational framework that effectively handles this ai model and its tb-size database to conduct accelerated protein structure prediction analyses in modern supercomputing environments. we deployed apace in the delta and polaris supercomputers, and quantified its performance for accurate protein structure predictions using four exemplar proteins: 6awo, 6oan, 7mez, and 6d6u. using up to 300 ensembles, distributed across 200 nvidia a100 gpus, we found that apace is up to two orders of magnitude faster than off-the-self alphafold2 implementations, reducing time-to-solution from weeks to minutes. this computational approach may be readily linked with robotics laboratories to automate and accelerate scientific discovery.',\n",
       "  'categories': 'q-bio.bm cs.ai cs.dc',\n",
       "  'doi': '10.1073/pnas.2311888121',\n",
       "  'created': '2023-08-15',\n",
       "  'updated': '2024-07-01',\n",
       "  'authors': ['hyun park', 'parth patel', 'roland haas', 'e. a. huerta'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.07954'},\n",
       " {'title': 'macroscopic behavior of populations of quadratic integrate-and-fire   neurons subject to non-gaussian white noise',\n",
       "  'id': '2308.08194',\n",
       "  'abstract': 'we study macroscopic behavior of populations of quadratic integrate-and-fire neurons subject to non-gaussian noises; we argue that these noises must be alpha-stable whenever they are delta-correlated (white). for the case of additive-in-voltage noise, we derive the governing equation of the dynamics of the characteristic function of the membrane voltage distribution and construct a linear-in-noise perturbation theory. specifically for the recurrent network with global synaptic coupling, we theoretically calculate the observables: population-mean membrane voltage and firing rate. the theoretical results are underpinned by the results of numerical simulation for homogeneous and heterogeneous populations. the possibility of the generalization of the pseudocumulant approach to the case of a fractional $\\\\alpha$ is examined for both irrational and fractional rational $\\\\alpha$. this examination seemingly suggests the pseudocumulant approach or its modifications to be employable only for the integer values of $\\\\alpha=1$ (cauchy noise) and $2$ (gaussian noise) within the physically meaningful range $(0;2]$. remarkably, the analysis for fractional $\\\\alpha$ indirectly revealed that, for the gaussian noise, the minimal asymptotically rigorous model reduction must involve three pseudocumulants and the two-pseudocumulant model reduction is an artificial approximation. this explains a surprising gain of accuracy for the three-pseudocumulant models as compared to the the two-pseudocumulant ones reported in the literature.',\n",
       "  'categories': 'cond-mat.stat-mech nlin.ao q-bio.nc',\n",
       "  'doi': '10.1063/5.0172735',\n",
       "  'created': '2023-08-16',\n",
       "  'updated': '2023-12-06',\n",
       "  'authors': ['denis s. goldobin',\n",
       "   'evelina v. permyakova',\n",
       "   'lyudmila s. klimenko'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.08194'},\n",
       " {'title': 'in distributive phosphorylation catalytic constants enable non-trivial   dynamics',\n",
       "  'id': '2308.08524',\n",
       "  'abstract': 'ordered distributive double phosphorylation is a recurrent motif in intracellular signaling and control. it is either sequential (where the site phosphorylated last is dephosphorylated first) or cyclic (where the site phosphorylated first is dephosphorylated first). sequential distributive double phosphorylation has been extensively studied and an inequality involving only the catalytic constants of kinase and phosphatase is known to be sufficient for multistationarity. as multistationarity is necessary for bistability it has been argued that these constants enable bistability.   here we show for cyclic distributive double phosphorylation that if its catalytic constants satisfy the very same inequality, then hopf bifurcations and hence sustained oscillations can occur. hence we argue that in distributive double phosphorylation (sequential or distributive) the catalytic constants enable non-trivial dynamics.   in fact, if the rate constant values in a network of cyclic distributive double phosphorylation are such that hopf bifurcations and sustained oscillations can occur, then a network of sequential distributive double phosphorylation with the same rate constant values will show multistationarity -- albeit for different values of the total concentrations. for cyclic distributive double phosphorylation we further describe a procedure to generate rate constant values where hopf bifurcations and hence sustained oscillations can occur. this may, for example, allow for an efficient sampling of oscillatory regions in parameter space.   our analysis is greatly simplified by the fact that it is possible to reduce the network of cyclic distributive double phosphorylation to what we call a network with a single extreme ray. we summarize key properties of these networks.',\n",
       "  'categories': 'q-bio.mn math.ds',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-16',\n",
       "  'updated': '2024-02-08',\n",
       "  'authors': ['carsten conradi', 'maya mincheva'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.08524'},\n",
       " {'title': 'implementation of the future of drug discovery: quantumbased machine   learning simulation (qmls)',\n",
       "  'id': '2308.08561',\n",
       "  'abstract': 'the research & development (r&d) phase of drug development is a lengthy and costly process. to revolutionize this process, we introduce our new concept qmls to shorten the whole r&d phase to three to six months and decrease the cost to merely fifty to eighty thousand usd. for hit generation, machine learning molecule generation (mlmg) generates possible hits according to the molecular structure of the target protein while the quantum simulation (qs) filters molecules from the primary essay based on the reaction and binding effectiveness with the target protein. then, for lead optimization, the resultant molecules generated and filtered from mlmg and qs are compared, and molecules that appear as a result of both processes will be made into dozens of molecular variations through machine learning molecule variation (mlmv), while others will only be made into a few variations. lastly, all optimized molecules would undergo multiple rounds of qs filtering with a high standard for reaction effectiveness and safety, creating a few dozen pre-clinical-trail-ready drugs. this paper is based on our first paper, where we pitched the concept of machine learning combined with quantum simulations. in this paper we will go over the detailed design and framework of qmls, including mlmg, mlmv, and qs.',\n",
       "  'categories': 'q-bio.bm cs.ai cs.lg',\n",
       "  'doi': '10.5281/zenodo.7983561',\n",
       "  'created': '2023-08-14',\n",
       "  'updated': '2024-09-05',\n",
       "  'authors': ['yifan zhou',\n",
       "   'yan shing liang',\n",
       "   'yew kee wong',\n",
       "   'haichuan qiu',\n",
       "   'yu xi wu',\n",
       "   'bin he'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.08561'},\n",
       " {'title': 'a hierarchy index for networks in the brain reveals a complex entangled   organizational structure',\n",
       "  'id': '2308.08898',\n",
       "  'abstract': 'networks involved in information processing often have their nodes arranged hierarchically, with the majority of connections occurring in adjacent levels. however, despite being an intuitively appealing concept, the hierarchical organization of large networks, such as those in the brain, are difficult to identify, especially in absence of additional information beyond that provided by the connectome. in this paper, we propose a framework to uncover the hierarchical structure of a given network, that identifies the nodes occupying each level as well as the sequential order of the levels. it involves optimizing a metric that we use to quantify the extent of hierarchy present in a network. applying this measure to various brain networks, ranging from the nervous system of the nematode caenorhabditis elegans to the human connectome, we unexpectedly find that they exhibit a common network architectural motif intertwining hierarchy and modularity. this suggests that brain networks may have evolved to simultaneously exploit the functional advantages of these two types of organizations, viz., relatively independent modules performing distributed processing in parallel and a hierarchical structure that allows sequential pooling of these multiple processing streams. an intriguing possibility is that this property we report may be common to information processing networks in general.',\n",
       "  'categories': 'q-bio.nc physics.bio-ph',\n",
       "  'doi': '10.1073/pnas.2314291121',\n",
       "  'created': '2023-08-17',\n",
       "  'updated': '',\n",
       "  'authors': ['anand pathak', 'shakti n. menon', 'sitabhra sinha'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.08898'},\n",
       " {'title': 'quantifying the biomimicry gap in biohybrid robot-fish pairs',\n",
       "  'id': '2308.08978',\n",
       "  'abstract': 'biohybrid systems in which robotic lures interact with animals have become compelling tools for probing and identifying the mechanisms underlying collective animal behavior. one key challenge lies in the transfer of social interaction models from simulations to reality, using robotics to validate the modeling hypotheses. this challenge arises in bridging what we term the \"biomimicry gap\", which is caused by imperfect robotic replicas, communication cues and physics constraints not incorporated in the simulations, that may elicit unrealistic behavioral responses in animals. in this work, we used a biomimetic lure of a rummy-nose tetra fish (hemigrammus rhodostomus) and a neural network (nn) model for generating biomimetic social interactions. through experiments with a biohybrid pair comprising a fish and the robotic lure, a pair of real fish, and simulations of pairs of fish, we demonstrate that our biohybrid system generates social interactions mirroring those of genuine fish pairs. our analyses highlight that: 1) the lure and nn maintain minimal deviation in real-world interactions compared to simulations and fish-only experiments, 2) our nn controls the robot efficiently in real-time, and 3) a comprehensive validation is crucial to bridge the biomimicry gap, ensuring realistic biohybrid systems.',\n",
       "  'categories': 'cs.ro cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-17',\n",
       "  'updated': '2024-03-17',\n",
       "  'authors': ['vaios papaspyros',\n",
       "   'guy theraulaz',\n",
       "   'clément sire',\n",
       "   'francesco mondada'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.08978'},\n",
       " {'title': 'sbsm-pro: support bio-sequence machine for proteins',\n",
       "  'id': '2308.10275',\n",
       "  'abstract': 'proteins play a pivotal role in biological systems. the use of machine learning algorithms for protein classification can assist and even guide biological experiments, offering crucial insights for biotechnological applications. we introduce the support bio-sequence machine for proteins (sbsm-pro), a model purpose-built for the classification of biological sequences. this model starts with raw sequences and groups amino acids based on their physicochemical properties. it incorporates sequence alignment to measure the similarities between proteins and uses a novel multiple kernel learning (mkl) approach to integrate various types of information, utilizing support vector machines for classification prediction. the results indicate that our model demonstrates commendable performance across ten datasets in terms of the identification of protein function and posttranslational modification. this research not only exemplifies state-of-the-art work in protein classification but also paves avenues for new directions in this domain, representing a beneficial endeavor in the development of platforms tailored for the classification of biological sequences. sbsm-pro is available for access at http://lab.malab.cn/soft/sbsm-pro/.',\n",
       "  'categories': 'q-bio.qm cs.lg',\n",
       "  'doi': '10.1007/s11432-024-4171-9',\n",
       "  'created': '2023-08-20',\n",
       "  'updated': '2023-11-04',\n",
       "  'authors': ['yizheng wang', 'yixiao zhai', 'yijie ding', 'quan zou'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.10275'},\n",
       " {'title': 'linking fast and slow: the case for generative models',\n",
       "  'id': '2308.10618',\n",
       "  'abstract': 'a pervasive challenge in neuroscience is testing whether neuronal connectivity changes over time due to specific causes, such as stimuli, events, or clinical interventions. recent hardware innovations and falling data storage costs enable longer, more naturalistic neuronal recordings. the implicit opportunity for understanding the self-organised brain calls for new analysis methods that link temporal scales: from the order of milliseconds over which neuronal dynamics evolve, to the order of minutes, days or even years over which experimental observations unfold. this review article demonstrates how hierarchical generative models and bayesian inference help to characterise neuronal activity across different time scales. crucially, these methods go beyond describing statistical associations among observations and enable inference about underlying mechanisms. we offer an overview of fundamental concepts in state-space modeling and suggest a taxonomy for these methods. additionally, we introduce key mathematical principles that underscore a separation of temporal scales, such as the slaving principle, and review bayesian methods that are being used to test hypotheses about the brain with multi-scale data. we hope that this review will serve as a useful primer for experimental and computational neuroscientists on the state of the art and current directions of travel in the complex systems modelling literature.',\n",
       "  'categories': 'q-bio.nc stat.ap',\n",
       "  'doi': '10.1162/netn_a_00343',\n",
       "  'created': '2023-08-21',\n",
       "  'updated': '',\n",
       "  'authors': ['johan medrano', 'karl j. friston', 'peter zeidman'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.10618'},\n",
       " {'title': 'automated mapping of virtual environments with visual predictive coding',\n",
       "  'id': '2308.10913',\n",
       "  'abstract': 'humans construct internal cognitive maps of their environment directly from sensory inputs without access to a system of explicit coordinates or distance measurements. while machine learning algorithms like slam utilize specialized visual inference procedures to identify visual features and construct spatial maps from visual and odometry data, the general nature of cognitive maps in the brain suggests a unified mapping algorithmic strategy that can generalize to auditory, tactile, and linguistic inputs. here, we demonstrate that predictive coding provides a natural and versatile neural network algorithm for constructing spatial maps using sensory data. we introduce a framework in which an agent navigates a virtual environment while engaging in visual predictive coding using a self-attention-equipped convolutional neural network. while learning a next image prediction task, the agent automatically constructs an internal representation of the environment that quantitatively reflects distances. the internal map enables the agent to pinpoint its location relative to landmarks using only visual information.the predictive coding network generates a vectorized encoding of the environment that supports vector navigation where individual latent space units delineate localized, overlapping neighborhoods in the environment. broadly, our work introduces predictive coding as a unified algorithmic framework for constructing cognitive maps that can naturally extend to the mapping of auditory, sensorimotor, and linguistic inputs.',\n",
       "  'categories': 'q-bio.nc cs.cv cs.lg eess.iv',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-20',\n",
       "  'updated': '2024-04-17',\n",
       "  'authors': ['james gornet', 'matthew thomson'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.10913'},\n",
       " {'title': 'bridging cognitive maps: a hierarchical active inference model of   spatial alternation tasks and the hippocampal-prefrontal circuit',\n",
       "  'id': '2308.11463',\n",
       "  'abstract': \"cognitive problem-solving benefits from cognitive maps aiding navigation and planning. previous studies revealed that cognitive maps for physical space navigation involve hippocampal (hc) allocentric codes, while cognitive maps for abstract task space engage medial prefrontal cortex (mpfc) task-specific codes. solving challenging cognitive tasks requires integrating these two types of maps. this is exemplified by spatial alternation tasks in multi-corridor settings, where animals like rodents are rewarded upon executing an alternation pattern in maze corridors. existing studies demonstrated the hc - mpfc circuit's engagement in spatial alternation tasks and that its disruption impairs task performance. yet, a comprehensive theory explaining how this circuit integrates task-related and spatial information is lacking. we advance a novel hierarchical active inference model clarifying how the hc - mpfc circuit enables the resolution of spatial alternation tasks, by merging physical and task-space cognitive maps. through a series of simulations, we demonstrate that the model's dual layers acquire effective cognitive maps for navigation within physical (hc map) and task (mpfc map) spaces, using a biologically-inspired approach: a clone-structured cognitive graph. the model solves spatial alternation tasks through reciprocal interactions between the two layers. importantly, disrupting inter-layer communication impairs difficult decisions, consistent with empirical findings. the same model showcases the ability to switch between multiple alternation rules. however, inhibiting message transmission between the two layers results in perseverative behavior, consistent with empirical findings. in summary, our model provides a mechanistic account of how the hc - mpfc circuit supports spatial alternation tasks and how its disruption impairs task performance.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-22',\n",
       "  'updated': '2024-07-27',\n",
       "  'authors': ['toon van de maele',\n",
       "   'bart dhoedt',\n",
       "   'tim verbelen',\n",
       "   'giovanni pezzulo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.11463'},\n",
       " {'title': 'unstable cores are the source of instability in chemical reaction   networks',\n",
       "  'id': '2308.11486',\n",
       "  'abstract': 'in biochemical networks, complex dynamical features such as superlinear growth and oscillations are classically considered a consequence of autocatalysis. for the large class of parameter-rich kinetic models, which includes generalized mass action kinetics and michaelis-menten kinetics, we show that certain submatrices of the stoichiometric matrix, so-called unstable cores, are sufficient for a reaction network to admit instability and potentially give rise to such complex dynamical behavior. the determinant of the submatrix distinguishes unstable-positive feedbacks, with a single real-positive eigenvalue, and unstable-negative feedbacks without real-positive eigenvalues. autocatalytic cores turn out to be exactly the unstable-positive feedbacks that are metzler matrices. thus there are sources of dynamical instability in chemical networks that are unrelated to autocatalysis. we use such intuition to design non-autocatalytic biochemical networks with superlinear growth and oscillations.',\n",
       "  'categories': 'q-bio.mn math.ds',\n",
       "  'doi': '10.1098/rspa.2023.0694',\n",
       "  'created': '2023-08-22',\n",
       "  'updated': '2024-03-05',\n",
       "  'authors': ['nicola vassena', 'peter f. stadler'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.11486'},\n",
       " {'title': 'a data-driven approach to morphogenesis under structural instability',\n",
       "  'id': '2308.11846',\n",
       "  'abstract': 'morphological development into evolutionary patterns under structural instability is ubiquitous in living systems and often of vital importance for engineering structures. here we propose a data-driven approach to understand and predict their spatiotemporal complexities. a machine-learning framework is proposed based on the physical modeling of morphogenesis triggered by internal or external forcing. digital libraries of structural patterns are constructed from the simulation data, which are then used to recognize the abnormalities, predict their development, and assist in risk assessment and prognosis. the capabilities to identify the key bifurcation characteristics and predict the history-dependent development from the global and local features are demonstrated by examples of brain growth and aerospace structural design, which offer guidelines for disease diagnosis/prognosis and instability-tolerant design.',\n",
       "  'categories': 'nlin.ps cs.cv q-bio.qm stat.ml',\n",
       "  'doi': '10.1016/j.xcrp.2024.101872',\n",
       "  'created': '2023-08-22',\n",
       "  'updated': '',\n",
       "  'authors': ['yingjie zhao', 'zhiping xu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.11846'},\n",
       " {'title': 'state-transition dynamics of resting-state functional magnetic resonance   imaging data: model comparison and test-to-retest analysis',\n",
       "  'id': '2308.11910',\n",
       "  'abstract': 'electroencephalogram (eeg) microstate analysis entails finding dynamics of quasi-stable and generally recurrent discrete states in multichannel eeg time series data and relating properties of the estimated state-transition dynamics to observables such as cognition and behavior. while microstate analysis has been widely employed to analyze eeg data, its use remains less prevalent in functional magnetic resonance imaging (fmri) data, largely due to the slower timescale of such data. in the present study, we extend various data clustering methods used in eeg microstate analysis to resting-state fmri data from healthy humans to extract their state-transition dynamics. we show that the quality of clustering is on par with that for various microstate analyses of eeg data. we then develop a method for examining test-retest reliability of the discrete-state transition dynamics between fmri sessions and show that the within-participant test-retest reliability is higher than between-participant test-retest reliability for different indices of state-transition dynamics, different networks, and different data sets. this result suggests that state-transition dynamics analysis of fmri data could discriminate between different individuals and is a promising tool for performing fingerprinting analysis of individuals.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1186/s12868-024-00854-3',\n",
       "  'created': '2023-08-23',\n",
       "  'updated': '2024-03-05',\n",
       "  'authors': ['saiful islam',\n",
       "   'pitambar khanra',\n",
       "   'johan nakuci',\n",
       "   'sarah f. muldoon',\n",
       "   'takamitsu watanabe',\n",
       "   'naoki masuda'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.11910'},\n",
       " {'title': 'will more expressive graph neural networks do better on generative   tasks?',\n",
       "  'id': '2308.11978',\n",
       "  'abstract': 'graph generation poses a significant challenge as it involves predicting a complete graph with multiple nodes and edges based on simply a given label. this task also carries fundamental importance to numerous real-world applications, including de-novo drug and molecular design. in recent years, several successful methods have emerged in the field of graph generation. however, these approaches suffer from two significant shortcomings: (1) the underlying graph neural network (gnn) architectures used in these methods are often underexplored; and (2) these methods are often evaluated on only a limited number of metrics. to fill this gap, we investigate the expressiveness of gnns under the context of the molecular graph generation task, by replacing the underlying gnns of graph generative models with more expressive gnns. specifically, we analyse the performance of six gnns in two different generative frameworks -- autoregressive generation models, such as gcpn and graphaf, and one-shot generation models, such as graphebm -- on six different molecular generative objectives on the zinc-250k dataset. through our extensive experiments, we demonstrate that advanced gnns can indeed improve the performance of gcpn, graphaf, and graphebm on molecular generation tasks, but gnn expressiveness is not a necessary condition for a good gnn-based generative model. moreover, we show that gcpn and graphaf with advanced gnns can achieve state-of-the-art results across 17 other non-gnn-based graph generative approaches, such as variational autoencoders and bayesian optimisation models, on the proposed molecular generative objectives (drd2, median1, median2), which are important metrics for de-novo molecular design.',\n",
       "  'categories': 'cs.lg cs.ai q-bio.bm stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-23',\n",
       "  'updated': '2024-02-20',\n",
       "  'authors': ['xiandong zou', 'xiangyu zhao', 'pietro liò', 'yiren zhao'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.11978'},\n",
       " {'title': 'critical learning periods emerge even in deep linear networks',\n",
       "  'id': '2308.12221',\n",
       "  'abstract': 'critical learning periods are periods early in development where temporary sensory deficits can have a permanent effect on behavior and learned representations. despite the radical differences between biological and artificial networks, critical learning periods have been empirically observed in both systems. this suggests that critical periods may be fundamental to learning and not an accident of biology. yet, why exactly critical periods emerge in deep networks is still an open question, and in particular it is unclear whether the critical periods observed in both systems depend on particular architectural or optimization details. to isolate the key underlying factors, we focus on deep linear network models, and show that, surprisingly, such networks also display much of the behavior seen in biology and artificial networks, while being amenable to analytical treatment. we show that critical periods depend on the depth of the model and structure of the data distribution. we also show analytically and in simulations that the learning of features is tied to competition between sources. finally, we extend our analysis to multi-task learning to show that pre-training on certain tasks can damage the transfer performance on new tasks, and show how this depends on the relationship between tasks and the duration of the pre-training stage. to the best of our knowledge, our work provides the first analytically tractable model that sheds light into why critical learning periods emerge in biological and artificial networks.',\n",
       "  'categories': 'cs.lg cs.ai q-bio.nc stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-23',\n",
       "  'updated': '2024-05-24',\n",
       "  'authors': ['michael kleinman', 'alessandro achille', 'stefano soatto'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.12221'},\n",
       " {'title': 'predicting drug solubility using different machine learning methods --   linear regression model with extracted chemical features vs graph   convolutional neural network',\n",
       "  'id': '2308.12325',\n",
       "  'abstract': 'predicting the solubility of given molecules remains crucial in the pharmaceutical industry. in this study, we revisited this extensively studied topic, leveraging the capabilities of contemporary computing resources. we employed two machine learning models: a linear regression model and a graph convolutional neural network (gcnn) model, using various experimental datasets. both methods yielded reasonable predictions, with the gcnn model exhibiting the highest level of performance. however, the present gcnn model has limited interpretability while the linear regression model allows scientists for a greater in-depth analysis of the underlying factors through feature importance analysis, although more human inputs and evaluations on the overall dataset is required. from the perspective of chemistry, using the linear regression model, we elucidated the impact of individual atom species and functional groups on overall solubility, highlighting the significance of comprehending how chemical structure influences chemical properties in the drug development process. it is learned that introducing oxygen atoms can increase the solubility of organic molecules, while almost all other hetero atoms except oxygen and nitrogen tend to decrease solubility.',\n",
       "  'categories': 'q-bio.qm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-23',\n",
       "  'updated': '2024-01-04',\n",
       "  'authors': ['john ho',\n",
       "   'zhao-heng yin',\n",
       "   'colin zhang',\n",
       "   'nicole guo',\n",
       "   'yang ha'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.12325'},\n",
       " {'title': 'network transformation-based analysis of biochemical systems',\n",
       "  'id': '2308.12586',\n",
       "  'abstract': 'a dynamical system obtains a wide variety of kinetic realizations, which is advantageous for the analysis of biochemical systems. a reaction network, derived from a dynamical system, may or may not possess some properties needed for a thorough analysis. we improve and extend the work of m. johnston \\\\cite{john2014} and hong et al. \\\\cite{hong2023} on network translations to network transformations, where the network is modified while preserving the dynamical system. these transformations can shrink, extend, or retain the stoichiometric subspace. here, we show that positive dependent network can be translated to a weakly reversible network. using the kinetic realizations of (1) calcium signaling in the olfactory system and (2) metabolic insulin signaling, we demonstrate the benefits of transformed systems with positive deficiency for analyzing biochemical systems. furthermore, we present an algorithm for a network transformation of a weakly reversible non-complex factorizable kinetic (nfk) system to a weakly reversible complex factorizable kinetic (cfk) system, thereby enhancing the subspace coincidence theorem for nfk systems of nazareno et al. \\\\cite{naza2019}. finally, using the transformed kinetic realization of monolignol biosynthesis in \\\\textit{populus xylem}, we study the structural and kinetic properties of transformed systems,including the invariance of concordance and variation of injectivity and mono-/multi-stationarity under network transformation.',\n",
       "  'categories': 'q-bio.mn math.ds',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-24',\n",
       "  'updated': '2024-04-30',\n",
       "  'authors': ['dylan antonio talabis', 'eduardo mendoza'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.12586'},\n",
       " {'title': 'genome replication in asynchronously growing microbial populations',\n",
       "  'id': '2308.12664',\n",
       "  'abstract': 'biological cells replicate their genomes in a well-planned manner. the dna replication program of an organism determines the timing at which different genomic regions are replicated, with fundamental consequences for cell homeostasis and genome stability. qualitatively, in a growing cell culture, one expects that genomic regions that are replicated early should be more abundant than regions that are replicated late. this abundance pattern can be experimentally measured using deep sequencing. however, a general quantitative theory to explain these data is still lacking. in this paper, we predict the abundance of dna fragments in asynchronously growing cultures from any given stochastic model of the dna replication program. as key examples, we present stochastic models of the dna replication programs in escherichia coli and in budding yeast. in both cases, our approach leads to analytical predictions that are in excellent agreement with experimental data and permit to infer key information about the replication program. in particular, our method is able to infer the locations of known replication origins in budding yeast with high accuracy. these examples demonstrate that our method can provide insight into a broad range of organisms, from bacteria to eukaryotes.',\n",
       "  'categories': 'q-bio.sc q-bio.pe',\n",
       "  'doi': '10.1371/journal.pcbi.1011753',\n",
       "  'created': '2023-08-24',\n",
       "  'updated': '2023-09-22',\n",
       "  'authors': ['florian pflug', 'deepak bhat', 'simone pigolotti'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.12664'},\n",
       " {'title': 'decoding natural images from eeg for object recognition',\n",
       "  'id': '2308.13234',\n",
       "  'abstract': 'electroencephalography (eeg) signals, known for convenient non-invasive acquisition but low signal-to-noise ratio, have recently gained substantial attention due to the potential to decode natural images. this paper presents a self-supervised framework to demonstrate the feasibility of learning image representations from eeg signals, particularly for object recognition. the framework utilizes image and eeg encoders to extract features from paired image stimuli and eeg responses. contrastive learning aligns these two modalities by constraining their similarity. with the framework, we attain significantly above-chance results on a comprehensive eeg-image dataset, achieving a top-1 accuracy of 15.6% and a top-5 accuracy of 42.8% in challenging 200-way zero-shot tasks. moreover, we perform extensive experiments to explore the biological plausibility by resolving the temporal, spatial, spectral, and semantic aspects of eeg signals. besides, we introduce attention modules to capture spatial correlations, providing implicit evidence of the brain activity perceived from eeg data. these findings yield valuable insights for neural decoding and brain-computer interfaces in real-world scenarios. the code will be released on https://github.com/eeyhsong/nice-eeg.',\n",
       "  'categories': 'cs.hc cs.ai eess.sp q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-25',\n",
       "  'updated': '2024-04-04',\n",
       "  'authors': ['yonghao song',\n",
       "   'bingchuan liu',\n",
       "   'xiang li',\n",
       "   'nanlin shi',\n",
       "   'yijun wang',\n",
       "   'xiaorong gao'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.13234'},\n",
       " {'title': 'alternating shrinking higher-order interactions for sparse neural   population activity',\n",
       "  'id': '2308.13257',\n",
       "  'abstract': \"neurons in living things work cooperatively and efficiently to process incoming sensory information, often exhibiting sparse and widespread population activity involving structured higher-order interactions. while there are statistical models based on continuous probability distributions for neurons' sparse firing rates, how the spiking activities of a large number of interacting neurons result in the sparse and widespread population activity remains unknown. here, for homogeneous (0,1) binary neurons, we provide sufficient conditions under which their spike-count population distribution converges to a sparse widespread distribution of the population spike rate in an infinitely large population of neurons. following the conditions, we propose new models belonging to an exponential family distribution in which the sign and magnitude of neurons' higher-order interactions alternate and shrink as the order increases. the distributions exhibit parameter-dependent sparsity on a bounded support for the population firing rate. the theory serves as a building block for developing prior distributions and neurons' non-linearity for spike-based sparse coding.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-25',\n",
       "  'updated': '2024-05-03',\n",
       "  'authors': ['ulises rodríguez-domínguez', 'hideaki shimazaki'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.13257'},\n",
       " {'title': 'robust core-periphery constrained transformer for domain adaptation',\n",
       "  'id': '2308.13515',\n",
       "  'abstract': \"unsupervised domain adaptation (uda) aims to learn transferable representation across domains. recently a few uda works have successfully applied transformer-based methods and achieved state-of-the-art (sota) results. however, it remains challenging when there exists a large domain gap between the source and target domain. inspired by humans' exceptional transferability abilities to adapt knowledge from familiar to uncharted domains, we try to apply the universally existing organizational structure in the human functional brain networks, i.e., the core-periphery principle to design the transformer and improve its uda performance. in this paper, we propose a novel brain-inspired robust core-periphery constrained transformer (rcct) for unsupervised domain adaptation, which brings a large margin of performance improvement on various datasets. specifically, in rcct, the self-attention operation across image patches is rescheduled by an adaptively learned weighted graph with the core-periphery structure (cp graph), where the information communication and exchange between images patches are manipulated and controlled by the connection strength, i.e., edge weight of the learned weighted cp graph. besides, since the data in domain adaptation tasks can be noisy, to improve the model robustness, we intentionally add perturbations to the patches in the latent space to ensure generating robust learned weighted core-periphery graphs. extensive evaluations are conducted on several widely tested uda benchmarks. our proposed rcct consistently performs best compared to existing works, including 88.3\\\\% on office-home, 95.0\\\\% on office-31, 90.7\\\\% on visda-2017, and 46.0\\\\% on domainnet.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-25',\n",
       "  'updated': '2024-02-10',\n",
       "  'authors': ['xiaowei yu', 'dajiang zhu', 'tianming liu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.13515'},\n",
       " {'title': 'semiparametric inference of effective reproduction number dynamics from   wastewater pathogen surveillance data',\n",
       "  'id': '2308.15770',\n",
       "  'abstract': 'concentrations of pathogen genomes measured in wastewater have recently become available as a new data source to use when modeling the spread of infectious diseases. one promising use for this data source is inference of the effective reproduction number, the average number of individuals a newly infected person will infect. we propose a model where new infections arrive according to a time-varying immigration rate which can be interpreted as an average number of secondary infections produced by one infectious individual per unit time. this model allows us to estimate the effective reproduction number from concentrations of pathogen genomes while avoiding difficult to verify assumptions about the dynamics of the susceptible population. as a byproduct of our primary goal, we also produce a new model for estimating the effective reproduction number from case data using the same framework. we test this modeling framework in an agent-based simulation study with a realistic data generating mechanism which accounts for the time-varying dynamics of pathogen shedding. finally, we apply our new model to estimating the effective reproduction number of sars-cov-2 in los angeles, california, using pathogen rna concentrations collected from a large wastewater treatment facility.',\n",
       "  'categories': 'stat.me q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-30',\n",
       "  'updated': '2024-06-21',\n",
       "  'authors': ['isaac h. goldstein',\n",
       "   'daniel m. parker',\n",
       "   'sunny jiang',\n",
       "   'volodymyr m. minin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.15770'},\n",
       " {'title': 'msgnn: multi-scale spatio-temporal graph neural network for epidemic   forecasting',\n",
       "  'id': '2308.15840',\n",
       "  'abstract': 'infectious disease forecasting has been a key focus and proved to be crucial in controlling epidemic. a recent trend is to develop forecast-ing models based on graph neural networks (gnns). however, existing gnn-based methods suffer from two key limitations: (1) current models broaden receptive fields by scaling the depth of gnns, which is insuffi-cient to preserve the semantics of long-range connectivity between distant but epidemic related areas. (2) previous approaches model epidemics within single spatial scale, while ignoring the multi-scale epidemic pat-terns derived from different scales. to address these deficiencies, we devise the multi-scale spatio-temporal graph neural network (msgnn) based on an innovative multi-scale view. to be specific, in the proposed msgnn model, we first devise a novel graph learning module, which directly captures long-range connectivity from trans-regional epidemic signals and integrates them into a multi-scale graph. based on the learned multi-scale graph, we utilize a newly designed graph convolution module to exploit multi-scale epidemic patterns. this module allows us to facilitate multi-scale epidemic modeling by mining both scale-shared and scale-specific pat-terns. experimental results on forecasting new cases of covid-19 in united state demonstrate the superiority of our method over state-of-arts. further analyses and visualization also show that msgnn offers not only accurate, but also robust and interpretable forecasting result.',\n",
       "  'categories': 'cs.lg cs.ai physics.soc-ph q-bio.pe',\n",
       "  'doi': '10.1007/s10618-024-01035-w',\n",
       "  'created': '2023-08-30',\n",
       "  'updated': '',\n",
       "  'authors': ['mingjie qiu', 'zhiyi tan', 'bing-kun bao'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.15840'},\n",
       " {'title': 'linking discrete and continuous models of cell birth and migration',\n",
       "  'id': '2308.16093',\n",
       "  'abstract': 'self-organisation of individuals within large collectives occurs throughout biology. mathematical models can help elucidate the individual-level mechanisms behind these dynamics, but analytical tractability often comes at the cost of biological intuition. discrete models provide straightforward interpretations by tracking each individual yet can be computationally expensive. alternatively, continuous models supply a large-scale perspective by representing the \"effective\" dynamics of infinite agents, but their results are often difficult to translate into experimentally relevant insights. we address this challenge by quantitatively linking spatio-temporal dynamics of continuous models and individual-based data in settings with biologically realistic, time-varying cell numbers. specifically, we introduce and fit scaling parameters in continuous models to account for discrepancies that can arise from low cell numbers and localised interactions. we illustrate our approach on an example motivated by zebrafish-skin pattern formation, in which we create a continuous framework describing the movement and proliferation of a single cell population by upscaling rules from a discrete model. our resulting continuous models accurately depict ensemble average agent-based solutions when migration or proliferation act alone. interestingly, the same parameters are not optimal when both processes act simultaneously, highlighting a rich difference in how combining migration and proliferation affects discrete and continuous dynamics.',\n",
       "  'categories': 'q-bio.cb',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-30',\n",
       "  'updated': '2024-05-07',\n",
       "  'authors': ['w. duncan martinson',\n",
       "   'alexandria volkening',\n",
       "   'markus schmidtchen',\n",
       "   'chandrasekhar venkataraman',\n",
       "   'josé a. carrillo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.16093'},\n",
       " {'title': 'retrobridge: modeling retrosynthesis with markov bridges',\n",
       "  'id': '2308.16212',\n",
       "  'abstract': 'retrosynthesis planning is a fundamental challenge in chemistry which aims at designing reaction pathways from commercially available starting materials to a target molecule. each step in multi-step retrosynthesis planning requires accurate prediction of possible precursor molecules given the target molecule and confidence estimates to guide heuristic search algorithms. we model single-step retrosynthesis planning as a distribution learning problem in a discrete state space. first, we introduce the markov bridge model, a generative framework aimed to approximate the dependency between two intractable discrete distributions accessible via a finite sample of coupled data points. our framework is based on the concept of a markov bridge, a markov process pinned at its endpoints. unlike diffusion-based methods, our markov bridge model does not need a tractable noise distribution as a sampling proxy and directly operates on the input product molecules as samples from the intractable prior distribution. we then address the retrosynthesis planning problem with our novel framework and introduce retrobridge, a template-free retrosynthesis modeling approach that achieves state-of-the-art results on standard evaluation benchmarks.',\n",
       "  'categories': 'q-bio.qm cs.lg q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-30',\n",
       "  'updated': '2024-03-26',\n",
       "  'authors': ['ilia igashov',\n",
       "   'arne schneuing',\n",
       "   'marwin segler',\n",
       "   'michael bronstein',\n",
       "   'bruno correia'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.16212'},\n",
       " {'title': 'identifiable specializations for ode models',\n",
       "  'id': '2308.16273',\n",
       "  'abstract': 'the parameter identifiability problem for a dynamical system is to determine whether the parameters of the system can be found from data for the outputs of the system. verifying whether the parameters are identifiable is a necessary first step before a meaningful parameter estimation can take place. non-identifiability occurs in practical models. to reparametrize a model to achieve identifiability is a challenge. the existing approaches have been shown to be useful for many important examples. however, these approaches are either limited to linear models and scaling parametrizations or are not guaranteed to find a reparametrization even if it exists. in the present paper, we prove that there always exists a locally identifiable model with the same input-output behaviour as the original one obtained from a given one by a partial specialization of the parameters. as an extra feature of our approach, the resulting (at least) locally identifiable reparameterization has the same shape: the monomials in the new state variables in the new model are formed in the same way as in the original model. furthermore, we give a sufficient observability condition for the existence of a state space transformation from the original model to the new one. our proof is constructive and can be translated to an algorithm, which we illustrate by several examples.',\n",
       "  'categories': 'eess.sy cs.sy math.ds q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-30',\n",
       "  'updated': '2024-07-08',\n",
       "  'authors': ['alexey ovchinnikov',\n",
       "   'anand pillay',\n",
       "   'gleb pogudin',\n",
       "   'thomas scanlon'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2308.16273'},\n",
       " {'title': 'approximate bayesian computation for markovian binary trees in   phylogenetics',\n",
       "  'id': '2309.00194',\n",
       "  'abstract': 'phylogenetic trees describe the relationships between species in the evolutionary process, and provide information about the rates of diversification. to understand the mechanisms behind macroevolution, we consider a class of multitype branching processes called markovian binary trees (mbts). mbts allow for trait-based variation in diversification rates, and provide a flexible and realistic probabilistic model for phylogenetic trees. we develop an approximate bayesian computation (abc) scheme to infer the rates of mbt parameters by exploiting the information in the shapes of phylogenetic trees. we evaluate the accuracy of this inference method using simulation studies, and find that our method is able to detect variation in the diversification rates, with accuracy comparable to, and generally better than, likelihood-based methods. in an application to a real-life phylogeny of squamata, we reinforce conclusions drawn from earlier studies, in particular supporting the existence of ovi-/viviparity transitions in both directions. our method demonstrates the potential for more complex models of evolution to be employed in phylogenetic inference, in conjunction with likelihood-free schemes.',\n",
       "  'categories': 'q-bio.pe math.st stat.ap stat.th',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-31',\n",
       "  'updated': '2024-10-03',\n",
       "  'authors': ['mingqi he', 'sophie hautphenne', 'yao-ban chan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.00194'},\n",
       " {'title': 'mathematical models of plasmodium vivax transmission: a scoping review',\n",
       "  'id': '2309.00274',\n",
       "  'abstract': \"plasmodium vivax is one of the most geographically widespread malaria parasites in the world due to its ability to remain dormant in the human liver as hypnozoites and subsequently reactivate after the initial infection (i.e. relapse infections). more than 80% of p. vivax infections are due to hypnozoite reactivation. mathematical modelling approaches have been widely applied to understand p. vivax dynamics and predict the impact of intervention outcomes. in this article, we provide a scoping review of mathematical models that capture p. vivax transmission dynamics published between january 1988 and may 2023 to provide a comprehensive summary of the mathematical models and techniques used to model p. vivax dynamics. we aim to assist researchers working on p. vivax transmission and other aspects of p. vivax malaria by highlighting best practices in currently published models and highlighting where future model development is required. we provide an overview of the different strategies used to incorporate the parasite's biology, use of multiple scales (within-host and population-level), superinfection, immunity, and treatment interventions. in most of the published literature, the rationale for different modelling approaches was driven by the research question at hand. some models focus on the parasites' complicated biology, while others incorporate simplified assumptions to avoid model complexity. overall, the existing literature on mathematical models for p. vivax encompasses various aspects of the parasite's dynamics. we recommend that future research should focus on refining how key aspects of p. vivax dynamics are modelled, including spatial heterogeneity in exposure risk, the accumulation of hypnozoite variation, the interaction between p. falciparum and p. vivax, acquisition of immunity, and recovery under superinfection.\",\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1371/journal.pcbi.1011931',\n",
       "  'created': '2023-09-01',\n",
       "  'updated': '2023-09-24',\n",
       "  'authors': ['md nurul anwar',\n",
       "   'lauren smith',\n",
       "   'angela devine',\n",
       "   'somya mehra',\n",
       "   'camelia r. walker',\n",
       "   'elizabeth ivory',\n",
       "   'eamon conway',\n",
       "   'ivo mueller',\n",
       "   'james m. mccaw',\n",
       "   'jennifer a. flegg',\n",
       "   'roslyn i. hickson'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.00274'},\n",
       " {'title': 'inverse modeling of time-delayed interactions via the dynamic-entropy   formalism',\n",
       "  'id': '2309.01229',\n",
       "  'abstract': 'although instantaneous interactions are unphysical, a large variety of maximum entropy statistical inference methods match the model-inferred and the empirically-measured equal-time correlation functions. focusing on collective motion of active units, this constraint is reasonable when the interaction timescale is much faster than that of the interacting units, as in starling flocks, yet it fails in a number of counter examples, as in leukocyte coordination (where signalling proteins diffuse among two cells). here, we relax this assumption and develop a path integral approach to maximum-entropy framework, which includes delay in signalling. our method is able to infer the strength of couplings and fields, but also the time required by the couplings to completely transfer information among the units. we demonstrate the validity of our approach providing excellent results on synthetic datasets of non-markovian trajectories generated by the heisenberg-kuramoto and vicsek models equipped with delayed interactions. as a proof of concept, we also apply the method to experiments on dendritic migration, where matching equal-time correlations results in a significant information loss.',\n",
       "  'categories': 'physics.bio-ph cond-mat.dis-nn q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-03',\n",
       "  'updated': '2024-07-10',\n",
       "  'authors': ['elena agliari',\n",
       "   'francesco alemanno',\n",
       "   'adriano barra',\n",
       "   'michele castellana',\n",
       "   'daniele lotito',\n",
       "   'matthieu piel'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.01229'},\n",
       " {'title': 'anomaly zones for uniformly sampled gene trees under the gene   duplication and loss model',\n",
       "  'id': '2309.01663',\n",
       "  'abstract': 'recently, there has been interest in extending long-known results about the multispecies coalescent tree to other models of gene trees. results about the gene duplication and loss (gdl) tree have mathematical proofs, including species tree identifiability, estimability, and sample complexity of popular algorithms like astral. here, this work is continued by characterizing the anomaly zones of uniformly sampled gene trees. the anomaly zone for species trees is the set of parameters where some discordant gene tree occurs with the maximal probability. the detection of anomalous gene trees is an important problem in phylogenomics, as their presence renders effective estimation methods to being positively misleading. under the multispecies coalescent, anomaly zones are known to exist for rooted species trees with as few as four species.   the gene duplication and loss process is a generalization of the generalized linear-birth death process to the rooted species tree, where each edge is treated as a single timeline with exponential-rate duplication and loss. the methods and results come from a detailed probabilistic analysis of trajectories observed from this stochastic process. it is shown that anomaly zones do not exist for rooted gdl balanced trees on four species, but do exist for rooted caterpillar trees, as with the multispecies coalescent.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-04',\n",
       "  'updated': '2024-03-28',\n",
       "  'authors': ['brandon legried'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.01663'},\n",
       " {'title': 'extreme first passage times for populations of identical rare events',\n",
       "  'id': '2309.01827',\n",
       "  'abstract': 'a collection of identical and independent rare event first passage times is considered. the problem of finding the fastest out of $n$ such events to occur is called an extreme first passage time. the rare event times are singular and limit to infinity as a positive parameter scaling the noise magnitude is reduced to zero. in contrast, previous work has shown that the mean of the fastest event time goes to zero in the limit of an infinite number of walkers. the combined limit is studied. in particular, the mean time and the most likely path taken by the fastest random walker are investigated. using techniques from large deviation theory, it is shown that there is a distinguished limit where the mean time for the fastest walker can take any positive value, depending on a single proportionality constant. furthermore, it is shown that the mean time and most likely path can be approximated using the solution to a variational problem related to the single-walker rare event.',\n",
       "  'categories': 'physics.bio-ph math.pr q-bio.sc',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-04',\n",
       "  'updated': '2024-04-24',\n",
       "  'authors': ['james maclaurin', 'jay m. newby'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.01827'},\n",
       " {'title': 'information transmission via molecular communication in astrobiological   environments',\n",
       "  'id': '2309.01924',\n",
       "  'abstract': \"the ubiquity of information transmission via molecular communication between cells is comprehensively documented on earth; this phenomenon might even have played a vital role in the origin(s) and early evolution of life. motivated by these considerations, a simple model for molecular communication entailing the diffusion of signaling molecules from transmitter to receiver is elucidated. the channel capacity $c$ (maximal rate of information transmission) and an optimistic heuristic estimate of the actual information transmission rate $\\\\mathcal{i}$ are derived for this communication system; the two quantities, especially the latter, are demonstrated to be broadly consistent with laboratory experiments and more sophisticated theoretical models. the channel capacity exhibits a potentially weak dependence on environmental parameters, whereas the actual information transmission rate may scale with the intercellular distance $d$ as $\\\\mathcal{i} \\\\propto d^{-4}$ and could vary substantially across settings. these two variables are roughly calculated for diverse astrobiological environments, ranging from earth's upper oceans ($c \\\\sim 3.1 \\\\times 10^3$ bits/s; $\\\\mathcal{i} \\\\sim 4.7 \\\\times 10^{-2}$ bits/s) and deep sea hydrothermal vents ($c \\\\sim 4.2 \\\\times 10^3$ bits/s; $\\\\mathcal{i} \\\\sim 1.2 \\\\times 10^{-1}$ bits/s) to the hydrocarbon lakes and seas of titan ($c \\\\sim 3.8 \\\\times 10^3$ bits/s; $\\\\mathcal{i} \\\\sim 2.6 \\\\times 10^{-1}$ bits/s).\",\n",
       "  'categories': 'astro-ph.ep physics.bio-ph physics.flu-dyn q-bio.mn q-bio.pe',\n",
       "  'doi': '10.1089/ast.2023.0069',\n",
       "  'created': '2023-09-04',\n",
       "  'updated': '2023-12-20',\n",
       "  'authors': ['manasvi lingam'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.01924'},\n",
       " {'title': 'the canonical equation of adaptive dynamics in individual-based models   with power law mutation rates',\n",
       "  'id': '2309.02148',\n",
       "  'abstract': \"in this paper, we consider an individual-based model with power law mutation probability. in this setting, we use the large population limit with a subsequent ``small mutations'' limit to derive the canonical equation of adaptive dynamics. for a one-dimensional trait space this corresponds to well established results and we can formulate a criterion for evolutionary branching in the spirit of champagnat and m\\\\'el\\\\'eard (2011). in higher dimensional trait spaces, we find that the speed at which the solution of the canonical equation moves through space is reduced due to mutations being restricted to the underlying grid on the trait space. however, as opposed to the canonical equation with rare mutations, we can explicitly calculate the path which the dominant trait will take without having to solve the equation itself.\",\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-05',\n",
       "  'updated': '2024-02-13',\n",
       "  'authors': ['tobias paul'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.02148'},\n",
       " {'title': 'a novel algebraic approach to time-reversible evolutionary models',\n",
       "  'id': '2309.02276',\n",
       "  'abstract': 'in the last years, algebraic tools have been proven useful in phylogenetic reconstruction and model selection through the study of phylogenetic invariants. however, up to now, the models studied from an algebraic viewpoint are either too general or too restrictive (as group-based models with a uniform stationary distribution) to be used in practice.   in this paper we provide a new framework to study time-reversible models, which are the most widely used by biologists. in our approach we consider algebraic time-reversible models on phylogenetic trees (as defined by allman and rhodes) and introduce a new inner product to make all transition matrices of the process diagonalizable through the same orthogonal eigenbasis. this framework generalizes the fourier transform widely used to work with group-based models and recovers some of the well known results. as illustration, we combine our technique with algebraic geometry tools to provide relevant phylogenetic invariants for trees evolving under the tamura-nei model of nucleotide substitution.',\n",
       "  'categories': 'q-bio.pe math.ag',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-05',\n",
       "  'updated': '2024-04-15',\n",
       "  'authors': ['marta casanellas', 'roser homs pons', 'angélica torres'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.02276'},\n",
       " {'title': 'information processing by neuron populations in the central nervous   system: mathematical structure of data and operations',\n",
       "  'id': '2309.02332',\n",
       "  'abstract': \"in the intricate architecture of the mammalian central nervous system, neurons form populations. axonal bundles communicate between these clusters using spike trains. however, these neuron populations' precise encoding and operations have yet to be discovered. in our analysis, the starting point is a state-of-the-art mechanistic model of a generic neuron endowed with plasticity. from this simple framework emerges a subtle mathematical construct: the representation and manipulation of information can be precisely characterized by an algebra of convex cones. furthermore, these neuron populations are not merely passive transmitters. they act as operators within this algebraic structure, mirroring the functionality of a low-level programming language. when these populations interconnect, they embody succinct yet potent algebraic expressions. these networks allow them to implement many operations, such as specialization, generalization, novelty detection, dimensionality reduction, inverse modeling, prediction, and associative memory. in broader terms, this work illuminates the potential of matrix embeddings in advancing our understanding in fields like cognitive science and ai. these embeddings enhance the capacity for concept processing and hierarchical description over their vector counterparts.\",\n",
       "  'categories': 'q-bio.nc cs.ai cs.lg cs.ne',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-05',\n",
       "  'updated': '2023-12-30',\n",
       "  'authors': ['martin n. p. nilsson'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.02332'},\n",
       " {'title': 'cooling down and waking up: feedback cooling switches an unconscious   neural computer into a conscious quantum computer',\n",
       "  'id': '2309.02708',\n",
       "  'abstract': 'this paper sets out a theory of how feedback cooling in the brain switches on consciousness. it explains how cooling reduces thermal noise to the point where macroscale quantum phenomena - crucially bose-einstein condensation and long-range coherence - can operate at body temperature. it takes the core idea from stapp that mind and brain interact via some sort of oscillator and then focuses on a likely candidate: neuronal arrays identified by stapp as cortical minicolumns. feedback cooling allows amplifiers to act as refrigerators, and when applied to minicolumns it is suggested that the units perform like quantum accelerators, solid-state devices devised to supercharge standard computers. when the accelerator is idle, as in sleep, we have a neural computer operating unconsciously, but when feedback cooling is activated by thalamocortical loops, it produces a bose-einstein condensate, quantum computation, and consciousness. the model explains how macroscale quantum phenomena can operate in a warm and noisy brain, how and why consciousness evolved, and gives insight into puzzling unconscious states like sleepwalking. the model is testable, predicting that cold states in the brain are detectable by magnetic resonance thermometry.',\n",
       "  'categories': 'q-bio.nc physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-21',\n",
       "  'updated': '2024-06-03',\n",
       "  'authors': ['andrew bell'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.02708'},\n",
       " {'title': 'effect of feedback between environment and finite population',\n",
       "  'id': '2309.03093',\n",
       "  'abstract': 'natural selection imply that any organisms including human being will evolve to improve its fitness advantage and the selected genotype or phenotype in equilibrium state will not vary over the time. however, evolutionary process of biological organisms in reality is greatly affected by the environmental change and historical accidents. in this research, we construct a co-evolutionary system to investigate the impact of species-environment feedback. when we talk about an invasion species or mutation, positive feedback is detrimental to the success of the invasion because positive feedback benefits a large number of individuals, whereas negative feedback benefits the invasion because negative feedback disadvantages a large number of individuals. in the case of a competition between two species with initially equal numbers of individuals, both positive and negative feedback will favor the species with low fitness, increasing its chances of taking over the whole population. the reason for this is that feedback allows initially inferior species to have greater fitness than initially dominating species in the early stages, emphasizing the importance of early random accident. our findings emphasize the significance of the evolutionary path driven by species-environment feedback.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1016/j.isci.2024.109055',\n",
       "  'created': '2023-09-05',\n",
       "  'updated': '2023-10-22',\n",
       "  'authors': ['jia-xu han', 'rui-wu wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.03093'},\n",
       " {'title': 'signatures of bayesian inference emerge from energy efficient synapses',\n",
       "  'id': '2309.03194',\n",
       "  'abstract': 'biological synaptic transmission is unreliable, and this unreliability likely degrades neural circuit performance. while there are biophysical mechanisms that can increase reliability, for instance by increasing vesicle release probability, these mechanisms cost energy. we examined four such mechanisms along with the associated scaling of the energetic costs. we then embedded these energetic costs for reliability in artificial neural networks (ann) with trainable stochastic synapses, and trained these networks on standard image classification tasks. the resulting networks revealed a tradeoff between circuit performance and the energetic cost of synaptic reliability. additionally, the optimised networks exhibited two testable predictions consistent with pre-existing experimental data. specifically, synapses with lower variability tended to have 1) higher input firing rates and 2) lower learning rates. surprisingly, these predictions also arise when synapse statistics are inferred through bayesian inference. indeed, we were able to find a formal, theoretical link between the performance-reliability cost tradeoff and bayesian inference. this connection suggests two incompatible possibilities: evolution may have chanced upon a scheme for implementing bayesian inference by optimising energy efficiency, or alternatively, energy efficient synapses may display signatures of bayesian inference without actually using bayes to reason about uncertainty.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-06',\n",
       "  'updated': '2024-07-01',\n",
       "  'authors': ['james malkin',\n",
       "   \"cian o'donnell\",\n",
       "   'conor houghton',\n",
       "   'laurence aitchison'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.03194'},\n",
       " {'title': 'learning a patent-informed biomedical knowledge graph reveals   technological potential of drug repositioning candidates',\n",
       "  'id': '2309.03227',\n",
       "  'abstract': \"drug repositioning-a promising strategy for discovering new therapeutic uses for existing drugs-has been increasingly explored in the computational science literature using biomedical databases. however, the technological potential of drug repositioning candidates has often been overlooked. this study presents a novel protocol to comprehensively analyse various sources such as pharmaceutical patents and biomedical databases, and identify drug repositioning candidates with both technological potential and scientific evidence. to this end, first, we constructed a scientific biomedical knowledge graph (s-bkg) comprising relationships between drugs, diseases, and genes derived from biomedical databases. our protocol involves identifying drugs that exhibit limited association with the target disease but are closely located in the s-bkg, as potential drug candidates. we constructed a patent-informed biomedical knowledge graph (p-bkg) by adding pharmaceutical patent information. finally, we developed a graph embedding protocol to ascertain the structure of the p-bkg, thereby calculating the relevance scores of those candidates with target disease-related patents to evaluate their technological potential. our case study on alzheimer's disease demonstrates its efficacy and feasibility, while the quantitative outcomes and systematic methods are expected to bridge the gap between computational discoveries and successful market applications in drug repositioning research.\",\n",
       "  'categories': 'cs.ai cs.cl cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-03',\n",
       "  'updated': '2024-07-24',\n",
       "  'authors': ['yongseung jegal',\n",
       "   'jaewoong choi',\n",
       "   'jiho lee',\n",
       "   'ki-su park',\n",
       "   'seyoung lee',\n",
       "   'janghyeok yoon'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.03227'},\n",
       " {'title': 'exponential cell division and allometric scaling in metabolic ecology',\n",
       "  'id': '2309.03425',\n",
       "  'abstract': \"one of the most fundamental rules in metabolic ecology is the allometric equation, which is a power-law scaling that describes the connection between body measurements and body size. the biological dynamics of this essentially empirical allometric equation, however, have yet to be properly addressed in cell level. in order to fill the gap between biological process in cell level and allometric scaling in metabolic ecology, we simply assumed a cell bipartition without limitation, and then exponential cells increased during their lifetime. two synchronous exponential increasing could generate a power-law scaling between body mass and an organ's weight. and the power-law scaling between body mass and metabolic rate may also be obtained by substituting an organ's weight with the weight of erythrocytes. based on the same assumption, the dynamic of cell proliferation reveal a complex exponential scaling between body mass and longevity rather than the previously reported power-law scaling. in other words, there is a quadratic relationship between longevity and logarithmic form of body mass. in these relationships, all parameters can be explained by indices in cell division and embryo.\",\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '10.1016/j.biosystems.2024.10519',\n",
       "  'created': '2023-09-06',\n",
       "  'updated': '',\n",
       "  'authors': ['jia-xu han', 'zhuangdong bai', 'rui-wu wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.03425'},\n",
       " {'title': 'insights into the inner workings of transformer models for protein   function prediction',\n",
       "  'id': '2309.03631',\n",
       "  'abstract': 'motivation: we explored how explainable artificial intelligence (xai) can help to shed light into the inner workings of neural networks for protein function prediction, by extending the widely used xai method of integrated gradients such that latent representations inside of transformer models, which were finetuned to gene ontology term and enzyme commission number prediction, can be inspected too. results: the approach enabled us to identify amino acids in the sequences that the transformers pay particular attention to, and to show that these relevant sequence parts reflect expectations from biology and chemistry, both in the embedding layer and inside of the model, where we identified transformer heads with a statistically significant correspondence of attribution maps with ground truth sequence annotations (e.g. transmembrane regions, active sites) across many proteins. availability and implementation: source code can be accessed at https://github.com/markuswenzel/xai-proteins .',\n",
       "  'categories': 'cs.lg q-bio.bm',\n",
       "  'doi': '10.1093/bioinformatics/btae031',\n",
       "  'created': '2023-09-07',\n",
       "  'updated': '2024-02-09',\n",
       "  'authors': ['markus wenzel', 'erik grüner', 'nils strodthoff'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.03631'},\n",
       " {'title': 'analyzing and comparing omicron lineage variants protein-protein   interaction network using centrality measure',\n",
       "  'id': '2309.03910',\n",
       "  'abstract': \"the worldwide spread of the omicron lineage variants has now been confirmed. it is crucial to understand the process of cellular life and to discover new drugs need to identify the important proteins in a protein interaction network (ppin). ppins are often represented by graphs in bioinformatics, which describe cell processes. there are some proteins that have significant influences on these tissues, and which play a crucial role in regulating them. the discovery of new drugs is aided by the study of significant proteins. these significant proteins can be found by reducing the graph and using graph analysis. studies examining protein interactions in the omicron lineage (b.1.1.529) and its variants (ba.5, ba.4, ba.3, ba.2, ba.1.1, ba.1) are not yet available. studying omicron has been intended to find a significant protein. 68 nodes represent 68 proteins and 52 edges represent the relationship among the protein in the network. a few entrality measures are computed namely page rank centrality (prc), degree centrality (dc), closeness centrality (cc), and betweenness centrality (bc) together with node degree and local clustering co-efficient (lcc). we also discover 18 network clusters using markov clustering. 8 significant proteins (candidate gene of omicron lineage variants) were detected among the 68 proteins, including ahsg, kcnk1, kcnq1, mapt, nr1h4, psmc2, ptpn11 and, ube21 which scored the highest among the omicron proteins. it is found that in the variant of omicron protein-protein interaction networks, the mapt protein's impact is the most significant.\",\n",
       "  'categories': 'q-bio.mn',\n",
       "  'doi': '10.1007/s42979-023-01685-5',\n",
       "  'created': '2023-08-09',\n",
       "  'updated': '',\n",
       "  'authors': ['mamata das', 'selvakumar k.', 'p. j. a. alphonse'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.03910'},\n",
       " {'title': 'evaluation of large language models for discovery of gene set function',\n",
       "  'id': '2309.04019',\n",
       "  'abstract': \"gene set analysis is a mainstay of functional genomics, but it relies on curated databases of gene functions that are incomplete. here we evaluate five large language models (llms) for their ability to discover the common biological functions represented by a gene set, substantiated by supporting rationale, citations and a confidence assessment. benchmarking against canonical gene sets from the gene ontology, gpt-4 confidently recovered the curated name or a more general concept (73% of cases), while benchmarking against random gene sets correctly yielded zero confidence. gemini-pro and mixtral-instruct showed ability in naming but were falsely confident for random sets, whereas llama2-70b had poor performance overall. in gene sets derived from 'omics data, gpt-4 identified novel functions not reported by classical functional enrichment (32% of cases), which independent review indicated were largely verifiable and not hallucinations. the ability to rapidly synthesize common gene functions positions llms as valuable 'omics assistants.\",\n",
       "  'categories': 'q-bio.gn cs.ai cs.cl q-bio.mn',\n",
       "  'doi': '10.1038/s41592-024-02525-x',\n",
       "  'created': '2023-09-07',\n",
       "  'updated': '2024-04-01',\n",
       "  'authors': ['mengzhou hu',\n",
       "   'sahar alkhairy',\n",
       "   'ingoo lee',\n",
       "   'rudolf t. pillich',\n",
       "   'dylan fong',\n",
       "   'kevin smith',\n",
       "   'robin bachelder',\n",
       "   'trey ideker',\n",
       "   'dexter pratt'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.04019'},\n",
       " {'title': 'meta predictive learning model of languages in neural circuits',\n",
       "  'id': '2309.04106',\n",
       "  'abstract': 'large language models based on self-attention mechanisms have achieved astonishing performances not only in natural language itself, but also in a variety of tasks of different nature. however, regarding processing language, our human brain may not operate using the same principle. then, a debate is established on the connection between brain computation and artificial self-supervision adopted in large language models. one of most influential hypothesis in brain computation is the predictive coding framework, which proposes to minimize the prediction error by local learning. however, the role of predictive coding and the associated credit assignment in language processing remains unknown. here, we propose a mean-field learning model within the predictive coding framework, assuming that the synaptic weight of each connection follows a spike and slab distribution, and only the distribution, rather than specific weights, is trained. this meta predictive learning is successfully validated on classifying handwritten digits where pixels are input to the network in sequence, and moreover on the toy and real language corpus. our model reveals that most of the connections become deterministic after learning, while the output connections have a higher level of variability. the performance of the resulting network ensemble changes continuously with data load, further improving with more training data, in analogy with the emergent behavior of large language models. therefore, our model provides a starting point to investigate the connection among brain computation, next-token prediction and general intelligence.',\n",
       "  'categories': 'cs.cl q-bio.nc',\n",
       "  'doi': '10.1103/physreve.109.044309',\n",
       "  'created': '2023-09-07',\n",
       "  'updated': '2023-10-09',\n",
       "  'authors': ['chan li', 'junbin qiu', 'haiping huang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.04106'},\n",
       " {'title': 'segmentanything helps microscopy images based automatic and quantitative   organoid detection and analysis',\n",
       "  'id': '2309.04190',\n",
       "  'abstract': 'organoids are self-organized 3d cell clusters that closely mimic the architecture and function of in vivo tissues and organs. quantification of organoid morphology helps in studying organ development, drug discovery, and toxicity assessment. recent microscopy techniques provide a potent tool to acquire organoid morphology features, but manual image analysis remains a labor and time-intensive process. thus, this paper proposes a comprehensive pipeline for microscopy analysis that leverages the segmentanything to precisely demarcate individual organoids. additionally, we introduce a set of morphological properties, including perimeter, area, radius, non-smoothness, and non-circularity, allowing researchers to analyze the organoid structures quantitatively and automatically. to validate the effectiveness of our approach, we conducted tests on bright-field images of human induced pluripotent stem cells (ipscs) derived neural-epithelial (ne) organoids. the results obtained from our automatic pipeline closely align with manual organoid detection and measurement, showcasing the capability of our proposed method in accelerating organoids morphology analysis.',\n",
       "  'categories': 'eess.iv cs.cv q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-08',\n",
       "  'updated': '2024-04-08',\n",
       "  'authors': ['xiaodan xing',\n",
       "   'chunling tang',\n",
       "   'yunzhe guo',\n",
       "   'nicholas kurniawan',\n",
       "   'guang yang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.04190'},\n",
       " {'title': 'vis-split: interactive hierarchical modeling for mrna expression   classification',\n",
       "  'id': '2309.04423',\n",
       "  'abstract': 'we propose an interactive visual analytics tool, vis-split, for partitioning a population of individuals into groups with similar gene signatures. vis-split allows users to interactively explore a dataset and exploit visual separations to build a classification model for specific cancers. the visualization components reveal gene expression and correlation to assist specific partitioning decisions, while also providing overviews for the decision model and clustered genetic signatures. we demonstrate the effectiveness of our framework through a case study and evaluate its usability with domain experts. our results show that vis-split can classify patients based on their genetic signatures to effectively gain insights into rna sequencing data, as compared to an existing classification system.',\n",
       "  'categories': 'cs.hc q-bio.qm',\n",
       "  'doi': '10.1109/vis54172.2023.00030',\n",
       "  'created': '2023-09-08',\n",
       "  'updated': '',\n",
       "  'authors': ['braden roper',\n",
       "   'james c. mathews',\n",
       "   'saad nadeem',\n",
       "   'ji hwan park'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.04423'},\n",
       " {'title': 'ormir_xct: a python package for high resolution peripheral quantitative   computed tomography image processing',\n",
       "  'id': '2309.04602',\n",
       "  'abstract': 'high resolution peripheral quantitative computed tomography (hr-pqct) is an imaging technique capable of imaging trabecular bone in-vivo. hr-pqct has a wide range of applications, primarily focused on bone to improve our understanding of musculoskeletal diseases, assess epidemiological associations, and evaluate the effects of pharmaceutical interventions. processing hr-pqct images has largely been supported using the scanner manufacturer scripting language (image processing language, ipl, scanco medical). however, by expanding image processing workflows outside of the scanner manufacturer software environment, users have the flexibility to apply more advanced mathematical techniques and leverage modern software packages to improve image processing. the ormir_xct python package was developed to reimplement some existing ipl workflows and provide an open and reproducible package allowing for the development of advanced hr-pqct data processing workflows.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.21105/joss.06084',\n",
       "  'created': '2023-09-08',\n",
       "  'updated': '',\n",
       "  'authors': ['michael t. kuczynski',\n",
       "   'nathan j. neeteson',\n",
       "   'kathryn s. stok',\n",
       "   'andrew j. burghardt',\n",
       "   'michelle a. espinosa hernandez',\n",
       "   'jared vicory',\n",
       "   'justin j. tse',\n",
       "   'pholpat durongbhan',\n",
       "   'serena bonaretti',\n",
       "   'andy kin on wong',\n",
       "   'steven k. boyd',\n",
       "   'sarah l. manske'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.04602'},\n",
       " {'title': 'open and reusable deep learning for pathology with wsinfer and qupath',\n",
       "  'id': '2309.04631',\n",
       "  'abstract': 'the field of digital pathology has seen a proliferation of deep learning models in recent years. despite substantial progress, it remains rare for other researchers and pathologists to be able to access models published in the literature and apply them to their own images. this is due to difficulties in both sharing and running models. to address these concerns, we introduce wsinfer: a new, open-source software ecosystem designed to make deep learning for pathology more streamlined and accessible. wsinfer comprises three main elements: 1) a python package and command line tool to efficiently apply patch-based deep learning inference to whole slide images; 2) a qupath extension that provides an alternative inference engine through user-friendly and interactive software, and 3) a model zoo, which enables pathology models and metadata to be easily shared in a standardized form. together, these contributions aim to encourage wider reuse, exploration, and interrogation of deep learning models for research purposes, by putting them into the hands of pathologists and eliminating a need for coding experience when accessed through qupath. the wsinfer source code is hosted on github and documentation is available at https://wsinfer.readthedocs.io.',\n",
       "  'categories': 'q-bio.to cs.cv',\n",
       "  'doi': '10.1038/s41698-024-00499-9',\n",
       "  'created': '2023-09-08',\n",
       "  'updated': '',\n",
       "  'authors': ['jakub r. kaczmarzyk',\n",
       "   \"alan o'callaghan\",\n",
       "   'fiona inglis',\n",
       "   'tahsin kurc',\n",
       "   'rajarsi gupta',\n",
       "   'erich bremer',\n",
       "   'peter bankhead',\n",
       "   'joel h. saltz'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.04631'},\n",
       " {'title': 'matlab plasmonic nanoparticle virion counting and interpretation system   in urban populations',\n",
       "  'id': '2309.04924',\n",
       "  'abstract': \"one of the biggest issues currently plaguing the field of medicine is the lack of an accurate and efficient form of disease diagnosis especially in urban settings such as major cities. for example, the two most commonly utilized test diagnosis systems, the pcr and rapid test, sacrifice either accuracy or speed to achieve the other, and this could slow down epidemiologists working to combat the spread. another issue currently present is the issue of viral quantification or the counting of virions within a nasal sample. these can provide doctors with crucial information in treating infections; however, the current mediums are underdeveloped and unstandardized. this project's goals were to 1) create an accurate and rapid rsv diagnostic test that could be replicated and utilized efficiently in urban settings and 2) design a viral quantification mechanism that counts the number of virions to provide more information to healthcare workers. this diagnostic test involved a system that pumped rsv-aggregated au-nanoparticles and unaggregated au-nanoparticles through a microcapillary, whose cross-section was intersected by two laser beams generating and detecting the nanobubbles. the signals between the unaggregated and aggregated nanobubbles were calibrated, and the number of rsv virions was recorded. the results yielded an accuracy of 99.99% and an average time of 5.2 minutes, validating that this design is both faster and more accurate compared to current tests. when cross-validated with poisson statistics, the virion counting system counted the number of virions with 98.52% accuracy. to verify the accuracy of our samples, the results were compared to clinical trials of nasal samples, and our diagnostic system predicted accurate diagnostics after statistical analysis. with further testing, this diagnostic method could replace current standards of testing, saving millions of lives every year.\",\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-09',\n",
       "  'updated': '2024-01-08',\n",
       "  'authors': ['bryan hong', 'jai pal'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.04924'},\n",
       " {'title': 'reconstructing the kinetic chemotaxis kernel using macroscopic data:   well-posedness and ill-posedness',\n",
       "  'id': '2309.05004',\n",
       "  'abstract': 'bacterial motion is steered by external stimuli (chemotaxis), and the motion described on the mesoscopic scale is uniquely determined by a parameter $k$ that models velocity change response from the bacteria. this parameter is called chemotaxis kernel. in a practical setting, it is inferred by experimental data. we deploy a pde-constrained optimization framework to perform this reconstruction using velocity-averaged, localized data taken in the interior of the domain. the problem can be well-posed or ill-posed depending on the data preparation and the experimental setup. in particular, we propose one specific design that guarantees numerical reconstructability and local convergence. this design is adapted to the discretization of $k$ in space and decouples the reconstruction of local values of $k$ into smaller cell problems, opening up parallelization opportunities. numerical evidences support the theoretical findings.',\n",
       "  'categories': 'math.na cs.na math.ap math.oc q-bio.cb',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-10',\n",
       "  'updated': '2024-11-22',\n",
       "  'authors': ['kathrin hellmuth',\n",
       "   'christian klingenberg',\n",
       "   'qin li',\n",
       "   'min tang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.05004'},\n",
       " {'title': 'is learning in biological neural networks based on stochastic gradient   descent? an analysis using stochastic processes',\n",
       "  'id': '2309.05102',\n",
       "  'abstract': 'in recent years, there has been an intense debate about how learning in biological neural networks (bnns) differs from learning in artificial neural networks. it is often argued that the updating of connections in the brain relies only on local information, and therefore a stochastic gradient-descent type optimization method cannot be used. in this paper, we study a stochastic model for supervised learning in bnns. we show that a (continuous) gradient step occurs approximately when each learning opportunity is processed by many local updates. this result suggests that stochastic gradient descent may indeed play a role in optimizing bnns.',\n",
       "  'categories': 'q-bio.nc cs.lg cs.ne math.pr',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-10',\n",
       "  'updated': '2024-04-10',\n",
       "  'authors': ['sören christensen', 'jan kallsen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.05102'},\n",
       " {'title': 'ecg-based estimation of respiratory modulation of av nodal conduction   during atrial fibrillation',\n",
       "  'id': '2309.05458',\n",
       "  'abstract': 'information about autonomic nervous system (ans) activity may be valuable for personalized atrial fibrillation (af) treatment but is not easily accessible from the ecg. in this study, we propose a new approach for ecg-based assessment of respiratory modulation in av nodal refractory period and conduction delay. a 1-dimensional convolutional neural network (1d-cnn) was trained to estimate respiratory modulation of av nodal conduction properties from 1-minute segments of rr series, respiration signals, and atrial fibrillatory rates (afr) using synthetic data that replicates clinical ecg-derived data. the synthetic data were generated using a network model of the av node and 4 million unique model parameter sets. the 1d-cnn was then used to analyze respiratory modulation in clinical deep breathing test data of 28 patients in af, where a ecg-derived respiration signal was extracted using a novel approach based on periodic component analysis. we demonstrated using synthetic data that the 1d-cnn can predict the respiratory modulation from rr series alone ($\\\\rho$ = 0.805) and that the addition of either respiration signal ($\\\\rho$ = 0.830), afr ($\\\\rho$ = 0.837), or both ($\\\\rho$ = 0.855) improves the prediction. results from analysis of clinical ecg data of 20 patients with sufficient signal quality suggest that respiratory modulation decreased in response to deep breathing for five patients, increased for five patients, and remained similar for ten patients, indicating a large inter-patient variability.',\n",
       "  'categories': 'physics.med-ph eess.sp q-bio.to',\n",
       "  'doi': '10.3389/fphys.2024.1281343',\n",
       "  'created': '2023-09-11',\n",
       "  'updated': '',\n",
       "  'authors': ['felix plappert',\n",
       "   'gunnar engström',\n",
       "   'pyotr g. platonov',\n",
       "   'mikael wallman',\n",
       "   'frida sandberg'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.05458'},\n",
       " {'title': \"maxwell's current in mitochondria and nerve\",\n",
       "  'id': '2309.05667',\n",
       "  'abstract': 'maxwell defined a true or total current in a way not widely used today. he said \"... true electric current ... is not the same thing as the current of conduction but that the time-variation of the electric displacement must be taken into account in estimating the total movement of electricity\". we show that true or total current is a universal property of electrodynamics independent of properties of matter. we use mathematics without a dielectric constant. the resulting maxwell current law is a generalization of the kirchhoff law of current used in circuit analysis, that also includes displacement current. the generalization is not a long-time low frequency approximation in contrast to traditional presentation of kirchhoff\\'s law. the maxwell current law does not require currents to be in circuits. it has been applied to three dimensional systems like the signaling system of nerve and muscle fibers. the maxwell current law clarifies the flow of electrons, protons, and ions in mitochondria that generate atp, the molecule that stores chemical energy throughout life. the currents are globally coupled because mitochondria are short. focusing on maxwell current reinterprets the classical chemiosmotic hypothesis of atp production. the conduction current of protons in mitochondria is driven by the protonmotive force including its component electrical potential, just as in the classical chemiosmotic hypothesis. the electrical potential is now the electrical potential as defined in physical sciences by maxwell partial differential equations. the conduction current is now just a part of the true current analyzed by maxwell. details of accumulation of charges do not have to be considered in analysis of true current because true current does not accumulate. it is true total current that provides the energy that generates the atp, not just the protonmotive force.',\n",
       "  'categories': 'physics.bio-ph physics.class-ph physics.hist-ph q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-08-30',\n",
       "  'updated': '2024-01-02',\n",
       "  'authors': ['robert s. eisenberg'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.05667'},\n",
       " {'title': 'the past, present, and future of the brain imaging data structure (bids)',\n",
       "  'id': '2309.05768',\n",
       "  'abstract': 'the brain imaging data structure (bids) is a community-driven standard for the organization of data and metadata from a growing range of neuroscience modalities. this paper is meant as a history of how the standard has developed and grown over time. we outline the principles behind the project, the mechanisms by which it has been extended, and some of the challenges being addressed as it evolves. we also discuss the lessons learned through the project, with the aim of enabling researchers in other domains to learn from the success of bids.',\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '10.1162/imag_a_00103',\n",
       "  'created': '2023-09-11',\n",
       "  'updated': '2024-01-08',\n",
       "  'authors': ['russell a. poldrack',\n",
       "   'christopher j. markiewicz',\n",
       "   'stefan appelhoff',\n",
       "   'yoni k. ashar',\n",
       "   'tibor auer',\n",
       "   'sylvain baillet',\n",
       "   'shashank bansal',\n",
       "   'leandro beltrachini',\n",
       "   'christian g. benar',\n",
       "   'giacomo bertazzoli',\n",
       "   'suyash bhogawar',\n",
       "   'ross w. blair',\n",
       "   'marta bortoletto',\n",
       "   'mathieu boudreau',\n",
       "   'teon l. brooks',\n",
       "   'vince d. calhoun',\n",
       "   'filippo maria castelli',\n",
       "   'patricia clement',\n",
       "   'alexander l cohen',\n",
       "   'julien cohen-adad',\n",
       "   \"sasha d'ambrosio\",\n",
       "   'gilles de hollander',\n",
       "   'maría de la iglesia-vayá',\n",
       "   'alejandro de la vega',\n",
       "   'arnaud delorme',\n",
       "   'orrin devinsky',\n",
       "   'dejan draschkow',\n",
       "   'eugene paul duff',\n",
       "   'elizabeth dupre',\n",
       "   'eric earl',\n",
       "   'oscar esteban',\n",
       "   'franklin w. feingold',\n",
       "   'guillaume flandin',\n",
       "   'anthony galassi',\n",
       "   'giuseppe gallitto',\n",
       "   'melanie ganz',\n",
       "   'rémi gau',\n",
       "   'james gholam',\n",
       "   'satrajit s. ghosh',\n",
       "   'alessio giacomel',\n",
       "   'ashley g gillman',\n",
       "   'padraig gleeson',\n",
       "   'alexandre gramfort',\n",
       "   'samuel guay',\n",
       "   'giacomo guidali',\n",
       "   'yaroslav o. halchenko',\n",
       "   'daniel a. handwerker',\n",
       "   'nell hardcastle',\n",
       "   'peer herholz',\n",
       "   'dora hermes',\n",
       "   'christopher j. honey',\n",
       "   'robert b. innis',\n",
       "   'horea-ioan ioanas',\n",
       "   'andrew jahn',\n",
       "   'agah karakuzu',\n",
       "   'david b. keator',\n",
       "   'gregory kiar',\n",
       "   'balint kincses',\n",
       "   'angela r. laird',\n",
       "   'jonathan c. lau',\n",
       "   'alberto lazari',\n",
       "   'jon haitz legarreta',\n",
       "   'adam li',\n",
       "   'xiangrui li',\n",
       "   'bradley c. love',\n",
       "   'hanzhang lu',\n",
       "   'camille maumet',\n",
       "   'giacomo mazzamuto',\n",
       "   'steven l. meisler',\n",
       "   'mark mikkelsen',\n",
       "   'henk mutsaerts',\n",
       "   'thomas e. nichols',\n",
       "   'aki nikolaidis',\n",
       "   'gustav nilsonne',\n",
       "   'guiomar niso',\n",
       "   'martin norgaard',\n",
       "   'thomas w okell',\n",
       "   'robert oostenveld',\n",
       "   'eduard ort',\n",
       "   'patrick j. park',\n",
       "   'mateusz pawlik',\n",
       "   'cyril r. pernet',\n",
       "   'franco pestilli',\n",
       "   'jan petr',\n",
       "   'christophe phillips',\n",
       "   'jean-baptiste poline',\n",
       "   'luca pollonini',\n",
       "   'pradeep reddy raamana',\n",
       "   'petra ritter',\n",
       "   'gaia rizzo',\n",
       "   'kay a. robbins',\n",
       "   'alexander p. rockhill',\n",
       "   'christine rogers',\n",
       "   'ariel rokem',\n",
       "   'chris rorden',\n",
       "   'alexandre routier',\n",
       "   'jose manuel saborit-torres',\n",
       "   'taylor salo',\n",
       "   'michael schirner',\n",
       "   'robert e. smith',\n",
       "   'tamas spisak',\n",
       "   'julia sprenger',\n",
       "   'nicole c. swann',\n",
       "   'martin szinte',\n",
       "   'sylvain takerkart',\n",
       "   'bertrand thirion',\n",
       "   'adam g. thomas',\n",
       "   'sajjad torabian',\n",
       "   'gael varoquaux',\n",
       "   'bradley voytek',\n",
       "   'julius welzel',\n",
       "   'martin wilson',\n",
       "   'tal yarkoni',\n",
       "   'krzysztof j. gorgolewski'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.05768'},\n",
       " {'title': 'rawhash2: mapping raw nanopore signals using hash-based seeding and   adaptive quantization',\n",
       "  'id': '2309.05771',\n",
       "  'abstract': 'summary: raw nanopore signals can be analyzed while they are being generated, a process known as real-time analysis. real-time analysis of raw signals is essential to utilize the unique features that nanopore sequencing provides, enabling the early stopping of the sequencing of a read or the entire sequencing run based on the analysis. the state-of-the-art mechanism, rawhash, offers the first hash-based efficient and accurate similarity identification between raw signals and a reference genome by quickly matching their hash values. in this work, we introduce rawhash2, which provides major improvements over rawhash, including a more sensitive quantization and chaining implementation, weighted mapping decisions, frequency filters to reduce ambiguous seed hits, minimizers for hash-based sketching, and support for the r10.4 flow cell version and various data formats such as pod5 and slow5. compared to rawhash, rawhash2 provides better f1 accuracy (on average by 10.57% and up to 20.25%) and better throughput (on average by 4.0x and up to 9.9x) than rawhash. availability and implementation: rawhash2 is available at https://github.com/cmu-safari/rawhash. we also provide the scripts to fully reproduce our results on our github page.',\n",
       "  'categories': 'q-bio.gn q-bio.qm',\n",
       "  'doi': '10.1093/bioinformatics/btae478',\n",
       "  'created': '2023-09-11',\n",
       "  'updated': '2024-08-13',\n",
       "  'authors': ['can firtina', 'melina soysal', 'joël lindegger', 'onur mutlu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.05771'},\n",
       " {'title': 'chemspaceal: an efficient active learning methodology applied to   protein-specific molecular generation',\n",
       "  'id': '2309.05853',\n",
       "  'abstract': 'the incredible capabilities of generative artificial intelligence models have inevitably led to their application in the domain of drug discovery. within this domain, the vastness of chemical space motivates the development of more efficient methods for identifying regions with molecules that exhibit desired characteristics. in this work, we present a computationally efficient active learning methodology that requires evaluation of only a subset of the generated data in the constructed sample space to successfully align a generative model with respect to a specified objective. we demonstrate the applicability of this methodology to targeted molecular generation by fine-tuning a gpt-based molecular generator toward a protein with fda-approved small-molecule inhibitors, c-abl kinase. remarkably, the model learns to generate molecules similar to the inhibitors without prior knowledge of their existence, and even reproduces two of them exactly. we also show that the methodology is effective for a protein without any commercially available small-molecule inhibitors, the hnh domain of the crispr-associated protein 9 (cas9) enzyme. we believe that the inherent generality of this method ensures that it will remain applicable as the exciting field of in silico molecular generation evolves. to facilitate implementation and reproducibility, we have made all of our software available through the open-source chemspaceal python package.',\n",
       "  'categories': 'cs.lg q-bio.bm',\n",
       "  'doi': '10.1021/acs.jcim.3c01456',\n",
       "  'created': '2023-09-11',\n",
       "  'updated': '2023-12-03',\n",
       "  'authors': ['gregory w. kyro',\n",
       "   'anton morgunov',\n",
       "   'rafael i. brent',\n",
       "   'victor s. batista'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.05853'},\n",
       " {'title': \"l\\\\'evy distributed fluctuations in the living cell cortex\",\n",
       "  'id': '2309.06226',\n",
       "  'abstract': \"the actomyosin cortex is an active material that provides animal cells with a strong but flexible exterior, whose mechanics, including non-gaussian fluctuations and occasional large displacements or cytoquakes, have defied explanation. we study the active fluctuations of the cortex using nanoscale tracking of arrays of flexible microposts adhered to multiple cultured cell types. when the confounding effects of static heterogeneity and tracking error are removed, the fluctuations are found to be heavy-tailed and well-described by a truncated l\\\\'evy alpha-stable distribution over a wide range of timescales, in multiple cell types. the largest random displacements closely resemble the earlier-reported cytoquakes, but notably, we find these cytoquakes are not due to earthquake-like cooperative rearrangement of many cytoskeletal elements. rather, they are indistinguishable from chance large excursions of a super-diffusive random process driven by heavy-tailed noise. the non-cooperative microscopic events driving these fluctuations need not be larger than the expected elastic energy of single tensed cortical actin filaments, and the implied distribution of microscopic event energies will need to be accounted for by future models of the cytoskeleton.\",\n",
       "  'categories': 'physics.bio-ph cond-mat.soft q-bio.sc',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-12',\n",
       "  'updated': '2024-11-07',\n",
       "  'authors': ['shankar sivarajan',\n",
       "   'yu shi',\n",
       "   'katherine m. xiang',\n",
       "   'clary rodríguez-cruz',\n",
       "   'christopher l. porter',\n",
       "   'geran m. kostecki',\n",
       "   'leslie tung',\n",
       "   'john c. crocker',\n",
       "   'daniel h. reich'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.06226'},\n",
       " {'title': 'reservoir computing model for multi-electrode electrophysiological data   analysis',\n",
       "  'id': '2309.06297',\n",
       "  'abstract': 'in this paper we present a computational model which decodes the spatio-temporal data from electro-physiological measurements of neuronal networks and reconstructs the network structure on a macroscopic domain, representing the connectivity between neuronal units. the model is based on reservoir computing network (rcn) approach, where experimental data is used as training and validation data. consequently, the model can be used to study the functionality of different neuronal cultures and simulate the network response to external stimuli.',\n",
       "  'categories': 'q-bio.qm cs.et physics.bio-ph q-bio.nc',\n",
       "  'doi': '10.1109/cibcb56990.2023.10264895',\n",
       "  'created': '2023-09-12',\n",
       "  'updated': '',\n",
       "  'authors': ['ilya auslender', 'lorenzo pavesi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.06297'},\n",
       " {'title': 'social \\\\textit{vs.} individual age-dependent costs of imperfect   vaccination',\n",
       "  'id': '2309.06336',\n",
       "  'abstract': \"in diseases with long-term immunity, vaccination is known to increase the average age at infection as a result of the decrease in the pathogen circulation. this implies that a vaccination campaign can have negative effects when a disease is more costly (financial or health-related costs) for higher ages. this work considers an age-structured population transmission model with imperfect vaccination. we aim to compare the social and individual costs of vaccination, assuming that disease costs are age-dependent, while the disease's dynamic is age-independent. a model coupling pathogen deterministic dynamics for a population consisting of juveniles and adults, assumed to be rational agents, is introduced. the parameter region for which vaccination has a positive social impact is fully characterized and the nash equilibrium of the vaccination game is obtained. finally, collective strategies designed to promote voluntary vaccination, without compromising social welfare, are discussed.\",\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-12',\n",
       "  'updated': '2024-06-03',\n",
       "  'authors': ['fabio a. c. c. chalub',\n",
       "   'paulo doutor',\n",
       "   'paula patrício',\n",
       "   'maria do céu soares'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.06336'},\n",
       " {'title': 'single-cell mutational burden distributions in birth-death processes',\n",
       "  'id': '2309.06355',\n",
       "  'abstract': 'genetic mutations are footprints of tumour growth. while mutation data in bulk samples has been used to infer evolutionary parameters hard to measure in vivo, the advent of single-cell data has led to strong interest in the mutational burden distribution (mbd) among tumour cells. we introduce dynamical matrices and recurrence relations to integrate this single-cell mbd with known statistics, and derive new analytical expressions. surprisingly, we find that the shape of the mbd is driven by cell lineage-level stochasticity rather than by the distribution of mutations in each cell division.',\n",
       "  'categories': 'q-bio.pe math.pr',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-12',\n",
       "  'updated': '2024-06-07',\n",
       "  'authors': ['christo morison', 'dudley stark', 'weini huang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.06355'},\n",
       " {'title': 'evolution of trust in structured populations',\n",
       "  'id': '2309.06636',\n",
       "  'abstract': 'the trust game, derived from an economics experiment, has recently attracted interest in the field of evolutionary dynamics. in a recent version of the evolutionary trust game, players adopt one of three strategies: investor, trustworthy trustee, or untrustworthy trustee. trustworthy trustees enhance and share the investment with the investor, whereas untrustworthy trustees retain the full amount, betraying the investor. following this setup, we investigate a two-player trust game, which is analytically feasible under weak selection. we explore the evolution of trust in structured populations, factoring in four strategy updating rules: pairwise comparison (pc), birth-death (bd), imitation (im), and death-birth (db). comparing structured populations with well-mixed populations, we arrive at two main conclusions. first, in the absence of untrustworthy trustees, there is a saddle point between investors and trustworthy trustees, with collaboration thriving best in well-mixed populations. the collaboration diminishes sequentially from db to im to pc/bd updating rules in structured populations. second, an invasion of untrustworthy trustees makes this saddle point unstable and leads to the extinction of investors. the 3-strategy system stabilizes at an equilibrium line where the trustworthy and untrustworthy trustees coexist. the stability span of trustworthy trustees is maximally extended under the pc and bd updating rules in structured populations, while it decreases in a sequence from im to db updating rules, with the well-mixed population being the least favorable. this research thus adds an analytical lens to the evolution of trust in structured populations.',\n",
       "  'categories': 'q-bio.pe nlin.ao physics.soc-ph',\n",
       "  'doi': '10.1016/j.amc.2024.128595',\n",
       "  'created': '2023-09-12',\n",
       "  'updated': '2024-02-09',\n",
       "  'authors': ['chaoqian wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.06636'},\n",
       " {'title': 'return to lacan: an approach to digital twin mind with free energy   principle',\n",
       "  'id': '2309.06707',\n",
       "  'abstract': \"free energy principle (fep) is a burgeoning theory in theoretical neuroscience that provides a universal law for modelling living systems of any scale. expecting a digital twin mind from this first principle, we propose a macro-level interpretation that bridge neuroscience and psychoanalysis through the lens of computational lacanian psychoanalysis. in this article, we claim three fundamental parallels between fep and lacanian psychoanalysis, and suggest a fep approach to formalizing lacan's theory. sharing the non-linear temporal structure that combines prediction and retrospection (logical time), both of two theories focus on epistemological questions that how systems represented themselves and external world, and those elements failed to be represented (lacks and free energy) significantly influence the systems' subsequent states. additionally, the fundamental hypothesis of fep that the precise state of environment is always concealed, accounts for object petit a, the core concept in lacan's theory. with neuropsychoanalytic mapping from three orders (the real, the symbolic, and the imaginary, rsi) onto brain regions, we propose a brain-wide fep model for a minimal definition of lacanian mind - composite state of rsi that is perturbated by desire running over the logical time. the fep-rsi model involves three fep units connected by respective free energy with a natural compliance with logical time, mimicking core dynamics of lacanian mind. the biological plausibility of current model is considered from perspectives of cognitive neuroscience. in conclusion, the fep-rsi model encapsulates a unified framework for digital twin modeling at the macro level.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-13',\n",
       "  'updated': '2024-07-05',\n",
       "  'authors': ['lingyu li', 'chunbo li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.06707'},\n",
       " {'title': 'optimal transport distances for directed, weighted graphs: a case study   with cell-cell communication networks',\n",
       "  'id': '2309.07030',\n",
       "  'abstract': 'comparing graphs by means of optimal transport has recently gained significant attention, as the distances induced by optimal transport provide both a principled metric between graphs as well as an interpretable description of the associated changes between graphs in terms of a transport plan. as the lack of symmetry introduces challenges in the typically considered formulations, optimal transport distances for graphs have mostly been developed for undirected graphs. here, we propose two distance measures to compare directed graphs based on variants of optimal transport: (i) an earth movers distance (wasserstein) and (ii) a gromov-wasserstein (gw) distance. we evaluate these two distances and discuss their relative performance for both simulated graph data and real-world directed cell-cell communication graphs, inferred from single-cell rna-seq data.',\n",
       "  'categories': 'cs.lg cs.si cs.sy eess.sy q-bio.gn q-bio.mn',\n",
       "  'doi': '10.1109/icassp48485.2024.10446503',\n",
       "  'created': '2023-09-13',\n",
       "  'updated': '2024-03-14',\n",
       "  'authors': ['james s. nagai', 'ivan g. costa', 'michael t. schaub'],\n",
       "  'affiliation': ['institute for computational genomics, rwth aachen medical faculty, germany',\n",
       "   'institute for computational genomics, rwth aachen medical faculty, germany',\n",
       "   'department of computer science, rwth aachen university, germany'],\n",
       "  'url': 'https://arxiv.org/abs/2309.07030'},\n",
       " {'title': 'discovering dynamic effective connectome of brain with bayesian dynamic   dag learning',\n",
       "  'id': '2309.07080',\n",
       "  'abstract': 'understanding the complex mechanisms of the brain can be unraveled by extracting the dynamic effective connectome (dec). recently, score-based directed acyclic graph (dag) discovery methods have shown significant improvements in extracting the causal structure and inferring effective connectivity. however, learning dec through these methods still faces two main challenges: one with the fundamental impotence of high-dimensional dynamic dag discovery methods and the other with the low quality of fmri data. in this paper, we introduce bayesian dynamic dag learning with m-matrices acyclicity characterization (bdyma) method to address the challenges in discovering dec. the presented dynamic causal model enables us to discover direct feedback loop edges as well. leveraging an unconstrained framework in the bdyma method leads to more accurate results in detecting high-dimensional networks, achieving sparser outcomes, making it particularly suitable for extracting dec. additionally, the score function of the bdyma method allows the incorporation of prior knowledge into the process of dynamic causal discovery which further enhances the accuracy of results. comprehensive simulations on synthetic data and experiments on human connectome project (hcp) data demonstrate that our method can handle both of the two main challenges, yielding more accurate and reliable dec compared to state-of-the-art and traditional methods. additionally, we investigate the trustworthiness of dti data as prior knowledge for dec discovery and show the improvements in dec discovery when the dti data is incorporated into the process.',\n",
       "  'categories': 'q-bio.nc cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-07',\n",
       "  'updated': '2024-03-09',\n",
       "  'authors': ['abdolmahdi bagheri',\n",
       "   'mohammad pasande',\n",
       "   'kevin bello',\n",
       "   'babak nadjar araabi',\n",
       "   'alireza akhondi-asl'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.07080'},\n",
       " {'title': 'computational limits to the legibility of the imaged human brain',\n",
       "  'id': '2309.07096',\n",
       "  'abstract': 'our knowledge of the organisation of the human brain at the population-level is yet to translate into power to predict functional differences at the individual-level, limiting clinical applications, and casting doubt on the generalisability of inferred mechanisms. it remains unknown whether the difficulty arises from the absence of individuating biological patterns within the brain, or from limited power to access them with the models and compute at our disposal. here we comprehensively investigate the resolvability of such patterns with data and compute at unprecedented scale. across 23 810 unique participants from uk biobank, we systematically evaluate the predictability of 25 individual biological characteristics, from all available combinations of structural and functional neuroimaging data. over 4526 gpu hours of computation, we train, optimize, and evaluate out-of-sample 700 individual predictive models, including fully-connected feed-forward neural networks of demographic, psychological, serological, chronic disease, and functional connectivity characteristics, and both uni- and multi-modal 3d convolutional neural network models of macro- and micro-structural brain imaging. we find a marked discrepancy between the high predictability of sex (balanced accuracy 99.7%), age (mean absolute error 2.048 years, r2 0.859), and weight (mean absolute error 2.609kg, r2 0.625), for which we set new state-of-the-art performance, and the surprisingly low predictability of other characteristics. neither structural nor functional imaging predicted psychology better than the coincidence of chronic disease (p<0.05). serology predicted chronic disease (p<0.05) and was best predicted by it (p<0.001), followed by structural neuroimaging (p<0.05). our findings suggest either more informative imaging or more powerful models are needed to decipher individual level characteristics from the human brain.',\n",
       "  'categories': 'q-bio.nc cs.cv eess.iv',\n",
       "  'doi': '10.1016/j.neuroimage.2024.120600',\n",
       "  'created': '2023-08-23',\n",
       "  'updated': '2024-04-02',\n",
       "  'authors': ['james k ruffle',\n",
       "   'robert j gray',\n",
       "   'samia mohinta',\n",
       "   'guilherme pombo',\n",
       "   'chaitanya kaul',\n",
       "   'harpreet hyare',\n",
       "   'geraint rees',\n",
       "   'parashkev nachev'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.07096'},\n",
       " {'title': 'a strong and simple deep learning baseline for bci mi decoding',\n",
       "  'id': '2309.07159',\n",
       "  'abstract': 'we propose eeg-simpleconv, a straightforward 1d convolutional neural network for motor imagery decoding in bci. our main motivation is to propose a simple and performing baseline to compare to, using only very standard ingredients from the literature. we evaluate its performance on four eeg motor imagery datasets, including simulated online setups, and compare it to recent deep learning and machine learning approaches. eeg-simpleconv is at least as good or far more efficient than other approaches, showing strong knowledge-transfer capabilities across subjects, at the cost of a low inference time. we advocate that using off-the-shelf ingredients rather than coming with ad-hoc solutions can significantly help the adoption of deep learning approaches for bci. we make the code of the models and the experiments accessible.',\n",
       "  'categories': 'eess.sp cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-11',\n",
       "  'updated': '2024-01-25',\n",
       "  'authors': ['yassine el ouahidi',\n",
       "   'vincent gripon',\n",
       "   'bastien pasdeloup',\n",
       "   'ghaith bouallegue',\n",
       "   'nicolas farrugia',\n",
       "   'giulia lioi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.07159'},\n",
       " {'title': 'simultaneous inference for generalized linear models with unmeasured   confounders',\n",
       "  'id': '2309.07261',\n",
       "  'abstract': 'tens of thousands of simultaneous hypothesis tests are routinely performed in genomic studies to identify differentially expressed genes. however, due to unmeasured confounders, many standard statistical approaches may be substantially biased. this paper investigates the large-scale hypothesis testing problem for multivariate generalized linear models in the presence of confounding effects. under arbitrary confounding mechanisms, we propose a unified statistical estimation and inference framework that harnesses orthogonal structures and integrates linear projections into three key stages. it begins by disentangling marginal and uncorrelated confounding effects to recover the latent coefficients. subsequently, latent factors and primary effects are jointly estimated through lasso-type optimization. finally, we incorporate projected and weighted bias-correction steps for hypothesis testing. theoretically, we establish the identification conditions of various effects and non-asymptotic error bounds. we show effective type-i error control of asymptotic $z$-tests as sample and response sizes approach infinity. numerical experiments demonstrate that the proposed method controls the false discovery rate by the benjamini-hochberg procedure and is more powerful than alternative methods. by comparing single-cell rna-seq counts from two groups of samples, we demonstrate the suitability of adjusting confounding effects when significant covariates are absent from the model.',\n",
       "  'categories': 'stat.me cs.lg q-bio.gn stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-13',\n",
       "  'updated': '2024-10-13',\n",
       "  'authors': ['jin-hong du', 'larry wasserman', 'kathryn roeder'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.07261'},\n",
       " {'title': 'diminished circadian and ultradian rhythms of human brain activity in   pathological tissue in vivo',\n",
       "  'id': '2309.07271',\n",
       "  'abstract': 'chronobiological rhythms, such as the circadian rhythm, have long been linked to neurological disorders, but it is currently unknown how pathological processes affect the expression of biological rhythms in the brain.   here, we use the unique opportunity of long-term, continuous intracranially recorded eeg from 38 patients (totalling 6338 hours) to delineate circadian (daily) and ultradian (minute to hourly) rhythms in different brain regions. we show that functional circadian and ultradian rhythms are diminished in pathological tissue, independent of regional variations. we further demonstrate that these diminished rhythms are persistent in time, regardless of load or occurrence of pathological events.   these findings provide evidence that brain pathology is functionally associated with persistently diminished chronobiological rhythms in vivo in humans, independent of regional variations or pathological events. future work interacting with, and restoring, these modulatory chronobiological rhythms may allow for novel therapies.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-13',\n",
       "  'updated': '2024-08-07',\n",
       "  'authors': ['christopher thornton',\n",
       "   'mariella panagiotopoulou',\n",
       "   'fahmida a chowdhury',\n",
       "   'beate diehl',\n",
       "   'john s duncan',\n",
       "   'sarah j gascoigne',\n",
       "   'guillermo besne',\n",
       "   'andrew w mcevoy',\n",
       "   'anna miserocchi',\n",
       "   'billy c smith',\n",
       "   'jane de tisi',\n",
       "   'peter n taylor',\n",
       "   'yujiang wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.07271'},\n",
       " {'title': \"a stein's method approach to the linear noise approximation for   stationary distributions of chemical reaction networks\",\n",
       "  'id': '2309.07386',\n",
       "  'abstract': \"stochastic chemical reaction networks are continuous time markov chain models that describe the time evolution of the molecular counts of species interacting stochastically via discrete reactions. such models are ubiquitous in systems and synthetic biology, but often have a large or infinite number of states, and thus are not amenable to computation and analysis. due to this, approximations that rely on the molecular counts and the volume being large are commonly used, with the most common being the reaction rate equations and the linear noise approximation. for finite time intervals, kurtz established the validity of the reaction rate equations and linear noise approximation, by proving law of large numbers and central limit theorem results respectively. however, the analogous question for the stationary distribution of the markov chain model has remained mostly unanswered, except for chemical reaction networks with special structures or bounded molecular counts. in this work, we use stein's method to obtain sufficient conditions for the stationary distribution of an appropriately scaled stochastic chemical reaction network to converge to the linear noise approximation as the system size goes to infinity. our results give non asymptotic bounds on the error in the reaction rate equations and in the linear noise approximation as applied to the stationary distribution. as a special case, we give conditions under which the global exponential stability of an equilibrium point of the reaction rate equations is sufficient to obtain our error bounds, thus permitting one to obtain conclusions about the markov chain by analyzing the deterministic reaction rate equations.\",\n",
       "  'categories': 'q-bio.qm math.pr physics.chem-ph q-bio.mn',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-13',\n",
       "  'updated': '2024-01-31',\n",
       "  'authors': ['theodore w. grunberg', 'domitilla del vecchio'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.07386'},\n",
       " {'title': 'a standardised open science framework for sharing and re-analysing   neural data acquired to continuous stimuli',\n",
       "  'id': '2309.07671',\n",
       "  'abstract': 'neurophysiology research has demonstrated that it is possible and valuable to investigate sensory processing in scenarios involving continuous sensory streams, such as speech and music. over the past 10 years or so, novel analytic frameworks combined with the growing participation in data sharing has led to a surge of publicly available datasets involving continuous sensory experiments. however, open science efforts in this domain of research remain scattered, lacking a cohesive set of guidelines. this paper presents an end-to-end open science framework for the storage, analysis, sharing, and re-analysis of neural data recorded during continuous sensory experiments. we propose a data structure that builds on existing custom structures (continuous-event neural data or cnd), providing precise naming conventions and data types, as well as a workflow for storing and loading data in the general-purpose bids structure. the framework has been designed to interface with existing eeg/meg analysis toolboxes, such as eelbrain, naplib, mne, and mtrf-toolbox. we present guidelines by taking both the user view (rapidly re-analyse existing data) and the experimenter view (store, analyse, and share), making the process straightforward and accessible. additionally, we introduce a web-based data browser that enables the effortless replication of published results and data re-analysis.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-14',\n",
       "  'updated': '2024-09-16',\n",
       "  'authors': ['giovanni m. di liberto',\n",
       "   'aaron nidiffer',\n",
       "   'michael j. crosse',\n",
       "   'nathaniel j. zuk',\n",
       "   'stephanie haro',\n",
       "   'giorgia cantisani',\n",
       "   'martin m. winchester',\n",
       "   'aoife igoe',\n",
       "   'ross mccrann',\n",
       "   'satwik chandra',\n",
       "   'edmund c. lalor',\n",
       "   'giacomo baruzzo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.07671'},\n",
       " {'title': 'virchow: a million-slide digital pathology foundation model',\n",
       "  'id': '2309.07778',\n",
       "  'abstract': \"the use of artificial intelligence to enable precision medicine and decision support systems through the analysis of pathology images has the potential to revolutionize the diagnosis and treatment of cancer. such applications will depend on models' abilities to capture the diverse patterns observed in pathology images. to address this challenge, we present virchow, a foundation model for computational pathology. using self-supervised learning empowered by the dinov2 algorithm, virchow is a vision transformer model with 632 million parameters trained on 1.5 million hematoxylin and eosin stained whole slide images from diverse tissue and specimen types, which is orders of magnitude more data than previous works. the virchow model enables the development of a pan-cancer detection system with 0.949 overall specimen-level auc across 17 different cancer types, while also achieving 0.937 auc on 7 rare cancer types. the virchow model sets the state-of-the-art on the internal and external image tile level benchmarks and slide level biomarker prediction tasks. the gains in performance highlight the importance of training on massive pathology image datasets, suggesting scaling up the data and network architecture can improve the accuracy for many high-impact computational pathology applications where limited amounts of training data are available.\",\n",
       "  'categories': 'eess.iv cs.cv cs.lg q-bio.to',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-14',\n",
       "  'updated': '2024-01-17',\n",
       "  'authors': ['eugene vorontsov',\n",
       "   'alican bozkurt',\n",
       "   'adam casson',\n",
       "   'george shaikovski',\n",
       "   'michal zelechowski',\n",
       "   'siqi liu',\n",
       "   'kristen severson',\n",
       "   'eric zimmermann',\n",
       "   'james hall',\n",
       "   'neil tenenholtz',\n",
       "   'nicolo fusi',\n",
       "   'philippe mathieu',\n",
       "   'alexander van eck',\n",
       "   'donghun lee',\n",
       "   'julian viret',\n",
       "   'eric robert',\n",
       "   'yi kan wang',\n",
       "   'jeremy d. kunz',\n",
       "   'matthew c. h. lee',\n",
       "   'jan bernhard',\n",
       "   'ran a. godrich',\n",
       "   'gerard oakley',\n",
       "   'ewan millar',\n",
       "   'matthew hanna',\n",
       "   'juan retamero',\n",
       "   'william a. moye',\n",
       "   'razik yousfi',\n",
       "   'christopher kanan',\n",
       "   'david klimstra',\n",
       "   'brandon rothrock',\n",
       "   'thomas j. fuchs'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.07778'},\n",
       " {'title': 'morphological stability for in silico models of avascular tumors',\n",
       "  'id': '2309.07889',\n",
       "  'abstract': \"the landscape of computational modeling in cancer systems biology is diverse, offering a spectrum of models and frameworks, each with its own trade-offs and advantages. ideally, models are meant to be useful in refining hypotheses, to sharpen experimental procedures and, in the longer run, even for applications in personalized medicine. one of the greatest challenges is to balance model realism and detail with experimental data to eventually produce useful data-driven models. we contribute to this quest by developing a transparent, highly parsimonious, first principles silico model of a growing avascular tumor. we initially formulate the physiological considerations and the specific model within a stochastic cell-based framework. we next formulate a corresponding mean-field model using partial differential equations which is amenable to mathematical analysis. despite a few notable differences between the two models, we are in this way able to successfully detail the impact of all parameters in the stability of the growth process and on the eventual tumor fate of the stochastic model. this facilitates the deduction of bayesian priors for a given situation, but also provides important insights into the underlying mechanism of tumor growth and progression. although the resulting model framework is relatively simple and transparent, it can still reproduce the full range of known emergent behavior. we identify a novel model instability arising from nutrient starvation and we also discuss additional insight concerning possible model additions and the effects of those. thanks to the framework's flexibility, such additions can be readily included whenever the relevant data become available.\",\n",
       "  'categories': 'math.ds q-bio.to',\n",
       "  'doi': '10.1007/s11538-024-01297-x',\n",
       "  'created': '2023-09-14',\n",
       "  'updated': '2024-03-18',\n",
       "  'authors': ['erik blom', 'stefan engblom'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.07889'},\n",
       " {'title': \"mathematical modeling of heterogeneous stem cell regeneration: from cell   division to waddington's epigenetic landscape\",\n",
       "  'id': '2309.08064',\n",
       "  'abstract': \"stem cell regeneration is a crucial biological process for most self-renewing tissues during the development and maintenance of tissue homeostasis. in developing the mathematical models of stem cell regeneration and tissue development, cell division is the core process connecting different scale biological processes and leading to changes in cell population number and the epigenetic state of cells. this chapter focuses on the primary strategies for modeling cell division in biological systems. the lagrange coordinate modeling approach considers gene network dynamics within each cell and random changes in cell states and model parameters during cell division. in contrast, the euler coordinate modeling approach formulates the evolution of cell population numbers with the same epigenetic state via a differential-integral equation. these strategies focus on different scale dynamics, respectively, and result in two methods of modeling waddington's epigenetic landscape: the fokker-planck equation and the differential-integral equation approaches. the differential-integral equation approach formulates the evolution of cell population density based on simple assumptions in cell proliferation, apoptosis, differentiation, and epigenetic state transitions during cell division. moreover, machine learning methods can establish low-dimensional macroscopic measurements of a cell based on single-cell rna sequencing data. the low dimensional measurements can quantify the epigenetic state of cells and become connections between static single-cell rna sequencing data with dynamic equations for tissue development processes. the differential-integral equation presented in this chapter provides a reasonable approach to understanding the complex biological processes of tissue development and tumor progression.\",\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-14',\n",
       "  'updated': '2024-01-16',\n",
       "  'authors': ['jinzhi lei'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.08064'},\n",
       " {'title': 'miml: multiplex image machine learning for high precision cell   classification via mechanical traits within microfluidic systems',\n",
       "  'id': '2309.08421',\n",
       "  'abstract': \"label-free cell classification is advantageous for supplying pristine cells for further use or examination, yet existing techniques frequently fall short in terms of specificity and speed. in this study, we address these limitations through the development of a novel machine learning framework, multiplex image machine learning (miml). this architecture uniquely combines label-free cell images with biomechanical property data, harnessing the vast, often underutilized morphological information intrinsic to each cell. by integrating both types of data, our model offers a more holistic understanding of the cellular properties, utilizing morphological information typically discarded in traditional machine learning models. this approach has led to a remarkable 98.3\\\\% accuracy in cell classification, a substantial improvement over models that only consider a single data type. miml has been proven effective in classifying white blood cells and tumor cells, with potential for broader application due to its inherent flexibility and transfer learning capability. it's particularly effective for cells with similar morphology but distinct biomechanical properties. this innovative approach has significant implications across various fields, from advancing disease diagnostics to understanding cellular behavior.\",\n",
       "  'categories': 'eess.iv cs.cv cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-15',\n",
       "  'updated': '2024-01-24',\n",
       "  'authors': ['khayrul islam', 'ratul paul', 'shen wang', 'yaling liu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.08421'},\n",
       " {'title': 'current and future directions in network biology',\n",
       "  'id': '2309.08478',\n",
       "  'abstract': 'network biology is an interdisciplinary field bridging computational and biological sciences that has proved pivotal in advancing the understanding of cellular functions and diseases across biological systems and scales. although the field has been around for two decades, it remains nascent. it has witnessed rapid evolution, accompanied by emerging challenges. these challenges stem from various factors, notably the growing complexity and volume of data together with the increased diversity of data types describing different tiers of biological organization. we discuss prevailing research directions in network biology and highlight areas of inference and comparison of biological networks, multimodal data integration and heterogeneous networks, higher-order network analysis, machine learning on networks, and network-based personalized medicine. following the overview of recent breakthroughs across these five areas, we offer a perspective on the future directions of network biology. additionally, we offer insights into scientific communities, educational initiatives, and the importance of fostering diversity within the field. this paper establishes a roadmap for an immediate and long-term vision for network biology.',\n",
       "  'categories': 'q-bio.mn',\n",
       "  'doi': '10.1093/bioadv/vbae099',\n",
       "  'created': '2023-09-15',\n",
       "  'updated': '2024-06-11',\n",
       "  'authors': ['marinka zitnik',\n",
       "   'michelle m. li',\n",
       "   'aydin wells',\n",
       "   'kimberly glass',\n",
       "   'deisy morselli gysi',\n",
       "   'arjun krishnan',\n",
       "   't. m. murali',\n",
       "   'predrag radivojac',\n",
       "   'sushmita roy',\n",
       "   'anaïs baudot',\n",
       "   'serdar bozdag',\n",
       "   'danny z. chen',\n",
       "   'lenore cowen',\n",
       "   'kapil devkota',\n",
       "   'anthony gitter',\n",
       "   'sara gosline',\n",
       "   'pengfei gu',\n",
       "   'pietro h. guzzi',\n",
       "   'heng huang',\n",
       "   'meng jiang',\n",
       "   'ziynet nesibe kesimoglu',\n",
       "   'mehmet koyuturk',\n",
       "   'jian ma',\n",
       "   'alexander r. pico',\n",
       "   'nataša pržulj',\n",
       "   'teresa m. przytycka',\n",
       "   'benjamin j. raphael',\n",
       "   'anna ritz',\n",
       "   'roded sharan',\n",
       "   'yang shen',\n",
       "   'mona singh',\n",
       "   'donna k. slonim',\n",
       "   'hanghang tong',\n",
       "   'xinan holly yang',\n",
       "   'byung-jun yoon',\n",
       "   'haiyuan yu',\n",
       "   'tijana milenković'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.08478'},\n",
       " {'title': 'using a quantitative assessment of propulsion biomechanics in wheelchair   racing to guide the design of personalized gloves: a case study',\n",
       "  'id': '2309.08726',\n",
       "  'abstract': \"this study with a t-52 class wheelchair racing athlete aimed to combine quantitative biomechanical measurements to the athlete's perception to design and test different prototypes of a new kind of rigid gloves. three personalized rigid gloves with various, fixed wrist extension angles were prototyped and tested on a treadmill in a biomechanics laboratory. the prototype with 45{\\\\deg} wrist extension was the athlete's favourite as it reduced his perception of effort. biomechanical assessment and user-experience data indicated that his favourite prototype increased wrist stability throughout the propulsion cycle while maintaining a very similar propulsion technique to the athlete's prior soft gloves. moreover, the inclusion of an innovative attachment system on the new gloves allowed the athlete to put his gloves on by himself, eliminating the need for external assistance and thus significantly increasing his autonomy. this multidisciplinary approach helped to prototype and develop a new rigid personalized gloves concept and is clearly a promising avenue to tailor adaptive sports equipment to an athlete's needs.\",\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-15',\n",
       "  'updated': '2024-01-22',\n",
       "  'authors': ['félix chénier',\n",
       "   'gerald parent',\n",
       "   'mikaël leblanc',\n",
       "   'colombe bélaise',\n",
       "   'mathieu andrieux'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.08726'},\n",
       " {'title': 'computational framework for the generation of one-dimensional vascular   models accounting for uncertainty in networks extracted from medical images',\n",
       "  'id': '2309.08779',\n",
       "  'abstract': \"patient-specific computational modeling is a popular, non-invasive method to answer medical questions. medical images are used to extract geometric domains necessary to create these models, providing a predictive tool for clinicians. however, in vivo imaging is subject to uncertainty, impacting vessel dimensions essential to the mathematical modeling process. while there are numerous programs available to provide information about vessel length, radii, and position, there is currently no exact way to determine and calibrate these features. this raises the question, if we are building patient-specific models based on uncertain measurements, how accurate are the geometries we extract and how can we best represent a patient's vasculature? in this study, we develop a novel framework to determine vessel dimensions using change points. we explore the impact of uncertainty in the network extraction process on hemodynamics by varying vessel dimensions and segmenting the same images multiple times. our analyses reveal that image segmentation, network size, and minor changes in radius and length have significant impacts on pressure and flow dynamics in rapidly branching structures and tapering vessels. accordingly, we conclude that it is critical to understand how uncertainty in network geometry propagates to fluid dynamics, especially in clinical applications.\",\n",
       "  'categories': 'q-bio.to q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-15',\n",
       "  'updated': '2024-05-08',\n",
       "  'authors': ['michelle a bartolo',\n",
       "   'alyssa m taylor-lapole',\n",
       "   'darsh gandhi',\n",
       "   'alexandria johnson',\n",
       "   'yaqi li',\n",
       "   'emma slack',\n",
       "   'isaiah stevens',\n",
       "   'zachary turner',\n",
       "   'justin d weigand',\n",
       "   'charles puelz',\n",
       "   'dirk husmeier',\n",
       "   'mette s olufsen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.08779'},\n",
       " {'title': \"bg-gan: generative ai enables representing brain structure-function   connections for alzheimer's disease\",\n",
       "  'id': '2309.08916',\n",
       "  'abstract': \"the relationship between brain structure and function is critical for revealing the pathogenesis of brain disease, including alzheimer's disease (ad). however, it is a great challenge to map brain structure-function connections due to various reasons. in this work, a bidirectional graph generative adversarial networks (bg-gan) is proposed to represent brain structure-function connections. specifically, by designing a module incorporating inner graph convolution network (innergcn), the generators of bg-gan can employ features of direct and indirect brain regions to learn the mapping function between structural domain and functional domain. besides, a new module named balancer is designed to counterpoise the optimization between generators and discriminators. by introducing the balancer into bg-gan, both the structural generator and functional generator can not only alleviate the issue of mode collapse but also learn complementarity of structural and functional features. experimental results using adni datasets show that the both the generated structure connections and generated function connections can improve the identification accuracy of ad. more importantly, based the proposed model, it is found that the relationship between brain structure and function is not a complete one-to-one correspondence. brain structure is the basis of brain function. the strong structural connections are almost accompanied by strong functional connections.\",\n",
       "  'categories': 'cs.ai eess.iv q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-16',\n",
       "  'updated': '2024-11-03',\n",
       "  'authors': ['changhong jing', 'chen ding', 'shuqiang wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.08916'},\n",
       " {'title': 'the connection between polymer collapse and the onset of jamming',\n",
       "  'id': '2309.09065',\n",
       "  'abstract': 'previous studies have shown that the interiors of proteins are densely packed, reaching packing fractions that are as large as those found for static packings of individual amino-acid-shaped particles. how can the interiors of proteins take on such high packing fractions given that amino acids are connected by peptide bonds and many amino acids are hydrophobic with attractive interactions? we investigate this question by comparing the structural and mechanical properties of collapsed attractive disk-shaped bead-spring polymers to those of three reference systems: static packings of repulsive disks, of attractive disks, and of repulsive disk-shaped bead-spring polymers. we show that attractive systems quenched to temperatures below the glass transition $t \\\\ll t_g$ and static packings of both repulsive disks and bead-spring polymers possess similar interior packing fractions. previous studies have shown that static packings of repulsive disks are isostatic at jamming onset, i.e. the number of contacts $n_c$ matches the number of degrees of freedom, which strongly influences their mechanical properties. we find that repulsive polymers are hypostatic at jamming onset, but effectively isostatic when including quartic modes. while attractive disk and polymer packings are hyperstatic, we identify a definition for interparticle contacts for which they can also be considered as effectively isostatic. as a result, we show that the mechanical properties (e.g. scaling of the potential energy with excess contact number and low-frequency contribution to the density of vibrational modes) of weakly attractive disk and polymer packings are similar to those of ${\\\\it isostatic}$ repulsive disk and polymer packings. our results demonstrate that static packings generated via attractive collapse or compression of repulsive particles possess similar structural and mechanical properties.',\n",
       "  'categories': 'cond-mat.soft q-bio.bm',\n",
       "  'doi': '10.1103/physreve.109.034406',\n",
       "  'created': '2023-09-16',\n",
       "  'updated': '',\n",
       "  'authors': ['alex t. grigas',\n",
       "   'aliza fisher',\n",
       "   'mark d. shattuck',\n",
       "   \"corey s. o'hern\"],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.09065'},\n",
       " {'title': 'structure to property: chemical element embeddings and a deep learning   approach for accurate prediction of chemical properties',\n",
       "  'id': '2309.09355',\n",
       "  'abstract': 'we introduce the elembert model for chemical classification tasks. it is based on deep learning techniques, such as a multilayer encoder architecture. we demonstrate the opportunities offered by our approach on sets of organic, inorganic and crystalline compounds. in particular, we developed and tested the model using the matbench and moleculenet benchmarks, which include crystal properties and drug design-related benchmarks. we also conduct an analysis of vector representations of chemical compounds, shedding light on the underlying patterns in structural data. our model exhibits exceptional predictive capabilities and proves universally applicable to molecular and material datasets. for instance, on the tox21 dataset, we achieved an average precision of 96%, surpassing the previously best result by 10%.',\n",
       "  'categories': 'physics.chem-ph cond-mat.mtrl-sci cs.lg physics.atm-clus q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-17',\n",
       "  'updated': '2024-08-17',\n",
       "  'authors': ['shokirbek shermukhamedov',\n",
       "   'dilorom mamurjonova',\n",
       "   'michael probst'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.09355'},\n",
       " {'title': 'interactions and migration rescuing ecological diversity',\n",
       "  'id': '2309.09900',\n",
       "  'abstract': \"how diversity is maintained in natural ecosystems is a long-standing question in theoretical ecology. by studying a system that combines ecological dynamics, heterogeneous interactions and spatial structure, we uncover a new mechanism for the survival of diversity-rich ecosystems in the presence of demographic fluctuations. for a single species, one finds a continuous phase transition between an extinction and a survival state, that falls into the universality class of directed percolation. here we show that the case of many species with heterogeneous interactions is different and richer. by merging theory and simulations, we demonstrate that with sufficiently strong demographic noise, the system exhibits behavior akin to the single-species case, undergoing a continuous transition. conversely, at low demographic noise, we observe unique features indicative of the ecosystem's complexity. the combined effects of the heterogeneity in the interaction network and migration enable the community to thrive, even in situations where demographic noise would lead to the extinction of isolated species. the emergence of mutualism induces the development of global bistability, accompanied by sudden tipping points. we present a way to predict the catastrophic shift from high diversity to extinction by probing responses to perturbations as an early warning signal.\",\n",
       "  'categories': 'q-bio.pe cond-mat.dis-nn cond-mat.stat-mech',\n",
       "  'doi': '10.1103/prxlife.2.013014',\n",
       "  'created': '2023-09-18',\n",
       "  'updated': '2024-02-05',\n",
       "  'authors': ['giulia garcia lorenzana', 'ada altieri', 'giulio biroli'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.09900'},\n",
       " {'title': 'dynamical network stability analysis of multiple biological ages   provides a framework for understanding the aging process',\n",
       "  'id': '2309.10005',\n",
       "  'abstract': 'widespread interest in non-destructive biomarkers of aging has led to a curse of plenty: a multitude of biological ages that each proffers a \\'true\\' health-adjusted age of an individual. while each measure provides salient information on the aging process, they are each univariate, in contrast to the \"hallmark\" and \"pillar\" theories of aging which are explicitly multidimensional, multicausal and multiscale. fortunately, multiple biological ages can be systematically combined into a multidimensional network representation. the interaction network between these biological ages permits analysis of the multidimensional effects of aging, as well as quantification of causal influences during both natural aging and, potentially, after anti-aging intervention. the behaviour of the system as a whole can then be explored using dynamical network stability analysis which identifies new, efficient biomarkers that quantify long term resilience scores on the timescale between measurements (years). we demonstrate this approach using a set of 8 biological ages from the longitudinal swedish adoption/twin study of aging (satsa). after extracting an interaction network between these biological ages, we observed that physiological age, a proxy for cardiometabolic health, serves as a central node in the network, implicating it as a key vulnerability for slow, age-related decline. we furthermore show that while the system as a whole is stable, there is a weakly stable direction along which recovery is slow - on the timescale of a human lifespan. this slow direction provides an aging biomarker which correlates strongly with chronological age and predicts longitudinal decline in health - suggesting that it estimates an important driver of age-related changes.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.1093/gerona/glae021',\n",
       "  'created': '2023-09-17',\n",
       "  'updated': '2024-01-20',\n",
       "  'authors': ['glen pridham', 'andrew d. rutenberg'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.10005'},\n",
       " {'title': 'characterizing the non-monotonic behavior of mutual information along   biochemical reaction cascades',\n",
       "  'id': '2309.10162',\n",
       "  'abstract': 'cells sense environmental signals and transmit information intracellularly through changes in the abundance of molecular components. such molecular abundances can be measured in single cells and exhibit significant heterogeneity in clonal populations even in identical environments. experimentally observed joint probability distributions can then be used to quantify the covariability and mutual information between molecular abundances along signaling cascades. however, because stationary state abundances along stochastic biochemical reaction cascades are not conditionally independent, their mutual information is not constrained by the data processing inequality. here, we report the conditions under which the mutual information between stationary state abundances increases along a cascade of biochemical reactions. this non-monotonic behavior can be intuitively understood in terms of noise propagation and time-averaging stochastic fluctuations that are short-lived compared to an extrinsic signal. our results re-emphasize that mutual information measurements of stationary state distributions of cellular components may be of limited utility for characterizing cellular signaling processes because they do not measure information transfer.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.1103/physreve.110.034309',\n",
       "  'created': '2023-09-18',\n",
       "  'updated': '2024-08-08',\n",
       "  'authors': ['raymond fan', 'andreas hilfinger'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.10162'},\n",
       " {'title': 'pathway realisability in chemical networks',\n",
       "  'id': '2309.10629',\n",
       "  'abstract': 'the exploration of pathways and alternative pathways that have a specific function is of interest in numerous chemical contexts. a framework for specifying and searching for pathways has previously been developed, but a focus on which of the many pathway solutions are realisable, or can be made realisable, is missing. realisable here means that there actually exists some sequencing of the reactions of the pathway that will execute the pathway. we present a method for analysing the realisability of pathways based on the reachability question in petri nets. for realisable pathways, our method also provides a certificate encoding an order of the reactions which realises the pathway. we present two extended notions of realisability of pathways, one of which is related to the concept of network catalysts. we exemplify our findings on the pentose phosphate pathway. furthermore, we discuss the relevance of our concepts for elucidating the choices often implicitly made when depicting pathways. lastly, we lay the foundation for the mathematical theory of realisability.',\n",
       "  'categories': 'q-bio.mn cs.dm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-19',\n",
       "  'updated': '2024-10-14',\n",
       "  'authors': ['jakob l. andersen',\n",
       "   'sissel banke',\n",
       "   'rolf fagerberg',\n",
       "   'christoph flamm',\n",
       "   'daniel merkle',\n",
       "   'peter f. stadler'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.10629'},\n",
       " {'title': 'an operating principle of the cerebral cortex, and a cellular mechanism   for attentional trial-and-error pattern learning and useful classification   extraction',\n",
       "  'id': '2309.10821',\n",
       "  'abstract': 'a feature of the brains of intelligent animals is the ability to learn to respond to an ensemble of active neuronal inputs with a behaviorally appropriate ensemble of active neuronal outputs. previously, a hypothesis was proposed on how this mechanism is implemented at the cellular level within the neocortical pyramidal neuron: the apical tuft or perisomatic inputs initiate \"guess\" neuron firings, while the basal dendrites identify input patterns based on excited synaptic clusters, with the cluster excitation strength adjusted based on reward feedback. this simple mechanism allows neurons to learn to classify their inputs in a surprisingly intelligent manner. here, we revise and extend this hypothesis. we modify synaptic plasticity rules to align with behavioral time scale synaptic plasticity (btsp) observed in hippocampal area ca1, making the framework more biophysically and behaviorally plausible. the neurons for the guess firings are selected in a voluntary manner via feedback connections to apical tufts in the neocortical layer 1, leading to dendritic ca2+ spikes with burst firing, which are postulated to be neural correlates of attentional, aware processing. once learned, the neuronal input classification is executed without voluntary or conscious control, enabling hierarchical incremental learning of classifications that is effective in our inherently classifiable world. in addition to voluntary, we propose that pyramidal neuron burst firing can be involuntary, also initiated via apical tuft inputs, drawing attention towards important cues such as novelty and noxious stimuli. we classify the excitations of neocortical pyramidal neurons into four categories based on their excitation pathway: attentional versus automatic and voluntary/acquired versus involuntary. additionally, we hypothesize that dendrites within pyramidal neuron minicolumn bundles are coupled via depolarization...',\n",
       "  'categories': 'q-bio.nc physics.bio-ph',\n",
       "  'doi': '10.3389/fncir.2024.1280604',\n",
       "  'created': '2023-08-21',\n",
       "  'updated': '2024-02-05',\n",
       "  'authors': ['marat m. rvachev'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.10821'},\n",
       " {'title': 'latent disentanglement in mesh variational autoencoders improves the   diagnosis of craniofacial syndromes and aids surgical planning',\n",
       "  'id': '2309.10825',\n",
       "  'abstract': 'the use of deep learning to undertake shape analysis of the complexities of the human head holds great promise. however, there have traditionally been a number of barriers to accurate modelling, especially when operating on both a global and local level. in this work, we will discuss the application of the swap disentangled variational autoencoder (sd-vae) with relevance to crouzon, apert and muenke syndromes. although syndrome classification is performed on the entire mesh, it is also possible, for the first time, to analyse the influence of each region of the head on the syndromic phenotype. by manipulating specific parameters of the generative model, and producing procedure-specific new shapes, it is also possible to simulate the outcome of a range of craniofacial surgical procedures. this opens new avenues to advance diagnosis, aids surgical planning and allows for the objective evaluation of surgical outcomes.',\n",
       "  'categories': 'eess.iv cs.lg q-bio.qm',\n",
       "  'doi': '10.1016/j.cmpb.2024.108395',\n",
       "  'created': '2023-09-05',\n",
       "  'updated': '',\n",
       "  'authors': ['simone foti',\n",
       "   'alexander j. rickart',\n",
       "   'bongjin koo',\n",
       "   \"eimear o' sullivan\",\n",
       "   'lara s. van de lande',\n",
       "   'athanasios papaioannou',\n",
       "   'roman khonsari',\n",
       "   'danail stoyanov',\n",
       "   'n. u. owase jeelani',\n",
       "   'silvia schievano',\n",
       "   'david j. dunaway',\n",
       "   'matthew j. clarkson'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.10825'},\n",
       " {'title': 'improving opioid use disorder risk modelling through behavioral and   genetic feature integration',\n",
       "  'id': '2309.10837',\n",
       "  'abstract': 'opioids are an effective analgesic for acute and chronic pain, but also carry a considerable risk of addiction leading to millions of opioid use disorder (oud) cases and tens of thousands of premature deaths in the united states yearly. estimating oud risk prior to prescription could improve the efficacy of treatment regimens, monitoring programs, and intervention strategies, but risk estimation is typically based on self-reported data or questionnaires. we develop an experimental design and computational methods that combine genetic variants associated with oud with behavioral features extracted from gps and wi-fi spatiotemporal coordinates to assess oud risk. since both oud mobility and genetic data do not exist for the same cohort, we develop algorithms to (1) generate mobility features from empirical distributions and (2) synthesize mobility and genetic samples assuming an expected level of disease co-occurrence. we show that integrating genetic and mobility modalities improves risk modelling using classification accuracy, area under the precision-recall and receiver operator characteristic curves, and $f_1$ score. interpreting the fitted models suggests that mobility features have more influence on oud risk, although the genetic contribution was significant, particularly in linear models. while there exist concerns with respect to privacy, security, bias, and generalizability that must be evaluated in clinical trials before being implemented in practice, our framework provides preliminary evidence that behavioral and genetic features may improve oud risk estimation to assist with personalized clinical decision-making.',\n",
       "  'categories': 'q-bio.qm cs.cy cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-19',\n",
       "  'updated': '2024-03-25',\n",
       "  'authors': ['sybille légitime',\n",
       "   'kaustubh prabhu',\n",
       "   'devin mcconnell',\n",
       "   'bing wang',\n",
       "   'dipak k. dey',\n",
       "   'derek aguiar'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.10837'},\n",
       " {'title': 'nanorobotics in medicine: a systematic review of advances, challenges,   and future prospects',\n",
       "  'id': '2309.10881',\n",
       "  'abstract': \"nanorobotics offers an emerging frontier in biomedicine, holding the potential to revolutionize diagnostic and therapeutic applications through its unique capabilities in manipulating biological systems at the nanoscale. following prisma guidelines, a comprehensive literature search was conducted using ieee xplore and pubmed databases, resulting in the identification and analysis of a total of 414 papers. the studies were filtered to include only those that addressed both nanorobotics and direct medical applications. our analysis traces the technology's evolution, highlighting its growing prominence in medicine as evidenced by the increasing number of publications over time. applications ranged from targeted drug delivery and single-cell manipulation to minimally invasive surgery and biosensing. despite the promise, limitations such as biocompatibility, precise control, and ethical concerns were also identified. this review aims to offer a thorough overview of the state of nanorobotics in medicine, drawing attention to current challenges and opportunities, and providing directions for future research in this rapidly advancing field.\",\n",
       "  'categories': 'cs.ro q-bio.to',\n",
       "  'doi': '10.33218/001c.92224',\n",
       "  'created': '2023-09-19',\n",
       "  'updated': '',\n",
       "  'authors': ['shishir rajendran',\n",
       "   'prathic sundararajan',\n",
       "   'ashi awasthi',\n",
       "   'suraj rajendran'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.10881'},\n",
       " {'title': 'migration feedback induces emergent ecotypes and abrupt transitions in   evolving populations',\n",
       "  'id': '2309.10884',\n",
       "  'abstract': 'we explore the connection between migration patterns and emergent behaviors of evolving populations in spatially heterogeneous environments. despite extensive studies in ecologically and medically important systems, a unifying framework that clarifies this connection and makes concrete predictions remains much needed. using a simple evolutionary model on a network of interconnected habitats with distinct fitness landscapes, we demonstrate a fundamental connection between migration feedback, emergent ecotypes, and an unusual form of discontinuous critical transition. we show how migration feedback generates spatially non-local niches in which emergent ecotypes can specialize. rugged fitness landscapes lead to a complex, yet understandable, phase diagram in which different ecotypes coexist under different migration patterns. the discontinuous transitions are distinct from the standard first-order phase transitions in statistical physics. they arise due to simultaneous transcritical bifurcations and exhibit a \"fine structure\" due to symmetry breaking between intra- and inter-ecotype interactions. we suggest feasible experiments to test our predictions.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-19',\n",
       "  'updated': '2024-01-08',\n",
       "  'authors': ['casey o. barkan', 'shenshen wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.10884'},\n",
       " {'title': 'negative spillover: a potential source of bias in pragmatic clinical   trials',\n",
       "  'id': '2309.10978',\n",
       "  'abstract': 'pragmatic clinical trials evaluate the effectiveness of health interventions in real-world settings. negative spillover can arise in a pragmatic trial if the study intervention affects how scarce resources are allocated between patients in the intervention and comparison groups. this can harm patients assigned to the control group and lead to overestimation of treatment effect. while this type of negative spillover is often addressed in trials of social welfare and public health interventions, there is little recognition of this source of bias in the medical literature. in this article, i examine what causes negative spillover and how it may have led clinical trial investigators to overestimate the effect of patient navigation, ai-based physiological alarms, and elective induction of labor. i also suggest ways to detect negative spillover and design trials that avoid this potential source of bias.',\n",
       "  'categories': 'stat.me q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-19',\n",
       "  'updated': '2024-03-26',\n",
       "  'authors': ['sean mann'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.10978'},\n",
       " {'title': 'quasi-stationary behavior of the stochastic fkpp equation on the circle',\n",
       "  'id': '2309.10998',\n",
       "  'abstract': 'we consider the stochastic fisher-kolmogorov-petrovsky-piscunov (fkpp) equation on the circle $\\\\mathbb{s}$, \\\\begin{equation*}   \\\\partial_t u(t,x) \\\\,= \\\\frac{\\\\alpha}{2}\\\\delta u +\\\\beta\\\\,u(1-u) + \\\\sqrt{\\\\gamma\\\\,u(1-u)}\\\\,\\\\dot{w}, \\\\qquad (t,x)\\\\in(0,\\\\infty)\\\\times \\\\mathbb{s}, \\\\end{equation*} where $\\\\dot{w}$ is space-time white noise. while any solution will eventually be absorbed at one of two states, the constant 1 and the constant 0 on the circle, essentially nothing had been established about the absorption time (also called the fixation time in population genetics), or about the long-time behavior prior to absorption. we establish the existence and uniqueness of the quasi-stationary distribution (qsd) for the solution of the stochastic fkpp. moreover, we show that the solution conditioned on not being absorbed at time $t$ converges to this unique qsd as $t\\\\to\\\\infty$, for any initial distribution, and characterize the leading-order asymptotics for the tail distribution of the fixation time. we obtain explicit calculations in the neutral case ($\\\\beta=0$), quantifying the effect of spatial diffusion on fixation time. we explicitly express the fixation rate in terms of the migration rate $\\\\alpha$ for all $\\\\alpha\\\\in (0,\\\\infty)$, finding in particular that the fixation rate is given by $\\\\gamma[1-\\\\frac{\\\\gamma}{12\\\\alpha}+\\\\mathcal{o}(\\\\frac{\\\\gamma^2}{\\\\alpha^2})]$ for fast migration and $\\\\pi^2\\\\alpha[1-\\\\frac{8\\\\alpha}{\\\\gamma}+\\\\mathcal{o}(\\\\frac{\\\\alpha^2}{\\\\gamma^2})]$ for slow migration. our proof relies on the observation that the absorbed (or killed) stochastic fkpp is dual to a system of $2$-type branching-coalescing brownian motions killed when one type dies off, and on leveraging the relationship between these two killed processes.',\n",
       "  'categories': 'math.pr math.ds math.fa math.sp q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-19',\n",
       "  'updated': '2024-01-09',\n",
       "  'authors': ['wai-tong louis fan', 'oliver tough'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.10998'},\n",
       " {'title': 'the topology and geometry of neural representations',\n",
       "  'id': '2309.11028',\n",
       "  'abstract': 'a central question for neuroscience is how to characterize brain representations of perceptual and cognitive content. an ideal characterization should distinguish different functional regions with robustness to noise and idiosyncrasies of individual brains that do not correspond to computational differences. previous studies have characterized brain representations by their representational geometry, which is defined by the representational dissimilarity matrix (rdm), a summary statistic that abstracts from the roles of individual neurons (or responses channels) and characterizes the discriminability of stimuli. here we explore a further step of abstraction: from the geometry to the topology of brain representations. we propose topological representational similarity analysis (trsa), an extension of representational similarity analysis (rsa) that uses a family of geo-topological summary statistics that generalizes the rdm to characterize the topology while de-emphasizing the geometry. we evaluate this new family of statistics in terms of the sensitivity and specificity for model selection using both simulations and fmri data. in the simulations, the ground truth is a data-generating layer representation in a neural network model and the models are the same and other layers in different model instances (trained from different random seeds). in fmri, the ground truth is a visual area and the models are the same and other areas measured in different subjects. results show that topology-sensitive characterizations of population codes are robust to noise and interindividual variability and maintain excellent sensitivity to the unique representational signatures of different neural network layers and brain regions. these methods enable researchers to calibrate comparisons among representations in brains and models to be sensitive to the geometry, the topology, or a combination of both.',\n",
       "  'categories': 'q-bio.nc cs.lg stat.me',\n",
       "  'doi': '10.1073/pnas.2317881121',\n",
       "  'created': '2023-09-19',\n",
       "  'updated': '2024-06-03',\n",
       "  'authors': ['baihan lin', 'nikolaus kriegeskorte'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.11028'},\n",
       " {'title': 'embed-search-align: dna sequence alignment using transformer models',\n",
       "  'id': '2309.11087',\n",
       "  'abstract': 'dna sequence alignment involves assigning short dna reads to the most probable locations on an extensive reference genome. this process is crucial for various genomic analyses, including variant calling, transcriptomics, and epigenomics. conventional methods, refined over decades, tackle this challenge in 2 steps: genome indexing followed by efficient search to locate likely positions for given reads. building on the success of large language models in encoding text into embeddings, where the distance metric captures semantic similarity, recent efforts have explored whether the same transformer architecture can produce embeddings for dna sequences. such models have shown early promise in classifying short dna sequences, such as detecting coding/non-coding regions, and enhancer, promoter sequences. however, performance at sequence classification tasks does not translate to sequence alignment, where it is necessary to search across the genome to align each read, a significantly longer-range task. we bridge this gap by framing the sequence alignment task for transformer models as an \"embed-search-align\" task. in this framework, a novel reference-free dna embedding model generates embeddings of reads and reference fragments, which are projected into a shared vector space where the read-fragment distance is used as a surrogate for alignment. technical contributions include: (1) contrastive loss for self-supervised training of dna sequence representations, facilitating rich reference-free, sequence-level embeddings, and (2) a dna vector store to enable search across fragments on a global scale. dna-esa is 99% accurate when aligning 250-length reads onto a human genome (3gb), rivaling conventional methods such as bowtie and bwa-mem. dna-esa exceeds the performance of 6 transformer model baselines such as nucleotide transformer, hyena-dna, and shows task transfer across chromosomes and species.',\n",
       "  'categories': 'q-bio.gn cs.ai',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-20',\n",
       "  'updated': '2024-10-18',\n",
       "  'authors': ['pavan holur',\n",
       "   'k. c. enevoldsen',\n",
       "   'shreyas rajesh',\n",
       "   'lajoyce mboning',\n",
       "   'thalia georgiou',\n",
       "   'louis-s. bouchard',\n",
       "   'matteo pellegrini',\n",
       "   'vwani roychowdhury'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.11087'},\n",
       " {'title': 'biological insights from ril-seq in bacteria',\n",
       "  'id': '2309.11399',\n",
       "  'abstract': 'bacteria reside in constantly changing environments and require rapid and precise adjustments of gene expression to ensure survival. small regulatory rnas (srnas) are a crucial element that bacteria utilize to achieve this. srnas are short rna molecules that modulate gene expression usually through base-pairing interactions with target rnas, primarily mrnas. these interactions can lead to either negative outcomes such as mrna degradation or translational repression or positive outcomes such as mrna stabilization or translation enhancement. in recent years, high-throughput approaches such as ril-seq (rna interaction by ligation and sequencing) revolutionized the srna field by enabling the identification of srna targets on a global scale, unveiling intricate srna-rna networks. in this review, we discuss the insights gained from investigating srna-rna networks in well-studied bacterial species as well as in under-studied bacterial species. having a complete understanding of srna-mediated regulation is critical for the development of new strategies for controlling bacterial growth and combating bacterial infections.',\n",
       "  'categories': 'q-bio.mn',\n",
       "  'doi': '10.1007/978-1-0716-4192-7_11',\n",
       "  'created': '2023-09-20',\n",
       "  'updated': '2024-11-24',\n",
       "  'authors': ['aviezer silverman', 'sahar melamed'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.11399'},\n",
       " {'title': 'brain-inspired computing with fluidic iontronic nanochannels',\n",
       "  'id': '2309.11438',\n",
       "  'abstract': \"the brain's remarkable and efficient information processing capability is driving research into brain-inspired (neuromorphic) computing paradigms. artificial aqueous ion channels are emerging as an exciting platform for neuromorphic computing, representing a departure from conventional solid-state devices by directly mimicking the brain's fluidic ion transport. supported by a quantitative theoretical model, we present easy to fabricate tapered microchannels that embed a conducting network of fluidic nanochannels between a colloidal structure. due to transient salt concentration polarisation our devices are volatile memristors (memory resistors) that are remarkably stable. the voltage-driven net salt flux and accumulation, that underpin the concentration polarisation, surprisingly combine into a diffusionlike quadratic dependence of the memory retention time on the channel length, allowing channel design for a specific timescale. we implement our device as a synaptic element for neuromorphic reservoir computing. individual channels distinguish various time series, that together represent (handwritten) numbers, for subsequent in-silico classification with a simple readout function. our results represent a significant step towards realising the promise of fluidic ion channels as a platform to emulate the rich aqueous dynamics of the brain.\",\n",
       "  'categories': 'cond-mat.soft q-bio.nc',\n",
       "  'doi': '10.1073/pnas.2320242121',\n",
       "  'created': '2023-09-20',\n",
       "  'updated': '2024-04-25',\n",
       "  'authors': ['t. m. kamsma',\n",
       "   'j. kim',\n",
       "   'k. kim',\n",
       "   'w. q. boon',\n",
       "   'c. spitoni',\n",
       "   'j. park',\n",
       "   'r. van roij'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.11438'},\n",
       " {'title': 'four universal growth regimes in degree-dependent first passage   percolation on spatial random graphs i',\n",
       "  'id': '2309.11840',\n",
       "  'abstract': 'one-dependent first passage percolation is a spreading process on a graph where the transmission time through each edge depends on the direct surroundings of the edge. in particular, the classical iid transmission time $l_{xy}$ is multiplied by $(w_xw_y)^\\\\mu$, a polynomial of the expected degrees $w_x, w_y$ of the endpoints of the edge $xy$, which we call the penalty function. beyond the markov case, we also allow any distribution for $l_{xy}$ with regularly varying distribution near $0$. we then run this process on three spatial scale-free random graph models: finite and infinite geometric inhomogeneous random graphs, and scale-free percolation. in these spatial models, the connection probability between two vertices depends on their spatial distance and on their expected degrees.   we show that as the penalty-function, i.e., $\\\\mu$ increases, the transmission time between two far away vertices sweeps through four universal phases: explosive (with tight transmission times), polylogarithmic, polynomial but strictly sublinear, and linear in the euclidean distance. the strictly polynomial growth phase here is a new phenomenon that so far was extremely rare in spatial graph models. the four growth phases are highly robust in the model parameters and are not restricted to phase boundaries. further, the transition points between the phases depend non-trivially on the main model parameters: the tail of the degree distribution, a long-range parameter governing the presence of long edges, and the behaviour of the distribution $l$ near $0$. in this paper we develop new methods to prove the upper bounds in all sub-explosive phases. our companion paper complements these results by providing matching lower bounds in the polynomial and linear regimes.',\n",
       "  'categories': 'math.pr cs.si math.co q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-21',\n",
       "  'updated': '2024-03-23',\n",
       "  'authors': ['júlia komjáthy',\n",
       "   'john lapinskas',\n",
       "   'johannes lengler',\n",
       "   'ulysse schaller'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.11840'},\n",
       " {'title': 'polynomial growth in degree-dependent first passage percolation on   spatial random graphs',\n",
       "  'id': '2309.11880',\n",
       "  'abstract': 'in this paper we study a version of (non-markovian) first passage percolation on graphs, where the transmission time between two connected vertices is non-iid, but increases by a penalty factor polynomial in their expected degrees. based on the exponent of the penalty-polynomial, this makes it increasingly harder to transmit to and from high-degree vertices. this choice is motivated by awareness or time-limitations. for the iid part of the transmission times we allow any nonnegative distribution with regularly varying behaviour at $0$. for the underlying graph models we choose spatial random graphs that have power-law degree distributions, so that the effect of the penalisation becomes visible: (finite and infinite) geometric inhomogeneous random graphs, and scale-free percolation. in these spatial models, the connection probability between two vertices depends on their spatial distance and on their expected degrees. we prove that upon increasing the penalty exponent, the transmission time between two far away vertices $x,y$ sweeps through four universal phases even for a single underlying graph: explosive (tight transmission times), polylogarithmic, polynomial but sublinear ($|x-y|^{\\\\eta_0+o(1)}$ for an explicit $\\\\eta_0<1$), and linear ($\\\\theta(|x-y|)$) in their euclidean distance. further, none of these phases are restricted to phase boundaries, and those are non-trivial in the main model parameters: the tail of the degree-distribution, a long-range parameter, and the exponent of regular variation of the iid part of the transmission times. in this paper we present proofs of lower bounds for the latter two phases and the upper bound for the linear phase. these complement the matching upper bounds for the polynomial regime in our companion paper.',\n",
       "  'categories': 'math.pr cs.si math.co q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-21',\n",
       "  'updated': '2024-10-02',\n",
       "  'authors': ['júlia komjáthy',\n",
       "   'john lapinskas',\n",
       "   'johannes lengler',\n",
       "   'ulysse schaller'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.11880'},\n",
       " {'title': 'genetic composition of supercritical branching populations under power   law mutation rates',\n",
       "  'id': '2309.12055',\n",
       "  'abstract': 'we aim at understanding the evolution of the genetic composition of cancer cell populations. to this aim, we consider a branching individual based model representing a cell population where cells divide, die and mutate along the edges of a finite directed graph $(v,e)$. the process starts with only one cell of trait $0$. following typical parameter values in cancer cell populations we study the model under \\\\textit{large population and power law mutation rates limit}, in the sense that the mutation probabilities are parameterized by negative powers of $n$ and the typical sizes of the population of interest are positive powers of $n$. under \\\\textit{non-increasing growth rate condition}, we describe the time evolution of the first-order asymptotics of the size of each subpopulation on the $log(n)$ time scale, as well as in the random time scale at which the initial population, resp. the total population, reaches the size $n^{t}$. in particular, such results allow for the perfect characterization of evolutionary pathways. without imposing any conditions on the growth rates, we describe the time evolution of the order of magnitude of each subpopulation, whose asymptotic limits are positive non-decreasing piecewise linear continuous functions.',\n",
       "  'categories': 'math.pr q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-21',\n",
       "  'updated': '2024-02-13',\n",
       "  'authors': ['vianney brouard'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.12055'},\n",
       " {'title': 'identifying vegetation patterns for a qualitative assessment of land   degradation using a cellular automata model and satellite imagery',\n",
       "  'id': '2309.12232',\n",
       "  'abstract': \"we aim to identify the spatial distribution of vegetation and its growth dynamics with the purpose of obtaining a qualitative assessment of vegetation characteristics tied to its condition, productivity and health, and to land degradation. to do so, we compare a statistical model of vegetation growth and land surface imagery derived vegetation indices. specifically, we analyze a stochastic cellular automata model and data obtained from satellite images, namely using the normalized difference vegetation index (ndvi) and the leaf area index (lai). in the experimental data, we look for areas where vegetation is broken into small patches and qualitatively compare it to the percolating, fragmented, and degraded states that appear in the cellular automata model. we model the periodic effect of seasons, finding numerical evidence of a periodic fragmentation and recovery phenomenology if the model parameters are sufficiently close to the model's percolation transition. we qualitatively recognize these effects in real-world vegetation images and consider them a signal of increased environmental stress and vulnerability. finally, we show an estimation of the environmental stress in land images by considering both the vegetation density and its clusterization.\",\n",
       "  'categories': 'cond-mat.stat-mech physics.soc-ph q-bio.pe',\n",
       "  'doi': '10.1103/physreve.110.024136',\n",
       "  'created': '2023-09-21',\n",
       "  'updated': '2024-09-13',\n",
       "  'authors': ['hediye yarahmadi',\n",
       "   'yves desille',\n",
       "   'john goold',\n",
       "   'francesca pietracaprina'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.12232'},\n",
       " {'title': 'mathematical modelling of the first hiv/zikv co-infection cases in   colombia and brazil',\n",
       "  'id': '2309.12385',\n",
       "  'abstract': 'this paper presents a mathematical model to investigate co-infection with hiv/aids and zika virus (zikv) in colombia and brazil, where the first cases were reported in 2015-2016. the model considers the sexual transmission dynamics of both viruses and vector-host interactions. we begin by exploring the qualitative behaviour of each model separately. then, we analyze the dynamics of the co-infection model using the thresholds and results defined separately for each model. the model also considers the impact of intervention strategies, such as, personal protection, antiretroviral therapy (art), and sexual protection (condoms use). using available parameter values for colombia and brazil, the model is calibrated to predict the potential effect of implementing combinations of those intervention strategies on the co-infection spread. according to these findings, transmission through sexual contact is a determining factor in the long-term behaviour of these two diseases. furthermore, it is important to note that co-infection with hiv and zikv may result in higher rates of hiv transmission and an increased risk of severe congenital disabilities linked to zikv infection. as a result, control measures have been implemented to limit the number of infected individuals and mosquitoes, with the aim of halting disease transmission. this study provides novel insights into the dynamics of hiv/zikv co-infection and highlights the importance of integrated intervention strategies in controlling the spread of these viruses, which may impact public health',\n",
       "  'categories': 'q-bio.pe math.ds',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-21',\n",
       "  'updated': '2024-04-15',\n",
       "  'authors': ['jhoana p. romero-leiton',\n",
       "   'idriss sekkak',\n",
       "   'julien arino',\n",
       "   'bouchra nasri'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.12385'},\n",
       " {'title': 'quantifying harmony between direct and indirect pathways in the basal   ganglia; healthy and parkinsonian states',\n",
       "  'id': '2309.12540',\n",
       "  'abstract': 'the basal ganglia (bg) show a variety of functions for motor and cognition. there are two competitive pathways in the bg; direct pathway (dp) which facilitates movement and indirect pathway (ip) which suppresses movement. it is well known that diverse functions of the bg may be made through \"balance\" between dp and ip. but, to the best of our knowledge, so far no quantitative analysis for such balance was done. in this paper, as a first time, we introduce the competition degree ${\\\\cal c}_d$ between dp and ip. then, by employing ${\\\\cal c}_d$, we quantify their competitive harmony (i.e., competition and cooperative interplay), which could lead to improving our understanding of the traditional \"balance\" so clearly and quantitatively. we first consider the case of normal dopamine (da) level of $\\\\phi^*=0.3$. in the case of phasic cortical input (10 hz), a healthy state with ${\\\\cal c}_d^* = 2.82$ (i.e., dp is 2.82 times stronger than ip) appears. in this case, normal movement occurs via harmony between dp and ip. next, we consider the case of decreased da level, $\\\\phi = \\\\phi^*(=0.3)~x_{da}$ ($1 > x_{da} \\\\geq 0$). with decreasing $x_{da}$ from 1, the competition degree ${\\\\cal c}_d$ between dp and ip decreases monotonically from ${\\\\cal c}_d^*$, which results in appearance of a pathological parkinsonian state with reduced ${\\\\cal c}_d$. in this parkinsonian state, strength of ip is much increased than that in the case of normal healthy state, leading to disharmony between dp and ip. due to such break-up of harmony between dp and ip, impaired movement occurs. finally, we also study treatment of the pathological parkinsonian state via recovery of harmony between dp and ip.',\n",
       "  'categories': 'q-bio.nc physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-21',\n",
       "  'updated': '2024-04-12',\n",
       "  'authors': ['sang-yoon kim', 'woochang lim'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.12540'},\n",
       " {'title': 'the role of the nucleus for cell mechanics: an elastic phase field   approach',\n",
       "  'id': '2309.12777',\n",
       "  'abstract': 'the nucleus of eukaryotic cells typically makes up around 30% of the cell volume and has significantly different mechanics, which can make it effectively up to ten times stiffer than the surrounding cytoplasm. therefore it is an important element for cell mechanics, but a quantitative understanding of its mechanical role during whole cell dynamics is largely missing. here we demonstrate that elastic phase fields can be used to describe dynamical cell processes in adhesive or confining environments in which the nucleus acts as a stiff inclusion. we first introduce and verify our computational method and then study several applications of large relevance. for cells on adhesive patterns, we find that nuclear stress is shielded by the adhesive pattern. for cell compression between two parallel plates, we obtain force-compression curves that allow us to extract an effective modulus for the cell-nucleus composite. for micropipette aspiration, the effect of the nucleus on the effective modulus is found to be much weaker, highlighting the complicated interplay between extracellular geometry and cell mechanics that is captured by our approach. we also show that our phase field approach can be used to investigate the effects of viscoelasticity and cortical tension.',\n",
       "  'categories': 'cond-mat.soft q-bio.cb',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-22',\n",
       "  'updated': '2024-03-26',\n",
       "  'authors': ['robert chojowski', 'ulrich s. schwarz', 'falko ziebert'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.12777'},\n",
       " {'title': 'emergent mechanisms for long timescales depend on training curriculum   and affect performance in memory tasks',\n",
       "  'id': '2309.12927',\n",
       "  'abstract': \"recurrent neural networks (rnns) in the brain and in silico excel at solving tasks with intricate temporal dependencies. long timescales required for solving such tasks can arise from properties of individual neurons (single-neuron timescale, $\\\\tau$, e.g., membrane time constant in biological neurons) or recurrent interactions among them (network-mediated timescale). however, the contribution of each mechanism for optimally solving memory-dependent tasks remains poorly understood. here, we train rnns to solve $n$-parity and $n$-delayed match-to-sample tasks with increasing memory requirements controlled by $n$ by simultaneously optimizing recurrent weights and $\\\\tau$s. we find that for both tasks rnns develop longer timescales with increasing $n$, but depending on the learning objective, they use different mechanisms. two distinct curricula define learning objectives: sequential learning of a single-$n$ (single-head) or simultaneous learning of multiple $n$s (multi-head). single-head networks increase their $\\\\tau$ with $n$ and are able to solve tasks for large $n$, but they suffer from catastrophic forgetting. however, multi-head networks, which are explicitly required to hold multiple concurrent memories, keep $\\\\tau$ constant and develop longer timescales through recurrent connectivity. moreover, we show that the multi-head curriculum increases training speed and network stability to ablations and perturbations, and allows rnns to generalize better to tasks beyond their training regime. this curriculum also significantly improves training grus and lstms for large-$n$ tasks. our results suggest that adapting timescales to task requirements via recurrent interactions allows learning more complex objectives and improves the rnn's performance.\",\n",
       "  'categories': 'cs.ne q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-22',\n",
       "  'updated': '2024-10-30',\n",
       "  'authors': ['sina khajehabdollahi',\n",
       "   'roxana zeraati',\n",
       "   'emmanouil giannakakis',\n",
       "   'tim jakob schäfer',\n",
       "   'georg martius',\n",
       "   'anna levina'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.12927'},\n",
       " {'title': 'graph-based permutation patterns for the analysis of task-related fmri   signals on dti networks in mild cognitive impairment',\n",
       "  'id': '2309.13083',\n",
       "  'abstract': 'permutation entropy ($pe$) is a powerful nonlinear analysis technique for univariate time series. recently, permutation entropy for graph signals ($peg$) has been proposed to extend pe to data residing on irregular domains. however, $peg$ is limited as it provides a single value to characterise a whole graph signal. here, we introduce a novel approach to evaluate graph signals \\\\emph{at the vertex level}: graph-based permutation patterns. synthetic datasets show the efficacy of our method. we reveal that dynamics in graph signals, undetectable with $peg$, can be discerned using our graph-based patterns. these are then validated in dti and fmri data acquired during a working memory task in mild cognitive impairment, where we explore functional brain signals on structural white matter networks. our findings suggest that graph-based permutation patterns in individual brain regions change as the disease progresses, demonstrating potential as a method of analyzing graph-signals at a granular scale.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-21',\n",
       "  'updated': '2024-01-16',\n",
       "  'authors': ['john stewart fabila-carrasco',\n",
       "   'avalon campbell-cousins',\n",
       "   'mario a. parra-rodriguez',\n",
       "   'javier escudero'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.13083'},\n",
       " {'title': 'forecasting treatment response with deep pharmacokinetic encoders',\n",
       "  'id': '2309.13135',\n",
       "  'abstract': 'forecasting healthcare time series data is vital for early detection of adverse outcomes and patient monitoring. however, forecasting is challenging in practice due to variable medication administration and unique pharmacokinetic (pk) properties for each patient. to address these challenges, we propose a novel hybrid global-local architecture and a pk encoder that informs deep learning models of patient-specific treatment effects. we showcase the efficacy of our approach in achieving significant accuracy gains for a blood glucose forecasting task using both realistically simulated and real-world data. our hybrid global-local architecture improves over patient-specific models by 15.8% on average. additionally, our pk encoder surpasses baselines by up to 16.4% on simulated data and 4.9% on real-world data for individual patients during critical events of severely high and low glucose levels.',\n",
       "  'categories': 'cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-22',\n",
       "  'updated': '2024-11-02',\n",
       "  'authors': ['willa potosnak',\n",
       "   'cristian challu',\n",
       "   'kin gutierrez olivares',\n",
       "   'keith dufendach',\n",
       "   'artur dubrawski'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.13135'},\n",
       " {'title': 'a closer look at parameter identifiability, model selection and handling   of censored data with bayesian inference in mathematical models of tumour   growth',\n",
       "  'id': '2309.13319',\n",
       "  'abstract': 'mathematical models (mms) are a powerful tool to help us understand and predict the dynamics of tumour growth under various conditions. in this work, we use 5 mms with an increasing number of parameters to explore how certain (often overlooked) decisions in estimating parameters from data of experimental tumour growth affect the outcome of the analysis. in particular, we propose a framework for including tumour volume measurements that fall outside the upper and lower limits of detection, which are normally discarded. we demonstrate how excluding censored data results in an overestimation of the initial tumour volume and the mm-predicted tumour volumes prior to the first measurements, and an underestimation of the carrying capacity and the mm-predicted tumour volumes beyond the latest measurable time points. we show in which way the choice of prior for the mm parameters can impact the posterior distributions, and illustrate that reporting the highest-likelihood parameters and their 95% credible interval can lead to confusing or misleading interpretations. we hope this work will encourage others to carefully consider choices made in parameter estimation and to adopt the approaches we put forward herein.',\n",
       "  'categories': 'q-bio.qm',\n",
       "  'doi': '10.1038/s41540-024-00409-6',\n",
       "  'created': '2023-09-23',\n",
       "  'updated': '',\n",
       "  'authors': ['jamie porthiyas',\n",
       "   'daniel nussey',\n",
       "   'catherine a. a. beauchemin',\n",
       "   'donald c. warren',\n",
       "   'christian quirouette',\n",
       "   'kathleen p. wilkie'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.13319'},\n",
       " {'title': 'sars-cov-2 wastewater genomic surveillance: approaches, challenges, and   opportunities',\n",
       "  'id': '2309.13326',\n",
       "  'abstract': 'during the sars-cov-2 pandemic, wastewater-based genomic surveillance (wwgs) emerged as an efficient viral surveillance tool that takes into account asymptomatic cases and can identify known and novel mutations and offers the opportunity to assign known virus lineages based on the detected mutations profiles. wwgs can also hint towards novel or cryptic lineages, but it is difficult to clearly identify and define novel lineages from wastewater (ww) alone. while wwgs has significant advantages in monitoring sars-cov-2 viral spread, technical challenges remain, including poor sequencing coverage and quality due to viral rna degradation. as a result, the viral rnas in wastewater have low concentrations and are often fragmented, making sequencing difficult. wwgs analysis requires advanced computational tools that are yet to be developed and benchmarked. the existing bioinformatics tools used to analyze wastewater sequencing data are often based on previously developed methods for quantifying the expression of transcripts or viral diversity. those methods were not developed for wastewater sequencing data specifically, and are not optimized to address unique challenges associated with wastewater. while specialized tools for analysis of wastewater sequencing data have also been developed recently, it remains to be seen how they will perform given the ongoing evolution of sars-cov-2 and the decline in testing and patient-based genomic surveillance. here, we discuss opportunities and challenges associated with wwgs, including sample preparation, sequencing technology, and bioinformatics methods.',\n",
       "  'categories': 'q-bio.gn',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-23',\n",
       "  'updated': '2024-01-30',\n",
       "  'authors': ['viorel munteanu',\n",
       "   'michael saldana',\n",
       "   'dumitru ciorba',\n",
       "   'viorel bostan',\n",
       "   'justin maine su',\n",
       "   'nadiia kasianchuk',\n",
       "   'nitesh kumar sharma',\n",
       "   'sergey knyazev',\n",
       "   'victor gordeev',\n",
       "   'eva aßmann',\n",
       "   'andrei lobiuc',\n",
       "   'mihai covasa',\n",
       "   'keith a. crandall',\n",
       "   'wenhao o. ouyang',\n",
       "   'nicholas c. wu',\n",
       "   'christopher mason',\n",
       "   'braden t tierney',\n",
       "   'alexander g lucaci',\n",
       "   'alex zelikovsky',\n",
       "   'fatemeh mohebbi',\n",
       "   'pavel skums',\n",
       "   'cynthia gibas',\n",
       "   'jessica schlueter',\n",
       "   'piotr rzymski',\n",
       "   'helena solo-gabriele',\n",
       "   'martin hölzer',\n",
       "   'adam smith',\n",
       "   'serghei mangul'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.13326'},\n",
       " {'title': 'categorization and analysis of 14 computational methods for estimating   cell potency from single-cell rna-seq data',\n",
       "  'id': '2309.13518',\n",
       "  'abstract': 'in single-cell rna sequencing (scrna-seq) analysis, a key challenge is inferring hidden cellular dynamics from static cell snapshots. various computational methods have been developed to address this, focusing on perspectives like pseudotime trajectories, rna velocities, and estimating the differentiation potential of cells, often referred to as \"cell potency.\" this review summarizes 14 methods for defining cell potency from scrna-seq data, categorizing them into average-based, entropy-based, and correlation-based methods based on how they summarize gene expression levels into a potency measure. we highlight the key similarities and differences within and between these categories, offering a high-level intuition for each method. additionally, we use unified mathematical notations to detail each method\\'s methodology and summarize their usage complexities, including parameters, required inputs, and differences between published descriptions and software implementations. we conclude that cell potency estimation remains an open question without a consensus on the optimal approach, emphasizing the need for benchmark datasets and studies. this review aims to provide a foundation for future benchmark studies, while also addressing the broader challenge of comparing methods that infer cellular dynamics from scrna-seq data through various perspectives, including pseudotime trajectories, rna velocities, and cell potency.',\n",
       "  'categories': 'q-bio.gn',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-23',\n",
       "  'updated': '2024-08-30',\n",
       "  'authors': ['qingyang wang',\n",
       "   'zhiqian zhai',\n",
       "   'qiuyu lian',\n",
       "   'dongyuan song',\n",
       "   'jingyi jessica li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.13518'},\n",
       " {'title': 'the rashomon importance distribution: getting rid of unstable, single   model-based variable importance',\n",
       "  'id': '2309.13775',\n",
       "  'abstract': 'quantifying variable importance is essential for answering high-stakes questions in fields like genetics, public policy, and medicine. current methods generally calculate variable importance for a given model trained on a given dataset. however, for a given dataset, there may be many models that explain the target outcome equally well; without accounting for all possible explanations, different researchers may arrive at many conflicting yet equally valid conclusions given the same data. additionally, even when accounting for all possible explanations for a given dataset, these insights may not generalize because not all good explanations are stable across reasonable data perturbations. we propose a new variable importance framework that quantifies the importance of a variable across the set of all good models and is stable across the data distribution. our framework is extremely flexible and can be integrated with most existing model classes and global variable importance metrics. we demonstrate through experiments that our framework recovers variable importance rankings for complex simulation setups where other methods fail. further, we show that our framework accurately estimates the true importance of a variable for the underlying data distribution. we provide theoretical guarantees on the consistency and finite sample error rates for our estimator. finally, we demonstrate its utility with a real-world case study exploring which genes are important for predicting hiv load in persons with hiv, highlighting an important gene that has not previously been studied in connection with hiv. code is available at https://github.com/jdonnelly36/rashomon_importance_distribution.',\n",
       "  'categories': 'cs.lg q-bio.gn stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-24',\n",
       "  'updated': '2024-04-01',\n",
       "  'authors': ['jon donnelly',\n",
       "   'srikar katta',\n",
       "   'cynthia rudin',\n",
       "   'edward p. browne'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.13775'},\n",
       " {'title': 'beam enumeration: probabilistic explainability for sample efficient   self-conditioned molecular design',\n",
       "  'id': '2309.13957',\n",
       "  'abstract': 'generative molecular design has moved from proof-of-concept to real-world applicability, as marked by the surge in very recent papers reporting experimental validation. key challenges in explainability and sample efficiency present opportunities to enhance generative design to directly optimize expensive high-fidelity oracles and provide actionable insights to domain experts. here, we propose beam enumeration to exhaustively enumerate the most probable sub-sequences from language-based molecular generative models and show that molecular substructures can be extracted. when coupled with reinforcement learning, extracted substructures become meaningful, providing a source of explainability and improving sample efficiency through self-conditioned generation. beam enumeration is generally applicable to any language-based molecular generative model and notably further improves the performance of the recently reported augmented memory algorithm, which achieved the new state-of-the-art on the practical molecular optimization benchmark for sample efficiency. the combined algorithm generates more high reward molecules and faster, given a fixed oracle budget. beam enumeration shows that improvements to explainability and sample efficiency for molecular design can be made synergistic.',\n",
       "  'categories': 'q-bio.bm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-25',\n",
       "  'updated': '2024-03-03',\n",
       "  'authors': ['jeff guo', 'philippe schwaller'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.13957'},\n",
       " {'title': 'using a probabilistic approach to derive a two-phase model of   flow-induced cell migration',\n",
       "  'id': '2309.13982',\n",
       "  'abstract': 'interstitial fluid flow is a feature of many solid tumours. in vitro experiments have shown that such fluid flow can direct tumour cell movement upstream or downstream depending on the balance between the competing mechanisms of tensotaxis and autologous chemotaxis. in this work we develop a probabilistic-continuum, two-phase model for cell migration in response to interstitial flow. we use a kinetic description for the cell-velocity probability density function, and model the flow-dependent stimuli as forcing terms which bias cell migration upstream and downstream. using velocity-space averaging, we reformulate the model as a system of continuum equations for the spatio-temporal evolution of the cell volume fraction and flux, in response to forcing terms which depend on the local direction and magnitude of the mechanochemical cues. we specialise our model to describe a one-dimensional cell layer subject to fluid flow. using a combination of numerical simulations and asymptotic analysis, we delineate the parameter regime where transitions from downstream to upstream cell migration occur. as has been observed experimentally, the model predicts downstream-oriented, chemotactic migration at low cell volume fractions, and upstream-oriented, tensotactic migration at larger volume fractions. we show that the locus of the critical volume fraction, at which the system transitions from downstream to upstream migration, is dominated by the ratio of the rate of chemokine secretion and advection. our model also predicts that, because the tensotactic stimulus depends strongly on the cell volume fraction, upstream, tensotaxis-dominated migration occurs only transiently when the cells are initially seeded, and transitions to downstream, chemotaxis-dominated migration occur at later times due to the dispersive effect of cell diffusion.',\n",
       "  'categories': 'q-bio.cb',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-25',\n",
       "  'updated': '2024-02-20',\n",
       "  'authors': ['yaron ben-ami',\n",
       "   'joe m. pitt-francis',\n",
       "   'philip k. maini',\n",
       "   'helen m. byrne'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.13982'},\n",
       " {'title': 'a stochastic explanation for observed local-to-global foraging states in   caenorhabditis elegans',\n",
       "  'id': '2309.15174',\n",
       "  'abstract': 'abrupt changes in behavior can often be associated with changes in underlying behavioral states. when placed off food, the foraging behavior of c. elegans can be described as a change between an initial local-search behavior characterized by a high rate of reorientations, followed by a global-search behavior characterized by sparse reorientations. this is commonly observed in individual worms, but when numerous worms are characterized, only about half appear to exhibit this behavior. we propose an alternative model that predicts both abrupt and continuous changes to reorientation that does not rely on behavioral states. this model is inspired by molecular dynamics modeling that defines the foraging reorientation rate as a decaying parameter. by stochastically sampling from the probability distribution defined by this rate, both abrupt and gradual changes to reorientation rates can occur, matching experimentally observed results. crucially, this model does not depend on behavioral states or information accumulation. even though abrupt behavioral changes do occur, they may not necessarily be indicative of abrupt changes in behavioral states, especially when abrupt changes are not universally observed in the population.',\n",
       "  'categories': 'q-bio.nc q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-26',\n",
       "  'updated': '2024-10-30',\n",
       "  'authors': ['andrew margolis', 'andrew gordus'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.15174'},\n",
       " {'title': 'apis: a paired ct-mri dataset for ischemic stroke segmentation challenge',\n",
       "  'id': '2309.15243',\n",
       "  'abstract': 'stroke is the second leading cause of mortality worldwide. immediate attention and diagnosis play a crucial role regarding patient prognosis. the key to diagnosis consists in localizing and delineating brain lesions. standard stroke examination protocols include the initial evaluation from a non-contrast ct scan to discriminate between hemorrhage and ischemia. however, non-contrast cts may lack sensitivity in detecting subtle ischemic changes in the acute phase. as a result, complementary diffusion-weighted mri studies are captured to provide valuable insights, allowing to recover and quantify stroke lesions. this work introduced apis, the first paired public dataset with ncct and adc studies of acute ischemic stroke patients. apis was presented as a challenge at the 20th ieee international symposium on biomedical imaging 2023, where researchers were invited to propose new computational strategies that leverage paired data and deal with lesion segmentation over ct sequences. despite all the teams employing specialized deep learning tools, the results suggest that the ischemic stroke segmentation task from ncct remains challenging. the annotated dataset remains accessible to the public upon registration, inviting the scientific community to deal with stroke characterization from ncct but guided with paired dwi information.',\n",
       "  'categories': 'eess.iv cs.cv q-bio.nc',\n",
       "  'doi': '10.1038/s41598-024-71273-x',\n",
       "  'created': '2023-09-26',\n",
       "  'updated': '',\n",
       "  'authors': ['santiago gómez',\n",
       "   'daniel mantilla',\n",
       "   'gustavo garzón',\n",
       "   'edgar rangel',\n",
       "   'andrés ortiz',\n",
       "   'franklin sierra-jerez',\n",
       "   'fabio martínez'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.15243'},\n",
       " {'title': 'structural identifiability analysis of linear   reaction-advection-diffusion processes in mathematical biology',\n",
       "  'id': '2309.15326',\n",
       "  'abstract': 'effective application of mathematical models to interpret biological data and make accurate predictions often requires that model parameters are identifiable. approaches to assess the so-called structural identifiability of models are well-established for ordinary differential equation models, yet there are no commonly adopted approaches that can be applied to assess the structural identifiability of the partial differential equation (pde) models that are requisite to capture spatial features inherent to many phenomena. the differential algebra approach to structural identifiability has recently been demonstrated to be applicable to several specific pde models. in this brief article, we present general methodology for performing structural identifiability analysis on partially observed reaction-advection-diffusion (rad) pde models that are linear in the unobserved quantities. we show that the differential algebra approach can always, in theory, be applied to such models. moreover, despite the perceived complexity introduced by the addition of advection and diffusion terms, identifiability of spatial analogues of non-spatial models cannot decrease in structural identifiability. we conclude by discussing future possibilities and the computational cost of performing structural identifiability analysis on more general pde models.',\n",
       "  'categories': 'q-bio.qm stat.me',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-26',\n",
       "  'updated': '2024-02-27',\n",
       "  'authors': ['alexander p browning',\n",
       "   'maria tască',\n",
       "   'carles falcó',\n",
       "   'ruth e baker'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.15326'},\n",
       " {'title': 'density estimation via measure transport: outlook for applications in   the biological sciences',\n",
       "  'id': '2309.15366',\n",
       "  'abstract': 'one among several advantages of measure transport methods is that they allow for a unified framework for processing and analysis of data distributed according to a wide class of probability measures. within this context, we present results from computational studies aimed at assessing the potential of measure transport techniques, specifically, the use of triangular transport maps, as part of a workflow intended to support research in the biological sciences. scenarios characterized by the availability of limited amount of sample data, which are common in domains such as radiation biology, are of particular interest. we find that when estimating a distribution density function given limited amount of sample data, adaptive transport maps are advantageous. in particular, statistics gathered from computing series of adaptive transport maps, trained on a series of randomly chosen subsets of the set of available data samples, leads to uncovering information hidden in the data. as a result, in the radiation biology application considered here, this approach provides a tool for generating hypotheses about gene relationships and their dynamics under radiation exposure.',\n",
       "  'categories': 'q-bio.qm cs.lg physics.bio-ph',\n",
       "  'doi': '10.1002/sam.11687',\n",
       "  'created': '2023-09-26',\n",
       "  'updated': '2024-05-12',\n",
       "  'authors': ['vanessa lopez-marrero',\n",
       "   'patrick r. johnstone',\n",
       "   'gilchan park',\n",
       "   'xihaier luo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.15366'},\n",
       " {'title': 'node-aligned graph-to-graph (nag2g): elevating template-free deep   learning approaches in single-step retrosynthesis',\n",
       "  'id': '2309.15798',\n",
       "  'abstract': \"single-step retrosynthesis (ssr) in organic chemistry is increasingly benefiting from deep learning (dl) techniques in computer-aided synthesis design. while template-free dl models are flexible and promising for retrosynthesis prediction, they often ignore vital 2d molecular information and struggle with atom alignment for node generation, resulting in lower performance compared to the template-based and semi-template-based methods. to address these issues, we introduce node-aligned graph-to-graph (nag2g), a transformer-based template-free dl model. nag2g combines 2d molecular graphs and 3d conformations to retain comprehensive molecular details and incorporates product-reactant atom mapping through node alignment which determines the order of the node-by-node graph outputs process in an auto-regressive manner. through rigorous benchmarking and detailed case studies, we have demonstrated that nag2g stands out with its remarkable predictive accuracy on the expansive datasets of uspto-50k and uspto-full. moreover, the model's practical utility is underscored by its successful prediction of synthesis pathways for multiple drug candidate molecules. this not only proves nag2g's robustness but also its potential to revolutionize the prediction of complex chemical synthesis processes for future synthetic route design tasks.\",\n",
       "  'categories': 'cs.lg physics.chem-ph q-bio.qm',\n",
       "  'doi': '10.1021/jacsau.3c00737',\n",
       "  'created': '2023-09-27',\n",
       "  'updated': '2024-03-25',\n",
       "  'authors': ['lin yao',\n",
       "   'wentao guo',\n",
       "   'zhen wang',\n",
       "   'shang xiang',\n",
       "   'wentan liu',\n",
       "   'guolin ke'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.15798'},\n",
       " {'title': 'to give or not to give: pandemic vaccine donation policy',\n",
       "  'id': '2309.15950',\n",
       "  'abstract': 'the global sars-cov-2 (covid-19) pandemic highlighted the challenge of equitable vaccine distribution between high- and low-income countries. many high-income countries were reluctant or slow to distribute extra doses of the vaccine to lower-income countries via the covid-19 vaccines global access (covax) collaboration. in addition to moral objections to such vaccine nationalism, vaccine inequity during a pandemic could contribute to the evolution of new variants of the virus and possibly increase total deaths, including in the high-income countries. using the covid-19 pandemic as a case study, we use the epidemiological model of holleran et al. that incorporates virus mutation. we identify realistic scenarios under which a donor country prefers to donate vaccines before distributing them locally in order to minimize local deaths during a pandemic. we demonstrate that a nondonor-first vaccination policy can delay, sometimes dramatically, the emergence of more-contagious variants. even more surprising, donating all vaccines is sometimes better for the donor country than a sharing policy in which half of the vaccines are donated and half are retained because of the impact donation can have on delaying the emergence of a more contagious virus. nondonor-first vaccine allocation is optimal in scenarios in which the local health impact of the vaccine is limited or when delaying emergence of a variant is especially valuable. in all cases, we find that vaccine distribution is not a zero-sum game between donor and nondonor countries. thus, in addition to moral reasons to avoid vaccine nationalism, donor nations can also realize local health benefits from donating vaccines. the insights yielded by this framework can be used to guide equitable vaccine distribution in future pandemics.',\n",
       "  'categories': 'q-bio.pe math.oc physics.soc-ph',\n",
       "  'doi': '10.1016/j.puhe.2024.05.011',\n",
       "  'created': '2023-09-27',\n",
       "  'updated': '',\n",
       "  'authors': ['abraham holleran', 'susan e. martonosi', 'michael veatch'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.15950'},\n",
       " {'title': 'confidence and second-order errors in cortical circuits',\n",
       "  'id': '2309.16046',\n",
       "  'abstract': 'minimization of cortical prediction errors has been considered a key computational goal of the cerebral cortex underlying perception, action and learning. however, it is still unclear how the cortex should form and use information about uncertainty in this process. here, we formally derive neural dynamics that minimize prediction errors under the assumption that cortical areas must not only predict the activity in other areas and sensory streams but also jointly project their confidence (inverse expected uncertainty) in their predictions. in the resulting neuronal dynamics, the integration of bottom-up and top-down cortical streams is dynamically modulated based on confidence in accordance with the bayesian principle. moreover, the theory predicts the existence of cortical second-order errors, comparing confidence and actual performance. these errors are propagated through the cortical hierarchy alongside classical prediction errors and are used to learn the weights of synapses responsible for formulating confidence. we propose a detailed mapping of the theory to cortical circuitry, discuss entailed functional interpretations and provide potential directions for experimental work.',\n",
       "  'categories': 'q-bio.nc cs.ne',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-27',\n",
       "  'updated': '2024-03-26',\n",
       "  'authors': ['arno granier',\n",
       "   'mihai a. petrovici',\n",
       "   'walter senn',\n",
       "   'katharina a. wilmes'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.16046'},\n",
       " {'title': 'language models in molecular discovery',\n",
       "  'id': '2309.16235',\n",
       "  'abstract': 'the success of language models, especially transformer-based architectures, has trickled into other domains giving rise to \"scientific language models\" that operate on small molecules, proteins or polymers. in chemistry, language models contribute to accelerating the molecule discovery cycle as evidenced by promising recent findings in early-stage drug discovery. here, we review the role of language models in molecular discovery, underlining their strength in de novo drug design, property prediction and reaction chemistry. we highlight valuable open-source software assets thus lowering the entry barrier to the field of scientific language modeling. last, we sketch a vision for future molecular design that combines a chatbot interface with access to computational chemistry tools. our contribution serves as a valuable resource for researchers, chemists, and ai enthusiasts interested in understanding how language models can and will be used to accelerate chemical discovery.',\n",
       "  'categories': 'physics.chem-ph cs.ai cs.cl cs.lg q-bio.bm',\n",
       "  'doi': '10.1007/978-981-97-4828-0_7',\n",
       "  'created': '2023-09-28',\n",
       "  'updated': '',\n",
       "  'authors': ['nikita janakarajan',\n",
       "   'tim erdmann',\n",
       "   'sarath swaminathan',\n",
       "   'teodoro laino',\n",
       "   'jannis born'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.16235'},\n",
       " {'title': 'si-sd: sleep interpreter through awake-guided cross-subject semantic   decoding',\n",
       "  'id': '2309.16457',\n",
       "  'abstract': 'understanding semantic content from brain activity during sleep represents a major goal in neuroscience. while studies in rodents have shown spontaneous neural reactivation of memories during sleep, capturing the semantic content of human sleep poses a significant challenge due to the absence of well-annotated sleep datasets and the substantial differences in neural patterns between wakefulness and sleep. to address these challenges, we designed a novel cognitive neuroscience experiment and collected a comprehensive, well-annotated electroencephalography (eeg) dataset from 134 subjects during both wakefulness and sleep. leveraging this benchmark dataset, we developed si-sd that enhances sleep semantic decoding through the position-wise alignment of neural latent sequence between wakefulness and sleep. in the 15-way classification task, our model achieves 24.12% and 21.39% top-1 accuracy on unseen subjects for nrem 2/3 and rem sleep, respectively, surpassing all other baselines. with additional fine-tuning, decoding performance improves to 30.32% and 31.65%, respectively. besides, inspired by previous neuroscientific findings, we systematically analyze how the \"slow oscillation\" event impacts decoding performance in nrem 2/3 sleep -- decoding performance on unseen subjects further improves to 40.02%. together, our findings and methodologies contribute to a promising neuro-ai framework for decoding brain activity during sleep.',\n",
       "  'categories': 'cs.lg eess.sp q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-28',\n",
       "  'updated': '2024-05-19',\n",
       "  'authors': ['hui zheng',\n",
       "   'zhong-tao chen',\n",
       "   'hai-teng wang',\n",
       "   'jian-yang zhou',\n",
       "   'lin zheng',\n",
       "   'pei-yang lin',\n",
       "   'yun-zhe liu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.16457'},\n",
       " {'title': 'on modeling airborne infection risk',\n",
       "  'id': '2309.16513',\n",
       "  'abstract': 'airborne infection risk analysis is usually performed for enclosed spaces where susceptible individuals are exposed to infectious airborne respiratory droplets by inhalation. it is usually based on exponential, dose-response models of which a widely used variant is the wells-riley (wr) model. we revisit this infection-risk estimate and extend it to the population level. we use an epidemiological model where the mode of pathogen transmission, either airborne or contact, is explicitly considered. we illustrate the link between epidemiological models and the wr model. we argue that airborne infection quanta are, up to an overall density, airborne infectious respiratory droplets modified by a parameter that depends on biological properties of the pathogen, physical properties of the droplet, and behavioural parameters of the individual. we calculate the time-dependent risk to be infected during the epidemic for two scenarios. we show how the epidemic infection risk depends on the viral latent period and the event time, the time infection occurs. the infection risk follows the dynamics of the infected population. as the latency period decreases, infection risk increases. the longer a susceptible is present in the epidemic, the higher is its risk of infection by equal exposure time to the mode of transmission.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1098/rsos.231976',\n",
       "  'created': '2023-09-28',\n",
       "  'updated': '2023-12-22',\n",
       "  'authors': ['yannis drossinos', 'nikolaos i. stilianakis'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.16513'},\n",
       " {'title': 'atomsurf : surface representation for learning on protein structures',\n",
       "  'id': '2309.16519',\n",
       "  'abstract': 'while there has been significant progress in evaluating and comparing different representations for learning on protein data, the role of surface-based learning approaches remains not well-understood. in particular, there is a lack of direct and fair benchmark comparison between the best available surface-based learning methods against alternative representations such as graphs. moreover, the few existing surface-based approaches either use surface information in isolation or, at best, perform global pooling between surface and graph-based architectures.   in this work, we fill this gap by first adapting a state-of-the-art surface encoder for protein learning tasks. we then perform a direct and fair comparison of the resulting method against alternative approaches within the atom3d benchmark, highlighting the limitations of pure surface-based learning. finally, we propose an integrated approach, which allows learned feature sharing between graphs and surface representations on the level of nodes and vertices $\\\\textit{across all layers}$.   we demonstrate that the resulting architecture achieves state-of-the-art results on all tasks in the atom3d benchmark, while adhering to the strict benchmark protocol, as well as more broadly on binding site identification and binding pocket classification. furthermore, we use coarsened surfaces and optimize our approach for efficiency, making our tool competitive in training and inference time with existing techniques. our code and data can be found online: $\\\\texttt{github.com/vincentx15/atomsurf}$',\n",
       "  'categories': 'cs.lg q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-28',\n",
       "  'updated': '2024-10-03',\n",
       "  'authors': ['vincent mallet',\n",
       "   'souhaib attaiki',\n",
       "   'yangyang miao',\n",
       "   'bruno correia',\n",
       "   'maks ovsjanikov'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.16519'},\n",
       " {'title': 'learnable real-time inference of molecular composition from diffuse   spectroscopy of brain tissue',\n",
       "  'id': '2309.16735',\n",
       "  'abstract': \"diffuse optical modalities such as broadband near-infrared spectroscopy (bnirs) and hyperspectral imaging (hsi) represent a promising alternative for low-cost, non-invasive, and fast monitoring of functional and structural properties of living tissue. particularly, the possibility of extracting the molecular composition of the tissue from the optical spectra in real-time deems the spectroscopy techniques as a unique diagnostic tool. however, no established method exists to streamline the inference of the biochemical composition from the optical spectrum for real-time applications such as surgical monitoring. in this paper, we analyse a machine learning technique for fast and accurate inference of changes in the molecular composition of brain tissue. we reconsider and propose modifications to the existing learnable methodology based on the beer-lambert law, which analytically connects the spectra with concentrations. we evaluate the method's applicability to linear and non-linear formulations of the beer-lambert law. the approach is tested on real data obtained from the bnirs- and hsi-based optical monitoring of brain tissue. the results demonstrate that the proposed method enables real-time molecular composition inference while maintaining the accuracy of traditional linear and non-linear optimization solvers. preliminary findings show that beer-lambert law-based spectral unmixing allows to contrast brain anatomy semantics such as the vessel tree and tumor area.\",\n",
       "  'categories': 'physics.bio-ph eess.iv q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-27',\n",
       "  'updated': '2024-08-15',\n",
       "  'authors': ['ivan ezhov',\n",
       "   'kevin scibilia',\n",
       "   'luca giannoni',\n",
       "   'florian kofler',\n",
       "   'ivan iliash',\n",
       "   'felix hsieh',\n",
       "   'suprosanna shit',\n",
       "   'charly caredda',\n",
       "   'fred lange',\n",
       "   'ilias tachtsidis',\n",
       "   'daniel rueckert'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.16735'},\n",
       " {'title': 'supervised learning models for early detection of albuminuria risk in   type-2 diabetes mellitus patients',\n",
       "  'id': '2309.16742',\n",
       "  'abstract': 'diabetes, especially t2dm, continues to be a significant health problem. one of the major concerns associated with diabetes is the development of its complications. diabetic nephropathy, one of the chronic complication of diabetes, adversely affects the kidneys, leading to kidney damage. diagnosing diabetic nephropathy involves considering various criteria, one of which is the presence of a pathologically significant quantity of albumin in urine, known as albuminuria. thus, early prediction of albuminuria in diabetic patients holds the potential for timely preventive measures. this study aimed to develop a supervised learning model to predict the risk of developing albuminuria in t2dm patients. the selected supervised learning algorithms included na\\\\\"ive bayes, support vector machine (svm), decision tree, random forest, adaboost, xgboost, and multi-layer perceptron (mlp). our private dataset, comprising 184 entries of diabetes complications risk factors, was used to train the algorithms. it consisted of 10 attributes as features and 1 attribute as the target (albuminuria). upon conducting the experiments, the mlp demonstrated superior performance compared to the other algorithms. it achieved accuracy and f1-score values as high as 0.74 and 0.75, respectively, making it suitable for screening purposes in predicting albuminuria in t2dm. nonetheless, further studies are warranted to enhance the model\\'s performance.',\n",
       "  'categories': 'cs.lg cs.ai q-bio.qm',\n",
       "  'doi': '10.1109/icaicta59291.2023.10390334',\n",
       "  'created': '2023-09-28',\n",
       "  'updated': '2024-01-27',\n",
       "  'authors': ['arief purnama muharram',\n",
       "   'dicky levenus tahapary',\n",
       "   'yeni dwi lestari',\n",
       "   'randy sarayar',\n",
       "   'valerie josephine dirjayanto'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.16742'},\n",
       " {'title': 'implicit gaussian process representation of vector fields over arbitrary   latent manifolds',\n",
       "  'id': '2309.16746',\n",
       "  'abstract': \"gaussian processes (gps) are popular nonparametric statistical models for learning unknown functions and quantifying the spatiotemporal uncertainty in data. recent works have extended gps to model scalar and vector quantities distributed over non-euclidean domains, including smooth manifolds appearing in numerous fields such as computer vision, dynamical systems, and neuroscience. however, these approaches assume that the manifold underlying the data is known, limiting their practical utility. we introduce rvgp, a generalisation of gps for learning vector signals over latent riemannian manifolds. our method uses positional encoding with eigenfunctions of the connection laplacian, associated with the tangent bundle, readily derived from common graph-based approximation of data. we demonstrate that rvgp possesses global regularity over the manifold, which allows it to super-resolve and inpaint vector fields while preserving singularities. furthermore, we use rvgp to reconstruct high-density neural dynamics derived from low-density eeg recordings in healthy individuals and alzheimer's patients. we show that vector field singularities are important disease markers and that their reconstruction leads to a comparable classification accuracy of disease states to high-density recordings. thus, our method overcomes a significant practical limitation in experimental and clinical applications.\",\n",
       "  'categories': 'cs.lg cs.ms physics.data-an q-bio.qm stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-28',\n",
       "  'updated': '2024-01-17',\n",
       "  'authors': ['robert l. peach',\n",
       "   'matteo vinao-carl',\n",
       "   'nir grossman',\n",
       "   'michael david',\n",
       "   'emma mallas',\n",
       "   'david sharp',\n",
       "   'paresh a. malhotra',\n",
       "   'pierre vandergheynst',\n",
       "   'adam gosztolai'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.16746'},\n",
       " {'title': 'intriguing properties of generative classifiers',\n",
       "  'id': '2309.16779',\n",
       "  'abstract': 'what is the best paradigm to recognize objects -- discriminative inference (fast but potentially prone to shortcut learning) or using a generative model (slow but potentially more robust)? we build on recent advances in generative modeling that turn text-to-image models into classifiers. this allows us to study their behavior and to compare them against discriminative models and human psychophysical data. we report four intriguing emergent properties of generative classifiers: they show a record-breaking human-like shape bias (99% for imagen), near human-level out-of-distribution accuracy, state-of-the-art alignment with human classification errors, and they understand certain perceptual illusions. our results indicate that while the current dominant paradigm for modeling human object recognition is discriminative inference, zero-shot generative models approximate human object recognition data surprisingly well.',\n",
       "  'categories': 'cs.cv cs.ai cs.lg q-bio.nc stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-28',\n",
       "  'updated': '2024-02-14',\n",
       "  'authors': ['priyank jaini', 'kevin clark', 'robert geirhos'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.16779'},\n",
       " {'title': 'a rigorous benchmarking of methods for sars-cov-2 lineage abundance   estimation in wastewater',\n",
       "  'id': '2309.16994',\n",
       "  'abstract': \"in light of the continuous transmission and evolution of sars-cov-2 coupled with a significant decline in clinical testing, there is a pressing need for scalable, cost-effective, long-term, passive surveillance tools to effectively monitor viral variants circulating in the population. wastewater genomic surveillance of sars-cov-2 has arrived as an alternative to clinical genomic surveillance, allowing to continuously monitor the prevalence of viral lineages in communities of various size at a fraction of the time, cost, and logistic effort and serving as an early warning system for emerging variants, critical for developed communities and especially for underserved ones. importantly, lineage prevalence estimates obtained with this approach aren't distorted by biases related to clinical testing accessibility and participation. however, the relative performance of bioinformatics methods used to measure relative lineage abundances from wastewater sequencing data is unknown, preventing both the research community and public health authorities from making informed decisions regarding computational tool selection. here, we perform comprehensive benchmarking of 18 bioinformatics methods for estimating the relative abundance of sars-cov-2 (sub)lineages in wastewater by using data from 36 in vitro mixtures of synthetic lineage and sublineage genomes. in addition, we use simulated data from 78 mixtures of lineages and sublineages co-occurring in the clinical setting with proportions mirroring their prevalence ratios observed in real data. importantly, we investigate how the accuracy of the evaluated methods is impacted by the sequencing technology used, the associated error rate, the read length, read depth, but also by the exposure of the synthetic rna mixtures to wastewater, with the goal of capturing the effects induced by the wastewater matrix, including rna fragmentation and degradation.\",\n",
       "  'categories': 'q-bio.gn',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-29',\n",
       "  'updated': '2024-01-21',\n",
       "  'authors': ['viorel munteanu',\n",
       "   'victor gordeev',\n",
       "   'michael saldana',\n",
       "   'eva aßmann',\n",
       "   'justin maine su',\n",
       "   'nicolae drabcinski',\n",
       "   'oksana zlenko',\n",
       "   'maryna kit',\n",
       "   'felicia iordachi',\n",
       "   'khooshbu kantibhai patel',\n",
       "   'abdullah al nahid',\n",
       "   'likhitha chittampalli',\n",
       "   'yidian xu',\n",
       "   'pavel skums',\n",
       "   'shelesh agrawal',\n",
       "   'martin hölzer',\n",
       "   'adam smith',\n",
       "   'alex zelikovsky',\n",
       "   'serghei mangul'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.16994'},\n",
       " {'title': 'fossil and present-day stromatolite ooids contain a meteoritic polymer   of glycine and iron',\n",
       "  'id': '2309.17195',\n",
       "  'abstract': 'hemoglycin, a space polymer of glycine and iron, has been identified in the carbonaceous chondritic meteorites allende, acfer 086, kaba, sutters mill and orgueil. its core form has a mass of 1494da and is basically an antiparallel pair of polyglycine strands linked at each end by an iron atom. the polymer forms two- and three- dimensional lattices with an inter-vertex distance of 4.9nm. here the extraction technique for meteorites is applied to a 2.1gya fossil stromatolite to reveal the presence of hemoglycin by mass spectrometry. intact ooids from a recent (3,000ya) stromatolite exhibited the same visible hemoglycin fluorescence in response to x-rays as an intact crystal from the orgueil meteorite. x-ray analysis confirmed the existence in ooids of an internal 3-dimensional lattice of 4.9nm inter-vertex spacing, matching the spacing of lattices in meteoritic crystals. ftir measurements of acid-treated ooid and a sutters mill meteoritic crystal both show the presence, via the splitting of the amide i band, of an extended anti-parallel beta sheet structure. it seems probable that the copious in-fall of carbonaceous meteoritic material, from archaean times onward, has left traces of hemoglycin in sedimentary carbonates and potentially has influenced ooid formation.',\n",
       "  'categories': 'physics.geo-ph astro-ph.ep q-bio.bm',\n",
       "  'doi': '10.1017/s1473550424000168',\n",
       "  'created': '2023-09-29',\n",
       "  'updated': '2024-03-29',\n",
       "  'authors': ['julie e m mcgeoch',\n",
       "   'anton j frommelt',\n",
       "   'robin l owen',\n",
       "   'gianfelice cinque',\n",
       "   'arthur mcclelland',\n",
       "   'david lageson',\n",
       "   'malcolm w mcgeoch'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.17195'},\n",
       " {'title': '3d-mol: a novel contrastive learning framework for molecular property   prediction with 3d information',\n",
       "  'id': '2309.17366',\n",
       "  'abstract': 'molecular property prediction, crucial for early drug candidate screening and optimization, has seen advancements with deep learning-based methods. while deep learning-based methods have advanced considerably, they often fall short in fully leveraging 3d spatial information. specifically, current molecular encoding techniques tend to inadequately extract spatial information, leading to ambiguous representations where a single one might represent multiple distinct molecules. moreover, existing molecular modeling methods focus predominantly on the most stable 3d conformations, neglecting other viable conformations present in reality. to address these issues, we propose 3d-mol, a novel approach designed for more accurate spatial structure representation. it deconstructs molecules into three hierarchical graphs to better extract geometric information. additionally, 3d-mol leverages contrastive learning for pretraining on 20 million unlabeled data, treating their conformations with identical topological structures as weighted positive pairs and contrasting ones as negatives, based on the similarity of their 3d conformation descriptors and fingerprints. we compare 3d-mol with various state-of-the-art baselines on 7 benchmarks and demonstrate our outstanding performance.',\n",
       "  'categories': 'q-bio.bm cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-09-28',\n",
       "  'updated': '2024-06-27',\n",
       "  'authors': ['taojie kuang', 'yiming ren', 'zhixiang ren'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2309.17366'},\n",
       " {'title': 'the analysis of the impact of fear in the presence of additional food   and prey refuge with nonlocal predator-prey models',\n",
       "  'id': '2310.01392',\n",
       "  'abstract': 'in a predator-prey interaction, many factors are present that affect the growth of the species, either positively or negatively. fear of predation is such a factor that causes psychological stress in a prey species, so their overall growth starts to decline. in this work, a predator-prey model is proposed where the prey species faces a reduction in their growth out of fear, and the predator is also provided with an alternative food source that helps the prey to hide in a safer place. the dynamics produce a wide range of interesting results, including the significance of the presence of a certain amount of fear or even prey refuge for population coexistence. the analysis is extended later to the nonlocal model to analyze how the non-equilibrium phenomena influence the dynamical behaviour. it is observed through numerical simulations that the scope of pattern formation reduces with the increase of fear level in the spatio-temporal model, whereas the incorporation of nonlocal interaction further increases the chance of species colonization.',\n",
       "  'categories': 'q-bio.pe math.ds',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-02',\n",
       "  'updated': '2024-06-14',\n",
       "  'authors': ['sangeeta saha', 'swadesh pal', 'roderick melnik'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.01392'},\n",
       " {'title': 'a mobile digital device proficiency performance test for cognitive   clinical research',\n",
       "  'id': '2310.01774',\n",
       "  'abstract': 'mobile device proficiency is increasingly important for everyday living, including to deliver healthcare services. human-device interactions represent a potential in cognitive neurology and aging research. although traditional pen-and-paper evaluations serve as valuable tools within public health strategies for population-scale cognitive assessments, digital devices could amplify cognitive assessment. however, even person-centered studies often fail to incorporate measures of mobile device proficiency and research with digital mobile technology frequently neglects these evaluations. besides that, cognitive screening, a fundamental part of brain health evaluation and a widely accepted strategy to identify high-risk individuals vulnerable to cognitive impairment and dementia, has research using digital devices for older adults in need for standardization. to address this shortfall, the digitau collaborative and interdisciplinary project is creating refined methodological parameters for the investigation of digital biomarkers. with careful consideration of cognitive design elements, here we describe the open-source and performance-based mobile device abilities test (mdat), a simple, low-cost, and reproductible open-sourced test framework. this result was achieved with a cross-sectional study population sample of 101 low and middle-income subjects aged 20 to 79 years old. partial least squares structural equation modeling (pls-sem) was used to assess the measurement of the construct. it was possible to achieve a reliable method with internal consistency, good content validity related to digital competences, and that does not have much interference with auto-perceived global functional disability, health self-perception, and motor dexterity. limitations for this method are discussed and paths to improve and establish better standards are highlighted.',\n",
       "  'categories': 'q-bio.nc cs.hc q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-02',\n",
       "  'updated': '2024-10-05',\n",
       "  'authors': ['alan cronemberger andrade',\n",
       "   'diógenes de souza bido',\n",
       "   'ana carolina bottura de barros',\n",
       "   'walter richard boot',\n",
       "   'paulo henrique ferreira bertolucci'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.01774'},\n",
       " {'title': 'dream: visual decoding from reversing human visual system',\n",
       "  'id': '2310.02265',\n",
       "  'abstract': 'in this work we present dream, an fmri-to-image method for reconstructing viewed images from brain activities, grounded on fundamental knowledge of the human visual system. we craft reverse pathways that emulate the hierarchical and parallel nature of how humans perceive the visual world. these tailored pathways are specialized to decipher semantics, color, and depth cues from fmri data, mirroring the forward pathways from visual stimuli to fmri recordings. to do so, two components mimic the inverse processes within the human visual system: the reverse visual association cortex (r-vac) which reverses pathways of this brain region, extracting semantics from fmri data; the reverse parallel pkm (r-pkm) component simultaneously predicting color and depth from fmri signals. the experiments indicate that our method outperforms the current state-of-the-art models in terms of the consistency of appearance, structure, and semantics. code will be made publicly available to facilitate further research in this field.',\n",
       "  'categories': 'cs.cv cs.lg eess.iv q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-03',\n",
       "  'updated': '2024-04-10',\n",
       "  'authors': ['weihao xia',\n",
       "   'raoul de charette',\n",
       "   'cengiz öztireli',\n",
       "   'jing-hao xue'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.02265'},\n",
       " {'title': 'from asynchronous states to griffiths phases and back: structural   heterogeneity and homeostasis in excitatory-inhibitory networks',\n",
       "  'id': '2310.02369',\n",
       "  'abstract': 'balanced neural networks -- in which excitatory and inhibitory inputs compensate each other on average -- give rise to a dynamical phase dominated by fluctuations called asynchronous state, crucial for brain functioning. however, structural disorder -- which is inherent to random networks -- can hinder such an excitation-inhibition balance. indeed, structural and synaptic heterogeneities can generate extended regions in phase space akin to critical points, called griffiths phases, with dynamical features very different from those of asynchronous states. here, we study a simple neural-network model with tunable levels of heterogeneity able to display these two types of dynamical regimes -- i.e., asynchronous states and griffiths phases -- putting them together within a single phase diagram. using this simple model, we are able to emphasize the crucial role played by synaptic plasticity and homeostasis to re-establish balance in intrinsically heterogeneous networks. overall, we shed light onto how diverse dynamical regimes, each with different functional advantages, can emerge from a given network as a result of self-organizing homeostatic mechanisms.',\n",
       "  'categories': 'cond-mat.stat-mech q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-03',\n",
       "  'updated': '2024-03-03',\n",
       "  'authors': ['jorge pretel',\n",
       "   'victor buendía',\n",
       "   'joaquín j. torres',\n",
       "   'miguel a. muñoz'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.02369'},\n",
       " {'title': 'numerical modeling of hydrogel scaffold anisotropy during   extrusion-based 3d printing for tissue engineering',\n",
       "  'id': '2310.02734',\n",
       "  'abstract': 'extrusion-based 3d printing is a recent and widely used tissue engineering tool that provides precise 3d control of bioinks to create organ-size biomaterial-based objects composed of hierarchically organized cellularized scaffolds. the internal organization of scaffold constituents should mimic the structural anisotropy of targeted tissue to stimulate proliferative cellular behavior during 3d cell culture. both the choice of polymers constituting the bioink and the topological properties during the extrusion process greatly influence the structural anisotropy and cellular response of tissue engineering constructs. the bioink used in our study was a hydrogel made of three constituents: fibrinogen, alginate and gelatin. these components provide biocompatibility, printability and conservation of 3d shape after printing. the topological properties in flowing polymers are dictated by macromolecule conformation i.e. their orientation and degree of stretch. in this study, we used the micro-macro approach to describe the orientation state of hydrogel macromolecules during the extrusion process, offering a two-scale description of fluid behavior. the goal of our study was to use the fokker-planck equation, which describes the evolution of probability distribution function over time, to represent the real state of the constituent population in the representative elementary volume within a hydrogel during extrusion-based 3d printing. our data suggest that for a tubular nozzle syringe, constituent orientation is driven by a high shear rate, which overcomes the fluid rheological behavior. also, the interaction coefficient, which represents the microscopic interaction between fluid particles overcomes hydrogel behavior for constituent orientation in the prediction model.',\n",
       "  'categories': 'q-bio.to q-bio.mn',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-04',\n",
       "  'updated': '2024-10-07',\n",
       "  'authors': ['v. t. mai',\n",
       "   'r. chatelin',\n",
       "   'e. -j. courtial',\n",
       "   'c. boulocher',\n",
       "   'r. rieger'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.02734'},\n",
       " {'title': 'algorithm to find new identifiable reparametrizations of parametric   rational ode models',\n",
       "  'id': '2310.03057',\n",
       "  'abstract': 'structural identifiability concerns the question of which unknown parameters of a model can be recovered from (perfect) input-output data. if all of the parameters of a model can be recovered from data, the model is said to be identifiable. however, in many models, there are parameters that can take on an infinite number of values but yield the same input-output data. in this case, those parameters and the model are called unidentifiable. the question is then what to do with an unidentifiable model. one can either adjust the model, if experimentally feasible, or try to find a reparametrization to make the model identifiable. in this paper, we take the latter approach. while existing approaches to find identifiable reparametrizations were limited to scaling reparametrizations or were not guaranteed to find a globally identifiable reparametrization even if it exists, we significantly broaden the class of models for which we can find a globally identifiable model with the same input-output behavior as the original one. we also prove that, for linear models, a globally identifiable reparametrization always exists and show that, for a certain class of linear compartmental models, an explicit reparametrization formula exists. we illustrate our method on several examples and provide detailed analysis in supplementary material on github.',\n",
       "  'categories': 'eess.sy cs.sy math.ag math.ds q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-04',\n",
       "  'updated': '2024-06-26',\n",
       "  'authors': ['nicolette meshkat', 'alexey ovchinnikov', 'thomas scanlon'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.03057'},\n",
       " {'title': 'molecule design by latent prompt transformer',\n",
       "  'id': '2310.03253',\n",
       "  'abstract': 'this paper proposes a latent prompt transformer model for solving challenging optimization problems such as molecule design, where the goal is to find molecules with optimal values of a target chemical or biological property that can be computed by an existing software. our proposed model consists of three components. (1) a latent vector whose prior distribution is modeled by a unet transformation of a gaussian white noise vector. (2) a molecule generation model that generates the string-based representation of molecule conditional on the latent vector in (1). we adopt the causal transformer model that takes the latent vector in (1) as prompt. (3) a property prediction model that predicts the value of the target property of a molecule based on a non-linear regression on the latent vector in (1). we call the proposed model the latent prompt transformer model. after initial training of the model on existing molecules and their property values, we then gradually shift the model distribution towards the region that supports desired values of the target property for the purpose of molecule design. our experiments show that our proposed model achieves state of the art performances on several benchmark molecule design tasks.',\n",
       "  'categories': 'cs.lg q-bio.bm stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-04',\n",
       "  'updated': '2024-02-05',\n",
       "  'authors': ['deqian kong', 'yuhao huang', 'jianwen xie', 'ying nian wu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.03253'},\n",
       " {'title': \"unbiased estimation of sampling variance for simpson's diversity index\",\n",
       "  'id': '2310.03439',\n",
       "  'abstract': \"quantification of measurement uncertainty is crucial for robust scientific inference, yet accurate estimates of this uncertainty remain elusive for ecological measures of diversity. here, we address this longstanding challenge by deriving a closed-form unbiased estimator for the sampling variance of simpson's diversity index. in numerical tests the estimator consistently outperforms existing approaches, particularly for applications in which species richness exceeds sample size. we apply the estimator to quantify biodiversity loss in marine ecosystems and to demonstrate ligand-dependent contributions of t cell receptor chains to specificity, illustrating its versatility across fields. the novel estimator provides researchers with a reliable method for comparing diversity between samples, essential for quantifying biodiversity trends and making informed conservation decisions.\",\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-05',\n",
       "  'updated': '2024-02-26',\n",
       "  'authors': ['andreas tiffeau-mayer'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.03439'},\n",
       " {'title': 'pepmlm: target sequence-conditioned generation of therapeutic peptide   binders via span masked language modeling',\n",
       "  'id': '2310.03842',\n",
       "  'abstract': 'target proteins that lack accessible binding pockets and conformational stability have posed increasing challenges for drug development. induced proximity strategies, such as protacs and molecular glues, have thus gained attention as pharmacological alternatives, but still require small molecule docking at binding pockets for targeted protein degradation. the computational design of protein-based binders presents unique opportunities to access \"undruggable\" targets, but have often relied on stable 3d structures or structure-influenced latent spaces for effective binder generation. in this work, we introduce pepmlm, a target sequence-conditioned generator of de novo linear peptide binders. by employing a novel span masking strategy that uniquely positions cognate peptide sequences at the c-terminus of target protein sequences, pepmlm fine-tunes the state-of-the-art esm-2 plm to fully reconstruct the binder region, achieving low perplexities matching or improving upon validated peptide-protein sequence pairs. after successful in silico benchmarking with alphafold-multimer, outperforming rfdiffusion on structured targets, we experimentally verify pepmlm\\'s efficacy via fusion of model-derived peptides to e3 ubiquitin ligase domains, demonstrating endogenous degradation of emergent viral phosphoproteins and huntington\\'s disease-driving proteins. in total, pepmlm enables the generative design of candidate binders to any target protein, without the requirement of target structure, empowering downstream therapeutic applications.',\n",
       "  'categories': 'q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-05',\n",
       "  'updated': '2024-08-11',\n",
       "  'authors': ['tianlai chen',\n",
       "   'madeleine dumas',\n",
       "   'rio watson',\n",
       "   'sophia vincoff',\n",
       "   'christina peng',\n",
       "   'lin zhao',\n",
       "   'lauren hong',\n",
       "   'sarah pertsemlidis',\n",
       "   'mayumi shaepers-cheu',\n",
       "   'tian zi wang',\n",
       "   'divya srijay',\n",
       "   'connor monticello',\n",
       "   'pranay vure',\n",
       "   'rishab pulugurta',\n",
       "   'kseniia kholina',\n",
       "   'shrey goel',\n",
       "   'matthew p. delisa',\n",
       "   'ray truant',\n",
       "   'hector c. aguilar',\n",
       "   'pranam chatterjee'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.03842'},\n",
       " {'title': 'improved prediction of ligand-protein binding affinities by   meta-modeling',\n",
       "  'id': '2310.03946',\n",
       "  'abstract': 'the accurate screening of candidate drug ligands against target proteins through computational approaches is of prime interest to drug development efforts. such virtual screening depends in part on methods to predict the binding affinity between ligands and proteins. many computational models for binding affinity prediction have been developed, but with varying results across targets. given that ensembling or meta-modeling approaches have shown great promise in reducing model-specific biases, we develop a framework to integrate published force-field-based empirical docking and sequence-based deep learning models. in building this framework, we evaluate many combinations of individual base models, training databases, and several meta-modeling approaches. we show that many of our meta-models significantly improve affinity predictions over base models. our best meta-models achieve comparable performance to state-of-the-art deep learning tools exclusively based on 3d structures, while allowing for improved database scalability and flexibility through the explicit inclusion of features such as physicochemical properties or molecular descriptors. we further demonstrate improved generalization capability by our models using a large-scale benchmark of affinity prediction as well as a virtual screening application benchmark. overall, we demonstrate that diverse modeling approaches can be ensembled together to gain meaningful improvement in binding affinity prediction.',\n",
       "  'categories': 'cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-05',\n",
       "  'updated': '2024-10-21',\n",
       "  'authors': ['ho-joon lee', 'prashant s. emani', 'mark b. gerstein'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.03946'},\n",
       " {'title': 'pgraphdta: improving drug target interaction prediction using protein   language models and contact maps',\n",
       "  'id': '2310.04017',\n",
       "  'abstract': 'developing and discovering new drugs is a complex and resource-intensive endeavor that often involves substantial costs, time investment, and safety concerns. a key aspect of drug discovery involves identifying novel drug-target (dt) interactions. existing computational methods for predicting dt interactions have primarily focused on binary classification tasks, aiming to determine whether a dt pair interacts or not. however, protein-ligand interactions exhibit a continuum of binding strengths, known as binding affinity, presenting a persistent challenge for accurate prediction. in this study, we investigate various techniques employed in drug target interaction (dti) prediction and propose novel enhancements to enhance their performance. our approaches include the integration of protein language models (plms) and the incorporation of contact map information as an inductive bias within current models. through extensive experimentation, we demonstrate that our proposed approaches outperform the baseline models considered in this study, presenting a compelling case for further development in this direction. we anticipate that the insights gained from this work will significantly narrow the search space for potential drugs targeting specific proteins, thereby accelerating drug discovery. code and data for pgraphdta are available at https://github.com/yijia-xiao/pgraphdta/.',\n",
       "  'categories': 'cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-06',\n",
       "  'updated': '2024-02-11',\n",
       "  'authors': ['rakesh bal', 'yijia xiao', 'wei wang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.04017'},\n",
       " {'title': 'from task structures to world models: what do llms know?',\n",
       "  'id': '2310.04276',\n",
       "  'abstract': 'in what sense does a large language model have knowledge? the answer to this question extends beyond the capabilities of a particular ai system, and challenges our assumptions about the nature of knowledge and intelligence. we answer by granting llms \"instrumental knowledge\"; knowledge defined by a certain set of abilities. we then ask how such knowledge is related to the more ordinary, \"worldly\" knowledge exhibited by human agents, and explore this in terms of the degree to which instrumental knowledge can be said to incorporate the structured world models of cognitive science. we discuss ways llms could recover degrees of worldly knowledge, and suggest such recovery will be governed by an implicit, resource-rational tradeoff between world models and task demands.',\n",
       "  'categories': 'cs.ai q-bio.nc',\n",
       "  'doi': '10.1016/j.tics.2024.02.008',\n",
       "  'created': '2023-10-06',\n",
       "  'updated': '',\n",
       "  'authors': ['ilker yildirim', 'l. a. paul'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.04276'},\n",
       " {'title': 'a perspective on neuroscience data standardization with neurodata   without borders',\n",
       "  'id': '2310.04317',\n",
       "  'abstract': 'neuroscience research has evolved to generate increasingly large and complex experimental data sets, and advanced data science tools are taking on central roles in neuroscience research. neurodata without borders (nwb), a standard language for neurophysiology data, has recently emerged as a powerful solution for data management, analysis, and sharing. we here discuss our efforts to implement nwb data science pipelines. we describe general principles and specific use cases that illustrate successes, challenges, and non-trivial decisions in software engineering. we hope that our experience can provide guidance for the neuroscience community and help bridge the gap between experimental neuroscience and data science.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-06',\n",
       "  'updated': '2024-01-22',\n",
       "  'authors': ['andrea pierré',\n",
       "   'tuan pham',\n",
       "   'jonah pearl',\n",
       "   'sandeep robert datta',\n",
       "   'jason t. ritt',\n",
       "   'alexander fleischmann'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.04317'},\n",
       " {'title': 'brainscuba: fine-grained natural language captions of visual cortex   selectivity',\n",
       "  'id': '2310.04420',\n",
       "  'abstract': 'understanding the functional organization of higher visual cortex is a central focus in neuroscience. past studies have primarily mapped the visual and semantic selectivity of neural populations using hand-selected stimuli, which may potentially bias results towards pre-existing hypotheses of visual cortex functionality. moving beyond conventional approaches, we introduce a data-driven method that generates natural language descriptions for images predicted to maximally activate individual voxels of interest. our method -- semantic captioning using brain alignments (\"brainscuba\") -- builds upon the rich embedding space learned by a contrastive vision-language model and utilizes a pre-trained large language model to generate interpretable captions. we validate our method through fine-grained voxel-level captioning across higher-order visual regions. we further perform text-conditioned image synthesis with the captions, and show that our images are semantically coherent and yield high predicted activations. finally, to demonstrate how our method enables scientific discovery, we perform exploratory investigations on the distribution of \"person\" representations in the brain, and discover fine-grained semantic selectivity in body-selective areas. unlike earlier studies that decode text, our method derives voxel-wise captions of semantic selectivity. our results show that brainscuba is a promising means for understanding functional preferences in the brain, and provides motivation for further hypothesis-driven investigation of visual cortex.',\n",
       "  'categories': 'cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-06',\n",
       "  'updated': '2024-05-03',\n",
       "  'authors': ['andrew f. luo',\n",
       "   'margaret m. henderson',\n",
       "   'michael j. tarr',\n",
       "   'leila wehbe'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.04420'},\n",
       " {'title': 'modeling the risk of in-person instruction during the covid-19 pandemic',\n",
       "  'id': '2310.04563',\n",
       "  'abstract': \"during the covid-19 pandemic, safely implementing in-person indoor instruction was a high priority for universities nationwide. to support this effort at the university, we developed a mathematical model for estimating the risk of sars-cov-2 transmission in university classrooms. this model was used to evaluate combinations of feasible interventions for classrooms at the university during the pandemic and optimize the set of interventions that would allow higher occupancy levels, matching the pre-pandemic numbers of in-person courses. importantly, we determined that requiring masking in dense classrooms with unrestricted seating with more than 90% of students vaccinated was easy to implement, incurred little logistical or financial cost, and allowed classes to be held at full capacity. a retrospective analysis at the end of the semester confirmed the model's assessment that the proposed classroom configuration would be safe. our framework is generalizable and was used to support reopening decisions at stanford university. in addition, our framework is flexible and applies to a wide range of indoor settings. it was repurposed for large university events and gatherings and could be used to support planning indoor space use to avoid transmission of infectious diseases across various industries, from secondary schools to movie theaters and restaurants.\",\n",
       "  'categories': 'stat.ap math.oc physics.soc-ph q-bio.qm stat.me',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-06',\n",
       "  'updated': '2024-02-19',\n",
       "  'authors': ['brian liu',\n",
       "   'yujia zhang',\n",
       "   'shane g. henderson',\n",
       "   'david b. shmoys',\n",
       "   'peter i. frazier'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.04563'},\n",
       " {'title': 'neural2speech: a transfer learning framework for neural-driven speech   reconstruction',\n",
       "  'id': '2310.04644',\n",
       "  'abstract': 'reconstructing natural speech from neural activity is vital for enabling direct communication via brain-computer interfaces. previous efforts have explored the conversion of neural recordings into speech using complex deep neural network (dnn) models trained on extensive neural recording data, which is resource-intensive under regular clinical constraints. however, achieving satisfactory performance in reconstructing speech from limited-scale neural recordings has been challenging, mainly due to the complexity of speech representations and the neural data constraints. to overcome these challenges, we propose a novel transfer learning framework for neural-driven speech reconstruction, called neural2speech, which consists of two distinct training phases. first, a speech autoencoder is pre-trained on readily available speech corpora to decode speech waveforms from the encoded speech representations. second, a lightweight adaptor is trained on the small-scale neural recordings to align the neural activity and the speech representation for decoding. remarkably, our proposed neural2speech demonstrates the feasibility of neural-driven speech reconstruction even with only 20 minutes of intracranial data, which significantly outperforms existing baseline methods in terms of speech fidelity and intelligibility.',\n",
       "  'categories': 'cs.sd eess.as q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-06',\n",
       "  'updated': '2024-01-31',\n",
       "  'authors': ['jiawei li',\n",
       "   'chunxu guo',\n",
       "   'li fu',\n",
       "   'lu fan',\n",
       "   'edward f. chang',\n",
       "   'yuanning li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.04644'},\n",
       " {'title': 'do self-supervised speech and language models extract similar   representations as human brain?',\n",
       "  'id': '2310.04645',\n",
       "  'abstract': 'speech and language models trained through self-supervised learning (ssl) demonstrate strong alignment with brain activity during speech and language perception. however, given their distinct training modalities, it remains unclear whether they correlate with the same neural aspects. we directly address this question by evaluating the brain prediction performance of two representative ssl models, wav2vec2.0 and gpt-2, designed for speech and language tasks. our findings reveal that both models accurately predict speech responses in the auditory cortex, with a significant correlation between their brain predictions. notably, shared speech contextual information between wav2vec2.0 and gpt-2 accounts for the majority of explained variance in brain activity, surpassing static semantic and lower-level acoustic-phonetic information. these results underscore the convergence of speech contextual representations in ssl models and their alignment with the neural network underlying speech perception, offering valuable insights into both ssl models and the neural basis of speech and language processing.',\n",
       "  'categories': 'q-bio.nc cs.ai cs.cl eess.as',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-06',\n",
       "  'updated': '2024-01-31',\n",
       "  'authors': ['peili chen',\n",
       "   'linyang he',\n",
       "   'li fu',\n",
       "   'lu fan',\n",
       "   'edward f. chang',\n",
       "   'yuanning li'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.04645'},\n",
       " {'title': 'keep moving: identifying task-relevant subspaces to maximise plasticity   for newly learned tasks',\n",
       "  'id': '2310.04741',\n",
       "  'abstract': \"continual learning algorithms strive to acquire new knowledge while preserving prior information. often, these algorithms emphasise stability and restrict network updates upon learning new tasks. in many cases, such restrictions come at a cost to the model's plasticity, i.e. the model's ability to adapt to the requirements of a new task. but is all change detrimental? here, we approach this question by proposing that activation spaces in neural networks can be decomposed into two subspaces: a readout range in which change affects prior tasks and a null space in which change does not alter prior performance. based on experiments with this novel technique, we show that, indeed, not all activation change is associated with forgetting. instead, only change in the subspace visible to the readout of a task can lead to decreased stability, while restricting change outside of this subspace is associated only with a loss of plasticity. analysing various commonly used algorithms, we show that regularisation-based techniques do not fully disentangle the two spaces and, as a result, restrict plasticity more than need be. we expand our results by investigating a linear model in which we can manipulate learning in the two subspaces directly and thus causally link activation changes to stability and plasticity. for hierarchical, nonlinear cases, we present an approximation that enables us to estimate functionally relevant subspaces at every layer of a deep nonlinear network, corroborating our previous insights. together, this work provides novel means to derive insights into the mechanisms behind stability and plasticity in continual learning and may serve as a diagnostic tool to guide developments of future continual learning algorithms that stabilise inference while allowing maximal space for learning.\",\n",
       "  'categories': 'cs.lg cs.cv q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-07',\n",
       "  'updated': '2024-06-20',\n",
       "  'authors': ['daniel anthes',\n",
       "   'sushrut thorat',\n",
       "   'peter könig',\n",
       "   'tim c. kietzmann'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.04741'},\n",
       " {'title': 'rawalign: accurate, fast, and scalable raw nanopore signal mapping via   combining seeding and alignment',\n",
       "  'id': '2310.05037',\n",
       "  'abstract': 'nanopore sequencers generate raw electrical signals representing the contents of a biological sequence molecule passing through the nanopore. these signals can be analyzed directly, avoiding basecalling entirely. we observe that while existing proposals for raw signal analysis typically do well in all metrics for small genomes (e.g., viral genomes), they all perform poorly for large genomes (e.g., the human genome). our goal is to analyze raw nanopore signals in an accurate, fast, and scalable manner. to this end, we propose rawalign, the first work to integrate fine-grained signal alignment into the state-of-the-art raw signal mapper. to enable accurate, fast, and scalable mapping with alignment, rawalign implements three algorithmic improvements and hardware acceleration via a vectorized implementation of fine-grained alignment. together, these significantly reduce the overhead of typically computationally expensive fine-grained alignment. our extensive evaluations on different use cases and various datasets show rawalign provides 1) the most accurate mapping for large genomes and 2) and on-par performance compared to rawhash (between 0.80x-1.08x), while achieving better performance than uncalled and sigmap by on average (geo. mean) 2.83x and 2.06x, respectively. availability: https://github.com/cmu-safari/rawalign.',\n",
       "  'categories': 'q-bio.gn q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-08',\n",
       "  'updated': '2024-10-23',\n",
       "  'authors': ['joël lindegger',\n",
       "   'can firtina',\n",
       "   'nika mansouri ghiasi',\n",
       "   'mohammad sadrosadati',\n",
       "   'mohammed alser',\n",
       "   'onur mutlu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.05037'},\n",
       " {'title': 'modeling motor control in continuous-time active inference: a survey',\n",
       "  'id': '2310.05144',\n",
       "  'abstract': 'the way the brain selects and controls actions is still widely debated. mainstream approaches based on optimal control focus on stimulus-response mappings that optimize cost functions. ideomotor theory and cybernetics propose a different perspective: they suggest that actions are selected and controlled by activating action effects and by continuously matching internal predictions with sensations. active inference offers a modern formulation of these ideas, in terms of inferential mechanisms and prediction-error-based control, which can be linked to neural mechanisms of living organisms. this article provides a technical illustration of active inference models in continuous time and a brief survey of active inference models that solve four kinds of control problems; namely, the control of goal-directed reaching movements, active sensing, the resolution of multisensory conflict during movement and the integration of decision-making and motor control. crucially, in active inference, all these different facets of motor control emerge from the same optimization process - namely, the minimization of free energy - and do not require designing separate cost functions. therefore, active inference provides a unitary perspective on various aspects of motor control that can inform both the study of biological control mechanisms and the design of artificial and robotic systems.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.1109/tcds.2023.3338491',\n",
       "  'created': '2023-10-08',\n",
       "  'updated': '',\n",
       "  'authors': ['matteo priorelli',\n",
       "   'federico maggiore',\n",
       "   'antonella maselli',\n",
       "   'francesco donnarumma',\n",
       "   'domenico maisto',\n",
       "   'francesco mannella',\n",
       "   'ivilin peev stoianov',\n",
       "   'giovanni pezzulo'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.05144'},\n",
       " {'title': \"longitudinal volumetric study for the progression of alzheimer's disease   from structural mri\",\n",
       "  'id': '2310.05558',\n",
       "  'abstract': \"alzheimer's disease (ad) is an irreversible neurodegenerative disorder affecting millions of individuals today. the prognosis of the disease solely depends on treating symptoms as they arise and proper caregiving, as there are no current medical preventative treatments apart from newly developing drugs which can, at most, slow the progression. thus, early detection of the disease at its most premature state is of paramount importance. this work aims to survey imaging biomarkers corresponding to the progression of ad and also reviews some of the existing feature extraction methods. a longitudinal study of structural mr images was performed for given temporal test subjects with ad selected randomly from the alzheimer's disease neuroimaging initiative (adni) database. a pipeline was implemented to study the data, including modern pre-processing techniques such as spatial image registration, skull stripping, inhomogeneity correction and tissue segmentation using an unsupervised learning approach using intensity histogram information. the temporal data across multiple visits is used to study the structural change in volumes of these tissue classes, namely, cerebrospinal fluid (csf), grey matter (gm), and white matter (wm) as the patients progressed further into the disease. to detect changes in volume trends, we also analyse the data with a modified mann-kendall statistic. the segmented features thus extracted and the subsequent trend analysis provide insights such as atrophy, increase or intolerable shifting of gm, wm and csf and should help in future research for automated analysis of alzheimer's detection with clinical domain explainability.\",\n",
       "  'categories': 'eess.iv physics.med-ph q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-09',\n",
       "  'updated': '2024-03-24',\n",
       "  'authors': ['prayas sanyal',\n",
       "   'srinjay mukherjee',\n",
       "   'arkapravo das',\n",
       "   'anindya sen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.05558'},\n",
       " {'title': 'of heading, posture and body rotations derived from data acquired by   animal-borne accelerometers, magnetometers and gyrometers, kernel density   estimation of the corresponding spherical distributions, and fine-scale   movement reconstruction',\n",
       "  'id': '2310.05820',\n",
       "  'abstract': \"this paper provides the mathematical expressions, formulated in classical trigonometric terms, that are required (i) to compute an animal's body orientations and rotations in 3d space from data acquired at high-frequency by on-board accelerometers, magnetometers and gyrometers, (ii) to relate 3d rotations to changes in 3d orientation, (iii) to represent their spherical distributions through kernel density estimation (kde), and (iv) to reconstruct fine-scale movements. the way the huge amount of data that is acquired by these on-board devices can be more easily managed is also considered.\",\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-09',\n",
       "  'updated': '2024-09-04',\n",
       "  'authors': ['simon benhamou'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.05820'},\n",
       " {'title': 'existence of two distinct time scales in the fairen-velarde model of   bacterial respiration',\n",
       "  'id': '2310.05822',\n",
       "  'abstract': 'we study the bacterial respiration through the numerical solution of the fairen-velarde coupled nonlinear differential equations. the instantaneous concentrations of the oxygen and the nutrients are computed. the fixed point solution and the stable limit cycle are found in different parameter ranges as predicted by the linearized differential equations. in a specified range of parameters, it is observed that the system spends some time near the stable limit cycle and eventually reaches the stable fixed point. this metastability has been investigated systematically. interestingly, it is observed that the system exhibits two distinctly different time scales in reaching the stable fixed points. the slow time scale of the metastable lifetime near the stable limit cycle and a fast time scale (after leaving the zone of limit cycle) in rushing towards the stable fixed point. the gross residence time, near the limit cycle (described by a slow time scale), can be reduced by varying the concentrations of nutrients. this idea can be used to control the harmful metastable lifespan of active bacteria.',\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1142/s0129183124500943',\n",
       "  'created': '2023-10-09',\n",
       "  'updated': '2023-12-05',\n",
       "  'authors': ['soumyadeep kundu', 'muktish acharyya'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.05822'},\n",
       " {'title': 'an evolutionary model of personality traits related to cooperative   behavior using a large language model',\n",
       "  'id': '2310.05976',\n",
       "  'abstract': 'this paper aims to shed light on the evolutionary dynamics of diverse and social populations by introducing the rich expressiveness of generative models into the trait expression of social agent-based evolutionary models. specifically, we focus on the evolution of personality traits in the context of a game-theoretic relationship as a situation in which inter-individual interests exert strong selection pressures. we construct an agent model in which linguistic descriptions of personality traits related to cooperative behavior are used as genes. the deterministic strategies extracted from large language model (llm) that make behavioral decisions based on these personality traits are used as behavioral traits. the population is evolved according to selection based on average payoff and mutation of genes by asking llm to slightly modify the parent gene toward cooperative or selfish. through preliminary experiments and analyses, we clarify that such a model can indeed exhibit the evolution of cooperative behavior based on the diverse and higher-order representation of personality traits. we also observed the repeated intrusion of cooperative and selfish personality traits through changes in the expression of personality traits, and found that the emerging words in the evolved gene well reflected the behavioral tendency of its personality in terms of their semantics.',\n",
       "  'categories': 'physics.soc-ph cs.ai cs.cl cs.gt cs.ma cs.ne q-bio.pe',\n",
       "  'doi': '10.1038/s41598-024-55903-y',\n",
       "  'created': '2023-10-03',\n",
       "  'updated': '',\n",
       "  'authors': ['reiji suzuki', 'takaya arita'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.05976'},\n",
       " {'title': 'probabilistic and maximum entropy modeling of chemical reaction systems:   characteristics and comparisons to mass action kinetic models',\n",
       "  'id': '2310.06135',\n",
       "  'abstract': 'we demonstrate and characterize a first-principles approach to modeling the mass action dynamics of metabolism. starting from a basic definition of entropy expressed as a multinomial probability density using boltzmann probabilities with standard chemical potentials, we derive and compare the free energy dissipation and the entropy production rates. we express the relation between the entropy production and the chemical master equation for modeling metabolism, which unifies chemical kinetics and chemical thermodynamics. subsequent implementation of an maximum free energy dissipation model for systems of coupled reactions is accomplished by using an approximation to the marcelin equation for mass action kinetics that maximizes the entropy production. because prediction uncertainty with respect to parameter variability is frequently a concern with mass action models utilizing rate constants, we compare and contrast the maximum entropy production model, which has its own set of rate parameters, to a population of standard mass action models in which the rate constants are randomly chosen. we show that a maximum entropy production model is characterized by a high probability of free energy dissipation rate, and likewise entropy production rate, relative to other models. we then characterize the variability of the maximum entropy production predictions with respect to uncertainties in parameters (standard free energies of formation) and with respect to ionic strengths typically found in a cell.',\n",
       "  'categories': 'physics.chem-ph q-bio.mn',\n",
       "  'doi': '10.1063/5.0180417',\n",
       "  'created': '2023-10-09',\n",
       "  'updated': '2024-02-25',\n",
       "  'authors': ['william r. cannon',\n",
       "   'samuel britton',\n",
       "   'mikahl banwarth-kuhn',\n",
       "   'mark alber'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.06135'},\n",
       " {'title': 'multilevel monte carlo for a class of partially observed processes in   neuroscience',\n",
       "  'id': '2310.06533',\n",
       "  'abstract': 'in this paper we consider bayesian parameter inference associated to a class of partially observed stochastic differential equations (sde) driven by jump processes. such type of models can be routinely found in applications, of which we focus upon the case of neuroscience. the data are assumed to be observed regularly in time and driven by the sde model with unknown parameters. in practice the sde may not have an analytically tractable solution and this leads naturally to a time-discretization. we adapt the multilevel markov chain monte carlo method of [11], which works with a hierarchy of time discretizations and show empirically and theoretically that this is preferable to using one single time discretization. the improvement is in terms of the computational cost needed to obtain a pre-specified numerical error. our approach is illustrated on models that are found in neuroscience.',\n",
       "  'categories': 'q-bio.nc stat.me',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-10',\n",
       "  'updated': '2024-11-30',\n",
       "  'authors': ['mohamed maama', 'ajay jasra', 'kengo kamatani'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.06533'},\n",
       " {'title': 'spatio-temporal modeling of co-dynamics of smallpox, measles and   pertussis in pre-healthcare finland',\n",
       "  'id': '2310.06538',\n",
       "  'abstract': 'infections are known to interact as previous infections may have an effect on risk of succumbing to a new infection. the co-dynamics can be mediated by immunosuppression or -modulation, shared environmental or climatic drivers, or competition for susceptible hosts. research and statistical methods in epidemiology often concentrate on large pooled datasets, or high quality data from cities, leaving rural areas underrepresented in literature. data considering rural populations are typically sparse and scarce, especially in the case of historical data sources, which may introduce considerable methodological challenges. in order to overcome many obstacles due to such data, we present a general bayesian spatio-temporal model for disease co-dynamics. applying the proposed model on historical (1820-1850) finnish parish register data, we study the spread of infectious diseases in pre-healthcare finland. we observe that measles, pertussis, and smallpox exhibit positively correlated dynamics, which could be attributed to immunosuppressive effects or, for example, the general weakening of the population due to recurring infections or poor nutritional conditions.',\n",
       "  'categories': 'stat.ap q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-10',\n",
       "  'updated': '2024-05-20',\n",
       "  'authors': ['tiia-maria pasanen',\n",
       "   'jouni helske',\n",
       "   'harri högmander',\n",
       "   'tarmo ketola'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.06538'},\n",
       " {'title': 'fabind: fast and accurate protein-ligand binding',\n",
       "  'id': '2310.06763',\n",
       "  'abstract': 'modeling the interaction between proteins and ligands and accurately predicting their binding structures is a critical yet challenging task in drug discovery. recent advancements in deep learning have shown promise in addressing this challenge, with sampling-based and regression-based methods emerging as two prominent approaches. however, these methods have notable limitations. sampling-based methods often suffer from low efficiency due to the need for generating multiple candidate structures for selection. on the other hand, regression-based methods offer fast predictions but may experience decreased accuracy. additionally, the variation in protein sizes often requires external modules for selecting suitable binding pockets, further impacting efficiency. in this work, we propose $\\\\mathbf{fabind}$, an end-to-end model that combines pocket prediction and docking to achieve accurate and fast protein-ligand binding. $\\\\mathbf{fabind}$ incorporates a unique ligand-informed pocket prediction module, which is also leveraged for docking pose estimation. the model further enhances the docking process by incrementally integrating the predicted pocket to optimize protein-ligand binding, reducing discrepancies between training and inference. through extensive experiments on benchmark datasets, our proposed $\\\\mathbf{fabind}$ demonstrates strong advantages in terms of effectiveness and efficiency compared to existing methods. our code is available at https://github.com/qizhipei/fabind',\n",
       "  'categories': 'cs.lg cs.ai q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-10',\n",
       "  'updated': '2024-01-08',\n",
       "  'authors': ['qizhi pei',\n",
       "   'kaiyuan gao',\n",
       "   'lijun wu',\n",
       "   'jinhua zhu',\n",
       "   'yingce xia',\n",
       "   'shufang xie',\n",
       "   'tao qin',\n",
       "   'kun he',\n",
       "   'tie-yan liu',\n",
       "   'rui yan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.06763'},\n",
       " {'title': 'cafa-evaluator: a python tool for benchmarking ontological   classification methods',\n",
       "  'id': '2310.06881',\n",
       "  'abstract': 'we present cafa-evaluator, a powerful python program designed to evaluate the performance of prediction methods on targets with hierarchical concept dependencies. it generalizes multi-label evaluation to modern ontologies where the prediction targets are drawn from a directed acyclic graph and achieves high efficiency by leveraging matrix computation and topological sorting. the program requirements include a small number of standard python libraries, making cafa-evaluator easy to maintain. the code replicates the critical assessment of protein function annotation (cafa) benchmarking, which evaluates predictions of the consistent subgraphs in gene ontology. owing to its reliability and accuracy, the organizers have selected cafa-evaluator as the official cafa evaluation software.',\n",
       "  'categories': 'q-bio.qm cs.pf',\n",
       "  'doi': '10.1093/bioadv/vbae043',\n",
       "  'created': '2023-10-10',\n",
       "  'updated': '2024-03-12',\n",
       "  'authors': ['damiano piovesan',\n",
       "   'davide zago',\n",
       "   'parnal joshi',\n",
       "   'm. clara de paolis kaluza',\n",
       "   'mahta mehdiabadi',\n",
       "   'rashika ramola',\n",
       "   'alexander miguel monzon',\n",
       "   'walter reade',\n",
       "   'iddo friedberg',\n",
       "   'predrag radivojac',\n",
       "   'silvio c. e. tosatto'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.06881'},\n",
       " {'title': 'score-based generative models for designing binding peptide backbones',\n",
       "  'id': '2310.07051',\n",
       "  'abstract': \"score-based generative models (sgms) have proven to be powerful tools for designing new proteins. designing proteins that bind a pre-specified target is highly relevant to a range of medical and industrial applications. despite the flurry of new sgms in the last year, there has been little systematic exploration of the impact of design choices in sgms for protein design. here we present loopgen, a flexible sgm framework for the design of short binding peptide structures. we apply our framework to design antibody binding loop structures conditional on a target epitope and evaluate a variety of modelling choices in sgm-based protein design. we demonstrate that modelling residue orientations in addition to positions improves not only the quality of the output structures but also their diversity. additionally, we identify variance schedules that result in significant performance improvements and observe patterns that may motivate the development of better schedules for protein design. finally, we develop three novel tests to evaluate whether the model generates structures that are appropriately conditioned on an epitope, demonstrating that loopgen's generated structures are dependent on the structure, sequence, and position of the epitope. our findings will help guide future development and evaluation of generative models for binding proteins.\",\n",
       "  'categories': 'q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-10',\n",
       "  'updated': '2024-09-26',\n",
       "  'authors': ['john d boom',\n",
       "   'matthew greenig',\n",
       "   'pietro sormanni',\n",
       "   'pietro liò'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.07051'},\n",
       " {'title': 'a reaction-diffusion-chemotaxis model for human population dynamics over   fractal terrains',\n",
       "  'id': '2310.07185',\n",
       "  'abstract': 'advection of entities induced by gradients in attractant concentration fields is observed via diffusiophoresis in colloids and via chemotaxis in microorganisms. mathematically, both diffusiophoresis and chemotaxis follow similar mathematical descriptions and display a variety of interesting behaviors that are not observed through other transport mechanisms. however, the application of such a mathematical framework has largely been restricted to soft matter research. in this article, we argue that this framework is more general and can be expanded to study human population dynamics. we assert that human populations also migrate chemotactically, but by sensing concentrations gradients in attractants such as resource availability, social connections, and safety indices. therefore, we extend the fisher-kpp reaction-diffusion model, foundational to human population dynamics, to incorporate chemotactic advection. furthermore, we introduce a fractal terrain to better mimic the human dispersal phenomena. simulations demonstrate that, by including chemotaxis of a population toward attractants which are dispersed heterogenously over fractal terrains, population hotspots can appear from from initially uniformly dispersed states whereas fisher-kpp without chemotaxis predicts a persistent tendency toward population uniformity. varying the chemotactic migration yields fine control over inter- or intra-population segregation and thus the population growth rates may be substantially altered by considering the population-attractant coupling. this framework may be useful for characterizing historical population separations, and furthermore is particularly pertinent for predicting emergence of new population hotspots as climate change is expected to cause large-scale human displacement, which may be dictated by chemotactic movement of humans due to evolving concentration gradients in safety indices.',\n",
       "  'categories': 'q-bio.pe cond-mat.soft physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-11',\n",
       "  'updated': '2024-10-03',\n",
       "  'authors': ['benjamin m. alessio', 'ankur gupta'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.07185'},\n",
       " {'title': 'epidemiological impacts of age structures on human malaria transmission',\n",
       "  'id': '2310.07275',\n",
       "  'abstract': 'malaria is one of the most common mosquito-borne diseases widespread in tropical and subtropical regions, causing thousands of deaths every year in the world. in a previous paper, we formulated an age-structured model containing three structural variables: (i) the chronological age of human and mosquito populations, (ii) the time since they are infected, and (iii) humans waning immunity (i.e. the progressive loss of protective antibodies after recovery). in the present paper, we expand the analysis of this age-structured model and focus on the derivation of entomological and epidemiological results commonly used in the literature, following the works of smith and mckenzie. we generalize their results to the age-structured case. in order to quantify the impact of neglecting structuring variables such as chronological age, we assigned values from the literature to our model parameters. while some parameters values are readily accessible from the literature, at least those about the human population, the parameters concerning mosquitoes are less commonly documented and the values of a number of them (e.g. mosquito survival in the presence or in absence of infection) can be discussed extensively.',\n",
       "  'categories': 'q-bio.pe math.ap',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-11',\n",
       "  'updated': '2024-05-22',\n",
       "  'authors': ['quentin richard',\n",
       "   'marc choisy',\n",
       "   'ramsès djidjou-demasse',\n",
       "   'thierry lefèvre'],\n",
       "  'affiliation': ['imag', 'oucru', 'mivegec', 'mivegec'],\n",
       "  'url': 'https://arxiv.org/abs/2310.07275'},\n",
       " {'title': 'biot5: enriching cross-modal integration in biology with chemical   knowledge and natural language associations',\n",
       "  'id': '2310.07276',\n",
       "  'abstract': 'recent advancements in biological research leverage the integration of molecules, proteins, and natural language to enhance drug discovery. however, current models exhibit several limitations, such as the generation of invalid molecular smiles, underutilization of contextual information, and equal treatment of structured and unstructured knowledge. to address these issues, we propose $\\\\mathbf{biot5}$, a comprehensive pre-training framework that enriches cross-modal integration in biology with chemical knowledge and natural language associations. $\\\\mathbf{biot5}$ utilizes selfies for $100%$ robust molecular representations and extracts knowledge from the surrounding context of bio-entities in unstructured biological literature. furthermore, $\\\\mathbf{biot5}$ distinguishes between structured and unstructured knowledge, leading to more effective utilization of information. after fine-tuning, biot5 shows superior performance across a wide range of tasks, demonstrating its strong capability of capturing underlying relations and properties of bio-entities. our code is available at $\\\\href{https://github.com/qizhipei/biot5}{github}$.',\n",
       "  'categories': 'cs.cl cs.ai cs.lg q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-11',\n",
       "  'updated': '2024-01-28',\n",
       "  'authors': ['qizhi pei',\n",
       "   'wei zhang',\n",
       "   'jinhua zhu',\n",
       "   'kehan wu',\n",
       "   'kaiyuan gao',\n",
       "   'lijun wu',\n",
       "   'yingce xia',\n",
       "   'rui yan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.07276'},\n",
       " {'title': 'ordinal characterization of similarity judgments',\n",
       "  'id': '2310.07543',\n",
       "  'abstract': 'characterizing judgments of similarity within a perceptual or semantic domain, and making inferences about the underlying structure of this domain from these judgments, has an increasingly important role in cognitive and systems neuroscience. we present a new framework for this purpose that makes limited assumptions about how perceptual distances are converted into similarity judgments. the approach starts from a dataset of empirical judgments of relative similarities: the fraction of times that a subject chooses one of two comparison stimuli to be more similar to a reference stimulus. these empirical judgments provide bayesian estimates of underling choice probabilities. from these estimates, we derive indices that characterize the set of judgments in three ways: compatibility with a symmetric dis-similarity, compatibility with an ultrametric space, and compatibility with an additive tree. each of the indices is derived from rank-order relationships among the choice probabilities that, as we show, are necessary and sufficient for local consistency with the three respective characteristics. we illustrate this approach with simulations and example psychophysical datasets of dis-similarity judgments in several visual domains and provide code that implements the analyses at https://github.com/jvlab/simrank.',\n",
       "  'categories': 'q-bio.nc q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-11',\n",
       "  'updated': '2024-09-05',\n",
       "  'authors': ['jonathan d. victor', 'guillermo aguilar', 'suniyya a. waraich'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.07543'},\n",
       " {'title': 'discrete and continuous mathematical models of sharp-fronted collective   cell migration and invasion',\n",
       "  'id': '2310.07938',\n",
       "  'abstract': 'mathematical models describing the spatial spreading and invasion of populations of biological cells are often developed in a continuum modelling framework using reaction-diffusion equations. while continuum models based on linear diffusion are routinely employed and known to capture key experimental observations, linear diffusion fails to predict well-defined sharp fronts that are often observed experimentally. this observation has motivated the use of nonlinear degenerate diffusion, however these nonlinear models and the associated parameters lack a clear biological motivation and interpretation. here we take a different approach by developing a stochastic discrete lattice-based model incorporating biologically-inspired mechanisms and then deriving the reaction-diffusion continuum limit. inspired by experimental observations, agents in the simulation deposit extracellular material, that we call a substrate, locally onto the lattice, and the motility of agents is taken to be proportional to the substrate density. discrete simulations that mimic a two--dimensional circular barrier assay illustrate how the discrete model supports both smooth and sharp-fronted density profiles depending on the rate of substrate deposition. coarse-graining the discrete model leads to a novel partial differential equation (pde) model whose solution accurately approximates averaged data from the discrete model. the new discrete model and pde approximation provides a simple, biologically motivated framework for modelling the spreading, growth and invasion of cell populations with well-defined sharp fronts',\n",
       "  'categories': 'nlin.cg q-bio.cb',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-11',\n",
       "  'updated': '2024-01-19',\n",
       "  'authors': ['matthew j simpson',\n",
       "   'keeley m murphy',\n",
       "   'scott w mccue',\n",
       "   'pascal r buenzli'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.07938'},\n",
       " {'title': 'multi-view variational autoencoder for missing value imputation in   untargeted metabolomics',\n",
       "  'id': '2310.07990',\n",
       "  'abstract': 'background: missing data is a common challenge in mass spectrometry-based metabolomics, which can lead to biased and incomplete analyses. the integration of whole-genome sequencing (wgs) data with metabolomics data has emerged as a promising approach to enhance the accuracy of data imputation in metabolomics studies. method: in this study, we propose a novel method that leverages the information from wgs data and reference metabolites to impute unknown metabolites. our approach utilizes a multi-view variational autoencoder to jointly model the burden score, polygenetic risk score (pgs), and linkage disequilibrium (ld) pruned single nucleotide polymorphisms (snps) for feature extraction and missing metabolomics data imputation. by learning the latent representations of both omics data, our method can effectively impute missing metabolomics values based on genomic information. results: we evaluate the performance of our method on empirical metabolomics datasets with missing values and demonstrate its superiority compared to conventional imputation techniques. using 35 template metabolites derived burden scores, pgs and ld-pruned snps, the proposed methods achieved r^2-scores > 0.01 for 71.55% of metabolites. conclusion: the integration of wgs data in metabolomics imputation not only improves data completeness but also enhances downstream analyses, paving the way for more comprehensive and accurate investigations of metabolic pathways and disease associations. our findings offer valuable insights into the potential benefits of utilizing wgs data for metabolomics data imputation and underscore the importance of leveraging multi-modal data integration in precision medicine research.',\n",
       "  'categories': 'q-bio.gn cs.ir cs.lg stat.ap',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-11',\n",
       "  'updated': '2024-03-12',\n",
       "  'authors': ['chen zhao',\n",
       "   'kuan-jui su',\n",
       "   'chong wu',\n",
       "   'xuewei cao',\n",
       "   'qiuying sha',\n",
       "   'wu li',\n",
       "   'zhe luo',\n",
       "   'tian qin',\n",
       "   'chuan qiu',\n",
       "   'lan juan zhao',\n",
       "   'anqi liu',\n",
       "   'lindong jiang',\n",
       "   'xiao zhang',\n",
       "   'hui shen',\n",
       "   'weihua zhou',\n",
       "   'hong-wen deng'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.07990'},\n",
       " {'title': 'how connectivity structure shapes rich and lazy learning in neural   circuits',\n",
       "  'id': '2310.08513',\n",
       "  'abstract': 'in theoretical neuroscience, recent work leverages deep learning tools to explore how some network attributes critically influence its learning dynamics. notably, initial weight distributions with small (resp. large) variance may yield a rich (resp. lazy) regime, where significant (resp. minor) changes to network states and representation are observed over the course of learning. however, in biology, neural circuit connectivity could exhibit a low-rank structure and therefore differs markedly from the random initializations generally used for these studies. as such, here we investigate how the structure of the initial weights -- in particular their effective rank -- influences the network learning regime. through both empirical and theoretical analyses, we discover that high-rank initializations typically yield smaller network changes indicative of lazier learning, a finding we also confirm with experimentally-driven initial connectivity in recurrent neural networks. conversely, low-rank initialization biases learning towards richer learning. importantly, however, as an exception to this rule, we find lazier learning can still occur with a low-rank initialization that aligns with task and data statistics. our research highlights the pivotal role of initial weight structures in shaping learning regimes, with implications for metabolic costs of plasticity and risks of catastrophic forgetting.',\n",
       "  'categories': 'cs.ne cs.ai q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-12',\n",
       "  'updated': '2024-02-19',\n",
       "  'authors': ['yuhan helena liu',\n",
       "   'aristide baratin',\n",
       "   'jonathan cornford',\n",
       "   'stefan mihalas',\n",
       "   'eric shea-brown',\n",
       "   'guillaume lajoie'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.08513'},\n",
       " {'title': 'phylogfn: phylogenetic inference with generative flow networks',\n",
       "  'id': '2310.08774',\n",
       "  'abstract': 'phylogenetics is a branch of computational biology that studies the evolutionary relationships among biological entities. its long history and numerous applications notwithstanding, inference of phylogenetic trees from sequence data remains challenging: the high complexity of tree space poses a significant obstacle for the current combinatorial and probabilistic techniques. in this paper, we adopt the framework of generative flow networks (gflownets) to tackle two core problems in phylogenetics: parsimony-based and bayesian phylogenetic inference. because gflownets are well-suited for sampling complex combinatorial structures, they are a natural choice for exploring and sampling from the multimodal posterior distribution over tree topologies and evolutionary distances. we demonstrate that our amortized posterior sampler, phylogfn, produces diverse and high-quality evolutionary hypotheses on real benchmark datasets. phylogfn is competitive with prior works in marginal likelihood estimation and achieves a closer fit to the target distribution than state-of-the-art variational inference methods. our code is available at https://github.com/zmy1116/phylogfn.',\n",
       "  'categories': 'q-bio.pe cs.lg stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-12',\n",
       "  'updated': '2024-03-24',\n",
       "  'authors': ['mingyang zhou',\n",
       "   'zichao yan',\n",
       "   'elliot layne',\n",
       "   'nikolay malkin',\n",
       "   'dinghuai zhang',\n",
       "   'moksh jain',\n",
       "   'mathieu blanchette',\n",
       "   'yoshua bengio'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.08774'},\n",
       " {'title': 'a deep neural network -- mechanistic hybrid model to predict   pharmacokinetics in rat',\n",
       "  'id': '2310.09167',\n",
       "  'abstract': 'an important aspect in the development of small molecules as drugs or agro-chemicals is their systemic availability after intravenous and oral administration. the prediction of the systemic availability from the chemical structure of a potential candidate is highly desirable, as it allows to focus the drug or agrochemical development on compounds with a favorable kinetic profile. however, such pre-dictions are challenging as the availability is the result of the complex interplay between molecular properties, biology and physiology and training data is rare. in this work we improve the hybrid model developed earlier [1]. we reduce the median fold change error for the total oral exposure from 2.85 to 2.35 and for intravenous administration from 1.95 to 1.62. this is achieved by training on a larger data set, improving the neural network architecture as well as the parametrization of mechanistic model. further, we extend our approach to predict additional endpoints and to handle different covariates, like sex and dosage form. in contrast to a pure machine learning model, our model is able to predict new end points on which it has not been trained. we demonstrate this feature by predicting the exposure over the first 24h, while the model has only been trained on the total exposure.',\n",
       "  'categories': 'q-bio.qm cs.ce cs.lg q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-13',\n",
       "  'updated': '2024-01-02',\n",
       "  'authors': ['florian führer',\n",
       "   'andrea gruber',\n",
       "   'holger diedam',\n",
       "   'andreas h. göller',\n",
       "   'stephan menz',\n",
       "   'sebastian schneckener'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.09167'},\n",
       " {'title': 'neural activity in quarks language: lattice field theory for a network   of real neurons',\n",
       "  'id': '2310.09178',\n",
       "  'abstract': 'brain-computer interfaces surged extraordinary developments in recent years, and a significant discrepancy now exists between the abundance of available data and the limited headway made in achieving a unified theoretical framework. this discrepancy becomes particularly pronounced when examining the collective neural activity at the micro- and meso-scale, where a coherent formalization that adequately describes neural interactions is still lacking. here, we introduce a mathematical framework to analyze systems of natural neurons and interpret the related empirical observations in terms of lattice field theory, an established paradigm from theoretical particle physics and statistical mechanics. our methods are tailored to interpret data from chronic neural interfaces, especially spike rasters from measurements of single neurons activity, and generalize the maximum entropy model for neural networks so that also the time evolution of the system is taken into account. this is obtained by bridging particle physics and neuroscience, paving the way to particle physics-inspired models of neocortex.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '10.3390/e26060495',\n",
       "  'created': '2023-10-13',\n",
       "  'updated': '2024-03-23',\n",
       "  'authors': ['giampiero bardella',\n",
       "   'simone franchini',\n",
       "   'liming pan',\n",
       "   'riccardo balzan',\n",
       "   'surabhi ramawat',\n",
       "   'emiliano brunamonti',\n",
       "   'pierpaolo pani',\n",
       "   'stefano ferraina'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.09178'},\n",
       " {'title': 'genome hybridization: a universal way for the origin and diversification   of organelles as well as the origin and speciation of eukaryotes',\n",
       "  'id': '2310.09758',\n",
       "  'abstract': 'the origin of organelles (mitochondrion, chloroplast and nucleus) remains enigmatic. the endosymbiotic hypothesis that chloroplasts, mitochondria and nuclei descend from the endosymbiotic cyanobacterium, bacterium and archaebacterium respectively is dominant yet uncompelling, while our discovery of de novo organelle biogenesis in the cyanobacterium tdx16 that had acquired the genome of its green algal host haematococcus pluvialis overturns this hypothesis. in light of organelle biogenesis in the cyanobacterium tdx16 in combination with the relevant cellular and molecular evidence, we propose genome hybridization hypothesis (ghh) that the origin of organelles and origin of eukaryotes as well as the diversification of organelles and speciation of eukaryotes are unified and achieved by genome hybridization: the endosymbiotic cyanobacteria or bacteria obtain genomes of their archaebacterial or eukaryotic hosts and hybridize with their own ones resulting in expanded genomes containing a mixture of hybrid prokaryotic genes and eukaryotic genes, and thus the cyanobacteria or bacteria have to compartmentalize to accommodate different genes for specialized function of photosynthesis (chloroplast), respiration (mitochondrion) and dna preservation (nucleus), and consequently turn into photosynthetic or heterotrophic eukaryotes. accordingly, eukaryotes and their organelles are of multiple origin, while the formation of cancer cells is the speciation of eukaryotes as cancer cells are new species of unicellular eukaryotes arising from bacteria. therefore, ghh provides a theoretical framework unifying evolutionary biology, cancer biology and cell biology and directing the integrated multidisciplinary research.',\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-15',\n",
       "  'updated': '2024-05-07',\n",
       "  'authors': ['qing-lin dong', 'xiang-ying xing'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.09758'},\n",
       " {'title': 'mental causation in a physical world: a self-causation model of downward   causation',\n",
       "  'id': '2310.10005',\n",
       "  'abstract': 'downward causation is self-causation, the causel effect from the whole to its parts, and is considered a promising theory for the problem of mental causation. however, it remains to be clarified how an irreducible but supervenient downward causal power can arise. here, we argue that a feedback control of lower micro-level synaptic weights using higher macro-level algebraic structural feedback errors is a model of downward causation. the feedback control consists of two mechanisms: observation of the feedback error and control of the feedback error. the two mechanisms can be implemented at two different levels of the hierarchy. macro-level feedback error is described by algebra of supervenient macro-level functions, independent of the external cause. the supervenient macro-level feedback error constraining of the micro synaptic weights through feedback control is downward causation.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-15',\n",
       "  'updated': '2024-10-24',\n",
       "  'authors': ['yoshiyuki ohmura', 'yasuo kuniyoshi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.10005'},\n",
       " {'title': 'universal characterization of epitope immunodominance from a multi-scale   model of clonal competition in germinal centers',\n",
       "  'id': '2310.10966',\n",
       "  'abstract': 'we introduce a novel, multi-scale model for affinity maturation, which aims to capture the intra-clonal, inter-clonal and epitope-specific organization of the b cell population in a germinal center. we describe the evolution of the b cell population via a quasispecies dynamics, with species corresponding to unique b cell receptors (bcrs), where the desired multi-scale structure is reflected on the mutational connectivity of the accessible bcr space, and on the statistical properties of its fitness landscape. within this mathematical framework, we study the competition among classes of bcrs targeting different antigen epitopes, and construct an effective \\\\emph{immunogenic space} where epitope immunodominance relations can be universally characterized. we finally study how varying the relative composition of a mixture of antigens with variable and conserved domains allows for a parametric exploration of this space, and identify general principles for the rational design of two-antigen cocktails.',\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-16',\n",
       "  'updated': '2024-01-31',\n",
       "  'authors': ['federica ferretti', 'mehran kardar'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.10966'},\n",
       " {'title': 'agent-based simulation reveals localized isolation key to saving lives   and resources',\n",
       "  'id': '2310.11121',\n",
       "  'abstract': 'in the realm of pandemic dynamics, understanding the intricate interplay between disease transmission, interventions, and immunity is pivotal for effective control strategies. through a rigorous agent-based computer simulation, we embarked on a comprehensive exploration, traversing unmitigated spread, lockdown scenarios, and the transformative potential of vaccination. we unveil that while quarantine unquestionably delays the pandemic peak, it does not act as an impenetrable barrier to halt the progression of infectious diseases. vaccination factor revealed a potent weapon against outbreaks higher vaccination percentage not only delayed infection peaks but also substantially curtailed their impact. our investigation into bond dilution below the percolation threshold presents an additional dimension to pandemic control. we observed that localized isolation through bond dilution offers a more resource-efficient targeted control strategy than blanket lockdowns or large-scale vaccination campaigns.',\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech physics.bio-ph',\n",
       "  'doi': '10.1016/j.physa.2024.130097',\n",
       "  'created': '2023-10-17',\n",
       "  'updated': '2024-09-24',\n",
       "  'authors': ['mintu karmakar'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.11121'},\n",
       " {'title': 'mismatch negativity: time for deconstruction',\n",
       "  'id': '2310.11247',\n",
       "  'abstract': 'error signals are the cornerstone of predictive coding and are widely considered essential to sensory perception and beyond. the mismatch negativity (mmn) is arguably the most emblematic and most studied brain error signal. it is affected in many brain disorders. however, its precise algorithmic function and the underlying physiology remain mysterious. over the past decade, theoretical and computational explanations have been put forward. they highlight a paradox: the mmn is considered a signature of context-dependent perceptual learning, although it is defined as an evoked response averaged across trials, thus neglecting the information carried by error signal fluctuations over time. we propose to deconstruct the mmn, by virtue of hypothesis driven computational approaches whose aim it to account for these fluctuations.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-17',\n",
       "  'updated': '2024-02-06',\n",
       "  'authors': ['francoise lecaignard', 'jeremie mattout'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.11247'},\n",
       " {'title': 'connecting the dots: range expansions across landscapes with quenched   noise',\n",
       "  'id': '2310.11563',\n",
       "  'abstract': \"when biological populations expand into new territory, the evolutionary outcomes can be strongly influenced by genetic drift, the random fluctuations in allele frequencies. meanwhile, spatial variability in the environment can also significantly influence the competition between sub-populations vying for space. little is known about the interplay of these intrinsic and extrinsic sources of noise in population dynamics: when does environmental heterogeneity dominate over genetic drift or vice versa, and what distinguishes their population genetics signatures? here, in the context of neutral evolution, we examine the interplay between a population's intrinsic, demographic noise and an extrinsic, quenched random noise provided by a heterogeneous environment. using a multi-species eden model, we simulate a population expanding over a landscape with random variations in local growth rates and measure how this variability affects genealogical tree structure, and thus genetic diversity. we find that, for strong heterogeneity, the genetic makeup of the expansion front is to a great extent predetermined by the set of fastest paths through the environment. the landscape-dependent statistics of these optimal paths then supersede those of the population's intrinsic noise as the main determinant of evolutionary dynamics. remarkably, the statistics for coalescence of genealogical lineages, derived from those deterministic paths, strongly resemble the statistics emerging from demographic noise alone in uniform landscapes. this cautions interpretations of coalescence statistics and raises new challenges for inferring past population dynamics.\",\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-17',\n",
       "  'updated': '2024-06-25',\n",
       "  'authors': ['jimmy gonzalez nuñez',\n",
       "   'jayson paulose',\n",
       "   'wolfram möbius',\n",
       "   'daniel a. beller'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.11563'},\n",
       " {'title': \"break-up and recovery of harmony between direct and indirect pathways in   the basal ganglia; huntington's disease and treatment\",\n",
       "  'id': '2310.11635',\n",
       "  'abstract': \"the basal ganglia (bg) in the brain exhibit diverse functions for motor, cognition, and emotion. such bg functions could be made via competitive harmony between the two competing pathways, direct pathway (dp) (facilitating movement) and indirect pathway (ip) (suppressing movement). as a result of break-up of harmony between dp and ip, there appear pathological states with disorder for movement, cognition, and psychiatry. in this paper, we are concerned about the huntington's disease (hd), which is a genetic neurodegenerative disorder causing involuntary movement and severe cognitive and psychiatric symptoms. for the hd, the number of d2 spns ($n_{\\\\rm d2}$) is decreased due to degenerative loss, and hence, by decreasing $x_{\\\\rm d2}$ (fraction of $n_{\\\\rm d2}$), we investigate break-up of harmony between dp and ip in terms of their competition degree ${\\\\cal c}_d$, given by the ratio of strength of dp (${\\\\cal s}_{dp}$) to strength of ip (${\\\\cal s}_{ip}$) (i.e., ${\\\\cal c}_d = {\\\\cal s}_{dp} / {\\\\cal s}_{ip}$). in the case of hd, the ip is under-active, in contrast to the case of parkinson's disease with over-active ip, which results in increase in ${\\\\cal c}_d$ (from the normal value). thus, hyperkinetic dyskinesia such as chorea (involuntary jerky movement) occurs. we also investigate treatment of hd, based on optogenetics and gp ablation, by increasing strength of ip, resulting in recovery of harmony between dp and ip. finally, we study effect of loss of healthy synapses of all the bg cells on hd. due to loss of healthy synapses, disharmony between dp and ip increases, leading to worsen symptoms of the hd.\",\n",
       "  'categories': 'q-bio.nc physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-17',\n",
       "  'updated': '2024-04-18',\n",
       "  'authors': ['sang-yoon kim', 'woochang lim'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.11635'},\n",
       " {'title': 'perceptual scales predicted by fisher information metrics',\n",
       "  'id': '2310.11759',\n",
       "  'abstract': 'perception is often viewed as a process that transforms physical variables, external to an observer, into internal psychological variables. such a process can be modeled by a function coined perceptual scale. the perceptual scale can be deduced from psychophysical measurements that consist in comparing the relative differences between stimuli (i.e. difference scaling experiments). however, this approach is often overlooked by the modeling and experimentation communities. here, we demonstrate the value of measuring the perceptual scale of classical (spatial frequency, orientation) and less classical physical variables (interpolation between textures) by embedding it in recent probabilistic modeling of perception. first, we show that the assumption that an observer has an internal representation of univariate parameters such as spatial frequency or orientation while stimuli are high-dimensional does not lead to contradictory predictions when following the theoretical framework. second, we show that the measured perceptual scale corresponds to the transduction function hypothesized in this framework. in particular, we demonstrate that it is related to the fisher information of the generative model that underlies perception and we test the predictions given by the generative model of different stimuli in a set a of difference scaling experiments. our main conclusion is that the perceptual scale is mostly driven by the stimulus power spectrum. finally, we propose that this measure of perceptual scale is a way to push further the notion of perceptual distances by estimating the perceptual geometry of images i.e. the path between images instead of simply the distance between those.',\n",
       "  'categories': 'q-bio.nc cs.cv cs.it math.it',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-18',\n",
       "  'updated': '2024-03-15',\n",
       "  'authors': ['jonathan vacher', 'pascal mamassian'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.11759'},\n",
       " {'title': 'towards parsimonious generative modeling of rna families',\n",
       "  'id': '2310.12700',\n",
       "  'abstract': 'generative probabilistic models emerge as a new paradigm in data-driven, evolution-informed design of biomolecular sequences. this paper introduces a novel approach, called edge activation direct coupling analysis (eadca), tailored to the characteristics of rna sequences, with a strong emphasis on simplicity, efficiency, and interpretability. eadca explicitly constructs sparse coevolutionary models for rna families, achieving performance levels comparable to more complex methods while utilizing a significantly lower number of parameters. our approach demonstrates efficiency in generating artificial rna sequences that closely resemble their natural counterparts in both statistical analyses and shape-map experiments, and in predicting the effect of mutations. notably, eadca provides a unique feature: estimating the number of potential functional sequences within a given rna family. for example, in the case of cyclic di-amp riboswitches (rf00379), our analysis suggests the existence of approximately $\\\\mathbf{10^{39}}$ functional nucleotide sequences. while huge compared to the known $< \\\\mathbf{4,000}$ natural sequences, this number represents only a tiny fraction of the vast pool of nearly $\\\\mathbf{10^{82}}$ possible nucleotide sequences of the same length (136 nucleotides). these results underscore the promise of sparse and interpretable generative models, such as eadca, in enhancing our understanding of the expansive rna sequence space.',\n",
       "  'categories': 'q-bio.bm cond-mat.stat-mech q-bio.gn q-bio.qm',\n",
       "  'doi': '10.1093/nar/gkae289',\n",
       "  'created': '2023-10-19',\n",
       "  'updated': '',\n",
       "  'authors': ['francesco calvanese',\n",
       "   'camille n. lambert',\n",
       "   'philippe nghe',\n",
       "   'francesco zamponi',\n",
       "   'martin weigt'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.12700'},\n",
       " {'title': 'a markovian dynamics for c. elegans behavior across scales',\n",
       "  'id': '2310.12883',\n",
       "  'abstract': 'how do we capture the breadth of behavior in animal movement, from rapid body twitches to aging? using high-resolution videos of the nematode worm $c. elegans$, we show that a single dynamics connects posture-scale fluctuations with trajectory diffusion, and longer-lived behavioral states. we take short posture sequences as an instantaneous behavioral measure, fixing the sequence length for maximal prediction. within the space of posture sequences we construct a fine-scale, maximum entropy partition so that transitions among microstates define a high-fidelity markov model, which we also use as a means of principled coarse-graining. we translate these dynamics into movement using resistive force theory, capturing the statistical properties of foraging trajectories. predictive across scales, we leverage the longest-lived eigenvectors of the inferred markov chain to perform a top-down subdivision of the worm\\'s foraging behavior, revealing both \"runs-and-pirouettes\" as well as previously uncharacterized finer-scale behaviors. we use our model to investigate the relevance of these fine-scale behaviors for foraging success, recovering a trade-off between local and global search strategies.',\n",
       "  'categories': 'physics.bio-ph nlin.cd q-bio.nc',\n",
       "  'doi': '10.1073/pnas.2318805121',\n",
       "  'created': '2023-10-19',\n",
       "  'updated': '2024-08-30',\n",
       "  'authors': ['antonio c. costa',\n",
       "   'tosif ahamed',\n",
       "   'david jordan',\n",
       "   'greg j. stephens'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.12883'},\n",
       " {'title': 'cousins of the vendi score: a family of similarity-based diversity   metrics for science and machine learning',\n",
       "  'id': '2310.12952',\n",
       "  'abstract': \"measuring diversity accurately is important for many scientific fields, including machine learning (ml), ecology, and chemistry. the vendi score was introduced as a generic similarity-based diversity metric that extends the hill number of order q=1 by leveraging ideas from quantum statistical mechanics. contrary to many diversity metrics in ecology, the vendi score accounts for similarity and does not require knowledge of the prevalence of the categories in the collection to be evaluated for diversity. however, the vendi score treats each item in a given collection with a level of sensitivity proportional to the item's prevalence. this is undesirable in settings where there is a significant imbalance in item prevalence. in this paper, we extend the other hill numbers using similarity to provide flexibility in allocating sensitivity to rare or common items. this leads to a family of diversity metrics -- vendi scores with different levels of sensitivity -- that can be used in a variety of applications. we study the properties of the scores in a synthetic controlled setting where the ground truth diversity is known. we then test their utility in improving molecular simulations via vendi sampling. finally, we use the vendi scores to better understand the behavior of image generative models in terms of memorization, duplication, diversity, and sample quality.\",\n",
       "  'categories': 'cs.lg physics.chem-ph q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-19',\n",
       "  'updated': '2024-05-04',\n",
       "  'authors': ['amey p. pasarkar', 'adji bousso dieng'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.12952'},\n",
       " {'title': 'getting aligned on representational alignment',\n",
       "  'id': '2310.13018',\n",
       "  'abstract': \"biological and artificial information processing systems form representations of the world that they can use to categorize, reason, plan, navigate, and make decisions. how can we measure the similarity between the representations formed by these diverse systems? do similarities in representations then translate into similar behavior? if so, then how can a system's representations be modified to better match those of another system? these questions pertaining to the study of representational alignment are at the heart of some of the most promising research areas in contemporary cognitive science, neuroscience, and machine learning. in this perspective, we survey the exciting recent developments in representational alignment research in the fields of cognitive science, neuroscience, and machine learning. despite their overlapping interests, there is limited knowledge transfer between these fields, so work in one field ends up duplicated in another, and useful innovations are not shared effectively. to improve communication, we propose a unifying framework that can serve as a common language for research on representational alignment, and map several streams of existing work across fields within our framework. we also lay out open problems in representational alignment where progress can benefit all three of these fields. we hope that this paper will catalyze cross-disciplinary collaboration and accelerate progress for all communities studying and developing information processing systems.\",\n",
       "  'categories': 'q-bio.nc cs.ai cs.lg cs.ne',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-18',\n",
       "  'updated': '2024-11-26',\n",
       "  'authors': ['ilia sucholutsky',\n",
       "   'lukas muttenthaler',\n",
       "   'adrian weller',\n",
       "   'andi peng',\n",
       "   'andreea bobu',\n",
       "   'been kim',\n",
       "   'bradley c. love',\n",
       "   'christopher j. cueva',\n",
       "   'erin grant',\n",
       "   'iris groen',\n",
       "   'jascha achterberg',\n",
       "   'joshua b. tenenbaum',\n",
       "   'katherine m. collins',\n",
       "   'katherine l. hermann',\n",
       "   'kerem oktar',\n",
       "   'klaus greff',\n",
       "   'martin n. hebart',\n",
       "   'nathan cloos',\n",
       "   'nikolaus kriegeskorte',\n",
       "   'nori jacoby',\n",
       "   'qiuyi zhang',\n",
       "   'raja marjieh',\n",
       "   'robert geirhos',\n",
       "   'sherol chen',\n",
       "   'simon kornblith',\n",
       "   'sunayana rane',\n",
       "   'talia konkle',\n",
       "   \"thomas p. o'connell\",\n",
       "   'thomas unterthiner',\n",
       "   'andrew k. lampinen',\n",
       "   'klaus-robert müller',\n",
       "   'mariya toneva',\n",
       "   'thomas l. griffiths'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.13018'},\n",
       " {'title': 'minimal mechanisms of microtubule length regulation in living cells',\n",
       "  'id': '2310.13666',\n",
       "  'abstract': 'the microtubule cytoskeleton is responsible for sustained, long-range intracellular transport of mrnas, proteins, and organelles in neurons. neuronal microtubules must be stable enough to ensure reliable transport, but they also undergo dynamic instability, as their plus and minus ends continuously switch between growth and shrinking. this process allows for continuous rebuilding of the cytoskeleton and for flexibility in injury settings. motivated by \\\\textit{in vivo} experimental data on microtubule behavior in \\\\textit{drosophila} neurons, we propose a mathematical model of dendritic microtubule dynamics, with a focus on understanding microtubule length, velocity, and state-duration distributions. we find that limitations on microtubule growth phases are needed for realistic dynamics, but the type of limiting mechanism leads to qualitatively different responses to plausible experimental perturbations. we therefore propose and investigate two minimally-complex length-limiting factors: limitation due to resource (tubulin) constraints and limitation due to catastrophe of large-length microtubules. we combine simulations of a detailed stochastic model with steady-state analysis of a mean-field ordinary differential equations model to map out qualitatively distinct parameter regimes. this provides a basis for predicting changes in microtubule dynamics, tubulin allocation, and the turnover rate of tubulin within microtubules in different experimental environments.',\n",
       "  'categories': 'math.ds q-bio.sc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-20',\n",
       "  'updated': '2024-03-04',\n",
       "  'authors': ['anna c nelson',\n",
       "   'melissa m rolls',\n",
       "   'maria-veronica ciocanel',\n",
       "   'scott a mckinley'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.13666'},\n",
       " {'title': 'compositional deep probabilistic models of dna encoded libraries',\n",
       "  'id': '2310.13769',\n",
       "  'abstract': 'dna-encoded library (del) has proven to be a powerful tool that utilizes combinatorially constructed small molecules to facilitate highly-efficient screening assays. these selection experiments, involving multiple stages of washing, elution, and identification of potent binders via unique dna barcodes, often generate complex data. this complexity can potentially mask the underlying signals, necessitating the application of computational tools such as machine learning to uncover valuable insights. we introduce a compositional deep probabilistic model of del data, del-compose, which decomposes molecular representations into their mono-synthon, di-synthon, and tri-synthon building blocks and capitalizes on the inherent hierarchical structure of these molecules by modeling latent reactions between embedded synthons. additionally, we investigate methods to improve the observation models for del count data such as integrating covariate factors to more effectively account for data noise. across two popular public benchmark datasets (ca-ix and hrp), our model demonstrates strong performance compared to count baselines, enriches the correct pharmacophores, and offers valuable insights via its intrinsic interpretable structure, thereby providing a robust tool for the analysis of del data.',\n",
       "  'categories': 'q-bio.qm stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-20',\n",
       "  'updated': '2024-02-13',\n",
       "  'authors': ['benson chen', 'mohammad m. sultan', 'theofanis karaletsos'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.13769'},\n",
       " {'title': 'noise robustness and metabolic load determine the principles of central   dogma regulation',\n",
       "  'id': '2310.13803',\n",
       "  'abstract': 'the processes of gene expression are inherently stochastic, even for essential genes required for growth. how does the cell maximize fitness in light of noise? to answer this question, we build a mathematical model to explore the trade-off between metabolic load and growth robustness. the model predicts novel principles of central dogma regulation: optimal protein expression levels for many genes are in vast overabundance. essential genes are transcribed above a lower limit of one message per cell cycle. gene expression is achieved by load balancing between transcription and translation. we present evidence that each of these novel regulatory principles is observed. these results reveal that robustness and metabolic load determine the global regulatory principles that govern gene expression processes, and these principles have broad implications for cellular function.',\n",
       "  'categories': 'q-bio.mn physics.bio-ph',\n",
       "  'doi': '10.1126/sciadv.ado3095',\n",
       "  'created': '2023-10-20',\n",
       "  'updated': '2024-08-15',\n",
       "  'authors': ['teresa w. lo',\n",
       "   'han james choi',\n",
       "   'dean huang',\n",
       "   'paul a. wiggins'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.13803'},\n",
       " {'title': 'computational and systems biology advances to enable bioagent-agnostic   signatures',\n",
       "  'id': '2310.13898',\n",
       "  'abstract': 'enumerated threat agent lists have long driven biodefense priorities. the global sars-cov-2 pandemic demonstrated the limitations of searching for known threat agents as compared to a more agnostic approach. recent technological advances are enabling agent-agnostic biodefense, especially through the integration of multi-modal observations of host-pathogen interactions directed by a human immunological model. although well-developed technical assays exist for many aspects of human-pathogen interaction, the analytic methods and pipelines to combine and holistically interpret the results of such assays are immature and require further investments to exploit new technologies. in this manuscript, we discuss potential immunologically based bioagent-agnostic approaches and the computational tool gaps the community should prioritize filling.',\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-20',\n",
       "  'updated': '2024-02-28',\n",
       "  'authors': ['andy lin',\n",
       "   'cameron torres',\n",
       "   'errett c. hobbs',\n",
       "   'jaydeep bardhan',\n",
       "   'stephen b. aley',\n",
       "   'charles t. spencer',\n",
       "   'karen l. taylor',\n",
       "   'tony chiang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.13898'},\n",
       " {'title': 'pre-training on large-scale generated docking conformations with   helixdock to unlock the potential of protein-ligand structure prediction   models',\n",
       "  'id': '2310.13913',\n",
       "  'abstract': 'protein-ligand structure prediction is an essential task in drug discovery, predicting the binding interactions between small molecules (ligands) and target proteins (receptors). recent advances have incorporated deep learning techniques to improve the accuracy of protein-ligand structure prediction. nevertheless, the experimental validation of docking conformations remains costly, it raises concerns regarding the generalizability of these deep learning-based methods due to the limited training data. in this work, we show that by pre-training on a large-scale docking conformation generated by traditional physics-based docking tools and then fine-tuning with a limited set of experimentally validated receptor-ligand complexes, we can obtain a protein-ligand structure prediction model with outstanding performance. specifically, this process involved the generation of 100 million docking conformations for protein-ligand pairings, an endeavor consuming roughly 1 million cpu core days. the proposed model, helixdock, aims to acquire the physical knowledge encapsulated by the physics-based docking tools during the pre-training phase. helixdock has been rigorously benchmarked against both physics-based and deep learning-based baselines, demonstrating its exceptional precision and robust transferability in predicting binding confirmation. in addition, our investigation reveals the scaling laws governing pre-trained protein-ligand structure prediction models, indicating a consistent enhancement in performance with increases in model parameters and the volume of pre-training data. moreover, we applied helixdock to several drug discovery-related tasks to validate its practical utility. helixdock demonstrates outstanding capabilities on both cross-docking and structure-based virtual screening benchmarks.',\n",
       "  'categories': 'cs.lg cs.ce q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-21',\n",
       "  'updated': '2024-05-22',\n",
       "  'authors': ['lihang liu',\n",
       "   'shanzhuo zhang',\n",
       "   'donglong he',\n",
       "   'xianbin ye',\n",
       "   'jingbo zhou',\n",
       "   'xiaonan zhang',\n",
       "   'yaoyao jiang',\n",
       "   'weiming diao',\n",
       "   'hang yin',\n",
       "   'hua chai',\n",
       "   'fan wang',\n",
       "   'jingzhou he',\n",
       "   'liang zheng',\n",
       "   'yonghui li',\n",
       "   'xiaomin fang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.13913'},\n",
       " {'title': 'unimap: universal smiles-graph representation learning',\n",
       "  'id': '2310.14216',\n",
       "  'abstract': \"molecular representation learning is fundamental for many drug related applications. most existing molecular pre-training models are limited in using single molecular modality, either smiles or graph representation. to effectively leverage both modalities, we argue that it is critical to capture the fine-grained 'semantics' between smiles and graph, because subtle sequence/graph differences may lead to contrary molecular properties. in this paper, we propose a universal smile-graph representation learning model, namely unimap. firstly, an embedding layer is employed to obtain the token and node/edge representation in smiles and graph, respectively. a multi-layer transformer is then utilized to conduct deep cross-modality fusion. specially, four kinds of pre-training tasks are designed for unimap, including multi-level cross-modality masking (cmm), smiles-graph matching (sgm), fragment-level alignment (fla), and domain knowledge learning (dkl). in this way, both global (i.e. sgm and dkl) and local (i.e. cmm and fla) alignments are integrated to achieve comprehensive cross-modality fusion. we evaluate unimap on various downstream tasks, i.e. molecular property prediction, drug-target affinity prediction and drug-drug interaction. experimental results show that unimap outperforms current state-of-the-art pre-training methods.we also visualize the learned representations to demonstrate the effect of multi-modality integration.\",\n",
       "  'categories': 'cs.lg cs.ai q-bio.bm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-22',\n",
       "  'updated': '2024-11-04',\n",
       "  'authors': ['shikun feng',\n",
       "   'lixin yang',\n",
       "   'yanwen huang',\n",
       "   'yuyan ni',\n",
       "   'weiying ma',\n",
       "   'yanyan lan'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.14216'},\n",
       " {'title': 'effects of phylogeny on coexistence in model communities',\n",
       "  'id': '2310.14392',\n",
       "  'abstract': \"species' interactions are shaped by their traits. thus, we expect traits -- in particular, trait (dis)similarity -- to play a central role in determining whether a particular set of species coexists. traits are, in turn, the outcome of an eco-evolutionary process summarized by a phylogenetic tree. therefore, the phylogenetic tree associated with a set of species should carry information about the dynamics and assembly properties of the community. many studies have highlighted the potentially complex ways in which this phylogenetic information is translated into species' ecological properties. however, much less emphasis has been placed on developing clear, quantitative expectations for community properties under a particular hypothesis. to address this gap, we couple a simple model of trait evolution on a phylogenetic tree with lotka-volterra community dynamics. this allows us to derive properties of a community of coexisting species as a function of the number of traits, tree topology and the size of the species pool. our analysis highlights how phylogenies, through traits, affect the coexistence of a set of species. together, these results provide much-needed baseline expectations for the ways in which evolutionary history, summarized by phylogeny, is reflected in the size and structure of ecological communities.\",\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-22',\n",
       "  'updated': '2024-10-12',\n",
       "  'authors': ['carlos a. servan',\n",
       "   'jose a. capitan',\n",
       "   'zachary r. miller',\n",
       "   'stefano allesina'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.14392'},\n",
       " {'title': 'spiking mode-based neural networks',\n",
       "  'id': '2310.14621',\n",
       "  'abstract': 'spiking neural networks play an important role in brain-like neuromorphic computations and in studying working mechanisms of neural circuits. one drawback of training a large scale spiking neural network is that updating all weights is quite expensive. furthermore, after training, all information related to the computational task is hidden into the weight matrix, prohibiting us from a transparent understanding of circuit mechanisms. therefore, in this work, we address these challenges by proposing a spiking mode-based training protocol, where the recurrent weight matrix is explained as a hopfield-like multiplication of three matrices: input, output modes and a score matrix. the first advantage is that the weight is interpreted by input and output modes and their associated scores characterizing the importance of each decomposition term. the number of modes is thus adjustable, allowing more degrees of freedom for modeling the experimental data. this significantly reduces the training cost because of significantly reduced space complexity for learning. training spiking networks is thus carried out in the mode-score space. the second advantage is that one can project the high dimensional neural activity (filtered spike train) in the state space onto the mode space which is typically of a low dimension, e.g., a few modes are sufficient to capture the shape of the underlying neural manifolds. we successfully apply our framework in two computational tasks -- digit classification and selective sensory integration tasks. our method accelerate the training of spiking neural networks by a hopfield-like decomposition, and moreover this training leads to low-dimensional attractor structures of high-dimensional neural dynamics.',\n",
       "  'categories': 'q-bio.nc cond-mat.dis-nn cs.ai cs.ne',\n",
       "  'doi': '10.1103/physreve.110.024306',\n",
       "  'created': '2023-10-23',\n",
       "  'updated': '2024-07-18',\n",
       "  'authors': ['zhanghan lin', 'haiping huang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.14621'},\n",
       " {'title': 'second-order group knockoffs with applications to gwas',\n",
       "  'id': '2310.15069',\n",
       "  'abstract': 'conditional testing via the knockoff framework allows one to identify -- among large number of possible explanatory variables -- those that carry unique information about an outcome of interest, and also provides a false discovery rate guarantee on the selection. this approach is particularly well suited to the analysis of genome wide association studies (gwas), which have the goal of identifying genetic variants which influence traits of medical relevance.   while conditional testing can be both more powerful and precise than traditional gwas analysis methods, its vanilla implementation encounters a difficulty common to all multivariate analysis methods: it is challenging to distinguish among multiple, highly correlated regressors. this impasse can be overcome by shifting the object of inference from single variables to groups of correlated variables. to achieve this, it is necessary to construct \"group knockoffs.\" while successful examples are already documented in the literature, this paper substantially expands the set of algorithms and software for group knockoffs. we focus in particular on second-order knockoffs, for which we describe correlation matrix approximations that are appropriate for gwas data and that result in considerable computational savings. we illustrate the effectiveness of the proposed methods with simulations and with the analysis of albuminuria data from the uk biobank.   the described algorithms are implemented in an open-source julia package knockoffs.jl, for which both r and python wrappers are available.',\n",
       "  'categories': 'stat.me q-bio.gn stat.ap',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-23',\n",
       "  'updated': '2024-03-03',\n",
       "  'authors': ['benjamin b chu',\n",
       "   'jiaqi gu',\n",
       "   'zhaomeng chen',\n",
       "   'tim morrison',\n",
       "   'emmanuel candes',\n",
       "   'zihuai he',\n",
       "   'chiara sabatti'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.15069'},\n",
       " {'title': 'hierarchical event descriptor library schema for eeg data annotation',\n",
       "  'id': '2310.15173',\n",
       "  'abstract': 'standardizing terminology to annotate electrophysiological events can improve both computational research and clinical care. sharing data enriched with standard terms can facilitate data exploration, from case studies to mega-analyses. the machine readability of such electrophysiological event annotations is essential for performing analyses efficiently across software tools and packages. hierarchical event descriptors (hed) provide a framework for describing events in neuroscience experiments. hed library schemas extend the standard hed schema vocabulary to include specialized vocabularies, such as standardized clinical terms for electrophysiological events. the standardized computer-based organized reporting of eeg (score) defines terms for annotating eeg events, including artifacts. this study developed a hed library schema for score, making the terms machine-readable. we demonstrate that the hed-score library schema can be used to annotate events in eeg data stored in the brain imaging data structure (bids). clinicians and researchers worldwide can now use the hed-score library schema to annotate and compute on electrophysiological data obtained from the human brain.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-04',\n",
       "  'updated': '2024-10-27',\n",
       "  'authors': ['dora hermes',\n",
       "   'tal pal attia',\n",
       "   'sándor beniczky',\n",
       "   'jorge bosch-bayard',\n",
       "   'arnaud delorme',\n",
       "   'brian nils lundstrom',\n",
       "   'christine rogers',\n",
       "   'stefan rampp',\n",
       "   'seyed yahya shirazi',\n",
       "   'dung truong',\n",
       "   'pedro valdes-sosa',\n",
       "   'greg worrell',\n",
       "   'scott makeig',\n",
       "   'kay robbins'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.15173'},\n",
       " {'title': 'reputation-based synergy and discounting mechanism promotes cooperation',\n",
       "  'id': '2310.15488',\n",
       "  'abstract': 'a good group reputation often facilitates more efficient synergistic teamwork in production activities. here we translate this simple motivation into a reputation-based synergy and discounting mechanism in the public goods game. specifically, the reputation type of a group, either good or bad determined by a reputation threshold, modifies the nonlinear payoff structure described by a unified reputation impact factor. results show that this reputation-based incentive mechanism could effectively promote cooperation compared with linear payoffs, despite the coexistence of synergy and discounting effects. notably, the complicated interactions between reputation impact and reputation threshold result in a sharp phase transition from full cooperation to full defection. we also find that the presence of a few discounting groups could increase the average payoffs of cooperators, leading to an interesting phenomenon that when the reputation threshold is raised, the gap between the average payoffs of cooperations and defectors increases while the overall payoff decreases. our work provides important insights into facilitating cooperation in social groups.',\n",
       "  'categories': 'physics.soc-ph q-bio.pe',\n",
       "  'doi': '10.1088/1367-2630/ad345d',\n",
       "  'created': '2023-10-23',\n",
       "  'updated': '2023-11-05',\n",
       "  'authors': ['wenqiang zhu',\n",
       "   'xin wang',\n",
       "   'chaoqian wang',\n",
       "   'longzhao liu',\n",
       "   'hongwei zheng',\n",
       "   'shaoting tang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.15488'},\n",
       " {'title': 'spectral-based detection of chromatin loops in multiplexed   super-resolution fish data',\n",
       "  'id': '2310.15701',\n",
       "  'abstract': \"involved in mitotic condensation, interaction of transcriptional regulatory elements or isolation of structural domains, understanding loop formation is becoming a paradigm in the deciphering of chromatin architecture and its functional role. despite the emergence of increasingly powerful genome visualization techniques, the high variability in cell populations and the randomness of conformations still make loop detection a challenge. we introduce a new approach for determining the presence and frequency of loops in a collection of experimental conformations obtained by multiplexed super-resolution imaging. based on a spectral approach, in conjunction with neural networks, this method offers a powerful tool to detect loops in large experimental data sets, both at the population and single cell level. the method's performance is confirmed by applying it to recently published experimental data, where it provides a detailed and statistically quantified description of the global architecture of the chromosomal region under study.\",\n",
       "  'categories': 'q-bio.qm cond-mat.soft physics.bio-ph',\n",
       "  'doi': '10.1038/s41467-024-51650-w',\n",
       "  'created': '2023-10-24',\n",
       "  'updated': '',\n",
       "  'authors': ['michaël liefsoens', 'timothy földes', 'maria barbi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.15701'},\n",
       " {'title': 'phenotype selection due to mutational robustness',\n",
       "  'id': '2310.15729',\n",
       "  'abstract': 'the mutation-selection mechanism of darwinian evolution gives rise not only to adaptation to environmental conditions but also to the enhancement of robustness against mutations. when two or more phenotypes have the same fitness value, the robustness distribution for different phenotypes can vary. thus, we expect that some phenotypes are favored in evolution and that some are hardly selected because of a selection bias for mutational robustness. in this study, we investigated this selection bias for phenotypes in a model of gene regulatory networks (grns) using numerical simulations. the model had one input gene accepting a signal from the outside and one output gene producing a target protein, and the fitness was high if the output for the full signal was much higher than that for no signal. the model exhibited three types of responses to changes in the input signal: monostable, toggle switch, and one-way switch. we regarded these three response types as three distinguishable phenotypes. we constructed a randomly generated set of grns using the multicanonical monte carlo method originally developed in statistical physics and compared it to the outcomes of evolutionary simulations. one-way switches were strongly suppressed during evolution because of their lack of mutational robustness. by examining one-way switch grns in detail, we found that mutationally robust grns obtained by evolutionary simulations and non-robust grns obtained by mcmc have different network structures. while robust grns have a common core motif, non-robust grns lack this motif. the bistability of non-robust grns is considered to be realized cooperatively by many genes, and these cooperative genotypes have been suppressed by evolution.',\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech physics.bio-ph',\n",
       "  'doi': '10.1371/journal.pone.0311058',\n",
       "  'created': '2023-10-24',\n",
       "  'updated': '2024-09-07',\n",
       "  'authors': ['macoto kikuchi'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.15729'},\n",
       " {'title': \"xeno amino acids: a look into biochemistry as we don't know it\",\n",
       "  'id': '2310.15857',\n",
       "  'abstract': \"would another origin of life resemble earth's biochemical use of amino acids? here we review current knowledge at three levels: 1) could other classes of chemical structure serve as building blocks for biopolymer structure and catalysis? amino acids now seem both readily available to, and a plausible chemical attractor for, life as we don't know it. amino acids thus remain important and tractable targets for astrobiological research. 2) if amino acids are used, would we expect the same l-alpha-structural subclass used by life? despite numerous ideas, it is not clear why life favors l-enantiomers. it seems clearer, however, why life on earth uses the shortest possible (alpha-) amino acid backbone, and why each carries only one side chain. however, assertions that other backbones are physicochemically impossible have relaxed into arguments that they are disadvantageous. 3) would we expect a similar set of side chains to those within the genetic code? not only do many plausible alternatives exist and evidence exists for both evolutionary advantage and physicochemical constraint for those encoded by life. overall, as focus shifts from amino acids as a chemical class to specific side chains used by post-luca biology, the probable role of physicochemical constraint diminishes relative to that of biological evolution. exciting opportunities now present themselves for laboratory work and computing to explore how changing the amino acid alphabet alters the universe of protein folds. near-term milestones include: a) expanding evidence about amino acids as attractors within chemical evolution; b) extending characterization of other backbones relative to biological proteins; c) merging computing and laboratory explorations of structures and functions unlocked by xeno peptides.\",\n",
       "  'categories': 'q-bio.bm q-bio.pe',\n",
       "  'doi': '10.3390/life13122281',\n",
       "  'created': '2023-10-24',\n",
       "  'updated': '2023-10-30',\n",
       "  'authors': ['sean m. brown', 'christopher mayer-bacon', 'stephen freeland'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.15857'},\n",
       " {'title': 'compressed representation of brain genetic transcription',\n",
       "  'id': '2310.16113',\n",
       "  'abstract': 'the architecture of the brain is too complex to be intuitively surveyable without the use of compressed representations that project its variation into a compact, navigable space. the task is especially challenging with high-dimensional data, such as gene expression, where the joint complexity of anatomical and transcriptional patterns demands maximum compression. established practice is to use standard principal component analysis (pca), whose computational felicity is offset by limited expressivity, especially at great compression ratios. employing whole-brain, voxel-wise allen brain atlas transcription data, here we systematically compare compressed representations based on the most widely supported linear and non-linear methods-pca, kernel pca, non-negative matrix factorization (nmf), t-stochastic neighbour embedding (t-sne), uniform manifold approximation and projection (umap), and deep auto-encoding-quantifying reconstruction fidelity, anatomical coherence, and predictive utility with respect to signalling, microstructural, and metabolic targets. we show that deep auto-encoders yield superior representations across all metrics of performance and target domains, supporting their use as the reference standard for representing transcription patterns in the human brain.',\n",
       "  'categories': 'cs.lg q-bio.gn q-bio.nc',\n",
       "  'doi': '10.1002/hbm.26795',\n",
       "  'created': '2023-10-24',\n",
       "  'updated': '2024-06-20',\n",
       "  'authors': ['james k ruffle',\n",
       "   'henry watkins',\n",
       "   'robert j gray',\n",
       "   'harpreet hyare',\n",
       "   'michel thiebaut de schotten',\n",
       "   'parashkev nachev'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.16113'},\n",
       " {'title': 'the rejection that defies anti-rejection drugs. chronic vascular   rejection (allograft vasculopathy): the role of terminology and linguistic   relativity',\n",
       "  'id': '2310.16182',\n",
       "  'abstract': 'solid organ transplantation has by now become a common medical procedure. owing to the introduction of new immunosuppressive drugs, the allograft loss due to acute rejection has been reduced significantly over time. tragically, the number of donor organs lost due to allograft vasculopathy (av), generally named chronic vascular rejection or chronic rejection, has remained significant and unchanged for decades. we argue that designation of av as chronic rejection, and its classification as a delayed long-lasting reaction of recipient immune effectors against donor alloantigens have given us the wrong impression that we have identified the necessary cause of the disease. however, whatever treatment options we have in the anti-rejection toolbox, despite their success in treating acute rejection, do not work for av. yet, the scientific community has continued to conceptualize and approach av within the alloimmunity and rejection model. due to unproductive research from the alloimmunity perspective, the number of transplanted hearts lost due to this pathology today is almost the same as it was fifty years ago. we believe that this phenomenon falls under the rubric of linguistic relativity, and that the language we chose to name the disease has restricted our cognitive ability to solve the problem. while the initial perception of the av as chronic rejection was logical and scientific, the subsequent experience revealed that such perception and approach have been fruitless, and likely are incorrect. considering our tragic failure to prevent and treat allograft vasculopathy using all available knowledge on alloimmunity and rejection, we must finally disassociate the former from the latter. a good way to start this process is to change the words we are using; particularly, the words we chose to name the disease. we have to step out of the alloimmunity rejection box',\n",
       "  'categories': 'q-bio.to',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-24',\n",
       "  'updated': '2024-08-20',\n",
       "  'authors': ['vladimir m. subbotin', 'michael v. subotin'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.16182'},\n",
       " {'title': 'sequencelab: a comprehensive benchmark of computational methods for   comparing genomic sequences',\n",
       "  'id': '2310.16908',\n",
       "  'abstract': 'computational complexity is a key limitation of genomic analyses. thus, over the last 30 years, researchers have proposed numerous fast heuristic methods that provide computational relief. comparing genomic sequences is one of the most fundamental computational steps in most genomic analyses. due to its high computational complexity, optimized exact and heuristic algorithms are still being developed. we find that these methods are highly sensitive to the underlying data, its quality, and various hyperparameters. despite their wide use, no in-depth analysis has been performed, potentially falsely discarding genetic sequences from further analysis and unnecessarily inflating computational costs. we provide the first analysis and benchmark of this heterogeneity. we deliver an actionable overview of the 11 most widely used state-of-the-art methods for comparing genomic sequences. we also inform readers about their advantages and downsides using thorough experimental evaluation and different real datasets from all major manufacturers (i.e., illumina, ont, and pacbio). sequencelab is publicly available at https://github.com/cmu-safari/sequencelab.',\n",
       "  'categories': 'q-bio.gn cs.ar q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-25',\n",
       "  'updated': '2024-01-21',\n",
       "  'authors': ['maximilian-david rumpf',\n",
       "   'mohammed alser',\n",
       "   'arvid e. gollwitzer',\n",
       "   'joel lindegger',\n",
       "   'nour almadhoun',\n",
       "   'can firtina',\n",
       "   'serghei mangul',\n",
       "   'onur mutlu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.16908'},\n",
       " {'title': 'improving performance in colorectal cancer histology decomposition using   deep and ensemble machine learning',\n",
       "  'id': '2310.16954',\n",
       "  'abstract': 'in routine colorectal cancer management, histologic samples stained with hematoxylin and eosin are commonly used. nonetheless, their potential for defining objective biomarkers for patient stratification and treatment selection is still being explored. the current gold standard relies on expensive and time-consuming genetic tests. however, recent research highlights the potential of convolutional neural networks (cnns) in facilitating the extraction of clinically relevant biomarkers from these readily available images. these cnn-based biomarkers can predict patient outcomes comparably to golden standards, with the added advantages of speed, automation, and minimal cost. the predictive potential of cnn-based biomarkers fundamentally relies on the ability of convolutional neural networks (cnns) to classify diverse tissue types from whole slide microscope images accurately. consequently, enhancing the accuracy of tissue class decomposition is critical to amplifying the prognostic potential of imaging-based biomarkers. this study introduces a hybrid deep and ensemble machine learning model that surpassed all preceding solutions for this classification task. our model achieved 96.74% accuracy on the external test set and 99.89% on the internal test set. recognizing the potential of these models in advancing the task, we have made them publicly available for further research and development.',\n",
       "  'categories': 'q-bio.qm cs.cv',\n",
       "  'doi': '10.1016/j.heliyon.2024.e37561',\n",
       "  'created': '2023-10-25',\n",
       "  'updated': '2024-09-24',\n",
       "  'authors': ['fabi prezja',\n",
       "   'leevi annala',\n",
       "   'sampsa kiiskinen',\n",
       "   'suvi lahtinen',\n",
       "   'timo ojala',\n",
       "   'pekka ruusuvuori',\n",
       "   'teijo kuopio'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.16954'},\n",
       " {'title': 't(w)o patch or not t(w)o patch: a novel additional food model',\n",
       "  'id': '2310.17003',\n",
       "  'abstract': 'a number of top down bio-control models have been proposed where the introduced predators\\' efficacy is enhanced via the provision of additional food (af). however, if the predator has a pest dependent monotone functional response, pest extinction is unattainable. in the current manuscript, we propose a model where a predator with pest dependent monotone functional response is introduced into a ``patch\" such as a prairie strip with af, and then disperses or drifts into a neighboring ``patch\" such as a crop field, to target a pest. we show the pest extinction state is attainable in the crop field and can be globally attracting.   the af model with patch structure can eliminate predator explosion present therein and can keep pest densities lower than the classical top-down bio-control model. we provide the first proof of the global stability of the interior equilibrium for the classical af model.   we also observe ``patch-specific chaos\" - the pest occupying the crop field can oscillate chaotically, while the pest in the prairie strip oscillates periodically. we discuss these results in light of bio-control strategies that utilize state-of-the-art farming practices such as prairie strips and drift and dispersal pressures driven by climate change.',\n",
       "  'categories': 'q-bio.pe math.ds',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-25',\n",
       "  'updated': '2024-08-29',\n",
       "  'authors': ['urvashi verma', 'aniket banerjee', 'rana d. parshad'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.17003'},\n",
       " {'title': 'phylogenetic invariants: straightforward from the general markov to   equivariant models',\n",
       "  'id': '2310.18053',\n",
       "  'abstract': 'in the last decade, some algebraic tools have been successfully applied to phylogenetic reconstruction. these tools are mainly based on the knowledge of equations describing algebraic varieties associated to phylogenetic trees evolving under markov processes of molecular substitution, the so called phylogenetic invariants. although the theory involved allows to explicitly obtain these equations for all equivariant models (which include some of the most popular nucleotide substitution models), practical uses of these algebraic tools have been restricted to the case of the general markov model. arguably, one of the reasons for this restriction is that knowledge of linear representation theory is required before making these equations explicit.   with the aim of enlarging the practical uses of algebraic phylogenetics, in this paper we prove that phylogenetic invariants for trees evolving under equivariant models can be derived from phylogenetic invariants for the general markov model, without the need of representation theory. our main result states that the algebraic variety corresponding to a phylogenetic tree evolving under an equivariant model is an irreducible component of the variety corresponding to the same tree under the general markov model cut with the linear space defined by the model. we also prove that, for any equivariant model, those phylogenetic invariants that are relevant for practical uses (e.g. tree reconstruction) can be simply deduced from a single rank constraint on the matrices obtained by flattening the joint distribution at the leaves of the tree. this condition can be easily tested from singular values of the matrices and extends our results from trees to phylogenetic networks.',\n",
       "  'categories': 'q-bio.pe math.ag',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-27',\n",
       "  'updated': '2024-02-28',\n",
       "  'authors': ['marta casanellas', 'jesús fernández-sánchez'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.18053'},\n",
       " {'title': 'polygenic dynamics underlying the response of quantitative traits to   directional selection',\n",
       "  'id': '2310.18106',\n",
       "  'abstract': \"we study the response of a quantitative trait to exponential directional selection in a finite haploid population at the genetic and the phenotypic level. we assume an infinite sites model, in which the number of new mutations per generation in the population follows a poisson distribution (with mean $\\\\theta$) and each mutation occurs at a new, previously monomorphic site. mutation effects are beneficial and drawn from a distribution. sites are unlinked and contribute additively to the trait. assuming that selection is stronger than random genetic drift, we model the initial phase of the dynamics by a supercritical galton-watson process. this enables us to obtain time-dependent results. we show that the copy-number distribution of the mutant in generation n, conditioned on non-extinction until n, is described accurately by the deterministic increase from an initial distribution with mean 1. this distribution is related to the absolutely continuous part $w^+$ of the random variable, typically denoted $w$, that characterizes the stochasticity accumulating during the mutant's sweep. on this basis, we derive explicitly the (approximate) time dependence of the mutant frequency distribution, of the expected mean and variance of the trait and of the expected number of segregating sites. unexpectedly, we obtain highly accurate approximations for all times, even for the quasi-stationary phase where we refine classical results. in addition, we find that $\\\\theta$ is the main determinant of the pattern of adaptation at the genetic level, i.e., whether the initial allele-frequency dynamics are best described by sweep-like patterns at few loci or small allele-frequency shifts at many. the selection strength determines primarily the rate of adaptation. the accuracy of our results is tested by comprehensive simulations in a wright-fisher framework.\",\n",
       "  'categories': 'q-bio.pe',\n",
       "  'doi': '10.1101/2023.02.23.529647',\n",
       "  'created': '2023-10-27',\n",
       "  'updated': '2024-04-09',\n",
       "  'authors': ['hannah götsch', 'reinhard bürger'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.18106'},\n",
       " {'title': 'leveraging machine learning models for peptide-protein interaction   prediction',\n",
       "  'id': '2310.18249',\n",
       "  'abstract': 'peptides play a pivotal role in a wide range of biological activities through participating in up to 40% protein-protein interactions in cellular processes. they also demonstrate remarkable specificity and efficacy, making them promising candidates for drug development. however, predicting peptide-protein complexes by traditional computational approaches, such as docking and molecular dynamics simulations, still remains a challenge due to high computational cost, flexible nature of peptides, and limited structural information of peptide-protein complexes. in recent years, the surge of available biological data has given rise to the development of an increasing number of machine learning models for predicting peptide-protein interactions. these models offer efficient solutions to address the challenges associated with traditional computational approaches. furthermore, they offer enhanced accuracy, robustness, and interpretability in their predictive outcomes. this review presents a comprehensive overview of machine learning and deep learning models that have emerged in recent years for the prediction of peptide-protein interactions.',\n",
       "  'categories': 'q-bio.bm q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-27',\n",
       "  'updated': '2024-02-07',\n",
       "  'authors': ['song yin', 'xuenan mi', 'diwakar shukla'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.18249'},\n",
       " {'title': 'bioimage.io chatbot: a community-driven ai assistant for integrative   computational bioimaging',\n",
       "  'id': '2310.18351',\n",
       "  'abstract': 'we present the bioimage$.$io chatbot, an ai assistant powered by large language models and supported by a community-driven knowledge base and toolset. this chatbot is designed to cater to a wide range of user needs through a flexible extension mechanism that spans from information retrieval to ai-enhanced analysis and microscopy control. embracing open-source principles, the chatbot is designed to evolve through community contributions. by simplifying navigation through the intricate bioimaging landscape, the bioimage$.$io chatbot empowers life sciences to progress by leveraging the collective expertise and innovation of its users.',\n",
       "  'categories': 'cs.ai q-bio.qm',\n",
       "  'doi': '10.5281/zenodo.10032227',\n",
       "  'created': '2023-10-23',\n",
       "  'updated': '2024-04-16',\n",
       "  'authors': ['wanlu lei',\n",
       "   'caterina fuster-barceló',\n",
       "   'gabriel reder',\n",
       "   'arrate muñoz-barrutia',\n",
       "   'wei ouyang'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.18351'},\n",
       " {'title': 'forum on immune digital twins: a meeting report',\n",
       "  'id': '2310.18374',\n",
       "  'abstract': \"medical digital twins are computational models of human biology relevant to a given medical condition, which can be tailored to an individual patient, thereby predicting the course of disease and individualized treatments, an important goal of personalized medicine. the immune system, which has a central role in many diseases, is highly heterogeneous between individuals, and thus poses a major challenge for this technology. if medical digital twins are to faithfully capture the characteristics of a patient's immune system, we need to answer many questions, such as: what do we need to know about the immune system to build mathematical models that reflect features of an individual? what data do we need to collect across the different scales of immune system action? what are the right modeling paradigms to properly capture immune system complexity? in february 2023, an international group of experts convened in lake nona, fl for two days to discuss these and other questions related to digital twins of the immune system. the group consisted of clinicians, immunologists, biologists, and mathematical modelers, representative of the interdisciplinary nature of medical digital twin development. a video recording of the entire event is available. this paper presents a synopsis of the discussions, brief descriptions of ongoing digital twin projects at different stages of progress. it also proposes a 5-year action plan for further developing this technology. the main recommendations are to identify and pursue a small number of promising use cases, to develop stimulation-specific assays of immune function in a clinical setting, and to develop a database of existing computational immune models, as well as advanced modeling technology and infrastructure.\",\n",
       "  'categories': 'q-bio.ot',\n",
       "  'doi': '10.1038/s41540-024-00345-5',\n",
       "  'created': '2023-10-26',\n",
       "  'updated': '',\n",
       "  'authors': ['reinhard laubenbacher',\n",
       "   'fred adler',\n",
       "   'gary an',\n",
       "   'filippo castiglione',\n",
       "   'stephen eubank',\n",
       "   'luis l. fonseca',\n",
       "   'james glazier',\n",
       "   'tomas helikar',\n",
       "   'marti jett-tilton',\n",
       "   'denise kirschner',\n",
       "   'paul macklin',\n",
       "   'borna mehrad',\n",
       "   'beth moore',\n",
       "   'virginia pasour',\n",
       "   'ilya shmulevich',\n",
       "   'amber smith',\n",
       "   'isabel voigt',\n",
       "   'thomas e. yankeelov',\n",
       "   'tjalf ziemssen'],\n",
       "  'affiliation': ['department of medicine, university of florida, gainesville, fl',\n",
       "   'department of mathematics and school of biological sciences, university of utah, salt lake city, ut',\n",
       "   'department of surgery, university of vermont, burlington, vt',\n",
       "   'biotechnology research center, technology innovation institute, abu dhabi, united arab emirates',\n",
       "   'biocomplexity institute and initiative, university of virginia, charlottesville, va',\n",
       "   'department of medicine, university of florida, gainesville, fl',\n",
       "   'department of intelligent systems engineering, indiana university, bloomington, in',\n",
       "   'department of biochemistry, university of nebraska, lincoln, ne',\n",
       "   'u.s. walter reed army institute of research, silver spring, md',\n",
       "   'department of microbiology and immunology, university of michigan, ann arbor, mi',\n",
       "   'department of intelligent systems engineering, indiana university, bloomington, in',\n",
       "   'department of medicine, university of florida, gainesville, fl',\n",
       "   'department of microbiology and immunology, university of michigan, ann arbor, mi',\n",
       "   'u.s. army research office, research triangle park, nc',\n",
       "   'institute for systems biology, seattle, wa',\n",
       "   'department of pediatrics, university of tennessee health science center, memphis, tn',\n",
       "   'center for clinical neuroscience, carl gustav carus university hospital, dresden, germany',\n",
       "   'department of biomedical engineering, oden institute for computational engineering and sciences, departments of biomedical engineering, diagnostic medicine, oncology, the university of texas, austin, tx, and department of imaging physics, the university of texas md anderson cancer center',\n",
       "   'center for clinical neuroscience, carl gustav carus university hospital, dresden, germany'],\n",
       "  'url': 'https://arxiv.org/abs/2310.18374'},\n",
       " {'title': 'liquid hopfield model: retrieval and localization in multicomponent   liquid mixtures',\n",
       "  'id': '2310.18853',\n",
       "  'abstract': 'biological mixtures, such as the cellular cytoplasm, are composed of a large number of different components. from this heterogeneity, ordered mesoscopic structures emerge, such as liquid phases with controlled composition. these structures compete with each other for the same components. this raises several questions, such as what types of interactions allow the retrieval of multiple ordered mesoscopic structures, and what are the physical limitations for the retrieval of said structures. in this work, we develop an analytically tractable model for liquids capable of retrieving states with target compositions. we name this model the liquid hopfield model in reference to corresponding work in the theory of associative neural networks. by solving this model, we show that non-linear repulsive interactions are necessary for retrieval of target structures. we demonstrate that this is because liquid mixtures at low temperatures tend to transition to phases with few components, a phenomenon that we term localization. taken together, our results demonstrate a trade-off between retrieval and localization phenomena in liquid mixtures.',\n",
       "  'categories': 'physics.bio-ph cond-mat.dis-nn cond-mat.soft q-bio.bm',\n",
       "  'doi': '10.1073/pnas.2320504121',\n",
       "  'created': '2023-10-28',\n",
       "  'updated': '2024-12-02',\n",
       "  'authors': ['rodrigo braz teixeira',\n",
       "   'giorgio carugno',\n",
       "   'izaak neri',\n",
       "   'pablo sartori'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.18853'},\n",
       " {'title': 'transport-of-intensity model for single-mask x-ray differential phase   contrast imaging',\n",
       "  'id': '2310.19087',\n",
       "  'abstract': 'x-ray phase contrast imaging holds great promise for improving the visibility of light-element materials such as soft tissues and tumors. single-mask differential phase contrastnimaging method stands out as a simple and effective approach to yield differential phase contrast. in this work, we introduce a novel model for a single-mask phase imaging system based on the transport-of-intensity equation. our model provides an accessible understanding of signal and contrast formation in single-mask x-ray phase imaging, offering a clear perspective on the image formation process, for example, the origin of alternate bright and dark fringes in phase contrast intensity images. aided by our model, we present an efficient retrieval method that yields differential phase contrast imagery in a single acquisition step. our model gives insight into the contrast generation and its dependence on the system geometry and imaging parameters in both the initial intensity image as well as in retrieved images. the model validity as well as the proposed retrieval method is demonstrated via both experimental results on a system developed in-house as well as with monte carlo simulations. in conclusion, our work not only provides a model for an intuitive visualization of image formation but also offers a method to optimize differential phase imaging setups, holding tremendous promise for advancing medical diagnostics and other applications.',\n",
       "  'categories': 'eess.iv physics.ins-det physics.med-ph physics.optics q-bio.to',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-29',\n",
       "  'updated': '2024-01-31',\n",
       "  'authors': ['jingcheng yuan', 'mini das'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.19087'},\n",
       " {'title': 'emergence of grid-like representations by training recurrent networks   with conformal normalization',\n",
       "  'id': '2310.19192',\n",
       "  'abstract': 'grid cells in the entorhinal cortex of mammalian brains exhibit striking hexagon grid firing patterns in their response maps as the animal (e.g., a rat) navigates in a 2d open environment. in this paper, we study the emergence of the hexagon grid patterns of grid cells based on a general recurrent neural network (rnn) model that captures the navigation process. the responses of grid cells collectively form a high dimensional vector, representing the 2d self-position of the agent. as the agent moves, the vector is transformed by an rnn that takes the velocity of the agent as input. we propose a simple yet general conformal normalization of the input velocity of the rnn, so that the local displacement of the position vector in the high-dimensional neural space is proportional to the local displacement of the agent in the 2d physical space, regardless of the direction of the input velocity. we apply this mechanism to both a linear rnn and nonlinear rnns. theoretically, we provide an understanding that explains the connection between conformal normalization and the emergence of hexagon grid patterns. empirically, we conduct extensive experiments to verify that conformal normalization is crucial for the emergence of hexagon grid patterns, across various types of rnns. the learned patterns share similar profiles to biological grid cells, and the topological properties of the patterns also align with our theoretical understanding.',\n",
       "  'categories': 'q-bio.nc cs.lg stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-29',\n",
       "  'updated': '2024-02-19',\n",
       "  'authors': ['dehong xu',\n",
       "   'ruiqi gao',\n",
       "   'wen-hao zhang',\n",
       "   'xue-xin wei',\n",
       "   'ying nian wu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.19192'},\n",
       " {'title': 'spatial information allows inference of the prevalence of direct   cell-to-cell viral infection',\n",
       "  'id': '2310.19481',\n",
       "  'abstract': 'the role of direct cell-to-cell spread in viral infections - where virions spread between host and susceptible cells without needing to be secreted into the extracellular environment - has come to be understood as essential to the dynamics of medically significant viruses like hepatitis c and influenza. recent work in both the experimental and mathematical modelling literature has attempted to quantify the prevalence of cell-to-cell infection compared to the conventional free virus route using a variety of methods and experimental data. however, estimates are subject to significant uncertainty and moreover rely on data collected by inhibiting one mode of infection by either chemical or physical factors, which may influence the other mode of infection to an extent which is difficult to quantify. in this work, we conduct a simulation-estimation study to probe the practical identifiability of the proportion of cell-to-cell infection, using two standard mathematical models and synthetic data that would likely be realistic to obtain in the laboratory. we show that this quantity cannot be estimated using non-spatial data alone, and that the collection of a data which describes the spatial structure of the infection is necessary to infer the proportion of cell-to-cell infection. our results provide guidance for the design of relevant experiments and mathematical tools for accurately inferring the prevalence of cell-to-cell infection in $\\\\textit{in vitro}$ and $\\\\textit{in vivo}$ contexts.',\n",
       "  'categories': 'q-bio.qm q-bio.to',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-30',\n",
       "  'updated': '2024-05-20',\n",
       "  'authors': ['thomas williams', 'james m. mccaw', 'james osborne'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.19481'},\n",
       " {'title': 'epidemic outbreak prediction using machine learning models',\n",
       "  'id': '2310.19760',\n",
       "  'abstract': \"in today's world,the risk of emerging and re-emerging epidemics have increased.the recent advancement in healthcare technology has made it possible to predict an epidemic outbreak in a region.early prediction of an epidemic outbreak greatly helps the authorities to be prepared with the necessary medications and logistics required to keep things in control. in this article, we try to predict the epidemic outbreak (influenza, hepatitis and malaria) for the state of new york, usa using machine and deep learning algorithms, and a portal has been created for the same which can alert the authorities and health care organizations of the region in case of an outbreak. the algorithm takes historical data to predict the possible number of cases for 5 weeks into the future. non-clinical factors like google search trends,social media data and weather data have also been used to predict the probability of an outbreak.\",\n",
       "  'categories': 'cs.lg cs.si q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-30',\n",
       "  'updated': '',\n",
       "  'authors': ['akshara pramod', 'js abhishek', 'suganthi k'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.19760'},\n",
       " {'title': 're-evaluating retrosynthesis algorithms with syntheseus',\n",
       "  'id': '2310.19796',\n",
       "  'abstract': 'automated synthesis planning has recently re-emerged as a research area at the intersection of chemistry and machine learning. despite the appearance of steady progress, we argue that imperfect benchmarks and inconsistent comparisons mask systematic shortcomings of existing techniques, and unnecessarily hamper progress. to remedy this, we present a synthesis planning library with an extensive benchmarking framework, called syntheseus, which promotes best practice by default, enabling consistent meaningful evaluation of single-step models and multi-step planning algorithms. we demonstrate the capabilities of syntheseus by re-evaluating several previous retrosynthesis algorithms, and find that the ranking of state-of-the-art models changes in controlled evaluation experiments. we end with guidance for future works in this area, and call the community to engage in the discussion on how to improve benchmarks for synthesis planning.',\n",
       "  'categories': 'cs.lg cs.ai q-bio.qm',\n",
       "  'doi': '10.1039/d4fd00093e',\n",
       "  'created': '2023-10-30',\n",
       "  'updated': '2024-09-06',\n",
       "  'authors': ['krzysztof maziarz',\n",
       "   'austin tripp',\n",
       "   'guoqing liu',\n",
       "   'megan stanley',\n",
       "   'shufang xie',\n",
       "   'piotr gaiński',\n",
       "   'philipp seidl',\n",
       "   'marwin segler'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.19796'},\n",
       " {'title': 'brain decoding: toward real-time reconstruction of visual perception',\n",
       "  'id': '2310.19812',\n",
       "  'abstract': 'in the past five years, the use of generative and foundational ai systems has greatly improved the decoding of brain activity. visual perception, in particular, can now be decoded from functional magnetic resonance imaging (fmri) with remarkable fidelity. this neuroimaging technique, however, suffers from a limited temporal resolution ($\\\\approx$0.5 hz) and thus fundamentally constrains its real-time usage. here, we propose an alternative approach based on magnetoencephalography (meg), a neuroimaging device capable of measuring brain activity with high temporal resolution ($\\\\approx$5,000 hz). for this, we develop an meg decoding model trained with both contrastive and regression objectives and consisting of three modules: i) pretrained embeddings obtained from the image, ii) an meg module trained end-to-end and iii) a pretrained image generator. our results are threefold: firstly, our meg decoder shows a 7x improvement of image-retrieval over classic linear decoders. second, late brain responses to images are best decoded with dinov2, a recent foundational image model. third, image retrievals and generations both suggest that high-level visual features can be decoded from meg signals, although the same approach applied to 7t fmri also recovers better low-level features. overall, these results, while preliminary, provide an important step towards the decoding -- in real-time -- of the visual processes continuously unfolding within the human brain.',\n",
       "  'categories': 'eess.iv cs.ai cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-18',\n",
       "  'updated': '2024-03-14',\n",
       "  'authors': ['yohann benchetrit', 'hubert banville', 'jean-rémi king'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.19812'},\n",
       " {'title': 'unmasking bias in ai: a systematic review of bias detection and   mitigation strategies in electronic health record-based models',\n",
       "  'id': '2310.19917',\n",
       "  'abstract': 'objectives: leveraging artificial intelligence (ai) in conjunction with electronic health records (ehrs) holds transformative potential to improve healthcare. yet, addressing bias in ai, which risks worsening healthcare disparities, cannot be overlooked. this study reviews methods to detect and mitigate diverse forms of bias in ai models developed using ehr data. methods: we conducted a systematic review following the preferred reporting items for systematic reviews and meta-analyses (prisma) guidelines, analyzing articles from pubmed, web of science, and ieee published between january 1, 2010, and dec 17, 2023. the review identified key biases, outlined strategies for detecting and mitigating bias throughout the ai model development process, and analyzed metrics for bias assessment. results: of the 450 articles retrieved, 20 met our criteria, revealing six major bias types: algorithmic, confounding, implicit, measurement, selection, and temporal. the ai models were primarily developed for predictive tasks in healthcare settings. four studies concentrated on the detection of implicit and algorithmic biases employing fairness metrics like statistical parity, equal opportunity, and predictive equity. sixty proposed various strategies for mitigating biases, especially targeting implicit and selection biases. these strategies, evaluated through both performance (e.g., accuracy, auroc) and fairness metrics, predominantly involved data collection and preprocessing techniques like resampling, reweighting, and transformation. discussion: this review highlights the varied and evolving nature of strategies to address bias in ehr-based ai models, emphasizing the urgent needs for the establishment of standardized, generalizable, and interpretable methodologies to foster the creation of ethical ai systems that promote fairness and equity in healthcare.',\n",
       "  'categories': 'cs.ai cs.cy cs.lg q-bio.qm',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-30',\n",
       "  'updated': '2024-07-01',\n",
       "  'authors': ['feng chen',\n",
       "   'liqin wang',\n",
       "   'julie hong',\n",
       "   'jiaqi jiang',\n",
       "   'li zhou'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.19917'},\n",
       " {'title': 'meta-learning strategies through value maximization in neural networks',\n",
       "  'id': '2310.19919',\n",
       "  'abstract': 'biological and artificial learning agents face numerous choices about how to learn, ranging from hyperparameter selection to aspects of task distributions like curricula. understanding how to make these meta-learning choices could offer normative accounts of cognitive control functions in biological learners and improve engineered systems. yet optimal strategies remain challenging to compute in modern deep networks due to the complexity of optimizing through the entire learning process. here we theoretically investigate optimal strategies in a tractable setting. we present a learning effort framework capable of efficiently optimizing control signals on a fully normative objective: discounted cumulative performance throughout learning. we obtain computational tractability by using average dynamical equations for gradient descent, available for simple neural network architectures. our framework accommodates a range of meta-learning and automatic curriculum learning methods in a unified normative setting. we apply this framework to investigate the effect of approximations in common meta-learning algorithms; infer aspects of optimal curricula; and compute optimal neuronal resource allocation in a continual learning setting. across settings, we find that control effort is most beneficial when applied to easier aspects of a task early in learning; followed by sustained effort on harder aspects. overall, the learning effort framework provides a tractable theoretical test bed to study normative benefits of interventions in a variety of learning systems, as well as a formal account of optimal cognitive control strategies over learning trajectories posited by established theories in cognitive neuroscience.',\n",
       "  'categories': 'cs.ne cs.lg q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-30',\n",
       "  'updated': '2024-07-15',\n",
       "  'authors': ['rodrigo carrasco-davis', 'javier masís', 'andrew m. saxe'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.19919'},\n",
       " {'title': 'competition between transient oscillations and early stochasticity in   exponentially growing populations',\n",
       "  'id': '2310.20011',\n",
       "  'abstract': 'it has been recently shown that the exponential growth rate of a population of bacterial cells starting from a single cell shows transient oscillations due to early synchronized bursts of division. these oscillations are enhanced by cell size regulation and contain information about single-cell growth statistics. here, we report a phase transition in these oscillations as a function of growth rate variability. below the transition point, these oscillations become asymptotically deterministic and can be measured experimentally, while above the transition point, the stochasticity in population growth dominates the oscillations and masks all the information about single cell growth statistics. the analytically calculated transition point, which roughly corresponds to $13\\\\%$ variability in single-cell growth rate, falls within physiologically relevant parameters. additionally, we show that the oscillations can stochastically emerge even when the initial state contains multiple cells with out-of-phase division cycles. we show that the amplitude and the phase of these oscillations are stochastic and would vary across repeated measurements with the same initial conditions. we provide analytic expressions as well as numerical estimates for the typical oscillation amplitude and the number of generations before the amplitude falls below a given measurement threshold for e. coli in multiple growth conditions.',\n",
       "  'categories': 'q-bio.pe physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-30',\n",
       "  'updated': '2024-08-01',\n",
       "  'authors': ['yaïr hein', 'farshid jafarpour'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.20011'},\n",
       " {'title': 'effective connectivity signatures in major depressive disorder: fmri   study using a multi-site dataset',\n",
       "  'id': '2310.20231',\n",
       "  'abstract': \"diagnosis of major depressive disorder (mdd) primarily relies on the patient's self-reported symptoms and a clinical evaluation. effective connectivity (ec) from resting-state functional magnetic resonance imaging (rs-fmri) analysis can reflect the directionality of connections between brain regions, making it a candidate method to classify mdd. this study used granger causality analysis to extract ec features from a large multi-site mdd dataset. the combat algorithm and multivariate linear regression were used to harmonize site difference and to remove age and sex covariates, respectively. two-sample t-tests and model-based feature selection methods were used to screen for highly discriminative ec features for mdd, and lightgbm was used to classify mdd. in this large-scale multi-site rs-fmri dataset, 97 ec features deemed highly discriminative for mdd were screened. in the nested five-fold cross-validation, the best classification model with the 97 ec features achieved accuracy, sensitivity, and specificity of 94.35%, 93.52%, and 95.25%, respectively. in another independent large dataset, which tested the generalization performance of the 97 ec features, the best classification models achieved 94.74%, 90.59%, and 96.75% for accuracy, sensitivity, and specificity, respectively. this work demonstrated that ec had a reasonable discriminative ability and supported the notion for using ec to potentially assist clinical diagnosis of mdd.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-31',\n",
       "  'updated': '2023-12-29',\n",
       "  'authors': ['peishan dai',\n",
       "   'yun shi',\n",
       "   'tong xiong',\n",
       "   'xiaoyan zhou',\n",
       "   'shenghui liao',\n",
       "   'zhongchao huang',\n",
       "   'xiaoping yi',\n",
       "   'bihong t. chen'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.20231'},\n",
       " {'title': \"a lipid rafts theory of alzheimer's disease\",\n",
       "  'id': '2310.20232',\n",
       "  'abstract': \"i present a theory of alzheimer's disease (ad) that explains its symptoms, pathology, and risk factors. to do this, i introduce a new theory of brain plasticity that elucidates the physiological roles of ad-related agents. new events generate synaptic and branching candidates competing for long-term enhancement. competition resolution crucially depends on the formation of membrane lipid rafts, which requires astrocyte-produced cholesterol. sporadic ad is caused by impaired formation of plasma membrane lipid rafts, which prevents the conversion of short- to long-term memory, and yields excessive tau phosphorylation, intracellular cholesterol accumulation, synaptic dysfunction, and neurodegeneration. amyloid beta (abeta) production is promoted by cholesterol during the switch to competition resolution, and cholesterol accumulation stimulates chronic abeta production, secretion, and aggregation. the theory addresses all of the major established facts known about the disease, and is supported by strong evidence.\",\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-31',\n",
       "  'updated': '2024-05-19',\n",
       "  'authors': ['ari rappoport'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.20232'},\n",
       " {'title': 'tensor formalism for predicting synaptic connections with ensemble   modeling or optimization',\n",
       "  'id': '2310.20309',\n",
       "  'abstract': 'theoretical neuroscientists often try to understand how the structure of a neural network relates to its function by focusing on structural features that would either follow from optimization or occur consistently across possible implementations. both optimization theories and ensemble modeling approaches have repeatedly proven their worth, and it would simplify theory building considerably if predictions from both theory types could be derived and tested simultaneously. here we show how tensor formalism from theoretical physics can be used to unify and solve many optimization and ensemble modeling approaches to predicting synaptic connectivity from neuronal responses. we specifically focus on analyzing the solution space of synaptic weights that allow a threshold-linear neural network to respond in a prescribed way to a limited number of input conditions. for optimization purposes, we compute the synaptic weight vector that minimizes an arbitrary quadratic loss function. for ensemble modeling, we identify synaptic weight features that occur consistently across all solutions bounded by an arbitrary ellipsoid. we derive a common solution to this suite of nonlinear problems by showing how each of them reduces to an equivalent linear problem that can be solved analytically. although identifying the equivalent linear problem is nontrivial, our tensor formalism provides an elegant geometrical perspective that allows us to solve the problem approximately in an analytical way or exactly using numeric methods. the final algorithm is applicable to a wide range of interesting neuroscience problems, and the associated geometric insights may carry over to other scientific problems that require constrained optimization.',\n",
       "  'categories': 'q-bio.nc cond-mat.dis-nn physics.bio-ph',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-31',\n",
       "  'updated': '2024-06-18',\n",
       "  'authors': ['tirthabir biswas', 'tianzhi lambus li', 'james e. fitzgerald'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.20309'},\n",
       " {'title': 'modular boundaries in recurrent neural networks',\n",
       "  'id': '2310.20601',\n",
       "  'abstract': 'recent theoretical and experimental work in neuroscience has focused on the representational and dynamical character of neural manifolds --subspaces in neural activity space wherein many neurons coactivate. importantly, neural populations studied under this \"neural manifold hypothesis\" are continuous and not cleanly divided into separate neural populations. this perspective clashes with the \"modular hypothesis\" of brain organization, wherein neural elements maintain an \"all-or-nothing\" affiliation with modules. in line with this modular hypothesis, recent research on recurrent neural networks suggests that multi-task networks become modular across training, such that different modules specialize for task-general dynamical motifs. if the modular hypothesis is true, then it would be important to use a dimensionality reduction technique that captures modular structure. here, we investigate the features of such a method. we leverage rnns as a model system to study the character of modular neural populations, using a community detection method from network science known as modularity maximization to partition neurons into distinct modules. these partitions allow us to ask the following question: do these modular boundaries matter to the system? ...',\n",
       "  'categories': 'q-bio.nc cs.ai cs.lg',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-31',\n",
       "  'updated': '2024-10-18',\n",
       "  'authors': ['jacob tanner',\n",
       "   'sina mansour l.',\n",
       "   'ludovico coletta',\n",
       "   'alessandro gozzi',\n",
       "   'richard f. betzel'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.20601'},\n",
       " {'title': 'interactions between different birds of prey as a random point process',\n",
       "  'id': '2310.20670',\n",
       "  'abstract': 'the two-dimensional coulomb gas is a one-parameter family of random point processes, depending on the inverse temperature $\\\\beta$. based on previous work, it is proposed as a simple statistical measure to quantify the intra- and interspecies repulsion among three different highly territorial birds of prey. using data from the area of the teutoburger wald over 20 years, we fit the nearest and next-to-nearest neighbour spacing distributions between the respective nests of goshawk, eagle owl and the previously examined common buzzard to $\\\\beta$ of the coulomb gas. within each species, the repulsion measured in this way deviates significantly from the poisson process of independent points in the plane. in contrast, the repulsion amongst each of two species is found to be considerably lower and closer to poisson. methodologically we investigate the influence of the terrain, of a shorter interaction range given by the two-dimensional yukawa interaction, and the statistical independence of the time moving average we use for the yearly ensembles of occupied nests. we also check that an artificial random displacement of the original nest positions of the order of the mean level spacing quickly destroys the repulsion measured by $\\\\beta> 0$. a simple, approximate analytical expression for the nearest neighbour spacing distribution derived from non-hermitian random matrix theory proves to be very useful.',\n",
       "  'categories': 'cond-mat.stat-mech math-ph math.mp q-bio.qm',\n",
       "  'doi': '10.1088/1742-5468/ad37be',\n",
       "  'created': '2023-10-31',\n",
       "  'updated': '2024-04-04',\n",
       "  'authors': ['gernot akemann',\n",
       "   'nayden chakarov',\n",
       "   'oliver krüger',\n",
       "   'adam mielke',\n",
       "   'meinolf ottensmann',\n",
       "   'patricia päßler'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2310.20670'},\n",
       " {'title': 'scx: a user-friendly tool for scrna-seq exploration',\n",
       "  'id': '2311.00012',\n",
       "  'abstract': 'single-cell rna sequencing (scrna-seq) has transformed our ability to explore biological systems. nevertheless, proficient expertise is essential for handling and interpreting the data. in this paper, we present scx, an r package built on the shiny framework that streamlines the analysis, exploration, and visualization of single-cell experiments. with an interactive graphic interface, implemented as a web application, scx provides easy access to key scrnaseq analyses, including marker identification, gene expression profiling, and differential gene expression analysis. additionally, scx seamlessly integrates with commonly used single-cell seurat and singlecellexperiment r objects, resulting in efficient processing and visualization of varied datasets. overall, scx serves as a valuable and user-friendly tool for effortless exploration and sharing of single-cell data, simplifying some of the complexities inherent in scrnaseq analysis.',\n",
       "  'categories': 'q-bio.gn',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-31',\n",
       "  'updated': '2024-03-07',\n",
       "  'authors': ['tomás vega waichman',\n",
       "   'm. luz vercesi',\n",
       "   'ariel a. berardino',\n",
       "   'maximiliano s. beckel',\n",
       "   'damiana giacomini',\n",
       "   'natalí b. rasetto',\n",
       "   'magalí herrero',\n",
       "   'daniela j. di bella',\n",
       "   'paola arlotta',\n",
       "   'alejandro f. schinder',\n",
       "   'ariel chernomoretz'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2311.00012'},\n",
       " {'title': 'limits on the accuracy of contact inhibition of locomotion',\n",
       "  'id': '2311.00085',\n",
       "  'abstract': \"cells that collide with each other repolarize away from contact, in a process called contact inhibition of locomotion (cil), which is necessary for correct development of the embryo. cil can occur even when cells make a micron-scale contact with a neighbor - much smaller than their size. how precisely can a cell sense cell-cell contact and repolarize in the correct direction? what factors control whether a cell recognizes it has contacted a neighbor? we propose a theoretical model for the limits of cil where cells recognize the presence of another cell by binding the protein ephrin with the eph receptor. this recognition is made difficult by the presence of interfering ligands that bind nonspecifically. both theoretical predictions and simulation results show that it becomes more difficult to sense cell-cell contact when it is difficult to distinguish ephrin from the interfering ligands, or when there are more interfering ligands, or when the contact width decreases. however, the error of estimating contact position remains almost constant when the contact width changes. this happens because the cell gains spatial information largely from the boundaries of cell-cell contact. we study using statistical decision theory the likelihood of a false positive cil event in the absence of cell-cell contact, and the likelihood of a false negative where cil does not occur when another cell is present. our results suggest that the cell is more likely to make incorrect decisions when the contact width is very small or so large that it nears the cell's perimeter. however, in general, we find that cells have the ability to make reasonably reliable cil decisions even for very narrow (micron-scale) contacts, even if the concentration of interfering ligands is ten times that of the correct ligands.\",\n",
       "  'categories': 'q-bio.cb physics.bio-ph',\n",
       "  'doi': '10.1103/physreve.109.054408',\n",
       "  'created': '2023-10-31',\n",
       "  'updated': '',\n",
       "  'authors': ['wei wang', 'brian a. camley'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2311.00085'},\n",
       " {'title': 'extracting the multiscale causal backbone of brain dynamics',\n",
       "  'id': '2311.00118',\n",
       "  'abstract': 'the bulk of the research effort on brain connectivity revolves around statistical associations among brain regions, which do not directly relate to the causal mechanisms governing brain dynamics. here we propose the multiscale causal backbone (mcb) of brain dynamics, shared by a set of individuals across multiple temporal scales, and devise a principled methodology to extract it.   our approach leverages recent advances in multiscale causal structure learning and optimizes the trade-off between the model fit and its complexity. empirical assessment on synthetic data shows the superiority of our methodology over a baseline based on canonical functional connectivity networks. when applied to resting-state fmri data, we find sparse mcbs for both the left and right brain hemispheres. thanks to its multiscale nature, our approach shows that at low-frequency bands, causal dynamics are driven by brain regions associated with high-level cognitive functions; at higher frequencies instead, nodes related to sensory processing play a crucial role. finally, our analysis of individual multiscale causal structures confirms the existence of a causal fingerprint of brain connectivity, thus supporting the existing extensive research in brain connectivity fingerprinting from a causal perspective.',\n",
       "  'categories': 'cs.lg q-bio.nc stat.ap stat.me stat.ml',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-31',\n",
       "  'updated': '2024-03-19',\n",
       "  'authors': [\"gabriele d'acunto\",\n",
       "   'francesco bonchi',\n",
       "   'gianmarco de francisci morales',\n",
       "   'giovanni petri'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2311.00118'},\n",
       " {'title': 'neuroformer: multimodal and multitask generative pretraining for brain   data',\n",
       "  'id': '2311.00136',\n",
       "  'abstract': \"state-of-the-art systems neuroscience experiments yield large-scale multimodal data, and these data sets require new tools for analysis. inspired by the success of large pretrained models in vision and language domains, we reframe the analysis of large-scale, cellular-resolution neuronal spiking data into an autoregressive spatiotemporal generation problem. neuroformer is a multimodal, multitask generative pretrained transformer (gpt) model that is specifically designed to handle the intricacies of data in systems neuroscience. it scales linearly with feature size, can process an arbitrary number of modalities, and is adaptable to downstream tasks, such as predicting behavior. we first trained neuroformer on simulated datasets, and found that it both accurately predicted simulated neuronal circuit activity, and also intrinsically inferred the underlying neural circuit connectivity, including direction. when pretrained to decode neural responses, the model predicted the behavior of a mouse with only few-shot fine-tuning, suggesting that the model begins learning how to do so directly from the neural representations themselves, without any explicit supervision. we used an ablation study to show that joint training on neuronal responses and behavior boosted performance, highlighting the model's ability to associate behavioral and neural representations in an unsupervised manner. these findings show that neuroformer can analyze neural datasets and their emergent properties, informing the development of models and hypotheses associated with the brain.\",\n",
       "  'categories': 'q-bio.nc cs.lg cs.ne',\n",
       "  'doi': '',\n",
       "  'created': '2023-10-31',\n",
       "  'updated': '2024-03-15',\n",
       "  'authors': ['antonis antoniades',\n",
       "   'yiyi yu',\n",
       "   'joseph canzano',\n",
       "   'william wang',\n",
       "   'spencer lavere smith'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2311.00136'},\n",
       " {'title': 'an open-source, three-dimensional growth model of the mandible',\n",
       "  'id': '2311.00392',\n",
       "  'abstract': 'the available reference data for the mandible and mandibular growth consists primarily of two-dimensional linear or angular measurements. the aim of this study was to create the first open-source, three-dimensional statistical shape model of the mandible that spans the complete growth period. computed tomography scans of 678 mandibles from children and young adults between 0 and 22 years old were included in the model. the mandibles were segmented using a semi-automatic or automatic (artificial intelligence-based) segmentation method. point correspondence among the samples was achieved by rigid registration, followed by non-rigid registration of a symmetrical template onto each sample. the registration process was validated with adequate results. principal component analysis was used to gain insight in the variation within the dataset and to investigate age-related changes and sexual dimorphism. the presented growth model is accessible globally and free-of-charge for scientists, physicians and forensic investigators for any kind of purpose deemed suitable. the versatility of the model opens up new possibilities in the fields of oral and maxillofacial surgery, forensic sciences or biological anthropology. in clinical settings, the model may aid diagnostic decision-making, treatment planning and treatment evaluation.',\n",
       "  'categories': 'q-bio.to',\n",
       "  'doi': '10.1016/j.compbiomed.2024.108455',\n",
       "  'created': '2023-11-01',\n",
       "  'updated': '2023-11-07',\n",
       "  'authors': ['cornelis klop',\n",
       "   'ruud schreurs',\n",
       "   'guido a de jong',\n",
       "   'edwin tm klinkenberg',\n",
       "   'valeria vespasiano',\n",
       "   'naomi l rood',\n",
       "   'valerie g niehe',\n",
       "   'vidija soerdjbalie-maikoe',\n",
       "   'alexia van goethem',\n",
       "   'bernadette s de bakker',\n",
       "   'thomas jj maal',\n",
       "   'jitske w nolte',\n",
       "   'alfred g becking'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2311.00392'},\n",
       " {'title': 'multiple possible patterns can emerge from virus-immune coevolution',\n",
       "  'id': '2311.00417',\n",
       "  'abstract': 'the adaptive immune system engages in an arms race with evolving viruses, trying to generate new responses to viral strains that continually move away from the set of variants that have already elicited a functional immune response. in previous work, it has been argued that this dynamical process can lead to a propagating pulse of ever-changing viral population and concomitant immune response. here, we introduce a new stochastic model of viral-host coevolution and a deterministic approximation thereof with which to study these dynamics. we show that there is indeed a possible pulse solution, but for a large host population size, the pulse becomes unstable to the generation of new infections in its wake, leading to an extended endemic infection pattern. this time-dependent endemic pattern eventually reaches a fluctuating steady-state when the infected population size reaches a large enough fraction of the total number of hosts.',\n",
       "  'categories': 'q-bio.pe cond-mat.stat-mech nlin.ps',\n",
       "  'doi': '',\n",
       "  'created': '2023-11-01',\n",
       "  'updated': '2024-06-01',\n",
       "  'authors': ['david a. kessler', 'herbert levine'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2311.00417'},\n",
       " {'title': 'statistics of c. elegans turning behavior reveals optimality under   biasing constraints',\n",
       "  'id': '2311.00616',\n",
       "  'abstract': 'animal locomotion is often subject to constraints arising from anatomical/physiological asymmetries. we use the nematode c.~elegans as a minimal model system to ask whether such constraints might shape locomotion patterns optimized during evolution. we focus on turning behaviours in two contrasting environmental contexts: (i) random exploration in the absence of strong stimuli and (ii) acute avoidance (escape) navigation upon encountering a strong aversive stimulus. we characterise the full repertoire of reorientation behaviours, including gradual reorientations and various posturally distinct sharp turns. during exploration, our measurements and theoretical modelling indicate that orientation fluctuations on short timescales are, on average, optimized to compensate the constraining gradual turn bias on long timescales. during escape, our data suggests that the reorientation is controlled not by an analog logic of continuous turn-amplitude modulations, but rather through the digital logic of selecting discrete turn types, leading to a symmetric escape performance despite strongly asymmetric turning biases.',\n",
       "  'categories': 'q-bio.nc',\n",
       "  'doi': '',\n",
       "  'created': '2023-11-01',\n",
       "  'updated': '2024-01-15',\n",
       "  'authors': ['w. mathijs rozemuller',\n",
       "   'steffen werner',\n",
       "   'antonio carlos costa',\n",
       "   \"liam o'shaughnessy\",\n",
       "   'greg j. stephens',\n",
       "   'thomas s. shimizu'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2311.00616'},\n",
       " {'title': 'adaptive dynamics of diverging fitness optima',\n",
       "  'id': '2311.00829',\n",
       "  'abstract': 'we analyse a non-local parabolic integro-differential equation modelling the evolutionary dynamics of a phenotypically-structured population in a changing environment. such models arise in a variety of contexts from climate change to chemotherapy to the ageing body. the main novelty is that there are two locally optimal traits, each of which shifts at a possibly different linear velocity. we determine sufficient conditions to guarantee extinction or persistence of the population in terms of associated eigenvalue problems. when it does not go extinct, we analyse the solution in the long time, small mutation limits. if the optimas have equal shift velocities, the solution concentrates on a point set of \"lagged optima\" which are strictly behind the true shifting optima. if the shift velocities are different, we determine that the solution in fact concentrates as a dirac delta function on the positive lagged optimum with maximum lagged fitness, which depends on the true optimum and the rate of shift. our results imply that for populations undergoing competition in temporally changing environments, both the true optimal fitness and the required rate of adaptation for each of the diverging optimal traits contribute to the eventual dominance of one trait.',\n",
       "  'categories': 'math.ap q-bio.pe',\n",
       "  'doi': '',\n",
       "  'created': '2023-11-01',\n",
       "  'updated': '2024-07-15',\n",
       "  'authors': ['manh hong duong', 'fabian spill', 'blaine van rensburg'],\n",
       "  'affiliation': [],\n",
       "  'url': 'https://arxiv.org/abs/2311.00829'},\n",
       " ...]"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# note that output is basically metadata\n",
    "output"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# # code to store meta data in case we need it down the line \n",
    "# with open('arxiv_metadata.csv', 'w', newline='') as file:\n",
    "    \n",
    "#     fieldnames = output[0].keys()\n",
    "    \n",
    "#     # Create CSV writer\n",
    "#     writer = csv.DictWriter(file, fieldnames=fieldnames)\n",
    "    \n",
    "#     # Write headers\n",
    "#     writer.writeheader()\n",
    "    \n",
    "#     # Write data rows\n",
    "#     writer.writerows(output)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# code to webscrape research papers using the metadata \n",
    "\n",
    "def fetch_paper_data(url):\n",
    "    time.sleep(3)\n",
    "    \n",
    "    headers = {\n",
    "        'User-Agent': 'Educational Bot (na648@drexel.edu)',\n",
    "        'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9'\n",
    "    }\n",
    "    \n",
    "    try:\n",
    "        #SENDING API REQUEST FOR THE URL\n",
    "        response = requests.get(url, headers=headers)\n",
    "        response.raise_for_status()\n",
    "        \n",
    "        soup = BeautifulSoup(response.text, 'html.parser')\n",
    "        \n",
    "        title = soup.find('h1', class_='title').text.strip()\n",
    "        abstract = soup.find('blockquote', class_='abstract').text.strip()\n",
    "        authors = [author.text.strip() for author in soup.find_all('div', class_='authors')]\n",
    "        \n",
    "        try:\n",
    "            #SEARCHING FOR THE PDF IN THE WEBSITE OF THE LINK\n",
    "            href = soup.find('a', class_='abs-button download-pdf')['href']\n",
    "            base = 'https://arxiv.org'\n",
    "            link = base + href\n",
    "            response = requests.get(link)\n",
    "            pdf_file = io.BytesIO(response.content)\n",
    "\n",
    "            try:\n",
    "                #reading the contents of the pdf\n",
    "                pdf_reader = PyPDF2.PdfReader(pdf_file)\n",
    "                text = \"\"\n",
    "                for page in pdf_reader.pages:\n",
    "                    text += page.extract_text()\n",
    "            except Exception as e:\n",
    "                print(f\"Error extracting PDF text from {url}: {e}\")\n",
    "                text = \"\"\n",
    "        except:\n",
    "            text = ''\n",
    "        \n",
    "        #Returning the extracted contents\n",
    "        return {\n",
    "            'title': title,\n",
    "            'abstract': abstract,\n",
    "            'authors': authors,\n",
    "            'body': text,\n",
    "            'url': url\n",
    "        }\n",
    "        \n",
    "    except requests.exceptions.RequestException as e:\n",
    "        print(f\"Error fetching {url}: {e}\")\n",
    "        return None\n",
    "\n",
    "def process_papers_incrementally(output, csv_path=None):\n",
    "    #PREPROCESSING THE DOWNLOADED DATA TO GENERATE FINAL DATASET\n",
    "    if csv_path is None:\n",
    "        csv_path = os.path.join(os.path.expanduser('~'), 'Downloads', 'research_papers.csv')\n",
    "    \n",
    "    # Track processed URLs to avoid duplicates\n",
    "    processed_urls = set()\n",
    "    \n",
    "    # If file exists, read processed URLs\n",
    "    if os.path.exists(csv_path):\n",
    "        try:\n",
    "            with open(csv_path, 'r', encoding='utf-8') as csvfile:\n",
    "                reader = csv.DictReader(csvfile)\n",
    "                processed_urls = {row['url'] for row in reader}\n",
    "        except Exception as e:\n",
    "            print(f\"Error reading existing CSV: {e}\")\n",
    "    \n",
    "    results = []\n",
    "    file_exists = os.path.exists(csv_path)\n",
    "    \n",
    "    for paper in output:\n",
    "        url = paper['url']\n",
    "        if url in processed_urls:\n",
    "            print(f\"Skipping already processed URL: {url}\")\n",
    "            continue\n",
    "            \n",
    "        print(f\"Processing: {url}\")\n",
    "        data = fetch_paper_data(url)\n",
    "        \n",
    "        if data:\n",
    "            results.append(data)\n",
    "            \n",
    "            try:\n",
    "                mode = 'a' if file_exists else 'w'\n",
    "                with open(csv_path, mode, newline='', encoding='utf-8') as csvfile:\n",
    "                    fieldnames = data.keys()\n",
    "                    writer = csv.DictWriter(csvfile, fieldnames=fieldnames)\n",
    "                    \n",
    "                    if not file_exists:  # Write header only for new file\n",
    "                        writer.writeheader()\n",
    "                        file_exists = True\n",
    "                    \n",
    "                    writer.writerow(data)\n",
    "                print(f\"Saved data for: {url}\")\n",
    "                processed_urls.add(url)\n",
    "            except Exception as e:\n",
    "                print(f\"Error saving data for {url}: {e}\")\n",
    "    \n",
    "    return results\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing: https://arxiv.org/abs/0905.3690\n",
      "Saved data for: https://arxiv.org/abs/0905.3690\n",
      "Processing: https://arxiv.org/abs/1103.2273\n",
      "Saved data for: https://arxiv.org/abs/1103.2273\n",
      "Processing: https://arxiv.org/abs/1201.1641\n",
      "Saved data for: https://arxiv.org/abs/1201.1641\n",
      "Processing: https://arxiv.org/abs/1204.5203\n",
      "Saved data for: https://arxiv.org/abs/1204.5203\n",
      "Processing: https://arxiv.org/abs/1206.3472\n",
      "Saved data for: https://arxiv.org/abs/1206.3472\n",
      "Processing: https://arxiv.org/abs/1210.7091\n",
      "Saved data for: https://arxiv.org/abs/1210.7091\n",
      "Processing: https://arxiv.org/abs/1211.7317\n",
      "Saved data for: https://arxiv.org/abs/1211.7317\n",
      "Processing: https://arxiv.org/abs/1307.4296\n",
      "Saved data for: https://arxiv.org/abs/1307.4296\n",
      "Processing: https://arxiv.org/abs/1403.7660\n",
      "Saved data for: https://arxiv.org/abs/1403.7660\n",
      "Processing: https://arxiv.org/abs/1407.4992\n",
      "Saved data for: https://arxiv.org/abs/1407.4992\n",
      "Processing: https://arxiv.org/abs/1409.7291\n",
      "Saved data for: https://arxiv.org/abs/1409.7291\n",
      "Processing: https://arxiv.org/abs/1410.5851\n",
      "Saved data for: https://arxiv.org/abs/1410.5851\n",
      "Processing: https://arxiv.org/abs/1410.6049\n",
      "Saved data for: https://arxiv.org/abs/1410.6049\n",
      "Processing: https://arxiv.org/abs/1410.6455\n",
      "Saved data for: https://arxiv.org/abs/1410.6455\n",
      "Processing: https://arxiv.org/abs/1411.2547\n",
      "Saved data for: https://arxiv.org/abs/1411.2547\n",
      "Processing: https://arxiv.org/abs/1411.2548\n",
      "Saved data for: https://arxiv.org/abs/1411.2548\n",
      "Processing: https://arxiv.org/abs/1411.2549\n",
      "Saved data for: https://arxiv.org/abs/1411.2549\n",
      "Processing: https://arxiv.org/abs/1411.2668\n",
      "Saved data for: https://arxiv.org/abs/1411.2668\n",
      "Processing: https://arxiv.org/abs/1503.02904\n",
      "Saved data for: https://arxiv.org/abs/1503.02904\n",
      "Processing: https://arxiv.org/abs/1503.07215\n",
      "Saved data for: https://arxiv.org/abs/1503.07215\n",
      "Processing: https://arxiv.org/abs/1503.08992\n",
      "Saved data for: https://arxiv.org/abs/1503.08992\n",
      "Processing: https://arxiv.org/abs/1507.02716\n",
      "Saved data for: https://arxiv.org/abs/1507.02716\n",
      "Processing: https://arxiv.org/abs/1508.03026\n",
      "Saved data for: https://arxiv.org/abs/1508.03026\n",
      "Processing: https://arxiv.org/abs/1509.06123\n",
      "Saved data for: https://arxiv.org/abs/1509.06123\n",
      "Processing: https://arxiv.org/abs/1510.03964\n",
      "Saved data for: https://arxiv.org/abs/1510.03964\n",
      "Processing: https://arxiv.org/abs/1512.01922\n",
      "Saved data for: https://arxiv.org/abs/1512.01922\n",
      "Processing: https://arxiv.org/abs/1512.08954\n",
      "Saved data for: https://arxiv.org/abs/1512.08954\n",
      "Processing: https://arxiv.org/abs/1601.07810\n",
      "Saved data for: https://arxiv.org/abs/1601.07810\n",
      "Processing: https://arxiv.org/abs/1602.03493\n",
      "Saved data for: https://arxiv.org/abs/1602.03493\n",
      "Processing: https://arxiv.org/abs/1603.03141\n",
      "Saved data for: https://arxiv.org/abs/1603.03141\n",
      "Processing: https://arxiv.org/abs/1607.04474\n",
      "Saved data for: https://arxiv.org/abs/1607.04474\n",
      "Processing: https://arxiv.org/abs/1607.05386\n",
      "Saved data for: https://arxiv.org/abs/1607.05386\n",
      "Processing: https://arxiv.org/abs/1608.00535\n",
      "Saved data for: https://arxiv.org/abs/1608.00535\n",
      "Processing: https://arxiv.org/abs/1609.04141\n",
      "Saved data for: https://arxiv.org/abs/1609.04141\n",
      "Processing: https://arxiv.org/abs/1610.01993\n",
      "Saved data for: https://arxiv.org/abs/1610.01993\n",
      "Processing: https://arxiv.org/abs/1610.09637\n",
      "Saved data for: https://arxiv.org/abs/1610.09637\n",
      "Processing: https://arxiv.org/abs/1612.00098\n",
      "Saved data for: https://arxiv.org/abs/1612.00098\n",
      "Processing: https://arxiv.org/abs/1612.04695\n",
      "Saved data for: https://arxiv.org/abs/1612.04695\n",
      "Processing: https://arxiv.org/abs/1612.08084\n",
      "Saved data for: https://arxiv.org/abs/1612.08084\n",
      "Processing: https://arxiv.org/abs/1612.09379\n",
      "Saved data for: https://arxiv.org/abs/1612.09379\n",
      "Processing: https://arxiv.org/abs/1703.04184\n",
      "Saved data for: https://arxiv.org/abs/1703.04184\n",
      "Processing: https://arxiv.org/abs/1704.03399\n",
      "Saved data for: https://arxiv.org/abs/1704.03399\n",
      "Processing: https://arxiv.org/abs/1707.02110\n",
      "Saved data for: https://arxiv.org/abs/1707.02110\n",
      "Processing: https://arxiv.org/abs/1709.10442\n",
      "Saved data for: https://arxiv.org/abs/1709.10442\n",
      "Processing: https://arxiv.org/abs/1711.03575\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "unknown widths : \n",
      "[0, IndirectObject(2289, 0, 4596143296)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(2292, 0, 4596143296)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(2294, 0, 4596143296)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(2296, 0, 4596143296)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(2298, 0, 4596143296)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(2300, 0, 4596143296)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(2302, 0, 4596143296)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(2304, 0, 4596143296)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(2306, 0, 4596143296)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(2308, 0, 4596143296)]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved data for: https://arxiv.org/abs/1711.03575\n",
      "Processing: https://arxiv.org/abs/1712.04633\n",
      "Saved data for: https://arxiv.org/abs/1712.04633\n",
      "Processing: https://arxiv.org/abs/1801.04057\n",
      "Saved data for: https://arxiv.org/abs/1801.04057\n",
      "Processing: https://arxiv.org/abs/1802.07480\n",
      "Saved data for: https://arxiv.org/abs/1802.07480\n",
      "Processing: https://arxiv.org/abs/1803.05136\n",
      "Saved data for: https://arxiv.org/abs/1803.05136\n",
      "Processing: https://arxiv.org/abs/1803.05275\n",
      "Saved data for: https://arxiv.org/abs/1803.05275\n",
      "Processing: https://arxiv.org/abs/1805.10235\n",
      "Saved data for: https://arxiv.org/abs/1805.10235\n",
      "Processing: https://arxiv.org/abs/1807.09932\n",
      "Saved data for: https://arxiv.org/abs/1807.09932\n",
      "Processing: https://arxiv.org/abs/1808.03386\n",
      "Saved data for: https://arxiv.org/abs/1808.03386\n",
      "Processing: https://arxiv.org/abs/1808.10806\n",
      "Saved data for: https://arxiv.org/abs/1808.10806\n",
      "Processing: https://arxiv.org/abs/1809.03930\n",
      "Saved data for: https://arxiv.org/abs/1809.03930\n",
      "Processing: https://arxiv.org/abs/1809.10990\n",
      "Saved data for: https://arxiv.org/abs/1809.10990\n",
      "Processing: https://arxiv.org/abs/1810.05848\n",
      "Saved data for: https://arxiv.org/abs/1810.05848\n",
      "Processing: https://arxiv.org/abs/1810.07215\n",
      "Saved data for: https://arxiv.org/abs/1810.07215\n",
      "Processing: https://arxiv.org/abs/1811.11443\n",
      "Saved data for: https://arxiv.org/abs/1811.11443\n",
      "Processing: https://arxiv.org/abs/1812.11850\n",
      "Saved data for: https://arxiv.org/abs/1812.11850\n",
      "Processing: https://arxiv.org/abs/1901.02314\n",
      "Saved data for: https://arxiv.org/abs/1901.02314\n",
      "Processing: https://arxiv.org/abs/1902.05515\n",
      "Saved data for: https://arxiv.org/abs/1902.05515\n",
      "Processing: https://arxiv.org/abs/1902.10353\n",
      "Saved data for: https://arxiv.org/abs/1902.10353\n",
      "Processing: https://arxiv.org/abs/1903.06036\n",
      "Saved data for: https://arxiv.org/abs/1903.06036\n",
      "Processing: https://arxiv.org/abs/1903.06375\n",
      "Saved data for: https://arxiv.org/abs/1903.06375\n",
      "Processing: https://arxiv.org/abs/1903.11448\n",
      "Saved data for: https://arxiv.org/abs/1903.11448\n",
      "Processing: https://arxiv.org/abs/1904.03236\n",
      "Saved data for: https://arxiv.org/abs/1904.03236\n",
      "Processing: https://arxiv.org/abs/1904.04579\n",
      "Saved data for: https://arxiv.org/abs/1904.04579\n",
      "Processing: https://arxiv.org/abs/1904.05675\n",
      "Saved data for: https://arxiv.org/abs/1904.05675\n",
      "Processing: https://arxiv.org/abs/1905.07818\n",
      "Saved data for: https://arxiv.org/abs/1905.07818\n",
      "Processing: https://arxiv.org/abs/1906.08246\n",
      "Saved data for: https://arxiv.org/abs/1906.08246\n",
      "Processing: https://arxiv.org/abs/1906.08612\n",
      "Saved data for: https://arxiv.org/abs/1906.08612\n",
      "Processing: https://arxiv.org/abs/1906.08804\n",
      "Saved data for: https://arxiv.org/abs/1906.08804\n",
      "Processing: https://arxiv.org/abs/1906.09465\n",
      "Saved data for: https://arxiv.org/abs/1906.09465\n",
      "Processing: https://arxiv.org/abs/1906.10819\n",
      "Saved data for: https://arxiv.org/abs/1906.10819\n",
      "Processing: https://arxiv.org/abs/1907.00070\n",
      "Saved data for: https://arxiv.org/abs/1907.00070\n",
      "Processing: https://arxiv.org/abs/1907.00816\n",
      "Saved data for: https://arxiv.org/abs/1907.00816\n",
      "Processing: https://arxiv.org/abs/1908.04206\n",
      "Saved data for: https://arxiv.org/abs/1908.04206\n",
      "Processing: https://arxiv.org/abs/1908.05923\n",
      "Saved data for: https://arxiv.org/abs/1908.05923\n",
      "Processing: https://arxiv.org/abs/1908.07092\n",
      "Saved data for: https://arxiv.org/abs/1908.07092\n",
      "Processing: https://arxiv.org/abs/1909.05006\n",
      "Saved data for: https://arxiv.org/abs/1909.05006\n",
      "Processing: https://arxiv.org/abs/1909.09769\n",
      "Saved data for: https://arxiv.org/abs/1909.09769\n",
      "Processing: https://arxiv.org/abs/1910.03263\n",
      "Saved data for: https://arxiv.org/abs/1910.03263\n",
      "Processing: https://arxiv.org/abs/1910.05421\n",
      "Saved data for: https://arxiv.org/abs/1910.05421\n",
      "Processing: https://arxiv.org/abs/1912.00518\n",
      "Saved data for: https://arxiv.org/abs/1912.00518\n",
      "Processing: https://arxiv.org/abs/1912.08735\n",
      "Saved data for: https://arxiv.org/abs/1912.08735\n",
      "Processing: https://arxiv.org/abs/1912.09205\n",
      "Saved data for: https://arxiv.org/abs/1912.09205\n",
      "Processing: https://arxiv.org/abs/1912.10641\n",
      "Saved data for: https://arxiv.org/abs/1912.10641\n",
      "Processing: https://arxiv.org/abs/1912.13190\n",
      "Saved data for: https://arxiv.org/abs/1912.13190\n",
      "Processing: https://arxiv.org/abs/2001.04794\n",
      "Saved data for: https://arxiv.org/abs/2001.04794\n",
      "Processing: https://arxiv.org/abs/2001.06530\n",
      "Saved data for: https://arxiv.org/abs/2001.06530\n",
      "Processing: https://arxiv.org/abs/2001.10788\n",
      "Saved data for: https://arxiv.org/abs/2001.10788\n",
      "Processing: https://arxiv.org/abs/2002.06816\n",
      "Saved data for: https://arxiv.org/abs/2002.06816\n",
      "Processing: https://arxiv.org/abs/2002.09626\n",
      "Saved data for: https://arxiv.org/abs/2002.09626\n",
      "Processing: https://arxiv.org/abs/2003.04601\n",
      "Saved data for: https://arxiv.org/abs/2003.04601\n",
      "Processing: https://arxiv.org/abs/2003.08809\n",
      "Saved data for: https://arxiv.org/abs/2003.08809\n",
      "Processing: https://arxiv.org/abs/2003.12493\n",
      "Saved data for: https://arxiv.org/abs/2003.12493\n",
      "Processing: https://arxiv.org/abs/2004.01399\n",
      "Saved data for: https://arxiv.org/abs/2004.01399\n",
      "Processing: https://arxiv.org/abs/2004.01562\n",
      "Saved data for: https://arxiv.org/abs/2004.01562\n",
      "Processing: https://arxiv.org/abs/2004.04229\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "incorrect startxref pointer(1)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved data for: https://arxiv.org/abs/2004.04229\n",
      "Processing: https://arxiv.org/abs/2004.08807\n",
      "Saved data for: https://arxiv.org/abs/2004.08807\n",
      "Processing: https://arxiv.org/abs/2004.10177\n",
      "Saved data for: https://arxiv.org/abs/2004.10177\n",
      "Processing: https://arxiv.org/abs/2005.00498\n",
      "Saved data for: https://arxiv.org/abs/2005.00498\n",
      "Processing: https://arxiv.org/abs/2005.05249\n",
      "Saved data for: https://arxiv.org/abs/2005.05249\n",
      "Processing: https://arxiv.org/abs/2005.06201\n",
      "Saved data for: https://arxiv.org/abs/2005.06201\n",
      "Processing: https://arxiv.org/abs/2006.04519\n",
      "Saved data for: https://arxiv.org/abs/2006.04519\n",
      "Processing: https://arxiv.org/abs/2006.06530\n",
      "Saved data for: https://arxiv.org/abs/2006.06530\n",
      "Processing: https://arxiv.org/abs/2006.11330\n",
      "Saved data for: https://arxiv.org/abs/2006.11330\n",
      "Processing: https://arxiv.org/abs/2006.11975\n",
      "Saved data for: https://arxiv.org/abs/2006.11975\n",
      "Processing: https://arxiv.org/abs/2007.01508\n",
      "Saved data for: https://arxiv.org/abs/2007.01508\n",
      "Processing: https://arxiv.org/abs/2008.00520\n",
      "Saved data for: https://arxiv.org/abs/2008.00520\n",
      "Processing: https://arxiv.org/abs/2008.09574\n",
      "Saved data for: https://arxiv.org/abs/2008.09574\n",
      "Processing: https://arxiv.org/abs/2008.11674\n",
      "Saved data for: https://arxiv.org/abs/2008.11674\n",
      "Processing: https://arxiv.org/abs/2009.01216\n",
      "Saved data for: https://arxiv.org/abs/2009.01216\n",
      "Processing: https://arxiv.org/abs/2009.03238\n",
      "Saved data for: https://arxiv.org/abs/2009.03238\n",
      "Processing: https://arxiv.org/abs/2009.11211\n",
      "Saved data for: https://arxiv.org/abs/2009.11211\n",
      "Processing: https://arxiv.org/abs/2010.03420\n",
      "Saved data for: https://arxiv.org/abs/2010.03420\n",
      "Processing: https://arxiv.org/abs/2010.09898\n",
      "Saved data for: https://arxiv.org/abs/2010.09898\n",
      "Processing: https://arxiv.org/abs/2010.11477\n",
      "Saved data for: https://arxiv.org/abs/2010.11477\n",
      "Processing: https://arxiv.org/abs/2011.02444\n",
      "Saved data for: https://arxiv.org/abs/2011.02444\n",
      "Processing: https://arxiv.org/abs/2011.04415\n",
      "Saved data for: https://arxiv.org/abs/2011.04415\n",
      "Processing: https://arxiv.org/abs/2011.04892\n",
      "Saved data for: https://arxiv.org/abs/2011.04892\n",
      "Processing: https://arxiv.org/abs/2011.06280\n",
      "Saved data for: https://arxiv.org/abs/2011.06280\n",
      "Processing: https://arxiv.org/abs/2011.07539\n",
      "Saved data for: https://arxiv.org/abs/2011.07539\n",
      "Processing: https://arxiv.org/abs/2011.08069\n",
      "Saved data for: https://arxiv.org/abs/2011.08069\n",
      "Processing: https://arxiv.org/abs/2011.09955\n",
      "Saved data for: https://arxiv.org/abs/2011.09955\n",
      "Processing: https://arxiv.org/abs/2012.02728\n",
      "Saved data for: https://arxiv.org/abs/2012.02728\n",
      "Processing: https://arxiv.org/abs/2012.04132\n",
      "Saved data for: https://arxiv.org/abs/2012.04132\n",
      "Processing: https://arxiv.org/abs/2012.04171\n",
      "Saved data for: https://arxiv.org/abs/2012.04171\n",
      "Processing: https://arxiv.org/abs/2012.06608\n",
      "Saved data for: https://arxiv.org/abs/2012.06608\n",
      "Processing: https://arxiv.org/abs/2012.07640\n",
      "Saved data for: https://arxiv.org/abs/2012.07640\n",
      "Processing: https://arxiv.org/abs/2012.07691\n",
      "Saved data for: https://arxiv.org/abs/2012.07691\n",
      "Processing: https://arxiv.org/abs/2012.10239\n",
      "Saved data for: https://arxiv.org/abs/2012.10239\n",
      "Processing: https://arxiv.org/abs/2012.10785\n",
      "Saved data for: https://arxiv.org/abs/2012.10785\n",
      "Processing: https://arxiv.org/abs/2012.11240\n",
      "Saved data for: https://arxiv.org/abs/2012.11240\n",
      "Processing: https://arxiv.org/abs/2101.00773\n",
      "Saved data for: https://arxiv.org/abs/2101.00773\n",
      "Processing: https://arxiv.org/abs/2101.07082\n",
      "Saved data for: https://arxiv.org/abs/2101.07082\n",
      "Processing: https://arxiv.org/abs/2101.09384\n",
      "Saved data for: https://arxiv.org/abs/2101.09384\n",
      "Processing: https://arxiv.org/abs/2101.11526\n",
      "Saved data for: https://arxiv.org/abs/2101.11526\n",
      "Processing: https://arxiv.org/abs/2102.02649\n",
      "Saved data for: https://arxiv.org/abs/2102.02649\n",
      "Processing: https://arxiv.org/abs/2102.03438\n",
      "Saved data for: https://arxiv.org/abs/2102.03438\n",
      "Processing: https://arxiv.org/abs/2102.03469\n",
      "Saved data for: https://arxiv.org/abs/2102.03469\n",
      "Processing: https://arxiv.org/abs/2102.03676\n",
      "Saved data for: https://arxiv.org/abs/2102.03676\n",
      "Processing: https://arxiv.org/abs/2102.04502\n",
      "Saved data for: https://arxiv.org/abs/2102.04502\n",
      "Processing: https://arxiv.org/abs/2102.06490\n",
      "Saved data for: https://arxiv.org/abs/2102.06490\n",
      "Processing: https://arxiv.org/abs/2102.06698\n",
      "Saved data for: https://arxiv.org/abs/2102.06698\n",
      "Processing: https://arxiv.org/abs/2102.08468\n",
      "Saved data for: https://arxiv.org/abs/2102.08468\n",
      "Processing: https://arxiv.org/abs/2103.03864\n",
      "Saved data for: https://arxiv.org/abs/2103.03864\n",
      "Processing: https://arxiv.org/abs/2103.05753\n",
      "Saved data for: https://arxiv.org/abs/2103.05753\n",
      "Processing: https://arxiv.org/abs/2103.13860\n",
      "Saved data for: https://arxiv.org/abs/2103.13860\n",
      "Processing: https://arxiv.org/abs/2103.14978\n",
      "Saved data for: https://arxiv.org/abs/2103.14978\n",
      "Processing: https://arxiv.org/abs/2104.01690\n",
      "Saved data for: https://arxiv.org/abs/2104.01690\n",
      "Processing: https://arxiv.org/abs/2104.04283\n",
      "Saved data for: https://arxiv.org/abs/2104.04283\n",
      "Processing: https://arxiv.org/abs/2105.02786\n",
      "Saved data for: https://arxiv.org/abs/2105.02786\n",
      "Processing: https://arxiv.org/abs/2106.02626\n",
      "Saved data for: https://arxiv.org/abs/2106.02626\n",
      "Processing: https://arxiv.org/abs/2106.03580\n",
      "Saved data for: https://arxiv.org/abs/2106.03580\n",
      "Processing: https://arxiv.org/abs/2106.04705\n",
      "Saved data for: https://arxiv.org/abs/2106.04705\n",
      "Processing: https://arxiv.org/abs/2106.05180\n",
      "Saved data for: https://arxiv.org/abs/2106.05180\n",
      "Processing: https://arxiv.org/abs/2106.06580\n",
      "Saved data for: https://arxiv.org/abs/2106.06580\n",
      "Processing: https://arxiv.org/abs/2106.07030\n",
      "Saved data for: https://arxiv.org/abs/2106.07030\n",
      "Processing: https://arxiv.org/abs/2106.07096\n",
      "Saved data for: https://arxiv.org/abs/2106.07096\n",
      "Processing: https://arxiv.org/abs/2106.11234\n",
      "Saved data for: https://arxiv.org/abs/2106.11234\n",
      "Processing: https://arxiv.org/abs/2106.13082\n",
      "Saved data for: https://arxiv.org/abs/2106.13082\n",
      "Processing: https://arxiv.org/abs/2106.15578\n",
      "Saved data for: https://arxiv.org/abs/2106.15578\n",
      "Processing: https://arxiv.org/abs/2107.00272\n",
      "Saved data for: https://arxiv.org/abs/2107.00272\n",
      "Processing: https://arxiv.org/abs/2107.03714\n",
      "Saved data for: https://arxiv.org/abs/2107.03714\n",
      "Processing: https://arxiv.org/abs/2107.12838\n",
      "Saved data for: https://arxiv.org/abs/2107.12838\n",
      "Processing: https://arxiv.org/abs/2108.00047\n",
      "Saved data for: https://arxiv.org/abs/2108.00047\n",
      "Processing: https://arxiv.org/abs/2108.04776\n",
      "Saved data for: https://arxiv.org/abs/2108.04776\n",
      "Processing: https://arxiv.org/abs/2108.11833\n",
      "Saved data for: https://arxiv.org/abs/2108.11833\n",
      "Processing: https://arxiv.org/abs/2108.13577\n",
      "Saved data for: https://arxiv.org/abs/2108.13577\n",
      "Processing: https://arxiv.org/abs/2108.13661\n",
      "Saved data for: https://arxiv.org/abs/2108.13661\n",
      "Processing: https://arxiv.org/abs/2109.00146\n",
      "Saved data for: https://arxiv.org/abs/2109.00146\n",
      "Processing: https://arxiv.org/abs/2109.00392\n",
      "Saved data for: https://arxiv.org/abs/2109.00392\n",
      "Processing: https://arxiv.org/abs/2109.01571\n",
      "Saved data for: https://arxiv.org/abs/2109.01571\n",
      "Processing: https://arxiv.org/abs/2109.03879\n",
      "Saved data for: https://arxiv.org/abs/2109.03879\n",
      "Processing: https://arxiv.org/abs/2109.07645\n",
      "Saved data for: https://arxiv.org/abs/2109.07645\n",
      "Processing: https://arxiv.org/abs/2109.13169\n",
      "Saved data for: https://arxiv.org/abs/2109.13169\n",
      "Processing: https://arxiv.org/abs/2110.01806\n",
      "Saved data for: https://arxiv.org/abs/2110.01806\n",
      "Processing: https://arxiv.org/abs/2110.05400\n",
      "Saved data for: https://arxiv.org/abs/2110.05400\n",
      "Processing: https://arxiv.org/abs/2110.07696\n",
      "Saved data for: https://arxiv.org/abs/2110.07696\n",
      "Processing: https://arxiv.org/abs/2110.10229\n",
      "Saved data for: https://arxiv.org/abs/2110.10229\n",
      "Processing: https://arxiv.org/abs/2110.12927\n",
      "Saved data for: https://arxiv.org/abs/2110.12927\n",
      "Processing: https://arxiv.org/abs/2110.13787\n",
      "Saved data for: https://arxiv.org/abs/2110.13787\n",
      "Processing: https://arxiv.org/abs/2110.14329\n",
      "Saved data for: https://arxiv.org/abs/2110.14329\n",
      "Processing: https://arxiv.org/abs/2110.14335\n",
      "Saved data for: https://arxiv.org/abs/2110.14335\n",
      "Processing: https://arxiv.org/abs/2111.00594\n",
      "Saved data for: https://arxiv.org/abs/2111.00594\n",
      "Processing: https://arxiv.org/abs/2111.02017\n",
      "Saved data for: https://arxiv.org/abs/2111.02017\n",
      "Processing: https://arxiv.org/abs/2111.02176\n",
      "Saved data for: https://arxiv.org/abs/2111.02176\n",
      "Processing: https://arxiv.org/abs/2111.02307\n",
      "Saved data for: https://arxiv.org/abs/2111.02307\n",
      "Processing: https://arxiv.org/abs/2111.03445\n",
      "Saved data for: https://arxiv.org/abs/2111.03445\n",
      "Processing: https://arxiv.org/abs/2111.03448\n",
      "Saved data for: https://arxiv.org/abs/2111.03448\n",
      "Processing: https://arxiv.org/abs/2111.04338\n",
      "Saved data for: https://arxiv.org/abs/2111.04338\n",
      "Processing: https://arxiv.org/abs/2111.04657\n",
      "Saved data for: https://arxiv.org/abs/2111.04657\n",
      "Processing: https://arxiv.org/abs/2111.04943\n",
      "Saved data for: https://arxiv.org/abs/2111.04943\n",
      "Processing: https://arxiv.org/abs/2111.07336\n",
      "Saved data for: https://arxiv.org/abs/2111.07336\n",
      "Processing: https://arxiv.org/abs/2111.08452\n",
      "Saved data for: https://arxiv.org/abs/2111.08452\n",
      "Processing: https://arxiv.org/abs/2111.09063\n",
      "Saved data for: https://arxiv.org/abs/2111.09063\n",
      "Processing: https://arxiv.org/abs/2111.10535\n",
      "Saved data for: https://arxiv.org/abs/2111.10535\n",
      "Processing: https://arxiv.org/abs/2111.10719\n",
      "Saved data for: https://arxiv.org/abs/2111.10719\n",
      "Processing: https://arxiv.org/abs/2111.11413\n",
      "Saved data for: https://arxiv.org/abs/2111.11413\n",
      "Processing: https://arxiv.org/abs/2111.12138\n",
      "Saved data for: https://arxiv.org/abs/2111.12138\n",
      "Processing: https://arxiv.org/abs/2111.12633\n",
      "Saved data for: https://arxiv.org/abs/2111.12633\n",
      "Processing: https://arxiv.org/abs/2111.13987\n",
      "Saved data for: https://arxiv.org/abs/2111.13987\n",
      "Processing: https://arxiv.org/abs/2111.14754\n",
      "Saved data for: https://arxiv.org/abs/2111.14754\n",
      "Processing: https://arxiv.org/abs/2112.02809\n",
      "Saved data for: https://arxiv.org/abs/2112.02809\n",
      "Processing: https://arxiv.org/abs/2112.04909\n",
      "Saved data for: https://arxiv.org/abs/2112.04909\n",
      "Processing: https://arxiv.org/abs/2112.05098\n",
      "Saved data for: https://arxiv.org/abs/2112.05098\n",
      "Processing: https://arxiv.org/abs/2112.06725\n",
      "Saved data for: https://arxiv.org/abs/2112.06725\n",
      "Processing: https://arxiv.org/abs/2112.10362\n",
      "Saved data for: https://arxiv.org/abs/2112.10362\n",
      "Processing: https://arxiv.org/abs/2112.10688\n",
      "Saved data for: https://arxiv.org/abs/2112.10688\n",
      "Processing: https://arxiv.org/abs/2112.11084\n",
      "Saved data for: https://arxiv.org/abs/2112.11084\n",
      "Processing: https://arxiv.org/abs/2112.12675\n",
      "Saved data for: https://arxiv.org/abs/2112.12675\n",
      "Processing: https://arxiv.org/abs/2112.14734\n",
      "Saved data for: https://arxiv.org/abs/2112.14734\n",
      "Processing: https://arxiv.org/abs/2112.14867\n",
      "Saved data for: https://arxiv.org/abs/2112.14867\n",
      "Processing: https://arxiv.org/abs/2112.14876\n",
      "Saved data for: https://arxiv.org/abs/2112.14876\n",
      "Processing: https://arxiv.org/abs/2201.00087\n",
      "Saved data for: https://arxiv.org/abs/2201.00087\n",
      "Processing: https://arxiv.org/abs/2201.03193\n",
      "Saved data for: https://arxiv.org/abs/2201.03193\n",
      "Processing: https://arxiv.org/abs/2201.06708\n",
      "Saved data for: https://arxiv.org/abs/2201.06708\n",
      "Processing: https://arxiv.org/abs/2201.09812\n",
      "Saved data for: https://arxiv.org/abs/2201.09812\n",
      "Processing: https://arxiv.org/abs/2201.13001\n",
      "Saved data for: https://arxiv.org/abs/2201.13001\n",
      "Processing: https://arxiv.org/abs/2201.13299\n",
      "Saved data for: https://arxiv.org/abs/2201.13299\n",
      "Processing: https://arxiv.org/abs/2202.00234\n",
      "Saved data for: https://arxiv.org/abs/2202.00234\n",
      "Processing: https://arxiv.org/abs/2202.02787\n",
      "Saved data for: https://arxiv.org/abs/2202.02787\n",
      "Processing: https://arxiv.org/abs/2202.06297\n",
      "Saved data for: https://arxiv.org/abs/2202.06297\n",
      "Processing: https://arxiv.org/abs/2202.07211\n",
      "Saved data for: https://arxiv.org/abs/2202.07211\n",
      "Processing: https://arxiv.org/abs/2202.07533\n",
      "Saved data for: https://arxiv.org/abs/2202.07533\n",
      "Processing: https://arxiv.org/abs/2202.10157\n",
      "Saved data for: https://arxiv.org/abs/2202.10157\n",
      "Processing: https://arxiv.org/abs/2202.12887\n",
      "Saved data for: https://arxiv.org/abs/2202.12887\n",
      "Processing: https://arxiv.org/abs/2202.13426\n",
      "Saved data for: https://arxiv.org/abs/2202.13426\n",
      "Processing: https://arxiv.org/abs/2202.13965\n",
      "Saved data for: https://arxiv.org/abs/2202.13965\n",
      "Processing: https://arxiv.org/abs/2203.00526\n",
      "Saved data for: https://arxiv.org/abs/2203.00526\n",
      "Processing: https://arxiv.org/abs/2203.00999\n",
      "Saved data for: https://arxiv.org/abs/2203.00999\n",
      "Processing: https://arxiv.org/abs/2203.01434\n",
      "Saved data for: https://arxiv.org/abs/2203.01434\n",
      "Processing: https://arxiv.org/abs/2203.07436\n",
      "Saved data for: https://arxiv.org/abs/2203.07436\n",
      "Processing: https://arxiv.org/abs/2203.10874\n",
      "Saved data for: https://arxiv.org/abs/2203.10874\n",
      "Processing: https://arxiv.org/abs/2203.11010\n",
      "Saved data for: https://arxiv.org/abs/2203.11010\n",
      "Processing: https://arxiv.org/abs/2203.11402\n",
      "Saved data for: https://arxiv.org/abs/2203.11402\n",
      "Processing: https://arxiv.org/abs/2203.11578\n",
      "Saved data for: https://arxiv.org/abs/2203.11578\n",
      "Processing: https://arxiv.org/abs/2203.13341\n",
      "Saved data for: https://arxiv.org/abs/2203.13341\n",
      "Processing: https://arxiv.org/abs/2203.16414\n",
      "Saved data for: https://arxiv.org/abs/2203.16414\n",
      "Processing: https://arxiv.org/abs/2203.17255\n",
      "Saved data for: https://arxiv.org/abs/2203.17255\n",
      "Processing: https://arxiv.org/abs/2204.01675\n",
      "Saved data for: https://arxiv.org/abs/2204.01675\n",
      "Processing: https://arxiv.org/abs/2204.03778\n",
      "Saved data for: https://arxiv.org/abs/2204.03778\n",
      "Processing: https://arxiv.org/abs/2204.05132\n",
      "Saved data for: https://arxiv.org/abs/2204.05132\n",
      "Processing: https://arxiv.org/abs/2204.07161\n",
      "Saved data for: https://arxiv.org/abs/2204.07161\n",
      "Processing: https://arxiv.org/abs/2204.07313\n",
      "Saved data for: https://arxiv.org/abs/2204.07313\n",
      "Processing: https://arxiv.org/abs/2204.09378\n",
      "Saved data for: https://arxiv.org/abs/2204.09378\n",
      "Processing: https://arxiv.org/abs/2204.12440\n",
      "Saved data for: https://arxiv.org/abs/2204.12440\n",
      "Processing: https://arxiv.org/abs/2204.14115\n",
      "Saved data for: https://arxiv.org/abs/2204.14115\n",
      "Processing: https://arxiv.org/abs/2205.02645\n",
      "Saved data for: https://arxiv.org/abs/2205.02645\n",
      "Processing: https://arxiv.org/abs/2205.03905\n",
      "Saved data for: https://arxiv.org/abs/2205.03905\n",
      "Processing: https://arxiv.org/abs/2205.03912\n",
      "Saved data for: https://arxiv.org/abs/2205.03912\n",
      "Processing: https://arxiv.org/abs/2205.04060\n",
      "Saved data for: https://arxiv.org/abs/2205.04060\n",
      "Processing: https://arxiv.org/abs/2205.04393\n",
      "Saved data for: https://arxiv.org/abs/2205.04393\n",
      "Processing: https://arxiv.org/abs/2205.05897\n",
      "Saved data for: https://arxiv.org/abs/2205.05897\n",
      "Processing: https://arxiv.org/abs/2205.07575\n",
      "Saved data for: https://arxiv.org/abs/2205.07575\n",
      "Processing: https://arxiv.org/abs/2205.08734\n",
      "Saved data for: https://arxiv.org/abs/2205.08734\n",
      "Processing: https://arxiv.org/abs/2205.08750\n",
      "Saved data for: https://arxiv.org/abs/2205.08750\n",
      "Processing: https://arxiv.org/abs/2205.09548\n",
      "Saved data for: https://arxiv.org/abs/2205.09548\n",
      "Processing: https://arxiv.org/abs/2205.10316\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " impossible to decode XFormObject /Im16\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved data for: https://arxiv.org/abs/2205.10316\n",
      "Processing: https://arxiv.org/abs/2205.10373\n",
      "Saved data for: https://arxiv.org/abs/2205.10373\n",
      "Processing: https://arxiv.org/abs/2205.12120\n",
      "Saved data for: https://arxiv.org/abs/2205.12120\n",
      "Processing: https://arxiv.org/abs/2205.13143\n",
      "Saved data for: https://arxiv.org/abs/2205.13143\n",
      "Processing: https://arxiv.org/abs/2205.13915\n",
      "Saved data for: https://arxiv.org/abs/2205.13915\n",
      "Processing: https://arxiv.org/abs/2205.14102\n",
      "Saved data for: https://arxiv.org/abs/2205.14102\n",
      "Processing: https://arxiv.org/abs/2206.01629\n",
      "Saved data for: https://arxiv.org/abs/2206.01629\n",
      "Processing: https://arxiv.org/abs/2206.02970\n",
      "Error saving data for https://arxiv.org/abs/2206.02970: 'utf-8' codec can't encode characters in position 34429-34430: surrogates not allowed\n",
      "Processing: https://arxiv.org/abs/2206.03026\n",
      "Saved data for: https://arxiv.org/abs/2206.03026\n",
      "Processing: https://arxiv.org/abs/2206.03269\n",
      "Saved data for: https://arxiv.org/abs/2206.03269\n",
      "Processing: https://arxiv.org/abs/2206.04234\n",
      "Saved data for: https://arxiv.org/abs/2206.04234\n",
      "Processing: https://arxiv.org/abs/2206.07921\n",
      "Saved data for: https://arxiv.org/abs/2206.07921\n",
      "Processing: https://arxiv.org/abs/2206.10501\n",
      "Error saving data for https://arxiv.org/abs/2206.10501: 'utf-8' codec can't encode character '\\ud835' in position 5942: surrogates not allowed\n",
      "Processing: https://arxiv.org/abs/2206.10814\n",
      "Saved data for: https://arxiv.org/abs/2206.10814\n",
      "Processing: https://arxiv.org/abs/2206.12508\n",
      "Saved data for: https://arxiv.org/abs/2206.12508\n",
      "Processing: https://arxiv.org/abs/2206.12687\n",
      "Saved data for: https://arxiv.org/abs/2206.12687\n",
      "Processing: https://arxiv.org/abs/2206.14794\n",
      "Saved data for: https://arxiv.org/abs/2206.14794\n",
      "Processing: https://arxiv.org/abs/2206.15238\n",
      "Saved data for: https://arxiv.org/abs/2206.15238\n",
      "Processing: https://arxiv.org/abs/2207.00574\n",
      "Saved data for: https://arxiv.org/abs/2207.00574\n",
      "Processing: https://arxiv.org/abs/2207.00982\n",
      "Saved data for: https://arxiv.org/abs/2207.00982\n",
      "Processing: https://arxiv.org/abs/2207.01581\n",
      "Saved data for: https://arxiv.org/abs/2207.01581\n",
      "Processing: https://arxiv.org/abs/2207.02320\n",
      "Saved data for: https://arxiv.org/abs/2207.02320\n",
      "Processing: https://arxiv.org/abs/2207.02880\n",
      "Saved data for: https://arxiv.org/abs/2207.02880\n",
      "Processing: https://arxiv.org/abs/2207.03717\n",
      "Saved data for: https://arxiv.org/abs/2207.03717\n",
      "Processing: https://arxiv.org/abs/2207.07682\n",
      "Saved data for: https://arxiv.org/abs/2207.07682\n",
      "Processing: https://arxiv.org/abs/2207.08277\n",
      "Saved data for: https://arxiv.org/abs/2207.08277\n",
      "Processing: https://arxiv.org/abs/2207.08883\n",
      "Saved data for: https://arxiv.org/abs/2207.08883\n",
      "Processing: https://arxiv.org/abs/2207.09551\n",
      "Saved data for: https://arxiv.org/abs/2207.09551\n",
      "Processing: https://arxiv.org/abs/2207.10571\n",
      "Saved data for: https://arxiv.org/abs/2207.10571\n",
      "Processing: https://arxiv.org/abs/2207.11282\n",
      "Saved data for: https://arxiv.org/abs/2207.11282\n",
      "Processing: https://arxiv.org/abs/2207.11414\n",
      "Saved data for: https://arxiv.org/abs/2207.11414\n",
      "Processing: https://arxiv.org/abs/2207.12328\n",
      "Saved data for: https://arxiv.org/abs/2207.12328\n",
      "Processing: https://arxiv.org/abs/2208.00637\n",
      "Saved data for: https://arxiv.org/abs/2208.00637\n",
      "Processing: https://arxiv.org/abs/2208.00778\n",
      "Saved data for: https://arxiv.org/abs/2208.00778\n",
      "Processing: https://arxiv.org/abs/2208.02657\n",
      "Saved data for: https://arxiv.org/abs/2208.02657\n",
      "Processing: https://arxiv.org/abs/2208.02726\n",
      "Saved data for: https://arxiv.org/abs/2208.02726\n",
      "Processing: https://arxiv.org/abs/2208.04852\n",
      "Saved data for: https://arxiv.org/abs/2208.04852\n",
      "Processing: https://arxiv.org/abs/2208.05515\n",
      "Saved data for: https://arxiv.org/abs/2208.05515\n",
      "Processing: https://arxiv.org/abs/2208.06348\n",
      "Saved data for: https://arxiv.org/abs/2208.06348\n",
      "Processing: https://arxiv.org/abs/2208.07397\n",
      "Saved data for: https://arxiv.org/abs/2208.07397\n",
      "Processing: https://arxiv.org/abs/2208.10230\n",
      "Saved data for: https://arxiv.org/abs/2208.10230\n",
      "Processing: https://arxiv.org/abs/2208.10962\n",
      "Saved data for: https://arxiv.org/abs/2208.10962\n",
      "Processing: https://arxiv.org/abs/2209.00950\n",
      "Saved data for: https://arxiv.org/abs/2209.00950\n",
      "Processing: https://arxiv.org/abs/2209.02044\n",
      "Saved data for: https://arxiv.org/abs/2209.02044\n",
      "Processing: https://arxiv.org/abs/2209.03324\n",
      "Saved data for: https://arxiv.org/abs/2209.03324\n",
      "Processing: https://arxiv.org/abs/2209.03924\n",
      "Saved data for: https://arxiv.org/abs/2209.03924\n",
      "Processing: https://arxiv.org/abs/2209.05935\n",
      "Saved data for: https://arxiv.org/abs/2209.05935\n",
      "Processing: https://arxiv.org/abs/2209.06862\n",
      "Saved data for: https://arxiv.org/abs/2209.06862\n",
      "Processing: https://arxiv.org/abs/2209.06948\n",
      "Saved data for: https://arxiv.org/abs/2209.06948\n",
      "Processing: https://arxiv.org/abs/2209.08066\n",
      "Saved data for: https://arxiv.org/abs/2209.08066\n",
      "Processing: https://arxiv.org/abs/2209.09075\n",
      "Saved data for: https://arxiv.org/abs/2209.09075\n",
      "Processing: https://arxiv.org/abs/2209.10545\n",
      "Saved data for: https://arxiv.org/abs/2209.10545\n",
      "Processing: https://arxiv.org/abs/2209.10837\n",
      "Saved data for: https://arxiv.org/abs/2209.10837\n",
      "Processing: https://arxiv.org/abs/2209.11737\n",
      "Saved data for: https://arxiv.org/abs/2209.11737\n",
      "Processing: https://arxiv.org/abs/2209.12635\n",
      "Saved data for: https://arxiv.org/abs/2209.12635\n",
      "Processing: https://arxiv.org/abs/2209.12994\n",
      "Saved data for: https://arxiv.org/abs/2209.12994\n",
      "Processing: https://arxiv.org/abs/2209.13371\n",
      "Saved data for: https://arxiv.org/abs/2209.13371\n",
      "Processing: https://arxiv.org/abs/2209.13865\n",
      "Saved data for: https://arxiv.org/abs/2209.13865\n",
      "Processing: https://arxiv.org/abs/2209.13886\n",
      "Saved data for: https://arxiv.org/abs/2209.13886\n",
      "Processing: https://arxiv.org/abs/2209.14297\n",
      "Saved data for: https://arxiv.org/abs/2209.14297\n",
      "Processing: https://arxiv.org/abs/2209.14683\n",
      "Saved data for: https://arxiv.org/abs/2209.14683\n",
      "Processing: https://arxiv.org/abs/2210.00392\n",
      "Saved data for: https://arxiv.org/abs/2210.00392\n",
      "Processing: https://arxiv.org/abs/2210.00802\n",
      "Saved data for: https://arxiv.org/abs/2210.00802\n",
      "Processing: https://arxiv.org/abs/2210.02116\n",
      "Saved data for: https://arxiv.org/abs/2210.02116\n",
      "Processing: https://arxiv.org/abs/2210.02366\n",
      "Saved data for: https://arxiv.org/abs/2210.02366\n",
      "Processing: https://arxiv.org/abs/2210.03639\n",
      "Saved data for: https://arxiv.org/abs/2210.03639\n",
      "Processing: https://arxiv.org/abs/2210.04627\n",
      "Saved data for: https://arxiv.org/abs/2210.04627\n",
      "Processing: https://arxiv.org/abs/2210.05988\n",
      "Saved data for: https://arxiv.org/abs/2210.05988\n",
      "Processing: https://arxiv.org/abs/2210.06891\n",
      "Saved data for: https://arxiv.org/abs/2210.06891\n",
      "Processing: https://arxiv.org/abs/2210.07310\n",
      "Saved data for: https://arxiv.org/abs/2210.07310\n",
      "Processing: https://arxiv.org/abs/2210.09574\n",
      "Saved data for: https://arxiv.org/abs/2210.09574\n",
      "Processing: https://arxiv.org/abs/2210.09986\n",
      "Saved data for: https://arxiv.org/abs/2210.09986\n",
      "Processing: https://arxiv.org/abs/2210.11364\n",
      "Saved data for: https://arxiv.org/abs/2210.11364\n",
      "Processing: https://arxiv.org/abs/2210.11474\n",
      "Saved data for: https://arxiv.org/abs/2210.11474\n",
      "Processing: https://arxiv.org/abs/2210.11638\n",
      "Saved data for: https://arxiv.org/abs/2210.11638\n",
      "Processing: https://arxiv.org/abs/2210.12064\n",
      "Saved data for: https://arxiv.org/abs/2210.12064\n",
      "Processing: https://arxiv.org/abs/2210.12072\n",
      "Saved data for: https://arxiv.org/abs/2210.12072\n",
      "Processing: https://arxiv.org/abs/2210.12204\n",
      "Saved data for: https://arxiv.org/abs/2210.12204\n",
      "Processing: https://arxiv.org/abs/2210.13004\n",
      "Saved data for: https://arxiv.org/abs/2210.13004\n",
      "Processing: https://arxiv.org/abs/2210.13113\n",
      "Saved data for: https://arxiv.org/abs/2210.13113\n",
      "Processing: https://arxiv.org/abs/2210.13695\n",
      "Saved data for: https://arxiv.org/abs/2210.13695\n",
      "Processing: https://arxiv.org/abs/2210.17270\n",
      "Saved data for: https://arxiv.org/abs/2210.17270\n",
      "Processing: https://arxiv.org/abs/2210.17309\n",
      "Saved data for: https://arxiv.org/abs/2210.17309\n",
      "Processing: https://arxiv.org/abs/2211.00581\n",
      "Saved data for: https://arxiv.org/abs/2211.00581\n",
      "Processing: https://arxiv.org/abs/2211.00646\n",
      "Saved data for: https://arxiv.org/abs/2211.00646\n",
      "Processing: https://arxiv.org/abs/2211.01867\n",
      "Saved data for: https://arxiv.org/abs/2211.01867\n",
      "Processing: https://arxiv.org/abs/2211.02407\n",
      "Saved data for: https://arxiv.org/abs/2211.02407\n",
      "Processing: https://arxiv.org/abs/2211.02523\n",
      "Saved data for: https://arxiv.org/abs/2211.02523\n",
      "Processing: https://arxiv.org/abs/2211.03079\n",
      "Saved data for: https://arxiv.org/abs/2211.03079\n",
      "Processing: https://arxiv.org/abs/2211.04125\n",
      "Saved data for: https://arxiv.org/abs/2211.04125\n",
      "Processing: https://arxiv.org/abs/2211.06302\n",
      "Saved data for: https://arxiv.org/abs/2211.06302\n",
      "Processing: https://arxiv.org/abs/2211.06928\n",
      "Saved data for: https://arxiv.org/abs/2211.06928\n",
      "Processing: https://arxiv.org/abs/2211.07393\n",
      "Saved data for: https://arxiv.org/abs/2211.07393\n",
      "Processing: https://arxiv.org/abs/2211.07869\n",
      "Saved data for: https://arxiv.org/abs/2211.07869\n",
      "Processing: https://arxiv.org/abs/2211.08157\n",
      "Saved data for: https://arxiv.org/abs/2211.08157\n",
      "Processing: https://arxiv.org/abs/2211.08527\n",
      "Saved data for: https://arxiv.org/abs/2211.08527\n",
      "Processing: https://arxiv.org/abs/2211.08618\n",
      "Saved data for: https://arxiv.org/abs/2211.08618\n",
      "Processing: https://arxiv.org/abs/2211.09005\n",
      "Saved data for: https://arxiv.org/abs/2211.09005\n",
      "Processing: https://arxiv.org/abs/2211.10542\n",
      "Saved data for: https://arxiv.org/abs/2211.10542\n",
      "Processing: https://arxiv.org/abs/2211.11169\n",
      "Saved data for: https://arxiv.org/abs/2211.11169\n",
      "Processing: https://arxiv.org/abs/2211.11214\n",
      "Saved data for: https://arxiv.org/abs/2211.11214\n",
      "Processing: https://arxiv.org/abs/2211.11965\n",
      "Saved data for: https://arxiv.org/abs/2211.11965\n",
      "Processing: https://arxiv.org/abs/2211.13507\n",
      "Saved data for: https://arxiv.org/abs/2211.13507\n",
      "Processing: https://arxiv.org/abs/2211.13704\n",
      "Saved data for: https://arxiv.org/abs/2211.13704\n",
      "Processing: https://arxiv.org/abs/2211.13943\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "unknown widths : \n",
      "[0, IndirectObject(645, 0, 5128143648)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(652, 0, 5128143648)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(645, 0, 5128143648)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(655, 0, 5128143648)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(657, 0, 5128143648)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(659, 0, 5128143648)]\n",
      "unknown widths : \n",
      "[0, IndirectObject(652, 0, 5128143648)]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Error saving data for https://arxiv.org/abs/2211.13943: 'utf-8' codec can't encode character '\\ud835' in position 12925: surrogates not allowed\n",
      "Processing: https://arxiv.org/abs/2211.14396\n",
      "Saved data for: https://arxiv.org/abs/2211.14396\n",
      "Processing: https://arxiv.org/abs/2211.15015\n",
      "Saved data for: https://arxiv.org/abs/2211.15015\n",
      "Processing: https://arxiv.org/abs/2211.15963\n",
      "Saved data for: https://arxiv.org/abs/2211.15963\n",
      "Processing: https://arxiv.org/abs/2212.00386\n",
      "Saved data for: https://arxiv.org/abs/2212.00386\n",
      "Processing: https://arxiv.org/abs/2212.00596\n",
      "Saved data for: https://arxiv.org/abs/2212.00596\n",
      "Processing: https://arxiv.org/abs/2212.00829\n",
      "Saved data for: https://arxiv.org/abs/2212.00829\n",
      "Processing: https://arxiv.org/abs/2212.01191\n",
      "Saved data for: https://arxiv.org/abs/2212.01191\n",
      "Processing: https://arxiv.org/abs/2212.02154\n",
      "Saved data for: https://arxiv.org/abs/2212.02154\n",
      "Processing: https://arxiv.org/abs/2212.02714\n",
      "Saved data for: https://arxiv.org/abs/2212.02714\n",
      "Processing: https://arxiv.org/abs/2212.03854\n",
      "Saved data for: https://arxiv.org/abs/2212.03854\n",
      "Processing: https://arxiv.org/abs/2212.04535\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "FloatObject (b'0.000-10332388') invalid; use 0.0 instead\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved data for: https://arxiv.org/abs/2212.04535\n",
      "Processing: https://arxiv.org/abs/2212.04740\n",
      "Saved data for: https://arxiv.org/abs/2212.04740\n",
      "Processing: https://arxiv.org/abs/2212.04937\n",
      "Saved data for: https://arxiv.org/abs/2212.04937\n",
      "Processing: https://arxiv.org/abs/2212.04953\n",
      "Saved data for: https://arxiv.org/abs/2212.04953\n",
      "Processing: https://arxiv.org/abs/2212.05354\n",
      "Saved data for: https://arxiv.org/abs/2212.05354\n",
      "Processing: https://arxiv.org/abs/2212.06361\n",
      "Saved data for: https://arxiv.org/abs/2212.06361\n",
      "Processing: https://arxiv.org/abs/2212.07225\n",
      "Saved data for: https://arxiv.org/abs/2212.07225\n",
      "Processing: https://arxiv.org/abs/2212.08145\n",
      "Saved data for: https://arxiv.org/abs/2212.08145\n",
      "Processing: https://arxiv.org/abs/2212.09179\n",
      "Saved data for: https://arxiv.org/abs/2212.09179\n",
      "Processing: https://arxiv.org/abs/2212.09832\n",
      "Saved data for: https://arxiv.org/abs/2212.09832\n",
      "Processing: https://arxiv.org/abs/2212.10249\n",
      "Saved data for: https://arxiv.org/abs/2212.10249\n",
      "Processing: https://arxiv.org/abs/2212.10789\n",
      "Saved data for: https://arxiv.org/abs/2212.10789\n",
      "Processing: https://arxiv.org/abs/2212.11367\n",
      "Saved data for: https://arxiv.org/abs/2212.11367\n",
      "Processing: https://arxiv.org/abs/2212.11997\n",
      "Saved data for: https://arxiv.org/abs/2212.11997\n",
      "Processing: https://arxiv.org/abs/2212.13443\n",
      "Saved data for: https://arxiv.org/abs/2212.13443\n",
      "Processing: https://arxiv.org/abs/2212.14041\n",
      "Saved data for: https://arxiv.org/abs/2212.14041\n",
      "Processing: https://arxiv.org/abs/2212.14445\n",
      "Saved data for: https://arxiv.org/abs/2212.14445\n",
      "Processing: https://arxiv.org/abs/2301.00148\n",
      "Saved data for: https://arxiv.org/abs/2301.00148\n",
      "Processing: https://arxiv.org/abs/2301.01445\n",
      "Saved data for: https://arxiv.org/abs/2301.01445\n",
      "Processing: https://arxiv.org/abs/2301.01642\n",
      "Saved data for: https://arxiv.org/abs/2301.01642\n",
      "Processing: https://arxiv.org/abs/2301.01829\n",
      "Saved data for: https://arxiv.org/abs/2301.01829\n",
      "Processing: https://arxiv.org/abs/2301.01847\n",
      "Saved data for: https://arxiv.org/abs/2301.01847\n",
      "Processing: https://arxiv.org/abs/2301.02607\n",
      "Saved data for: https://arxiv.org/abs/2301.02607\n",
      "Processing: https://arxiv.org/abs/2301.02623\n",
      "Saved data for: https://arxiv.org/abs/2301.02623\n",
      "Processing: https://arxiv.org/abs/2301.04091\n",
      "Saved data for: https://arxiv.org/abs/2301.04091\n",
      "Processing: https://arxiv.org/abs/2301.04355\n",
      "Saved data for: https://arxiv.org/abs/2301.04355\n",
      "Processing: https://arxiv.org/abs/2301.05917\n",
      "Saved data for: https://arxiv.org/abs/2301.05917\n",
      "Processing: https://arxiv.org/abs/2301.07016\n",
      "Saved data for: https://arxiv.org/abs/2301.07016\n",
      "Processing: https://arxiv.org/abs/2301.07175\n",
      "Saved data for: https://arxiv.org/abs/2301.07175\n",
      "Processing: https://arxiv.org/abs/2301.07363\n",
      "Saved data for: https://arxiv.org/abs/2301.07363\n",
      "Processing: https://arxiv.org/abs/2301.07386\n",
      "Saved data for: https://arxiv.org/abs/2301.07386\n",
      "Processing: https://arxiv.org/abs/2301.07394\n",
      "Saved data for: https://arxiv.org/abs/2301.07394\n",
      "Processing: https://arxiv.org/abs/2301.07599\n",
      "Saved data for: https://arxiv.org/abs/2301.07599\n",
      "Processing: https://arxiv.org/abs/2301.08023\n",
      "Saved data for: https://arxiv.org/abs/2301.08023\n",
      "Processing: https://arxiv.org/abs/2301.10748\n",
      "Saved data for: https://arxiv.org/abs/2301.10748\n",
      "Processing: https://arxiv.org/abs/2301.10774\n",
      "Saved data for: https://arxiv.org/abs/2301.10774\n",
      "Processing: https://arxiv.org/abs/2301.11669\n",
      "Saved data for: https://arxiv.org/abs/2301.11669\n",
      "Processing: https://arxiv.org/abs/2301.11758\n",
      "Saved data for: https://arxiv.org/abs/2301.11758\n",
      "Processing: https://arxiv.org/abs/2301.11764\n",
      "Saved data for: https://arxiv.org/abs/2301.11764\n",
      "Processing: https://arxiv.org/abs/2302.01924\n",
      "Saved data for: https://arxiv.org/abs/2302.01924\n",
      "Processing: https://arxiv.org/abs/2302.02488\n",
      "Saved data for: https://arxiv.org/abs/2302.02488\n",
      "Processing: https://arxiv.org/abs/2302.02520\n",
      "Saved data for: https://arxiv.org/abs/2302.02520\n",
      "Processing: https://arxiv.org/abs/2302.02942\n",
      "Saved data for: https://arxiv.org/abs/2302.02942\n",
      "Processing: https://arxiv.org/abs/2302.03038\n",
      "Saved data for: https://arxiv.org/abs/2302.03038\n",
      "Processing: https://arxiv.org/abs/2302.03210\n",
      "Saved data for: https://arxiv.org/abs/2302.03210\n",
      "Processing: https://arxiv.org/abs/2302.04313\n",
      "Saved data for: https://arxiv.org/abs/2302.04313\n",
      "Processing: https://arxiv.org/abs/2302.04611\n",
      "Saved data for: https://arxiv.org/abs/2302.04611\n",
      "Processing: https://arxiv.org/abs/2302.04939\n",
      "Saved data for: https://arxiv.org/abs/2302.04939\n",
      "Processing: https://arxiv.org/abs/2302.05281\n",
      "Saved data for: https://arxiv.org/abs/2302.05281\n",
      "Processing: https://arxiv.org/abs/2302.05562\n",
      "Saved data for: https://arxiv.org/abs/2302.05562\n",
      "Processing: https://arxiv.org/abs/2302.06089\n",
      "Saved data for: https://arxiv.org/abs/2302.06089\n",
      "Processing: https://arxiv.org/abs/2302.06120\n",
      "Saved data for: https://arxiv.org/abs/2302.06120\n",
      "Processing: https://arxiv.org/abs/2302.06573\n",
      "Saved data for: https://arxiv.org/abs/2302.06573\n",
      "Processing: https://arxiv.org/abs/2302.06842\n",
      "Saved data for: https://arxiv.org/abs/2302.06842\n",
      "Processing: https://arxiv.org/abs/2302.07005\n",
      "Saved data for: https://arxiv.org/abs/2302.07005\n",
      "Processing: https://arxiv.org/abs/2302.07868\n",
      "Saved data for: https://arxiv.org/abs/2302.07868\n",
      "Processing: https://arxiv.org/abs/2302.08062\n",
      "Saved data for: https://arxiv.org/abs/2302.08062\n",
      "Processing: https://arxiv.org/abs/2302.08695\n",
      "Saved data for: https://arxiv.org/abs/2302.08695\n",
      "Processing: https://arxiv.org/abs/2302.08985\n",
      "Saved data for: https://arxiv.org/abs/2302.08985\n",
      "Processing: https://arxiv.org/abs/2302.09109\n",
      "Saved data for: https://arxiv.org/abs/2302.09109\n",
      "Processing: https://arxiv.org/abs/2302.09151\n",
      "Saved data for: https://arxiv.org/abs/2302.09151\n",
      "Processing: https://arxiv.org/abs/2302.09402\n",
      "Saved data for: https://arxiv.org/abs/2302.09402\n",
      "Processing: https://arxiv.org/abs/2302.09445\n",
      "Saved data for: https://arxiv.org/abs/2302.09445\n",
      "Processing: https://arxiv.org/abs/2302.10163\n",
      "Saved data for: https://arxiv.org/abs/2302.10163\n",
      "Processing: https://arxiv.org/abs/2302.10662\n",
      "Saved data for: https://arxiv.org/abs/2302.10662\n",
      "Processing: https://arxiv.org/abs/2302.11351\n",
      "Saved data for: https://arxiv.org/abs/2302.11351\n",
      "Processing: https://arxiv.org/abs/2302.11419\n",
      "Saved data for: https://arxiv.org/abs/2302.11419\n",
      "Processing: https://arxiv.org/abs/2302.11681\n",
      "Saved data for: https://arxiv.org/abs/2302.11681\n",
      "Processing: https://arxiv.org/abs/2302.12177\n",
      "Saved data for: https://arxiv.org/abs/2302.12177\n",
      "Processing: https://arxiv.org/abs/2302.12542\n",
      "Saved data for: https://arxiv.org/abs/2302.12542\n",
      "Processing: https://arxiv.org/abs/2302.12838\n",
      "Saved data for: https://arxiv.org/abs/2302.12838\n",
      "Processing: https://arxiv.org/abs/2302.12968\n",
      "Saved data for: https://arxiv.org/abs/2302.12968\n",
      "Processing: https://arxiv.org/abs/2302.13268\n",
      "Saved data for: https://arxiv.org/abs/2302.13268\n",
      "Processing: https://arxiv.org/abs/2302.13358\n",
      "Saved data for: https://arxiv.org/abs/2302.13358\n",
      "Processing: https://arxiv.org/abs/2302.13385\n",
      "Saved data for: https://arxiv.org/abs/2302.13385\n",
      "Processing: https://arxiv.org/abs/2302.13440\n",
      "Saved data for: https://arxiv.org/abs/2302.13440\n",
      "Processing: https://arxiv.org/abs/2302.13711\n",
      "Saved data for: https://arxiv.org/abs/2302.13711\n",
      "Processing: https://arxiv.org/abs/2302.14700\n",
      "Saved data for: https://arxiv.org/abs/2302.14700\n",
      "Processing: https://arxiv.org/abs/2303.00864\n",
      "Saved data for: https://arxiv.org/abs/2303.00864\n",
      "Processing: https://arxiv.org/abs/2303.01444\n",
      "Saved data for: https://arxiv.org/abs/2303.01444\n",
      "Processing: https://arxiv.org/abs/2303.01643\n",
      "Saved data for: https://arxiv.org/abs/2303.01643\n",
      "Processing: https://arxiv.org/abs/2303.01650\n",
      "Saved data for: https://arxiv.org/abs/2303.01650\n",
      "Processing: https://arxiv.org/abs/2303.01653\n",
      "Saved data for: https://arxiv.org/abs/2303.01653\n",
      "Processing: https://arxiv.org/abs/2303.02079\n",
      "Saved data for: https://arxiv.org/abs/2303.02079\n",
      "Processing: https://arxiv.org/abs/2303.02195\n",
      "Saved data for: https://arxiv.org/abs/2303.02195\n",
      "Processing: https://arxiv.org/abs/2303.02529\n",
      "Saved data for: https://arxiv.org/abs/2303.02529\n",
      "Processing: https://arxiv.org/abs/2303.02981\n",
      "Saved data for: https://arxiv.org/abs/2303.02981\n",
      "Processing: https://arxiv.org/abs/2303.03660\n",
      "Saved data for: https://arxiv.org/abs/2303.03660\n",
      "Processing: https://arxiv.org/abs/2303.03960\n",
      "Saved data for: https://arxiv.org/abs/2303.03960\n",
      "Processing: https://arxiv.org/abs/2303.04467\n",
      "Saved data for: https://arxiv.org/abs/2303.04467\n",
      "Processing: https://arxiv.org/abs/2303.04902\n",
      "Saved data for: https://arxiv.org/abs/2303.04902\n",
      "Processing: https://arxiv.org/abs/2303.05174\n",
      "Saved data for: https://arxiv.org/abs/2303.05174\n",
      "Processing: https://arxiv.org/abs/2303.05287\n",
      "Saved data for: https://arxiv.org/abs/2303.05287\n",
      "Processing: https://arxiv.org/abs/2303.05390\n",
      "Saved data for: https://arxiv.org/abs/2303.05390\n",
      "Processing: https://arxiv.org/abs/2303.05561\n",
      "Saved data for: https://arxiv.org/abs/2303.05561\n",
      "Processing: https://arxiv.org/abs/2303.05585\n",
      "Saved data for: https://arxiv.org/abs/2303.05585\n",
      "Processing: https://arxiv.org/abs/2303.05917\n",
      "Saved data for: https://arxiv.org/abs/2303.05917\n",
      "Processing: https://arxiv.org/abs/2303.06050\n",
      "Saved data for: https://arxiv.org/abs/2303.06050\n",
      "Processing: https://arxiv.org/abs/2303.06183\n",
      "Saved data for: https://arxiv.org/abs/2303.06183\n",
      "Processing: https://arxiv.org/abs/2303.06965\n",
      "Saved data for: https://arxiv.org/abs/2303.06965\n",
      "Processing: https://arxiv.org/abs/2303.07534\n",
      "Saved data for: https://arxiv.org/abs/2303.07534\n",
      "Processing: https://arxiv.org/abs/2303.07540\n",
      "Saved data for: https://arxiv.org/abs/2303.07540\n",
      "Processing: https://arxiv.org/abs/2303.08107\n",
      "Saved data for: https://arxiv.org/abs/2303.08107\n",
      "Processing: https://arxiv.org/abs/2303.08158\n",
      "Saved data for: https://arxiv.org/abs/2303.08158\n",
      "Processing: https://arxiv.org/abs/2303.09094\n",
      "Saved data for: https://arxiv.org/abs/2303.09094\n",
      "Processing: https://arxiv.org/abs/2303.09666\n",
      "Saved data for: https://arxiv.org/abs/2303.09666\n",
      "Processing: https://arxiv.org/abs/2303.09669\n",
      "Saved data for: https://arxiv.org/abs/2303.09669\n",
      "Processing: https://arxiv.org/abs/2303.09874\n",
      "Saved data for: https://arxiv.org/abs/2303.09874\n",
      "Processing: https://arxiv.org/abs/2303.09906\n",
      "Saved data for: https://arxiv.org/abs/2303.09906\n",
      "Processing: https://arxiv.org/abs/2303.10724\n",
      "Saved data for: https://arxiv.org/abs/2303.10724\n",
      "Processing: https://arxiv.org/abs/2303.10996\n",
      "Saved data for: https://arxiv.org/abs/2303.10996\n",
      "Processing: https://arxiv.org/abs/2303.11342\n",
      "Saved data for: https://arxiv.org/abs/2303.11342\n",
      "Processing: https://arxiv.org/abs/2303.11475\n",
      "Saved data for: https://arxiv.org/abs/2303.11475\n",
      "Processing: https://arxiv.org/abs/2303.11593\n",
      "Saved data for: https://arxiv.org/abs/2303.11593\n",
      "Processing: https://arxiv.org/abs/2303.11783\n",
      "Saved data for: https://arxiv.org/abs/2303.11783\n",
      "Processing: https://arxiv.org/abs/2303.11833\n",
      "Saved data for: https://arxiv.org/abs/2303.11833\n",
      "Processing: https://arxiv.org/abs/2303.11909\n",
      "Saved data for: https://arxiv.org/abs/2303.11909\n",
      "Processing: https://arxiv.org/abs/2303.12654\n",
      "Saved data for: https://arxiv.org/abs/2303.12654\n",
      "Processing: https://arxiv.org/abs/2303.12806\n",
      "Saved data for: https://arxiv.org/abs/2303.12806\n",
      "Processing: https://arxiv.org/abs/2303.13332\n",
      "Saved data for: https://arxiv.org/abs/2303.13332\n",
      "Processing: https://arxiv.org/abs/2303.13528\n",
      "Saved data for: https://arxiv.org/abs/2303.13528\n",
      "Processing: https://arxiv.org/abs/2303.13608\n",
      "Saved data for: https://arxiv.org/abs/2303.13608\n",
      "Processing: https://arxiv.org/abs/2303.13702\n",
      "Saved data for: https://arxiv.org/abs/2303.13702\n",
      "Processing: https://arxiv.org/abs/2303.13848\n",
      "Saved data for: https://arxiv.org/abs/2303.13848\n",
      "Processing: https://arxiv.org/abs/2303.14159\n",
      "Saved data for: https://arxiv.org/abs/2303.14159\n",
      "Processing: https://arxiv.org/abs/2303.14995\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[42], line 1\u001b[0m\n\u001b[0;32m----> 1\u001b[0m results \u001b[38;5;241m=\u001b[39m process_papers_incrementally(output)\n",
      "Cell \u001b[0;32mIn[41], line 83\u001b[0m, in \u001b[0;36mprocess_papers_incrementally\u001b[0;34m(output, csv_path)\u001b[0m\n\u001b[1;32m     80\u001b[0m     \u001b[38;5;28;01mcontinue\u001b[39;00m\n\u001b[1;32m     82\u001b[0m \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;124mf\u001b[39m\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mProcessing: \u001b[39m\u001b[38;5;132;01m{\u001b[39;00murl\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m\"\u001b[39m)\n\u001b[0;32m---> 83\u001b[0m data \u001b[38;5;241m=\u001b[39m fetch_paper_data(url)\n\u001b[1;32m     85\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m data:\n\u001b[1;32m     86\u001b[0m     results\u001b[38;5;241m.\u001b[39mappend(data)\n",
      "Cell \u001b[0;32mIn[41], line 18\u001b[0m, in \u001b[0;36mfetch_paper_data\u001b[0;34m(url)\u001b[0m\n\u001b[1;32m     12\u001b[0m headers \u001b[38;5;241m=\u001b[39m {\n\u001b[1;32m     13\u001b[0m     \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mUser-Agent\u001b[39m\u001b[38;5;124m'\u001b[39m: \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mEducational Bot (na648@drexel.edu)\u001b[39m\u001b[38;5;124m'\u001b[39m,\n\u001b[1;32m     14\u001b[0m     \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mAccept\u001b[39m\u001b[38;5;124m'\u001b[39m: \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mtext/html,application/xhtml+xml,application/xml;q=0.9\u001b[39m\u001b[38;5;124m'\u001b[39m\n\u001b[1;32m     15\u001b[0m }\n\u001b[1;32m     17\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m---> 18\u001b[0m     response \u001b[38;5;241m=\u001b[39m requests\u001b[38;5;241m.\u001b[39mget(url, headers\u001b[38;5;241m=\u001b[39mheaders)\n\u001b[1;32m     19\u001b[0m     response\u001b[38;5;241m.\u001b[39mraise_for_status()\n\u001b[1;32m     21\u001b[0m     soup \u001b[38;5;241m=\u001b[39m BeautifulSoup(response\u001b[38;5;241m.\u001b[39mtext, \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mhtml.parser\u001b[39m\u001b[38;5;124m'\u001b[39m)\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/site-packages/requests/api.py:73\u001b[0m, in \u001b[0;36mget\u001b[0;34m(url, params, **kwargs)\u001b[0m\n\u001b[1;32m     62\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mget\u001b[39m(url, params\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs):\n\u001b[1;32m     63\u001b[0m \u001b[38;5;250m    \u001b[39m\u001b[38;5;124mr\u001b[39m\u001b[38;5;124;03m\"\"\"Sends a GET request.\u001b[39;00m\n\u001b[1;32m     64\u001b[0m \n\u001b[1;32m     65\u001b[0m \u001b[38;5;124;03m    :param url: URL for the new :class:`Request` object.\u001b[39;00m\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     70\u001b[0m \u001b[38;5;124;03m    :rtype: requests.Response\u001b[39;00m\n\u001b[1;32m     71\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[0;32m---> 73\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m request(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mget\u001b[39m\u001b[38;5;124m\"\u001b[39m, url, params\u001b[38;5;241m=\u001b[39mparams, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/site-packages/requests/api.py:59\u001b[0m, in \u001b[0;36mrequest\u001b[0;34m(method, url, **kwargs)\u001b[0m\n\u001b[1;32m     55\u001b[0m \u001b[38;5;66;03m# By using the 'with' statement we are sure the session is closed, thus we\u001b[39;00m\n\u001b[1;32m     56\u001b[0m \u001b[38;5;66;03m# avoid leaving sockets open which can trigger a ResourceWarning in some\u001b[39;00m\n\u001b[1;32m     57\u001b[0m \u001b[38;5;66;03m# cases, and look like a memory leak in others.\u001b[39;00m\n\u001b[1;32m     58\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m sessions\u001b[38;5;241m.\u001b[39mSession() \u001b[38;5;28;01mas\u001b[39;00m session:\n\u001b[0;32m---> 59\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m session\u001b[38;5;241m.\u001b[39mrequest(method\u001b[38;5;241m=\u001b[39mmethod, url\u001b[38;5;241m=\u001b[39murl, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/site-packages/requests/sessions.py:589\u001b[0m, in \u001b[0;36mSession.request\u001b[0;34m(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)\u001b[0m\n\u001b[1;32m    584\u001b[0m send_kwargs \u001b[38;5;241m=\u001b[39m {\n\u001b[1;32m    585\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mtimeout\u001b[39m\u001b[38;5;124m\"\u001b[39m: timeout,\n\u001b[1;32m    586\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mallow_redirects\u001b[39m\u001b[38;5;124m\"\u001b[39m: allow_redirects,\n\u001b[1;32m    587\u001b[0m }\n\u001b[1;32m    588\u001b[0m send_kwargs\u001b[38;5;241m.\u001b[39mupdate(settings)\n\u001b[0;32m--> 589\u001b[0m resp \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msend(prep, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39msend_kwargs)\n\u001b[1;32m    591\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m resp\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/site-packages/requests/sessions.py:703\u001b[0m, in \u001b[0;36mSession.send\u001b[0;34m(self, request, **kwargs)\u001b[0m\n\u001b[1;32m    700\u001b[0m start \u001b[38;5;241m=\u001b[39m preferred_clock()\n\u001b[1;32m    702\u001b[0m \u001b[38;5;66;03m# Send the request\u001b[39;00m\n\u001b[0;32m--> 703\u001b[0m r \u001b[38;5;241m=\u001b[39m adapter\u001b[38;5;241m.\u001b[39msend(request, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n\u001b[1;32m    705\u001b[0m \u001b[38;5;66;03m# Total elapsed time of the request (approximately)\u001b[39;00m\n\u001b[1;32m    706\u001b[0m elapsed \u001b[38;5;241m=\u001b[39m preferred_clock() \u001b[38;5;241m-\u001b[39m start\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/site-packages/requests/adapters.py:667\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[0;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[1;32m    664\u001b[0m     timeout \u001b[38;5;241m=\u001b[39m TimeoutSauce(connect\u001b[38;5;241m=\u001b[39mtimeout, read\u001b[38;5;241m=\u001b[39mtimeout)\n\u001b[1;32m    666\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 667\u001b[0m     resp \u001b[38;5;241m=\u001b[39m conn\u001b[38;5;241m.\u001b[39murlopen(\n\u001b[1;32m    668\u001b[0m         method\u001b[38;5;241m=\u001b[39mrequest\u001b[38;5;241m.\u001b[39mmethod,\n\u001b[1;32m    669\u001b[0m         url\u001b[38;5;241m=\u001b[39murl,\n\u001b[1;32m    670\u001b[0m         body\u001b[38;5;241m=\u001b[39mrequest\u001b[38;5;241m.\u001b[39mbody,\n\u001b[1;32m    671\u001b[0m         headers\u001b[38;5;241m=\u001b[39mrequest\u001b[38;5;241m.\u001b[39mheaders,\n\u001b[1;32m    672\u001b[0m         redirect\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mFalse\u001b[39;00m,\n\u001b[1;32m    673\u001b[0m         assert_same_host\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mFalse\u001b[39;00m,\n\u001b[1;32m    674\u001b[0m         preload_content\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mFalse\u001b[39;00m,\n\u001b[1;32m    675\u001b[0m         decode_content\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mFalse\u001b[39;00m,\n\u001b[1;32m    676\u001b[0m         retries\u001b[38;5;241m=\u001b[39m\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mmax_retries,\n\u001b[1;32m    677\u001b[0m         timeout\u001b[38;5;241m=\u001b[39mtimeout,\n\u001b[1;32m    678\u001b[0m         chunked\u001b[38;5;241m=\u001b[39mchunked,\n\u001b[1;32m    679\u001b[0m     )\n\u001b[1;32m    681\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (ProtocolError, \u001b[38;5;167;01mOSError\u001b[39;00m) \u001b[38;5;28;01mas\u001b[39;00m err:\n\u001b[1;32m    682\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m(err, request\u001b[38;5;241m=\u001b[39mrequest)\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/site-packages/urllib3/connectionpool.py:789\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[0;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, preload_content, decode_content, **response_kw)\u001b[0m\n\u001b[1;32m    786\u001b[0m response_conn \u001b[38;5;241m=\u001b[39m conn \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m release_conn \u001b[38;5;28;01melse\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m\n\u001b[1;32m    788\u001b[0m \u001b[38;5;66;03m# Make the request on the HTTPConnection object\u001b[39;00m\n\u001b[0;32m--> 789\u001b[0m response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_make_request(\n\u001b[1;32m    790\u001b[0m     conn,\n\u001b[1;32m    791\u001b[0m     method,\n\u001b[1;32m    792\u001b[0m     url,\n\u001b[1;32m    793\u001b[0m     timeout\u001b[38;5;241m=\u001b[39mtimeout_obj,\n\u001b[1;32m    794\u001b[0m     body\u001b[38;5;241m=\u001b[39mbody,\n\u001b[1;32m    795\u001b[0m     headers\u001b[38;5;241m=\u001b[39mheaders,\n\u001b[1;32m    796\u001b[0m     chunked\u001b[38;5;241m=\u001b[39mchunked,\n\u001b[1;32m    797\u001b[0m     retries\u001b[38;5;241m=\u001b[39mretries,\n\u001b[1;32m    798\u001b[0m     response_conn\u001b[38;5;241m=\u001b[39mresponse_conn,\n\u001b[1;32m    799\u001b[0m     preload_content\u001b[38;5;241m=\u001b[39mpreload_content,\n\u001b[1;32m    800\u001b[0m     decode_content\u001b[38;5;241m=\u001b[39mdecode_content,\n\u001b[1;32m    801\u001b[0m     \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mresponse_kw,\n\u001b[1;32m    802\u001b[0m )\n\u001b[1;32m    804\u001b[0m \u001b[38;5;66;03m# Everything went great!\u001b[39;00m\n\u001b[1;32m    805\u001b[0m clean_exit \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mTrue\u001b[39;00m\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/site-packages/urllib3/connectionpool.py:536\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[0;34m(self, conn, method, url, body, headers, retries, timeout, chunked, response_conn, preload_content, decode_content, enforce_content_length)\u001b[0m\n\u001b[1;32m    534\u001b[0m \u001b[38;5;66;03m# Receive the response from the server\u001b[39;00m\n\u001b[1;32m    535\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 536\u001b[0m     response \u001b[38;5;241m=\u001b[39m conn\u001b[38;5;241m.\u001b[39mgetresponse()\n\u001b[1;32m    537\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (BaseSSLError, \u001b[38;5;167;01mOSError\u001b[39;00m) \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m    538\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_raise_timeout(err\u001b[38;5;241m=\u001b[39me, url\u001b[38;5;241m=\u001b[39murl, timeout_value\u001b[38;5;241m=\u001b[39mread_timeout)\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/site-packages/urllib3/connection.py:507\u001b[0m, in \u001b[0;36mHTTPConnection.getresponse\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    504\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mresponse\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m HTTPResponse\n\u001b[1;32m    506\u001b[0m \u001b[38;5;66;03m# Get the response from http.client.HTTPConnection\u001b[39;00m\n\u001b[0;32m--> 507\u001b[0m httplib_response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28msuper\u001b[39m()\u001b[38;5;241m.\u001b[39mgetresponse()\n\u001b[1;32m    509\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m    510\u001b[0m     assert_header_parsing(httplib_response\u001b[38;5;241m.\u001b[39mmsg)\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/http/client.py:1428\u001b[0m, in \u001b[0;36mHTTPConnection.getresponse\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   1426\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m   1427\u001b[0m     \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m-> 1428\u001b[0m         response\u001b[38;5;241m.\u001b[39mbegin()\n\u001b[1;32m   1429\u001b[0m     \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m:\n\u001b[1;32m   1430\u001b[0m         \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mclose()\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/http/client.py:331\u001b[0m, in \u001b[0;36mHTTPResponse.begin\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    329\u001b[0m \u001b[38;5;66;03m# read until we get a non-100 response\u001b[39;00m\n\u001b[1;32m    330\u001b[0m \u001b[38;5;28;01mwhile\u001b[39;00m \u001b[38;5;28;01mTrue\u001b[39;00m:\n\u001b[0;32m--> 331\u001b[0m     version, status, reason \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_read_status()\n\u001b[1;32m    332\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m status \u001b[38;5;241m!=\u001b[39m CONTINUE:\n\u001b[1;32m    333\u001b[0m         \u001b[38;5;28;01mbreak\u001b[39;00m\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/http/client.py:292\u001b[0m, in \u001b[0;36mHTTPResponse._read_status\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    291\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21m_read_status\u001b[39m(\u001b[38;5;28mself\u001b[39m):\n\u001b[0;32m--> 292\u001b[0m     line \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mstr\u001b[39m(\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mfp\u001b[38;5;241m.\u001b[39mreadline(_MAXLINE \u001b[38;5;241m+\u001b[39m \u001b[38;5;241m1\u001b[39m), \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124miso-8859-1\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    293\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mlen\u001b[39m(line) \u001b[38;5;241m>\u001b[39m _MAXLINE:\n\u001b[1;32m    294\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m LineTooLong(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mstatus line\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/socket.py:720\u001b[0m, in \u001b[0;36mSocketIO.readinto\u001b[0;34m(self, b)\u001b[0m\n\u001b[1;32m    718\u001b[0m \u001b[38;5;28;01mwhile\u001b[39;00m \u001b[38;5;28;01mTrue\u001b[39;00m:\n\u001b[1;32m    719\u001b[0m     \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 720\u001b[0m         \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_sock\u001b[38;5;241m.\u001b[39mrecv_into(b)\n\u001b[1;32m    721\u001b[0m     \u001b[38;5;28;01mexcept\u001b[39;00m timeout:\n\u001b[1;32m    722\u001b[0m         \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_timeout_occurred \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mTrue\u001b[39;00m\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/ssl.py:1251\u001b[0m, in \u001b[0;36mSSLSocket.recv_into\u001b[0;34m(self, buffer, nbytes, flags)\u001b[0m\n\u001b[1;32m   1247\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m flags \u001b[38;5;241m!=\u001b[39m \u001b[38;5;241m0\u001b[39m:\n\u001b[1;32m   1248\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mValueError\u001b[39;00m(\n\u001b[1;32m   1249\u001b[0m           \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mnon-zero flags not allowed in calls to recv_into() on \u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m\"\u001b[39m \u001b[38;5;241m%\u001b[39m\n\u001b[1;32m   1250\u001b[0m           \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m\u001b[38;5;18m__class__\u001b[39m)\n\u001b[0;32m-> 1251\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mread(nbytes, buffer)\n\u001b[1;32m   1252\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m   1253\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28msuper\u001b[39m()\u001b[38;5;241m.\u001b[39mrecv_into(buffer, nbytes, flags)\n",
      "File \u001b[0;32m/opt/anaconda3/lib/python3.12/ssl.py:1103\u001b[0m, in \u001b[0;36mSSLSocket.read\u001b[0;34m(self, len, buffer)\u001b[0m\n\u001b[1;32m   1101\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m   1102\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m buffer \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[0;32m-> 1103\u001b[0m         \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_sslobj\u001b[38;5;241m.\u001b[39mread(\u001b[38;5;28mlen\u001b[39m, buffer)\n\u001b[1;32m   1104\u001b[0m     \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[1;32m   1105\u001b[0m         \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_sslobj\u001b[38;5;241m.\u001b[39mread(\u001b[38;5;28mlen\u001b[39m)\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "results = process_papers_incrementally(output)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "base",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
